<!DOCTYPE html><html lang="en"><!--js-app.mako--><head>
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">

        <!-- Set meta description -->
        
        <meta name="description" content="Galaxy is a community-driven web-based analysis platform for life science research.">

        <title>
            Galaxy
        </title>
        
        <style type="text/css">:root, :host {
  --fa-font-solid: normal 900 1em/1 "Font Awesome 6 Solid";
  --fa-font-regular: normal 400 1em/1 "Font Awesome 6 Regular";
  --fa-font-light: normal 300 1em/1 "Font Awesome 6 Light";
  --fa-font-thin: normal 100 1em/1 "Font Awesome 6 Thin";
  --fa-font-duotone: normal 900 1em/1 "Font Awesome 6 Duotone";
  --fa-font-sharp-solid: normal 900 1em/1 "Font Awesome 6 Sharp";
  --fa-font-brands: normal 400 1em/1 "Font Awesome 6 Brands";
}

svg:not(:root).svg-inline--fa, svg:not(:host).svg-inline--fa {
  overflow: visible;
  box-sizing: content-box;
}

.svg-inline--fa {
  display: var(--fa-display, inline-block);
  height: 1em;
  overflow: visible;
  vertical-align: -0.125em;
}
.svg-inline--fa.fa-2xs {
  vertical-align: 0.1em;
}
.svg-inline--fa.fa-xs {
  vertical-align: 0em;
}
.svg-inline--fa.fa-sm {
  vertical-align: -0.0714285705em;
}
.svg-inline--fa.fa-lg {
  vertical-align: -0.2em;
}
.svg-inline--fa.fa-xl {
  vertical-align: -0.25em;
}
.svg-inline--fa.fa-2xl {
  vertical-align: -0.3125em;
}
.svg-inline--fa.fa-pull-left {
  margin-right: var(--fa-pull-margin, 0.3em);
  width: auto;
}
.svg-inline--fa.fa-pull-right {
  margin-left: var(--fa-pull-margin, 0.3em);
  width: auto;
}
.svg-inline--fa.fa-li {
  width: var(--fa-li-width, 2em);
  top: 0.25em;
}
.svg-inline--fa.fa-fw {
  width: var(--fa-fw-width, 1.25em);
}

.fa-layers svg.svg-inline--fa {
  bottom: 0;
  left: 0;
  margin: auto;
  position: absolute;
  right: 0;
  top: 0;
}

.fa-layers-counter, .fa-layers-text {
  display: inline-block;
  position: absolute;
  text-align: center;
}

.fa-layers {
  display: inline-block;
  height: 1em;
  position: relative;
  text-align: center;
  vertical-align: -0.125em;
  width: 1em;
}
.fa-layers svg.svg-inline--fa {
  -webkit-transform-origin: center center;
          transform-origin: center center;
}

.fa-layers-text {
  left: 50%;
  top: 50%;
  -webkit-transform: translate(-50%, -50%);
          transform: translate(-50%, -50%);
  -webkit-transform-origin: center center;
          transform-origin: center center;
}

.fa-layers-counter {
  background-color: var(--fa-counter-background-color, #ff253a);
  border-radius: var(--fa-counter-border-radius, 1em);
  box-sizing: border-box;
  color: var(--fa-inverse, #fff);
  line-height: var(--fa-counter-line-height, 1);
  max-width: var(--fa-counter-max-width, 5em);
  min-width: var(--fa-counter-min-width, 1.5em);
  overflow: hidden;
  padding: var(--fa-counter-padding, 0.25em 0.5em);
  right: var(--fa-right, 0);
  text-overflow: ellipsis;
  top: var(--fa-top, 0);
  -webkit-transform: scale(var(--fa-counter-scale, 0.25));
          transform: scale(var(--fa-counter-scale, 0.25));
  -webkit-transform-origin: top right;
          transform-origin: top right;
}

.fa-layers-bottom-right {
  bottom: var(--fa-bottom, 0);
  right: var(--fa-right, 0);
  top: auto;
  -webkit-transform: scale(var(--fa-layers-scale, 0.25));
          transform: scale(var(--fa-layers-scale, 0.25));
  -webkit-transform-origin: bottom right;
          transform-origin: bottom right;
}

.fa-layers-bottom-left {
  bottom: var(--fa-bottom, 0);
  left: var(--fa-left, 0);
  right: auto;
  top: auto;
  -webkit-transform: scale(var(--fa-layers-scale, 0.25));
          transform: scale(var(--fa-layers-scale, 0.25));
  -webkit-transform-origin: bottom left;
          transform-origin: bottom left;
}

.fa-layers-top-right {
  top: var(--fa-top, 0);
  right: var(--fa-right, 0);
  -webkit-transform: scale(var(--fa-layers-scale, 0.25));
          transform: scale(var(--fa-layers-scale, 0.25));
  -webkit-transform-origin: top right;
          transform-origin: top right;
}

.fa-layers-top-left {
  left: var(--fa-left, 0);
  right: auto;
  top: var(--fa-top, 0);
  -webkit-transform: scale(var(--fa-layers-scale, 0.25));
          transform: scale(var(--fa-layers-scale, 0.25));
  -webkit-transform-origin: top left;
          transform-origin: top left;
}

.fa-1x {
  font-size: 1em;
}

.fa-2x {
  font-size: 2em;
}

.fa-3x {
  font-size: 3em;
}

.fa-4x {
  font-size: 4em;
}

.fa-5x {
  font-size: 5em;
}

.fa-6x {
  font-size: 6em;
}

.fa-7x {
  font-size: 7em;
}

.fa-8x {
  font-size: 8em;
}

.fa-9x {
  font-size: 9em;
}

.fa-10x {
  font-size: 10em;
}

.fa-2xs {
  font-size: 0.625em;
  line-height: 0.1em;
  vertical-align: 0.225em;
}

.fa-xs {
  font-size: 0.75em;
  line-height: 0.0833333337em;
  vertical-align: 0.125em;
}

.fa-sm {
  font-size: 0.875em;
  line-height: 0.0714285718em;
  vertical-align: 0.0535714295em;
}

.fa-lg {
  font-size: 1.25em;
  line-height: 0.05em;
  vertical-align: -0.075em;
}

.fa-xl {
  font-size: 1.5em;
  line-height: 0.0416666682em;
  vertical-align: -0.125em;
}

.fa-2xl {
  font-size: 2em;
  line-height: 0.03125em;
  vertical-align: -0.1875em;
}

.fa-fw {
  text-align: center;
  width: 1.25em;
}

.fa-ul {
  list-style-type: none;
  margin-left: var(--fa-li-margin, 2.5em);
  padding-left: 0;
}
.fa-ul > li {
  position: relative;
}

.fa-li {
  left: calc(var(--fa-li-width, 2em) * -1);
  position: absolute;
  text-align: center;
  width: var(--fa-li-width, 2em);
  line-height: inherit;
}

.fa-border {
  border-color: var(--fa-border-color, #eee);
  border-radius: var(--fa-border-radius, 0.1em);
  border-style: var(--fa-border-style, solid);
  border-width: var(--fa-border-width, 0.08em);
  padding: var(--fa-border-padding, 0.2em 0.25em 0.15em);
}

.fa-pull-left {
  float: left;
  margin-right: var(--fa-pull-margin, 0.3em);
}

.fa-pull-right {
  float: right;
  margin-left: var(--fa-pull-margin, 0.3em);
}

.fa-beat {
  -webkit-animation-name: fa-beat;
          animation-name: fa-beat;
  -webkit-animation-delay: var(--fa-animation-delay, 0s);
          animation-delay: var(--fa-animation-delay, 0s);
  -webkit-animation-direction: var(--fa-animation-direction, normal);
          animation-direction: var(--fa-animation-direction, normal);
  -webkit-animation-duration: var(--fa-animation-duration, 1s);
          animation-duration: var(--fa-animation-duration, 1s);
  -webkit-animation-iteration-count: var(--fa-animation-iteration-count, infinite);
          animation-iteration-count: var(--fa-animation-iteration-count, infinite);
  -webkit-animation-timing-function: var(--fa-animation-timing, ease-in-out);
          animation-timing-function: var(--fa-animation-timing, ease-in-out);
}

.fa-bounce {
  -webkit-animation-name: fa-bounce;
          animation-name: fa-bounce;
  -webkit-animation-delay: var(--fa-animation-delay, 0s);
          animation-delay: var(--fa-animation-delay, 0s);
  -webkit-animation-direction: var(--fa-animation-direction, normal);
          animation-direction: var(--fa-animation-direction, normal);
  -webkit-animation-duration: var(--fa-animation-duration, 1s);
          animation-duration: var(--fa-animation-duration, 1s);
  -webkit-animation-iteration-count: var(--fa-animation-iteration-count, infinite);
          animation-iteration-count: var(--fa-animation-iteration-count, infinite);
  -webkit-animation-timing-function: var(--fa-animation-timing, cubic-bezier(0.28, 0.84, 0.42, 1));
          animation-timing-function: var(--fa-animation-timing, cubic-bezier(0.28, 0.84, 0.42, 1));
}

.fa-fade {
  -webkit-animation-name: fa-fade;
          animation-name: fa-fade;
  -webkit-animation-delay: var(--fa-animation-delay, 0s);
          animation-delay: var(--fa-animation-delay, 0s);
  -webkit-animation-direction: var(--fa-animation-direction, normal);
          animation-direction: var(--fa-animation-direction, normal);
  -webkit-animation-duration: var(--fa-animation-duration, 1s);
          animation-duration: var(--fa-animation-duration, 1s);
  -webkit-animation-iteration-count: var(--fa-animation-iteration-count, infinite);
          animation-iteration-count: var(--fa-animation-iteration-count, infinite);
  -webkit-animation-timing-function: var(--fa-animation-timing, cubic-bezier(0.4, 0, 0.6, 1));
          animation-timing-function: var(--fa-animation-timing, cubic-bezier(0.4, 0, 0.6, 1));
}

.fa-beat-fade {
  -webkit-animation-name: fa-beat-fade;
          animation-name: fa-beat-fade;
  -webkit-animation-delay: var(--fa-animation-delay, 0s);
          animation-delay: var(--fa-animation-delay, 0s);
  -webkit-animation-direction: var(--fa-animation-direction, normal);
          animation-direction: var(--fa-animation-direction, normal);
  -webkit-animation-duration: var(--fa-animation-duration, 1s);
          animation-duration: var(--fa-animation-duration, 1s);
  -webkit-animation-iteration-count: var(--fa-animation-iteration-count, infinite);
          animation-iteration-count: var(--fa-animation-iteration-count, infinite);
  -webkit-animation-timing-function: var(--fa-animation-timing, cubic-bezier(0.4, 0, 0.6, 1));
          animation-timing-function: var(--fa-animation-timing, cubic-bezier(0.4, 0, 0.6, 1));
}

.fa-flip {
  -webkit-animation-name: fa-flip;
          animation-name: fa-flip;
  -webkit-animation-delay: var(--fa-animation-delay, 0s);
          animation-delay: var(--fa-animation-delay, 0s);
  -webkit-animation-direction: var(--fa-animation-direction, normal);
          animation-direction: var(--fa-animation-direction, normal);
  -webkit-animation-duration: var(--fa-animation-duration, 1s);
          animation-duration: var(--fa-animation-duration, 1s);
  -webkit-animation-iteration-count: var(--fa-animation-iteration-count, infinite);
          animation-iteration-count: var(--fa-animation-iteration-count, infinite);
  -webkit-animation-timing-function: var(--fa-animation-timing, ease-in-out);
          animation-timing-function: var(--fa-animation-timing, ease-in-out);
}

.fa-shake {
  -webkit-animation-name: fa-shake;
          animation-name: fa-shake;
  -webkit-animation-delay: var(--fa-animation-delay, 0s);
          animation-delay: var(--fa-animation-delay, 0s);
  -webkit-animation-direction: var(--fa-animation-direction, normal);
          animation-direction: var(--fa-animation-direction, normal);
  -webkit-animation-duration: var(--fa-animation-duration, 1s);
          animation-duration: var(--fa-animation-duration, 1s);
  -webkit-animation-iteration-count: var(--fa-animation-iteration-count, infinite);
          animation-iteration-count: var(--fa-animation-iteration-count, infinite);
  -webkit-animation-timing-function: var(--fa-animation-timing, linear);
          animation-timing-function: var(--fa-animation-timing, linear);
}

.fa-spin {
  -webkit-animation-name: fa-spin;
          animation-name: fa-spin;
  -webkit-animation-delay: var(--fa-animation-delay, 0s);
          animation-delay: var(--fa-animation-delay, 0s);
  -webkit-animation-direction: var(--fa-animation-direction, normal);
          animation-direction: var(--fa-animation-direction, normal);
  -webkit-animation-duration: var(--fa-animation-duration, 2s);
          animation-duration: var(--fa-animation-duration, 2s);
  -webkit-animation-iteration-count: var(--fa-animation-iteration-count, infinite);
          animation-iteration-count: var(--fa-animation-iteration-count, infinite);
  -webkit-animation-timing-function: var(--fa-animation-timing, linear);
          animation-timing-function: var(--fa-animation-timing, linear);
}

.fa-spin-reverse {
  --fa-animation-direction: reverse;
}

.fa-pulse,
.fa-spin-pulse {
  -webkit-animation-name: fa-spin;
          animation-name: fa-spin;
  -webkit-animation-direction: var(--fa-animation-direction, normal);
          animation-direction: var(--fa-animation-direction, normal);
  -webkit-animation-duration: var(--fa-animation-duration, 1s);
          animation-duration: var(--fa-animation-duration, 1s);
  -webkit-animation-iteration-count: var(--fa-animation-iteration-count, infinite);
          animation-iteration-count: var(--fa-animation-iteration-count, infinite);
  -webkit-animation-timing-function: var(--fa-animation-timing, steps(8));
          animation-timing-function: var(--fa-animation-timing, steps(8));
}

@media (prefers-reduced-motion: reduce) {
  .fa-beat,
.fa-bounce,
.fa-fade,
.fa-beat-fade,
.fa-flip,
.fa-pulse,
.fa-shake,
.fa-spin,
.fa-spin-pulse {
    -webkit-animation-delay: -1ms;
            animation-delay: -1ms;
    -webkit-animation-duration: 1ms;
            animation-duration: 1ms;
    -webkit-animation-iteration-count: 1;
            animation-iteration-count: 1;
    transition-delay: 0s;
    transition-duration: 0s;
  }
}
@-webkit-keyframes fa-beat {
  0%, 90% {
    -webkit-transform: scale(1);
            transform: scale(1);
  }
  45% {
    -webkit-transform: scale(var(--fa-beat-scale, 1.25));
            transform: scale(var(--fa-beat-scale, 1.25));
  }
}
@keyframes fa-beat {
  0%, 90% {
    -webkit-transform: scale(1);
            transform: scale(1);
  }
  45% {
    -webkit-transform: scale(var(--fa-beat-scale, 1.25));
            transform: scale(var(--fa-beat-scale, 1.25));
  }
}
@-webkit-keyframes fa-bounce {
  0% {
    -webkit-transform: scale(1, 1) translateY(0);
            transform: scale(1, 1) translateY(0);
  }
  10% {
    -webkit-transform: scale(var(--fa-bounce-start-scale-x, 1.1), var(--fa-bounce-start-scale-y, 0.9)) translateY(0);
            transform: scale(var(--fa-bounce-start-scale-x, 1.1), var(--fa-bounce-start-scale-y, 0.9)) translateY(0);
  }
  30% {
    -webkit-transform: scale(var(--fa-bounce-jump-scale-x, 0.9), var(--fa-bounce-jump-scale-y, 1.1)) translateY(var(--fa-bounce-height, -0.5em));
            transform: scale(var(--fa-bounce-jump-scale-x, 0.9), var(--fa-bounce-jump-scale-y, 1.1)) translateY(var(--fa-bounce-height, -0.5em));
  }
  50% {
    -webkit-transform: scale(var(--fa-bounce-land-scale-x, 1.05), var(--fa-bounce-land-scale-y, 0.95)) translateY(0);
            transform: scale(var(--fa-bounce-land-scale-x, 1.05), var(--fa-bounce-land-scale-y, 0.95)) translateY(0);
  }
  57% {
    -webkit-transform: scale(1, 1) translateY(var(--fa-bounce-rebound, -0.125em));
            transform: scale(1, 1) translateY(var(--fa-bounce-rebound, -0.125em));
  }
  64% {
    -webkit-transform: scale(1, 1) translateY(0);
            transform: scale(1, 1) translateY(0);
  }
  100% {
    -webkit-transform: scale(1, 1) translateY(0);
            transform: scale(1, 1) translateY(0);
  }
}
@keyframes fa-bounce {
  0% {
    -webkit-transform: scale(1, 1) translateY(0);
            transform: scale(1, 1) translateY(0);
  }
  10% {
    -webkit-transform: scale(var(--fa-bounce-start-scale-x, 1.1), var(--fa-bounce-start-scale-y, 0.9)) translateY(0);
            transform: scale(var(--fa-bounce-start-scale-x, 1.1), var(--fa-bounce-start-scale-y, 0.9)) translateY(0);
  }
  30% {
    -webkit-transform: scale(var(--fa-bounce-jump-scale-x, 0.9), var(--fa-bounce-jump-scale-y, 1.1)) translateY(var(--fa-bounce-height, -0.5em));
            transform: scale(var(--fa-bounce-jump-scale-x, 0.9), var(--fa-bounce-jump-scale-y, 1.1)) translateY(var(--fa-bounce-height, -0.5em));
  }
  50% {
    -webkit-transform: scale(var(--fa-bounce-land-scale-x, 1.05), var(--fa-bounce-land-scale-y, 0.95)) translateY(0);
            transform: scale(var(--fa-bounce-land-scale-x, 1.05), var(--fa-bounce-land-scale-y, 0.95)) translateY(0);
  }
  57% {
    -webkit-transform: scale(1, 1) translateY(var(--fa-bounce-rebound, -0.125em));
            transform: scale(1, 1) translateY(var(--fa-bounce-rebound, -0.125em));
  }
  64% {
    -webkit-transform: scale(1, 1) translateY(0);
            transform: scale(1, 1) translateY(0);
  }
  100% {
    -webkit-transform: scale(1, 1) translateY(0);
            transform: scale(1, 1) translateY(0);
  }
}
@-webkit-keyframes fa-fade {
  50% {
    opacity: var(--fa-fade-opacity, 0.4);
  }
}
@keyframes fa-fade {
  50% {
    opacity: var(--fa-fade-opacity, 0.4);
  }
}
@-webkit-keyframes fa-beat-fade {
  0%, 100% {
    opacity: var(--fa-beat-fade-opacity, 0.4);
    -webkit-transform: scale(1);
            transform: scale(1);
  }
  50% {
    opacity: 1;
    -webkit-transform: scale(var(--fa-beat-fade-scale, 1.125));
            transform: scale(var(--fa-beat-fade-scale, 1.125));
  }
}
@keyframes fa-beat-fade {
  0%, 100% {
    opacity: var(--fa-beat-fade-opacity, 0.4);
    -webkit-transform: scale(1);
            transform: scale(1);
  }
  50% {
    opacity: 1;
    -webkit-transform: scale(var(--fa-beat-fade-scale, 1.125));
            transform: scale(var(--fa-beat-fade-scale, 1.125));
  }
}
@-webkit-keyframes fa-flip {
  50% {
    -webkit-transform: rotate3d(var(--fa-flip-x, 0), var(--fa-flip-y, 1), var(--fa-flip-z, 0), var(--fa-flip-angle, -180deg));
            transform: rotate3d(var(--fa-flip-x, 0), var(--fa-flip-y, 1), var(--fa-flip-z, 0), var(--fa-flip-angle, -180deg));
  }
}
@keyframes fa-flip {
  50% {
    -webkit-transform: rotate3d(var(--fa-flip-x, 0), var(--fa-flip-y, 1), var(--fa-flip-z, 0), var(--fa-flip-angle, -180deg));
            transform: rotate3d(var(--fa-flip-x, 0), var(--fa-flip-y, 1), var(--fa-flip-z, 0), var(--fa-flip-angle, -180deg));
  }
}
@-webkit-keyframes fa-shake {
  0% {
    -webkit-transform: rotate(-15deg);
            transform: rotate(-15deg);
  }
  4% {
    -webkit-transform: rotate(15deg);
            transform: rotate(15deg);
  }
  8%, 24% {
    -webkit-transform: rotate(-18deg);
            transform: rotate(-18deg);
  }
  12%, 28% {
    -webkit-transform: rotate(18deg);
            transform: rotate(18deg);
  }
  16% {
    -webkit-transform: rotate(-22deg);
            transform: rotate(-22deg);
  }
  20% {
    -webkit-transform: rotate(22deg);
            transform: rotate(22deg);
  }
  32% {
    -webkit-transform: rotate(-12deg);
            transform: rotate(-12deg);
  }
  36% {
    -webkit-transform: rotate(12deg);
            transform: rotate(12deg);
  }
  40%, 100% {
    -webkit-transform: rotate(0deg);
            transform: rotate(0deg);
  }
}
@keyframes fa-shake {
  0% {
    -webkit-transform: rotate(-15deg);
            transform: rotate(-15deg);
  }
  4% {
    -webkit-transform: rotate(15deg);
            transform: rotate(15deg);
  }
  8%, 24% {
    -webkit-transform: rotate(-18deg);
            transform: rotate(-18deg);
  }
  12%, 28% {
    -webkit-transform: rotate(18deg);
            transform: rotate(18deg);
  }
  16% {
    -webkit-transform: rotate(-22deg);
            transform: rotate(-22deg);
  }
  20% {
    -webkit-transform: rotate(22deg);
            transform: rotate(22deg);
  }
  32% {
    -webkit-transform: rotate(-12deg);
            transform: rotate(-12deg);
  }
  36% {
    -webkit-transform: rotate(12deg);
            transform: rotate(12deg);
  }
  40%, 100% {
    -webkit-transform: rotate(0deg);
            transform: rotate(0deg);
  }
}
@-webkit-keyframes fa-spin {
  0% {
    -webkit-transform: rotate(0deg);
            transform: rotate(0deg);
  }
  100% {
    -webkit-transform: rotate(360deg);
            transform: rotate(360deg);
  }
}
@keyframes fa-spin {
  0% {
    -webkit-transform: rotate(0deg);
            transform: rotate(0deg);
  }
  100% {
    -webkit-transform: rotate(360deg);
            transform: rotate(360deg);
  }
}
.fa-rotate-90 {
  -webkit-transform: rotate(90deg);
          transform: rotate(90deg);
}

.fa-rotate-180 {
  -webkit-transform: rotate(180deg);
          transform: rotate(180deg);
}

.fa-rotate-270 {
  -webkit-transform: rotate(270deg);
          transform: rotate(270deg);
}

.fa-flip-horizontal {
  -webkit-transform: scale(-1, 1);
          transform: scale(-1, 1);
}

.fa-flip-vertical {
  -webkit-transform: scale(1, -1);
          transform: scale(1, -1);
}

.fa-flip-both,
.fa-flip-horizontal.fa-flip-vertical {
  -webkit-transform: scale(-1, -1);
          transform: scale(-1, -1);
}

.fa-rotate-by {
  -webkit-transform: rotate(var(--fa-rotate-angle, none));
          transform: rotate(var(--fa-rotate-angle, none));
}

.fa-stack {
  display: inline-block;
  vertical-align: middle;
  height: 2em;
  position: relative;
  width: 2.5em;
}

.fa-stack-1x,
.fa-stack-2x {
  bottom: 0;
  left: 0;
  margin: auto;
  position: absolute;
  right: 0;
  top: 0;
  z-index: var(--fa-stack-z-index, auto);
}

.svg-inline--fa.fa-stack-1x {
  height: 1em;
  width: 1.25em;
}
.svg-inline--fa.fa-stack-2x {
  height: 2em;
  width: 2.5em;
}

.fa-inverse {
  color: var(--fa-inverse, #fff);
}

.sr-only,
.fa-sr-only {
  position: absolute;
  width: 1px;
  height: 1px;
  padding: 0;
  margin: -1px;
  overflow: hidden;
  clip: rect(0, 0, 0, 0);
  white-space: nowrap;
  border-width: 0;
}

.sr-only-focusable:not(:focus),
.fa-sr-only-focusable:not(:focus) {
  position: absolute;
  width: 1px;
  height: 1px;
  padding: 0;
  margin: -1px;
  overflow: hidden;
  clip: rect(0, 0, 0, 0);
  white-space: nowrap;
  border-width: 0;
}

.svg-inline--fa .fa-primary {
  fill: var(--fa-primary-color, currentColor);
  opacity: var(--fa-primary-opacity, 1);
}

.svg-inline--fa .fa-secondary {
  fill: var(--fa-secondary-color, currentColor);
  opacity: var(--fa-secondary-opacity, 0.4);
}

.svg-inline--fa.fa-swap-opacity .fa-primary {
  opacity: var(--fa-secondary-opacity, 0.4);
}

.svg-inline--fa.fa-swap-opacity .fa-secondary {
  opacity: var(--fa-primary-opacity, 1);
}

.svg-inline--fa mask .fa-primary,
.svg-inline--fa mask .fa-secondary {
  fill: black;
}

.fad.fa-inverse,
.fa-duotone.fa-inverse {
  color: var(--fa-inverse, #fff);
}</style><link rel="index" href="/">
        
        <link href="/static/dist/base.css?v=1703115796000" media="screen" rel="stylesheet" type="text/css">
        <link href="/static/style/jquery-ui/smoothness/jquery-ui.css?v=1703115796000" media="screen" rel="stylesheet" type="text/css">
    <style type="text/css">.ti-tag-input[data-v-108f4f13] {
  background-color: transparent;
  color: inherit;
  border: none;
  padding: 0px;
  margin: 0px;
  display: flex;
  top: 0px;
  position: absolute;
  width: 100%;
  line-height: inherit;
}
.ti-tag-input[data-v-108f4f13]::-ms-clear {
  display: none;
}
input[data-v-108f4f13]:focus {
  outline: none;
}
input[disabled][data-v-108f4f13] {
  background-color: transparent;
}

/*# sourceURL=C:/Users/johan/dev/vue-tags-input/vue-tags-input/C:/Users/johan/dev/vue-tags-input/vue-tags-input/tag-input.vue */
/*# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkM6L1VzZXJzL2pvaGFuL2Rldi92dWUtdGFncy1pbnB1dC92dWUtdGFncy1pbnB1dC9DOi9Vc2Vycy9qb2hhbi9kZXYvdnVlLXRhZ3MtaW5wdXQvdnVlLXRhZ3MtaW5wdXQvdGFnLWlucHV0LnZ1ZSJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFBQTtFQUNFLDhCQUE4QjtFQUM5QixlQUFlO0VBQ2YsYUFBYTtFQUNiLGFBQWE7RUFDYixZQUFZO0VBQ1osY0FBYztFQUNkLFNBQVM7RUFDVCxtQkFBbUI7RUFDbkIsWUFBWTtFQUNaLHFCQUFxQjtDQUFFO0FBRXpCO0VBQ0UsY0FBYztDQUFFO0FBRWxCO0VBQ0UsY0FBYztDQUFFO0FBRWxCO0VBQ0UsOEJBQThCO0NBQUUiLCJmaWxlIjoidGFnLWlucHV0LnZ1ZT92dWUmdHlwZT1zdHlsZSZpbmRleD0wJmlkPTEwOGY0ZjEzJmxhbmc9Y3NzJnNjb3BlZD10cnVlJiIsInNvdXJjZXNDb250ZW50IjpbIi50aS10YWctaW5wdXQge1xuICBiYWNrZ3JvdW5kLWNvbG9yOiB0cmFuc3BhcmVudDtcbiAgY29sb3I6IGluaGVyaXQ7XG4gIGJvcmRlcjogbm9uZTtcbiAgcGFkZGluZzogMHB4O1xuICBtYXJnaW46IDBweDtcbiAgZGlzcGxheTogZmxleDtcbiAgdG9wOiAwcHg7XG4gIHBvc2l0aW9uOiBhYnNvbHV0ZTtcbiAgd2lkdGg6IDEwMCU7XG4gIGxpbmUtaGVpZ2h0OiBpbmhlcml0OyB9XG5cbi50aS10YWctaW5wdXQ6Oi1tcy1jbGVhciB7XG4gIGRpc3BsYXk6IG5vbmU7IH1cblxuaW5wdXQ6Zm9jdXMge1xuICBvdXRsaW5lOiBub25lOyB9XG5cbmlucHV0W2Rpc2FibGVkXSB7XG4gIGJhY2tncm91bmQtY29sb3I6IHRyYW5zcGFyZW50OyB9XG4iXSwic291cmNlUm9vdCI6IiJ9 */</style><style type="text/css">@font-face {
  font-family: 'icomoon';
  src: url(data:application/vnd.ms-fontobject;base64,aAUAAMQEAAABAAIAAAAAAAAAAAAAAAAAAAABAJABAAAAAExQAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAUdPJHwAAAAAAAAAAAAAAAAAAAAAAAA4AaQBjAG8AbQBvAG8AbgAAAA4AUgBlAGcAdQBsAGEAcgAAABYAVgBlAHIAcwBpAG8AbgAgADEALgAwAAAADgBpAGMAbwBtAG8AbwBuAAAAAAAAAQAAAAsAgAADADBPUy8yDxIFrAAAALwAAABgY21hcBdW0okAAAEcAAAAVGdhc3AAAAAQAAABcAAAAAhnbHlmpZ+jMAAAAXgAAAD8aGVhZA/FmAgAAAJ0AAAANmhoZWEHgAPIAAACrAAAACRobXR4EgABvgAAAtAAAAAcbG9jYQCSAOIAAALsAAAAEG1heHAACQAfAAAC/AAAACBuYW1lmUoJ+wAAAxwAAAGGcG9zdAADAAAAAASkAAAAIAADA4ABkAAFAAACmQLMAAAAjwKZAswAAAHrADMBCQAAAAAAAAAAAAAAAAAAAAEQAAAAAAAAAAAAAAAAAAAAAEAAAOkCA8D/wABAA8AAQAAAAAEAAAAAAAAAAAAAACAAAAAAAAMAAAADAAAAHAABAAMAAAAcAAMAAQAAABwABAA4AAAACgAIAAIAAgABACDpAv/9//8AAAAAACDpAP/9//8AAf/jFwQAAwABAAAAAAAAAAAAAAABAAH//wAPAAEAAAAAAAAAAAACAAA3OQEAAAAAAQAAAAAAAAAAAAIAADc5AQAAAAABAAAAAAAAAAAAAgAANzkBAAAAAAEAVgEBA74CgQAcAAABMhceARcWFwcmJy4BJyYjIgYHFyERFzY3PgE3NgIWSkNDbykpF2QQIB9VMzQ5P3AtnP6AmB0iIkspKAJVFxhSODlCIDMrKz4REislmgGAmhkVFBwICAABANYAgQMqAtUACwAAAQcXBycHJzcnNxc3Ayru7jzu7jzu7jzu7gKZ7u487u487u487u4AAQCSAIEDgAK9AAUAACUBFwEnNwGAAcQ8/gDuPPkBxDz+AO48AAAAAAEAAAAAAAAfydNRXw889QALBAAAAAAA1nUqGwAAAADWdSobAAAAAAO+AtUAAAAIAAIAAAAAAAAAAQAAA8D/wAAABAAAAAAAA74AAQAAAAAAAAAAAAAAAAAAAAcEAAAAAAAAAAAAAAACAAAABAAAVgQAANYEAACSAAAAAAAKABQAHgBQAGoAfgABAAAABwAdAAEAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAAADgCuAAEAAAAAAAEABwAAAAEAAAAAAAIABwBgAAEAAAAAAAMABwA2AAEAAAAAAAQABwB1AAEAAAAAAAUACwAVAAEAAAAAAAYABwBLAAEAAAAAAAoAGgCKAAMAAQQJAAEADgAHAAMAAQQJAAIADgBnAAMAAQQJAAMADgA9AAMAAQQJAAQADgB8AAMAAQQJAAUAFgAgAAMAAQQJAAYADgBSAAMAAQQJAAoANACkaWNvbW9vbgBpAGMAbwBtAG8AbwBuVmVyc2lvbiAxLjAAVgBlAHIAcwBpAG8AbgAgADEALgAwaWNvbW9vbgBpAGMAbwBtAG8AbwBuaWNvbW9vbgBpAGMAbwBtAG8AbwBuUmVndWxhcgBSAGUAZwB1AGwAYQByaWNvbW9vbgBpAGMAbwBtAG8AbwBuRm9udCBnZW5lcmF0ZWQgYnkgSWNvTW9vbi4ARgBvAG4AdAAgAGcAZQBuAGUAcgBhAHQAZQBkACAAYgB5ACAASQBjAG8ATQBvAG8AbgAuAAAAAwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==);
  src: url(data:application/vnd.ms-fontobject;base64,aAUAAMQEAAABAAIAAAAAAAAAAAAAAAAAAAABAJABAAAAAExQAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAUdPJHwAAAAAAAAAAAAAAAAAAAAAAAA4AaQBjAG8AbQBvAG8AbgAAAA4AUgBlAGcAdQBsAGEAcgAAABYAVgBlAHIAcwBpAG8AbgAgADEALgAwAAAADgBpAGMAbwBtAG8AbwBuAAAAAAAAAQAAAAsAgAADADBPUy8yDxIFrAAAALwAAABgY21hcBdW0okAAAEcAAAAVGdhc3AAAAAQAAABcAAAAAhnbHlmpZ+jMAAAAXgAAAD8aGVhZA/FmAgAAAJ0AAAANmhoZWEHgAPIAAACrAAAACRobXR4EgABvgAAAtAAAAAcbG9jYQCSAOIAAALsAAAAEG1heHAACQAfAAAC/AAAACBuYW1lmUoJ+wAAAxwAAAGGcG9zdAADAAAAAASkAAAAIAADA4ABkAAFAAACmQLMAAAAjwKZAswAAAHrADMBCQAAAAAAAAAAAAAAAAAAAAEQAAAAAAAAAAAAAAAAAAAAAEAAAOkCA8D/wABAA8AAQAAAAAEAAAAAAAAAAAAAACAAAAAAAAMAAAADAAAAHAABAAMAAAAcAAMAAQAAABwABAA4AAAACgAIAAIAAgABACDpAv/9//8AAAAAACDpAP/9//8AAf/jFwQAAwABAAAAAAAAAAAAAAABAAH//wAPAAEAAAAAAAAAAAACAAA3OQEAAAAAAQAAAAAAAAAAAAIAADc5AQAAAAABAAAAAAAAAAAAAgAANzkBAAAAAAEAVgEBA74CgQAcAAABMhceARcWFwcmJy4BJyYjIgYHFyERFzY3PgE3NgIWSkNDbykpF2QQIB9VMzQ5P3AtnP6AmB0iIkspKAJVFxhSODlCIDMrKz4REislmgGAmhkVFBwICAABANYAgQMqAtUACwAAAQcXBycHJzcnNxc3Ayru7jzu7jzu7jzu7gKZ7u487u487u487u4AAQCSAIEDgAK9AAUAACUBFwEnNwGAAcQ8/gDuPPkBxDz+AO48AAAAAAEAAAAAAAAfydNRXw889QALBAAAAAAA1nUqGwAAAADWdSobAAAAAAO+AtUAAAAIAAIAAAAAAAAAAQAAA8D/wAAABAAAAAAAA74AAQAAAAAAAAAAAAAAAAAAAAcEAAAAAAAAAAAAAAACAAAABAAAVgQAANYEAACSAAAAAAAKABQAHgBQAGoAfgABAAAABwAdAAEAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAAADgCuAAEAAAAAAAEABwAAAAEAAAAAAAIABwBgAAEAAAAAAAMABwA2AAEAAAAAAAQABwB1AAEAAAAAAAUACwAVAAEAAAAAAAYABwBLAAEAAAAAAAoAGgCKAAMAAQQJAAEADgAHAAMAAQQJAAIADgBnAAMAAQQJAAMADgA9AAMAAQQJAAQADgB8AAMAAQQJAAUAFgAgAAMAAQQJAAYADgBSAAMAAQQJAAoANACkaWNvbW9vbgBpAGMAbwBtAG8AbwBuVmVyc2lvbiAxLjAAVgBlAHIAcwBpAG8AbgAgADEALgAwaWNvbW9vbgBpAGMAbwBtAG8AbwBuaWNvbW9vbgBpAGMAbwBtAG8AbwBuUmVndWxhcgBSAGUAZwB1AGwAYQByaWNvbW9vbgBpAGMAbwBtAG8AbwBuRm9udCBnZW5lcmF0ZWQgYnkgSWNvTW9vbi4ARgBvAG4AdAAgAGcAZQBuAGUAcgBhAHQAZQBkACAAYgB5ACAASQBjAG8ATQBvAG8AbgAuAAAAAwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==#iefix) format("embedded-opentype"), url(data:font/ttf;base64,AAEAAAALAIAAAwAwT1MvMg8SBawAAAC8AAAAYGNtYXAXVtKJAAABHAAAAFRnYXNwAAAAEAAAAXAAAAAIZ2x5ZqWfozAAAAF4AAAA/GhlYWQPxZgIAAACdAAAADZoaGVhB4ADyAAAAqwAAAAkaG10eBIAAb4AAALQAAAAHGxvY2EAkgDiAAAC7AAAABBtYXhwAAkAHwAAAvwAAAAgbmFtZZlKCfsAAAMcAAABhnBvc3QAAwAAAAAEpAAAACAAAwOAAZAABQAAApkCzAAAAI8CmQLMAAAB6wAzAQkAAAAAAAAAAAAAAAAAAAABEAAAAAAAAAAAAAAAAAAAAABAAADpAgPA/8AAQAPAAEAAAAABAAAAAAAAAAAAAAAgAAAAAAADAAAAAwAAABwAAQADAAAAHAADAAEAAAAcAAQAOAAAAAoACAACAAIAAQAg6QL//f//AAAAAAAg6QD//f//AAH/4xcEAAMAAQAAAAAAAAAAAAAAAQAB//8ADwABAAAAAAAAAAAAAgAANzkBAAAAAAEAAAAAAAAAAAACAAA3OQEAAAAAAQAAAAAAAAAAAAIAADc5AQAAAAABAFYBAQO+AoEAHAAAATIXHgEXFhcHJicuAScmIyIGBxchERc2Nz4BNzYCFkpDQ28pKRdkECAfVTM0OT9wLZz+gJgdIiJLKSgCVRcYUjg5QiAzKys+ERIrJZoBgJoZFRQcCAgAAQDWAIEDKgLVAAsAAAEHFwcnByc3JzcXNwMq7u487u487u487u4Cme7uPO7uPO7uPO7uAAEAkgCBA4ACvQAFAAAlARcBJzcBgAHEPP4A7jz5AcQ8/gDuPAAAAAABAAAAAAAAH8nTUV8PPPUACwQAAAAAANZ1KhsAAAAA1nUqGwAAAAADvgLVAAAACAACAAAAAAAAAAEAAAPA/8AAAAQAAAAAAAO+AAEAAAAAAAAAAAAAAAAAAAAHBAAAAAAAAAAAAAAAAgAAAAQAAFYEAADWBAAAkgAAAAAACgAUAB4AUABqAH4AAQAAAAcAHQABAAAAAAACAAAAAAAAAAAAAAAAAAAAAAAAAA4ArgABAAAAAAABAAcAAAABAAAAAAACAAcAYAABAAAAAAADAAcANgABAAAAAAAEAAcAdQABAAAAAAAFAAsAFQABAAAAAAAGAAcASwABAAAAAAAKABoAigADAAEECQABAA4ABwADAAEECQACAA4AZwADAAEECQADAA4APQADAAEECQAEAA4AfAADAAEECQAFABYAIAADAAEECQAGAA4AUgADAAEECQAKADQApGljb21vb24AaQBjAG8AbQBvAG8AblZlcnNpb24gMS4wAFYAZQByAHMAaQBvAG4AIAAxAC4AMGljb21vb24AaQBjAG8AbQBvAG8Abmljb21vb24AaQBjAG8AbQBvAG8AblJlZ3VsYXIAUgBlAGcAdQBsAGEAcmljb21vb24AaQBjAG8AbQBvAG8AbkZvbnQgZ2VuZXJhdGVkIGJ5IEljb01vb24uAEYAbwBuAHQAIABnAGUAbgBlAHIAYQB0AGUAZAAgAGIAeQAgAEkAYwBvAE0AbwBvAG4ALgAAAAMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=) format("truetype"), url(data:font/woff;base64,d09GRgABAAAAAAUQAAsAAAAABMQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABPUy8yAAABCAAAAGAAAABgDxIFrGNtYXAAAAFoAAAAVAAAAFQXVtKJZ2FzcAAAAbwAAAAIAAAACAAAABBnbHlmAAABxAAAAPwAAAD8pZ+jMGhlYWQAAALAAAAANgAAADYPxZgIaGhlYQAAAvgAAAAkAAAAJAeAA8hobXR4AAADHAAAABwAAAAcEgABvmxvY2EAAAM4AAAAEAAAABAAkgDibWF4cAAAA0gAAAAgAAAAIAAJAB9uYW1lAAADaAAAAYYAAAGGmUoJ+3Bvc3QAAATwAAAAIAAAACAAAwAAAAMDgAGQAAUAAAKZAswAAACPApkCzAAAAesAMwEJAAAAAAAAAAAAAAAAAAAAARAAAAAAAAAAAAAAAAAAAAAAQAAA6QIDwP/AAEADwABAAAAAAQAAAAAAAAAAAAAAIAAAAAAAAwAAAAMAAAAcAAEAAwAAABwAAwABAAAAHAAEADgAAAAKAAgAAgACAAEAIOkC//3//wAAAAAAIOkA//3//wAB/+MXBAADAAEAAAAAAAAAAAAAAAEAAf//AA8AAQAAAAAAAAAAAAIAADc5AQAAAAABAAAAAAAAAAAAAgAANzkBAAAAAAEAAAAAAAAAAAACAAA3OQEAAAAAAQBWAQEDvgKBABwAAAEyFx4BFxYXByYnLgEnJiMiBgcXIREXNjc+ATc2AhZKQ0NvKSkXZBAgH1UzNDk/cC2c/oCYHSIiSykoAlUXGFI4OUIgMysrPhESKyWaAYCaGRUUHAgIAAEA1gCBAyoC1QALAAABBxcHJwcnNyc3FzcDKu7uPO7uPO7uPO7uApnu7jzu7jzu7jzu7gABAJIAgQOAAr0ABQAAJQEXASc3AYABxDz+AO48+QHEPP4A7jwAAAAAAQAAAAAAAB/J01FfDzz1AAsEAAAAAADWdSobAAAAANZ1KhsAAAAAA74C1QAAAAgAAgAAAAAAAAABAAADwP/AAAAEAAAAAAADvgABAAAAAAAAAAAAAAAAAAAABwQAAAAAAAAAAAAAAAIAAAAEAABWBAAA1gQAAJIAAAAAAAoAFAAeAFAAagB+AAEAAAAHAB0AAQAAAAAAAgAAAAAAAAAAAAAAAAAAAAAAAAAOAK4AAQAAAAAAAQAHAAAAAQAAAAAAAgAHAGAAAQAAAAAAAwAHADYAAQAAAAAABAAHAHUAAQAAAAAABQALABUAAQAAAAAABgAHAEsAAQAAAAAACgAaAIoAAwABBAkAAQAOAAcAAwABBAkAAgAOAGcAAwABBAkAAwAOAD0AAwABBAkABAAOAHwAAwABBAkABQAWACAAAwABBAkABgAOAFIAAwABBAkACgA0AKRpY29tb29uAGkAYwBvAG0AbwBvAG5WZXJzaW9uIDEuMABWAGUAcgBzAGkAbwBuACAAMQAuADBpY29tb29uAGkAYwBvAG0AbwBvAG5pY29tb29uAGkAYwBvAG0AbwBvAG5SZWd1bGFyAFIAZQBnAHUAbABhAHJpY29tb29uAGkAYwBvAG0AbwBvAG5Gb250IGdlbmVyYXRlZCBieSBJY29Nb29uLgBGAG8AbgB0ACAAZwBlAG4AZQByAGEAdABlAGQAIABiAHkAIABJAGMAbwBNAG8AbwBuAC4AAAADAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA) format("woff");
  font-weight: normal;
  font-style: normal;
}
[class^="ti-icon-"][data-v-61d92e31], [class*=" ti-icon-"][data-v-61d92e31] {
  font-family: 'icomoon' !important;
  speak: none;
  font-style: normal;
  font-weight: normal;
  font-variant: normal;
  text-transform: none;
  line-height: 1;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}
.ti-icon-check[data-v-61d92e31]:before {
  content: "\e902";
}
.ti-icon-close[data-v-61d92e31]:before {
  content: "\e901";
}
.ti-icon-undo[data-v-61d92e31]:before {
  content: "\e900";
}
ul[data-v-61d92e31] {
  margin: 0px;
  padding: 0px;
  list-style-type: none;
}
*[data-v-61d92e31], *[data-v-61d92e31]:before, *[data-v-61d92e31]:after {
  box-sizing: border-box;
}
input[data-v-61d92e31]:focus {
  outline: none;
}
input[disabled][data-v-61d92e31] {
  background-color: transparent;
}
.vue-tags-input[data-v-61d92e31] {
  max-width: 450px;
  position: relative;
  background-color: #fff;
}
div.vue-tags-input.disabled[data-v-61d92e31] {
  opacity: 0.5;
}
div.vue-tags-input.disabled *[data-v-61d92e31] {
    cursor: default;
}
.ti-input[data-v-61d92e31] {
  border: 1px solid #ccc;
  display: flex;
  padding: 4px;
  flex-wrap: wrap;
}
.ti-tags[data-v-61d92e31] {
  display: flex;
  flex-wrap: wrap;
  width: 100%;
  line-height: 1em;
}
.ti-tag[data-v-61d92e31] {
  background-color: #5C6BC0;
  color: #fff;
  border-radius: 2px;
  display: flex;
  padding: 3px 5px;
  margin: 2px;
  font-size: .85em;
}
.ti-tag[data-v-61d92e31]:focus {
    outline: none;
}
.ti-tag .ti-content[data-v-61d92e31] {
    display: flex;
    align-items: center;
}
.ti-tag .ti-tag-center[data-v-61d92e31] {
    position: relative;
}
.ti-tag span[data-v-61d92e31] {
    line-height: .85em;
}
.ti-tag span.ti-hidden[data-v-61d92e31] {
    padding-left: 14px;
    visibility: hidden;
    height: 0px;
    white-space: pre;
}
.ti-tag .ti-actions[data-v-61d92e31] {
    margin-left: 2px;
    display: flex;
    align-items: center;
    font-size: 1.15em;
}
.ti-tag .ti-actions i[data-v-61d92e31] {
      cursor: pointer;
}
.ti-tag[data-v-61d92e31]:last-child {
    margin-right: 4px;
}
.ti-tag.ti-invalid[data-v-61d92e31], .ti-tag.ti-tag.ti-deletion-mark[data-v-61d92e31] {
    background-color: #e54d42;
}
.ti-new-tag-input-wrapper[data-v-61d92e31] {
  display: flex;
  flex: 1 0 auto;
  padding: 3px 5px;
  margin: 2px;
  font-size: .85em;
}
.ti-new-tag-input-wrapper input[data-v-61d92e31] {
    flex: 1 0 auto;
    min-width: 100px;
    border: none;
    padding: 0px;
    margin: 0px;
}
.ti-new-tag-input[data-v-61d92e31] {
  line-height: initial;
}
.ti-autocomplete[data-v-61d92e31] {
  border: 1px solid #ccc;
  border-top: none;
  position: absolute;
  width: 100%;
  background-color: #fff;
  z-index: 20;
}
.ti-item > div[data-v-61d92e31] {
  cursor: pointer;
  padding: 3px 6px;
  width: 100%;
}
.ti-selected-item[data-v-61d92e31] {
  background-color: #5C6BC0;
  color: #fff;
}

/*# sourceURL=C:/Users/johan/dev/vue-tags-input/vue-tags-input/C:/Users/johan/dev/vue-tags-input/vue-tags-input/vue-tags-input.scss */
/*# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkM6L1VzZXJzL2pvaGFuL2Rldi92dWUtdGFncy1pbnB1dC92dWUtdGFncy1pbnB1dC9DOi9Vc2Vycy9qb2hhbi9kZXYvdnVlLXRhZ3MtaW5wdXQvdnVlLXRhZ3MtaW5wdXQvdnVlLXRhZ3MtaW5wdXQuc2NzcyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFBQTtFQUNFLHVCQUF1QjtFQUN2QixtQ0FBOEM7RUFDOUMsK0pBQXVNO0VBQ3ZNLG9CQUFvQjtFQUNwQixtQkFBbUI7Q0FBRTtBQUV2QjtFQUNFLGtDQUFrQztFQUNsQyxZQUFZO0VBQ1osbUJBQW1CO0VBQ25CLG9CQUFvQjtFQUNwQixxQkFBcUI7RUFDckIscUJBQXFCO0VBQ3JCLGVBQWU7RUFDZixvQ0FBb0M7RUFDcEMsbUNBQW1DO0NBQUU7QUFFdkM7RUFDRSxpQkFBaUI7Q0FBRTtBQUVyQjtFQUNFLGlCQUFpQjtDQUFFO0FBRXJCO0VBQ0UsaUJBQWlCO0NBQUU7QUFFckI7RUFDRSxZQUFZO0VBQ1osYUFBYTtFQUNiLHNCQUFzQjtDQUFFO0FBRTFCO0VBQ0UsdUJBQXVCO0NBQUU7QUFFM0I7RUFDRSxjQUFjO0NBQUU7QUFFbEI7RUFDRSw4QkFBOEI7Q0FBRTtBQUVsQztFQUNFLGlCQUFpQjtFQUNqQixtQkFBbUI7RUFDbkIsdUJBQXVCO0NBQUU7QUFFM0I7RUFDRSxhQUFhO0NBQUU7QUFDZjtJQUNFLGdCQUFnQjtDQUFFO0FBRXRCO0VBQ0UsdUJBQXVCO0VBQ3ZCLGNBQWM7RUFDZCxhQUFhO0VBQ2IsZ0JBQWdCO0NBQUU7QUFFcEI7RUFDRSxjQUFjO0VBQ2QsZ0JBQWdCO0VBQ2hCLFlBQVk7RUFDWixpQkFBaUI7Q0FBRTtBQUVyQjtFQUNFLDBCQUEwQjtFQUMxQixZQUFZO0VBQ1osbUJBQW1CO0VBQ25CLGNBQWM7RUFDZCxpQkFBaUI7RUFDakIsWUFBWTtFQUNaLGlCQUFpQjtDQUFFO0FBQ25CO0lBQ0UsY0FBYztDQUFFO0FBQ2xCO0lBQ0UsY0FBYztJQUNkLG9CQUFvQjtDQUFFO0FBQ3hCO0lBQ0UsbUJBQW1CO0NBQUU7QUFDdkI7SUFDRSxtQkFBbUI7Q0FBRTtBQUN2QjtJQUNFLG1CQUFtQjtJQUNuQixtQkFBbUI7SUFDbkIsWUFBWTtJQUNaLGlCQUFpQjtDQUFFO0FBQ3JCO0lBQ0UsaUJBQWlCO0lBQ2pCLGNBQWM7SUFDZCxvQkFBb0I7SUFDcEIsa0JBQWtCO0NBQUU7QUFDcEI7TUFDRSxnQkFBZ0I7Q0FBRTtBQUN0QjtJQUNFLGtCQUFrQjtDQUFFO0FBQ3RCO0lBQ0UsMEJBQTBCO0NBQUU7QUFFaEM7RUFDRSxjQUFjO0VBQ2QsZUFBZTtFQUNmLGlCQUFpQjtFQUNqQixZQUFZO0VBQ1osaUJBQWlCO0NBQUU7QUFDbkI7SUFDRSxlQUFlO0lBQ2YsaUJBQWlCO0lBQ2pCLGFBQWE7SUFDYixhQUFhO0lBQ2IsWUFBWTtDQUFFO0FBRWxCO0VBQ0UscUJBQXFCO0NBQUU7QUFFekI7RUFDRSx1QkFBdUI7RUFDdkIsaUJBQWlCO0VBQ2pCLG1CQUFtQjtFQUNuQixZQUFZO0VBQ1osdUJBQXVCO0VBQ3ZCLFlBQVk7Q0FBRTtBQUVoQjtFQUNFLGdCQUFnQjtFQUNoQixpQkFBaUI7RUFDakIsWUFBWTtDQUFFO0FBRWhCO0VBQ0UsMEJBQTBCO0VBQzFCLFlBQVk7Q0FBRSIsImZpbGUiOiJ2dWUtdGFncy1pbnB1dC5zY3NzP3Z1ZSZ0eXBlPXN0eWxlJmluZGV4PTAmaWQ9NjFkOTJlMzEmbGFuZz1zY3NzJnNjb3BlZD10cnVlJiIsInNvdXJjZXNDb250ZW50IjpbIkBmb250LWZhY2Uge1xuICBmb250LWZhbWlseTogJ2ljb21vb24nO1xuICBzcmM6IHVybChcIi4vYXNzZXRzL2ZvbnRzL2ljb21vb24uZW90PzdncmxzZVwiKTtcbiAgc3JjOiB1cmwoXCIuL2Fzc2V0cy9mb250cy9pY29tb29uLmVvdD83Z3Jsc2UjaWVmaXhcIikgZm9ybWF0KFwiZW1iZWRkZWQtb3BlbnR5cGVcIiksIHVybChcIi4vYXNzZXRzL2ZvbnRzL2ljb21vb24udHRmPzdncmxzZVwiKSBmb3JtYXQoXCJ0cnVldHlwZVwiKSwgdXJsKFwiLi9hc3NldHMvZm9udHMvaWNvbW9vbi53b2ZmPzdncmxzZVwiKSBmb3JtYXQoXCJ3b2ZmXCIpO1xuICBmb250LXdlaWdodDogbm9ybWFsO1xuICBmb250LXN0eWxlOiBub3JtYWw7IH1cblxuW2NsYXNzXj1cInRpLWljb24tXCJdLCBbY2xhc3MqPVwiIHRpLWljb24tXCJdIHtcbiAgZm9udC1mYW1pbHk6ICdpY29tb29uJyAhaW1wb3J0YW50O1xuICBzcGVhazogbm9uZTtcbiAgZm9udC1zdHlsZTogbm9ybWFsO1xuICBmb250LXdlaWdodDogbm9ybWFsO1xuICBmb250LXZhcmlhbnQ6IG5vcm1hbDtcbiAgdGV4dC10cmFuc2Zvcm06IG5vbmU7XG4gIGxpbmUtaGVpZ2h0OiAxO1xuICAtd2Via2l0LWZvbnQtc21vb3RoaW5nOiBhbnRpYWxpYXNlZDtcbiAgLW1vei1vc3gtZm9udC1zbW9vdGhpbmc6IGdyYXlzY2FsZTsgfVxuXG4udGktaWNvbi1jaGVjazpiZWZvcmUge1xuICBjb250ZW50OiBcIlxcZTkwMlwiOyB9XG5cbi50aS1pY29uLWNsb3NlOmJlZm9yZSB7XG4gIGNvbnRlbnQ6IFwiXFxlOTAxXCI7IH1cblxuLnRpLWljb24tdW5kbzpiZWZvcmUge1xuICBjb250ZW50OiBcIlxcZTkwMFwiOyB9XG5cbnVsIHtcbiAgbWFyZ2luOiAwcHg7XG4gIHBhZGRpbmc6IDBweDtcbiAgbGlzdC1zdHlsZS10eXBlOiBub25lOyB9XG5cbiosICo6YmVmb3JlLCAqOmFmdGVyIHtcbiAgYm94LXNpemluZzogYm9yZGVyLWJveDsgfVxuXG5pbnB1dDpmb2N1cyB7XG4gIG91dGxpbmU6IG5vbmU7IH1cblxuaW5wdXRbZGlzYWJsZWRdIHtcbiAgYmFja2dyb3VuZC1jb2xvcjogdHJhbnNwYXJlbnQ7IH1cblxuLnZ1ZS10YWdzLWlucHV0IHtcbiAgbWF4LXdpZHRoOiA0NTBweDtcbiAgcG9zaXRpb246IHJlbGF0aXZlO1xuICBiYWNrZ3JvdW5kLWNvbG9yOiAjZmZmOyB9XG5cbmRpdi52dWUtdGFncy1pbnB1dC5kaXNhYmxlZCB7XG4gIG9wYWNpdHk6IDAuNTsgfVxuICBkaXYudnVlLXRhZ3MtaW5wdXQuZGlzYWJsZWQgKiB7XG4gICAgY3Vyc29yOiBkZWZhdWx0OyB9XG5cbi50aS1pbnB1dCB7XG4gIGJvcmRlcjogMXB4IHNvbGlkICNjY2M7XG4gIGRpc3BsYXk6IGZsZXg7XG4gIHBhZGRpbmc6IDRweDtcbiAgZmxleC13cmFwOiB3cmFwOyB9XG5cbi50aS10YWdzIHtcbiAgZGlzcGxheTogZmxleDtcbiAgZmxleC13cmFwOiB3cmFwO1xuICB3aWR0aDogMTAwJTtcbiAgbGluZS1oZWlnaHQ6IDFlbTsgfVxuXG4udGktdGFnIHtcbiAgYmFja2dyb3VuZC1jb2xvcjogIzVDNkJDMDtcbiAgY29sb3I6ICNmZmY7XG4gIGJvcmRlci1yYWRpdXM6IDJweDtcbiAgZGlzcGxheTogZmxleDtcbiAgcGFkZGluZzogM3B4IDVweDtcbiAgbWFyZ2luOiAycHg7XG4gIGZvbnQtc2l6ZTogLjg1ZW07IH1cbiAgLnRpLXRhZzpmb2N1cyB7XG4gICAgb3V0bGluZTogbm9uZTsgfVxuICAudGktdGFnIC50aS1jb250ZW50IHtcbiAgICBkaXNwbGF5OiBmbGV4O1xuICAgIGFsaWduLWl0ZW1zOiBjZW50ZXI7IH1cbiAgLnRpLXRhZyAudGktdGFnLWNlbnRlciB7XG4gICAgcG9zaXRpb246IHJlbGF0aXZlOyB9XG4gIC50aS10YWcgc3BhbiB7XG4gICAgbGluZS1oZWlnaHQ6IC44NWVtOyB9XG4gIC50aS10YWcgc3Bhbi50aS1oaWRkZW4ge1xuICAgIHBhZGRpbmctbGVmdDogMTRweDtcbiAgICB2aXNpYmlsaXR5OiBoaWRkZW47XG4gICAgaGVpZ2h0OiAwcHg7XG4gICAgd2hpdGUtc3BhY2U6IHByZTsgfVxuICAudGktdGFnIC50aS1hY3Rpb25zIHtcbiAgICBtYXJnaW4tbGVmdDogMnB4O1xuICAgIGRpc3BsYXk6IGZsZXg7XG4gICAgYWxpZ24taXRlbXM6IGNlbnRlcjtcbiAgICBmb250LXNpemU6IDEuMTVlbTsgfVxuICAgIC50aS10YWcgLnRpLWFjdGlvbnMgaSB7XG4gICAgICBjdXJzb3I6IHBvaW50ZXI7IH1cbiAgLnRpLXRhZzpsYXN0LWNoaWxkIHtcbiAgICBtYXJnaW4tcmlnaHQ6IDRweDsgfVxuICAudGktdGFnLnRpLWludmFsaWQsIC50aS10YWcudGktdGFnLnRpLWRlbGV0aW9uLW1hcmsge1xuICAgIGJhY2tncm91bmQtY29sb3I6ICNlNTRkNDI7IH1cblxuLnRpLW5ldy10YWctaW5wdXQtd3JhcHBlciB7XG4gIGRpc3BsYXk6IGZsZXg7XG4gIGZsZXg6IDEgMCBhdXRvO1xuICBwYWRkaW5nOiAzcHggNXB4O1xuICBtYXJnaW46IDJweDtcbiAgZm9udC1zaXplOiAuODVlbTsgfVxuICAudGktbmV3LXRhZy1pbnB1dC13cmFwcGVyIGlucHV0IHtcbiAgICBmbGV4OiAxIDAgYXV0bztcbiAgICBtaW4td2lkdGg6IDEwMHB4O1xuICAgIGJvcmRlcjogbm9uZTtcbiAgICBwYWRkaW5nOiAwcHg7XG4gICAgbWFyZ2luOiAwcHg7IH1cblxuLnRpLW5ldy10YWctaW5wdXQge1xuICBsaW5lLWhlaWdodDogaW5pdGlhbDsgfVxuXG4udGktYXV0b2NvbXBsZXRlIHtcbiAgYm9yZGVyOiAxcHggc29saWQgI2NjYztcbiAgYm9yZGVyLXRvcDogbm9uZTtcbiAgcG9zaXRpb246IGFic29sdXRlO1xuICB3aWR0aDogMTAwJTtcbiAgYmFja2dyb3VuZC1jb2xvcjogI2ZmZjtcbiAgei1pbmRleDogMjA7IH1cblxuLnRpLWl0ZW0gPiBkaXYge1xuICBjdXJzb3I6IHBvaW50ZXI7XG4gIHBhZGRpbmc6IDNweCA2cHg7XG4gIHdpZHRoOiAxMDAlOyB9XG5cbi50aS1zZWxlY3RlZC1pdGVtIHtcbiAgYmFja2dyb3VuZC1jb2xvcjogIzVDNkJDMDtcbiAgY29sb3I6ICNmZmY7IH1cbiJdLCJzb3VyY2VSb290IjoiIn0= */</style><style type="text/css">.jqstooltip { position: absolute;left: 0px;top: 0px;visibility: hidden;background: rgb(0, 0, 0) transparent;background-color: rgba(0,0,0,0.6);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr=#99000000, endColorstr=#99000000);-ms-filter: "progid:DXImageTransform.Microsoft.gradient(startColorstr=#99000000, endColorstr=#99000000)";color: white;font: 10px arial, san serif;text-align: left;white-space: nowrap;padding: 5px;border: 1px solid white;z-index: 10000;}.jqsfield { color: white;font: 10px arial, san serif;text-align: left;}</style><script type="text/javascript">parcelRequire=function(e,r,n,t){var i="function"==typeof parcelRequire&&parcelRequire,o="function"==typeof require&&require;function u(n,t){if(!r[n]){if(!e[n]){var f="function"==typeof parcelRequire&&parcelRequire;if(!t&&f)return f(n,!0);if(i)return i(n,!0);if(o&&"string"==typeof n)return o(n);var c=new Error("Cannot find module '"+n+"'");throw c.code="MODULE_NOT_FOUND",c}p.resolve=function(r){return e[n][1][r]||r},p.cache={};var l=r[n]=new u.Module(n);e[n][0].call(l.exports,p,l,l.exports,this)}return r[n].exports;function p(e){return u(p.resolve(e))}}u.isParcelRequire=!0,u.Module=function(e){this.id=e,this.bundle=u,this.exports={}},u.modules=e,u.cache=r,u.parent=i,u.register=function(r,n){e[r]=[function(e,r){r.exports=n},{}]};for(var f=0;f<n.length;f++)u(n[f]);if(n.length){var c=u(n[n.length-1]);"object"==typeof exports&&"undefined"!=typeof module?module.exports=c:"function"==typeof define&&define.amd?define(function(){return c}):t&&(this[t]=c)}return u}({"g5I+":[function(require,module,exports) {

var t,e,n=module.exports={};function r(){throw new Error("setTimeout has not been defined")}function o(){throw new Error("clearTimeout has not been defined")}function i(e){if(t===setTimeout)return setTimeout(e,0);if((t===r||!t)&&setTimeout)return t=setTimeout,setTimeout(e,0);try{return t(e,0)}catch(n){try{return t.call(null,e,0)}catch(n){return t.call(this,e,0)}}}function u(t){if(e===clearTimeout)return clearTimeout(t);if((e===o||!e)&&clearTimeout)return e=clearTimeout,clearTimeout(t);try{return e(t)}catch(n){try{return e.call(null,t)}catch(n){return e.call(this,t)}}}!function(){try{t="function"==typeof setTimeout?setTimeout:r}catch(n){t=r}try{e="function"==typeof clearTimeout?clearTimeout:o}catch(n){e=o}}();var c,s=[],l=!1,a=-1;function f(){l&&c&&(l=!1,c.length?s=c.concat(s):a=-1,s.length&&h())}function h(){if(!l){var t=i(f);l=!0;for(var e=s.length;e;){for(c=s,s=[];++a<e;)c&&c[a].run();a=-1,e=s.length}c=null,l=!1,u(t)}}function m(t,e){this.fun=t,this.array=e}function p(){}n.nextTick=function(t){var e=new Array(arguments.length-1);if(arguments.length>1)for(var n=1;n<arguments.length;n++)e[n-1]=arguments[n];s.push(new m(t,e)),1!==s.length||l||i(h)},m.prototype.run=function(){this.fun.apply(null,this.array)},n.title="browser",n.browser=!0,n.env={},n.argv=[],n.version="",n.versions={},n.on=p,n.addListener=p,n.once=p,n.off=p,n.removeListener=p,n.removeAllListeners=p,n.emit=p,n.prependListener=p,n.prependOnceListener=p,n.listeners=function(t){return[]},n.binding=function(t){throw new Error("process.binding is not supported")},n.cwd=function(){return"/"},n.chdir=function(t){throw new Error("process.chdir is not supported")},n.umask=function(){return 0};
},{}],"HlZQ":[function(require,module,exports) {
var global = arguments[3];
var process = require("process");
var define;
var e,t=arguments[3],n=require("process");!function(e,t){"use strict";"object"==typeof module&&"object"==typeof module.exports?module.exports=e.document?t(e,!0):function(e){if(!e.document)throw new Error("jQuery requires a window with a document");return t(e)}:t(e)}("undefined"!=typeof window?window:this,function(t,n){"use strict";var r=[],i=t.document,o=Object.getPrototypeOf,s=r.slice,a=r.concat,u=r.push,c=r.indexOf,f={},l=f.toString,p=f.hasOwnProperty,d=p.toString,h=d.call(Object),g={},v=function(e){return"function"==typeof e&&"number"!=typeof e.nodeType},y=function(e){return null!=e&&e===e.window},m={type:!0,src:!0,noModule:!0};function x(e,t,n){var r,o=(t=t||i).createElement("script");if(o.text=e,n)for(r in m)n[r]&&(o[r]=n[r]);t.head.appendChild(o).parentNode.removeChild(o)}function b(e){return null==e?e+"":"object"==typeof e||"function"==typeof e?f[l.call(e)]||"object":typeof e}var w=function(e,t){return new w.fn.init(e,t)},T=/^[\s\uFEFF\xA0]+|[\s\uFEFF\xA0]+$/g;function C(e){var t=!!e&&"length"in e&&e.length,n=b(e);return!v(e)&&!y(e)&&("array"===n||0===t||"number"==typeof t&&t>0&&t-1 in e)}w.fn=w.prototype={jquery:"3.3.1",constructor:w,length:0,toArray:function(){return s.call(this)},get:function(e){return null==e?s.call(this):e<0?this[e+this.length]:this[e]},pushStack:function(e){var t=w.merge(this.constructor(),e);return t.prevObject=this,t},each:function(e){return w.each(this,e)},map:function(e){return this.pushStack(w.map(this,function(t,n){return e.call(t,n,t)}))},slice:function(){return this.pushStack(s.apply(this,arguments))},first:function(){return this.eq(0)},last:function(){return this.eq(-1)},eq:function(e){var t=this.length,n=+e+(e<0?t:0);return this.pushStack(n>=0&&n<t?[this[n]]:[])},end:function(){return this.prevObject||this.constructor()},push:u,sort:r.sort,splice:r.splice},w.extend=w.fn.extend=function(){var e,t,n,r,i,o,s=arguments[0]||{},a=1,u=arguments.length,c=!1;for("boolean"==typeof s&&(c=s,s=arguments[a]||{},a++),"object"==typeof s||v(s)||(s={}),a===u&&(s=this,a--);a<u;a++)if(null!=(e=arguments[a]))for(t in e)n=s[t],s!==(r=e[t])&&(c&&r&&(w.isPlainObject(r)||(i=Array.isArray(r)))?(i?(i=!1,o=n&&Array.isArray(n)?n:[]):o=n&&w.isPlainObject(n)?n:{},s[t]=w.extend(c,o,r)):void 0!==r&&(s[t]=r));return s},w.extend({expando:"jQuery"+("3.3.1"+Math.random()).replace(/\D/g,""),isReady:!0,error:function(e){throw new Error(e)},noop:function(){},isPlainObject:function(e){var t,n;return!(!e||"[object Object]"!==l.call(e))&&(!(t=o(e))||"function"==typeof(n=p.call(t,"constructor")&&t.constructor)&&d.call(n)===h)},isEmptyObject:function(e){var t;for(t in e)return!1;return!0},globalEval:function(e){x(e)},each:function(e,t){var n,r=0;if(C(e))for(n=e.length;r<n&&!1!==t.call(e[r],r,e[r]);r++);else for(r in e)if(!1===t.call(e[r],r,e[r]))break;return e},trim:function(e){return null==e?"":(e+"").replace(T,"")},makeArray:function(e,t){var n=t||[];return null!=e&&(C(Object(e))?w.merge(n,"string"==typeof e?[e]:e):u.call(n,e)),n},inArray:function(e,t,n){return null==t?-1:c.call(t,e,n)},merge:function(e,t){for(var n=+t.length,r=0,i=e.length;r<n;r++)e[i++]=t[r];return e.length=i,e},grep:function(e,t,n){for(var r=[],i=0,o=e.length,s=!n;i<o;i++)!t(e[i],i)!==s&&r.push(e[i]);return r},map:function(e,t,n){var r,i,o=0,s=[];if(C(e))for(r=e.length;o<r;o++)null!=(i=t(e[o],o,n))&&s.push(i);else for(o in e)null!=(i=t(e[o],o,n))&&s.push(i);return a.apply([],s)},guid:1,support:g}),"function"==typeof Symbol&&(w.fn[Symbol.iterator]=r[Symbol.iterator]),w.each("Boolean Number String Function Array Date RegExp Object Error Symbol".split(" "),function(e,t){f["[object "+t+"]"]=t.toLowerCase()});var S=function(e){var t,n,r,i,o,s,a,u,c,f,l,p,d,h,g,v,y,m,x,b="sizzle"+1*new Date,w=e.document,T=0,C=0,S=se(),k=se(),E=se(),D=function(e,t){return e===t&&(l=!0),0},A={}.hasOwnProperty,N=[],L=N.pop,q=N.push,H=N.push,j=N.slice,P=function(e,t){for(var n=0,r=e.length;n<r;n++)if(e[n]===t)return n;return-1},M="checked|selected|async|autofocus|autoplay|controls|defer|disabled|hidden|ismap|loop|multiple|open|readonly|required|scoped",O="[\\x20\\t\\r\\n\\f]",R="(?:\\\\.|[\\w-]|[^\0-\\xa0])+",W="\\["+O+"*("+R+")(?:"+O+"*([*^$|!~]?=)"+O+"*(?:'((?:\\\\.|[^\\\\'])*)'|\"((?:\\\\.|[^\\\\\"])*)\"|("+R+"))|)"+O+"*\\]",$=":("+R+")(?:\\((('((?:\\\\.|[^\\\\'])*)'|\"((?:\\\\.|[^\\\\\"])*)\")|((?:\\\\.|[^\\\\()[\\]]|"+W+")*)|.*)\\)|)",B=new RegExp(O+"+","g"),I=new RegExp("^"+O+"+|((?:^|[^\\\\])(?:\\\\.)*)"+O+"+$","g"),F=new RegExp("^"+O+"*,"+O+"*"),_=new RegExp("^"+O+"*([>+~]|"+O+")"+O+"*"),z=new RegExp("="+O+"*([^\\]'\"]*?)"+O+"*\\]","g"),U=new RegExp($),V=new RegExp("^"+R+"$"),X={ID:new RegExp("^#("+R+")"),CLASS:new RegExp("^\\.("+R+")"),TAG:new RegExp("^("+R+"|[*])"),ATTR:new RegExp("^"+W),PSEUDO:new RegExp("^"+$),CHILD:new RegExp("^:(only|first|last|nth|nth-last)-(child|of-type)(?:\\("+O+"*(even|odd|(([+-]|)(\\d*)n|)"+O+"*(?:([+-]|)"+O+"*(\\d+)|))"+O+"*\\)|)","i"),bool:new RegExp("^(?:"+M+")$","i"),needsContext:new RegExp("^"+O+"*[>+~]|:(even|odd|eq|gt|lt|nth|first|last)(?:\\("+O+"*((?:-\\d)?\\d*)"+O+"*\\)|)(?=[^-]|$)","i")},Y=/^(?:input|select|textarea|button)$/i,G=/^h\d$/i,J=/^[^{]+\{\s*\[native \w/,K=/^(?:#([\w-]+)|(\w+)|\.([\w-]+))$/,Z=/[+~]/,Q=new RegExp("\\\\([\\da-f]{1,6}"+O+"?|("+O+")|.)","ig"),ee=function(e,t,n){var r="0x"+t-65536;return r!=r||n?t:r<0?String.fromCharCode(r+65536):String.fromCharCode(r>>10|55296,1023&r|56320)},te=/([\0-\x1f\x7f]|^-?\d)|^-$|[^\0-\x1f\x7f-\uFFFF\w-]/g,ne=function(e,t){return t?"\0"===e?"�":e.slice(0,-1)+"\\"+e.charCodeAt(e.length-1).toString(16)+" ":"\\"+e},re=function(){p()},ie=me(function(e){return!0===e.disabled&&("form"in e||"label"in e)},{dir:"parentNode",next:"legend"});try{H.apply(N=j.call(w.childNodes),w.childNodes),N[w.childNodes.length].nodeType}catch(Ce){H={apply:N.length?function(e,t){q.apply(e,j.call(t))}:function(e,t){for(var n=e.length,r=0;e[n++]=t[r++];);e.length=n-1}}}function oe(e,t,r,i){var o,a,c,f,l,h,y,m=t&&t.ownerDocument,T=t?t.nodeType:9;if(r=r||[],"string"!=typeof e||!e||1!==T&&9!==T&&11!==T)return r;if(!i&&((t?t.ownerDocument||t:w)!==d&&p(t),t=t||d,g)){if(11!==T&&(l=K.exec(e)))if(o=l[1]){if(9===T){if(!(c=t.getElementById(o)))return r;if(c.id===o)return r.push(c),r}else if(m&&(c=m.getElementById(o))&&x(t,c)&&c.id===o)return r.push(c),r}else{if(l[2])return H.apply(r,t.getElementsByTagName(e)),r;if((o=l[3])&&n.getElementsByClassName&&t.getElementsByClassName)return H.apply(r,t.getElementsByClassName(o)),r}if(n.qsa&&!E[e+" "]&&(!v||!v.test(e))){if(1!==T)m=t,y=e;else if("object"!==t.nodeName.toLowerCase()){for((f=t.getAttribute("id"))?f=f.replace(te,ne):t.setAttribute("id",f=b),a=(h=s(e)).length;a--;)h[a]="#"+f+" "+ye(h[a]);y=h.join(","),m=Z.test(e)&&ge(t.parentNode)||t}if(y)try{return H.apply(r,m.querySelectorAll(y)),r}catch(C){}finally{f===b&&t.removeAttribute("id")}}}return u(e.replace(I,"$1"),t,r,i)}function se(){var e=[];return function t(n,i){return e.push(n+" ")>r.cacheLength&&delete t[e.shift()],t[n+" "]=i}}function ae(e){return e[b]=!0,e}function ue(e){var t=d.createElement("fieldset");try{return!!e(t)}catch(Ce){return!1}finally{t.parentNode&&t.parentNode.removeChild(t),t=null}}function ce(e,t){for(var n=e.split("|"),i=n.length;i--;)r.attrHandle[n[i]]=t}function fe(e,t){var n=t&&e,r=n&&1===e.nodeType&&1===t.nodeType&&e.sourceIndex-t.sourceIndex;if(r)return r;if(n)for(;n=n.nextSibling;)if(n===t)return-1;return e?1:-1}function le(e){return function(t){return"input"===t.nodeName.toLowerCase()&&t.type===e}}function pe(e){return function(t){var n=t.nodeName.toLowerCase();return("input"===n||"button"===n)&&t.type===e}}function de(e){return function(t){return"form"in t?t.parentNode&&!1===t.disabled?"label"in t?"label"in t.parentNode?t.parentNode.disabled===e:t.disabled===e:t.isDisabled===e||t.isDisabled!==!e&&ie(t)===e:t.disabled===e:"label"in t&&t.disabled===e}}function he(e){return ae(function(t){return t=+t,ae(function(n,r){for(var i,o=e([],n.length,t),s=o.length;s--;)n[i=o[s]]&&(n[i]=!(r[i]=n[i]))})})}function ge(e){return e&&void 0!==e.getElementsByTagName&&e}for(t in n=oe.support={},o=oe.isXML=function(e){var t=e&&(e.ownerDocument||e).documentElement;return!!t&&"HTML"!==t.nodeName},p=oe.setDocument=function(e){var t,i,s=e?e.ownerDocument||e:w;return s!==d&&9===s.nodeType&&s.documentElement?(h=(d=s).documentElement,g=!o(d),w!==d&&(i=d.defaultView)&&i.top!==i&&(i.addEventListener?i.addEventListener("unload",re,!1):i.attachEvent&&i.attachEvent("onunload",re)),n.attributes=ue(function(e){return e.className="i",!e.getAttribute("className")}),n.getElementsByTagName=ue(function(e){return e.appendChild(d.createComment("")),!e.getElementsByTagName("*").length}),n.getElementsByClassName=J.test(d.getElementsByClassName),n.getById=ue(function(e){return h.appendChild(e).id=b,!d.getElementsByName||!d.getElementsByName(b).length}),n.getById?(r.filter.ID=function(e){var t=e.replace(Q,ee);return function(e){return e.getAttribute("id")===t}},r.find.ID=function(e,t){if(void 0!==t.getElementById&&g){var n=t.getElementById(e);return n?[n]:[]}}):(r.filter.ID=function(e){var t=e.replace(Q,ee);return function(e){var n=void 0!==e.getAttributeNode&&e.getAttributeNode("id");return n&&n.value===t}},r.find.ID=function(e,t){if(void 0!==t.getElementById&&g){var n,r,i,o=t.getElementById(e);if(o){if((n=o.getAttributeNode("id"))&&n.value===e)return[o];for(i=t.getElementsByName(e),r=0;o=i[r++];)if((n=o.getAttributeNode("id"))&&n.value===e)return[o]}return[]}}),r.find.TAG=n.getElementsByTagName?function(e,t){return void 0!==t.getElementsByTagName?t.getElementsByTagName(e):n.qsa?t.querySelectorAll(e):void 0}:function(e,t){var n,r=[],i=0,o=t.getElementsByTagName(e);if("*"===e){for(;n=o[i++];)1===n.nodeType&&r.push(n);return r}return o},r.find.CLASS=n.getElementsByClassName&&function(e,t){if(void 0!==t.getElementsByClassName&&g)return t.getElementsByClassName(e)},y=[],v=[],(n.qsa=J.test(d.querySelectorAll))&&(ue(function(e){h.appendChild(e).innerHTML="<a id='"+b+"'></a><select id='"+b+"-\r\\' msallowcapture=''><option selected=''></option></select>",e.querySelectorAll("[msallowcapture^='']").length&&v.push("[*^$]="+O+"*(?:''|\"\")"),e.querySelectorAll("[selected]").length||v.push("\\["+O+"*(?:value|"+M+")"),e.querySelectorAll("[id~="+b+"-]").length||v.push("~="),e.querySelectorAll(":checked").length||v.push(":checked"),e.querySelectorAll("a#"+b+"+*").length||v.push(".#.+[+~]")}),ue(function(e){e.innerHTML="<a href='' disabled='disabled'></a><select disabled='disabled'><option/></select>";var t=d.createElement("input");t.setAttribute("type","hidden"),e.appendChild(t).setAttribute("name","D"),e.querySelectorAll("[name=d]").length&&v.push("name"+O+"*[*^$|!~]?="),2!==e.querySelectorAll(":enabled").length&&v.push(":enabled",":disabled"),h.appendChild(e).disabled=!0,2!==e.querySelectorAll(":disabled").length&&v.push(":enabled",":disabled"),e.querySelectorAll("*,:x"),v.push(",.*:")})),(n.matchesSelector=J.test(m=h.matches||h.webkitMatchesSelector||h.mozMatchesSelector||h.oMatchesSelector||h.msMatchesSelector))&&ue(function(e){n.disconnectedMatch=m.call(e,"*"),m.call(e,"[s!='']:x"),y.push("!=",$)}),v=v.length&&new RegExp(v.join("|")),y=y.length&&new RegExp(y.join("|")),t=J.test(h.compareDocumentPosition),x=t||J.test(h.contains)?function(e,t){var n=9===e.nodeType?e.documentElement:e,r=t&&t.parentNode;return e===r||!(!r||1!==r.nodeType||!(n.contains?n.contains(r):e.compareDocumentPosition&&16&e.compareDocumentPosition(r)))}:function(e,t){if(t)for(;t=t.parentNode;)if(t===e)return!0;return!1},D=t?function(e,t){if(e===t)return l=!0,0;var r=!e.compareDocumentPosition-!t.compareDocumentPosition;return r||(1&(r=(e.ownerDocument||e)===(t.ownerDocument||t)?e.compareDocumentPosition(t):1)||!n.sortDetached&&t.compareDocumentPosition(e)===r?e===d||e.ownerDocument===w&&x(w,e)?-1:t===d||t.ownerDocument===w&&x(w,t)?1:f?P(f,e)-P(f,t):0:4&r?-1:1)}:function(e,t){if(e===t)return l=!0,0;var n,r=0,i=e.parentNode,o=t.parentNode,s=[e],a=[t];if(!i||!o)return e===d?-1:t===d?1:i?-1:o?1:f?P(f,e)-P(f,t):0;if(i===o)return fe(e,t);for(n=e;n=n.parentNode;)s.unshift(n);for(n=t;n=n.parentNode;)a.unshift(n);for(;s[r]===a[r];)r++;return r?fe(s[r],a[r]):s[r]===w?-1:a[r]===w?1:0},d):d},oe.matches=function(e,t){return oe(e,null,null,t)},oe.matchesSelector=function(e,t){if((e.ownerDocument||e)!==d&&p(e),t=t.replace(z,"='$1']"),n.matchesSelector&&g&&!E[t+" "]&&(!y||!y.test(t))&&(!v||!v.test(t)))try{var r=m.call(e,t);if(r||n.disconnectedMatch||e.document&&11!==e.document.nodeType)return r}catch(Ce){}return oe(t,d,null,[e]).length>0},oe.contains=function(e,t){return(e.ownerDocument||e)!==d&&p(e),x(e,t)},oe.attr=function(e,t){(e.ownerDocument||e)!==d&&p(e);var i=r.attrHandle[t.toLowerCase()],o=i&&A.call(r.attrHandle,t.toLowerCase())?i(e,t,!g):void 0;return void 0!==o?o:n.attributes||!g?e.getAttribute(t):(o=e.getAttributeNode(t))&&o.specified?o.value:null},oe.escape=function(e){return(e+"").replace(te,ne)},oe.error=function(e){throw new Error("Syntax error, unrecognized expression: "+e)},oe.uniqueSort=function(e){var t,r=[],i=0,o=0;if(l=!n.detectDuplicates,f=!n.sortStable&&e.slice(0),e.sort(D),l){for(;t=e[o++];)t===e[o]&&(i=r.push(o));for(;i--;)e.splice(r[i],1)}return f=null,e},i=oe.getText=function(e){var t,n="",r=0,o=e.nodeType;if(o){if(1===o||9===o||11===o){if("string"==typeof e.textContent)return e.textContent;for(e=e.firstChild;e;e=e.nextSibling)n+=i(e)}else if(3===o||4===o)return e.nodeValue}else for(;t=e[r++];)n+=i(t);return n},(r=oe.selectors={cacheLength:50,createPseudo:ae,match:X,attrHandle:{},find:{},relative:{">":{dir:"parentNode",first:!0}," ":{dir:"parentNode"},"+":{dir:"previousSibling",first:!0},"~":{dir:"previousSibling"}},preFilter:{ATTR:function(e){return e[1]=e[1].replace(Q,ee),e[3]=(e[3]||e[4]||e[5]||"").replace(Q,ee),"~="===e[2]&&(e[3]=" "+e[3]+" "),e.slice(0,4)},CHILD:function(e){return e[1]=e[1].toLowerCase(),"nth"===e[1].slice(0,3)?(e[3]||oe.error(e[0]),e[4]=+(e[4]?e[5]+(e[6]||1):2*("even"===e[3]||"odd"===e[3])),e[5]=+(e[7]+e[8]||"odd"===e[3])):e[3]&&oe.error(e[0]),e},PSEUDO:function(e){var t,n=!e[6]&&e[2];return X.CHILD.test(e[0])?null:(e[3]?e[2]=e[4]||e[5]||"":n&&U.test(n)&&(t=s(n,!0))&&(t=n.indexOf(")",n.length-t)-n.length)&&(e[0]=e[0].slice(0,t),e[2]=n.slice(0,t)),e.slice(0,3))}},filter:{TAG:function(e){var t=e.replace(Q,ee).toLowerCase();return"*"===e?function(){return!0}:function(e){return e.nodeName&&e.nodeName.toLowerCase()===t}},CLASS:function(e){var t=S[e+" "];return t||(t=new RegExp("(^|"+O+")"+e+"("+O+"|$)"))&&S(e,function(e){return t.test("string"==typeof e.className&&e.className||void 0!==e.getAttribute&&e.getAttribute("class")||"")})},ATTR:function(e,t,n){return function(r){var i=oe.attr(r,e);return null==i?"!="===t:!t||(i+="","="===t?i===n:"!="===t?i!==n:"^="===t?n&&0===i.indexOf(n):"*="===t?n&&i.indexOf(n)>-1:"$="===t?n&&i.slice(-n.length)===n:"~="===t?(" "+i.replace(B," ")+" ").indexOf(n)>-1:"|="===t&&(i===n||i.slice(0,n.length+1)===n+"-"))}},CHILD:function(e,t,n,r,i){var o="nth"!==e.slice(0,3),s="last"!==e.slice(-4),a="of-type"===t;return 1===r&&0===i?function(e){return!!e.parentNode}:function(t,n,u){var c,f,l,p,d,h,g=o!==s?"nextSibling":"previousSibling",v=t.parentNode,y=a&&t.nodeName.toLowerCase(),m=!u&&!a,x=!1;if(v){if(o){for(;g;){for(p=t;p=p[g];)if(a?p.nodeName.toLowerCase()===y:1===p.nodeType)return!1;h=g="only"===e&&!h&&"nextSibling"}return!0}if(h=[s?v.firstChild:v.lastChild],s&&m){for(x=(d=(c=(f=(l=(p=v)[b]||(p[b]={}))[p.uniqueID]||(l[p.uniqueID]={}))[e]||[])[0]===T&&c[1])&&c[2],p=d&&v.childNodes[d];p=++d&&p&&p[g]||(x=d=0)||h.pop();)if(1===p.nodeType&&++x&&p===t){f[e]=[T,d,x];break}}else if(m&&(x=d=(c=(f=(l=(p=t)[b]||(p[b]={}))[p.uniqueID]||(l[p.uniqueID]={}))[e]||[])[0]===T&&c[1]),!1===x)for(;(p=++d&&p&&p[g]||(x=d=0)||h.pop())&&((a?p.nodeName.toLowerCase()!==y:1!==p.nodeType)||!++x||(m&&((f=(l=p[b]||(p[b]={}))[p.uniqueID]||(l[p.uniqueID]={}))[e]=[T,x]),p!==t)););return(x-=i)===r||x%r==0&&x/r>=0}}},PSEUDO:function(e,t){var n,i=r.pseudos[e]||r.setFilters[e.toLowerCase()]||oe.error("unsupported pseudo: "+e);return i[b]?i(t):i.length>1?(n=[e,e,"",t],r.setFilters.hasOwnProperty(e.toLowerCase())?ae(function(e,n){for(var r,o=i(e,t),s=o.length;s--;)e[r=P(e,o[s])]=!(n[r]=o[s])}):function(e){return i(e,0,n)}):i}},pseudos:{not:ae(function(e){var t=[],n=[],r=a(e.replace(I,"$1"));return r[b]?ae(function(e,t,n,i){for(var o,s=r(e,null,i,[]),a=e.length;a--;)(o=s[a])&&(e[a]=!(t[a]=o))}):function(e,i,o){return t[0]=e,r(t,null,o,n),t[0]=null,!n.pop()}}),has:ae(function(e){return function(t){return oe(e,t).length>0}}),contains:ae(function(e){return e=e.replace(Q,ee),function(t){return(t.textContent||t.innerText||i(t)).indexOf(e)>-1}}),lang:ae(function(e){return V.test(e||"")||oe.error("unsupported lang: "+e),e=e.replace(Q,ee).toLowerCase(),function(t){var n;do{if(n=g?t.lang:t.getAttribute("xml:lang")||t.getAttribute("lang"))return(n=n.toLowerCase())===e||0===n.indexOf(e+"-")}while((t=t.parentNode)&&1===t.nodeType);return!1}}),target:function(t){var n=e.location&&e.location.hash;return n&&n.slice(1)===t.id},root:function(e){return e===h},focus:function(e){return e===d.activeElement&&(!d.hasFocus||d.hasFocus())&&!!(e.type||e.href||~e.tabIndex)},enabled:de(!1),disabled:de(!0),checked:function(e){var t=e.nodeName.toLowerCase();return"input"===t&&!!e.checked||"option"===t&&!!e.selected},selected:function(e){return e.parentNode&&e.parentNode.selectedIndex,!0===e.selected},empty:function(e){for(e=e.firstChild;e;e=e.nextSibling)if(e.nodeType<6)return!1;return!0},parent:function(e){return!r.pseudos.empty(e)},header:function(e){return G.test(e.nodeName)},input:function(e){return Y.test(e.nodeName)},button:function(e){var t=e.nodeName.toLowerCase();return"input"===t&&"button"===e.type||"button"===t},text:function(e){var t;return"input"===e.nodeName.toLowerCase()&&"text"===e.type&&(null==(t=e.getAttribute("type"))||"text"===t.toLowerCase())},first:he(function(){return[0]}),last:he(function(e,t){return[t-1]}),eq:he(function(e,t,n){return[n<0?n+t:n]}),even:he(function(e,t){for(var n=0;n<t;n+=2)e.push(n);return e}),odd:he(function(e,t){for(var n=1;n<t;n+=2)e.push(n);return e}),lt:he(function(e,t,n){for(var r=n<0?n+t:n;--r>=0;)e.push(r);return e}),gt:he(function(e,t,n){for(var r=n<0?n+t:n;++r<t;)e.push(r);return e})}}).pseudos.nth=r.pseudos.eq,{radio:!0,checkbox:!0,file:!0,password:!0,image:!0})r.pseudos[t]=le(t);for(t in{submit:!0,reset:!0})r.pseudos[t]=pe(t);function ve(){}function ye(e){for(var t=0,n=e.length,r="";t<n;t++)r+=e[t].value;return r}function me(e,t,n){var r=t.dir,i=t.next,o=i||r,s=n&&"parentNode"===o,a=C++;return t.first?function(t,n,i){for(;t=t[r];)if(1===t.nodeType||s)return e(t,n,i);return!1}:function(t,n,u){var c,f,l,p=[T,a];if(u){for(;t=t[r];)if((1===t.nodeType||s)&&e(t,n,u))return!0}else for(;t=t[r];)if(1===t.nodeType||s)if(f=(l=t[b]||(t[b]={}))[t.uniqueID]||(l[t.uniqueID]={}),i&&i===t.nodeName.toLowerCase())t=t[r]||t;else{if((c=f[o])&&c[0]===T&&c[1]===a)return p[2]=c[2];if(f[o]=p,p[2]=e(t,n,u))return!0}return!1}}function xe(e){return e.length>1?function(t,n,r){for(var i=e.length;i--;)if(!e[i](t,n,r))return!1;return!0}:e[0]}function be(e,t,n,r,i){for(var o,s=[],a=0,u=e.length,c=null!=t;a<u;a++)(o=e[a])&&(n&&!n(o,r,i)||(s.push(o),c&&t.push(a)));return s}function we(e,t,n,r,i,o){return r&&!r[b]&&(r=we(r)),i&&!i[b]&&(i=we(i,o)),ae(function(o,s,a,u){var c,f,l,p=[],d=[],h=s.length,g=o||function(e,t,n){for(var r=0,i=t.length;r<i;r++)oe(e,t[r],n);return n}(t||"*",a.nodeType?[a]:a,[]),v=!e||!o&&t?g:be(g,p,e,a,u),y=n?i||(o?e:h||r)?[]:s:v;if(n&&n(v,y,a,u),r)for(c=be(y,d),r(c,[],a,u),f=c.length;f--;)(l=c[f])&&(y[d[f]]=!(v[d[f]]=l));if(o){if(i||e){if(i){for(c=[],f=y.length;f--;)(l=y[f])&&c.push(v[f]=l);i(null,y=[],c,u)}for(f=y.length;f--;)(l=y[f])&&(c=i?P(o,l):p[f])>-1&&(o[c]=!(s[c]=l))}}else y=be(y===s?y.splice(h,y.length):y),i?i(null,s,y,u):H.apply(s,y)})}function Te(e){for(var t,n,i,o=e.length,s=r.relative[e[0].type],a=s||r.relative[" "],u=s?1:0,f=me(function(e){return e===t},a,!0),l=me(function(e){return P(t,e)>-1},a,!0),p=[function(e,n,r){var i=!s&&(r||n!==c)||((t=n).nodeType?f(e,n,r):l(e,n,r));return t=null,i}];u<o;u++)if(n=r.relative[e[u].type])p=[me(xe(p),n)];else{if((n=r.filter[e[u].type].apply(null,e[u].matches))[b]){for(i=++u;i<o&&!r.relative[e[i].type];i++);return we(u>1&&xe(p),u>1&&ye(e.slice(0,u-1).concat({value:" "===e[u-2].type?"*":""})).replace(I,"$1"),n,u<i&&Te(e.slice(u,i)),i<o&&Te(e=e.slice(i)),i<o&&ye(e))}p.push(n)}return xe(p)}return ve.prototype=r.filters=r.pseudos,r.setFilters=new ve,s=oe.tokenize=function(e,t){var n,i,o,s,a,u,c,f=k[e+" "];if(f)return t?0:f.slice(0);for(a=e,u=[],c=r.preFilter;a;){for(s in n&&!(i=F.exec(a))||(i&&(a=a.slice(i[0].length)||a),u.push(o=[])),n=!1,(i=_.exec(a))&&(n=i.shift(),o.push({value:n,type:i[0].replace(I," ")}),a=a.slice(n.length)),r.filter)!(i=X[s].exec(a))||c[s]&&!(i=c[s](i))||(n=i.shift(),o.push({value:n,type:s,matches:i}),a=a.slice(n.length));if(!n)break}return t?a.length:a?oe.error(e):k(e,u).slice(0)},a=oe.compile=function(e,t){var n,i=[],o=[],a=E[e+" "];if(!a){for(t||(t=s(e)),n=t.length;n--;)(a=Te(t[n]))[b]?i.push(a):o.push(a);(a=E(e,function(e,t){var n=t.length>0,i=e.length>0,o=function(o,s,a,u,f){var l,h,v,y=0,m="0",x=o&&[],b=[],w=c,C=o||i&&r.find.TAG("*",f),S=T+=null==w?1:Math.random()||.1,k=C.length;for(f&&(c=s===d||s||f);m!==k&&null!=(l=C[m]);m++){if(i&&l){for(h=0,s||l.ownerDocument===d||(p(l),a=!g);v=e[h++];)if(v(l,s||d,a)){u.push(l);break}f&&(T=S)}n&&((l=!v&&l)&&y--,o&&x.push(l))}if(y+=m,n&&m!==y){for(h=0;v=t[h++];)v(x,b,s,a);if(o){if(y>0)for(;m--;)x[m]||b[m]||(b[m]=L.call(u));b=be(b)}H.apply(u,b),f&&!o&&b.length>0&&y+t.length>1&&oe.uniqueSort(u)}return f&&(T=S,c=w),x};return n?ae(o):o}(o,i))).selector=e}return a},u=oe.select=function(e,t,n,i){var o,u,c,f,l,p="function"==typeof e&&e,d=!i&&s(e=p.selector||e);if(n=n||[],1===d.length){if((u=d[0]=d[0].slice(0)).length>2&&"ID"===(c=u[0]).type&&9===t.nodeType&&g&&r.relative[u[1].type]){if(!(t=(r.find.ID(c.matches[0].replace(Q,ee),t)||[])[0]))return n;p&&(t=t.parentNode),e=e.slice(u.shift().value.length)}for(o=X.needsContext.test(e)?0:u.length;o--&&(c=u[o],!r.relative[f=c.type]);)if((l=r.find[f])&&(i=l(c.matches[0].replace(Q,ee),Z.test(u[0].type)&&ge(t.parentNode)||t))){if(u.splice(o,1),!(e=i.length&&ye(u)))return H.apply(n,i),n;break}}return(p||a(e,d))(i,t,!g,n,!t||Z.test(e)&&ge(t.parentNode)||t),n},n.sortStable=b.split("").sort(D).join("")===b,n.detectDuplicates=!!l,p(),n.sortDetached=ue(function(e){return 1&e.compareDocumentPosition(d.createElement("fieldset"))}),ue(function(e){return e.innerHTML="<a href='#'></a>","#"===e.firstChild.getAttribute("href")})||ce("type|href|height|width",function(e,t,n){if(!n)return e.getAttribute(t,"type"===t.toLowerCase()?1:2)}),n.attributes&&ue(function(e){return e.innerHTML="<input/>",e.firstChild.setAttribute("value",""),""===e.firstChild.getAttribute("value")})||ce("value",function(e,t,n){if(!n&&"input"===e.nodeName.toLowerCase())return e.defaultValue}),ue(function(e){return null==e.getAttribute("disabled")})||ce(M,function(e,t,n){var r;if(!n)return!0===e[t]?t.toLowerCase():(r=e.getAttributeNode(t))&&r.specified?r.value:null}),oe}(t);w.find=S,w.expr=S.selectors,w.expr[":"]=w.expr.pseudos,w.uniqueSort=w.unique=S.uniqueSort,w.text=S.getText,w.isXMLDoc=S.isXML,w.contains=S.contains,w.escapeSelector=S.escape;var k=function(e,t,n){for(var r=[],i=void 0!==n;(e=e[t])&&9!==e.nodeType;)if(1===e.nodeType){if(i&&w(e).is(n))break;r.push(e)}return r},E=function(e,t){for(var n=[];e;e=e.nextSibling)1===e.nodeType&&e!==t&&n.push(e);return n},D=w.expr.match.needsContext;function A(e,t){return e.nodeName&&e.nodeName.toLowerCase()===t.toLowerCase()}var N=/^<([a-z][^\/\0>:\x20\t\r\n\f]*)[\x20\t\r\n\f]*\/?>(?:<\/\1>|)$/i;function L(e,t,n){return v(t)?w.grep(e,function(e,r){return!!t.call(e,r,e)!==n}):t.nodeType?w.grep(e,function(e){return e===t!==n}):"string"!=typeof t?w.grep(e,function(e){return c.call(t,e)>-1!==n}):w.filter(t,e,n)}w.filter=function(e,t,n){var r=t[0];return n&&(e=":not("+e+")"),1===t.length&&1===r.nodeType?w.find.matchesSelector(r,e)?[r]:[]:w.find.matches(e,w.grep(t,function(e){return 1===e.nodeType}))},w.fn.extend({find:function(e){var t,n,r=this.length,i=this;if("string"!=typeof e)return this.pushStack(w(e).filter(function(){for(t=0;t<r;t++)if(w.contains(i[t],this))return!0}));for(n=this.pushStack([]),t=0;t<r;t++)w.find(e,i[t],n);return r>1?w.uniqueSort(n):n},filter:function(e){return this.pushStack(L(this,e||[],!1))},not:function(e){return this.pushStack(L(this,e||[],!0))},is:function(e){return!!L(this,"string"==typeof e&&D.test(e)?w(e):e||[],!1).length}});var q,H=/^(?:\s*(<[\w\W]+>)[^>]*|#([\w-]+))$/;(w.fn.init=function(e,t,n){var r,o;if(!e)return this;if(n=n||q,"string"==typeof e){if(!(r="<"===e[0]&&">"===e[e.length-1]&&e.length>=3?[null,e,null]:H.exec(e))||!r[1]&&t)return!t||t.jquery?(t||n).find(e):this.constructor(t).find(e);if(r[1]){if(t=t instanceof w?t[0]:t,w.merge(this,w.parseHTML(r[1],t&&t.nodeType?t.ownerDocument||t:i,!0)),N.test(r[1])&&w.isPlainObject(t))for(r in t)v(this[r])?this[r](t[r]):this.attr(r,t[r]);return this}return(o=i.getElementById(r[2]))&&(this[0]=o,this.length=1),this}return e.nodeType?(this[0]=e,this.length=1,this):v(e)?void 0!==n.ready?n.ready(e):e(w):w.makeArray(e,this)}).prototype=w.fn,q=w(i);var j=/^(?:parents|prev(?:Until|All))/,P={children:!0,contents:!0,next:!0,prev:!0};function M(e,t){for(;(e=e[t])&&1!==e.nodeType;);return e}w.fn.extend({has:function(e){var t=w(e,this),n=t.length;return this.filter(function(){for(var e=0;e<n;e++)if(w.contains(this,t[e]))return!0})},closest:function(e,t){var n,r=0,i=this.length,o=[],s="string"!=typeof e&&w(e);if(!D.test(e))for(;r<i;r++)for(n=this[r];n&&n!==t;n=n.parentNode)if(n.nodeType<11&&(s?s.index(n)>-1:1===n.nodeType&&w.find.matchesSelector(n,e))){o.push(n);break}return this.pushStack(o.length>1?w.uniqueSort(o):o)},index:function(e){return e?"string"==typeof e?c.call(w(e),this[0]):c.call(this,e.jquery?e[0]:e):this[0]&&this[0].parentNode?this.first().prevAll().length:-1},add:function(e,t){return this.pushStack(w.uniqueSort(w.merge(this.get(),w(e,t))))},addBack:function(e){return this.add(null==e?this.prevObject:this.prevObject.filter(e))}}),w.each({parent:function(e){var t=e.parentNode;return t&&11!==t.nodeType?t:null},parents:function(e){return k(e,"parentNode")},parentsUntil:function(e,t,n){return k(e,"parentNode",n)},next:function(e){return M(e,"nextSibling")},prev:function(e){return M(e,"previousSibling")},nextAll:function(e){return k(e,"nextSibling")},prevAll:function(e){return k(e,"previousSibling")},nextUntil:function(e,t,n){return k(e,"nextSibling",n)},prevUntil:function(e,t,n){return k(e,"previousSibling",n)},siblings:function(e){return E((e.parentNode||{}).firstChild,e)},children:function(e){return E(e.firstChild)},contents:function(e){return A(e,"iframe")?e.contentDocument:(A(e,"template")&&(e=e.content||e),w.merge([],e.childNodes))}},function(e,t){w.fn[e]=function(n,r){var i=w.map(this,t,n);return"Until"!==e.slice(-5)&&(r=n),r&&"string"==typeof r&&(i=w.filter(r,i)),this.length>1&&(P[e]||w.uniqueSort(i),j.test(e)&&i.reverse()),this.pushStack(i)}});var O=/[^\x20\t\r\n\f]+/g;function R(e){return e}function W(e){throw e}function $(e,t,n,r){var i;try{e&&v(i=e.promise)?i.call(e).done(t).fail(n):e&&v(i=e.then)?i.call(e,t,n):t.apply(void 0,[e].slice(r))}catch(e){n.apply(void 0,[e])}}w.Callbacks=function(e){e="string"==typeof e?function(e){var t={};return w.each(e.match(O)||[],function(e,n){t[n]=!0}),t}(e):w.extend({},e);var t,n,r,i,o=[],s=[],a=-1,u=function(){for(i=i||e.once,r=t=!0;s.length;a=-1)for(n=s.shift();++a<o.length;)!1===o[a].apply(n[0],n[1])&&e.stopOnFalse&&(a=o.length,n=!1);e.memory||(n=!1),t=!1,i&&(o=n?[]:"")},c={add:function(){return o&&(n&&!t&&(a=o.length-1,s.push(n)),function t(n){w.each(n,function(n,r){v(r)?e.unique&&c.has(r)||o.push(r):r&&r.length&&"string"!==b(r)&&t(r)})}(arguments),n&&!t&&u()),this},remove:function(){return w.each(arguments,function(e,t){for(var n;(n=w.inArray(t,o,n))>-1;)o.splice(n,1),n<=a&&a--}),this},has:function(e){return e?w.inArray(e,o)>-1:o.length>0},empty:function(){return o&&(o=[]),this},disable:function(){return i=s=[],o=n="",this},disabled:function(){return!o},lock:function(){return i=s=[],n||t||(o=n=""),this},locked:function(){return!!i},fireWith:function(e,n){return i||(n=[e,(n=n||[]).slice?n.slice():n],s.push(n),t||u()),this},fire:function(){return c.fireWith(this,arguments),this},fired:function(){return!!r}};return c},w.extend({Deferred:function(e){var n=[["notify","progress",w.Callbacks("memory"),w.Callbacks("memory"),2],["resolve","done",w.Callbacks("once memory"),w.Callbacks("once memory"),0,"resolved"],["reject","fail",w.Callbacks("once memory"),w.Callbacks("once memory"),1,"rejected"]],r="pending",i={state:function(){return r},always:function(){return o.done(arguments).fail(arguments),this},catch:function(e){return i.then(null,e)},pipe:function(){var e=arguments;return w.Deferred(function(t){w.each(n,function(n,r){var i=v(e[r[4]])&&e[r[4]];o[r[1]](function(){var e=i&&i.apply(this,arguments);e&&v(e.promise)?e.promise().progress(t.notify).done(t.resolve).fail(t.reject):t[r[0]+"With"](this,i?[e]:arguments)})}),e=null}).promise()},then:function(e,r,i){var o=0;function s(e,n,r,i){return function(){var a=this,u=arguments,c=function(){var t,c;if(!(e<o)){if((t=r.apply(a,u))===n.promise())throw new TypeError("Thenable self-resolution");c=t&&("object"==typeof t||"function"==typeof t)&&t.then,v(c)?i?c.call(t,s(o,n,R,i),s(o,n,W,i)):(o++,c.call(t,s(o,n,R,i),s(o,n,W,i),s(o,n,R,n.notifyWith))):(r!==R&&(a=void 0,u=[t]),(i||n.resolveWith)(a,u))}},f=i?c:function(){try{c()}catch(t){w.Deferred.exceptionHook&&w.Deferred.exceptionHook(t,f.stackTrace),e+1>=o&&(r!==W&&(a=void 0,u=[t]),n.rejectWith(a,u))}};e?f():(w.Deferred.getStackHook&&(f.stackTrace=w.Deferred.getStackHook()),t.setTimeout(f))}}return w.Deferred(function(t){n[0][3].add(s(0,t,v(i)?i:R,t.notifyWith)),n[1][3].add(s(0,t,v(e)?e:R)),n[2][3].add(s(0,t,v(r)?r:W))}).promise()},promise:function(e){return null!=e?w.extend(e,i):i}},o={};return w.each(n,function(e,t){var s=t[2],a=t[5];i[t[1]]=s.add,a&&s.add(function(){r=a},n[3-e][2].disable,n[3-e][3].disable,n[0][2].lock,n[0][3].lock),s.add(t[3].fire),o[t[0]]=function(){return o[t[0]+"With"](this===o?void 0:this,arguments),this},o[t[0]+"With"]=s.fireWith}),i.promise(o),e&&e.call(o,o),o},when:function(e){var t=arguments.length,n=t,r=Array(n),i=s.call(arguments),o=w.Deferred(),a=function(e){return function(n){r[e]=this,i[e]=arguments.length>1?s.call(arguments):n,--t||o.resolveWith(r,i)}};if(t<=1&&($(e,o.done(a(n)).resolve,o.reject,!t),"pending"===o.state()||v(i[n]&&i[n].then)))return o.then();for(;n--;)$(i[n],a(n),o.reject);return o.promise()}});var B=/^(Eval|Internal|Range|Reference|Syntax|Type|URI)Error$/;w.Deferred.exceptionHook=function(e,n){t.console&&t.console.warn&&e&&B.test(e.name)&&t.console.warn("jQuery.Deferred exception: "+e.message,e.stack,n)},w.readyException=function(e){t.setTimeout(function(){throw e})};var I=w.Deferred();function F(){i.removeEventListener("DOMContentLoaded",F),t.removeEventListener("load",F),w.ready()}w.fn.ready=function(e){return I.then(e).catch(function(e){w.readyException(e)}),this},w.extend({isReady:!1,readyWait:1,ready:function(e){(!0===e?--w.readyWait:w.isReady)||(w.isReady=!0,!0!==e&&--w.readyWait>0||I.resolveWith(i,[w]))}}),w.ready.then=I.then,"complete"===i.readyState||"loading"!==i.readyState&&!i.documentElement.doScroll?t.setTimeout(w.ready):(i.addEventListener("DOMContentLoaded",F),t.addEventListener("load",F));var _=function(e,t,n,r,i,o,s){var a=0,u=e.length,c=null==n;if("object"===b(n))for(a in i=!0,n)_(e,t,a,n[a],!0,o,s);else if(void 0!==r&&(i=!0,v(r)||(s=!0),c&&(s?(t.call(e,r),t=null):(c=t,t=function(e,t,n){return c.call(w(e),n)})),t))for(;a<u;a++)t(e[a],n,s?r:r.call(e[a],a,t(e[a],n)));return i?e:c?t.call(e):u?t(e[0],n):o},z=/^-ms-/,U=/-([a-z])/g;function V(e,t){return t.toUpperCase()}function X(e){return e.replace(z,"ms-").replace(U,V)}var Y=function(e){return 1===e.nodeType||9===e.nodeType||!+e.nodeType};function G(){this.expando=w.expando+G.uid++}G.uid=1,G.prototype={cache:function(e){var t=e[this.expando];return t||(t={},Y(e)&&(e.nodeType?e[this.expando]=t:Object.defineProperty(e,this.expando,{value:t,configurable:!0}))),t},set:function(e,t,n){var r,i=this.cache(e);if("string"==typeof t)i[X(t)]=n;else for(r in t)i[X(r)]=t[r];return i},get:function(e,t){return void 0===t?this.cache(e):e[this.expando]&&e[this.expando][X(t)]},access:function(e,t,n){return void 0===t||t&&"string"==typeof t&&void 0===n?this.get(e,t):(this.set(e,t,n),void 0!==n?n:t)},remove:function(e,t){var n,r=e[this.expando];if(void 0!==r){if(void 0!==t){n=(t=Array.isArray(t)?t.map(X):(t=X(t))in r?[t]:t.match(O)||[]).length;for(;n--;)delete r[t[n]]}(void 0===t||w.isEmptyObject(r))&&(e.nodeType?e[this.expando]=void 0:delete e[this.expando])}},hasData:function(e){var t=e[this.expando];return void 0!==t&&!w.isEmptyObject(t)}};var J=new G,K=new G,Z=/^(?:\{[\w\W]*\}|\[[\w\W]*\])$/,Q=/[A-Z]/g;function ee(e,t,n){var r;if(void 0===n&&1===e.nodeType)if(r="data-"+t.replace(Q,"-$&").toLowerCase(),"string"==typeof(n=e.getAttribute(r))){try{n=function(e){return"true"===e||"false"!==e&&("null"===e?null:e===+e+""?+e:Z.test(e)?JSON.parse(e):e)}(n)}catch(i){}K.set(e,t,n)}else n=void 0;return n}w.extend({hasData:function(e){return K.hasData(e)||J.hasData(e)},data:function(e,t,n){return K.access(e,t,n)},removeData:function(e,t){K.remove(e,t)},_data:function(e,t,n){return J.access(e,t,n)},_removeData:function(e,t){J.remove(e,t)}}),w.fn.extend({data:function(e,t){var n,r,i,o=this[0],s=o&&o.attributes;if(void 0===e){if(this.length&&(i=K.get(o),1===o.nodeType&&!J.get(o,"hasDataAttrs"))){for(n=s.length;n--;)s[n]&&0===(r=s[n].name).indexOf("data-")&&(r=X(r.slice(5)),ee(o,r,i[r]));J.set(o,"hasDataAttrs",!0)}return i}return"object"==typeof e?this.each(function(){K.set(this,e)}):_(this,function(t){var n;if(o&&void 0===t)return void 0!==(n=K.get(o,e))?n:void 0!==(n=ee(o,e))?n:void 0;this.each(function(){K.set(this,e,t)})},null,t,arguments.length>1,null,!0)},removeData:function(e){return this.each(function(){K.remove(this,e)})}}),w.extend({queue:function(e,t,n){var r;if(e)return t=(t||"fx")+"queue",r=J.get(e,t),n&&(!r||Array.isArray(n)?r=J.access(e,t,w.makeArray(n)):r.push(n)),r||[]},dequeue:function(e,t){t=t||"fx";var n=w.queue(e,t),r=n.length,i=n.shift(),o=w._queueHooks(e,t);"inprogress"===i&&(i=n.shift(),r--),i&&("fx"===t&&n.unshift("inprogress"),delete o.stop,i.call(e,function(){w.dequeue(e,t)},o)),!r&&o&&o.empty.fire()},_queueHooks:function(e,t){var n=t+"queueHooks";return J.get(e,n)||J.access(e,n,{empty:w.Callbacks("once memory").add(function(){J.remove(e,[t+"queue",n])})})}}),w.fn.extend({queue:function(e,t){var n=2;return"string"!=typeof e&&(t=e,e="fx",n--),arguments.length<n?w.queue(this[0],e):void 0===t?this:this.each(function(){var n=w.queue(this,e,t);w._queueHooks(this,e),"fx"===e&&"inprogress"!==n[0]&&w.dequeue(this,e)})},dequeue:function(e){return this.each(function(){w.dequeue(this,e)})},clearQueue:function(e){return this.queue(e||"fx",[])},promise:function(e,t){var n,r=1,i=w.Deferred(),o=this,s=this.length,a=function(){--r||i.resolveWith(o,[o])};for("string"!=typeof e&&(t=e,e=void 0),e=e||"fx";s--;)(n=J.get(o[s],e+"queueHooks"))&&n.empty&&(r++,n.empty.add(a));return a(),i.promise(t)}});var te=/[+-]?(?:\d*\.|)\d+(?:[eE][+-]?\d+|)/.source,ne=new RegExp("^(?:([+-])=|)("+te+")([a-z%]*)$","i"),re=["Top","Right","Bottom","Left"],ie=function(e,t){return"none"===(e=t||e).style.display||""===e.style.display&&w.contains(e.ownerDocument,e)&&"none"===w.css(e,"display")},oe=function(e,t,n,r){var i,o,s={};for(o in t)s[o]=e.style[o],e.style[o]=t[o];for(o in i=n.apply(e,r||[]),t)e.style[o]=s[o];return i};function se(e,t,n,r){var i,o,s=20,a=r?function(){return r.cur()}:function(){return w.css(e,t,"")},u=a(),c=n&&n[3]||(w.cssNumber[t]?"":"px"),f=(w.cssNumber[t]||"px"!==c&&+u)&&ne.exec(w.css(e,t));if(f&&f[3]!==c){for(u/=2,c=c||f[3],f=+u||1;s--;)w.style(e,t,f+c),(1-o)*(1-(o=a()/u||.5))<=0&&(s=0),f/=o;f*=2,w.style(e,t,f+c),n=n||[]}return n&&(f=+f||+u||0,i=n[1]?f+(n[1]+1)*n[2]:+n[2],r&&(r.unit=c,r.start=f,r.end=i)),i}var ae={};function ue(e){var t,n=e.ownerDocument,r=e.nodeName,i=ae[r];return i||(t=n.body.appendChild(n.createElement(r)),i=w.css(t,"display"),t.parentNode.removeChild(t),"none"===i&&(i="block"),ae[r]=i,i)}function ce(e,t){for(var n,r,i=[],o=0,s=e.length;o<s;o++)(r=e[o]).style&&(n=r.style.display,t?("none"===n&&(i[o]=J.get(r,"display")||null,i[o]||(r.style.display="")),""===r.style.display&&ie(r)&&(i[o]=ue(r))):"none"!==n&&(i[o]="none",J.set(r,"display",n)));for(o=0;o<s;o++)null!=i[o]&&(e[o].style.display=i[o]);return e}w.fn.extend({show:function(){return ce(this,!0)},hide:function(){return ce(this)},toggle:function(e){return"boolean"==typeof e?e?this.show():this.hide():this.each(function(){ie(this)?w(this).show():w(this).hide()})}});var fe=/^(?:checkbox|radio)$/i,le=/<([a-z][^\/\0>\x20\t\r\n\f]+)/i,pe=/^$|^module$|\/(?:java|ecma)script/i,de={option:[1,"<select multiple='multiple'>","</select>"],thead:[1,"<table>","</table>"],col:[2,"<table><colgroup>","</colgroup></table>"],tr:[2,"<table><tbody>","</tbody></table>"],td:[3,"<table><tbody><tr>","</tr></tbody></table>"],_default:[0,"",""]};function he(e,t){var n;return n=void 0!==e.getElementsByTagName?e.getElementsByTagName(t||"*"):void 0!==e.querySelectorAll?e.querySelectorAll(t||"*"):[],void 0===t||t&&A(e,t)?w.merge([e],n):n}function ge(e,t){for(var n=0,r=e.length;n<r;n++)J.set(e[n],"globalEval",!t||J.get(t[n],"globalEval"))}de.optgroup=de.option,de.tbody=de.tfoot=de.colgroup=de.caption=de.thead,de.th=de.td;var ve,ye,me=/<|&#?\w+;/;function xe(e,t,n,r,i){for(var o,s,a,u,c,f,l=t.createDocumentFragment(),p=[],d=0,h=e.length;d<h;d++)if((o=e[d])||0===o)if("object"===b(o))w.merge(p,o.nodeType?[o]:o);else if(me.test(o)){for(s=s||l.appendChild(t.createElement("div")),a=(le.exec(o)||["",""])[1].toLowerCase(),u=de[a]||de._default,s.innerHTML=u[1]+w.htmlPrefilter(o)+u[2],f=u[0];f--;)s=s.lastChild;w.merge(p,s.childNodes),(s=l.firstChild).textContent=""}else p.push(t.createTextNode(o));for(l.textContent="",d=0;o=p[d++];)if(r&&w.inArray(o,r)>-1)i&&i.push(o);else if(c=w.contains(o.ownerDocument,o),s=he(l.appendChild(o),"script"),c&&ge(s),n)for(f=0;o=s[f++];)pe.test(o.type||"")&&n.push(o);return l}ve=i.createDocumentFragment().appendChild(i.createElement("div")),(ye=i.createElement("input")).setAttribute("type","radio"),ye.setAttribute("checked","checked"),ye.setAttribute("name","t"),ve.appendChild(ye),g.checkClone=ve.cloneNode(!0).cloneNode(!0).lastChild.checked,ve.innerHTML="<textarea>x</textarea>",g.noCloneChecked=!!ve.cloneNode(!0).lastChild.defaultValue;var be=i.documentElement,we=/^key/,Te=/^(?:mouse|pointer|contextmenu|drag|drop)|click/,Ce=/^([^.]*)(?:\.(.+)|)/;function Se(){return!0}function ke(){return!1}function Ee(){try{return i.activeElement}catch(e){}}function De(e,t,n,r,i,o){var s,a;if("object"==typeof t){for(a in"string"!=typeof n&&(r=r||n,n=void 0),t)De(e,a,n,r,t[a],o);return e}if(null==r&&null==i?(i=n,r=n=void 0):null==i&&("string"==typeof n?(i=r,r=void 0):(i=r,r=n,n=void 0)),!1===i)i=ke;else if(!i)return e;return 1===o&&(s=i,(i=function(e){return w().off(e),s.apply(this,arguments)}).guid=s.guid||(s.guid=w.guid++)),e.each(function(){w.event.add(this,t,i,r,n)})}w.event={global:{},add:function(e,t,n,r,i){var o,s,a,u,c,f,l,p,d,h,g,v=J.get(e);if(v)for(n.handler&&(n=(o=n).handler,i=o.selector),i&&w.find.matchesSelector(be,i),n.guid||(n.guid=w.guid++),(u=v.events)||(u=v.events={}),(s=v.handle)||(s=v.handle=function(t){return void 0!==w&&w.event.triggered!==t.type?w.event.dispatch.apply(e,arguments):void 0}),c=(t=(t||"").match(O)||[""]).length;c--;)d=g=(a=Ce.exec(t[c])||[])[1],h=(a[2]||"").split(".").sort(),d&&(l=w.event.special[d]||{},d=(i?l.delegateType:l.bindType)||d,l=w.event.special[d]||{},f=w.extend({type:d,origType:g,data:r,handler:n,guid:n.guid,selector:i,needsContext:i&&w.expr.match.needsContext.test(i),namespace:h.join(".")},o),(p=u[d])||((p=u[d]=[]).delegateCount=0,l.setup&&!1!==l.setup.call(e,r,h,s)||e.addEventListener&&e.addEventListener(d,s)),l.add&&(l.add.call(e,f),f.handler.guid||(f.handler.guid=n.guid)),i?p.splice(p.delegateCount++,0,f):p.push(f),w.event.global[d]=!0)},remove:function(e,t,n,r,i){var o,s,a,u,c,f,l,p,d,h,g,v=J.hasData(e)&&J.get(e);if(v&&(u=v.events)){for(c=(t=(t||"").match(O)||[""]).length;c--;)if(d=g=(a=Ce.exec(t[c])||[])[1],h=(a[2]||"").split(".").sort(),d){for(l=w.event.special[d]||{},p=u[d=(r?l.delegateType:l.bindType)||d]||[],a=a[2]&&new RegExp("(^|\\.)"+h.join("\\.(?:.*\\.|)")+"(\\.|$)"),s=o=p.length;o--;)f=p[o],!i&&g!==f.origType||n&&n.guid!==f.guid||a&&!a.test(f.namespace)||r&&r!==f.selector&&("**"!==r||!f.selector)||(p.splice(o,1),f.selector&&p.delegateCount--,l.remove&&l.remove.call(e,f));s&&!p.length&&(l.teardown&&!1!==l.teardown.call(e,h,v.handle)||w.removeEvent(e,d,v.handle),delete u[d])}else for(d in u)w.event.remove(e,d+t[c],n,r,!0);w.isEmptyObject(u)&&J.remove(e,"handle events")}},dispatch:function(e){var t,n,r,i,o,s,a=w.event.fix(e),u=new Array(arguments.length),c=(J.get(this,"events")||{})[a.type]||[],f=w.event.special[a.type]||{};for(u[0]=a,t=1;t<arguments.length;t++)u[t]=arguments[t];if(a.delegateTarget=this,!f.preDispatch||!1!==f.preDispatch.call(this,a)){for(s=w.event.handlers.call(this,a,c),t=0;(i=s[t++])&&!a.isPropagationStopped();)for(a.currentTarget=i.elem,n=0;(o=i.handlers[n++])&&!a.isImmediatePropagationStopped();)a.rnamespace&&!a.rnamespace.test(o.namespace)||(a.handleObj=o,a.data=o.data,void 0!==(r=((w.event.special[o.origType]||{}).handle||o.handler).apply(i.elem,u))&&!1===(a.result=r)&&(a.preventDefault(),a.stopPropagation()));return f.postDispatch&&f.postDispatch.call(this,a),a.result}},handlers:function(e,t){var n,r,i,o,s,a=[],u=t.delegateCount,c=e.target;if(u&&c.nodeType&&!("click"===e.type&&e.button>=1))for(;c!==this;c=c.parentNode||this)if(1===c.nodeType&&("click"!==e.type||!0!==c.disabled)){for(o=[],s={},n=0;n<u;n++)void 0===s[i=(r=t[n]).selector+" "]&&(s[i]=r.needsContext?w(i,this).index(c)>-1:w.find(i,this,null,[c]).length),s[i]&&o.push(r);o.length&&a.push({elem:c,handlers:o})}return c=this,u<t.length&&a.push({elem:c,handlers:t.slice(u)}),a},addProp:function(e,t){Object.defineProperty(w.Event.prototype,e,{enumerable:!0,configurable:!0,get:v(t)?function(){if(this.originalEvent)return t(this.originalEvent)}:function(){if(this.originalEvent)return this.originalEvent[e]},set:function(t){Object.defineProperty(this,e,{enumerable:!0,configurable:!0,writable:!0,value:t})}})},fix:function(e){return e[w.expando]?e:new w.Event(e)},special:{load:{noBubble:!0},focus:{trigger:function(){if(this!==Ee()&&this.focus)return this.focus(),!1},delegateType:"focusin"},blur:{trigger:function(){if(this===Ee()&&this.blur)return this.blur(),!1},delegateType:"focusout"},click:{trigger:function(){if("checkbox"===this.type&&this.click&&A(this,"input"))return this.click(),!1},_default:function(e){return A(e.target,"a")}},beforeunload:{postDispatch:function(e){void 0!==e.result&&e.originalEvent&&(e.originalEvent.returnValue=e.result)}}}},w.removeEvent=function(e,t,n){e.removeEventListener&&e.removeEventListener(t,n)},w.Event=function(e,t){if(!(this instanceof w.Event))return new w.Event(e,t);e&&e.type?(this.originalEvent=e,this.type=e.type,this.isDefaultPrevented=e.defaultPrevented||void 0===e.defaultPrevented&&!1===e.returnValue?Se:ke,this.target=e.target&&3===e.target.nodeType?e.target.parentNode:e.target,this.currentTarget=e.currentTarget,this.relatedTarget=e.relatedTarget):this.type=e,t&&w.extend(this,t),this.timeStamp=e&&e.timeStamp||Date.now(),this[w.expando]=!0},w.Event.prototype={constructor:w.Event,isDefaultPrevented:ke,isPropagationStopped:ke,isImmediatePropagationStopped:ke,isSimulated:!1,preventDefault:function(){var e=this.originalEvent;this.isDefaultPrevented=Se,e&&!this.isSimulated&&e.preventDefault()},stopPropagation:function(){var e=this.originalEvent;this.isPropagationStopped=Se,e&&!this.isSimulated&&e.stopPropagation()},stopImmediatePropagation:function(){var e=this.originalEvent;this.isImmediatePropagationStopped=Se,e&&!this.isSimulated&&e.stopImmediatePropagation(),this.stopPropagation()}},w.each({altKey:!0,bubbles:!0,cancelable:!0,changedTouches:!0,ctrlKey:!0,detail:!0,eventPhase:!0,metaKey:!0,pageX:!0,pageY:!0,shiftKey:!0,view:!0,char:!0,charCode:!0,key:!0,keyCode:!0,button:!0,buttons:!0,clientX:!0,clientY:!0,offsetX:!0,offsetY:!0,pointerId:!0,pointerType:!0,screenX:!0,screenY:!0,targetTouches:!0,toElement:!0,touches:!0,which:function(e){var t=e.button;return null==e.which&&we.test(e.type)?null!=e.charCode?e.charCode:e.keyCode:!e.which&&void 0!==t&&Te.test(e.type)?1&t?1:2&t?3:4&t?2:0:e.which}},w.event.addProp),w.each({mouseenter:"mouseover",mouseleave:"mouseout",pointerenter:"pointerover",pointerleave:"pointerout"},function(e,t){w.event.special[e]={delegateType:t,bindType:t,handle:function(e){var n,r=e.relatedTarget,i=e.handleObj;return r&&(r===this||w.contains(this,r))||(e.type=i.origType,n=i.handler.apply(this,arguments),e.type=t),n}}}),w.fn.extend({on:function(e,t,n,r){return De(this,e,t,n,r)},one:function(e,t,n,r){return De(this,e,t,n,r,1)},off:function(e,t,n){var r,i;if(e&&e.preventDefault&&e.handleObj)return r=e.handleObj,w(e.delegateTarget).off(r.namespace?r.origType+"."+r.namespace:r.origType,r.selector,r.handler),this;if("object"==typeof e){for(i in e)this.off(i,t,e[i]);return this}return!1!==t&&"function"!=typeof t||(n=t,t=void 0),!1===n&&(n=ke),this.each(function(){w.event.remove(this,e,n,t)})}});var Ae=/<(?!area|br|col|embed|hr|img|input|link|meta|param)(([a-z][^\/\0>\x20\t\r\n\f]*)[^>]*)\/>/gi,Ne=/<script|<style|<link/i,Le=/checked\s*(?:[^=]|=\s*.checked.)/i,qe=/^\s*<!(?:\[CDATA\[|--)|(?:\]\]|--)>\s*$/g;function He(e,t){return A(e,"table")&&A(11!==t.nodeType?t:t.firstChild,"tr")&&w(e).children("tbody")[0]||e}function je(e){return e.type=(null!==e.getAttribute("type"))+"/"+e.type,e}function Pe(e){return"true/"===(e.type||"").slice(0,5)?e.type=e.type.slice(5):e.removeAttribute("type"),e}function Me(e,t){var n,r,i,o,s,a,u,c;if(1===t.nodeType){if(J.hasData(e)&&(o=J.access(e),s=J.set(t,o),c=o.events))for(i in delete s.handle,s.events={},c)for(n=0,r=c[i].length;n<r;n++)w.event.add(t,i,c[i][n]);K.hasData(e)&&(a=K.access(e),u=w.extend({},a),K.set(t,u))}}function Oe(e,t,n,r){t=a.apply([],t);var i,o,s,u,c,f,l=0,p=e.length,d=p-1,h=t[0],y=v(h);if(y||p>1&&"string"==typeof h&&!g.checkClone&&Le.test(h))return e.each(function(i){var o=e.eq(i);y&&(t[0]=h.call(this,i,o.html())),Oe(o,t,n,r)});if(p&&(o=(i=xe(t,e[0].ownerDocument,!1,e,r)).firstChild,1===i.childNodes.length&&(i=o),o||r)){for(u=(s=w.map(he(i,"script"),je)).length;l<p;l++)c=i,l!==d&&(c=w.clone(c,!0,!0),u&&w.merge(s,he(c,"script"))),n.call(e[l],c,l);if(u)for(f=s[s.length-1].ownerDocument,w.map(s,Pe),l=0;l<u;l++)c=s[l],pe.test(c.type||"")&&!J.access(c,"globalEval")&&w.contains(f,c)&&(c.src&&"module"!==(c.type||"").toLowerCase()?w._evalUrl&&w._evalUrl(c.src):x(c.textContent.replace(qe,""),f,c))}return e}function Re(e,t,n){for(var r,i=t?w.filter(t,e):e,o=0;null!=(r=i[o]);o++)n||1!==r.nodeType||w.cleanData(he(r)),r.parentNode&&(n&&w.contains(r.ownerDocument,r)&&ge(he(r,"script")),r.parentNode.removeChild(r));return e}w.extend({htmlPrefilter:function(e){return e.replace(Ae,"<$1></$2>")},clone:function(e,t,n){var r,i,o,s,a,u,c,f=e.cloneNode(!0),l=w.contains(e.ownerDocument,e);if(!(g.noCloneChecked||1!==e.nodeType&&11!==e.nodeType||w.isXMLDoc(e)))for(s=he(f),r=0,i=(o=he(e)).length;r<i;r++)a=o[r],u=s[r],c=void 0,"input"===(c=u.nodeName.toLowerCase())&&fe.test(a.type)?u.checked=a.checked:"input"!==c&&"textarea"!==c||(u.defaultValue=a.defaultValue);if(t)if(n)for(o=o||he(e),s=s||he(f),r=0,i=o.length;r<i;r++)Me(o[r],s[r]);else Me(e,f);return(s=he(f,"script")).length>0&&ge(s,!l&&he(e,"script")),f},cleanData:function(e){for(var t,n,r,i=w.event.special,o=0;void 0!==(n=e[o]);o++)if(Y(n)){if(t=n[J.expando]){if(t.events)for(r in t.events)i[r]?w.event.remove(n,r):w.removeEvent(n,r,t.handle);n[J.expando]=void 0}n[K.expando]&&(n[K.expando]=void 0)}}}),w.fn.extend({detach:function(e){return Re(this,e,!0)},remove:function(e){return Re(this,e)},text:function(e){return _(this,function(e){return void 0===e?w.text(this):this.empty().each(function(){1!==this.nodeType&&11!==this.nodeType&&9!==this.nodeType||(this.textContent=e)})},null,e,arguments.length)},append:function(){return Oe(this,arguments,function(e){1!==this.nodeType&&11!==this.nodeType&&9!==this.nodeType||He(this,e).appendChild(e)})},prepend:function(){return Oe(this,arguments,function(e){if(1===this.nodeType||11===this.nodeType||9===this.nodeType){var t=He(this,e);t.insertBefore(e,t.firstChild)}})},before:function(){return Oe(this,arguments,function(e){this.parentNode&&this.parentNode.insertBefore(e,this)})},after:function(){return Oe(this,arguments,function(e){this.parentNode&&this.parentNode.insertBefore(e,this.nextSibling)})},empty:function(){for(var e,t=0;null!=(e=this[t]);t++)1===e.nodeType&&(w.cleanData(he(e,!1)),e.textContent="");return this},clone:function(e,t){return e=null!=e&&e,t=null==t?e:t,this.map(function(){return w.clone(this,e,t)})},html:function(e){return _(this,function(e){var t=this[0]||{},n=0,r=this.length;if(void 0===e&&1===t.nodeType)return t.innerHTML;if("string"==typeof e&&!Ne.test(e)&&!de[(le.exec(e)||["",""])[1].toLowerCase()]){e=w.htmlPrefilter(e);try{for(;n<r;n++)1===(t=this[n]||{}).nodeType&&(w.cleanData(he(t,!1)),t.innerHTML=e);t=0}catch(i){}}t&&this.empty().append(e)},null,e,arguments.length)},replaceWith:function(){var e=[];return Oe(this,arguments,function(t){var n=this.parentNode;w.inArray(this,e)<0&&(w.cleanData(he(this)),n&&n.replaceChild(t,this))},e)}}),w.each({appendTo:"append",prependTo:"prepend",insertBefore:"before",insertAfter:"after",replaceAll:"replaceWith"},function(e,t){w.fn[e]=function(e){for(var n,r=[],i=w(e),o=i.length-1,s=0;s<=o;s++)n=s===o?this:this.clone(!0),w(i[s])[t](n),u.apply(r,n.get());return this.pushStack(r)}});var We=new RegExp("^("+te+")(?!px)[a-z%]+$","i"),$e=function(e){var n=e.ownerDocument.defaultView;return n&&n.opener||(n=t),n.getComputedStyle(e)},Be=new RegExp(re.join("|"),"i");function Ie(e,t,n){var r,i,o,s,a=e.style;return(n=n||$e(e))&&(""!==(s=n.getPropertyValue(t)||n[t])||w.contains(e.ownerDocument,e)||(s=w.style(e,t)),!g.pixelBoxStyles()&&We.test(s)&&Be.test(t)&&(r=a.width,i=a.minWidth,o=a.maxWidth,a.minWidth=a.maxWidth=a.width=s,s=n.width,a.width=r,a.minWidth=i,a.maxWidth=o)),void 0!==s?s+"":s}function Fe(e,t){return{get:function(){if(!e())return(this.get=t).apply(this,arguments);delete this.get}}}!function(){function e(){if(f){c.style.cssText="position:absolute;left:-11111px;width:60px;margin-top:1px;padding:0;border:0",f.style.cssText="position:relative;display:block;box-sizing:border-box;overflow:scroll;margin:auto;border:1px;padding:1px;width:60%;top:1%",be.appendChild(c).appendChild(f);var e=t.getComputedStyle(f);r="1%"!==e.top,u=12===n(e.marginLeft),f.style.right="60%",a=36===n(e.right),o=36===n(e.width),f.style.position="absolute",s=36===f.offsetWidth||"absolute",be.removeChild(c),f=null}}function n(e){return Math.round(parseFloat(e))}var r,o,s,a,u,c=i.createElement("div"),f=i.createElement("div");f.style&&(f.style.backgroundClip="content-box",f.cloneNode(!0).style.backgroundClip="",g.clearCloneStyle="content-box"===f.style.backgroundClip,w.extend(g,{boxSizingReliable:function(){return e(),o},pixelBoxStyles:function(){return e(),a},pixelPosition:function(){return e(),r},reliableMarginLeft:function(){return e(),u},scrollboxSize:function(){return e(),s}}))}();var _e=/^(none|table(?!-c[ea]).+)/,ze=/^--/,Ue={position:"absolute",visibility:"hidden",display:"block"},Ve={letterSpacing:"0",fontWeight:"400"},Xe=["Webkit","Moz","ms"],Ye=i.createElement("div").style;function Ge(e){var t=w.cssProps[e];return t||(t=w.cssProps[e]=function(e){if(e in Ye)return e;for(var t=e[0].toUpperCase()+e.slice(1),n=Xe.length;n--;)if((e=Xe[n]+t)in Ye)return e}(e)||e),t}function Je(e,t,n){var r=ne.exec(t);return r?Math.max(0,r[2]-(n||0))+(r[3]||"px"):t}function Ke(e,t,n,r,i,o){var s="width"===t?1:0,a=0,u=0;if(n===(r?"border":"content"))return 0;for(;s<4;s+=2)"margin"===n&&(u+=w.css(e,n+re[s],!0,i)),r?("content"===n&&(u-=w.css(e,"padding"+re[s],!0,i)),"margin"!==n&&(u-=w.css(e,"border"+re[s]+"Width",!0,i))):(u+=w.css(e,"padding"+re[s],!0,i),"padding"!==n?u+=w.css(e,"border"+re[s]+"Width",!0,i):a+=w.css(e,"border"+re[s]+"Width",!0,i));return!r&&o>=0&&(u+=Math.max(0,Math.ceil(e["offset"+t[0].toUpperCase()+t.slice(1)]-o-u-a-.5))),u}function Ze(e,t,n){var r=$e(e),i=Ie(e,t,r),o="border-box"===w.css(e,"boxSizing",!1,r),s=o;if(We.test(i)){if(!n)return i;i="auto"}return s=s&&(g.boxSizingReliable()||i===e.style[t]),("auto"===i||!parseFloat(i)&&"inline"===w.css(e,"display",!1,r))&&(i=e["offset"+t[0].toUpperCase()+t.slice(1)],s=!0),(i=parseFloat(i)||0)+Ke(e,t,n||(o?"border":"content"),s,r,i)+"px"}function Qe(e,t,n,r,i){return new Qe.prototype.init(e,t,n,r,i)}w.extend({cssHooks:{opacity:{get:function(e,t){if(t){var n=Ie(e,"opacity");return""===n?"1":n}}}},cssNumber:{animationIterationCount:!0,columnCount:!0,fillOpacity:!0,flexGrow:!0,flexShrink:!0,fontWeight:!0,lineHeight:!0,opacity:!0,order:!0,orphans:!0,widows:!0,zIndex:!0,zoom:!0},cssProps:{},style:function(e,t,n,r){if(e&&3!==e.nodeType&&8!==e.nodeType&&e.style){var i,o,s,a=X(t),u=ze.test(t),c=e.style;if(u||(t=Ge(a)),s=w.cssHooks[t]||w.cssHooks[a],void 0===n)return s&&"get"in s&&void 0!==(i=s.get(e,!1,r))?i:c[t];"string"===(o=typeof n)&&(i=ne.exec(n))&&i[1]&&(n=se(e,t,i),o="number"),null!=n&&n==n&&("number"===o&&(n+=i&&i[3]||(w.cssNumber[a]?"":"px")),g.clearCloneStyle||""!==n||0!==t.indexOf("background")||(c[t]="inherit"),s&&"set"in s&&void 0===(n=s.set(e,n,r))||(u?c.setProperty(t,n):c[t]=n))}},css:function(e,t,n,r){var i,o,s,a=X(t);return ze.test(t)||(t=Ge(a)),(s=w.cssHooks[t]||w.cssHooks[a])&&"get"in s&&(i=s.get(e,!0,n)),void 0===i&&(i=Ie(e,t,r)),"normal"===i&&t in Ve&&(i=Ve[t]),""===n||n?(o=parseFloat(i),!0===n||isFinite(o)?o||0:i):i}}),w.each(["height","width"],function(e,t){w.cssHooks[t]={get:function(e,n,r){if(n)return!_e.test(w.css(e,"display"))||e.getClientRects().length&&e.getBoundingClientRect().width?Ze(e,t,r):oe(e,Ue,function(){return Ze(e,t,r)})},set:function(e,n,r){var i,o=$e(e),s="border-box"===w.css(e,"boxSizing",!1,o),a=r&&Ke(e,t,r,s,o);return s&&g.scrollboxSize()===o.position&&(a-=Math.ceil(e["offset"+t[0].toUpperCase()+t.slice(1)]-parseFloat(o[t])-Ke(e,t,"border",!1,o)-.5)),a&&(i=ne.exec(n))&&"px"!==(i[3]||"px")&&(e.style[t]=n,n=w.css(e,t)),Je(0,n,a)}}}),w.cssHooks.marginLeft=Fe(g.reliableMarginLeft,function(e,t){if(t)return(parseFloat(Ie(e,"marginLeft"))||e.getBoundingClientRect().left-oe(e,{marginLeft:0},function(){return e.getBoundingClientRect().left}))+"px"}),w.each({margin:"",padding:"",border:"Width"},function(e,t){w.cssHooks[e+t]={expand:function(n){for(var r=0,i={},o="string"==typeof n?n.split(" "):[n];r<4;r++)i[e+re[r]+t]=o[r]||o[r-2]||o[0];return i}},"margin"!==e&&(w.cssHooks[e+t].set=Je)}),w.fn.extend({css:function(e,t){return _(this,function(e,t,n){var r,i,o={},s=0;if(Array.isArray(t)){for(r=$e(e),i=t.length;s<i;s++)o[t[s]]=w.css(e,t[s],!1,r);return o}return void 0!==n?w.style(e,t,n):w.css(e,t)},e,t,arguments.length>1)}}),w.Tween=Qe,Qe.prototype={constructor:Qe,init:function(e,t,n,r,i,o){this.elem=e,this.prop=n,this.easing=i||w.easing._default,this.options=t,this.start=this.now=this.cur(),this.end=r,this.unit=o||(w.cssNumber[n]?"":"px")},cur:function(){var e=Qe.propHooks[this.prop];return e&&e.get?e.get(this):Qe.propHooks._default.get(this)},run:function(e){var t,n=Qe.propHooks[this.prop];return this.options.duration?this.pos=t=w.easing[this.easing](e,this.options.duration*e,0,1,this.options.duration):this.pos=t=e,this.now=(this.end-this.start)*t+this.start,this.options.step&&this.options.step.call(this.elem,this.now,this),n&&n.set?n.set(this):Qe.propHooks._default.set(this),this}},Qe.prototype.init.prototype=Qe.prototype,Qe.propHooks={_default:{get:function(e){var t;return 1!==e.elem.nodeType||null!=e.elem[e.prop]&&null==e.elem.style[e.prop]?e.elem[e.prop]:(t=w.css(e.elem,e.prop,""))&&"auto"!==t?t:0},set:function(e){w.fx.step[e.prop]?w.fx.step[e.prop](e):1!==e.elem.nodeType||null==e.elem.style[w.cssProps[e.prop]]&&!w.cssHooks[e.prop]?e.elem[e.prop]=e.now:w.style(e.elem,e.prop,e.now+e.unit)}}},Qe.propHooks.scrollTop=Qe.propHooks.scrollLeft={set:function(e){e.elem.nodeType&&e.elem.parentNode&&(e.elem[e.prop]=e.now)}},w.easing={linear:function(e){return e},swing:function(e){return.5-Math.cos(e*Math.PI)/2},_default:"swing"},w.fx=Qe.prototype.init,w.fx.step={};var et,tt,nt=/^(?:toggle|show|hide)$/,rt=/queueHooks$/;function it(){tt&&(!1===i.hidden&&t.requestAnimationFrame?t.requestAnimationFrame(it):t.setTimeout(it,w.fx.interval),w.fx.tick())}function ot(){return t.setTimeout(function(){et=void 0}),et=Date.now()}function st(e,t){var n,r=0,i={height:e};for(t=t?1:0;r<4;r+=2-t)i["margin"+(n=re[r])]=i["padding"+n]=e;return t&&(i.opacity=i.width=e),i}function at(e,t,n){for(var r,i=(ut.tweeners[t]||[]).concat(ut.tweeners["*"]),o=0,s=i.length;o<s;o++)if(r=i[o].call(n,t,e))return r}function ut(e,t,n){var r,i,o=0,s=ut.prefilters.length,a=w.Deferred().always(function(){delete u.elem}),u=function(){if(i)return!1;for(var t=et||ot(),n=Math.max(0,c.startTime+c.duration-t),r=1-(n/c.duration||0),o=0,s=c.tweens.length;o<s;o++)c.tweens[o].run(r);return a.notifyWith(e,[c,r,n]),r<1&&s?n:(s||a.notifyWith(e,[c,1,0]),a.resolveWith(e,[c]),!1)},c=a.promise({elem:e,props:w.extend({},t),opts:w.extend(!0,{specialEasing:{},easing:w.easing._default},n),originalProperties:t,originalOptions:n,startTime:et||ot(),duration:n.duration,tweens:[],createTween:function(t,n){var r=w.Tween(e,c.opts,t,n,c.opts.specialEasing[t]||c.opts.easing);return c.tweens.push(r),r},stop:function(t){var n=0,r=t?c.tweens.length:0;if(i)return this;for(i=!0;n<r;n++)c.tweens[n].run(1);return t?(a.notifyWith(e,[c,1,0]),a.resolveWith(e,[c,t])):a.rejectWith(e,[c,t]),this}}),f=c.props;for(!function(e,t){var n,r,i,o,s;for(n in e)if(i=t[r=X(n)],o=e[n],Array.isArray(o)&&(i=o[1],o=e[n]=o[0]),n!==r&&(e[r]=o,delete e[n]),(s=w.cssHooks[r])&&"expand"in s)for(n in o=s.expand(o),delete e[r],o)n in e||(e[n]=o[n],t[n]=i);else t[r]=i}(f,c.opts.specialEasing);o<s;o++)if(r=ut.prefilters[o].call(c,e,f,c.opts))return v(r.stop)&&(w._queueHooks(c.elem,c.opts.queue).stop=r.stop.bind(r)),r;return w.map(f,at,c),v(c.opts.start)&&c.opts.start.call(e,c),c.progress(c.opts.progress).done(c.opts.done,c.opts.complete).fail(c.opts.fail).always(c.opts.always),w.fx.timer(w.extend(u,{elem:e,anim:c,queue:c.opts.queue})),c}w.Animation=w.extend(ut,{tweeners:{"*":[function(e,t){var n=this.createTween(e,t);return se(n.elem,e,ne.exec(t),n),n}]},tweener:function(e,t){v(e)?(t=e,e=["*"]):e=e.match(O);for(var n,r=0,i=e.length;r<i;r++)n=e[r],ut.tweeners[n]=ut.tweeners[n]||[],ut.tweeners[n].unshift(t)},prefilters:[function(e,t,n){var r,i,o,s,a,u,c,f,l="width"in t||"height"in t,p=this,d={},h=e.style,g=e.nodeType&&ie(e),v=J.get(e,"fxshow");for(r in n.queue||(null==(s=w._queueHooks(e,"fx")).unqueued&&(s.unqueued=0,a=s.empty.fire,s.empty.fire=function(){s.unqueued||a()}),s.unqueued++,p.always(function(){p.always(function(){s.unqueued--,w.queue(e,"fx").length||s.empty.fire()})})),t)if(i=t[r],nt.test(i)){if(delete t[r],o=o||"toggle"===i,i===(g?"hide":"show")){if("show"!==i||!v||void 0===v[r])continue;g=!0}d[r]=v&&v[r]||w.style(e,r)}if((u=!w.isEmptyObject(t))||!w.isEmptyObject(d))for(r in l&&1===e.nodeType&&(n.overflow=[h.overflow,h.overflowX,h.overflowY],null==(c=v&&v.display)&&(c=J.get(e,"display")),"none"===(f=w.css(e,"display"))&&(c?f=c:(ce([e],!0),c=e.style.display||c,f=w.css(e,"display"),ce([e]))),("inline"===f||"inline-block"===f&&null!=c)&&"none"===w.css(e,"float")&&(u||(p.done(function(){h.display=c}),null==c&&(f=h.display,c="none"===f?"":f)),h.display="inline-block")),n.overflow&&(h.overflow="hidden",p.always(function(){h.overflow=n.overflow[0],h.overflowX=n.overflow[1],h.overflowY=n.overflow[2]})),u=!1,d)u||(v?"hidden"in v&&(g=v.hidden):v=J.access(e,"fxshow",{display:c}),o&&(v.hidden=!g),g&&ce([e],!0),p.done(function(){for(r in g||ce([e]),J.remove(e,"fxshow"),d)w.style(e,r,d[r])})),u=at(g?v[r]:0,r,p),r in v||(v[r]=u.start,g&&(u.end=u.start,u.start=0))}],prefilter:function(e,t){t?ut.prefilters.unshift(e):ut.prefilters.push(e)}}),w.speed=function(e,t,n){var r=e&&"object"==typeof e?w.extend({},e):{complete:n||!n&&t||v(e)&&e,duration:e,easing:n&&t||t&&!v(t)&&t};return w.fx.off?r.duration=0:"number"!=typeof r.duration&&(r.duration in w.fx.speeds?r.duration=w.fx.speeds[r.duration]:r.duration=w.fx.speeds._default),null!=r.queue&&!0!==r.queue||(r.queue="fx"),r.old=r.complete,r.complete=function(){v(r.old)&&r.old.call(this),r.queue&&w.dequeue(this,r.queue)},r},w.fn.extend({fadeTo:function(e,t,n,r){return this.filter(ie).css("opacity",0).show().end().animate({opacity:t},e,n,r)},animate:function(e,t,n,r){var i=w.isEmptyObject(e),o=w.speed(t,n,r),s=function(){var t=ut(this,w.extend({},e),o);(i||J.get(this,"finish"))&&t.stop(!0)};return s.finish=s,i||!1===o.queue?this.each(s):this.queue(o.queue,s)},stop:function(e,t,n){var r=function(e){var t=e.stop;delete e.stop,t(n)};return"string"!=typeof e&&(n=t,t=e,e=void 0),t&&!1!==e&&this.queue(e||"fx",[]),this.each(function(){var t=!0,i=null!=e&&e+"queueHooks",o=w.timers,s=J.get(this);if(i)s[i]&&s[i].stop&&r(s[i]);else for(i in s)s[i]&&s[i].stop&&rt.test(i)&&r(s[i]);for(i=o.length;i--;)o[i].elem!==this||null!=e&&o[i].queue!==e||(o[i].anim.stop(n),t=!1,o.splice(i,1));!t&&n||w.dequeue(this,e)})},finish:function(e){return!1!==e&&(e=e||"fx"),this.each(function(){var t,n=J.get(this),r=n[e+"queue"],i=n[e+"queueHooks"],o=w.timers,s=r?r.length:0;for(n.finish=!0,w.queue(this,e,[]),i&&i.stop&&i.stop.call(this,!0),t=o.length;t--;)o[t].elem===this&&o[t].queue===e&&(o[t].anim.stop(!0),o.splice(t,1));for(t=0;t<s;t++)r[t]&&r[t].finish&&r[t].finish.call(this);delete n.finish})}}),w.each(["toggle","show","hide"],function(e,t){var n=w.fn[t];w.fn[t]=function(e,r,i){return null==e||"boolean"==typeof e?n.apply(this,arguments):this.animate(st(t,!0),e,r,i)}}),w.each({slideDown:st("show"),slideUp:st("hide"),slideToggle:st("toggle"),fadeIn:{opacity:"show"},fadeOut:{opacity:"hide"},fadeToggle:{opacity:"toggle"}},function(e,t){w.fn[e]=function(e,n,r){return this.animate(t,e,n,r)}}),w.timers=[],w.fx.tick=function(){var e,t=0,n=w.timers;for(et=Date.now();t<n.length;t++)(e=n[t])()||n[t]!==e||n.splice(t--,1);n.length||w.fx.stop(),et=void 0},w.fx.timer=function(e){w.timers.push(e),w.fx.start()},w.fx.interval=13,w.fx.start=function(){tt||(tt=!0,it())},w.fx.stop=function(){tt=null},w.fx.speeds={slow:600,fast:200,_default:400},w.fn.delay=function(e,n){return e=w.fx&&w.fx.speeds[e]||e,n=n||"fx",this.queue(n,function(n,r){var i=t.setTimeout(n,e);r.stop=function(){t.clearTimeout(i)}})},function(){var e=i.createElement("input"),t=i.createElement("select").appendChild(i.createElement("option"));e.type="checkbox",g.checkOn=""!==e.value,g.optSelected=t.selected,(e=i.createElement("input")).value="t",e.type="radio",g.radioValue="t"===e.value}();var ct,ft=w.expr.attrHandle;w.fn.extend({attr:function(e,t){return _(this,w.attr,e,t,arguments.length>1)},removeAttr:function(e){return this.each(function(){w.removeAttr(this,e)})}}),w.extend({attr:function(e,t,n){var r,i,o=e.nodeType;if(3!==o&&8!==o&&2!==o)return void 0===e.getAttribute?w.prop(e,t,n):(1===o&&w.isXMLDoc(e)||(i=w.attrHooks[t.toLowerCase()]||(w.expr.match.bool.test(t)?ct:void 0)),void 0!==n?null===n?void w.removeAttr(e,t):i&&"set"in i&&void 0!==(r=i.set(e,n,t))?r:(e.setAttribute(t,n+""),n):i&&"get"in i&&null!==(r=i.get(e,t))?r:null==(r=w.find.attr(e,t))?void 0:r)},attrHooks:{type:{set:function(e,t){if(!g.radioValue&&"radio"===t&&A(e,"input")){var n=e.value;return e.setAttribute("type",t),n&&(e.value=n),t}}}},removeAttr:function(e,t){var n,r=0,i=t&&t.match(O);if(i&&1===e.nodeType)for(;n=i[r++];)e.removeAttribute(n)}}),ct={set:function(e,t,n){return!1===t?w.removeAttr(e,n):e.setAttribute(n,n),n}},w.each(w.expr.match.bool.source.match(/\w+/g),function(e,t){var n=ft[t]||w.find.attr;ft[t]=function(e,t,r){var i,o,s=t.toLowerCase();return r||(o=ft[s],ft[s]=i,i=null!=n(e,t,r)?s:null,ft[s]=o),i}});var lt=/^(?:input|select|textarea|button)$/i,pt=/^(?:a|area)$/i;function dt(e){return(e.match(O)||[]).join(" ")}function ht(e){return e.getAttribute&&e.getAttribute("class")||""}function gt(e){return Array.isArray(e)?e:"string"==typeof e&&e.match(O)||[]}w.fn.extend({prop:function(e,t){return _(this,w.prop,e,t,arguments.length>1)},removeProp:function(e){return this.each(function(){delete this[w.propFix[e]||e]})}}),w.extend({prop:function(e,t,n){var r,i,o=e.nodeType;if(3!==o&&8!==o&&2!==o)return 1===o&&w.isXMLDoc(e)||(t=w.propFix[t]||t,i=w.propHooks[t]),void 0!==n?i&&"set"in i&&void 0!==(r=i.set(e,n,t))?r:e[t]=n:i&&"get"in i&&null!==(r=i.get(e,t))?r:e[t]},propHooks:{tabIndex:{get:function(e){var t=w.find.attr(e,"tabindex");return t?parseInt(t,10):lt.test(e.nodeName)||pt.test(e.nodeName)&&e.href?0:-1}}},propFix:{for:"htmlFor",class:"className"}}),g.optSelected||(w.propHooks.selected={get:function(e){var t=e.parentNode;return t&&t.parentNode&&t.parentNode.selectedIndex,null},set:function(e){var t=e.parentNode;t&&(t.selectedIndex,t.parentNode&&t.parentNode.selectedIndex)}}),w.each(["tabIndex","readOnly","maxLength","cellSpacing","cellPadding","rowSpan","colSpan","useMap","frameBorder","contentEditable"],function(){w.propFix[this.toLowerCase()]=this}),w.fn.extend({addClass:function(e){var t,n,r,i,o,s,a,u=0;if(v(e))return this.each(function(t){w(this).addClass(e.call(this,t,ht(this)))});if((t=gt(e)).length)for(;n=this[u++];)if(i=ht(n),r=1===n.nodeType&&" "+dt(i)+" "){for(s=0;o=t[s++];)r.indexOf(" "+o+" ")<0&&(r+=o+" ");i!==(a=dt(r))&&n.setAttribute("class",a)}return this},removeClass:function(e){var t,n,r,i,o,s,a,u=0;if(v(e))return this.each(function(t){w(this).removeClass(e.call(this,t,ht(this)))});if(!arguments.length)return this.attr("class","");if((t=gt(e)).length)for(;n=this[u++];)if(i=ht(n),r=1===n.nodeType&&" "+dt(i)+" "){for(s=0;o=t[s++];)for(;r.indexOf(" "+o+" ")>-1;)r=r.replace(" "+o+" "," ");i!==(a=dt(r))&&n.setAttribute("class",a)}return this},toggleClass:function(e,t){var n=typeof e,r="string"===n||Array.isArray(e);return"boolean"==typeof t&&r?t?this.addClass(e):this.removeClass(e):v(e)?this.each(function(n){w(this).toggleClass(e.call(this,n,ht(this),t),t)}):this.each(function(){var t,i,o,s;if(r)for(i=0,o=w(this),s=gt(e);t=s[i++];)o.hasClass(t)?o.removeClass(t):o.addClass(t);else void 0!==e&&"boolean"!==n||((t=ht(this))&&J.set(this,"__className__",t),this.setAttribute&&this.setAttribute("class",t||!1===e?"":J.get(this,"__className__")||""))})},hasClass:function(e){var t,n,r=0;for(t=" "+e+" ";n=this[r++];)if(1===n.nodeType&&(" "+dt(ht(n))+" ").indexOf(t)>-1)return!0;return!1}});var vt=/\r/g;w.fn.extend({val:function(e){var t,n,r,i=this[0];return arguments.length?(r=v(e),this.each(function(n){var i;1===this.nodeType&&(null==(i=r?e.call(this,n,w(this).val()):e)?i="":"number"==typeof i?i+="":Array.isArray(i)&&(i=w.map(i,function(e){return null==e?"":e+""})),(t=w.valHooks[this.type]||w.valHooks[this.nodeName.toLowerCase()])&&"set"in t&&void 0!==t.set(this,i,"value")||(this.value=i))})):i?(t=w.valHooks[i.type]||w.valHooks[i.nodeName.toLowerCase()])&&"get"in t&&void 0!==(n=t.get(i,"value"))?n:"string"==typeof(n=i.value)?n.replace(vt,""):null==n?"":n:void 0}}),w.extend({valHooks:{option:{get:function(e){var t=w.find.attr(e,"value");return null!=t?t:dt(w.text(e))}},select:{get:function(e){var t,n,r,i=e.options,o=e.selectedIndex,s="select-one"===e.type,a=s?null:[],u=s?o+1:i.length;for(r=o<0?u:s?o:0;r<u;r++)if(((n=i[r]).selected||r===o)&&!n.disabled&&(!n.parentNode.disabled||!A(n.parentNode,"optgroup"))){if(t=w(n).val(),s)return t;a.push(t)}return a},set:function(e,t){for(var n,r,i=e.options,o=w.makeArray(t),s=i.length;s--;)((r=i[s]).selected=w.inArray(w.valHooks.option.get(r),o)>-1)&&(n=!0);return n||(e.selectedIndex=-1),o}}}}),w.each(["radio","checkbox"],function(){w.valHooks[this]={set:function(e,t){if(Array.isArray(t))return e.checked=w.inArray(w(e).val(),t)>-1}},g.checkOn||(w.valHooks[this].get=function(e){return null===e.getAttribute("value")?"on":e.value})}),g.focusin="onfocusin"in t;var yt=/^(?:focusinfocus|focusoutblur)$/,mt=function(e){e.stopPropagation()};w.extend(w.event,{trigger:function(e,n,r,o){var s,a,u,c,f,l,d,h,g=[r||i],m=p.call(e,"type")?e.type:e,x=p.call(e,"namespace")?e.namespace.split("."):[];if(a=h=u=r=r||i,3!==r.nodeType&&8!==r.nodeType&&!yt.test(m+w.event.triggered)&&(m.indexOf(".")>-1&&(x=m.split("."),m=x.shift(),x.sort()),f=m.indexOf(":")<0&&"on"+m,(e=e[w.expando]?e:new w.Event(m,"object"==typeof e&&e)).isTrigger=o?2:3,e.namespace=x.join("."),e.rnamespace=e.namespace?new RegExp("(^|\\.)"+x.join("\\.(?:.*\\.|)")+"(\\.|$)"):null,e.result=void 0,e.target||(e.target=r),n=null==n?[e]:w.makeArray(n,[e]),d=w.event.special[m]||{},o||!d.trigger||!1!==d.trigger.apply(r,n))){if(!o&&!d.noBubble&&!y(r)){for(c=d.delegateType||m,yt.test(c+m)||(a=a.parentNode);a;a=a.parentNode)g.push(a),u=a;u===(r.ownerDocument||i)&&g.push(u.defaultView||u.parentWindow||t)}for(s=0;(a=g[s++])&&!e.isPropagationStopped();)h=a,e.type=s>1?c:d.bindType||m,(l=(J.get(a,"events")||{})[e.type]&&J.get(a,"handle"))&&l.apply(a,n),(l=f&&a[f])&&l.apply&&Y(a)&&(e.result=l.apply(a,n),!1===e.result&&e.preventDefault());return e.type=m,o||e.isDefaultPrevented()||d._default&&!1!==d._default.apply(g.pop(),n)||!Y(r)||f&&v(r[m])&&!y(r)&&((u=r[f])&&(r[f]=null),w.event.triggered=m,e.isPropagationStopped()&&h.addEventListener(m,mt),r[m](),e.isPropagationStopped()&&h.removeEventListener(m,mt),w.event.triggered=void 0,u&&(r[f]=u)),e.result}},simulate:function(e,t,n){var r=w.extend(new w.Event,n,{type:e,isSimulated:!0});w.event.trigger(r,null,t)}}),w.fn.extend({trigger:function(e,t){return this.each(function(){w.event.trigger(e,t,this)})},triggerHandler:function(e,t){var n=this[0];if(n)return w.event.trigger(e,t,n,!0)}}),g.focusin||w.each({focus:"focusin",blur:"focusout"},function(e,t){var n=function(e){w.event.simulate(t,e.target,w.event.fix(e))};w.event.special[t]={setup:function(){var r=this.ownerDocument||this,i=J.access(r,t);i||r.addEventListener(e,n,!0),J.access(r,t,(i||0)+1)},teardown:function(){var r=this.ownerDocument||this,i=J.access(r,t)-1;i?J.access(r,t,i):(r.removeEventListener(e,n,!0),J.remove(r,t))}}});var xt=t.location,bt=Date.now(),wt=/\?/;w.parseXML=function(e){var n;if(!e||"string"!=typeof e)return null;try{n=(new t.DOMParser).parseFromString(e,"text/xml")}catch(r){n=void 0}return n&&!n.getElementsByTagName("parsererror").length||w.error("Invalid XML: "+e),n};var Tt=/\[\]$/,Ct=/\r?\n/g,St=/^(?:submit|button|image|reset|file)$/i,kt=/^(?:input|select|textarea|keygen)/i;function Et(e,t,n,r){var i;if(Array.isArray(t))w.each(t,function(t,i){n||Tt.test(e)?r(e,i):Et(e+"["+("object"==typeof i&&null!=i?t:"")+"]",i,n,r)});else if(n||"object"!==b(t))r(e,t);else for(i in t)Et(e+"["+i+"]",t[i],n,r)}w.param=function(e,t){var n,r=[],i=function(e,t){var n=v(t)?t():t;r[r.length]=encodeURIComponent(e)+"="+encodeURIComponent(null==n?"":n)};if(Array.isArray(e)||e.jquery&&!w.isPlainObject(e))w.each(e,function(){i(this.name,this.value)});else for(n in e)Et(n,e[n],t,i);return r.join("&")},w.fn.extend({serialize:function(){return w.param(this.serializeArray())},serializeArray:function(){return this.map(function(){var e=w.prop(this,"elements");return e?w.makeArray(e):this}).filter(function(){var e=this.type;return this.name&&!w(this).is(":disabled")&&kt.test(this.nodeName)&&!St.test(e)&&(this.checked||!fe.test(e))}).map(function(e,t){var n=w(this).val();return null==n?null:Array.isArray(n)?w.map(n,function(e){return{name:t.name,value:e.replace(Ct,"\r\n")}}):{name:t.name,value:n.replace(Ct,"\r\n")}}).get()}});var Dt=/%20/g,At=/#.*$/,Nt=/([?&])_=[^&]*/,Lt=/^(.*?):[ \t]*([^\r\n]*)$/gm,qt=/^(?:GET|HEAD)$/,Ht=/^\/\//,jt={},Pt={},Mt="*/".concat("*"),Ot=i.createElement("a");function Rt(e){return function(t,n){"string"!=typeof t&&(n=t,t="*");var r,i=0,o=t.toLowerCase().match(O)||[];if(v(n))for(;r=o[i++];)"+"===r[0]?(r=r.slice(1)||"*",(e[r]=e[r]||[]).unshift(n)):(e[r]=e[r]||[]).push(n)}}function Wt(e,t,n,r){var i={},o=e===Pt;function s(a){var u;return i[a]=!0,w.each(e[a]||[],function(e,a){var c=a(t,n,r);return"string"!=typeof c||o||i[c]?o?!(u=c):void 0:(t.dataTypes.unshift(c),s(c),!1)}),u}return s(t.dataTypes[0])||!i["*"]&&s("*")}function $t(e,t){var n,r,i=w.ajaxSettings.flatOptions||{};for(n in t)void 0!==t[n]&&((i[n]?e:r||(r={}))[n]=t[n]);return r&&w.extend(!0,e,r),e}Ot.href=xt.href,w.extend({active:0,lastModified:{},etag:{},ajaxSettings:{url:xt.href,type:"GET",isLocal:/^(?:about|app|app-storage|.+-extension|file|res|widget):$/.test(xt.protocol),global:!0,processData:!0,async:!0,contentType:"application/x-www-form-urlencoded; charset=UTF-8",accepts:{"*":Mt,text:"text/plain",html:"text/html",xml:"application/xml, text/xml",json:"application/json, text/javascript"},contents:{xml:/\bxml\b/,html:/\bhtml/,json:/\bjson\b/},responseFields:{xml:"responseXML",text:"responseText",json:"responseJSON"},converters:{"* text":String,"text html":!0,"text json":JSON.parse,"text xml":w.parseXML},flatOptions:{url:!0,context:!0}},ajaxSetup:function(e,t){return t?$t($t(e,w.ajaxSettings),t):$t(w.ajaxSettings,e)},ajaxPrefilter:Rt(jt),ajaxTransport:Rt(Pt),ajax:function(e,n){"object"==typeof e&&(n=e,e=void 0),n=n||{};var r,o,s,a,u,c,f,l,p,d,h=w.ajaxSetup({},n),g=h.context||h,v=h.context&&(g.nodeType||g.jquery)?w(g):w.event,y=w.Deferred(),m=w.Callbacks("once memory"),x=h.statusCode||{},b={},T={},C="canceled",S={readyState:0,getResponseHeader:function(e){var t;if(f){if(!a)for(a={};t=Lt.exec(s);)a[t[1].toLowerCase()]=t[2];t=a[e.toLowerCase()]}return null==t?null:t},getAllResponseHeaders:function(){return f?s:null},setRequestHeader:function(e,t){return null==f&&(e=T[e.toLowerCase()]=T[e.toLowerCase()]||e,b[e]=t),this},overrideMimeType:function(e){return null==f&&(h.mimeType=e),this},statusCode:function(e){var t;if(e)if(f)S.always(e[S.status]);else for(t in e)x[t]=[x[t],e[t]];return this},abort:function(e){var t=e||C;return r&&r.abort(t),k(0,t),this}};if(y.promise(S),h.url=((e||h.url||xt.href)+"").replace(Ht,xt.protocol+"//"),h.type=n.method||n.type||h.method||h.type,h.dataTypes=(h.dataType||"*").toLowerCase().match(O)||[""],null==h.crossDomain){c=i.createElement("a");try{c.href=h.url,c.href=c.href,h.crossDomain=Ot.protocol+"//"+Ot.host!=c.protocol+"//"+c.host}catch(E){h.crossDomain=!0}}if(h.data&&h.processData&&"string"!=typeof h.data&&(h.data=w.param(h.data,h.traditional)),Wt(jt,h,n,S),f)return S;for(p in(l=w.event&&h.global)&&0==w.active++&&w.event.trigger("ajaxStart"),h.type=h.type.toUpperCase(),h.hasContent=!qt.test(h.type),o=h.url.replace(At,""),h.hasContent?h.data&&h.processData&&0===(h.contentType||"").indexOf("application/x-www-form-urlencoded")&&(h.data=h.data.replace(Dt,"+")):(d=h.url.slice(o.length),h.data&&(h.processData||"string"==typeof h.data)&&(o+=(wt.test(o)?"&":"?")+h.data,delete h.data),!1===h.cache&&(o=o.replace(Nt,"$1"),d=(wt.test(o)?"&":"?")+"_="+bt+++d),h.url=o+d),h.ifModified&&(w.lastModified[o]&&S.setRequestHeader("If-Modified-Since",w.lastModified[o]),w.etag[o]&&S.setRequestHeader("If-None-Match",w.etag[o])),(h.data&&h.hasContent&&!1!==h.contentType||n.contentType)&&S.setRequestHeader("Content-Type",h.contentType),S.setRequestHeader("Accept",h.dataTypes[0]&&h.accepts[h.dataTypes[0]]?h.accepts[h.dataTypes[0]]+("*"!==h.dataTypes[0]?", "+Mt+"; q=0.01":""):h.accepts["*"]),h.headers)S.setRequestHeader(p,h.headers[p]);if(h.beforeSend&&(!1===h.beforeSend.call(g,S,h)||f))return S.abort();if(C="abort",m.add(h.complete),S.done(h.success),S.fail(h.error),r=Wt(Pt,h,n,S)){if(S.readyState=1,l&&v.trigger("ajaxSend",[S,h]),f)return S;h.async&&h.timeout>0&&(u=t.setTimeout(function(){S.abort("timeout")},h.timeout));try{f=!1,r.send(b,k)}catch(E){if(f)throw E;k(-1,E)}}else k(-1,"No Transport");function k(e,n,i,a){var c,p,d,b,T,C=n;f||(f=!0,u&&t.clearTimeout(u),r=void 0,s=a||"",S.readyState=e>0?4:0,c=e>=200&&e<300||304===e,i&&(b=function(e,t,n){for(var r,i,o,s,a=e.contents,u=e.dataTypes;"*"===u[0];)u.shift(),void 0===r&&(r=e.mimeType||t.getResponseHeader("Content-Type"));if(r)for(i in a)if(a[i]&&a[i].test(r)){u.unshift(i);break}if(u[0]in n)o=u[0];else{for(i in n){if(!u[0]||e.converters[i+" "+u[0]]){o=i;break}s||(s=i)}o=o||s}if(o)return o!==u[0]&&u.unshift(o),n[o]}(h,S,i)),b=function(e,t,n,r){var i,o,s,a,u,c={},f=e.dataTypes.slice();if(f[1])for(s in e.converters)c[s.toLowerCase()]=e.converters[s];for(o=f.shift();o;)if(e.responseFields[o]&&(n[e.responseFields[o]]=t),!u&&r&&e.dataFilter&&(t=e.dataFilter(t,e.dataType)),u=o,o=f.shift())if("*"===o)o=u;else if("*"!==u&&u!==o){if(!(s=c[u+" "+o]||c["* "+o]))for(i in c)if((a=i.split(" "))[1]===o&&(s=c[u+" "+a[0]]||c["* "+a[0]])){!0===s?s=c[i]:!0!==c[i]&&(o=a[0],f.unshift(a[1]));break}if(!0!==s)if(s&&e.throws)t=s(t);else try{t=s(t)}catch(E){return{state:"parsererror",error:s?E:"No conversion from "+u+" to "+o}}}return{state:"success",data:t}}(h,b,S,c),c?(h.ifModified&&((T=S.getResponseHeader("Last-Modified"))&&(w.lastModified[o]=T),(T=S.getResponseHeader("etag"))&&(w.etag[o]=T)),204===e||"HEAD"===h.type?C="nocontent":304===e?C="notmodified":(C=b.state,p=b.data,c=!(d=b.error))):(d=C,!e&&C||(C="error",e<0&&(e=0))),S.status=e,S.statusText=(n||C)+"",c?y.resolveWith(g,[p,C,S]):y.rejectWith(g,[S,C,d]),S.statusCode(x),x=void 0,l&&v.trigger(c?"ajaxSuccess":"ajaxError",[S,h,c?p:d]),m.fireWith(g,[S,C]),l&&(v.trigger("ajaxComplete",[S,h]),--w.active||w.event.trigger("ajaxStop")))}return S},getJSON:function(e,t,n){return w.get(e,t,n,"json")},getScript:function(e,t){return w.get(e,void 0,t,"script")}}),w.each(["get","post"],function(e,t){w[t]=function(e,n,r,i){return v(n)&&(i=i||r,r=n,n=void 0),w.ajax(w.extend({url:e,type:t,dataType:i,data:n,success:r},w.isPlainObject(e)&&e))}}),w._evalUrl=function(e){return w.ajax({url:e,type:"GET",dataType:"script",cache:!0,async:!1,global:!1,throws:!0})},w.fn.extend({wrapAll:function(e){var t;return this[0]&&(v(e)&&(e=e.call(this[0])),t=w(e,this[0].ownerDocument).eq(0).clone(!0),this[0].parentNode&&t.insertBefore(this[0]),t.map(function(){for(var e=this;e.firstElementChild;)e=e.firstElementChild;return e}).append(this)),this},wrapInner:function(e){return v(e)?this.each(function(t){w(this).wrapInner(e.call(this,t))}):this.each(function(){var t=w(this),n=t.contents();n.length?n.wrapAll(e):t.append(e)})},wrap:function(e){var t=v(e);return this.each(function(n){w(this).wrapAll(t?e.call(this,n):e)})},unwrap:function(e){return this.parent(e).not("body").each(function(){w(this).replaceWith(this.childNodes)}),this}}),w.expr.pseudos.hidden=function(e){return!w.expr.pseudos.visible(e)},w.expr.pseudos.visible=function(e){return!!(e.offsetWidth||e.offsetHeight||e.getClientRects().length)},w.ajaxSettings.xhr=function(){try{return new t.XMLHttpRequest}catch(e){}};var Bt={0:200,1223:204},It=w.ajaxSettings.xhr();g.cors=!!It&&"withCredentials"in It,g.ajax=It=!!It,w.ajaxTransport(function(e){var n,r;if(g.cors||It&&!e.crossDomain)return{send:function(i,o){var s,a=e.xhr();if(a.open(e.type,e.url,e.async,e.username,e.password),e.xhrFields)for(s in e.xhrFields)a[s]=e.xhrFields[s];for(s in e.mimeType&&a.overrideMimeType&&a.overrideMimeType(e.mimeType),e.crossDomain||i["X-Requested-With"]||(i["X-Requested-With"]="XMLHttpRequest"),i)a.setRequestHeader(s,i[s]);n=function(e){return function(){n&&(n=r=a.onload=a.onerror=a.onabort=a.ontimeout=a.onreadystatechange=null,"abort"===e?a.abort():"error"===e?"number"!=typeof a.status?o(0,"error"):o(a.status,a.statusText):o(Bt[a.status]||a.status,a.statusText,"text"!==(a.responseType||"text")||"string"!=typeof a.responseText?{binary:a.response}:{text:a.responseText},a.getAllResponseHeaders()))}},a.onload=n(),r=a.onerror=a.ontimeout=n("error"),void 0!==a.onabort?a.onabort=r:a.onreadystatechange=function(){4===a.readyState&&t.setTimeout(function(){n&&r()})},n=n("abort");try{a.send(e.hasContent&&e.data||null)}catch(u){if(n)throw u}},abort:function(){n&&n()}}}),w.ajaxPrefilter(function(e){e.crossDomain&&(e.contents.script=!1)}),w.ajaxSetup({accepts:{script:"text/javascript, application/javascript, application/ecmascript, application/x-ecmascript"},contents:{script:/\b(?:java|ecma)script\b/},converters:{"text script":function(e){return w.globalEval(e),e}}}),w.ajaxPrefilter("script",function(e){void 0===e.cache&&(e.cache=!1),e.crossDomain&&(e.type="GET")}),w.ajaxTransport("script",function(e){var t,n;if(e.crossDomain)return{send:function(r,o){t=w("<script>").prop({charset:e.scriptCharset,src:e.url}).on("load error",n=function(e){t.remove(),n=null,e&&o("error"===e.type?404:200,e.type)}),i.head.appendChild(t[0])},abort:function(){n&&n()}}});var Ft,_t=[],zt=/(=)\?(?=&|$)|\?\?/;w.ajaxSetup({jsonp:"callback",jsonpCallback:function(){var e=_t.pop()||w.expando+"_"+bt++;return this[e]=!0,e}}),w.ajaxPrefilter("json jsonp",function(e,n,r){var i,o,s,a=!1!==e.jsonp&&(zt.test(e.url)?"url":"string"==typeof e.data&&0===(e.contentType||"").indexOf("application/x-www-form-urlencoded")&&zt.test(e.data)&&"data");if(a||"jsonp"===e.dataTypes[0])return i=e.jsonpCallback=v(e.jsonpCallback)?e.jsonpCallback():e.jsonpCallback,a?e[a]=e[a].replace(zt,"$1"+i):!1!==e.jsonp&&(e.url+=(wt.test(e.url)?"&":"?")+e.jsonp+"="+i),e.converters["script json"]=function(){return s||w.error(i+" was not called"),s[0]},e.dataTypes[0]="json",o=t[i],t[i]=function(){s=arguments},r.always(function(){void 0===o?w(t).removeProp(i):t[i]=o,e[i]&&(e.jsonpCallback=n.jsonpCallback,_t.push(i)),s&&v(o)&&o(s[0]),s=o=void 0}),"script"}),g.createHTMLDocument=((Ft=i.implementation.createHTMLDocument("").body).innerHTML="<form></form><form></form>",2===Ft.childNodes.length),w.parseHTML=function(e,t,n){return"string"!=typeof e?[]:("boolean"==typeof t&&(n=t,t=!1),t||(g.createHTMLDocument?((r=(t=i.implementation.createHTMLDocument("")).createElement("base")).href=i.location.href,t.head.appendChild(r)):t=i),s=!n&&[],(o=N.exec(e))?[t.createElement(o[1])]:(o=xe([e],t,s),s&&s.length&&w(s).remove(),w.merge([],o.childNodes)));var r,o,s},w.fn.load=function(e,t,n){var r,i,o,s=this,a=e.indexOf(" ");return a>-1&&(r=dt(e.slice(a)),e=e.slice(0,a)),v(t)?(n=t,t=void 0):t&&"object"==typeof t&&(i="POST"),s.length>0&&w.ajax({url:e,type:i||"GET",dataType:"html",data:t}).done(function(e){o=arguments,s.html(r?w("<div>").append(w.parseHTML(e)).find(r):e)}).always(n&&function(e,t){s.each(function(){n.apply(this,o||[e.responseText,t,e])})}),this},w.each(["ajaxStart","ajaxStop","ajaxComplete","ajaxError","ajaxSuccess","ajaxSend"],function(e,t){w.fn[t]=function(e){return this.on(t,e)}}),w.expr.pseudos.animated=function(e){return w.grep(w.timers,function(t){return e===t.elem}).length},w.offset={setOffset:function(e,t,n){var r,i,o,s,a,u,c=w.css(e,"position"),f=w(e),l={};"static"===c&&(e.style.position="relative"),a=f.offset(),o=w.css(e,"top"),u=w.css(e,"left"),("absolute"===c||"fixed"===c)&&(o+u).indexOf("auto")>-1?(s=(r=f.position()).top,i=r.left):(s=parseFloat(o)||0,i=parseFloat(u)||0),v(t)&&(t=t.call(e,n,w.extend({},a))),null!=t.top&&(l.top=t.top-a.top+s),null!=t.left&&(l.left=t.left-a.left+i),"using"in t?t.using.call(e,l):f.css(l)}},w.fn.extend({offset:function(e){if(arguments.length)return void 0===e?this:this.each(function(t){w.offset.setOffset(this,e,t)});var t,n,r=this[0];return r?r.getClientRects().length?(t=r.getBoundingClientRect(),n=r.ownerDocument.defaultView,{top:t.top+n.pageYOffset,left:t.left+n.pageXOffset}):{top:0,left:0}:void 0},position:function(){if(this[0]){var e,t,n,r=this[0],i={top:0,left:0};if("fixed"===w.css(r,"position"))t=r.getBoundingClientRect();else{for(t=this.offset(),n=r.ownerDocument,e=r.offsetParent||n.documentElement;e&&(e===n.body||e===n.documentElement)&&"static"===w.css(e,"position");)e=e.parentNode;e&&e!==r&&1===e.nodeType&&((i=w(e).offset()).top+=w.css(e,"borderTopWidth",!0),i.left+=w.css(e,"borderLeftWidth",!0))}return{top:t.top-i.top-w.css(r,"marginTop",!0),left:t.left-i.left-w.css(r,"marginLeft",!0)}}},offsetParent:function(){return this.map(function(){for(var e=this.offsetParent;e&&"static"===w.css(e,"position");)e=e.offsetParent;return e||be})}}),w.each({scrollLeft:"pageXOffset",scrollTop:"pageYOffset"},function(e,t){var n="pageYOffset"===t;w.fn[e]=function(r){return _(this,function(e,r,i){var o;if(y(e)?o=e:9===e.nodeType&&(o=e.defaultView),void 0===i)return o?o[t]:e[r];o?o.scrollTo(n?o.pageXOffset:i,n?i:o.pageYOffset):e[r]=i},e,r,arguments.length)}}),w.each(["top","left"],function(e,t){w.cssHooks[t]=Fe(g.pixelPosition,function(e,n){if(n)return n=Ie(e,t),We.test(n)?w(e).position()[t]+"px":n})}),w.each({Height:"height",Width:"width"},function(e,t){w.each({padding:"inner"+e,content:t,"":"outer"+e},function(n,r){w.fn[r]=function(i,o){var s=arguments.length&&(n||"boolean"!=typeof i),a=n||(!0===i||!0===o?"margin":"border");return _(this,function(t,n,i){var o;return y(t)?0===r.indexOf("outer")?t["inner"+e]:t.document.documentElement["client"+e]:9===t.nodeType?(o=t.documentElement,Math.max(t.body["scroll"+e],o["scroll"+e],t.body["offset"+e],o["offset"+e],o["client"+e])):void 0===i?w.css(t,n,a):w.style(t,n,i,a)},t,s?i:void 0,s)}})}),w.each("blur focus focusin focusout resize scroll click dblclick mousedown mouseup mousemove mouseover mouseout mouseenter mouseleave change select submit keydown keypress keyup contextmenu".split(" "),function(e,t){w.fn[t]=function(e,n){return arguments.length>0?this.on(t,null,e,n):this.trigger(t)}}),w.fn.extend({hover:function(e,t){return this.mouseenter(e).mouseleave(t||e)}}),w.fn.extend({bind:function(e,t,n){return this.on(e,null,t,n)},unbind:function(e,t){return this.off(e,null,t)},delegate:function(e,t,n,r){return this.on(t,e,n,r)},undelegate:function(e,t,n){return 1===arguments.length?this.off(e,"**"):this.off(t,e||"**",n)}}),w.proxy=function(e,t){var n,r,i;if("string"==typeof t&&(n=e[t],t=e,e=n),v(e))return r=s.call(arguments,2),(i=function(){return e.apply(t||this,r.concat(s.call(arguments)))}).guid=e.guid=e.guid||w.guid++,i},w.holdReady=function(e){e?w.readyWait++:w.ready(!0)},w.isArray=Array.isArray,w.parseJSON=JSON.parse,w.nodeName=A,w.isFunction=v,w.isWindow=y,w.camelCase=X,w.type=b,w.now=Date.now,w.isNumeric=function(e){var t=w.type(e);return("number"===t||"string"===t)&&!isNaN(e-parseFloat(e))},"function"==typeof e&&e.amd&&e("jquery",[],function(){return w});var Ut=t.jQuery,Vt=t.$;return w.noConflict=function(e){return t.$===w&&(t.$=Vt),e&&t.jQuery===w&&(t.jQuery=Ut),w},n||(t.jQuery=t.$=w),w});
},{"process":"g5I+"}],"+cfP":[function(require,module,exports) {
var define;
var e;!function(e){e(["jquery"],function(e){return function(){var t,n,s,o=0,i={error:"error",info:"info",success:"success",warning:"warning"},a={clear:function(n,s){var o=u();t||r(o);l(n,o,s)||function(n){for(var s=t.children(),o=s.length-1;o>=0;o--)l(e(s[o]),n)}(o)},remove:function(n){var s=u();t||r(s);if(n&&0===e(":focus",n).length)return void p(n);t.children().length&&t.remove()},error:function(e,t,n){return d({type:i.error,iconClass:u().iconClasses.error,message:e,optionsOverride:n,title:t})},getContainer:r,info:function(e,t,n){return d({type:i.info,iconClass:u().iconClasses.info,message:e,optionsOverride:n,title:t})},options:{},subscribe:function(e){n=e},success:function(e,t,n){return d({type:i.success,iconClass:u().iconClasses.success,message:e,optionsOverride:n,title:t})},version:"2.1.4",warning:function(e,t,n){return d({type:i.warning,iconClass:u().iconClasses.warning,message:e,optionsOverride:n,title:t})}};return a;function r(n,s){return n||(n=u()),(t=e("#"+n.containerId)).length?t:(s&&(t=function(n){return(t=e("<div/>").attr("id",n.containerId).addClass(n.positionClass)).appendTo(e(n.target)),t}(n)),t)}function l(t,n,s){var o=!(!s||!s.force)&&s.force;return!(!t||!o&&0!==e(":focus",t).length)&&(t[n.hideMethod]({duration:n.hideDuration,easing:n.hideEasing,complete:function(){p(t)}}),!0)}function c(e){n&&n(e)}function d(n){var i=u(),a=n.iconClass||i.iconClass;if(void 0!==n.optionsOverride&&(i=e.extend(i,n.optionsOverride),a=n.optionsOverride.iconClass||a),!function(e,t){if(e.preventDuplicates){if(t.message===s)return!0;s=t.message}return!1}(i,n)){o++,t=r(i,!0);var l=null,d=e("<div/>"),g=e("<div/>"),m=e("<div/>"),f=e("<div/>"),h=e(i.closeHtml),v={intervalId:null,hideEta:null,maxHideTime:null},C={toastId:o,state:"visible",startTime:new Date,options:i,map:n};return n.iconClass&&d.addClass(i.toastClass).addClass(a),function(){if(n.title){var e=n.title;i.escapeHtml&&(e=w(n.title)),g.append(e).addClass(i.titleClass),d.append(g)}}(),function(){if(n.message){var e=n.message;i.escapeHtml&&(e=w(n.message)),m.append(e).addClass(i.messageClass),d.append(m)}}(),i.closeButton&&(h.addClass(i.closeClass).attr("role","button"),d.prepend(h)),i.progressBar&&(f.addClass(i.progressClass),d.prepend(f)),i.rtl&&d.addClass("rtl"),i.newestOnTop?t.prepend(d):t.append(d),function(){var e="";switch(n.iconClass){case"toast-success":case"toast-info":e="polite";break;default:e="assertive"}d.attr("aria-live",e)}(),d.hide(),d[i.showMethod]({duration:i.showDuration,easing:i.showEasing,complete:i.onShown}),i.timeOut>0&&(l=setTimeout(T,i.timeOut),v.maxHideTime=parseFloat(i.timeOut),v.hideEta=(new Date).getTime()+v.maxHideTime,i.progressBar&&(v.intervalId=setInterval(D,10))),function(){i.closeOnHover&&d.hover(b,O);!i.onclick&&i.tapToDismiss&&d.click(T);i.closeButton&&h&&h.click(function(e){e.stopPropagation?e.stopPropagation():void 0!==e.cancelBubble&&!0!==e.cancelBubble&&(e.cancelBubble=!0),i.onCloseClick&&i.onCloseClick(e),T(!0)});i.onclick&&d.click(function(e){i.onclick(e),T()})}(),c(C),i.debug&&console&&console.log(C),d}function w(e){return null==e&&(e=""),e.replace(/&/g,"&amp;").replace(/"/g,"&quot;").replace(/'/g,"&#39;").replace(/</g,"&lt;").replace(/>/g,"&gt;")}function T(t){var n=t&&!1!==i.closeMethod?i.closeMethod:i.hideMethod,s=t&&!1!==i.closeDuration?i.closeDuration:i.hideDuration,o=t&&!1!==i.closeEasing?i.closeEasing:i.hideEasing;if(!e(":focus",d).length||t)return clearTimeout(v.intervalId),d[n]({duration:s,easing:o,complete:function(){p(d),clearTimeout(l),i.onHidden&&"hidden"!==C.state&&i.onHidden(),C.state="hidden",C.endTime=new Date,c(C)}})}function O(){(i.timeOut>0||i.extendedTimeOut>0)&&(l=setTimeout(T,i.extendedTimeOut),v.maxHideTime=parseFloat(i.extendedTimeOut),v.hideEta=(new Date).getTime()+v.maxHideTime)}function b(){clearTimeout(l),v.hideEta=0,d.stop(!0,!0)[i.showMethod]({duration:i.showDuration,easing:i.showEasing})}function D(){var e=(v.hideEta-(new Date).getTime())/v.maxHideTime*100;f.width(e+"%")}}function u(){return e.extend({},{tapToDismiss:!0,toastClass:"toast",containerId:"toast-container",debug:!1,showMethod:"fadeIn",showDuration:300,showEasing:"swing",onShown:void 0,hideMethod:"fadeOut",hideDuration:1e3,hideEasing:"swing",onHidden:void 0,closeMethod:!1,closeDuration:!1,closeEasing:!1,closeOnHover:!0,extendedTimeOut:1e3,iconClasses:{error:"toast-error",info:"toast-info",success:"toast-success",warning:"toast-warning"},iconClass:"toast-info",positionClass:"toast-top-right",timeOut:5e3,titleClass:"toast-title",messageClass:"toast-message",escapeHtml:!1,target:"body",closeHtml:'<button type="button">&times;</button>',closeClass:"toast-close-button",newestOnTop:!0,preventDuplicates:!1,progressBar:!1,progressClass:"toast-progress",rtl:!1},a.options)}function p(e){t||(t=r()),e.is(":visible")||(e.remove(),e=null,0===t.children().length&&(t.remove(),s=void 0))}}()})}("function"==typeof e&&e.amd?e:function(e,t){"undefined"!=typeof module&&module.exports?module.exports=t(require("jquery")):window.toastr=t(window.jQuery)});
},{"jquery":"HlZQ"}],"ZKus":[function(require,module,exports) {
var global = arguments[3];
var define;
var n,r=arguments[3];!function(){var t="object"==typeof self&&self.self===self&&self||"object"==typeof r&&r.global===r&&r||this||{},e=t._,u=Array.prototype,i=Object.prototype,o="undefined"!=typeof Symbol?Symbol.prototype:null,c=u.push,a=u.slice,f=i.toString,l=i.hasOwnProperty,s=Array.isArray,p=Object.keys,v=Object.create,y=function(){},h=function(n){return n instanceof h?n:this instanceof h?void(this._wrapped=n):new h(n)};"undefined"==typeof exports||exports.nodeType?t._=h:("undefined"!=typeof module&&!module.nodeType&&module.exports&&(exports=module.exports=h),exports._=h),h.VERSION="1.9.1";var d,m=function(n,r,t){if(void 0===r)return n;switch(null==t?3:t){case 1:return function(t){return n.call(r,t)};case 3:return function(t,e,u){return n.call(r,t,e,u)};case 4:return function(t,e,u,i){return n.call(r,t,e,u,i)}}return function(){return n.apply(r,arguments)}},g=function(n,r,t){return h.iteratee!==d?h.iteratee(n,r):null==n?h.identity:h.isFunction(n)?m(n,r,t):h.isObject(n)&&!h.isArray(n)?h.matcher(n):h.property(n)};h.iteratee=d=function(n,r){return g(n,r,1/0)};var _=function(n,r){return r=null==r?n.length-1:+r,function(){for(var t=Math.max(arguments.length-r,0),e=Array(t),u=0;u<t;u++)e[u]=arguments[u+r];switch(r){case 0:return n.call(this,e);case 1:return n.call(this,arguments[0],e);case 2:return n.call(this,arguments[0],arguments[1],e)}var i=Array(r+1);for(u=0;u<r;u++)i[u]=arguments[u];return i[r]=e,n.apply(this,i)}},b=function(n){if(!h.isObject(n))return{};if(v)return v(n);y.prototype=n;var r=new y;return y.prototype=null,r},A=function(n){return function(r){return null==r?void 0:r[n]}},O=function(n,r){return null!=n&&l.call(n,r)},w=function(n,r){for(var t=r.length,e=0;e<t;e++){if(null==n)return;n=n[r[e]]}return t?n:void 0},S=Math.pow(2,53)-1,k=A("length"),x=function(n){var r=k(n);return"number"==typeof r&&r>=0&&r<=S};h.each=h.forEach=function(n,r,t){var e,u;if(r=m(r,t),x(n))for(e=0,u=n.length;e<u;e++)r(n[e],e,n);else{var i=h.keys(n);for(e=0,u=i.length;e<u;e++)r(n[i[e]],i[e],n)}return n},h.map=h.collect=function(n,r,t){r=g(r,t);for(var e=!x(n)&&h.keys(n),u=(e||n).length,i=Array(u),o=0;o<u;o++){var c=e?e[o]:o;i[o]=r(n[c],c,n)}return i};var M=function(n){return function(r,t,e,u){var i=arguments.length>=3;return function(r,t,e,u){var i=!x(r)&&h.keys(r),o=(i||r).length,c=n>0?0:o-1;for(u||(e=r[i?i[c]:c],c+=n);c>=0&&c<o;c+=n){var a=i?i[c]:c;e=t(e,r[a],a,r)}return e}(r,m(t,u,4),e,i)}};h.reduce=h.foldl=h.inject=M(1),h.reduceRight=h.foldr=M(-1),h.find=h.detect=function(n,r,t){var e=(x(n)?h.findIndex:h.findKey)(n,r,t);if(void 0!==e&&-1!==e)return n[e]},h.filter=h.select=function(n,r,t){var e=[];return r=g(r,t),h.each(n,function(n,t,u){r(n,t,u)&&e.push(n)}),e},h.reject=function(n,r,t){return h.filter(n,h.negate(g(r)),t)},h.every=h.all=function(n,r,t){r=g(r,t);for(var e=!x(n)&&h.keys(n),u=(e||n).length,i=0;i<u;i++){var o=e?e[i]:i;if(!r(n[o],o,n))return!1}return!0},h.some=h.any=function(n,r,t){r=g(r,t);for(var e=!x(n)&&h.keys(n),u=(e||n).length,i=0;i<u;i++){var o=e?e[i]:i;if(r(n[o],o,n))return!0}return!1},h.contains=h.includes=h.include=function(n,r,t,e){return x(n)||(n=h.values(n)),("number"!=typeof t||e)&&(t=0),h.indexOf(n,r,t)>=0},h.invoke=_(function(n,r,t){var e,u;return h.isFunction(r)?u=r:h.isArray(r)&&(e=r.slice(0,-1),r=r[r.length-1]),h.map(n,function(n){var i=u;if(!i){if(e&&e.length&&(n=w(n,e)),null==n)return;i=n[r]}return null==i?i:i.apply(n,t)})}),h.pluck=function(n,r){return h.map(n,h.property(r))},h.where=function(n,r){return h.filter(n,h.matcher(r))},h.findWhere=function(n,r){return h.find(n,h.matcher(r))},h.max=function(n,r,t){var e,u,i=-1/0,o=-1/0;if(null==r||"number"==typeof r&&"object"!=typeof n[0]&&null!=n)for(var c=0,a=(n=x(n)?n:h.values(n)).length;c<a;c++)null!=(e=n[c])&&e>i&&(i=e);else r=g(r,t),h.each(n,function(n,t,e){((u=r(n,t,e))>o||u===-1/0&&i===-1/0)&&(i=n,o=u)});return i},h.min=function(n,r,t){var e,u,i=1/0,o=1/0;if(null==r||"number"==typeof r&&"object"!=typeof n[0]&&null!=n)for(var c=0,a=(n=x(n)?n:h.values(n)).length;c<a;c++)null!=(e=n[c])&&e<i&&(i=e);else r=g(r,t),h.each(n,function(n,t,e){((u=r(n,t,e))<o||u===1/0&&i===1/0)&&(i=n,o=u)});return i},h.shuffle=function(n){return h.sample(n,1/0)},h.sample=function(n,r,t){if(null==r||t)return x(n)||(n=h.values(n)),n[h.random(n.length-1)];var e=x(n)?h.clone(n):h.values(n),u=k(e);r=Math.max(Math.min(r,u),0);for(var i=u-1,o=0;o<r;o++){var c=h.random(o,i),a=e[o];e[o]=e[c],e[c]=a}return e.slice(0,r)},h.sortBy=function(n,r,t){var e=0;return r=g(r,t),h.pluck(h.map(n,function(n,t,u){return{value:n,index:e++,criteria:r(n,t,u)}}).sort(function(n,r){var t=n.criteria,e=r.criteria;if(t!==e){if(t>e||void 0===t)return 1;if(t<e||void 0===e)return-1}return n.index-r.index}),"value")};var F=function(n,r){return function(t,e,u){var i=r?[[],[]]:{};return e=g(e,u),h.each(t,function(r,u){var o=e(r,u,t);n(i,r,o)}),i}};h.groupBy=F(function(n,r,t){O(n,t)?n[t].push(r):n[t]=[r]}),h.indexBy=F(function(n,r,t){n[t]=r}),h.countBy=F(function(n,r,t){O(n,t)?n[t]++:n[t]=1});var E=/[^\ud800-\udfff]|[\ud800-\udbff][\udc00-\udfff]|[\ud800-\udfff]/g;h.toArray=function(n){return n?h.isArray(n)?a.call(n):h.isString(n)?n.match(E):x(n)?h.map(n,h.identity):h.values(n):[]},h.size=function(n){return null==n?0:x(n)?n.length:h.keys(n).length},h.partition=F(function(n,r,t){n[t?0:1].push(r)},!0),h.first=h.head=h.take=function(n,r,t){return null==n||n.length<1?null==r?void 0:[]:null==r||t?n[0]:h.initial(n,n.length-r)},h.initial=function(n,r,t){return a.call(n,0,Math.max(0,n.length-(null==r||t?1:r)))},h.last=function(n,r,t){return null==n||n.length<1?null==r?void 0:[]:null==r||t?n[n.length-1]:h.rest(n,Math.max(0,n.length-r))},h.rest=h.tail=h.drop=function(n,r,t){return a.call(n,null==r||t?1:r)},h.compact=function(n){return h.filter(n,Boolean)};var I=function(n,r,t,e){for(var u=(e=e||[]).length,i=0,o=k(n);i<o;i++){var c=n[i];if(x(c)&&(h.isArray(c)||h.isArguments(c)))if(r)for(var a=0,f=c.length;a<f;)e[u++]=c[a++];else I(c,r,t,e),u=e.length;else t||(e[u++]=c)}return e};h.flatten=function(n,r){return I(n,r,!1)},h.without=_(function(n,r){return h.difference(n,r)}),h.uniq=h.unique=function(n,r,t,e){h.isBoolean(r)||(e=t,t=r,r=!1),null!=t&&(t=g(t,e));for(var u=[],i=[],o=0,c=k(n);o<c;o++){var a=n[o],f=t?t(a,o,n):a;r&&!t?(o&&i===f||u.push(a),i=f):t?h.contains(i,f)||(i.push(f),u.push(a)):h.contains(u,a)||u.push(a)}return u},h.union=_(function(n){return h.uniq(I(n,!0,!0))}),h.intersection=function(n){for(var r=[],t=arguments.length,e=0,u=k(n);e<u;e++){var i=n[e];if(!h.contains(r,i)){var o;for(o=1;o<t&&h.contains(arguments[o],i);o++);o===t&&r.push(i)}}return r},h.difference=_(function(n,r){return r=I(r,!0,!0),h.filter(n,function(n){return!h.contains(r,n)})}),h.unzip=function(n){for(var r=n&&h.max(n,k).length||0,t=Array(r),e=0;e<r;e++)t[e]=h.pluck(n,e);return t},h.zip=_(h.unzip),h.object=function(n,r){for(var t={},e=0,u=k(n);e<u;e++)r?t[n[e]]=r[e]:t[n[e][0]]=n[e][1];return t};var T=function(n){return function(r,t,e){t=g(t,e);for(var u=k(r),i=n>0?0:u-1;i>=0&&i<u;i+=n)if(t(r[i],i,r))return i;return-1}};h.findIndex=T(1),h.findLastIndex=T(-1),h.sortedIndex=function(n,r,t,e){for(var u=(t=g(t,e,1))(r),i=0,o=k(n);i<o;){var c=Math.floor((i+o)/2);t(n[c])<u?i=c+1:o=c}return i};var N=function(n,r,t){return function(e,u,i){var o=0,c=k(e);if("number"==typeof i)n>0?o=i>=0?i:Math.max(i+c,o):c=i>=0?Math.min(i+1,c):i+c+1;else if(t&&i&&c)return e[i=t(e,u)]===u?i:-1;if(u!=u)return(i=r(a.call(e,o,c),h.isNaN))>=0?i+o:-1;for(i=n>0?o:c-1;i>=0&&i<c;i+=n)if(e[i]===u)return i;return-1}};h.indexOf=N(1,h.findIndex,h.sortedIndex),h.lastIndexOf=N(-1,h.findLastIndex),h.range=function(n,r,t){null==r&&(r=n||0,n=0),t||(t=r<n?-1:1);for(var e=Math.max(Math.ceil((r-n)/t),0),u=Array(e),i=0;i<e;i++,n+=t)u[i]=n;return u},h.chunk=function(n,r){if(null==r||r<1)return[];for(var t=[],e=0,u=n.length;e<u;)t.push(a.call(n,e,e+=r));return t};var B=function(n,r,t,e,u){if(!(e instanceof r))return n.apply(t,u);var i=b(n.prototype),o=n.apply(i,u);return h.isObject(o)?o:i};h.bind=_(function(n,r,t){if(!h.isFunction(n))throw new TypeError("Bind must be called on a function");var e=_(function(u){return B(n,e,r,this,t.concat(u))});return e}),h.partial=_(function(n,r){var t=h.partial.placeholder,e=function(){for(var u=0,i=r.length,o=Array(i),c=0;c<i;c++)o[c]=r[c]===t?arguments[u++]:r[c];for(;u<arguments.length;)o.push(arguments[u++]);return B(n,e,this,this,o)};return e}),h.partial.placeholder=h,h.bindAll=_(function(n,r){var t=(r=I(r,!1,!1)).length;if(t<1)throw new Error("bindAll must be passed function names");for(;t--;){var e=r[t];n[e]=h.bind(n[e],n)}}),h.memoize=function(n,r){var t=function(e){var u=t.cache,i=""+(r?r.apply(this,arguments):e);return O(u,i)||(u[i]=n.apply(this,arguments)),u[i]};return t.cache={},t},h.delay=_(function(n,r,t){return setTimeout(function(){return n.apply(null,t)},r)}),h.defer=h.partial(h.delay,h,1),h.throttle=function(n,r,t){var e,u,i,o,c=0;t||(t={});var a=function(){c=!1===t.leading?0:h.now(),e=null,o=n.apply(u,i),e||(u=i=null)},f=function(){var f=h.now();c||!1!==t.leading||(c=f);var l=r-(f-c);return u=this,i=arguments,l<=0||l>r?(e&&(clearTimeout(e),e=null),c=f,o=n.apply(u,i),e||(u=i=null)):e||!1===t.trailing||(e=setTimeout(a,l)),o};return f.cancel=function(){clearTimeout(e),c=0,e=u=i=null},f},h.debounce=function(n,r,t){var e,u,i=function(r,t){e=null,t&&(u=n.apply(r,t))},o=_(function(o){if(e&&clearTimeout(e),t){var c=!e;e=setTimeout(i,r),c&&(u=n.apply(this,o))}else e=h.delay(i,r,this,o);return u});return o.cancel=function(){clearTimeout(e),e=null},o},h.wrap=function(n,r){return h.partial(r,n)},h.negate=function(n){return function(){return!n.apply(this,arguments)}},h.compose=function(){var n=arguments,r=n.length-1;return function(){for(var t=r,e=n[r].apply(this,arguments);t--;)e=n[t].call(this,e);return e}},h.after=function(n,r){return function(){if(--n<1)return r.apply(this,arguments)}},h.before=function(n,r){var t;return function(){return--n>0&&(t=r.apply(this,arguments)),n<=1&&(r=null),t}},h.once=h.partial(h.before,2),h.restArguments=_;var R=!{toString:null}.propertyIsEnumerable("toString"),K=["valueOf","isPrototypeOf","toString","propertyIsEnumerable","hasOwnProperty","toLocaleString"],z=function(n,r){var t=K.length,e=n.constructor,u=h.isFunction(e)&&e.prototype||i,o="constructor";for(O(n,o)&&!h.contains(r,o)&&r.push(o);t--;)(o=K[t])in n&&n[o]!==u[o]&&!h.contains(r,o)&&r.push(o)};h.keys=function(n){if(!h.isObject(n))return[];if(p)return p(n);var r=[];for(var t in n)O(n,t)&&r.push(t);return R&&z(n,r),r},h.allKeys=function(n){if(!h.isObject(n))return[];var r=[];for(var t in n)r.push(t);return R&&z(n,r),r},h.values=function(n){for(var r=h.keys(n),t=r.length,e=Array(t),u=0;u<t;u++)e[u]=n[r[u]];return e},h.mapObject=function(n,r,t){r=g(r,t);for(var e=h.keys(n),u=e.length,i={},o=0;o<u;o++){var c=e[o];i[c]=r(n[c],c,n)}return i},h.pairs=function(n){for(var r=h.keys(n),t=r.length,e=Array(t),u=0;u<t;u++)e[u]=[r[u],n[r[u]]];return e},h.invert=function(n){for(var r={},t=h.keys(n),e=0,u=t.length;e<u;e++)r[n[t[e]]]=t[e];return r},h.functions=h.methods=function(n){var r=[];for(var t in n)h.isFunction(n[t])&&r.push(t);return r.sort()};var q=function(n,r){return function(t){var e=arguments.length;if(r&&(t=Object(t)),e<2||null==t)return t;for(var u=1;u<e;u++)for(var i=arguments[u],o=n(i),c=o.length,a=0;a<c;a++){var f=o[a];r&&void 0!==t[f]||(t[f]=i[f])}return t}};h.extend=q(h.allKeys),h.extendOwn=h.assign=q(h.keys),h.findKey=function(n,r,t){r=g(r,t);for(var e,u=h.keys(n),i=0,o=u.length;i<o;i++)if(r(n[e=u[i]],e,n))return e};var D,P,W=function(n,r,t){return r in t};h.pick=_(function(n,r){var t={},e=r[0];if(null==n)return t;h.isFunction(e)?(r.length>1&&(e=m(e,r[1])),r=h.allKeys(n)):(e=W,r=I(r,!1,!1),n=Object(n));for(var u=0,i=r.length;u<i;u++){var o=r[u],c=n[o];e(c,o,n)&&(t[o]=c)}return t}),h.omit=_(function(n,r){var t,e=r[0];return h.isFunction(e)?(e=h.negate(e),r.length>1&&(t=r[1])):(r=h.map(I(r,!1,!1),String),e=function(n,t){return!h.contains(r,t)}),h.pick(n,e,t)}),h.defaults=q(h.allKeys,!0),h.create=function(n,r){var t=b(n);return r&&h.extendOwn(t,r),t},h.clone=function(n){return h.isObject(n)?h.isArray(n)?n.slice():h.extend({},n):n},h.tap=function(n,r){return r(n),n},h.isMatch=function(n,r){var t=h.keys(r),e=t.length;if(null==n)return!e;for(var u=Object(n),i=0;i<e;i++){var o=t[i];if(r[o]!==u[o]||!(o in u))return!1}return!0},D=function(n,r,t,e){if(n===r)return 0!==n||1/n==1/r;if(null==n||null==r)return!1;if(n!=n)return r!=r;var u=typeof n;return("function"===u||"object"===u||"object"==typeof r)&&P(n,r,t,e)},P=function(n,r,t,e){n instanceof h&&(n=n._wrapped),r instanceof h&&(r=r._wrapped);var u=f.call(n);if(u!==f.call(r))return!1;switch(u){case"[object RegExp]":case"[object String]":return""+n==""+r;case"[object Number]":return+n!=+n?+r!=+r:0==+n?1/+n==1/r:+n==+r;case"[object Date]":case"[object Boolean]":return+n==+r;case"[object Symbol]":return o.valueOf.call(n)===o.valueOf.call(r)}var i="[object Array]"===u;if(!i){if("object"!=typeof n||"object"!=typeof r)return!1;var c=n.constructor,a=r.constructor;if(c!==a&&!(h.isFunction(c)&&c instanceof c&&h.isFunction(a)&&a instanceof a)&&"constructor"in n&&"constructor"in r)return!1}e=e||[];for(var l=(t=t||[]).length;l--;)if(t[l]===n)return e[l]===r;if(t.push(n),e.push(r),i){if((l=n.length)!==r.length)return!1;for(;l--;)if(!D(n[l],r[l],t,e))return!1}else{var s,p=h.keys(n);if(l=p.length,h.keys(r).length!==l)return!1;for(;l--;)if(s=p[l],!O(r,s)||!D(n[s],r[s],t,e))return!1}return t.pop(),e.pop(),!0},h.isEqual=function(n,r){return D(n,r)},h.isEmpty=function(n){return null==n||(x(n)&&(h.isArray(n)||h.isString(n)||h.isArguments(n))?0===n.length:0===h.keys(n).length)},h.isElement=function(n){return!(!n||1!==n.nodeType)},h.isArray=s||function(n){return"[object Array]"===f.call(n)},h.isObject=function(n){var r=typeof n;return"function"===r||"object"===r&&!!n},h.each(["Arguments","Function","String","Number","Date","RegExp","Error","Symbol","Map","WeakMap","Set","WeakSet"],function(n){h["is"+n]=function(r){return f.call(r)==="[object "+n+"]"}}),h.isArguments(arguments)||(h.isArguments=function(n){return O(n,"callee")});var C=t.document&&t.document.childNodes;"function"!=typeof/./&&"object"!=typeof Int8Array&&"function"!=typeof C&&(h.isFunction=function(n){return"function"==typeof n||!1}),h.isFinite=function(n){return!h.isSymbol(n)&&isFinite(n)&&!isNaN(parseFloat(n))},h.isNaN=function(n){return h.isNumber(n)&&isNaN(n)},h.isBoolean=function(n){return!0===n||!1===n||"[object Boolean]"===f.call(n)},h.isNull=function(n){return null===n},h.isUndefined=function(n){return void 0===n},h.has=function(n,r){if(!h.isArray(r))return O(n,r);for(var t=r.length,e=0;e<t;e++){var u=r[e];if(null==n||!l.call(n,u))return!1;n=n[u]}return!!t},h.noConflict=function(){return t._=e,this},h.identity=function(n){return n},h.constant=function(n){return function(){return n}},h.noop=function(){},h.property=function(n){return h.isArray(n)?function(r){return w(r,n)}:A(n)},h.propertyOf=function(n){return null==n?function(){}:function(r){return h.isArray(r)?w(n,r):n[r]}},h.matcher=h.matches=function(n){return n=h.extendOwn({},n),function(r){return h.isMatch(r,n)}},h.times=function(n,r,t){var e=Array(Math.max(0,n));r=m(r,t,1);for(var u=0;u<n;u++)e[u]=r(u);return e},h.random=function(n,r){return null==r&&(r=n,n=0),n+Math.floor(Math.random()*(r-n+1))},h.now=Date.now||function(){return(new Date).getTime()};var J={"&":"&amp;","<":"&lt;",">":"&gt;",'"':"&quot;","'":"&#x27;","`":"&#x60;"},U=h.invert(J),V=function(n){var r=function(r){return n[r]},t="(?:"+h.keys(n).join("|")+")",e=RegExp(t),u=RegExp(t,"g");return function(n){return n=null==n?"":""+n,e.test(n)?n.replace(u,r):n}};h.escape=V(J),h.unescape=V(U),h.result=function(n,r,t){h.isArray(r)||(r=[r]);var e=r.length;if(!e)return h.isFunction(t)?t.call(n):t;for(var u=0;u<e;u++){var i=null==n?void 0:n[r[u]];void 0===i&&(i=t,u=e),n=h.isFunction(i)?i.call(n):i}return n};var $=0;h.uniqueId=function(n){var r=++$+"";return n?n+r:r},h.templateSettings={evaluate:/<%([\s\S]+?)%>/g,interpolate:/<%=([\s\S]+?)%>/g,escape:/<%-([\s\S]+?)%>/g};var G=/(.)^/,H={"'":"'","\\":"\\","\r":"r","\n":"n","\u2028":"u2028","\u2029":"u2029"},Q=/\\|'|\r|\n|\u2028|\u2029/g,X=function(n){return"\\"+H[n]};h.template=function(n,r,t){!r&&t&&(r=t),r=h.defaults({},r,h.templateSettings);var e,u=RegExp([(r.escape||G).source,(r.interpolate||G).source,(r.evaluate||G).source].join("|")+"|$","g"),i=0,o="__p+='";n.replace(u,function(r,t,e,u,c){return o+=n.slice(i,c).replace(Q,X),i=c+r.length,t?o+="'+\n((__t=("+t+"))==null?'':_.escape(__t))+\n'":e?o+="'+\n((__t=("+e+"))==null?'':__t)+\n'":u&&(o+="';\n"+u+"\n__p+='"),r}),o+="';\n",r.variable||(o="with(obj||{}){\n"+o+"}\n"),o="var __t,__p='',__j=Array.prototype.join,print=function(){__p+=__j.call(arguments,'');};\n"+o+"return __p;\n";try{e=new Function(r.variable||"obj","_",o)}catch(f){throw f.source=o,f}var c=function(n){return e.call(this,n,h)},a=r.variable||"obj";return c.source="function("+a+"){\n"+o+"}",c},h.chain=function(n){var r=h(n);return r._chain=!0,r};var Y=function(n,r){return n._chain?h(r).chain():r};h.mixin=function(n){return h.each(h.functions(n),function(r){var t=h[r]=n[r];h.prototype[r]=function(){var n=[this._wrapped];return c.apply(n,arguments),Y(this,t.apply(h,n))}}),h},h.mixin(h),h.each(["pop","push","reverse","shift","sort","splice","unshift"],function(n){var r=u[n];h.prototype[n]=function(){var t=this._wrapped;return r.apply(t,arguments),"shift"!==n&&"splice"!==n||0!==t.length||delete t[0],Y(this,t)}}),h.each(["concat","join","slice"],function(n){var r=u[n];h.prototype[n]=function(){return Y(this,r.apply(this._wrapped,arguments))}}),h.prototype.value=function(){return this._wrapped},h.prototype.valueOf=h.prototype.toJSON=h.prototype.value,h.prototype.toString=function(){return String(this._wrapped)},"function"==typeof n&&n.amd&&n("underscore",[],function(){return h})}();
},{}],"5kFN":[function(require,module,exports) {
var global = arguments[3];
var define;
var t,e=arguments[3];!function(i){var n="object"==typeof self&&self.self===self&&self||"object"==typeof e&&e.global===e&&e;if("function"==typeof t&&t.amd)t(["underscore","jquery","exports"],function(t,e,r){n.Backbone=i(n,r,t,e)});else if("undefined"!=typeof exports){var r,s=require("underscore");try{r=require("jquery")}catch(h){}i(n,exports,s,r)}else n.Backbone=i(n,{},n._,n.jQuery||n.Zepto||n.ender||n.$)}(function(t,e,i,n){var r=t.Backbone,s=Array.prototype.slice;e.VERSION="1.3.3",e.$=n,e.noConflict=function(){return t.Backbone=r,this},e.emulateHTTP=!1,e.emulateJSON=!1;var h=function(t,e,n){i.each(e,function(e,r){i[r]&&(t.prototype[r]=function(t,e,n){switch(t){case 1:return function(){return i[e](this[n])};case 2:return function(t){return i[e](this[n],t)};case 3:return function(t,r){return i[e](this[n],a(t,this),r)};case 4:return function(t,r,s){return i[e](this[n],a(t,this),r,s)};default:return function(){var t=s.call(arguments);return t.unshift(this[n]),i[e].apply(i,t)}}}(e,r,n))})},a=function(t,e){return i.isFunction(t)?t:i.isObject(t)&&!e._isModel(t)?o(t):i.isString(t)?function(e){return e.get(t)}:t},o=function(t){var e=i.matches(t);return function(t){return e(t.attributes)}},u=e.Events={},c=/\s+/,l=function(t,e,n,r,s){var h,a=0;if(n&&"object"==typeof n){void 0!==r&&"context"in s&&void 0===s.context&&(s.context=r);for(h=i.keys(n);a<h.length;a++)e=l(t,e,h[a],n[h[a]],s)}else if(n&&c.test(n))for(h=n.split(c);a<h.length;a++)e=t(e,h[a],r,s);else e=t(e,n,r,s);return e};u.on=function(t,e,i){return d(this,t,e,i)};var d=function(t,e,i,n,r){(t._events=l(f,t._events||{},e,i,{context:n,ctx:t,listening:r}),r)&&((t._listeners||(t._listeners={}))[r.id]=r);return t};u.listenTo=function(t,e,n){if(!t)return this;var r=t._listenId||(t._listenId=i.uniqueId("l")),s=this._listeningTo||(this._listeningTo={}),h=s[r];if(!h){var a=this._listenId||(this._listenId=i.uniqueId("l"));h=s[r]={obj:t,objId:r,id:a,listeningTo:s,count:0}}return d(t,e,n,this,h),this};var f=function(t,e,i,n){if(i){var r=t[e]||(t[e]=[]),s=n.context,h=n.ctx,a=n.listening;a&&a.count++,r.push({callback:i,context:s,ctx:s||h,listening:a})}return t};u.off=function(t,e,i){return this._events?(this._events=l(g,this._events,t,e,{context:i,listeners:this._listeners}),this):this},u.stopListening=function(t,e,n){var r=this._listeningTo;if(!r)return this;for(var s=t?[t._listenId]:i.keys(r),h=0;h<s.length;h++){var a=r[s[h]];if(!a)break;a.obj.off(e,n,this)}return this};var g=function(t,e,n,r){if(t){var s,h=0,a=r.context,o=r.listeners;if(e||n||a){for(var u=e?[e]:i.keys(t);h<u.length;h++){var c=t[e=u[h]];if(!c)break;for(var l=[],d=0;d<c.length;d++){var f=c[d];n&&n!==f.callback&&n!==f.callback._callback||a&&a!==f.context?l.push(f):(s=f.listening)&&0==--s.count&&(delete o[s.id],delete s.listeningTo[s.objId])}l.length?t[e]=l:delete t[e]}return t}for(var g=i.keys(o);h<g.length;h++)delete o[(s=o[g[h]]).id],delete s.listeningTo[s.objId]}};u.once=function(t,e,n){var r=l(v,{},t,e,i.bind(this.off,this));return"string"==typeof t&&null==n&&(e=void 0),this.on(r,e,n)},u.listenToOnce=function(t,e,n){var r=l(v,{},e,n,i.bind(this.stopListening,this,t));return this.listenTo(t,r)};var v=function(t,e,n,r){if(n){var s=t[e]=i.once(function(){r(e,s),n.apply(this,arguments)});s._callback=n}return t};u.trigger=function(t){if(!this._events)return this;for(var e=Math.max(0,arguments.length-1),i=Array(e),n=0;n<e;n++)i[n]=arguments[n+1];return l(p,this._events,t,void 0,i),this};var p=function(t,e,i,n){if(t){var r=t[e],s=t.all;r&&s&&(s=s.slice()),r&&m(r,n),s&&m(s,[e].concat(n))}return t},m=function(t,e){var i,n=-1,r=t.length,s=e[0],h=e[1],a=e[2];switch(e.length){case 0:for(;++n<r;)(i=t[n]).callback.call(i.ctx);return;case 1:for(;++n<r;)(i=t[n]).callback.call(i.ctx,s);return;case 2:for(;++n<r;)(i=t[n]).callback.call(i.ctx,s,h);return;case 3:for(;++n<r;)(i=t[n]).callback.call(i.ctx,s,h,a);return;default:for(;++n<r;)(i=t[n]).callback.apply(i.ctx,e);return}};u.bind=u.on,u.unbind=u.off,i.extend(e,u);var _=e.Model=function(t,e){var n=t||{};e||(e={}),this.cid=i.uniqueId(this.cidPrefix),this.attributes={},e.collection&&(this.collection=e.collection),e.parse&&(n=this.parse(n,e)||{});var r=i.result(this,"defaults");n=i.defaults(i.extend({},r,n),r),this.set(n,e),this.changed={},this.initialize.apply(this,arguments)};i.extend(_.prototype,u,{changed:null,validationError:null,idAttribute:"id",cidPrefix:"c",initialize:function(){},toJSON:function(t){return i.clone(this.attributes)},sync:function(){return e.sync.apply(this,arguments)},get:function(t){return this.attributes[t]},escape:function(t){return i.escape(this.get(t))},has:function(t){return null!=this.get(t)},matches:function(t){return!!i.iteratee(t,this)(this.attributes)},set:function(t,e,n){if(null==t)return this;var r;if("object"==typeof t?(r=t,n=e):(r={})[t]=e,n||(n={}),!this._validate(r,n))return!1;var s=n.unset,h=n.silent,a=[],o=this._changing;this._changing=!0,o||(this._previousAttributes=i.clone(this.attributes),this.changed={});var u=this.attributes,c=this.changed,l=this._previousAttributes;for(var d in r)e=r[d],i.isEqual(u[d],e)||a.push(d),i.isEqual(l[d],e)?delete c[d]:c[d]=e,s?delete u[d]:u[d]=e;if(this.idAttribute in r&&(this.id=this.get(this.idAttribute)),!h){a.length&&(this._pending=n);for(var f=0;f<a.length;f++)this.trigger("change:"+a[f],this,u[a[f]],n)}if(o)return this;if(!h)for(;this._pending;)n=this._pending,this._pending=!1,this.trigger("change",this,n);return this._pending=!1,this._changing=!1,this},unset:function(t,e){return this.set(t,void 0,i.extend({},e,{unset:!0}))},clear:function(t){var e={};for(var n in this.attributes)e[n]=void 0;return this.set(e,i.extend({},t,{unset:!0}))},hasChanged:function(t){return null==t?!i.isEmpty(this.changed):i.has(this.changed,t)},changedAttributes:function(t){if(!t)return!!this.hasChanged()&&i.clone(this.changed);var e=this._changing?this._previousAttributes:this.attributes,n={};for(var r in t){var s=t[r];i.isEqual(e[r],s)||(n[r]=s)}return!!i.size(n)&&n},previous:function(t){return null!=t&&this._previousAttributes?this._previousAttributes[t]:null},previousAttributes:function(){return i.clone(this._previousAttributes)},fetch:function(t){t=i.extend({parse:!0},t);var e=this,n=t.success;return t.success=function(i){var r=t.parse?e.parse(i,t):i;if(!e.set(r,t))return!1;n&&n.call(t.context,e,i,t),e.trigger("sync",e,i,t)},$(this,t),this.sync("read",this,t)},save:function(t,e,n){var r;null==t||"object"==typeof t?(r=t,n=e):(r={})[t]=e;var s=(n=i.extend({validate:!0,parse:!0},n)).wait;if(r&&!s){if(!this.set(r,n))return!1}else if(!this._validate(r,n))return!1;var h=this,a=n.success,o=this.attributes;n.success=function(t){h.attributes=o;var e=n.parse?h.parse(t,n):t;if(s&&(e=i.extend({},r,e)),e&&!h.set(e,n))return!1;a&&a.call(n.context,h,t,n),h.trigger("sync",h,t,n)},$(this,n),r&&s&&(this.attributes=i.extend({},o,r));var u=this.isNew()?"create":n.patch?"patch":"update";"patch"!==u||n.attrs||(n.attrs=r);var c=this.sync(u,this,n);return this.attributes=o,c},destroy:function(t){t=t?i.clone(t):{};var e=this,n=t.success,r=t.wait,s=function(){e.stopListening(),e.trigger("destroy",e,e.collection,t)};t.success=function(i){r&&s(),n&&n.call(t.context,e,i,t),e.isNew()||e.trigger("sync",e,i,t)};var h=!1;return this.isNew()?i.defer(t.success):($(this,t),h=this.sync("delete",this,t)),r||s(),h},url:function(){var t=i.result(this,"urlRoot")||i.result(this.collection,"url")||U();if(this.isNew())return t;var e=this.get(this.idAttribute);return t.replace(/[^\/]$/,"$&/")+encodeURIComponent(e)},parse:function(t,e){return t},clone:function(){return new this.constructor(this.attributes)},isNew:function(){return!this.has(this.idAttribute)},isValid:function(t){return this._validate({},i.extend({},t,{validate:!0}))},_validate:function(t,e){if(!e.validate||!this.validate)return!0;t=i.extend({},this.attributes,t);var n=this.validationError=this.validate(t,e)||null;return!n||(this.trigger("invalid",this,n,i.extend(e,{validationError:n})),!1)}});h(_,{keys:1,values:1,pairs:1,invert:1,pick:0,omit:0,chain:1,isEmpty:1},"attributes");var y=e.Collection=function(t,e){e||(e={}),e.model&&(this.model=e.model),void 0!==e.comparator&&(this.comparator=e.comparator),this._reset(),this.initialize.apply(this,arguments),t&&this.reset(t,i.extend({silent:!0},e))},b={add:!0,remove:!0,merge:!0},x={add:!0,remove:!1},w=function(t,e,i){i=Math.min(Math.max(i,0),t.length);var n,r=Array(t.length-i),s=e.length;for(n=0;n<r.length;n++)r[n]=t[n+i];for(n=0;n<s;n++)t[n+i]=e[n];for(n=0;n<r.length;n++)t[n+s+i]=r[n]};i.extend(y.prototype,u,{model:_,initialize:function(){},toJSON:function(t){return this.map(function(e){return e.toJSON(t)})},sync:function(){return e.sync.apply(this,arguments)},add:function(t,e){return this.set(t,i.extend({merge:!1},e,x))},remove:function(t,e){e=i.extend({},e);var n=!i.isArray(t);t=n?[t]:t.slice();var r=this._removeModels(t,e);return!e.silent&&r.length&&(e.changes={added:[],merged:[],removed:r},this.trigger("update",this,e)),n?r[0]:r},set:function(t,e){if(null!=t){(e=i.extend({},b,e)).parse&&!this._isModel(t)&&(t=this.parse(t,e)||[]);var n=!i.isArray(t);t=n?[t]:t.slice();var r=e.at;null!=r&&(r=+r),r>this.length&&(r=this.length),r<0&&(r+=this.length+1);var s,h,a=[],o=[],u=[],c=[],l={},d=e.add,f=e.merge,g=e.remove,v=!1,p=this.comparator&&null==r&&!1!==e.sort,m=i.isString(this.comparator)?this.comparator:null;for(h=0;h<t.length;h++){s=t[h];var _=this.get(s);if(_){if(f&&s!==_){var y=this._isModel(s)?s.attributes:s;e.parse&&(y=_.parse(y,e)),_.set(y,e),u.push(_),p&&!v&&(v=_.hasChanged(m))}l[_.cid]||(l[_.cid]=!0,a.push(_)),t[h]=_}else d&&(s=t[h]=this._prepareModel(s,e))&&(o.push(s),this._addReference(s,e),l[s.cid]=!0,a.push(s))}if(g){for(h=0;h<this.length;h++)l[(s=this.models[h]).cid]||c.push(s);c.length&&this._removeModels(c,e)}var x=!1,E=!p&&d&&g;if(a.length&&E?(x=this.length!==a.length||i.some(this.models,function(t,e){return t!==a[e]}),this.models.length=0,w(this.models,a,0),this.length=this.models.length):o.length&&(p&&(v=!0),w(this.models,o,null==r?this.length:r),this.length=this.models.length),v&&this.sort({silent:!0}),!e.silent){for(h=0;h<o.length;h++)null!=r&&(e.index=r+h),(s=o[h]).trigger("add",s,this,e);(v||x)&&this.trigger("sort",this,e),(o.length||c.length||u.length)&&(e.changes={added:o,removed:c,merged:u},this.trigger("update",this,e))}return n?t[0]:t}},reset:function(t,e){e=e?i.clone(e):{};for(var n=0;n<this.models.length;n++)this._removeReference(this.models[n],e);return e.previousModels=this.models,this._reset(),t=this.add(t,i.extend({silent:!0},e)),e.silent||this.trigger("reset",this,e),t},push:function(t,e){return this.add(t,i.extend({at:this.length},e))},pop:function(t){var e=this.at(this.length-1);return this.remove(e,t)},unshift:function(t,e){return this.add(t,i.extend({at:0},e))},shift:function(t){var e=this.at(0);return this.remove(e,t)},slice:function(){return s.apply(this.models,arguments)},get:function(t){if(null!=t)return this._byId[t]||this._byId[this.modelId(t.attributes||t)]||t.cid&&this._byId[t.cid]},has:function(t){return null!=this.get(t)},at:function(t){return t<0&&(t+=this.length),this.models[t]},where:function(t,e){return this[e?"find":"filter"](t)},findWhere:function(t){return this.where(t,!0)},sort:function(t){var e=this.comparator;if(!e)throw new Error("Cannot sort a set without a comparator");t||(t={});var n=e.length;return i.isFunction(e)&&(e=i.bind(e,this)),1===n||i.isString(e)?this.models=this.sortBy(e):this.models.sort(e),t.silent||this.trigger("sort",this,t),this},pluck:function(t){return this.map(t+"")},fetch:function(t){var e=(t=i.extend({parse:!0},t)).success,n=this;return t.success=function(i){var r=t.reset?"reset":"set";n[r](i,t),e&&e.call(t.context,n,i,t),n.trigger("sync",n,i,t)},$(this,t),this.sync("read",this,t)},create:function(t,e){var n=(e=e?i.clone(e):{}).wait;if(!(t=this._prepareModel(t,e)))return!1;n||this.add(t,e);var r=this,s=e.success;return e.success=function(t,e,i){n&&r.add(t,i),s&&s.call(i.context,t,e,i)},t.save(null,e),t},parse:function(t,e){return t},clone:function(){return new this.constructor(this.models,{model:this.model,comparator:this.comparator})},modelId:function(t){return t[this.model.prototype.idAttribute||"id"]},_reset:function(){this.length=0,this.models=[],this._byId={}},_prepareModel:function(t,e){if(this._isModel(t))return t.collection||(t.collection=this),t;(e=e?i.clone(e):{}).collection=this;var n=new this.model(t,e);return n.validationError?(this.trigger("invalid",this,n.validationError,e),!1):n},_removeModels:function(t,e){for(var i=[],n=0;n<t.length;n++){var r=this.get(t[n]);if(r){var s=this.indexOf(r);this.models.splice(s,1),this.length--,delete this._byId[r.cid];var h=this.modelId(r.attributes);null!=h&&delete this._byId[h],e.silent||(e.index=s,r.trigger("remove",r,this,e)),i.push(r),this._removeReference(r,e)}}return i},_isModel:function(t){return t instanceof _},_addReference:function(t,e){this._byId[t.cid]=t;var i=this.modelId(t.attributes);null!=i&&(this._byId[i]=t),t.on("all",this._onModelEvent,this)},_removeReference:function(t,e){delete this._byId[t.cid];var i=this.modelId(t.attributes);null!=i&&delete this._byId[i],this===t.collection&&delete t.collection,t.off("all",this._onModelEvent,this)},_onModelEvent:function(t,e,i,n){if(e){if(("add"===t||"remove"===t)&&i!==this)return;if("destroy"===t&&this.remove(e,n),"change"===t){var r=this.modelId(e.previousAttributes()),s=this.modelId(e.attributes);r!==s&&(null!=r&&delete this._byId[r],null!=s&&(this._byId[s]=e))}}this.trigger.apply(this,arguments)}});h(y,{forEach:3,each:3,map:3,collect:3,reduce:0,foldl:0,inject:0,reduceRight:0,foldr:0,find:3,detect:3,filter:3,select:3,reject:3,every:3,all:3,some:3,any:3,include:3,includes:3,contains:3,invoke:0,max:3,min:3,toArray:1,size:1,first:3,head:3,take:3,initial:3,rest:3,tail:3,drop:3,last:3,without:0,difference:0,indexOf:3,shuffle:1,lastIndexOf:3,isEmpty:1,chain:1,sample:3,partition:3,groupBy:3,countBy:3,sortBy:3,indexBy:3,findIndex:3,findLastIndex:3},"models");var E=e.View=function(t){this.cid=i.uniqueId("view"),i.extend(this,i.pick(t,S)),this._ensureElement(),this.initialize.apply(this,arguments)},I=/^(\S+)\s*(.*)$/,S=["model","collection","el","id","attributes","className","tagName","events"];i.extend(E.prototype,u,{tagName:"div",$:function(t){return this.$el.find(t)},initialize:function(){},render:function(){return this},remove:function(){return this._removeElement(),this.stopListening(),this},_removeElement:function(){this.$el.remove()},setElement:function(t){return this.undelegateEvents(),this._setElement(t),this.delegateEvents(),this},_setElement:function(t){this.$el=t instanceof e.$?t:e.$(t),this.el=this.$el[0]},delegateEvents:function(t){if(t||(t=i.result(this,"events")),!t)return this;for(var e in this.undelegateEvents(),t){var n=t[e];if(i.isFunction(n)||(n=this[n]),n){var r=e.match(I);this.delegate(r[1],r[2],i.bind(n,this))}}return this},delegate:function(t,e,i){return this.$el.on(t+".delegateEvents"+this.cid,e,i),this},undelegateEvents:function(){return this.$el&&this.$el.off(".delegateEvents"+this.cid),this},undelegate:function(t,e,i){return this.$el.off(t+".delegateEvents"+this.cid,e,i),this},_createElement:function(t){return document.createElement(t)},_ensureElement:function(){if(this.el)this.setElement(i.result(this,"el"));else{var t=i.extend({},i.result(this,"attributes"));this.id&&(t.id=i.result(this,"id")),this.className&&(t.class=i.result(this,"className")),this.setElement(this._createElement(i.result(this,"tagName"))),this._setAttributes(t)}},_setAttributes:function(t){this.$el.attr(t)}}),e.sync=function(t,n,r){var s=k[t];i.defaults(r||(r={}),{emulateHTTP:e.emulateHTTP,emulateJSON:e.emulateJSON});var h={type:s,dataType:"json"};if(r.url||(h.url=i.result(n,"url")||U()),null!=r.data||!n||"create"!==t&&"update"!==t&&"patch"!==t||(h.contentType="application/json",h.data=JSON.stringify(r.attrs||n.toJSON(r))),r.emulateJSON&&(h.contentType="application/x-www-form-urlencoded",h.data=h.data?{model:h.data}:{}),r.emulateHTTP&&("PUT"===s||"DELETE"===s||"PATCH"===s)){h.type="POST",r.emulateJSON&&(h.data._method=s);var a=r.beforeSend;r.beforeSend=function(t){if(t.setRequestHeader("X-HTTP-Method-Override",s),a)return a.apply(this,arguments)}}"GET"===h.type||r.emulateJSON||(h.processData=!1);var o=r.error;r.error=function(t,e,i){r.textStatus=e,r.errorThrown=i,o&&o.call(r.context,t,e,i)};var u=r.xhr=e.ajax(i.extend(h,r));return n.trigger("request",n,u,r),u};var k={create:"POST",update:"PUT",patch:"PATCH",delete:"DELETE",read:"GET"};e.ajax=function(){return e.$.ajax.apply(e.$,arguments)};var T=e.Router=function(t){t||(t={}),t.routes&&(this.routes=t.routes),this._bindRoutes(),this.initialize.apply(this,arguments)},P=/\((.*?)\)/g,H=/(\(\?)?:\w+/g,C=/\*\w+/g,A=/[\-{}\[\]+?.,\\\^$|#\s]/g;i.extend(T.prototype,u,{initialize:function(){},route:function(t,n,r){i.isRegExp(t)||(t=this._routeToRegExp(t)),i.isFunction(n)&&(r=n,n=""),r||(r=this[n]);var s=this;return e.history.route(t,function(i){var h=s._extractParameters(t,i);!1!==s.execute(r,h,n)&&(s.trigger.apply(s,["route:"+n].concat(h)),s.trigger("route",n,h),e.history.trigger("route",s,n,h))}),this},execute:function(t,e,i){t&&t.apply(this,e)},navigate:function(t,i){return e.history.navigate(t,i),this},_bindRoutes:function(){if(this.routes){this.routes=i.result(this,"routes");for(var t,e=i.keys(this.routes);null!=(t=e.pop());)this.route(t,this.routes[t])}},_routeToRegExp:function(t){return t=t.replace(A,"\\$&").replace(P,"(?:$1)?").replace(H,function(t,e){return e?t:"([^/?]+)"}).replace(C,"([^?]*?)"),new RegExp("^"+t+"(?:\\?([\\s\\S]*))?$")},_extractParameters:function(t,e){var n=t.exec(e).slice(1);return i.map(n,function(t,e){return e===n.length-1?t||null:t?decodeURIComponent(t):null})}});var R=e.History=function(){this.handlers=[],this.checkUrl=i.bind(this.checkUrl,this),"undefined"!=typeof window&&(this.location=window.location,this.history=window.history)},j=/^[#\/]|\s+$/g,N=/^\/+|\/+$/g,O=/#.*$/;R.started=!1,i.extend(R.prototype,u,{interval:50,atRoot:function(){return this.location.pathname.replace(/[^\/]$/,"$&/")===this.root&&!this.getSearch()},matchRoot:function(){return this.decodeFragment(this.location.pathname).slice(0,this.root.length-1)+"/"===this.root},decodeFragment:function(t){return decodeURI(t.replace(/%25/g,"%2525"))},getSearch:function(){var t=this.location.href.replace(/#.*/,"").match(/\?.+/);return t?t[0]:""},getHash:function(t){var e=(t||this).location.href.match(/#(.*)$/);return e?e[1]:""},getPath:function(){var t=this.decodeFragment(this.location.pathname+this.getSearch()).slice(this.root.length-1);return"/"===t.charAt(0)?t.slice(1):t},getFragment:function(t){return null==t&&(t=this._usePushState||!this._wantsHashChange?this.getPath():this.getHash()),t.replace(j,"")},start:function(t){if(R.started)throw new Error("Backbone.history has already been started");if(R.started=!0,this.options=i.extend({root:"/"},this.options,t),this.root=this.options.root,this._wantsHashChange=!1!==this.options.hashChange,this._hasHashChange="onhashchange"in window&&(void 0===document.documentMode||document.documentMode>7),this._useHashChange=this._wantsHashChange&&this._hasHashChange,this._wantsPushState=!!this.options.pushState,this._hasPushState=!(!this.history||!this.history.pushState),this._usePushState=this._wantsPushState&&this._hasPushState,this.fragment=this.getFragment(),this.root=("/"+this.root+"/").replace(N,"/"),this._wantsHashChange&&this._wantsPushState){if(!this._hasPushState&&!this.atRoot()){var e=this.root.slice(0,-1)||"/";return this.location.replace(e+"#"+this.getPath()),!0}this._hasPushState&&this.atRoot()&&this.navigate(this.getHash(),{replace:!0})}if(!this._hasHashChange&&this._wantsHashChange&&!this._usePushState){this.iframe=document.createElement("iframe"),this.iframe.src="javascript:0",this.iframe.style.display="none",this.iframe.tabIndex=-1;var n=document.body,r=n.insertBefore(this.iframe,n.firstChild).contentWindow;r.document.open(),r.document.close(),r.location.hash="#"+this.fragment}var s=window.addEventListener||function(t,e){return attachEvent("on"+t,e)};if(this._usePushState?s("popstate",this.checkUrl,!1):this._useHashChange&&!this.iframe?s("hashchange",this.checkUrl,!1):this._wantsHashChange&&(this._checkUrlInterval=setInterval(this.checkUrl,this.interval)),!this.options.silent)return this.loadUrl()},stop:function(){var t=window.removeEventListener||function(t,e){return detachEvent("on"+t,e)};this._usePushState?t("popstate",this.checkUrl,!1):this._useHashChange&&!this.iframe&&t("hashchange",this.checkUrl,!1),this.iframe&&(document.body.removeChild(this.iframe),this.iframe=null),this._checkUrlInterval&&clearInterval(this._checkUrlInterval),R.started=!1},route:function(t,e){this.handlers.unshift({route:t,callback:e})},checkUrl:function(t){var e=this.getFragment();if(e===this.fragment&&this.iframe&&(e=this.getHash(this.iframe.contentWindow)),e===this.fragment)return!1;this.iframe&&this.navigate(e),this.loadUrl()},loadUrl:function(t){return!!this.matchRoot()&&(t=this.fragment=this.getFragment(t),i.some(this.handlers,function(e){if(e.route.test(t))return e.callback(t),!0}))},navigate:function(t,e){if(!R.started)return!1;e&&!0!==e||(e={trigger:!!e}),t=this.getFragment(t||"");var i=this.root;""!==t&&"?"!==t.charAt(0)||(i=i.slice(0,-1)||"/");var n=i+t;if(t=this.decodeFragment(t.replace(O,"")),this.fragment!==t){if(this.fragment=t,this._usePushState)this.history[e.replace?"replaceState":"pushState"]({},document.title,n);else{if(!this._wantsHashChange)return this.location.assign(n);if(this._updateHash(this.location,t,e.replace),this.iframe&&t!==this.getHash(this.iframe.contentWindow)){var r=this.iframe.contentWindow;e.replace||(r.document.open(),r.document.close()),this._updateHash(r.location,t,e.replace)}}return e.trigger?this.loadUrl(t):void 0}},_updateHash:function(t,e,i){if(i){var n=t.href.replace(/(javascript:|#).*$/,"");t.replace(n+"#"+e)}else t.hash="#"+e}}),e.history=new R;_.extend=y.extend=T.extend=E.extend=R.extend=function(t,e){var n,r=this;return n=t&&i.has(t,"constructor")?t.constructor:function(){return r.apply(this,arguments)},i.extend(n,r,e),n.prototype=i.create(r.prototype,t),n.prototype.constructor=n,n.__super__=r.prototype,n};var U=function(){throw new Error('A "url" property or function must be specified')},$=function(t,e){var i=e.error;e.error=function(n){i&&i.call(e.context,t,n,e),t.trigger("error",t,n,e)}};return e});
},{"underscore":"ZKus","jquery":"HlZQ"}],"wXLC":[function(require,module,exports) {
"use strict";var r=c(require("toastr")),n=c(require("backbone")),i=c(require("underscore"));function c(r){if(r&&r.__esModule)return r;var n={};if(null!=r)for(var i in r)if(Object.prototype.hasOwnProperty.call(r,i)){var c=Object.defineProperty&&Object.getOwnPropertyDescriptor?Object.getOwnPropertyDescriptor(r,i):{};c.get||c.set?Object.defineProperty(n,i,c):n[i]=r[i]}return n.default=r,n}window.TourGenerator=n.View.extend({initialize:function(n){var c=this;c.toolId=n.toolId,c.toolVersion=n.toolVersion,$("#execute").attr("tour_id","execute"),r.info("Tour generation might take some time."),$.getJSON("/api/webhooks/tour_generator/data/",{tool_id:c.toolId,tool_version:c.toolVersion},function(n){if(n.success)if(n.data.useDatasets){var g=window.bundleEntries.getGalaxyInstance();g.currHistoryPanel.refreshContents(),setTimeout(function(){var f=[],w=0;i.each(n.data.hids,function(r){var n=g.currHistoryPanel.collection.where({hid:r})[0];n&&f.push(n)}),f.length===n.data.hids.length?i.each(f,function(r){"ok"===r.get("state")?w++:r.on("change:state",function(r){"ok"===r.get("state")&&w++,w===f.length&&c._generateTour(n.data.tour)})}):(r.warning("Cannot generate a tour."),console.error("Some of the test datasets cannot be found in the history."))},1500)}else c._generateTour(n.data.tour);else r.warning("Cannot generate a tour."),console.error("Tour Generator: "+n.error)})},_generateTour:function(r){var n=window.bundleEntries.getGalaxyInstance().giveTourWithData(r);$("#execute").on("mousedown",function(){n&&n.end()})}});
},{"toastr":"+cfP","backbone":"5kFN","underscore":"ZKus"}]},{},["wXLC"], null)</script><script type="text/javascript">(() => {
    let gtnWebhookLoaded = false;
    let lastUpdate = 0;
    const urlParams = new URLSearchParams(document.location.search);
    const autoLoadTutorial =
        urlParams.get("autoload_gtn_tutorial") === null ? "" : urlParams.get("autoload_gtn_tutorial");

    function removeOverlay() {
        const container = document.getElementById("gtn-container");
        if (container) {
            container.style.visibility = "hidden";
        }
    }

    function showOverlay() {
        document.getElementById("gtn-container").style.visibility = "visible";
    }

    function getIframeUrl() {
        let loc;
        try {
            loc = document.getElementById("gtn-embed").contentWindow.location.pathname;
        } catch (e) {
            loc = null;
        }
        return loc;
    }

    function getIframeScroll() {
        let loc;
        try {
            loc = parseInt(document.getElementById("gtn-embed").contentWindow.scrollY);
        } catch (e) {
            loc = 0;
        }
        return loc;
    }

    function persistLocation() {
        // Don't save every scroll event.
        const time = new Date().getTime();
        if (time - lastUpdate < 1000) {
            return;
        }
        lastUpdate = time;
        window.localStorage.setItem("gtn-in-galaxy", `${getIframeScroll()} ${getIframeUrl()}`);
    }

    function addIframe() {
        let url;
        let message;
        let onloadscroll;
        gtnWebhookLoaded = true;
        let storedData = false;
        let safe = false;

        // Test for the presence of /training-material/. If that is available we
        // can opt in the fancy click-to-run features. Otherwise we fallback to
        // displaying the real GTN.
        fetch("/training-material/")
            .then((response) => {
                if (!response.ok) {
                    url = `https://training.galaxyproject.org/training-material/${autoLoadTutorial}?utm_source=webhook&utm_medium=noproxy&utm_campaign=gxy`;
                    message = `
                        <span>
                            <a href="https://docs.galaxyproject.org/en/master/admin/special_topics/gtn.html">Click to run</a> unavailable.
                        </span>`;
                } else {
                    safe = true;

                    const storedLocation = window.localStorage.getItem("gtn-in-galaxy");
                    if (
                        storedLocation !== null &&
                        storedLocation.split(" ")[1] !== undefined &&
                        storedLocation.split(" ")[1].startsWith("/training-material/")
                    ) {
                        onloadscroll = storedLocation.split(" ")[0];
                        url = storedLocation.split(" ")[1];
                    } else {
                        url = `/training-material/${autoLoadTutorial}?utm_source=webhook&utm_medium=proxy&utm_campaign=gxy`;
                    }
                    message = "";
                }
            })
            .then(() => {
                document.querySelector("body.full-content").insertAdjacentHTML(
                    "afterbegin",
                    `
                    <div id="gtn-container">
                        <div id="gtn-screen-overlay"></div>
                        <div id="gtn-screen">
                            <div id="gtn-header">
                                <iframe id="gtn-embed" src="${url}" width="80%" height="80%"></iframe>
                                ${message}
                            </div>
                        </div>
                    </div>`
                );

                // Clicking outside of GTN closes it
                document.getElementById("gtn-screen").addEventListener("click", () => {
                    removeOverlay();
                });

                // Only setup the listener if it won't crash things.
                if (safe) {
                    // Listen to the scroll position
                    document.getElementById("gtn-embed").contentWindow.addEventListener("scroll", () => {
                        persistLocation();
                    });
                }

                // Depends on the iframe being present
                document.getElementById("gtn-embed").addEventListener("load", () => {
                    // Save our current location when possible
                    const gtnEmbed = document.getElementById("gtn-embed");
                    if (onloadscroll !== undefined) {
                        document.getElementById("gtn-embed").contentWindow.scrollTo(0, parseInt(onloadscroll));
                        onloadscroll = undefined;
                    }

                    if (safe) {
                        persistLocation();
                    }
                    // Add the class to the entire GTN page
                    document
                        .getElementById("gtn-embed")
                        .contentDocument.getElementsByTagName("body")[0]
                        .classList.add("galaxy-proxy-active");

                    const gtnToolElements = document
                        .getElementById("gtn-embed")
                        .contentDocument.querySelectorAll("span[data-tool],a[data-tool]");

                    // Buttonify
                    gtnToolElements.forEach(function (el) {
                        el.addEventListener("click", function (e) {
                            let target = e.target;

                            // Sometimes we get the i or the strong, not the parent.
                            if (e.target.tagName.toLowerCase() !== "span" && e.target.tagName.toLowerCase() !== "a") {
                                target = e.target.parentElement;
                            }

                            tool_id = target.dataset.tool;

                            if (tool_id === "upload1" || tool_id === "upload") {
                                document.getElementById("tool-panel-upload-button").click();
                            } else {
                                Galaxy.router.push({ path: `/?tool_id=${encodeURIComponent(tool_id)}` });
                            }
                            removeOverlay();
                        });
                    });

                    const gtnWorkflowElements = document
                        .getElementById("gtn-embed")
                        .contentDocument.querySelectorAll("span[data-workflow],a[data-workflow]");

                    // Buttonify
                    gtnWorkflowElements.forEach(function (el) {
                        el.addEventListener("click", (e) => {
                            let target = e.target;

                            // Sometimes we get the i or the strong, not the parent.
                            if (e.target.tagName.toLowerCase() !== "span" && e.target.tagName.toLowerCase() !== "a") {
                                target = e.target.parentElement;
                            }

                            trs_url = target.dataset.workflow;
                            Galaxy.router.push({
                                path: `/workflows/trs_import?trs_url=${encodeURIComponent(trs_url)}&run_form=true`,
                            });
                            removeOverlay();
                        });
                    });
                });
            });
    }
    /* The masthead icon may not exist yet when this webhook executes; we need this to wait for that to happen.
     * elementReady function from gist:
     * https://gist.github.com/jwilson8767/db379026efcbd932f64382db4b02853e
     */
    function elementReady(selector) {
        return new Promise((resolve, reject) => {
            let el = document.querySelector(selector);
            if (el) {
                resolve(el);
            }
            new MutationObserver((mutationRecords, observer) => {
                // Query for elements matching the specified selector
                Array.from(document.querySelectorAll(selector)).forEach((element) => {
                    resolve(element);
                    //Once we have resolved we don't need the observer anymore.
                    observer.disconnect();
                });
            }).observe(document.documentElement, {
                childList: true,
                subtree: true,
            });
        });
    }

    elementReady("#gtn a").then((el) => {
        // External stuff may also have attached a click handler here (vue-based masthead)
        // replace with a clean copy of the node to remove all that cruft.
        clean = el.cloneNode(true);
        el.parentNode.replaceChild(clean, el);
        clean.addEventListener("click", (e) => {
            e.preventDefault();
            e.stopPropagation();
            if (!gtnWebhookLoaded) {
                addIframe();
            } else {
                showOverlay();
            }
        });
        if (autoLoadTutorial) {
            clean.click();
        }
    });

    // Remove the overlay on escape button click
    document.addEventListener("keydown", (event) => {
        // Check for escape button - "Escape" (modern browsers), "27" (old browsers)
        if (event.key === "Escape" || event.keyCode === 27) {
            removeOverlay();
        }
    });
})();
</script><style>#gtn-screen {	position: fixed;	z-index: 5100;	width:100%;	height:100%;}#gtn-screen-overlay {	position: fixed;	top:0;	left:0;	background: rgba(224, 224, 224, 0.75);	z-index: 5000;	width:100%;	height:100%;	opacity: 2;}#gtn-header {	position: fixed;	height: 100%;	width: 100%;	z-index: 5200;	display: flex;	justify-content: center;	flex-direction: column;	align-items: center;}</style><script type="text/javascript">(function () {
    function hideNewsOverlay() {
        const container = document.getElementById("news-container");
        if (container) {
            container.style.visibility = "hidden";
        }
    }

    function newsSeen() {
        // When it's seen, remove fa, add far.
        const newsIconSpan = document.querySelector("#news .fa-bell");
        newsIconSpan.classList.remove("fa");
        newsIconSpan.classList.add("far");
        window.localStorage.setItem("galaxy-news-seen-release", Galaxy.config.version_major);
    }

    function newsUnseen() {
        // When there is news, remove far, add fa for (default -- same as fas) solid style.
        const newsIconSpan = document.querySelector("#news .fa-bell");
        newsIconSpan.classList.remove("far");
        newsIconSpan.classList.add("fa");
    }

    /* The masthead icon may not exist yet when this webhook executes; we need this to wait for that to happen.
     * elementReady function from gist:
     * https://gist.github.com/jwilson8767/db379026efcbd932f64382db4b02853e
     */
    function elementReadyNews(selector) {
        return new Promise((resolve, reject) => {
            const el = document.querySelector(selector);
            if (el) {
                resolve(el);
            }
            new MutationObserver((mutationRecords, observer) => {
                // Query for elements matching the specified selector
                Array.from(document.querySelectorAll(selector)).forEach((element) => {
                    resolve(element);
                    //Once we have resolved we don't need the observer anymore.
                    observer.disconnect();
                });
            }).observe(document.documentElement, {
                childList: true,
                subtree: true,
            });
        });
    }

    elementReadyNews("#news a").then((el) => {
        // External stuff may also have attached a click handler here (vue-based masthead)
        // replace with a clean copy of the node to remove all that cruft.
        clean = el.cloneNode(true);
        el.parentNode.replaceChild(clean, el);

        let currentGalaxyVersion = Galaxy.config.version_major;

        const releaseNotes = `https://docs.galaxyproject.org/en/latest/releases/${currentGalaxyVersion}_announce_user.html`;
        const lastSeenVersion = window.localStorage.getItem("galaxy-news-seen-release");
        // Check that they've seen the current version's release notes.
        if (lastSeenVersion != currentGalaxyVersion) {
            newsUnseen();
        } else {
            newsSeen();
        }

        clean.addEventListener("click", (e) => {
            e.preventDefault();
            e.stopPropagation();

            // If element doesn't exist, add it.
            if (document.getElementById("news-container") == null) {
                document.querySelector("body.full-content").insertAdjacentHTML(
                    "afterbegin",
                    `
                    <div id="news-container" style="visibility: hidden">
                        <div id="news-screen-overlay"></div>
                        <div id="news-screen">
                            <div id="news-header">
                                <iframe id="news-embed" src="${releaseNotes}" width="80%" height="80%"></iframe>
                            </div>
                        </div>
                    </div>`
                );
                // Clicking outside of GTN closes it
                document.getElementById("news-screen").addEventListener("click", () => {
                    hideNewsOverlay();
                });
            }
            document.getElementById("news-container").style.visibility = "visible";
            newsSeen();
        });
    });

    // Remove the overlay on escape button click
    document.addEventListener("keydown", (e) => {
        // Check for escape button - "27"
        if (e.which === 27 || e.keyCode === 27) {
            hideNewsOverlay();
        }
    });
})();
</script><style>#news-screen {	position: fixed;	z-index: 202;	width:100%;	height:100%;}#news-screen-overlay {	position: fixed;	top:0;	left:0;	background: rgba(224, 224, 224, 0.75);	z-index: 201;	width:100%;	height:100%;	opacity: 2;}#news-header {	position: fixed;	height: 100%;	width: 100%;	z-index: 203;	display: flex;	justify-content: center;	flex-direction: column;	align-items: center;}</style></head>

    <body scroll="no" class="full-content modal-open" data-modal-open-count="1"><div class="ui-modal"></div><div class="ui-modal"></div>
        <!-- Provide mount point for application -->
        <main>
            <div id="app" style="--masthead-color: #003399;"><div id="everything"><div id="background"></div> <nav data-v-e2d015b2="" class="navbar justify-content-between navbar-dark navbar-expand" id="masthead" aria-label="Main"><ul data-v-e2d015b2="" class="navbar-nav"><a data-v-e2d015b2="" title="Home" aria-label="homepage" href="/" target="_self" class="navbar-brand ml-2 mr-2"><img data-v-e2d015b2="" alt="logo" src="/static/favicon.svg"> <!----></a> <span data-v-e2d015b2="" class="navbar-text px-2">
            Europe
        </span></ul> <ul data-v-e2d015b2="" class="navbar-nav"><li data-v-e2d015b2="" id="analysis" title="Tools and Current History" class="nav-item active"><a href="/" target="_top" class="nav-link nav-icon"><span class="sr-only">Tools and Current History</span> <span class="fa fa-fw fa-home"></span> <!----></a></li><li data-v-e2d015b2="" id="workflow" title="Chain tools into workflows" class="nav-item"><a href="/workflows/list" target="_parent" class="nav-link">
        Workflow
    </a></li><li data-v-e2d015b2="" id="visualization" title="Visualize datasets" class="nav-item"><a href="/visualizations" target="_parent" class="nav-link">
        Visualize
    </a></li><li data-v-e2d015b2="" id="shared" class="nav-item b-nav-dropdown dropdown" href="#" title="Access published resources"><a id="shared__BV_toggle_" role="button" aria-haspopup="true" aria-expanded="false" aria-controls="shared__BV_toggle_menu_" href="#shared" target="_self" class="nav-link dropdown-toggle"><span>Shared Data</span></a><ul tabindex="-1" aria-labelledby="shared__BV_toggle_" id="shared__BV_toggle_menu_" class="dropdown-menu"> <li role="presentation"><a role="menuitem" title="" href="/libraries" target="_top" class="dropdown-item">
            Data Libraries
        </a></li><li role="presentation"><a role="menuitem" title="" href="/histories/list_published" target="_parent" class="dropdown-item">
            Histories
        </a></li><li role="presentation"><a role="menuitem" title="" href="/workflows/list_published" target="_parent" class="dropdown-item">
            Workflows
        </a></li><li role="presentation"><a role="menuitem" title="" href="/visualizations/list_published" target="_parent" class="dropdown-item">
            Visualizations
        </a></li><li role="presentation"><a role="menuitem" title="" href="/pages/list_published" target="_parent" class="dropdown-item">
            Pages
        </a></li></ul></li><li data-v-e2d015b2="" id="help" class="nav-item b-nav-dropdown dropdown" href="#" title="Support, contact, and community"><a id="help__BV_toggle_" role="button" aria-haspopup="true" aria-expanded="false" aria-controls="help__BV_toggle_menu_" href="#help" target="_self" class="nav-link dropdown-toggle"><span>Help</span></a><ul tabindex="-1" aria-labelledby="help__BV_toggle_" id="help__BV_toggle_menu_" class="dropdown-menu"> <li role="presentation"><a role="menuitem" title="" href="https://help.galaxyproject.org/c/usegalaxy-eu-support" rel="noopener" target="_blank" class="dropdown-item">
            Galaxy Help
        </a></li><li role="presentation"><a role="menuitem" title="" href="https://galaxyproject.org/support/" rel="noopener" target="_blank" class="dropdown-item">
            Support
        </a></li><li role="presentation"><a role="menuitem" title="" href="https://www.youtube.com/c/galaxyproject" rel="noopener" target="_blank" class="dropdown-item">
            Videos
        </a></li><li role="presentation"><a role="menuitem" title="" href="https://galaxyproject.org/" rel="noopener" target="_blank" class="dropdown-item">
            Community Hub
        </a></li><li role="presentation"><a role="menuitem" title="" href="https://galaxyproject.org/citing-galaxy" rel="noopener" target="_blank" class="dropdown-item">
            How to Cite Galaxy
        </a></li><li role="presentation"><a role="menuitem" title="" href="/tours" target="_parent" class="dropdown-item">
            Interactive Tours
        </a></li><li role="presentation"><a role="menuitem" title="" href="/welcome/new" target="_parent" class="dropdown-item">
            Introduction to Galaxy
        </a></li><li role="presentation"><a role="menuitem" title="" href="/about" target="_parent" class="dropdown-item">
            About
        </a></li><li role="presentation"><a role="menuitem" title="" href="https://usegalaxy.eu/terms" rel="noopener" target="_blank" class="dropdown-item">
            Terms and Conditions
        </a></li></ul></li><li data-v-e2d015b2="" id="user" title="Log in or register a new account" class="nav-item loggedout-only"><a href="/login" target="_top" class="nav-link">
        Log in or Register
    </a></li> <li data-v-e2d015b2="" id="interactive" title="See Running Interactive Tools" class="nav-item" style="display: none;"><a href="/interactivetool_entry_points/list" target="_parent" class="nav-link nav-icon"><span class="sr-only">See Running Interactive Tools</span> <span class="fa fa-fw fa-cogs"></span> <!----></a></li> <li data-v-e2d015b2="" id="gtn" title="See Galaxy Training Materials" class="nav-item"><a href="#" target="_parent" class="nav-link nav-icon"><span class="sr-only">See Galaxy Training Materials</span> <span class="fa fa-fw fa-graduation-cap"></span> <!----></a></li><li data-v-e2d015b2="" id="news" title="See the Galaxy Release Notes" class="nav-item"><a href="#" target="_parent" class="nav-link nav-icon"><span class="sr-only">See the Galaxy Release Notes</span> <span class="fa fa-fw fa-bell"></span> <!----></a></li> <li data-v-e2d015b2="" id="enable-window-manager" title="Enable/Disable Window Manager" class="nav-item"><a href="#" target="_parent" class="nav-link nav-icon"><span class="sr-only">Enable/Disable Window Manager</span> <span class="fa fa-fw fa-th"></span> <!----></a></li> <!----></ul> <div data-v-47889573="" data-v-e2d015b2=""><div data-v-47889573="" class="quota-meter d-flex align-items-center"><a data-v-47889573="" title="Using 0 b. Log in for details." data-description="storage dashboard link" href="/storage" target="_self" tabindex="-1" aria-disabled="true" class="quota-progress disabled"><div data-v-47889573="" class="progress"><div data-v-47889573="" role="progressbar" aria-valuemin="0" aria-valuemax="100" aria-valuenow="0" class="progress-bar bg-success" aria-label="Quota usage" style="width: 0%;"></div></div> <span data-v-47889573="">Using 0%</span></a></div></div></nav> <!----> <!----> <div id="columns" class="d-flex"><!----> <div id="left" class="d-flex flex-panel-left"><div class="d-flex flex-column" style="width: 300px;"><div data-v-9db6ef8c="" aria-labelledby="toolbox-heading" class="unified-panel d-flex flex-column"><div data-v-9db6ef8c="" unselectable="on"><div data-v-9db6ef8c="" class="unified-panel-header-inner"><nav data-v-9db6ef8c="" class="d-flex justify-content-between mx-3 my-2"><h2 data-v-9db6ef8c="" id="toolbox-heading" class="m-1 h-sm">Tools</h2> <div data-v-9db6ef8c="" class="panel-header-buttons"><div data-v-9db6ef8c="" role="group" class="btn-group"><button data-v-9db6ef8c="" aria-label="Show favorite tools" title="Log in to Favorite Tools" type="button" disabled="disabled" class="btn panel-header-button-toolbox btn-link btn-sm disabled"><svg aria-hidden="true" focusable="false" data-prefix="far" data-icon="star" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512" class="svg-inline--fa fa-star"><path fill="currentColor" d="M528.1 171.5L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6zM388.6 312.3l23.7 138.4L288 385.4l-124.3 65.3 23.7-138.4-100.6-98 139-20.2 62.2-126 62.2 126 139 20.2-100.6 98z" class=""></path></svg></button> <div data-v-9db6ef8c="" class="dropdown b-dropdown tool-panel-dropdown btn-group" title="Show panel options" aria-label="View all tool panel configurations" id="__BVID__104"><!----><button aria-haspopup="menu" aria-expanded="false" type="button" class="btn dropdown-toggle btn-link btn-sm" id="__BVID__104__BV_toggle_"><span class="sr-only">View all tool panel configurations</span></button><ul role="menu" tabindex="-1" class="dropdown-menu dropdown-menu-right" aria-labelledby="__BVID__104__BV_toggle_"> <li role="presentation"><a data-panel-id="default" role="menuitem" href="#" target="_self" class="dropdown-item active"><span fixed-width="" class="fa fa-check"></span> <span title="Galaxy's fully configured toolbox panel with all sections, tools, and configured workflows loaded." class="ml-1">Full Tool Panel</span></a></li> <li role="presentation"><header id="_bv_ontology_group_dd_header" class="dropdown-header"><small class="font-weight-bold">...by Ontology</small></header><ul id="ontology" role="group" aria-describedby="_bv_ontology_group_dd_header" class="list-unstyled"> <li role="presentation"><a data-panel-id="ontology:edam_operations" role="menuitem" href="#" target="_self" class="dropdown-item"><span fixed-width="" class="fa fa-sitemap"></span> <span title="Tools are grouped using annotated EDAM operation information (if available)." class="ml-1">EDAM Operations</span></a></li><li role="presentation"><a data-panel-id="ontology:edam_topics" role="menuitem" href="#" target="_self" class="dropdown-item"><span fixed-width="" class="fa fa-sitemap"></span> <span title="Tools are grouped using annotated EDAM topic information (if available)." class="ml-1">EDAM Topics</span></a></li></ul></li> <!----> </ul></div></div></div></nav></div></div> <div data-v-9db6ef8c="" class="unified-panel-controls"><div data-v-9db6ef8c=""><!----> <div role="group" class="input-group mb-3"><!----><input type="text" placeholder="search tools" autocomplete="off" class="search-query form-control form-control-sm" id="__BVID__117"> <div class="input-group-append"><button aria-haspopup="true" title="Toggle Advanced Search" data-description="toggle advanced search" type="button" aria-pressed="false" autocomplete="off" class="btn btn-secondary btn-sm"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="angle-double-down" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-angle-double-down fa-fw"><path fill="currentColor" d="M143 256.3L7 120.3c-9.4-9.4-9.4-24.6 0-33.9l22.6-22.6c9.4-9.4 24.6-9.4 33.9 0l96.4 96.4 96.4-96.4c9.4-9.4 24.6-9.4 33.9 0L313 86.3c9.4 9.4 9.4 24.6 0 33.9l-136 136c-9.4 9.5-24.6 9.5-34 .1zm34 192l136-136c9.4-9.4 9.4-24.6 0-33.9l-22.6-22.6c-9.4-9.4-24.6-9.4-33.9 0L160 352.1l-96.4-96.4c-9.4-9.4-24.6-9.4-33.9 0L7 278.3c-9.4 9.4-9.4 24.6 0 33.9l136 136c9.4 9.5 24.6 9.5 34 .1z" class=""></path></svg></button> <button aria-haspopup="true" title="Clear Search (esc)" data-description="reset query" type="button" class="btn search-clear btn-secondary btn-sm"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="times" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 352 512" class="svg-inline--fa fa-times fa-fw"><path fill="currentColor" d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z" class=""></path></svg></button></div><!----></div> <!----></div> <section data-v-9db6ef8c=""><button data-v-9db6ef8c="" id="activity-upload" aria-label="Download from URL or upload files from disk" title="Download from URL or upload files from disk" type="button" class="btn upload-button btn-secondary btn-sm"><div class="progress"><div class="progress-bar progress-bar-notransition progress-bar-" style="width: 0%;"></div></div> <span class="position-relative"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="upload" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="mr-1 svg-inline--fa fa-upload"><path fill="currentColor" d="M296 384h-80c-13.3 0-24-10.7-24-24V192h-87.7c-17.8 0-26.7-21.5-14.1-34.1L242.3 5.7c7.5-7.5 19.8-7.5 27.3 0l152.2 152.2c12.6 12.6 3.7 34.1-14.1 34.1H320v168c0 13.3-10.7 24-24 24zm216-8v112c0 13.3-10.7 24-24 24H24c-13.3 0-24-10.7-24-24V376c0-13.3 10.7-24 24-24h136v8c0 30.9 25.1 56 56 56h80c30.9 0 56-25.1 56-56v-8h136c13.3 0 24 10.7 24 24zm-124 88c0-11-9-20-20-20s-20 9-20 20 9 20 20 20 20-9 20-20zm64 0c0-11-9-20-20-20s-20 9-20 20 9 20 20 20 20-9 20-20z" class=""></path></svg> <b>Upload Data</b></span></button> <!----> <!----></section></div> <div data-v-9db6ef8c="" class="unified-panel-body"><div data-v-9db6ef8c="" class="toolMenuContainer"><div data-v-9db6ef8c="" class="toolMenu"><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Get Data
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Send Data
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Collection Operations
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c=""><div data-v-9b7407e4="" data-v-45afbd0d="" tabindex="0" class="tool-panel-label">
    General Text Tools
    <!----></div></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Text Manipulation
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Convert Formats
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Filter and Sort
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Join, Subtract and Group
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c=""><div data-v-9b7407e4="" data-v-45afbd0d="" tabindex="0" class="tool-panel-label">
    Genomic File Manipulation
    <!----></div></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Convert Formats
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                FASTA/FASTQ
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Quality Control
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                SAM/BAM
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                BED
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                VCF/BCF
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Nanopore
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c=""><div data-v-9b7407e4="" data-v-45afbd0d="" tabindex="0" class="tool-panel-label">
    Common Genomics Tools
    <!----></div></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Operate on Genomic Intervals
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Fetch Sequences / Alignments
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c=""><div data-v-9b7407e4="" data-v-45afbd0d="" tabindex="0" class="tool-panel-label">
    Genomics Analysis
    <!----></div></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Annotation
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Multiple Alignments
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Assembly
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Mapping
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Variant Calling
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Genome editing
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                RNA-Seq
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                RNA Analysis
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Peak Calling
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Epigenetics
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Phylogenetics
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Phenotype Association
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Single-cell
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Spatial Omics
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c=""><div data-v-9b7407e4="" data-v-45afbd0d="" tabindex="0" class="tool-panel-label">
    Genomics Toolkits
    <!----></div></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Picard
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                deepTools
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Gemini
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                EMBOSS
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                NCBI Blast
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                HiCExplorer
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                RAD-seq
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                GraphClust
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                MiModD
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c=""><div data-v-9b7407e4="" data-v-45afbd0d="" tabindex="0" class="tool-panel-label">
    Metagenomics
    <!----></div></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Metagenomic Analysis
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Qiime
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Mothur
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                DNA Metabarcoding
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Data and Metadata Management
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Virology
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c=""><div data-v-9b7407e4="" data-v-45afbd0d="" tabindex="0" class="tool-panel-label">
    Proteomics, Metabolomics, Chemistry
    <!----></div></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Proteomics
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Metabolomics
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                ChemicalToolBox
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c=""><div data-v-9b7407e4="" data-v-45afbd0d="" tabindex="0" class="tool-panel-label">
    Statistics and Visualisation
    <!----></div></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Statistics
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Machine Learning
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Graph/Display Data
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Muon Spectroscopy
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c=""><div data-v-9b7407e4="" data-v-45afbd0d="" tabindex="0" class="tool-panel-label">
    Miscellaneous Tools
    <!----></div></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Evolution
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Motif Tools
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Test Tools
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                GIS Data Handling
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Animal Detection on Acoustic Recordings
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Compute indicators for satellite remote sensing
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Regional Variation
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Genome Diversity
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Climate Analysis
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Apollo
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Imaging
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Species abundance
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Compute indicators for turnover boulders fields
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Ecoregionalization
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Astronomy
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Interactive tools
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                OBO Ontology manipulation
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                HCA-Single Cell
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Other Tools
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Biodiversity data exploration
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Sanger Sequencing
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Extract Features
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                QIIME 2
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                Metagenomic Analysis
            </span> <!----></a></div> <!----></div><div data-v-45afbd0d="" data-v-9db6ef8c="" class="tool-panel-section"><div data-v-45afbd0d="" class="toolSectionTitle tool-menu-section-default"><a data-v-45afbd0d="" href="javascript:void(0)" class="title-link"><span data-v-45afbd0d="" class="name">
                GATK Tools
            </span> <!----></a></div> <!----></div></div> <div data-v-45afbd0d="" data-v-9db6ef8c=""><div data-v-9b7407e4="" data-v-45afbd0d="" tabindex="0" class="tool-panel-label">
    Workflows
    <!----></div></div> <div data-v-9db6ef8c="" id="internal-workflows" class="toolSectionBody"><div data-v-9db6ef8c="" class="toolSectionBg"></div> <div data-v-9db6ef8c="" class="toolTitle"><a data-v-9db6ef8c="" href="/workflows/list" class="title-link">All workflows</a></div></div></div></div></div> <div class="flex-panel-footer d-flex px-2 py-1"><div class="cursor-pointer"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="chevron-left" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-chevron-left"><path fill="currentColor" d="M34.52 239.03L228.87 44.69c9.37-9.37 24.57-9.37 33.94 0l22.67 22.67c9.36 9.36 9.37 24.52.04 33.9L131.49 256l154.02 154.75c9.34 9.38 9.32 24.54-.04 33.9l-22.67 22.67c-9.37 9.37-24.57 9.37-33.94 0L34.52 272.97c-9.37-9.37-9.37-24.57 0-33.94z" class=""></path></svg> <div id="left-collapse" class="interaction-area"></div></div> <div class="flex-fill"></div> <div id="left-drag"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="grip-lines-vertical" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 512" class="svg-inline--fa fa-grip-lines-vertical"><path fill="currentColor" d="M96 496V16c0-8.8-7.2-16-16-16H48c-8.8 0-16 7.2-16 16v480c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16zm128 0V16c0-8.8-7.2-16-16-16h-32c-8.8 0-16 7.2-16 16v480c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16z" class=""></path></svg> <div class="interaction-area cursor-grab"></div></div></div></div> <!----></div> <div id="center" class="overflow-auto p-3 w-100"><iframe id="galaxy_main" name="galaxy_main" src="" frameborder="0" title="galaxy frame" width="100%" height="100%" class="center-frame" style="display: none;"></iframe> <div class="h-100"><div tool_id="toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.74+galaxy0"><!----> <!----> <!----> <!----> <!----> <div data-v-6d7325b0="" class="position-relative" itemscope="itemscope" itemtype="https://schema.org/CreativeWork"><div data-v-6d7325b0="" class="underlay sticky-top"></div> <div data-v-6d7325b0="" class="tool-header sticky-top bg-secondary px-2 py-1 rounded"><div data-v-6d7325b0="" class="d-flex justify-content-between"><div data-v-6d7325b0="" class="py-1 d-flex flex-wrap flex-gapx-1"><span data-v-6d7325b0=""><svg data-v-6d7325b0="" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="wrench" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="fa-fw svg-inline--fa fa-wrench"><path data-v-6d7325b0="" fill="currentColor" d="M507.73 109.1c-2.24-9.03-13.54-12.09-20.12-5.51l-74.36 74.36-67.88-11.31-11.31-67.88 74.36-74.36c6.62-6.62 3.43-17.9-5.66-20.16-47.38-11.74-99.55.91-136.58 37.93-39.64 39.64-50.55 97.1-34.05 147.2L18.74 402.76c-24.99 24.99-24.99 65.51 0 90.5 24.99 24.99 65.51 24.99 90.5 0l213.21-213.21c50.12 16.71 107.47 5.68 147.37-34.22 37.07-37.07 49.7-89.32 37.91-136.73zM64 472c-13.25 0-24-10.75-24-24 0-13.26 10.75-24 24-24s24 10.74 24 24c0 13.25-10.75 24-24 24z" class=""></path></svg> <h1 data-v-b01e12ae="" data-v-6d7325b0="" class="heading h-text font-weight-bold inline" itemprop="name"><!----> FastQC</h1></span> <span data-v-6d7325b0="" itemprop="description">Read Quality reports</span> <span data-v-6d7325b0="">(Galaxy Version 0.74+galaxy0)</span></div> <div data-v-6d7325b0="" class="d-flex flex-nowrap align-items-start flex-gapx-1"><div data-v-6d7325b0="" role="group" class="tool-card-buttons btn-group"><!----> <div data-v-6d7325b0="" class="dropdown b-dropdown tool-versions btn-group" title="Versions" aria-label="Select Versions" id="__BVID__140"><!----><button aria-haspopup="false" aria-expanded="false" type="button" class="btn dropdown-toggle btn-link btn-sm dropdown-toggle-no-caret" id="__BVID__140__BV_toggle_"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="cubes" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-cubes"><path fill="currentColor" d="M488.6 250.2L392 214V105.5c0-15-9.3-28.4-23.4-33.7l-100-37.5c-8.1-3.1-17.1-3.1-25.3 0l-100 37.5c-14.1 5.3-23.4 18.7-23.4 33.7V214l-96.6 36.2C9.3 255.5 0 268.9 0 283.9V394c0 13.6 7.7 26.1 19.9 32.2l100 50c10.1 5.1 22.1 5.1 32.2 0l103.9-52 103.9 52c10.1 5.1 22.1 5.1 32.2 0l100-50c12.2-6.1 19.9-18.6 19.9-32.2V283.9c0-15-9.3-28.4-23.4-33.7zM358 214.8l-85 31.9v-68.2l85-37v73.3zM154 104.1l102-38.2 102 38.2v.6l-102 41.4-102-41.4v-.6zm84 291.1l-85 42.5v-79.1l85-38.8v75.4zm0-112l-102 41.4-102-41.4v-.6l102-38.2 102 38.2v.6zm240 112l-85 42.5v-79.1l85-38.8v75.4zm0-112l-102 41.4-102-41.4v-.6l102-38.2 102 38.2v.6z" class=""></path></svg></button><ul role="button" tabindex="-1" class="dropdown-menu dropdown-menu-right" aria-labelledby="__BVID__140__BV_toggle_"> <li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item active"><span><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="check" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-check"><path fill="currentColor" d="M173.898 439.404l-166.4-166.4c-9.997-9.997-9.997-26.206 0-36.204l36.203-36.204c9.997-9.998 26.207-9.998 36.204 0L192 312.69 432.095 72.596c9.997-9.997 26.207-9.997 36.204 0l36.203 36.204c9.997 9.997 9.997 26.206 0 36.204l-294.4 294.401c-9.998 9.997-26.207 9.997-36.204-.001z" class=""></path></svg> <span>Selected</span> 0.74+galaxy0 </span></a></li><li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item"><span><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="cube" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-cube"><path fill="currentColor" d="M239.1 6.3l-208 78c-18.7 7-31.1 25-31.1 45v225.1c0 18.2 10.3 34.8 26.5 42.9l208 104c13.5 6.8 29.4 6.8 42.9 0l208-104c16.3-8.1 26.5-24.8 26.5-42.9V129.3c0-20-12.4-37.9-31.1-44.9l-208-78C262 2.2 250 2.2 239.1 6.3zM256 68.4l192 72v1.1l-192 78-192-78v-1.1l192-72zm32 356V275.5l160-65v133.9l-160 80z" class=""></path></svg> <span>Switch to</span> 0.73+galaxy0
        </span></a></li><li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item"><span><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="cube" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-cube"><path fill="currentColor" d="M239.1 6.3l-208 78c-18.7 7-31.1 25-31.1 45v225.1c0 18.2 10.3 34.8 26.5 42.9l208 104c13.5 6.8 29.4 6.8 42.9 0l208-104c16.3-8.1 26.5-24.8 26.5-42.9V129.3c0-20-12.4-37.9-31.1-44.9l-208-78C262 2.2 250 2.2 239.1 6.3zM256 68.4l192 72v1.1l-192 78-192-78v-1.1l192-72zm32 356V275.5l160-65v133.9l-160 80z" class=""></path></svg> <span>Switch to</span> 0.72+galaxy1
        </span></a></li><li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item"><span><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="cube" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-cube"><path fill="currentColor" d="M239.1 6.3l-208 78c-18.7 7-31.1 25-31.1 45v225.1c0 18.2 10.3 34.8 26.5 42.9l208 104c13.5 6.8 29.4 6.8 42.9 0l208-104c16.3-8.1 26.5-24.8 26.5-42.9V129.3c0-20-12.4-37.9-31.1-44.9l-208-78C262 2.2 250 2.2 239.1 6.3zM256 68.4l192 72v1.1l-192 78-192-78v-1.1l192-72zm32 356V275.5l160-65v133.9l-160 80z" class=""></path></svg> <span>Switch to</span> 0.72
        </span></a></li><li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item"><span><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="cube" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-cube"><path fill="currentColor" d="M239.1 6.3l-208 78c-18.7 7-31.1 25-31.1 45v225.1c0 18.2 10.3 34.8 26.5 42.9l208 104c13.5 6.8 29.4 6.8 42.9 0l208-104c16.3-8.1 26.5-24.8 26.5-42.9V129.3c0-20-12.4-37.9-31.1-44.9l-208-78C262 2.2 250 2.2 239.1 6.3zM256 68.4l192 72v1.1l-192 78-192-78v-1.1l192-72zm32 356V275.5l160-65v133.9l-160 80z" class=""></path></svg> <span>Switch to</span> 0.71
        </span></a></li><li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item"><span><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="cube" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-cube"><path fill="currentColor" d="M239.1 6.3l-208 78c-18.7 7-31.1 25-31.1 45v225.1c0 18.2 10.3 34.8 26.5 42.9l208 104c13.5 6.8 29.4 6.8 42.9 0l208-104c16.3-8.1 26.5-24.8 26.5-42.9V129.3c0-20-12.4-37.9-31.1-44.9l-208-78C262 2.2 250 2.2 239.1 6.3zM256 68.4l192 72v1.1l-192 78-192-78v-1.1l192-72zm32 356V275.5l160-65v133.9l-160 80z" class=""></path></svg> <span>Switch to</span> 0.69
        </span></a></li><li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item"><span><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="cube" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-cube"><path fill="currentColor" d="M239.1 6.3l-208 78c-18.7 7-31.1 25-31.1 45v225.1c0 18.2 10.3 34.8 26.5 42.9l208 104c13.5 6.8 29.4 6.8 42.9 0l208-104c16.3-8.1 26.5-24.8 26.5-42.9V129.3c0-20-12.4-37.9-31.1-44.9l-208-78C262 2.2 250 2.2 239.1 6.3zM256 68.4l192 72v1.1l-192 78-192-78v-1.1l192-72zm32 356V275.5l160-65v133.9l-160 80z" class=""></path></svg> <span>Switch to</span> 0.68
        </span></a></li><li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item"><span><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="cube" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-cube"><path fill="currentColor" d="M239.1 6.3l-208 78c-18.7 7-31.1 25-31.1 45v225.1c0 18.2 10.3 34.8 26.5 42.9l208 104c13.5 6.8 29.4 6.8 42.9 0l208-104c16.3-8.1 26.5-24.8 26.5-42.9V129.3c0-20-12.4-37.9-31.1-44.9l-208-78C262 2.2 250 2.2 239.1 6.3zM256 68.4l192 72v1.1l-192 78-192-78v-1.1l192-72zm32 356V275.5l160-65v133.9l-160 80z" class=""></path></svg> <span>Switch to</span> 0.67
        </span></a></li><li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item"><span><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="cube" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-cube"><path fill="currentColor" d="M239.1 6.3l-208 78c-18.7 7-31.1 25-31.1 45v225.1c0 18.2 10.3 34.8 26.5 42.9l208 104c13.5 6.8 29.4 6.8 42.9 0l208-104c16.3-8.1 26.5-24.8 26.5-42.9V129.3c0-20-12.4-37.9-31.1-44.9l-208-78C262 2.2 250 2.2 239.1 6.3zM256 68.4l192 72v1.1l-192 78-192-78v-1.1l192-72zm32 356V275.5l160-65v133.9l-160 80z" class=""></path></svg> <span>Switch to</span> 0.65
        </span></a></li><li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item"><span><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="cube" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-cube"><path fill="currentColor" d="M239.1 6.3l-208 78c-18.7 7-31.1 25-31.1 45v225.1c0 18.2 10.3 34.8 26.5 42.9l208 104c13.5 6.8 29.4 6.8 42.9 0l208-104c16.3-8.1 26.5-24.8 26.5-42.9V129.3c0-20-12.4-37.9-31.1-44.9l-208-78C262 2.2 250 2.2 239.1 6.3zM256 68.4l192 72v1.1l-192 78-192-78v-1.1l192-72zm32 356V275.5l160-65v133.9l-160 80z" class=""></path></svg> <span>Switch to</span> 0.64
        </span></a></li><li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item"><span><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="cube" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-cube"><path fill="currentColor" d="M239.1 6.3l-208 78c-18.7 7-31.1 25-31.1 45v225.1c0 18.2 10.3 34.8 26.5 42.9l208 104c13.5 6.8 29.4 6.8 42.9 0l208-104c16.3-8.1 26.5-24.8 26.5-42.9V129.3c0-20-12.4-37.9-31.1-44.9l-208-78C262 2.2 250 2.2 239.1 6.3zM256 68.4l192 72v1.1l-192 78-192-78v-1.1l192-72zm32 356V275.5l160-65v133.9l-160 80z" class=""></path></svg> <span>Switch to</span> 0.63
        </span></a></li><li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item"><span><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="cube" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-cube"><path fill="currentColor" d="M239.1 6.3l-208 78c-18.7 7-31.1 25-31.1 45v225.1c0 18.2 10.3 34.8 26.5 42.9l208 104c13.5 6.8 29.4 6.8 42.9 0l208-104c16.3-8.1 26.5-24.8 26.5-42.9V129.3c0-20-12.4-37.9-31.1-44.9l-208-78C262 2.2 250 2.2 239.1 6.3zM256 68.4l192 72v1.1l-192 78-192-78v-1.1l192-72zm32 356V275.5l160-65v133.9l-160 80z" class=""></path></svg> <span>Switch to</span> 0.62
        </span></a></li></ul></div> <div data-v-6d7325b0="" class="dropdown b-dropdown tool-dropdown btn-group" title="" aria-label="View all Options" id="__BVID__167" data-original-title="Options"><!----><button aria-haspopup="false" aria-expanded="false" type="button" class="btn dropdown-toggle btn-link btn-sm dropdown-toggle-no-caret" id="__BVID__167__BV_toggle_"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="caret-down" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-caret-down"><path fill="currentColor" d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z" class=""></path></svg></button><ul role="button" tabindex="-1" class="dropdown-menu dropdown-menu-right" aria-labelledby="__BVID__167__BV_toggle_" style=""> <li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="link" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-link"><path fill="currentColor" d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z" class=""></path></svg><span>Copy Link</span></a></li> <li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item"><svg aria-hidden="true" focusable="false" data-prefix="far" data-icon="copy" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="svg-inline--fa fa-copy"><path fill="currentColor" d="M433.941 65.941l-51.882-51.882A48 48 0 0 0 348.118 0H176c-26.51 0-48 21.49-48 48v48H48c-26.51 0-48 21.49-48 48v320c0 26.51 21.49 48 48 48h224c26.51 0 48-21.49 48-48v-48h80c26.51 0 48-21.49 48-48V99.882a48 48 0 0 0-14.059-33.941zM266 464H54a6 6 0 0 1-6-6V150a6 6 0 0 1 6-6h74v224c0 26.51 21.49 48 48 48h96v42a6 6 0 0 1-6 6zm128-96H182a6 6 0 0 1-6-6V54a6 6 0 0 1 6-6h106v88c0 13.255 10.745 24 24 24h88v202a6 6 0 0 1-6 6zm6-256h-64V48h9.632c1.591 0 3.117.632 4.243 1.757l48.368 48.368a6 6 0 0 1 1.757 4.243V112z" class=""></path></svg><span>Copy Tool ID</span></a></li> <!----> <div><li role="presentation" tabindex="0"><a role="menuitem" href="#" target="_self" class="dropdown-item"><svg aria-hidden="true" focusable="false" data-prefix="far" data-icon="eye" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512" class="svg-inline--fa fa-eye"><path fill="currentColor" d="M288 144a110.94 110.94 0 0 0-31.24 5 55.4 55.4 0 0 1 7.24 27 56 56 0 0 1-56 56 55.4 55.4 0 0 1-27-7.24A111.71 111.71 0 1 0 288 144zm284.52 97.4C518.29 135.59 410.93 64 288 64S57.68 135.64 3.48 241.41a32.35 32.35 0 0 0 0 29.19C57.71 376.41 165.07 448 288 448s230.32-71.64 284.52-177.41a32.35 32.35 0 0 0 0-29.19zM288 400c-98.65 0-189.09-55-237.93-144C98.91 167 189.34 112 288 112s189.09 55 237.93 144C477.1 345 386.66 400 288 400z" class=""></path></svg><span>View Tool source</span></a></li> <!----></div> <li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg><span>See in Tool Shed</span></a></li> <li role="presentation"><a role="menuitem" href="#" target="_self" class="dropdown-item"><span class="fa fa-puzzle-piece"></span>Generate Tour
    </a></li></ul></div> <!----> <!----> <!----></div> <button title="Run tool: FastQC (0.74+galaxy0)" type="button" class="btn d-flex flex-nowrap align-items-center text-nowrap btn-sm btn-primary" id="execute" data-v-6d7325b0=""><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="play" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="mr-2 svg-inline--fa fa-play"><path fill="currentColor" d="M424.4 214.7L72.4 6.6C43.8-10.3 0 6.1 0 47.9V464c0 37.5 40.7 60.1 72.4 41.3l352-208c31.4-18.5 31.5-64.1 0-82.6z" class=""></path></svg>Run Tool
</button></div></div></div> <div data-v-6d7325b0="" id="tool-card-body"><!----> <!----> <div data-v-6d7325b0="" class="mt-2 mb-4"><div data-v-b01e12ae="" class="separator heading" data-v-6d7325b0=""><div data-v-b01e12ae="" class="stripe"></div> <h2 data-v-b01e12ae="" class="h-sm font-weight-bold"> Tool Parameters </h2> <div data-v-b01e12ae="" class="stripe"></div></div> <div data-v-6d7325b0=""><div><div data-v-af72d116="" id="form-element-input_file" class="ui-form-element section-row alert alert-info"><div data-v-af72d116="" class="ui-form-error"><svg data-v-af72d116="" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="exclamation" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 192 512" class="mr-1 svg-inline--fa fa-exclamation"><path data-v-af72d116="" fill="currentColor" d="M176 432c0 44.112-35.888 80-80 80s-80-35.888-80-80 35.888-80 80-80 80 35.888 80 80zM25.26 25.199l13.6 272C39.499 309.972 50.041 320 62.83 320h66.34c12.789 0 23.331-10.028 23.97-22.801l13.6-272C167.425 11.49 156.496 0 142.77 0H49.23C35.504 0 24.575 11.49 25.26 25.199z" class=""></path></svg> <span data-v-af72d116="" class="ui-form-error-text">Please provide a value for this option.</span></div> <div data-v-af72d116="" class="ui-form-title"><span data-v-af72d116="" class="ui-form-title-text"><label data-v-af72d116="" for="input_file">Raw read data from your current history</label></span> <span data-v-af72d116="" title="required" class="ui-form-title-star warning">
            *
            <span data-v-af72d116="" class="ui-form-title-message warning"> required </span></span></div> <div data-v-af72d116="" data-label="Raw read data from your current history" class="ui-form-field"><div data-v-af72d116="" data-label="Raw read data from your current history"><div class="ui-select-content d-flex flex-row flex-wrap flex-lg-nowrap"><div><div class="ui-options pr-lg-2" style="display: block;"><div class="mt-2" style="display: none;"></div><div class="ui-options-menu" style="display: block;"></div><div class="ui-radiobutton" style="display: inline-block;"><label role="button" class="ui-option active" data-original-title="" title=""><i class="fa fa-file-o no-padding"></i><input type="radio" name="uid-5" value="0" style="display: none;"></label><label role="button" class="ui-option" data-original-title="" title=""><i class="fa fa-files-o no-padding"></i><input type="radio" name="uid-5" value="1" style="display: none;"></label><label role="button" class="ui-option" data-original-title="" title=""><i class="fa fa-folder-o no-padding"></i><input type="radio" name="uid-5" value="2" style="display: none;"></label></div></div></div><div class="overflow-auto w-100 py-2 py-lg-0 pr-lg-2"><div class="ui-select" id="uid-2" style="display: block;"><div class="select2-container select select2-container-disabled" id="s2id_uid-2_select" title="" style="display: block;"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-3">No fastq, fastq.gz, fastq.bz2, bam or sam dataset available.</span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen3" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-3" id="s2id_autogen3" disabled=""><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen3_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-3" id="s2id_autogen3_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-3">   </ul></div></div><div class="select select2-offscreen" id="uid-2_select" title="" style="display: block;" tabindex="-1"></div><div class="icon-dropdown fa fa-caret-down" style="display: block;"></div></div><div class="ui-select-multiple ui-select" id="uid-3" style="display: none;"><select class="select" id="uid-3_select" multiple="" disabled="" style="display: inline-block;"><option value="__null__">No fastq, fastq.gz, fastq.bz2, bam or sam dataset available.</option></select><div class="icon-dropdown fa fa-caret-down" style="display: none;"></div><div class="icon-resize" style="display: block;"><i class="fa fa-angle-double-right fa-rotate-45" style="display: inline;"></i></div></div><div class="ui-select" id="uid-4" style="display: none;"><div class="select2-container select select2-container-disabled" id="s2id_uid-4_select" title="" style="display: block;"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-4">No fastq, fastq.gz, fastq.bz2, bam or sam dataset collection available.</span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen4" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-4" id="s2id_autogen4" disabled=""><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen4_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-4" id="s2id_autogen4_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-4">   </ul></div></div><div class="select select2-offscreen" id="uid-4_select" title="" style="display: block;" tabindex="-1"></div><div class="icon-dropdown fa fa-caret-down" style="display: block;"></div></div><div class="form-text text-muted" style="clear: both; display: none;">
                    <i class="fa fa-sitemap"></i>
                    <span>
                        This is a batch mode input field. Separate jobs will be triggered for each dataset selection.
                    </span>
                </div><div class="form-text text-muted" style="clear: both; display: none;">
                    <i class="fa fa-sitemap"></i>
                    <span>
                        This is a batch mode input field. Separate jobs will be triggered for each dataset selection.
                    </span>
                <div class="ui-options" style="display: block;"><div class="mt-2" style="display: none;"></div><div class="ui-options-menu" style="display: block;"></div><div class="ui-radiobutton" style="display: inline-block;"><label role="button" class="ui-option active" data-original-title="" title=""><i class="fa fa-chain no-padding"></i><input type="radio" name="uid-1" value="false" style="display: none;"></label><label role="button" class="ui-option" data-original-title="" title=""><i class="fa fa-chain-broken no-padding"></i><input type="radio" name="uid-1" value="true" style="display: none;"></label></div></div></div></div><div style="margin-right: 5px;"><button type="button" class="ui-button-default btn btn-secondary" id="uid-7" style="display: inline-block;" data-original-title="" title=""><i class="icon fa fa-upload"></i><span class="title"></span><div class="progress" style="display: none;"><div class="progress-bar" style="width: 0%;"></div></div></button></div><div><button type="button" class="ui-button-default btn btn-secondary" id="uid-6" style="display: inline-block;" data-original-title="" title=""><i class="icon fa fa-folder-open-o"></i><span class="title"></span><div class="progress" style="display: none;"><div class="progress-bar" style="width: 0%;"></div></div></button></div></div></div></div> <!----> <!----></div></div><div><div data-v-af72d116="" id="form-element-contaminants" class="ui-form-element section-row"><!----> <div data-v-af72d116="" class="ui-form-title"><span data-v-af72d116="" class="ui-form-title-text"><label data-v-af72d116="" for="contaminants">Contaminant list</label></span> <span data-v-af72d116="" class="ui-form-title-message">
            - optional
        </span></div> <div data-v-af72d116="" data-label="Contaminant list" class="ui-form-field"><div data-v-af72d116="" data-label="Contaminant list"><div class="ui-select-content d-flex flex-row flex-wrap flex-lg-nowrap"><div><div class="ui-options pr-lg-2" style="display: block;"><div class="mt-2" style="display: none;"></div><div class="ui-options-menu" style="display: block;"></div><div class="ui-radiobutton" style="display: inline-block;"><label role="button" class="ui-option active" data-original-title="" title=""><i class="fa fa-file-o no-padding"></i><input type="radio" name="uid-12" value="0" style="display: none;"></label><label role="button" class="ui-option" data-original-title="" title=""><i class="fa fa-files-o no-padding"></i><input type="radio" name="uid-12" value="1" style="display: none;"></label><label role="button" class="ui-option" data-original-title="" title=""><i class="fa fa-folder-o no-padding"></i><input type="radio" name="uid-12" value="2" style="display: none;"></label></div></div></div><div class="overflow-auto w-100 py-2 py-lg-0 pr-lg-2"><div class="ui-select" id="uid-9" style="display: block;"><div class="select2-container select select2-container-disabled" id="s2id_uid-9_select" title="" style="display: block;"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-7">No tabular dataset available.</span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen7" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-7" id="s2id_autogen7" disabled=""><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen7_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-7" id="s2id_autogen7_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-7">   </ul></div></div><div class="select select2-offscreen" id="uid-9_select" title="" style="display: block;" tabindex="-1"></div><div class="icon-dropdown fa fa-caret-down" style="display: block;"></div></div><div class="ui-select-multiple ui-select" id="uid-10" style="display: none;"><select class="select" id="uid-10_select" multiple="" disabled="" style="display: inline-block;"><option value="__null__">No tabular dataset available.</option></select><div class="icon-dropdown fa fa-caret-down" style="display: none;"></div><div class="icon-resize" style="display: block;"><i class="fa fa-angle-double-right fa-rotate-45" style="display: inline;"></i></div></div><div class="ui-select" id="uid-11" style="display: none;"><div class="select2-container select select2-container-disabled" id="s2id_uid-11_select" title="" style="display: block;"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-8">No tabular dataset collection available.</span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen8" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-8" id="s2id_autogen8" disabled=""><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen8_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-8" id="s2id_autogen8_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-8">   </ul></div></div><div class="select select2-offscreen" id="uid-11_select" title="" style="display: block;" tabindex="-1"></div><div class="icon-dropdown fa fa-caret-down" style="display: block;"></div></div><div class="form-text text-muted" style="clear: both; display: none;">
                    <i class="fa fa-sitemap"></i>
                    <span>
                        This is a batch mode input field. Separate jobs will be triggered for each dataset selection.
                    </span>
                </div><div class="form-text text-muted" style="clear: both; display: none;">
                    <i class="fa fa-sitemap"></i>
                    <span>
                        This is a batch mode input field. Separate jobs will be triggered for each dataset selection.
                    </span>
                <div class="ui-options" style="display: block;"><div class="mt-2" style="display: none;"></div><div class="ui-options-menu" style="display: block;"></div><div class="ui-radiobutton" style="display: inline-block;"><label role="button" class="ui-option active" data-original-title="" title=""><i class="fa fa-chain no-padding"></i><input type="radio" name="uid-8" value="false" style="display: none;"></label><label role="button" class="ui-option" data-original-title="" title=""><i class="fa fa-chain-broken no-padding"></i><input type="radio" name="uid-8" value="true" style="display: none;"></label></div></div></div></div><div style="margin-right: 5px;"><button type="button" class="ui-button-default btn btn-secondary" id="uid-14" style="display: inline-block;" data-original-title="" title=""><i class="icon fa fa-upload"></i><span class="title"></span><div class="progress" style="display: none;"><div class="progress-bar" style="width: 0%;"></div></div></button></div><div><button type="button" class="ui-button-default btn btn-secondary" id="uid-13" style="display: inline-block;" data-original-title="" title=""><i class="icon fa fa-folder-open-o"></i><span class="title"></span><div class="progress" style="display: none;"><div class="progress-bar" style="width: 0%;"></div></div></button></div></div></div></div> <!----> <span data-v-af72d116="" class="ui-form-info form-text text-muted">tab delimited file with 2 columns: name and sequence.  For example: Illumina Small RNA RT Primer CAAGCAGAAGACGGCATACGA</span></div></div><div><div data-v-af72d116="" id="form-element-adapters" class="ui-form-element section-row"><!----> <div data-v-af72d116="" class="ui-form-title"><span data-v-af72d116="" class="ui-form-title-text"><label data-v-af72d116="" for="adapters">Adapter list</label></span> <span data-v-af72d116="" class="ui-form-title-message">
            - optional
        </span></div> <div data-v-af72d116="" data-label="Adapter list" class="ui-form-field"><div data-v-af72d116="" data-label="Adapter list"><div class="ui-select-content d-flex flex-row flex-wrap flex-lg-nowrap"><div><div class="ui-options pr-lg-2" style="display: block;"><div class="mt-2" style="display: none;"></div><div class="ui-options-menu" style="display: block;"></div><div class="ui-radiobutton" style="display: inline-block;"><label role="button" class="ui-option active" data-original-title="" title=""><i class="fa fa-file-o no-padding"></i><input type="radio" name="uid-19" value="0" style="display: none;"></label><label role="button" class="ui-option" data-original-title="" title=""><i class="fa fa-files-o no-padding"></i><input type="radio" name="uid-19" value="1" style="display: none;"></label><label role="button" class="ui-option" data-original-title="" title=""><i class="fa fa-folder-o no-padding"></i><input type="radio" name="uid-19" value="2" style="display: none;"></label></div></div></div><div class="overflow-auto w-100 py-2 py-lg-0 pr-lg-2"><div class="ui-select" id="uid-16" style="display: block;"><div class="select2-container select select2-container-disabled" id="s2id_uid-16_select" title="" style="display: block;"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-11">No tabular dataset available.</span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen11" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-11" id="s2id_autogen11" disabled=""><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen11_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-11" id="s2id_autogen11_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-11">   </ul></div></div><div class="select select2-offscreen" id="uid-16_select" title="" style="display: block;" tabindex="-1"></div><div class="icon-dropdown fa fa-caret-down" style="display: block;"></div></div><div class="ui-select-multiple ui-select" id="uid-17" style="display: none;"><select class="select" id="uid-17_select" multiple="" disabled="" style="display: inline-block;"><option value="__null__">No tabular dataset available.</option></select><div class="icon-dropdown fa fa-caret-down" style="display: none;"></div><div class="icon-resize" style="display: block;"><i class="fa fa-angle-double-right fa-rotate-45" style="display: inline;"></i></div></div><div class="ui-select" id="uid-18" style="display: none;"><div class="select2-container select select2-container-disabled" id="s2id_uid-18_select" title="" style="display: block;"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-12">No tabular dataset collection available.</span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen12" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-12" id="s2id_autogen12" disabled=""><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen12_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-12" id="s2id_autogen12_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-12">   </ul></div></div><div class="select select2-offscreen" id="uid-18_select" title="" style="display: block;" tabindex="-1"></div><div class="icon-dropdown fa fa-caret-down" style="display: block;"></div></div><div class="form-text text-muted" style="clear: both; display: none;">
                    <i class="fa fa-sitemap"></i>
                    <span>
                        This is a batch mode input field. Separate jobs will be triggered for each dataset selection.
                    </span>
                </div><div class="form-text text-muted" style="clear: both; display: none;">
                    <i class="fa fa-sitemap"></i>
                    <span>
                        This is a batch mode input field. Separate jobs will be triggered for each dataset selection.
                    </span>
                <div class="ui-options" style="display: block;"><div class="mt-2" style="display: none;"></div><div class="ui-options-menu" style="display: block;"></div><div class="ui-radiobutton" style="display: inline-block;"><label role="button" class="ui-option active" data-original-title="" title=""><i class="fa fa-chain no-padding"></i><input type="radio" name="uid-15" value="false" style="display: none;"></label><label role="button" class="ui-option" data-original-title="" title=""><i class="fa fa-chain-broken no-padding"></i><input type="radio" name="uid-15" value="true" style="display: none;"></label></div></div></div></div><div style="margin-right: 5px;"><button type="button" class="ui-button-default btn btn-secondary" id="uid-21" style="display: inline-block;" data-original-title="" title=""><i class="icon fa fa-upload"></i><span class="title"></span><div class="progress" style="display: none;"><div class="progress-bar" style="width: 0%;"></div></div></button></div><div><button type="button" class="ui-button-default btn btn-secondary" id="uid-20" style="display: inline-block;" data-original-title="" title=""><i class="icon fa fa-folder-open-o"></i><span class="title"></span><div class="progress" style="display: none;"><div class="progress-bar" style="width: 0%;"></div></div></button></div></div></div></div> <!----> <span data-v-af72d116="" class="ui-form-info form-text text-muted">List of adapters adapter sequences which will be explicity searched against the library. It should be a tab-delimited file with 2 columns: name and sequence. (--adapters)</span></div></div><div><div data-v-af72d116="" id="form-element-limits" class="ui-form-element section-row"><!----> <div data-v-af72d116="" class="ui-form-title"><span data-v-af72d116="" class="ui-form-title-text"><label data-v-af72d116="" for="limits">Submodule and Limit specifing file</label></span> <span data-v-af72d116="" class="ui-form-title-message">
            - optional
        </span></div> <div data-v-af72d116="" data-label="Submodule and Limit specifing file" class="ui-form-field"><div data-v-af72d116="" data-label="Submodule and Limit specifing file"><div class="ui-select-content d-flex flex-row flex-wrap flex-lg-nowrap"><div><div class="ui-options pr-lg-2" style="display: block;"><div class="mt-2" style="display: none;"></div><div class="ui-options-menu" style="display: block;"></div><div class="ui-radiobutton" style="display: inline-block;"><label role="button" class="ui-option active" data-original-title="" title=""><i class="fa fa-file-o no-padding"></i><input type="radio" name="uid-26" value="0" style="display: none;"></label><label role="button" class="ui-option" data-original-title="" title=""><i class="fa fa-files-o no-padding"></i><input type="radio" name="uid-26" value="1" style="display: none;"></label><label role="button" class="ui-option" data-original-title="" title=""><i class="fa fa-folder-o no-padding"></i><input type="radio" name="uid-26" value="2" style="display: none;"></label></div></div></div><div class="overflow-auto w-100 py-2 py-lg-0 pr-lg-2"><div class="ui-select" id="uid-23" style="display: block;"><div class="select2-container select select2-container-disabled" id="s2id_uid-23_select" title="" style="display: block;"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-15">No txt dataset available.</span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen15" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-15" id="s2id_autogen15" disabled=""><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen15_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-15" id="s2id_autogen15_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-15">   </ul></div></div><div class="select select2-offscreen" id="uid-23_select" title="" style="display: block;" tabindex="-1"></div><div class="icon-dropdown fa fa-caret-down" style="display: block;"></div></div><div class="ui-select-multiple ui-select" id="uid-24" style="display: none;"><select class="select" id="uid-24_select" multiple="" disabled="" style="display: inline-block;"><option value="__null__">No txt dataset available.</option></select><div class="icon-dropdown fa fa-caret-down" style="display: none;"></div><div class="icon-resize" style="display: block;"><i class="fa fa-angle-double-right fa-rotate-45" style="display: inline;"></i></div></div><div class="ui-select" id="uid-25" style="display: none;"><div class="select2-container select select2-container-disabled" id="s2id_uid-25_select" title="" style="display: block;"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-16">No txt dataset collection available.</span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen16" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-16" id="s2id_autogen16" disabled=""><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen16_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-16" id="s2id_autogen16_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-16">   </ul></div></div><div class="select select2-offscreen" id="uid-25_select" title="" style="display: block;" tabindex="-1"></div><div class="icon-dropdown fa fa-caret-down" style="display: block;"></div></div><div class="form-text text-muted" style="clear: both; display: none;">
                    <i class="fa fa-sitemap"></i>
                    <span>
                        This is a batch mode input field. Separate jobs will be triggered for each dataset selection.
                    </span>
                </div><div class="form-text text-muted" style="clear: both; display: none;">
                    <i class="fa fa-sitemap"></i>
                    <span>
                        This is a batch mode input field. Separate jobs will be triggered for each dataset selection.
                    </span>
                <div class="ui-options" style="display: block;"><div class="mt-2" style="display: none;"></div><div class="ui-options-menu" style="display: block;"></div><div class="ui-radiobutton" style="display: inline-block;"><label role="button" class="ui-option active" data-original-title="" title=""><i class="fa fa-chain no-padding"></i><input type="radio" name="uid-22" value="false" style="display: none;"></label><label role="button" class="ui-option" data-original-title="" title=""><i class="fa fa-chain-broken no-padding"></i><input type="radio" name="uid-22" value="true" style="display: none;"></label></div></div></div></div><div style="margin-right: 5px;"><button type="button" class="ui-button-default btn btn-secondary" id="uid-28" style="display: inline-block;" data-original-title="" title=""><i class="icon fa fa-upload"></i><span class="title"></span><div class="progress" style="display: none;"><div class="progress-bar" style="width: 0%;"></div></div></button></div><div><button type="button" class="ui-button-default btn btn-secondary" id="uid-27" style="display: inline-block;" data-original-title="" title=""><i class="icon fa fa-folder-open-o"></i><span class="title"></span><div class="progress" style="display: none;"><div class="progress-bar" style="width: 0%;"></div></div></button></div></div></div></div> <!----> <span data-v-af72d116="" class="ui-form-info form-text text-muted">a file that specifies which submodules are to be executed (default=all) and also specifies the thresholds for the each submodules warning parameter</span></div></div><div><div data-v-af72d116="" id="form-element-nogroup" class="ui-form-element section-row"><!----> <div data-v-af72d116="" class="ui-form-title"><span data-v-af72d116="" class="ui-form-title-text"><label data-v-af72d116="" for="nogroup">Disable grouping of bases for reads &gt;50bp</label></span> <!----></div> <div data-v-af72d116="" data-label="Disable grouping of bases for reads >50bp" class="ui-form-field"><div data-v-af72d116="" class="no-highlight custom-control custom-switch" id="nogroup"><input type="checkbox" class="custom-control-input" value="true" id="__BVID__202"><label class="custom-control-label" for="__BVID__202">
    No
</label></div></div> <!----> <span data-v-af72d116="" class="ui-form-info form-text text-muted">Using this option will cause fastqc to crash and burn if you use it on really long reads, and your plots may end up a ridiculous size. You have been warned! (--nogroup)</span></div></div><div><div data-v-af72d116="" id="form-element-min_length" class="ui-form-element section-row"><!----> <div data-v-af72d116="" class="ui-form-title"><span data-v-af72d116="" class="ui-form-title-text"><label data-v-af72d116="" for="min_length">Lower limit on the length of the sequence to be shown in the report</label></span> <span data-v-af72d116="" class="ui-form-title-message">
            - optional
        </span></div> <div data-v-af72d116="" data-label="Lower limit on the length of the sequence to be shown in the report" class="ui-form-field"><div data-v-af72d116="" id="min_length"><!----> <div class="row align-items-center"><div class="col"><input type="number" step="1" class="form-control form-control-sm" id="__BVID__205"></div> <!----></div></div></div> <!----> <span data-v-af72d116="" class="ui-form-info form-text text-muted"> As long as you set this to a value greater or equal to your longest read length then this will be the sequence length used to create your read groups.  This can be useful for making directly comaparable statistics from datasets with somewhat variable read lengths. (--min_length)</span></div></div><div><div data-v-af72d116="" id="form-element-kmers" class="ui-form-element section-row"><!----> <div data-v-af72d116="" class="ui-form-title"><span data-v-af72d116="" class="ui-form-title-text"><label data-v-af72d116="" for="kmers">Length of Kmer to look for</label></span> <span data-v-af72d116="" title="required" class="ui-form-title-star">
            *
            <!----></span></div> <div data-v-af72d116="" data-label="Length of Kmer to look for" class="ui-form-field"><div data-v-af72d116="" id="kmers"><!----> <div class="row align-items-center"><div class="col-sm-4"><input type="number" step="1" class="form-control form-control-sm" id="__BVID__209"></div> <div class="pl-0 col"><input type="range" min="2" max="10" step="1" class="custom-range" id="__BVID__210"></div></div></div></div> <!----> <span data-v-af72d116="" class="ui-form-info form-text text-muted">Note: the Kmer test is disabled and needs to be enabled using a custom Submodule and limits file (--kmers)</span></div></div></div></div> <!----> <!----></div> <button title="Run tool: FastQC (0.74+galaxy0)" type="button" class="btn d-flex flex-nowrap align-items-center text-nowrap mt-3 mb-3 btn-primary" data-v-6d7325b0=""><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="play" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="mr-2 svg-inline--fa fa-play"><path fill="currentColor" d="M424.4 214.7L72.4 6.6C43.8-10.3 0 6.1 0 47.9V464c0 37.5 40.7 60.1 72.4 41.3l352-208c31.4-18.5 31.5-64.1 0-82.6z" class=""></path></svg>Run Tool
</button> <div data-v-6d7325b0=""><div data-v-6d7325b0="" class="mt-2 mb-4"><div data-v-b01e12ae="" data-v-6d7325b0="" class="separator heading"><div data-v-b01e12ae="" class="stripe"></div> <h2 data-v-b01e12ae="" class="h-sm font-weight-bold"> Help </h2> <div data-v-b01e12ae="" class="stripe"></div></div> <div data-v-05cd6a49="" data-v-6d7325b0="" class="form-help form-text"><p class="infomark"><strong>Purpose</strong></p>
<p>FastQC aims to provide a simple way to do some quality control checks on raw
sequence data coming from high throughput sequencing pipelines.
It provides a set of analyses which you can use to get a quick
impression of whether your data has any problems of
which you should be aware before doing any further analysis.</p>
<p>The main functions of FastQC are:</p>
<ul class="simple">
<li>Import of data from BAM, SAM or FastQ/FastQ.gz files (any variant),</li>
<li>Providing a quick overview to tell you in which areas there may be problems</li>
<li>Summary graphs and tables to quickly assess your data</li>
<li>Export of results to an HTML based permanent report</li>
<li>Offline operation to allow automated generation of reports without running the interactive application</li>
</ul>
<hr class="docutils">
<p class="infomark"><strong>FastQC</strong></p>
<p>This is a Galaxy wrapper. It merely exposes the external package <a class="reference external" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">FastQC</a> which is documented at <a class="reference external" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">FastQC</a>
Kindly acknowledge it as well as this tool if you use it.
FastQC incorporates the <a class="reference external" href="https://broadinstitute.github.io/picard/" target="_blank">Picard-tools</a> libraries for SAM/BAM processing.</p>
<p>The contaminants file parameter was borrowed from the independently developed
fastqcwrapper contributed to the Galaxy Community Tool Shed by J. Johnson.
Adaption to version 0.11.2 by T. McGowan.</p>
<hr class="docutils">
<p class="infomark"><strong>Inputs and outputs</strong></p>
<p><a class="reference external" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">FastQC</a> is the best place to look for documentation - it's very good.
A summary follows below for those in a tearing hurry.</p>
<p>This wrapper will accept a Galaxy fastq, fastq.gz, sam or bam as the input read file to check.
It will also take an optional file containing a list of contaminants information, in the form of
a tab-delimited file with 2 columns, name and sequence. As another option the tool takes a custom
limits.txt file that allows setting the warning thresholds for the different modules and also specifies
which modules to include in the output.</p>
<p>The tool produces a basic text and a HTML output file that contain all of the results, including the following:</p>
<ul class="simple">
<li>Basic Statistics</li>
<li>Per base sequence quality</li>
<li>Per sequence quality scores</li>
<li>Per base sequence content</li>
<li>Per base GC content</li>
<li>Per sequence GC content</li>
<li>Per base N content</li>
<li>Sequence Length Distribution</li>
<li>Sequence Duplication Levels</li>
<li>Overrepresented sequences</li>
<li>Kmer Content</li>
</ul>
<dl class="docutils">
<dt>All except Basic Statistics and Overrepresented sequences are plots.</dt>
<dd></dd>
</dl>
</div></div> <div data-v-6d7325b0="" class="mt-2 mb-4"><div data-v-b01e12ae="" class="separator heading"><div data-v-b01e12ae="" class="stripe"></div> <h2 data-v-b01e12ae="" class="h-sm font-weight-bold">Tutorials</h2> <div data-v-b01e12ae="" class="stripe"></div></div> <p>
        There are 40 tutorials available which use this tool.
        <span> These tutorials include training for the current version of the tool. </span> <a target="_blank" href="https://training.galaxyproject.org/training-material/by-tool/devteam/fastqc.html">
            View all tutorials referencing this tool.
         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></p> <button type="button" class="btn ui-link btn-secondary collapsed" aria-expanded="false" aria-controls="collapse-0" style="overflow-anchor: none;"><b>
            Tutorials available in 10
            categories
        </b> <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="caret-down" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-caret-down"><path fill="currentColor" d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z" class=""></path></svg></button> <div id="collapse-0" class="collapse" style="display: none;"><div><button type="button" class="btn ui-link ml-3 btn-secondary collapsed" aria-expanded="false" aria-controls="collapse-0-Assembly" style="overflow-anchor: none;">
                Assembly (5)
                <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="caret-down" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-caret-down"><path fill="currentColor" d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z" class=""></path></svg></button> <div id="collapse-0-Assembly" class="collapse" style="display: none;"><ul class="d-flex flex-column my-1"><li><a target="_blank" href="https://training.galaxyproject.org/topics/assembly/tutorials/general-introduction/tutorial.html" class="ml-2">
                            An Introduction to Genome Assembly
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/assembly/tutorials/largegenome/tutorial.html" class="ml-2">
                            Large genome assembly and polishing
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/assembly/tutorials/metagenomics-assembly/tutorial.html" class="ml-2">
                            Assembly of metagenomic sequencing data
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/assembly/tutorials/mrsa-illumina/tutorial.html" class="ml-2">
                            Genome Assembly of MRSA using Illumina MiSeq Data
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/assembly/tutorials/unicycler-assembly/tutorial.html" class="ml-2">
                            Unicycler Assembly
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li></ul></div></div><div><button type="button" class="btn ui-link ml-3 btn-secondary collapsed" aria-expanded="false" aria-controls="collapse-0-Ecology" style="overflow-anchor: none;">
                Ecology (3)
                <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="caret-down" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-caret-down"><path fill="currentColor" d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z" class=""></path></svg></button> <div id="collapse-0-Ecology" class="collapse" style="display: none;"><ul class="d-flex flex-column my-1"><li><a target="_blank" href="https://training.galaxyproject.org/topics/ecology/tutorials/Obitools-metabarcoding/tutorial.html" class="ml-2">
                            Metabarcoding/eDNA through Obitools
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/ecology/tutorials/de-novo-rad-seq/tutorial.html" class="ml-2">
                            RAD-Seq de-novo data analysis
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/ecology/tutorials/ref-based-rad-seq/tutorial.html" class="ml-2">
                            RAD-Seq Reference-based data analysis
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li></ul></div></div><div><button type="button" class="btn ui-link ml-3 btn-secondary collapsed" aria-expanded="false" aria-controls="collapse-0-Epigenetics" style="overflow-anchor: none;">
                Epigenetics (5)
                <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="caret-down" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-caret-down"><path fill="currentColor" d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z" class=""></path></svg></button> <div id="collapse-0-Epigenetics" class="collapse" style="display: none;"><ul class="d-flex flex-column my-1"><li><a target="_blank" href="https://training.galaxyproject.org/topics/epigenetics/tutorials/atac-seq/tutorial.html" class="ml-2">
                            ATAC-Seq data analysis
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/epigenetics/tutorials/estrogen-receptor-binding-site-identification/tutorial.html" class="ml-2">
                            Identification of the binding sites of the Estrogen receptor
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/epigenetics/tutorials/formation_of_super-structures_on_xi/tutorial.html" class="ml-2">
                            Formation of the Super-Structures on the Inactive X
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/epigenetics/tutorials/methylation-seq/tutorial.html" class="ml-2">
                            DNA Methylation data analysis
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/epigenetics/tutorials/tal1-binding-site-identification/tutorial.html" class="ml-2">
                            Identification of the binding sites of the T-cell acute lymphocytic leukemia protein 1 (TAL1)
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li></ul></div></div><div><button type="button" class="btn ui-link ml-3 btn-secondary collapsed" aria-expanded="false" aria-controls="collapse-0-Genome-Annotation" style="overflow-anchor: none;">
                Genome Annotation (1)
                <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="caret-down" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-caret-down"><path fill="currentColor" d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z" class=""></path></svg></button> <div id="collapse-0-Genome-Annotation" class="collapse" style="display: none;"><ul class="d-flex flex-column my-1"><li><a target="_blank" href="https://training.galaxyproject.org/topics/genome-annotation/tutorials/crispr-screen/tutorial.html" class="ml-2">
                            CRISPR screen analysis
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li></ul></div></div><div><button type="button" class="btn ui-link ml-3 btn-secondary collapsed" aria-expanded="false" aria-controls="collapse-0-Introduction-to-Galaxy-Analyses" style="overflow-anchor: none;">
                Introduction to Galaxy Analyses (1)
                <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="caret-down" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-caret-down"><path fill="currentColor" d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z" class=""></path></svg></button> <div id="collapse-0-Introduction-to-Galaxy-Analyses" class="collapse" style="display: none;"><ul class="d-flex flex-column my-1"><li><a target="_blank" href="https://training.galaxyproject.org/topics/introduction/tutorials/galaxy-intro-short/tutorial.html" class="ml-2">
                            A short introduction to Galaxy
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li></ul></div></div><div><button type="button" class="btn ui-link ml-3 btn-secondary collapsed" aria-expanded="false" aria-controls="collapse-0-Metagenomics" style="overflow-anchor: none;">
                Metagenomics (6)
                <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="caret-down" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-caret-down"><path fill="currentColor" d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z" class=""></path></svg></button> <div id="collapse-0-Metagenomics" class="collapse" style="display: none;"><ul class="d-flex flex-column my-1"><li><a target="_blank" href="https://training.galaxyproject.org/topics/metagenomics/tutorials/beer-data-analysis/tutorial.html" class="ml-2">
                            Identification of the micro-organisms in a beer using Nanopore sequencing
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/metagenomics/tutorials/metagenomics-assembly/tutorial.html" class="ml-2">
                            Assembly of metagenomic sequencing data
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/metagenomics/tutorials/metatranscriptomics/tutorial.html" class="ml-2">
                            Metatranscriptomics analysis using microbiome RNA-seq data
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/metagenomics/tutorials/metatranscriptomics-short/tutorial.html" class="ml-2">
                            Metatranscriptomics analysis using microbiome RNA-seq data (short)
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.html" class="ml-2">
                            16S Microbial analysis with Nanopore data
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html" class="ml-2">
                            Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li></ul></div></div><div><button type="button" class="btn ui-link ml-3 btn-secondary collapsed" aria-expanded="false" aria-controls="collapse-0-Sequence-analysis" style="overflow-anchor: none;">
                Sequence analysis (3)
                <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="caret-down" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-caret-down"><path fill="currentColor" d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z" class=""></path></svg></button> <div id="collapse-0-Sequence-analysis" class="collapse" style="display: none;"><ul class="d-flex flex-column my-1"><li><a target="_blank" href="https://training.galaxyproject.org/topics/sequence-analysis/tutorials/mapping/tutorial.html" class="ml-2">
                            Mapping
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/sequence-analysis/tutorials/quality-control/tutorial.html" class="ml-2">
                            Quality Control
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.html" class="ml-2">
                            SARS-CoV-2 Viral Sample Alignment and Variant Visualization
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li></ul></div></div><div><button type="button" class="btn ui-link ml-3 btn-secondary collapsed" aria-expanded="false" aria-controls="collapse-0-Single-Cell" style="overflow-anchor: none;">
                Single Cell (2)
                <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="caret-down" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-caret-down"><path fill="currentColor" d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z" class=""></path></svg></button> <div id="collapse-0-Single-Cell" class="collapse" style="display: none;"><ul class="d-flex flex-column my-1"><li><a target="_blank" href="https://training.galaxyproject.org/topics/single-cell/tutorials/scatac-preprocessing-tenx/tutorial.html" class="ml-2">
                            Pre-processing of 10X Single-Cell ATAC-seq Datasets
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/single-cell/tutorials/scrna-umis/tutorial.html" class="ml-2">
                            Understanding Barcodes
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li></ul></div></div><div><button type="button" class="btn ui-link ml-3 btn-secondary collapsed" aria-expanded="false" aria-controls="collapse-0-Transcriptomics" style="overflow-anchor: none;">
                Transcriptomics (9)
                <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="caret-down" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-caret-down"><path fill="currentColor" d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z" class=""></path></svg></button> <div id="collapse-0-Transcriptomics" class="collapse" style="display: none;"><ul class="d-flex flex-column my-1"><li><a target="_blank" href="https://training.galaxyproject.org/topics/transcriptomics/tutorials/clipseq/tutorial.html" class="ml-2">
                            CLIP-Seq data analysis from pre-processing to motif detection
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/transcriptomics/tutorials/de-novo/tutorial.html" class="ml-2">
                            De novo transcriptome reconstruction with RNA-Seq
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.html" class="ml-2">
                            Genome-wide alternative splicing analysis
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/transcriptomics/tutorials/full-de-novo/tutorial.html" class="ml-2">
                            De novo transcriptome assembly, annotation, and differential expression analysis
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/transcriptomics/tutorials/mirna-target-finder/tutorial.html" class="ml-2">
                            Whole transcriptome analysis of Arabidopsis thaliana
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/transcriptomics/tutorials/ref-based/tutorial.html" class="ml-2">
                            Reference-based RNA-Seq data analysis
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/transcriptomics/tutorials/rna-interactome/tutorial.html" class="ml-2">
                            RNA-RNA interactome data analysis
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/transcriptomics/tutorials/rna-seq-reads-to-counts/tutorial.html" class="ml-2">
                            1: RNA-Seq reads to counts
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/transcriptomics/tutorials/srna/tutorial.html" class="ml-2">
                            Differential abundance testing of small RNAs
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li></ul></div></div><div><button type="button" class="btn ui-link ml-3 btn-secondary collapsed" aria-expanded="false" aria-controls="collapse-0-Variant-Analysis" style="overflow-anchor: none;">
                Variant Analysis (5)
                <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="caret-down" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-caret-down"><path fill="currentColor" d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z" class=""></path></svg></button> <div id="collapse-0-Variant-Analysis" class="collapse" style="display: none;"><ul class="d-flex flex-column my-1"><li><a target="_blank" href="https://training.galaxyproject.org/topics/variant-analysis/tutorials/exome-seq/tutorial.html" class="ml-2">
                            Exome sequencing data analysis for diagnosing a genetic disease
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/variant-analysis/tutorials/non-dip/tutorial.html" class="ml-2">
                            Calling variants in non-diploid systems
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/variant-analysis/tutorials/somatic-variant-discovery/tutorial.html" class="ml-2">
                            Somatic Variant Discovery from WES Data Using Control-FREEC
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/variant-analysis/tutorials/somatic-variants/tutorial.html" class="ml-2">
                            Identification of somatic and germline variants from tumor and normal sample pairs
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li><li><a target="_blank" href="https://training.galaxyproject.org/topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html" class="ml-2">
                            M. tuberculosis Variant Analysis
                         <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></li></ul></div></div></div></div> <div data-v-3acb8ac3="" data-v-6d7325b0="" class="card tool-footer"><!----><!----><div data-v-3acb8ac3="" class="card-body"><!----><!----><div data-v-3acb8ac3="" class="mb-1"><span data-v-3acb8ac3="" class="footer-section-name">Citations</span> <button data-v-3acb8ac3="" title="Copy all citations as BibTeX" type="button" class="btn btn-link btn-sm" style="cursor: pointer;"><svg data-v-3acb8ac3="" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="copy" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="svg-inline--fa fa-copy"><path data-v-3acb8ac3="" fill="currentColor" d="M320 448v40c0 13.255-10.745 24-24 24H24c-13.255 0-24-10.745-24-24V120c0-13.255 10.745-24 24-24h72v296c0 30.879 25.121 56 56 56h168zm0-344V0H152c-13.255 0-24 10.745-24 24v368c0 13.255 10.745 24 24 24h272c13.255 0 24-10.745 24-24V128H344c-13.2 0-24-10.8-24-24zm120.971-31.029L375.029 7.029A24 24 0 0 0 358.059 0H352v96h96v-6.059a24 24 0 0 0-7.029-16.97z" class=""></path></svg></button> <div data-v-3acb8ac3="" class="formatted-reference">
    -
    <span><div class="csl-bib-body">
  <div data-csl-entry-id="andrews_s" class="csl-entry">Andrews, S. (n.d.). <i>FastQC A Quality Control tool for High Throughput Sequence Data</i>. http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</div>
</div></span> <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">Visit Citation <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a></div></div> <div data-v-3acb8ac3="" class="mb-1"><span data-v-3acb8ac3="" class="footer-section-name">Requirements</span> <a data-v-3acb8ac3="" title="Learn more about Galaxy Requirements" href="https://galaxyproject.org/tools/requirements/" target="_blank">
            See details <svg data-v-3acb8ac3="" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><path data-v-3acb8ac3="" fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a> <div data-v-3acb8ac3="">
            - fastqc
            <span data-v-3acb8ac3=""> (Version 0.12.1) </span></div></div> <!----> <div data-v-3acb8ac3="" class="mb-1"><span data-v-3acb8ac3="" class="footer-section-name">References</span> <div data-v-3acb8ac3="">
            -
            
                bio.tools: fastqc (<a data-v-3acb8ac3="" href="https://bio.tools/fastqc" target="_blank">bio.tools
                    <svg data-v-3acb8ac3="" aria-labelledby="svg-inline--fa-title-fOKfEtuNauVy" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><title data-v-3acb8ac3="" id="svg-inline--fa-title-fOKfEtuNauVy" class="">Visit bio.tools reference</title><path data-v-3acb8ac3="" fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a>) (<a data-v-3acb8ac3="" href="https://openebench.bsc.es/tool/fastqc" target="_blank">OpenEBench
                    <svg data-v-3acb8ac3="" aria-labelledby="svg-inline--fa-title-KFxZaHxZEocN" data-prefix="fas" data-icon="external-link-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-external-link-alt"><title data-v-3acb8ac3="" id="svg-inline--fa-title-KFxZaHxZEocN" class="">Visit OpenEBench reference</title><path data-v-3acb8ac3="" fill="currentColor" d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z" class=""></path></svg></a>)
            </div></div> <!----></div><!----><!----></div></div></div></div></div></div> <div id="right" class="d-flex flex-panel-right"><div class="d-flex flex-column" style="width: 300px;"><div class="d-flex flex-column history-index" id="current-history-panel"><section class="history-layout d-flex flex-column w-100"><div><nav aria-label="current history management" class="d-flex justify-content-between mx-3 my-2"><h2 class="m-1 h-sm">History</h2> <div role="group" class="btn-group"><button data-description="create new history" title="Log in to Create new history" type="button" disabled="disabled" class="btn create-hist-btn btn-link btn-sm disabled"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="plus" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="svg-inline--fa fa-plus fa-fw"><path fill="currentColor" d="M416 208H272V64c0-17.67-14.33-32-32-32h-32c-17.67 0-32 14.33-32 32v144H32c-17.67 0-32 14.33-32 32v32c0 17.67 14.33 32 32 32h144v144c0 17.67 14.33 32 32 32h32c17.67 0 32-14.33 32-32V304h144c17.67 0 32-14.33 32-32v-32c0-17.67-14.33-32-32-32z" class=""></path></svg></button> <button data-description="switch to another history" title="Log in to Switch to history" type="button" disabled="disabled" class="btn btn-link btn-sm disabled"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="exchange-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-exchange-alt fa-fw"><path fill="currentColor" d="M0 168v-16c0-13.255 10.745-24 24-24h360V80c0-21.367 25.899-32.042 40.971-16.971l80 80c9.372 9.373 9.372 24.569 0 33.941l-80 80C409.956 271.982 384 261.456 384 240v-48H24c-13.255 0-24-10.745-24-24zm488 152H128v-48c0-21.314-25.862-32.08-40.971-16.971l-80 80c-9.372 9.373-9.372 24.569 0 33.941l80 80C102.057 463.997 128 453.437 128 432v-48h360c13.255 0 24-10.745 24-24v-16c0-13.255-10.745-24-24-24z" class=""></path></svg></button> <div class="dropdown b-dropdown btn-group" title="History options" data-description="history options" id="__BVID__231"><!----><button aria-haspopup="menu" aria-expanded="false" type="button" class="btn dropdown-toggle btn-link btn-sm text-decoration-none" id="__BVID__231__BV_toggle_"><span class="sr-only">History Options</span></button><ul role="menu" tabindex="-1" class="dropdown-menu history-options-button-menu" aria-labelledby="__BVID__231__BV_toggle_"> <li role="presentation"><p class="b-dropdown-text"><span>You have 1 histories.</span></p></li> <li role="presentation"><a data-description="switch to multi history view" title="Log in to Open History Multiview" role="menuitem" href="#" target="_self" tabindex="-1" aria-disabled="true" class="dropdown-item disabled"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="columns" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="mr-1 svg-inline--fa fa-columns fa-fw"><path fill="currentColor" d="M464 32H48C21.49 32 0 53.49 0 80v352c0 26.51 21.49 48 48 48h416c26.51 0 48-21.49 48-48V80c0-26.51-21.49-48-48-48zM224 416H64V160h160v256zm224 0H288V160h160v256z" class=""></path></svg> <span>Show Histories Side-by-Side</span></a></li> <li role="presentation"><hr role="separator" aria-orientation="horizontal" class="dropdown-divider"></li> <li role="presentation"><a title="Resume all Paused Jobs in this History" role="menuitem" href="#" target="_self" class="dropdown-item"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="play" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="mr-1 svg-inline--fa fa-play fa-fw"><path fill="currentColor" d="M424.4 214.7L72.4 6.6C43.8-10.3 0 6.1 0 47.9V464c0 37.5 40.7 60.1 72.4 41.3l352-208c31.4-18.5 31.5-64.1 0-82.6z" class=""></path></svg> <span>Resume Paused Jobs</span></a></li> <li role="presentation"><hr role="separator" aria-orientation="horizontal" class="dropdown-divider"></li> <li role="presentation"><a title="Log in to Copy History to a New History" role="menuitem" href="#" target="_self" tabindex="-1" aria-disabled="true" class="dropdown-item disabled"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="copy" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="mr-1 svg-inline--fa fa-copy fa-fw"><path fill="currentColor" d="M320 448v40c0 13.255-10.745 24-24 24H24c-13.255 0-24-10.745-24-24V120c0-13.255 10.745-24 24-24h72v296c0 30.879 25.121 56 56 56h168zm0-344V0H152c-13.255 0-24 10.745-24 24v368c0 13.255 10.745 24 24 24h272c13.255 0 24-10.745 24-24V128H344c-13.2 0-24-10.8-24-24zm120.971-31.029L375.029 7.029A24 24 0 0 0 358.059 0H352v96h96v-6.059a24 24 0 0 0-7.029-16.97z" class=""></path></svg> <span>Copy this History</span></a></li> <li role="presentation"><a title="Permanently Delete History" role="menuitem" href="#" target="_self" class="dropdown-item"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="trash" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="mr-1 svg-inline--fa fa-trash fa-fw"><path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z" class=""></path></svg> <span>Delete this History</span></a></li> <li role="presentation"><a title="Export Citations for all Tools used in this History" role="menuitem" href="#" target="_self" class="dropdown-item"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="stream" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="mr-1 svg-inline--fa fa-stream fa-fw"><path fill="currentColor" d="M16 128h416c8.84 0 16-7.16 16-16V48c0-8.84-7.16-16-16-16H16C7.16 32 0 39.16 0 48v64c0 8.84 7.16 16 16 16zm480 80H80c-8.84 0-16 7.16-16 16v64c0 8.84 7.16 16 16 16h416c8.84 0 16-7.16 16-16v-64c0-8.84-7.16-16-16-16zm-64 176H16c-8.84 0-16 7.16-16 16v64c0 8.84 7.16 16 16 16h416c8.84 0 16-7.16 16-16v-64c0-8.84-7.16-16-16-16z" class=""></path></svg> <span>Export Tool Citations</span></a></li> <li role="presentation"><a data-description="export to file" title="Export and Download History as a File" role="menuitem" href="#" target="_self" class="dropdown-item"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="file-archive" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512" class="mr-1 svg-inline--fa fa-file-archive fa-fw"><path fill="currentColor" d="M377 105L279.1 7c-4.5-4.5-10.6-7-17-7H256v128h128v-6.1c0-6.3-2.5-12.4-7-16.9zM128.4 336c-17.9 0-32.4 12.1-32.4 27 0 15 14.6 27 32.5 27s32.4-12.1 32.4-27-14.6-27-32.5-27zM224 136V0h-63.6v32h-32V0H24C10.7 0 0 10.7 0 24v464c0 13.3 10.7 24 24 24h336c13.3 0 24-10.7 24-24V160H248c-13.2 0-24-10.8-24-24zM95.9 32h32v32h-32zm32.3 384c-33.2 0-58-30.4-51.4-62.9L96.4 256v-32h32v-32h-32v-32h32v-32h-32V96h32V64h32v32h-32v32h32v32h-32v32h32v32h-32v32h22.1c5.7 0 10.7 4.1 11.8 9.7l17.3 87.7c6.4 32.4-18.4 62.6-51.4 62.6z" class=""></path></svg> <span>Export History to File</span></a></li> <li role="presentation"><a data-description="archive history" title="Log in to Archive this History" role="menuitem" href="#" target="_self" tabindex="-1" aria-disabled="true" class="dropdown-item disabled"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="archive" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="mr-1 svg-inline--fa fa-archive fa-fw"><path fill="currentColor" d="M32 448c0 17.7 14.3 32 32 32h384c17.7 0 32-14.3 32-32V160H32v288zm160-212c0-6.6 5.4-12 12-12h104c6.6 0 12 5.4 12 12v8c0 6.6-5.4 12-12 12H204c-6.6 0-12-5.4-12-12v-8zM480 32H32C14.3 32 0 46.3 0 64v48c0 8.8 7.2 16 16 16h480c8.8 0 16-7.2 16-16V64c0-17.7-14.3-32-32-32z" class=""></path></svg> <span>Archive History</span></a></li> <li role="presentation"><a title="Log in to Convert History to Workflow" role="menuitem" href="#" target="_self" tabindex="-1" aria-disabled="true" class="dropdown-item disabled"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="file-export" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512" class="mr-1 svg-inline--fa fa-file-export fa-fw"><path fill="currentColor" d="M384 121.9c0-6.3-2.5-12.4-7-16.9L279.1 7c-4.5-4.5-10.6-7-17-7H256v128h128zM571 308l-95.7-96.4c-10.1-10.1-27.4-3-27.4 11.3V288h-64v64h64v65.2c0 14.3 17.3 21.4 27.4 11.3L571 332c6.6-6.6 6.6-17.4 0-24zm-379 28v-32c0-8.8 7.2-16 16-16h176V160H248c-13.2 0-24-10.8-24-24V0H24C10.7 0 0 10.7 0 24v464c0 13.3 10.7 24 24 24h336c13.3 0 24-10.7 24-24V352H208c-8.8 0-16-7.2-16-16z" class=""></path></svg> <span>Extract Workflow</span></a></li> <li role="presentation"><a title="Log in to Display Workflow Invocations" role="menuitem" href="#" target="_self" tabindex="-1" aria-disabled="true" class="dropdown-item disabled"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="sitemap" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 512" class="fa-rotate-270 mr-1 svg-inline--fa fa-sitemap fa-fw"><path fill="currentColor" d="M128 352H32c-17.67 0-32 14.33-32 32v96c0 17.67 14.33 32 32 32h96c17.67 0 32-14.33 32-32v-96c0-17.67-14.33-32-32-32zm-24-80h192v48h48v-48h192v48h48v-57.59c0-21.17-17.23-38.41-38.41-38.41H344v-64h40c17.67 0 32-14.33 32-32V32c0-17.67-14.33-32-32-32H256c-17.67 0-32 14.33-32 32v96c0 17.67 14.33 32 32 32h40v64H94.41C73.23 224 56 241.23 56 262.41V320h48v-48zm264 80h-96c-17.67 0-32 14.33-32 32v96c0 17.67 14.33 32 32 32h96c17.67 0 32-14.33 32-32v-96c0-17.67-14.33-32-32-32zm240 0h-96c-17.67 0-32 14.33-32 32v96c0 17.67 14.33 32 32 32h96c17.67 0 32-14.33 32-32v-96c0-17.67-14.33-32-32-32z" class=""></path></svg> <span>Show Invocations</span></a></li> <li role="presentation"><hr role="separator" aria-orientation="horizontal" class="dropdown-divider"></li> <li role="presentation"><a title="Log in to Share or Publish this History" data-description="share or publish" role="menuitem" href="#" target="_self" tabindex="-1" aria-disabled="true" class="dropdown-item disabled"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="share-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="mr-1 svg-inline--fa fa-share-alt fa-fw"><path fill="currentColor" d="M352 320c-22.608 0-43.387 7.819-59.79 20.895l-102.486-64.054a96.551 96.551 0 0 0 0-41.683l102.486-64.054C308.613 184.181 329.392 192 352 192c53.019 0 96-42.981 96-96S405.019 0 352 0s-96 42.981-96 96c0 7.158.79 14.13 2.276 20.841L155.79 180.895C139.387 167.819 118.608 160 96 160c-53.019 0-96 42.981-96 96s42.981 96 96 96c22.608 0 43.387-7.819 59.79-20.895l102.486 64.054A96.301 96.301 0 0 0 256 416c0 53.019 42.981 96 96 96s96-42.981 96-96-42.981-96-96-96z" class=""></path></svg> <span>Share or Publish</span></a></li> <li role="presentation"><a title="Log in to Set who can View or Edit this History" role="menuitem" href="#" target="_self" tabindex="-1" aria-disabled="true" class="dropdown-item disabled"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="user-lock" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 512" class="mr-1 svg-inline--fa fa-user-lock fa-fw"><path fill="currentColor" d="M224 256A128 128 0 1 0 96 128a128 128 0 0 0 128 128zm96 64a63.08 63.08 0 0 1 8.1-30.5c-4.8-.5-9.5-1.5-14.5-1.5h-16.7a174.08 174.08 0 0 1-145.8 0h-16.7A134.43 134.43 0 0 0 0 422.4V464a48 48 0 0 0 48 48h280.9a63.54 63.54 0 0 1-8.9-32zm288-32h-32v-80a80 80 0 0 0-160 0v80h-32a32 32 0 0 0-32 32v160a32 32 0 0 0 32 32h224a32 32 0 0 0 32-32V320a32 32 0 0 0-32-32zM496 432a32 32 0 1 1 32-32 32 32 0 0 1-32 32zm32-144h-64v-80a32 32 0 0 1 64 0z" class=""></path></svg> <span>Set Permissions</span></a></li> <li role="presentation"><a title="Log in to Make this History Private" role="menuitem" href="#" target="_self" tabindex="-1" aria-disabled="true" class="dropdown-item disabled"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="lock" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="mr-1 svg-inline--fa fa-lock fa-fw"><path fill="currentColor" d="M400 224h-24v-72C376 68.2 307.8 0 224 0S72 68.2 72 152v72H48c-26.5 0-48 21.5-48 48v192c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V272c0-26.5-21.5-48-48-48zm-104 0H152v-72c0-39.7 32.3-72 72-72s72 32.3 72 72v72z" class=""></path></svg> <span>Make Private</span></a></li></ul></div></div></nav> <div id="selector-history-modal" style="display: none;"><div id="selector-history-modal___BV_modal_outer_" class="history-selector-modal" style="position: absolute; z-index: 1040;"><div id="selector-history-modal" role="dialog" aria-hidden="true" aria-labelledby="selector-history-modal___BV_modal_title_" aria-describedby="selector-history-modal___BV_modal_body_" class="modal fade" style="display: none;"><div class="modal-dialog modal-md"><!----><div id="selector-history-modal___BV_modal_content_" tabindex="-1" class="modal-content"><header id="selector-history-modal___BV_modal_header_" class="modal-header"><h5 id="selector-history-modal___BV_modal_title_" class="modal-title"><h2 data-v-b01e12ae="" class="heading h-sm inline"><!----> Switch to history</h2></h5><button type="button" aria-label="Close" class="close">×</button></header><div id="selector-history-modal___BV_modal_body_" class="modal-body"> <fieldset class="form-group" id="__BVID__261"><!----><div><input type="search" placeholder="Search Filter" class="form-control" id="__BVID__262"><!----><!----><small tabindex="-1" class="form-text text-muted" id="__BVID__261__BV_description_">Filter histories</small></div></fieldset> <!----> <div class="history-selector-modal-list"><div class="list-group"><button data-pk="e6ef9ff086fa4c6a" type="button" class="list-group-item list-group-item-action current"><div class="d-flex justify-content-between align-items-center"><h3 data-v-b01e12ae="" class="heading h-text font-weight-bold inline"><!----> 
                            Unnamed history
                            <i data-v-b01e12ae="">(Current)</i></h3> <div class="d-flex align-items-center flex-gapx-1"><span title="Amount of items in history" class="badge badge-secondary badge-pill">
                                0 items
                            </span> <span title="Last edited" class="badge badge-secondary badge-pill"><span title="2023-12-28T15:45:51.791Z" class="utc-time utc-time-elapsed">
    in less than a minute
</span></span></div></div> <!----> <!----> <div class="d-flex justify-content-end align-items-center mt-1"><div role="group" class="btn-group"><!----> <button title="Open in multi-view" type="button" class="btn p-0 px-1 btn-link"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="columns" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-columns"><path fill="currentColor" d="M464 32H48C21.49 32 0 53.49 0 80v352c0 26.51 21.49 48 48 48h416c26.51 0 48-21.49 48-48V80c0-26.51-21.49-48-48-48zM224 416H64V160h160v256zm224 0H288V160h160v256z" class=""></path></svg></button> <button title="Open in center panel" type="button" class="btn p-0 px-1 btn-link"><svg aria-hidden="true" focusable="false" data-prefix="far" data-icon="list-alt" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-list-alt"><path fill="currentColor" d="M464 32H48C21.49 32 0 53.49 0 80v352c0 26.51 21.49 48 48 48h416c26.51 0 48-21.49 48-48V80c0-26.51-21.49-48-48-48zm-6 400H54a6 6 0 0 1-6-6V86a6 6 0 0 1 6-6h404a6 6 0 0 1 6 6v340a6 6 0 0 1-6 6zm-42-92v24c0 6.627-5.373 12-12 12H204c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h200c6.627 0 12 5.373 12 12zm0-96v24c0 6.627-5.373 12-12 12H204c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h200c6.627 0 12 5.373 12 12zm0-96v24c0 6.627-5.373 12-12 12H204c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h200c6.627 0 12 5.373 12 12zm-252 12c0 19.882-16.118 36-36 36s-36-16.118-36-36 16.118-36 36-36 36 16.118 36 36zm0 96c0 19.882-16.118 36-36 36s-36-16.118-36-36 16.118-36 36-36 36 16.118 36 36zm0 96c0 19.882-16.118 36-36 36s-36-16.118-36-36 16.118-36 36-36 36 16.118 36 36z" class=""></path></svg></button></div></div></button> <div><div class="list-end my-2"><span>- 1 history loaded -</span></div> <div class="b-overlay-wrap position-relative"><!----></div></div></div></div></div><footer id="selector-history-modal___BV_modal_footer_" class="modal-footer"><!----> <span>Click a history to switch to it</span></footer></div><!----></div></div><!----></div></div> <!----> <!----> <!----> <!----></div> <div class="content-operations-filters mx-3"><div role="group" class="input-group"><!----><input type="text" placeholder="search datasets" class="form-control form-control-sm" data-description="filter text input" id="__BVID__279"> <div class="input-group-append"><button aria-haspopup="true" data-description="show advanced filter toggle" title="Show Advanced Filter" aria-label="Show advanced filter" type="button" aria-pressed="false" autocomplete="off" class="btn btn-secondary btn-sm"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="angle-double-down" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-angle-double-down fa-fw"><path fill="currentColor" d="M143 256.3L7 120.3c-9.4-9.4-9.4-24.6 0-33.9l22.6-22.6c9.4-9.4 24.6-9.4 33.9 0l96.4 96.4 96.4-96.4c9.4-9.4 24.6-9.4 33.9 0L313 86.3c9.4 9.4 9.4 24.6 0 33.9l-136 136c-9.4 9.5-24.6 9.5-34 .1zm34 192l136-136c9.4-9.4 9.4-24.6 0-33.9l-22.6-22.6c-9.4-9.4-24.6-9.4-33.9 0L160 352.1l-96.4-96.4c-9.4-9.4-24.6-9.4-33.9 0L7 278.3c-9.4 9.4-9.4 24.6 0 33.9l136 136c9.4 9.5 24.6 9.5 34 .1z" class=""></path></svg></button> <button aria-haspopup="true" title="Clear Filters (esc)" aria-label="Clear filters" data-description="clear filters" type="button" class="btn btn-secondary btn-sm"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="times" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 352 512" class="svg-inline--fa fa-times fa-fw"><path fill="currentColor" d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z" class=""></path></svg></button></div><!----></div> <!----></div> <section><section data-description="edit details" class="m-3 details"><button data-description="editor toggle" title="Log in to Rename History" type="button" disabled="disabled" aria-pressed="false" autocomplete="off" class="btn edit-button ml-1 float-right btn-link btn-sm disabled"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="pen" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-pen"><path fill="currentColor" d="M290.74 93.24l128.02 128.02-277.99 277.99-114.14 12.6C11.35 513.54-1.56 500.62.14 485.34l12.7-114.22 277.9-277.88zm207.2-19.06l-60.11-60.11c-18.75-18.75-49.16-18.75-67.91 0l-56.55 56.55 128.02 128.02 56.55-56.55c18.75-18.76 18.75-49.16 0-67.91z" class=""></path></svg></button> <h3 data-description="name display" class="my-2 text-break">Unnamed history</h3> <div><!----> <div data-v-094f1a6c="" class="stateless-tags tags mt-2 px-1"><div data-v-094f1a6c="" class="pl-1 pb-2"><div data-v-094f1a6c="" class="d-inline"> <!----> <!----></div></div></div></div></section> <!----> <div class="history-size my-1 d-flex justify-content-between"><button title="History Size" data-description="storage dashboard button" type="button" class="btn rounded-0 text-decoration-none history-storage-overview-button btn-link btn-sm"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="database" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="svg-inline--fa fa-database"><path fill="currentColor" d="M448 73.143v45.714C448 159.143 347.667 192 224 192S0 159.143 0 118.857V73.143C0 32.857 100.333 0 224 0s224 32.857 224 73.143zM448 176v102.857C448 319.143 347.667 352 224 352S0 319.143 0 278.857V176c48.125 33.143 136.208 48.572 224 48.572S399.874 209.143 448 176zm0 160v102.857C448 479.143 347.667 512 224 512S0 479.143 0 438.857V336c48.125 33.143 136.208 48.572 224 48.572S399.874 369.143 448 336z" class=""></path></svg> <span>0 B</span></button> <div role="group" class="btn-group"><!----> <!----> <div role="group" class="btn-group"><button title="Show active" data-description="show active items button" type="button" class="btn rounded-0 text-decoration-none btn-link btn-sm"><span class="fa fa-map-marker"></span> <span>0</span></button> <!----> <!----> <button title="Last refreshed 3 seconds ago" type="button" class="btn rounded-0 text-decoration-none history-refresh-button btn-link btn-sm"><span class="fa fa-sync"></span></button></div> <!----></div></div> <section><nav class="content-operations d-flex justify-content-between bg-secondary"><div role="group" class="btn-group"><button title="Select Items" type="button" disabled="disabled" aria-pressed="false" autocomplete="off" class="btn show-history-content-selectors-btn rounded-0 btn-link btn-sm disabled"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="check-square" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="svg-inline--fa fa-check-square"><path fill="currentColor" d="M400 480H48c-26.51 0-48-21.49-48-48V80c0-26.51 21.49-48 48-48h352c26.51 0 48 21.49 48 48v352c0 26.51-21.49 48-48 48zm-204.686-98.059l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.248-16.379-6.249-22.628 0L184 302.745l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.25 16.379 6.25 22.628.001z" class=""></path></svg></button> <button title="Collapse Items" type="button" disabled="disabled" class="btn rounded-0 btn-link btn-sm disabled"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="compress" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="svg-inline--fa fa-compress"><path fill="currentColor" d="M436 192H312c-13.3 0-24-10.7-24-24V44c0-6.6 5.4-12 12-12h40c6.6 0 12 5.4 12 12v84h84c6.6 0 12 5.4 12 12v40c0 6.6-5.4 12-12 12zm-276-24V44c0-6.6-5.4-12-12-12h-40c-6.6 0-12 5.4-12 12v84H12c-6.6 0-12 5.4-12 12v40c0 6.6 5.4 12 12 12h124c13.3 0 24-10.7 24-24zm0 300V344c0-13.3-10.7-24-24-24H12c-6.6 0-12 5.4-12 12v40c0 6.6 5.4 12 12 12h84v84c0 6.6 5.4 12 12 12h40c6.6 0 12-5.4 12-12zm192 0v-84h84c6.6 0 12-5.4 12-12v-40c0-6.6-5.4-12-12-12H312c-13.3 0-24 10.7-24 24v124c0 6.6 5.4 12 12 12h40c6.6 0 12-5.4 12-12z" class=""></path></svg></button></div> <div role="group" class="btn-group" style="display: none;"><!----> <!----></div> <section><div class="dropdown b-dropdown rounded-0 btn-group" data-description="history action menu" id="__BVID__294"><!----><button aria-haspopup="menu" aria-expanded="false" type="button" class="btn dropdown-toggle btn-link btn-sm text-decoration-none rounded-0 dropdown-toggle-no-caret" id="__BVID__294__BV_toggle_"><span class="sr-only">History actions</span> <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="cog" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="svg-inline--fa fa-cog"><path fill="currentColor" d="M487.4 315.7l-42.6-24.6c4.3-23.2 4.3-47 0-70.2l42.6-24.6c4.9-2.8 7.1-8.6 5.5-14-11.1-35.6-30-67.8-54.7-94.6-3.8-4.1-10-5.1-14.8-2.3L380.8 110c-17.9-15.4-38.5-27.3-60.8-35.1V25.8c0-5.6-3.9-10.5-9.4-11.7-36.7-8.2-74.3-7.8-109.2 0-5.5 1.2-9.4 6.1-9.4 11.7V75c-22.2 7.9-42.8 19.8-60.8 35.1L88.7 85.5c-4.9-2.8-11-1.9-14.8 2.3-24.7 26.7-43.6 58.9-54.7 94.6-1.7 5.4.6 11.2 5.5 14L67.3 221c-4.3 23.2-4.3 47 0 70.2l-42.6 24.6c-4.9 2.8-7.1 8.6-5.5 14 11.1 35.6 30 67.8 54.7 94.6 3.8 4.1 10 5.1 14.8 2.3l42.6-24.6c17.9 15.4 38.5 27.3 60.8 35.1v49.2c0 5.6 3.9 10.5 9.4 11.7 36.7 8.2 74.3 7.8 109.2 0 5.5-1.2 9.4-6.1 9.4-11.7v-49.2c22.2-7.9 42.8-19.8 60.8-35.1l42.6 24.6c4.9 2.8 11 1.9 14.8-2.3 24.7-26.7 43.6-58.9 54.7-94.6 1.5-5.5-.7-11.3-5.6-14.1zM256 336c-44.1 0-80-35.9-80-80s35.9-80 80-80 80 35.9 80 80-35.9 80-80 80z" class=""></path></svg></button><ul role="menu" tabindex="-1" class="dropdown-menu" aria-labelledby="__BVID__294__BV_toggle_"> <li role="presentation"><p id="history-op-all-content" class="b-dropdown-text"><span>With entire history...</span></p></li> <li role="presentation"><a data-description="copy datasets" role="menuitem" href="#" target="_self" class="dropdown-item"><span>Copy Datasets</span></a></li> <!----> <!----> <!----></ul></div> <!----> <!----> <!----></section></nav></section> <!----> <!----></section> <section class="position-relative flex-grow-1 scroller"><!----> <div><div><div role="alert" aria-live="polite" aria-atomic="true" class="alert m-2 alert-info"><!----><h4 class="mb-1"><i class="fa fa-info-circle empty-message"></i> <span>This history is empty.</span></h4> <p><a href="#">You can load your own data</a> <span>or</span> <a href="#">get data from an external source</a>.
    </p></div></div></div></section></section></div> <div class="flex-panel-footer d-flex px-2 py-1"><div id="right-drag"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="grip-lines-vertical" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 512" class="svg-inline--fa fa-grip-lines-vertical"><path fill="currentColor" d="M96 496V16c0-8.8-7.2-16-16-16H48c-8.8 0-16 7.2-16 16v480c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16zm128 0V16c0-8.8-7.2-16-16-16h-32c-8.8 0-16 7.2-16 16v480c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16z" class=""></path></svg> <div class="interaction-area cursor-grab"></div></div> <div class="flex-fill"></div> <div class="cursor-pointer"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="chevron-right" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" class="svg-inline--fa fa-chevron-right"><path fill="currentColor" d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z" class=""></path></svg> <div id="right-collapse" class="interaction-area"></div></div></div></div> <!----></div> <!----></div></div> <div id="dd-helper"></div> <!----> <!----> <div id="__BVID__91___BV_modal_outer_" style="position: absolute; z-index: 1040;"><div role="dialog" aria-hidden="true" class="modal fade ui-modal" id="__BVID__91" aria-describedby="__BVID__91___BV_modal_body_" style="display: none;"><div class="modal-dialog modal-md upload-dialog"><!----><div tabindex="-1" class="modal-content" id="__BVID__91___BV_modal_content_"><header class="modal-header no-separator" id="__BVID__91___BV_modal_header_"><h2 tabindex="0" class="title h-sm">Upload from Disk or Web</h2></header><div class="modal-body upload-dialog-body" id="__BVID__91___BV_modal_body_"> <div class="tabs" title="Upload from Disk or Web" modalstatic="true" id="__BVID__309"><!----><div class=""><ul role="tablist" class="nav nav-tabs" id="__BVID__309__BV_tab_controls_"><!----><li role="presentation" class="nav-item"><a id="tab-title-link-regular" role="tab" aria-selected="true" aria-setsize="4" aria-posinset="1" aria-controls="regular" href="#" target="_self" class="nav-link active">Regular</a></li><li role="presentation" class="nav-item"><a id="tab-title-link-composite" role="tab" tabindex="-1" aria-selected="false" aria-setsize="4" aria-posinset="2" aria-controls="composite" href="#" target="_self" class="nav-link">Composite</a></li><li role="presentation" class="nav-item"><a id="tab-title-link-collection" role="tab" tabindex="-1" aria-selected="false" aria-setsize="4" aria-posinset="3" aria-controls="collection" href="#" target="_self" class="nav-link">Collection</a></li><li role="presentation" class="nav-item"><a id="tab-title-link-rule-based" role="tab" tabindex="-1" aria-selected="false" aria-setsize="4" aria-posinset="4" aria-controls="rule-based" href="#" target="_self" class="nav-link">Rule-based</a></li><!----></ul></div><div class="tab-content" id="__BVID__309__BV_tab_container_"><div role="tabpanel" id="regular" aria-hidden="false" aria-labelledby="tab-title-link-regular" class="tab-pane active" style=""><div class="upload-view-default"><div class="upload-top"><div class="upload-top-info">&nbsp;</div></div> <div class="upload-box" style="height: 300px;"><div class="upload-helper"><i class="fa fa-files-o"></i>Drop files here</div> <table class="upload-table ui-table-striped" style="display: none;"><thead><tr><th>Name</th> <th>Size</th> <th>Type</th> <th>Genome</th> <th>Settings</th> <th>Status</th> <th></th></tr></thead> <tbody></tbody></table><input type="file" style="display: none" multiple=""><div class="ui-limitloader" style="display: none;">...only the first 50 entries are visible.</div></div> <div class="upload-footer"><span class="upload-footer-title">Type (set all):</span> <div class="select2-container upload-footer-extension" id="s2id_autogen17"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-18">Auto-detect</span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen18" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-18" id="s2id_autogen18"><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen18_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-18" id="s2id_autogen18_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-18">   </ul></div></div><select tabindex="-1" title="" class="select2-offscreen"><option value="auto">Auto-detect</option><option value="ab1">ab1</option><option value="agilentbrukeryep.d.tar">agilentbrukeryep.d.tar</option><option value="agilentmasshunter.d.tar">agilentmasshunter.d.tar</option><option value="agp">agp</option><option value="ampvis2">ampvis2</option><option value="arb">arb</option><option value="arb.gz">arb.gz</option><option value="arff">arff</option><option value="asn1">asn1</option><option value="asn1-binary">asn1-binary</option><option value="augustus">augustus</option><option value="avi">avi</option><option value="axt">axt</option><option value="bam">bam</option><option value="bcf">bcf</option><option value="bcf_uncompressed">bcf_uncompressed</option><option value="bcsl.model">bcsl.model</option><option value="bcsl.ts">bcsl.ts</option><option value="bed">bed</option><option value="bed12">bed12</option><option value="bed6">bed6</option><option value="bed_tabix.gz">bed_tabix.gz</option><option value="bedgraph">bedgraph</option><option value="bedstrict">bedstrict</option><option value="bgzip">bgzip</option><option value="bigbed">bigbed</option><option value="bigwig">bigwig</option><option value="bil">bil</option><option value="biom1">biom1</option><option value="biom2">biom2</option><option value="blastxml">blastxml</option><option value="blib">blib</option><option value="bref3">bref3</option><option value="brukerbaf.d.tar">brukerbaf.d.tar</option><option value="brukertdf.d.tar">brukertdf.d.tar</option><option value="bus">bus</option><option value="castep">castep</option><option value="cel">cel</option><option value="cell">cell</option><option value="chira.sqlite">chira.sqlite</option><option value="chrint">chrint</option><option value="cif">cif</option><option value="cisml">cisml</option><option value="ckpt">ckpt</option><option value="cmap">cmap</option><option value="cml">cml</option><option value="consensusxml">consensusxml</option><option value="cool">cool</option><option value="cps">cps</option><option value="cpt">cpt</option><option value="cram">cram</option><option value="csfasta">csfasta</option><option value="csv">csv</option><option value="ct">ct</option><option value="ctl.result">ctl.result</option><option value="cuffdiff.sqlite">cuffdiff.sqlite</option><option value="cxb">cxb</option><option value="d3_hierarchy">d3_hierarchy</option><option value="daa">daa</option><option value="dada2_dada">dada2_dada</option><option value="dada2_errorrates">dada2_errorrates</option><option value="dada2_mergepairs">dada2_mergepairs</option><option value="dada2_sequencetable">dada2_sequencetable</option><option value="dada2_uniques">dada2_uniques</option><option value="dbn">dbn</option><option value="dbnsfp.tabular">dbnsfp.tabular</option><option value="dcd">dcd</option><option value="deeptools_compute_matrix_archive">deeptools_compute_matrix_archive</option><option value="deeptools_coverage_matrix">deeptools_coverage_matrix</option><option value="den_fmt">den_fmt</option><option value="dlib">dlib</option><option value="drf">drf</option><option value="dta">dta</option><option value="dta2d">dta2d</option><option value="dzi">dzi</option><option value="edr">edr</option><option value="edta">edta</option><option value="eland">eland</option><option value="elandmulti">elandmulti</option><option value="elib">elib</option><option value="embl">embl</option><option value="encodepeak">encodepeak</option><option value="excel.xls">excel.xls</option><option value="expression.json">expression.json</option><option value="extxyz">extxyz</option><option value="fai">fai</option><option value="fast5">fast5</option><option value="fast5.tar">fast5.tar</option><option value="fast5.tar.bz2">fast5.tar.bz2</option><option value="fast5.tar.gz">fast5.tar.gz</option><option value="fasta">fasta</option><option value="fasta.gz">fasta.gz</option><option value="fastg">fastg</option><option value="fastq">fastq</option><option value="fastq.bz2">fastq.bz2</option><option value="fastq.gz">fastq.gz</option><option value="fastqcssanger">fastqcssanger</option><option value="fastqcssanger.bz2">fastqcssanger.bz2</option><option value="fastqcssanger.gz">fastqcssanger.gz</option><option value="fastqillumina">fastqillumina</option><option value="fastqillumina.bz2">fastqillumina.bz2</option><option value="fastqillumina.gz">fastqillumina.gz</option><option value="fastqsanger">fastqsanger</option><option value="fastqsanger.bz2">fastqsanger.bz2</option><option value="fastqsanger.gz">fastqsanger.gz</option><option value="fastqsolexa">fastqsolexa</option><option value="fastqsolexa.bz2">fastqsolexa.bz2</option><option value="fastqsolexa.gz">fastqsolexa.gz</option><option value="fcs">fcs</option><option value="featurexml">featurexml</option><option value="ffdata">ffdata</option><option value="ffindex">ffindex</option><option value="fits">fits</option><option value="flowclr">flowclr</option><option value="flowframe">flowframe</option><option value="flowmfi">flowmfi</option><option value="flowscore">flowscore</option><option value="flowset">flowset</option><option value="flowstat1">flowstat1</option><option value="flowstat2">flowstat2</option><option value="flowstat3">flowstat3</option><option value="flowtext">flowtext</option><option value="flv">flv</option><option value="fps">fps</option><option value="fqtoc">fqtoc</option><option value="fsom">fsom</option><option value="gafa.sqlite">gafa.sqlite</option><option value="gal">gal</option><option value="gatk_dbsnp">gatk_dbsnp</option><option value="gatk_interval">gatk_interval</option><option value="gatk_recal">gatk_recal</option><option value="gatk_report">gatk_report</option><option value="gatk_tranche">gatk_tranche</option><option value="gemini.sqlite">gemini.sqlite</option><option value="genbank">genbank</option><option value="genbank.gz">genbank.gz</option><option value="genenotebook">genenotebook</option><option value="geojson">geojson</option><option value="gfa1">gfa1</option><option value="gfa1.gz">gfa1.gz</option><option value="gfa2">gfa2</option><option value="gfa2.gz">gfa2.gz</option><option value="gff">gff</option><option value="gff3">gff3</option><option value="gff3.bz2">gff3.bz2</option><option value="gff3.gz">gff3.gz</option><option value="gff_tabix.gz">gff_tabix.gz</option><option value="gii">gii</option><option value="gii.gz">gii.gz</option><option value="gmsh.geo">gmsh.geo</option><option value="gmsh.msh">gmsh.msh</option><option value="gpr">gpr</option><option value="grd">grd</option><option value="grd.tgz">grd.tgz</option><option value="grib">grib</option><option value="gro">gro</option><option value="gtf">gtf</option><option value="h5">h5</option><option value="h5ad">h5ad</option><option value="h5mlm">h5mlm</option><option value="hdf4">hdf4</option><option value="hdr">hdr</option><option value="hdt">hdt</option><option value="hep.root">hep.root</option><option value="hexrd.accepted_orientations">hexrd.accepted_orientations</option><option value="hexrd.eta_ome.npz">hexrd.eta_ome.npz</option><option value="hexrd.images.npz">hexrd.images.npz</option><option value="hexrd.materials.h5">hexrd.materials.h5</option><option value="hexrd.scored_orientations.npz">hexrd.scored_orientations.npz</option><option value="hexrd.yml">hexrd.yml</option><option value="hhr">hhr</option><option value="hic_matrix">hic_matrix</option><option value="hlf">hlf</option><option value="hmm2">hmm2</option><option value="hmm3">hmm3</option><option value="ibd">ibd</option><option value="icm">icm</option><option value="idat">idat</option><option value="idat">idat</option><option value="idpdb">idpdb</option><option value="idxml">idxml</option><option value="imgt.json">imgt.json</option><option value="inchi">inchi</option><option value="inpcrd">inpcrd</option><option value="intermine_tabular">intermine_tabular</option><option value="interval">interval</option><option value="ipynb">ipynb</option><option value="itp">itp</option><option value="jellyfish">jellyfish</option><option value="jp2">jp2</option><option value="jpg">jpg</option><option value="json">json</option><option value="jsonld">jsonld</option><option value="juicer_medium_tabix.gz">juicer_medium_tabix.gz</option><option value="kallisto.idx">kallisto.idx</option><option value="kroenik">kroenik</option><option value="lav">lav</option><option value="len">len</option><option value="loom">loom</option><option value="maf">maf</option><option value="mascotxml">mascotxml</option><option value="maskinfo-asn1">maskinfo-asn1</option><option value="maskinfo-asn1-binary">maskinfo-asn1-binary</option><option value="mat">mat</option><option value="mcool">mcool</option><option value="mdp">mdp</option><option value="memepsp">memepsp</option><option value="memexml">memexml</option><option value="meryldb">meryldb</option><option value="metacyto_clr.txt">metacyto_clr.txt</option><option value="metacyto_stats.txt">metacyto_stats.txt</option><option value="metacyto_summary.txt">metacyto_summary.txt</option><option value="mgf">mgf</option><option value="mkv">mkv</option><option value="mol">mol</option><option value="mol2">mol2</option><option value="mongodb">mongodb</option><option value="mothur.accnos">mothur.accnos</option><option value="mothur.align">mothur.align</option><option value="mothur.align.check">mothur.align.check</option><option value="mothur.align.report">mothur.align.report</option><option value="mothur.axes">mothur.axes</option><option value="mothur.cons.taxonomy">mothur.cons.taxonomy</option><option value="mothur.count_table">mothur.count_table</option><option value="mothur.design">mothur.design</option><option value="mothur.dist">mothur.dist</option><option value="mothur.filter">mothur.filter</option><option value="mothur.filtered.masked.quan">mothur.filtered.masked.quan</option><option value="mothur.filtered.quan">mothur.filtered.quan</option><option value="mothur.freq">mothur.freq</option><option value="mothur.groups">mothur.groups</option><option value="mothur.list">mothur.list</option><option value="mothur.lower.dist">mothur.lower.dist</option><option value="mothur.map">mothur.map</option><option value="mothur.masked.quan">mothur.masked.quan</option><option value="mothur.names">mothur.names</option><option value="mothur.oligos">mothur.oligos</option><option value="mothur.otu">mothur.otu</option><option value="mothur.otu.corr">mothur.otu.corr</option><option value="mothur.otulabels">mothur.otulabels</option><option value="mothur.pair.dist">mothur.pair.dist</option><option value="mothur.quan">mothur.quan</option><option value="mothur.rabund">mothur.rabund</option><option value="mothur.rdp.taxonomy">mothur.rdp.taxonomy</option><option value="mothur.ref.taxonomy">mothur.ref.taxonomy</option><option value="mothur.relabund">mothur.relabund</option><option value="mothur.sabund">mothur.sabund</option><option value="mothur.seq.taxonomy">mothur.seq.taxonomy</option><option value="mothur.sff.flow">mothur.sff.flow</option><option value="mothur.shared">mothur.shared</option><option value="mothur.square.dist">mothur.square.dist</option><option value="mothur.summary">mothur.summary</option><option value="mothur.tax.summary">mothur.tax.summary</option><option value="mothur.tre">mothur.tre</option><option value="mov">mov</option><option value="mp3">mp3</option><option value="mp4">mp4</option><option value="mpg">mpg</option><option value="mrc">mrc</option><option value="mrm">mrm</option><option value="ms2">ms2</option><option value="msh">msh</option><option value="msp">msp</option><option value="mtx">mtx</option><option value="mz.sqlite">mz.sqlite</option><option value="mz5">mz5</option><option value="mzdata">mzdata</option><option value="mzid">mzid</option><option value="mzml">mzml</option><option value="mzq">mzq</option><option value="mzqc">mzqc</option><option value="mztab">mztab</option><option value="mztab2">mztab2</option><option value="mzxml">mzxml</option><option value="n3">n3</option><option value="ncbi_genome_dataset.zip">ncbi_genome_dataset.zip</option><option value="ncbitaxonomy.sqlite">ncbitaxonomy.sqlite</option><option value="ndx">ndx</option><option value="neper.mscell">neper.mscell</option><option value="neper.points">neper.points</option><option value="neper.points.tsv">neper.points.tsv</option><option value="neper.tesr">neper.tesr</option><option value="neper.tess">neper.tess</option><option value="netcdf">netcdf</option><option value="newick">newick</option><option value="nex">nex</option><option value="nhx">nhx</option><option value="nii1">nii1</option><option value="nii1.gz">nii1.gz</option><option value="nii2">nii2</option><option value="nii2.gz">nii2.gz</option><option value="nmrml">nmrml</option><option value="no_unzip.zip">no_unzip.zip</option><option value="npz">npz</option><option value="nt">nt</option><option value="obo">obo</option><option value="odgi">odgi</option><option value="ome.tiff">ome.tiff</option><option value="onnx">onnx</option><option value="osw">osw</option><option value="owl">owl</option><option value="oxlicg">oxlicg</option><option value="oxligl">oxligl</option><option value="oxling">oxling</option><option value="oxliss">oxliss</option><option value="oxlist">oxlist</option><option value="oxlits">oxlits</option><option value="paf">paf</option><option value="paf.gz">paf.gz</option><option value="par">par</option><option value="param">param</option><option value="paramxml">paramxml</option><option value="parquet">parquet</option><option value="pdb">pdb</option><option value="pdbqt">pdbqt</option><option value="pdf">pdf</option><option value="peff">peff</option><option value="peplist">peplist</option><option value="peptideshaker_archive">peptideshaker_archive</option><option value="pepxml">pepxml</option><option value="pepxml.tsv">pepxml.tsv</option><option value="phar">phar</option><option value="phylip">phylip</option><option value="phyloseq">phyloseq</option><option value="phyloxml">phyloxml</option><option value="picard_interval_list">picard_interval_list</option><option value="pileup">pileup</option><option value="pithya.model">pithya.model</option><option value="pithya.property">pithya.property</option><option value="pithya.result">pithya.result</option><option value="plyascii">plyascii</option><option value="plybinary">plybinary</option><option value="png">png</option><option value="postgresql">postgresql</option><option value="pov">pov</option><option value="pqp">pqp</option><option value="pqr">pqr</option><option value="pretext">pretext</option><option value="prmtop">prmtop</option><option value="probam">probam</option><option value="probed">probed</option><option value="protobuf2">protobuf2</option><option value="protobuf3">protobuf3</option><option value="protxml">protxml</option><option value="protxml.tsv">protxml.tsv</option><option value="psl">psl</option><option value="psms">psms</option><option value="pssm-asn1">pssm-asn1</option><option value="ptkscmp">ptkscmp</option><option value="qcml">qcml</option><option value="qcml">qcml</option><option value="qiime2.tabular">qiime2.tabular</option><option value="qname_sorted.bam">qname_sorted.bam</option><option value="qual454">qual454</option><option value="qualillumina">qualillumina</option><option value="qualsolexa">qualsolexa</option><option value="qualsolid">qualsolid</option><option value="qza">qza</option><option value="qzv">qzv</option><option value="rdata">rdata</option><option value="rdata.camera.negative">rdata.camera.negative</option><option value="rdata.camera.positive">rdata.camera.positive</option><option value="rdata.camera.quick">rdata.camera.quick</option><option value="rdata.msnbase.raw">rdata.msnbase.raw</option><option value="rdata.sce">rdata.sce</option><option value="rdata.xcms.fillpeaks">rdata.xcms.fillpeaks</option><option value="rdata.xcms.findchrompeaks">rdata.xcms.findchrompeaks</option><option value="rdata.xcms.group">rdata.xcms.group</option><option value="rdata.xcms.raw">rdata.xcms.raw</option><option value="rdata.xcms.retcor">rdata.xcms.retcor</option><option value="rdf">rdf</option><option value="rdock_as">rdock_as</option><option value="rds">rds</option><option value="rma6">rma6</option><option value="rna_eps">rna_eps</option><option value="sam">sam</option><option value="sbml">sbml</option><option value="scf">scf</option><option value="scidx">scidx</option><option value="scool">scool</option><option value="scool">scool</option><option value="sdf">sdf</option><option value="searchgui_archive">searchgui_archive</option><option value="sf3">sf3</option><option value="sff">sff</option><option value="sif">sif</option><option value="smat">smat</option><option value="smi">smi</option><option value="snaphmm">snaphmm</option><option value="snpeffdb">snpeffdb</option><option value="snpmatrix">snpmatrix</option><option value="snptest">snptest</option><option value="source.c">source.c</option><option value="source.cpp">source.cpp</option><option value="source.cs">source.cs</option><option value="source.go">source.go</option><option value="source.h">source.h</option><option value="source.py">source.py</option><option value="source.rs">source.rs</option><option value="spec.xml">spec.xml</option><option value="splib_noindex">splib_noindex</option><option value="sqlite">sqlite</option><option value="sqmass">sqmass</option><option value="sra">sra</option><option value="sra_manifest.tabular">sra_manifest.tabular</option><option value="star">star</option><option value="stl">stl</option><option value="stockholm">stockholm</option><option value="storm.check">storm.check</option><option value="storm.sample">storm.sample</option><option value="svg">svg</option><option value="tabular">tabular</option><option value="tabular.gz">tabular.gz</option><option value="tandem">tandem</option><option value="tar">tar</option><option value="taxonomy">taxonomy</option><option value="tck">tck</option><option value="textgrid">textgrid</option><option value="tgz">tgz</option><option value="thermo.raw">thermo.raw</option><option value="tiff">tiff</option><option value="toml">toml</option><option value="top">top</option><option value="tpr">tpr</option><option value="trackhub">trackhub</option><option value="trafoxml">trafoxml</option><option value="trafoxml">trafoxml</option><option value="traml">traml</option><option value="trk">trk</option><option value="trr">trr</option><option value="tsv">tsv</option><option value="ttl">ttl</option><option value="twobit">twobit</option><option value="txt">txt</option><option value="uniprotxml">uniprotxml</option><option value="unsorted.bam">unsorted.bam</option><option value="vcf">vcf</option><option value="vcf_bgzip">vcf_bgzip</option><option value="vel">vel</option><option value="vg">vg</option><option value="visium.tar.gz">visium.tar.gz</option><option value="vtkascii">vtkascii</option><option value="vtkbinary">vtkbinary</option><option value="watersmasslynx.raw.tar">watersmasslynx.raw.tar</option><option value="wav">wav</option><option value="wiff.tar">wiff.tar</option><option value="wiff2.tar">wiff2.tar</option><option value="wig">wig</option><option value="xg">xg</option><option value="xgmml">xgmml</option><option value="xlsx">xlsx</option><option value="xmfa">xmfa</option><option value="xml">xml</option><option value="xquest.xml">xquest.xml</option><option value="xsd">xsd</option><option value="xtc">xtc</option><option value="xvg">xvg</option><option value="xyz">xyz</option><option value="yaml">yaml</option><option value="zip">zip</option><option value="zset.geof">zset.geof</option></select> <span class="upload-footer-extension-info upload-icon-button fa fa-search"></span> <span class="upload-footer-title">Genome (set all):</span> <div class="select2-container upload-footer-genome" id="s2id_autogen19"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-20">
                unspecified (?)
            </span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen20" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-20" id="s2id_autogen20"><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen20_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-20" id="s2id_autogen20_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-20">   </ul></div></div><select tabindex="-1" title="" class="select2-offscreen"><option value="?">
                unspecified (?)
            </option><option value="anoGam1">
                A. gambiae Feb. 2003 (IAGEC MOZ2/anoGam1) (anoGam1)
            </option><option value="anoGam3">
                A. gambiae Oct. 2006 (AgamP3/anoGam3) (anoGam3)
            </option><option value="apiMel4">
                A. mellifera 04 Nov 2010 (Amel_4.5/apiMel4) (apiMel4)
            </option><option value="apiMel2">
                A. mellifera Jan. 2005 (Baylor 2.0/apiMel2) (apiMel2)
            </option><option value="apiMel1">
                A. mellifera July 2004 (Baylor 1.2/apiMel1) (apiMel1)
            </option><option value="GCF_001704415.2">
                ARS1 Aug. 2016 Capra hircus (goat) (San Clemente 2016 USDA) (GCF_001704415.2)
            </option><option value="oryAfe1">
                Aardvark May 2012 (OryAfe1.0/oryAfe1) (oryAfe1)
            </option><option value="12997">
                Acaryochloris marina MBIC11017 (12997)
            </option><option value="19259">
                Acholeplasma laidlawii PG-8A (19259)
            </option><option value="acidCryp_JF_5">
                Acidiphilium cryptum JF-5 (acidCryp_JF_5)
            </option><option value="acidBact_ELLIN345">
                Acidobacteria bacterium Ellin345 (acidBact_ELLIN345)
            </option><option value="acidCell_11B">
                Acidothermus cellulolyticus 11B (acidCell_11B)
            </option><option value="15708">
                Acidovorax avenae subsp. citrulli AAC00-1 (15708)
            </option><option value="acidJS42">
                Acidovorax sp. JS42 (acidJS42)
            </option><option value="13001">
                Acinetobacter baumannii (13001)
            </option><option value="28921">
                Acinetobacter baumannii (28921)
            </option><option value="17477">
                Acinetobacter baumannii ATCC 17978 (17477)
            </option><option value="aciBauTYTH_1">
                Acinetobacter baumannii Oct. 2012 (aciBauTYTH_1/aciBauTYTH_1) (aciBauTYTH_1)
            </option><option value="acinSp_ADP1">
                Acinetobacter sp. ADP1 (acinSp_ADP1)
            </option><option value="Actinidia_chinensis_FEI_Lab_v1.0">
                Actinidia chinensis FEI Lab v1.0
            </option><option value="actiPleu_L20">
                Actinobacillus pleuropneumoniae L20 (actiPleu_L20)
            </option><option value="19135">
                Actinobacillus pleuropneumoniae serovar 3 str. JL03 (19135)
            </option><option value="13370">
                Actinobacillus succinogenes 130Z (13370)
            </option><option value="pygAde1">
                Adelie penguin Jun 2014 (ASM69910v1/pygAde1) (pygAde1)
            </option><option value="aeroHydr_ATCC7966">
                Aeromonas hydrophila subsp. hydrophila ATCC 7966 (aeroHydr_ATCC7966)
            </option><option value="16723">
                Aeromonas salmonicida subsp. salmonicida A449 (16723)
            </option><option value="aerPer1">
                Aeropyrum pernix K1 (aerPer1)
            </option><option value="xenLae9.2">
                African Clawed Frog (xenLae9.2)
            </option><option value="xenLae2">
                African clawed frog Aug. 2016 (Xenopus_laevis_v2/xenLae2) (xenLae2)
            </option><option value="afrOth13">
                Afrotheria Apr. 24. 2006 (UCSC Recon/afrOth13) (afrOth13)
            </option><option value="283">
                Agrobacterium tumefaciens str. C58 (283)
            </option><option value="alcaBork_SK2">
                Alcanivorax borkumensis SK2 (alcaBork_SK2)
            </option><option value="alkaEhrl_MLHE_1">
                Alkalilimnicola ehrlichei MLHE-1 (alkaEhrl_MLHE_1)
            </option><option value="13006">
                Alkaliphilus metalliredigens QYMF (13006)
            </option><option value="16083">
                Alkaliphilus oremlandii OhILAs (16083)
            </option><option value="vicPac1">
                Alpaca Jul. 2008 (Broad/vicPac1) (vicPac1)
            </option><option value="vicPac2">
                Alpaca Mar. 2013 (Vicugna_pacos-2.0.1/vicPac2) (vicPac2)
            </option><option value="poeFor1">
                Amazon molly 28 Oct-2013 (Poecilia_formosa-5.1.2/poeFor1) (poeFor1)
            </option><option value="Amborella_trichopoda_Amborella_V1.0">
                Amborella trichopoda Amborella V1.0
            </option><option value="Amborella_trichopoda_JGI_v1.0">
                Amborella trichopoda JGI v1.0
            </option><option value="allMis1">
                American alligator Aug. 2012 (allMis0.2/allMis1) (allMis1)
            </option><option value="allMis0">
                American alligator November 2011 (UCSC aMiss_v0.1c27/allMis0) (allMis0)
            </option><option value="corBra1">
                American crow Jun 2014 (ASM69197v1/corBra1) (corBra1)
            </option><option value="phoRub1">
                American flamingo May 2014 (ASM68726v1/phoRub1) (phoRub1)
            </option><option value="panTig1">
                Amur tiger 06 Sep 2013 (PanTig1.0/panTig1) (panTig1)
            </option><option value="anabVari_ATCC29413">
                Anabaena variabilis ATCC 29413 (anabVari_ATCC29413)
            </option><option value="anaeDeha_2CP_C">
                Anaeromyxobacter dehalogenans 2CP-C (anaeDeha_2CP_C)
            </option><option value="17729">
                Anaeromyxobacter sp. Fw109-5 (17729)
            </option><option value="Ananas_comosus_JGI_v3.0">
                Ananas comosus JGI v3.0
            </option><option value="anapMarg_ST_MARIES">
                Anaplasma marginale str. St. Maries (anapMarg_ST_MARIES)
            </option><option value="336">
                Anaplasma phagocytophilum HZ (336)
            </option><option value="calAnn1">
                Anna's hummingbird Jun 2014 (ASM69908v1/calAnn1) (calAnn1)
            </option><option value="Apostasia_shenzhenica_v1.0">
                Apostasia shenzhenica v1.0
            </option><option value="aquiAeol">
                Aquifex aeolicus VF5 (aquiAeol)
            </option><option value="Araly1">
                Arabidopsis lyrata (Araly1)
            </option><option value="Arabidopsis_lyrata_JGI_v1.0">
                Arabidopsis lyrata JGI v1.0
            </option><option value="Arabidopsis_lyrata_JGI_v2.1">
                Arabidopsis lyrata JGI v2.1
            </option><option value="TAIR10">
                Arabidopsis thaliana
            </option><option value="Arabidopsis_thaliana_COL-0_Araport11">
                Arabidopsis thaliana COL-0 Araport11
            </option><option value="Arabidopsis_thaliana_COL-0_TAIR10">
                Arabidopsis thaliana COL-0 TAIR10
            </option><option value="Arabidopsis_thaliana_TAIR10">
                Arabidopsis thaliana TAIR10 (Arabidopsis_thaliana_TAIR10) (Arabidopsis_thaliana_TAIR10)
            </option><option value="araTha1">
                Arabidopsis thaliana TAIR7 (araTha1)
            </option><option value="arabidopsis_tair8">
                Arabidopsis thaliana TAIR8 (arabidopsis_tair8)
            </option><option value="arabidopsis">
                Arabidopsis thaliana TAIR9 (arabidopsis)
            </option><option value="Arachis_ipaensis_Peanut_Base_v1.0">
                Arachis ipaensis Peanut Base v1.0
            </option><option value="archFulg1">
                Archaeoglobus fulgidus DSM 4304 (archFulg1)
            </option><option value="16319">
                Arcobacter butzleri RM4018 (16319)
            </option><option value="letCam1">
                Arctic lamprey Sep 2013 (LetJap1.0/letCam1) (letCam1)
            </option><option value="dasNov3">
                Armadillo Dec. 2011 (Baylor/dasNov3) (dasNov3)
            </option><option value="dasNov2">
                Armadillo Jul. 2008 (Broad/dasNov2) (dasNov2)
            </option><option value="dasNov1">
                Armadillo May 2005 (Broad/dasNov1) (dasNov1)
            </option><option value="12512">
                Arthrobacter aurescens TC1 (12512)
            </option><option value="arthFB24">
                Arthrobacter sp. FB24 (arthFB24)
            </option><option value="Asparagus_officinalis_v1.1">
                Asparagus officinalis v1.1
            </option><option value="13478">
                Aster yellows witches'-broom phytoplasma AYWB (13478)
            </option><option value="gadMor1">
                Atlantic cod May 2010 (Genofisk GadMor_May2010/gadMor1) (gadMor1)
            </option><option value="13217">
                Azoarcus sp. BH72 (13217)
            </option><option value="azoaSp_EBN1">
                Azoarcus sp. EbN1 (azoaSp_EBN1)
            </option><option value="azorCaul2">
                Azorhizobium caulinodans ORS 571 (azorCaul2)
            </option><option value="bruMal1">
                B. malayi Sep. 2007 (TIGR 1.0/bruMal1) (bruMal1)
            </option><option value="papAnu4">
                Baboon Apr. 2017 (Panu_3.0/papAnu4) (papAnu4)
            </option><option value="papAnu2">
                Baboon Mar. 2012 (Baylor Panu_2.0/papAnu2) (papAnu2)
            </option><option value="papHam1">
                Baboon Nov. 2008 (Baylor Pham_1.0/papHam1) (papHam1)
            </option><option value="13403">
                Bacillus amyloliquefaciens FZB42 (13403)
            </option><option value="10784">
                Bacillus anthracis str. 'Ames Ancestor' (10784)
            </option><option value="baciAnth_AMES">
                Bacillus anthracis str. Ames (baciAnth_AMES)
            </option><option value="10878">
                Bacillus anthracis str. Sterne (10878)
            </option><option value="74">
                Bacillus cereus ATCC 10987 (74)
            </option><option value="384">
                Bacillus cereus ATCC 14579 (384)
            </option><option value="12468">
                Bacillus cereus E33L (12468)
            </option><option value="13624">
                Bacillus cereus subsp. cytotoxis NVH 391-98 (13624)
            </option><option value="13291">
                Bacillus clausii KSM-K16 (13291)
            </option><option value="baciHalo">
                Bacillus halodurans C-125 (baciHalo)
            </option><option value="12388">
                Bacillus licheniformis ATCC 14580 (12388)
            </option><option value="13082">
                Bacillus licheniformis ATCC 14580 (13082)
            </option><option value="20391">
                Bacillus pumilus SAFR-032 (20391)
            </option><option value="baciSubt">
                Bacillus subtilis subsp. subtilis str. 168 (baciSubt)
            </option><option value="10877">
                Bacillus thuringiensis serovar konkukian str. 97-27 (10877)
            </option><option value="18255">
                Bacillus thuringiensis str. Al Hakam (18255)
            </option><option value="13623">
                Bacillus weihenstephanensis KBAB4 (13623)
            </option><option value="46">
                Bacteroides fragilis NCTC 9343 (46)
            </option><option value="13067">
                Bacteroides fragilis YCH46 (13067)
            </option><option value="bactThet_VPI_5482">
                Bacteroides thetaiotaomicron VPI-5482 (bactThet_VPI_5482)
            </option><option value="13378">
                Bacteroides vulgatus ATCC 8482 (13378)
            </option><option value="camFer1">
                Bactrian camel Dec 2011 (CB1/camFer1) (camFer1)
            </option><option value="halLeu1">
                Bald eagle Aug 2014 (Haliaeetus_leucocephalus-4.0/halLeu1) (halLeu1)
            </option><option value="apaVit1">
                Bar tailed trogon 16 Jun 2014 (ASM70340v1/apaVit1) (apaVit1)
            </option><option value="tytAlb1">
                Barn owl May 2014 (ASM68720v1/tytAlb1) (tytAlb1)
            </option><option value="16249">
                Bartonella bacilliformis KC583 (16249)
            </option><option value="bartHens_HOUSTON_1">
                Bartonella henselae str. Houston-1 (bartHens_HOUSTON_1)
            </option><option value="44">
                Bartonella quintana str. Toulouse (44)
            </option><option value="28109">
                Bartonella tribocorum CIP 105476 (28109)
            </option><option value="baumCica_HOMALODISCA">
                Baumannia cicadellinicola str. Hc (Homalodisca coagulata) (baumCica_HOMALODISCA)
            </option><option value="bdelBact">
                Bdellovibrio bacteriovorus HD100 (bdelBact)
            </option><option value="Beta_vulgaris_RefBeet_1.1">
                Beta vulgaris RefBeet 1.1
            </option><option value="Beta_vulgaris_RefBeet_v1.2">
                Beta vulgaris RefBeet v1.2
            </option><option value="stePar1">
                Bicolor damselfish May 2014 (Stegastes_partitus-1.0.2/stePar1) (stePar1)
            </option><option value="16321">
                Bifidobacterium adolescentis ATCC 15703 (16321)
            </option><option value="bifiLong">
                Bifidobacterium longum NCC2705 (bifiLong)
            </option><option value="eptFus1">
                Big brown bat Jul 2012 (EptFus1.0/eptFus1) (eptFus1)
            </option><option value="bisBis1">
                Bison Oct. 2014 (Bison_UMD1.0/bisBis1) (bisBis1)
            </option><option value="pteAle1">
                Black flying-fox Aug 2012 (ASM32557v1/pteAle1) (pteAle1)
            </option><option value="notCor1">
                Black rockcod Jul 2014 (NC01/notCor1) (notCor1)
            </option><option value="panPan2">
                Bonobo Aug. 2015 (MPI-EVA panpan1.1/panPan2) (panPan2)
            </option><option value="panPan1">
                Bonobo May 2012 (Max-Planck/panPan1) (panPan1)
            </option><option value="panPan3">
                Bonobo May 2020 (Mhudiblu_PPA_v0/panPan3) (panPan3)
            </option><option value="bordBron">
                Bordetella bronchiseptica RB50 (bordBron)
            </option><option value="25">
                Bordetella parapertussis 12822 (25)
            </option><option value="26">
                Bordetella pertussis Tohama I (26)
            </option><option value="28135">
                Bordetella petrii DSM 12804 (28135)
            </option><option value="borEut13">
                Boreoeutherian Apr. 24. 2006 (UCSC Recon/borEut13) (borEut13)
            </option><option value="canHg12">
                Boreoeutherian Nov. 19. 2005 (UCSC Recon/canHg12) (canHg12)
            </option><option value="17057">
                Borrelia afzelii PKo (17057)
            </option><option value="borrBurg">
                Borrelia burgdorferi B31 (borrBurg)
            </option><option value="12554">
                Borrelia garinii PBi (12554)
            </option><option value="Brachypodium_distachyon_JGI_v3.1">
                Brachypodium distachyon JGI v3.1
            </option><option value="Brachypodium_distachyon_MIPS_1.2">
                Brachypodium distachyon MIPS 1.2
            </option><option value="bradJapo">
                Bradyrhizobium japonicum USDA 110 (bradJapo)
            </option><option value="16137">
                Bradyrhizobium sp. BTAi1 (16137)
            </option><option value="19575">
                Bradyrhizobium sp. ORS278 (19575)
            </option><option value="myoBra1">
                Brandt's myotis (bat) 28 Jun 2013 (ASM41265v1/myoBra1) (myoBra1)
            </option><option value="braNey1">
                Braney test Jul. 2013 (braNey1/braNey1) (braNey1)
            </option><option value="Brassica_oleracea_Ensembl_Genomes_v2.1">
                Brassica oleracea Ensembl Genomes v2.1
            </option><option value="Brassica_rapa_Annotation_v1.2_on_assembly_v1.1_Phytozome_9.1">
                Brassica rapa Annotation v1.2 on assembly v1.1 (Phytozome 9.1)
            </option><option value="Brassica_rapa_JGI_FPsc1.3">
                Brassica rapa JGI FPsc1.3
            </option><option value="aptMan1">
                Brown kiwi Jun. 2015 (MPI-EVA AptMant0/aptMan1) (aptMan1)
            </option><option value="mesUni1">
                Brown roatelo Jun 2014 (ASM69576v1/mesUni1) (mesUni1)
            </option><option value="9619">
                Brucella abortus biovar 1 str. 9-941 (9619)
            </option><option value="20243">
                Brucella canis ATCC 23365 (20243)
            </option><option value="brucMeli">
                Brucella melitensis 16M (brucMeli)
            </option><option value="16203">
                Brucella melitensis biovar Abortus 2308 (16203)
            </option><option value="12514">
                Brucella ovis ATCC 25840 (12514)
            </option><option value="320">
                Brucella suis 1330 (320)
            </option><option value="20371">
                Brucella suis ATCC 23445 (20371)
            </option><option value="octDeg1">
                Brush-tailed rat Apr 2012 (OctDeg1.0/octDeg1) (octDeg1)
            </option><option value="buchSp">
                Buchnera aphidicola str. APS (Acyrthosiphon pisum) (buchSp)
            </option><option value="256">
                Buchnera aphidicola str. Bp (Baizongia pistaciae) (256)
            </option><option value="16372">
                Buchnera aphidicola str. Cc (Cinara cedri) (16372)
            </option><option value="312">
                Buchnera aphidicola str. Sg (Schizaphis graminum) (312)
            </option><option value="melUnd1">
                Budgerigar Sep. 2011 (WUSTL v6.3/melUnd1) (melUnd1)
            </option><option value="bunEbo1">
                Bundibugyo ebolavirus Aug. 2014 (NC_014373.1/bunEbo1) (bunEbo1)
            </option><option value="burkCepa_AMMD">
                Burkholderia ambifaria AMMD (burkCepa_AMMD)
            </option><option value="burkCeno_AU_1054">
                Burkholderia cenocepacia AU 1054 (burkCeno_AU_1054)
            </option><option value="burkCeno_HI2424">
                Burkholderia cenocepacia HI2424 (burkCeno_HI2424)
            </option><option value="17929">
                Burkholderia cenocepacia MC0-3 (17929)
            </option><option value="burkMall_ATCC23344">
                Burkholderia mallei ATCC 23344 (burkMall_ATCC23344)
            </option><option value="13943">
                Burkholderia mallei NCTC 10229 (13943)
            </option><option value="13946">
                Burkholderia mallei NCTC 10247 (13946)
            </option><option value="13947">
                Burkholderia mallei SAVP1 (13947)
            </option><option value="17407">
                Burkholderia multivorans ATCC 17616 (17407)
            </option><option value="burkPseu_1106A">
                Burkholderia pseudomallei 1106a (burkPseu_1106A)
            </option><option value="13954">
                Burkholderia pseudomallei 1710b (13954)
            </option><option value="13953">
                Burkholderia pseudomallei 668 (13953)
            </option><option value="178">
                Burkholderia pseudomallei K96243 (178)
            </option><option value="burk383">
                Burkholderia sp. 383 (burk383)
            </option><option value="burkThai_E264">
                Burkholderia thailandensis E264 (burkThai_E264)
            </option><option value="burkViet_G4">
                Burkholderia vietnamiensis G4 (burkViet_G4)
            </option><option value="burkXeno_LB400">
                Burkholderia xenovorans LB400 (burkXeno_LB400)
            </option><option value="pytBiv1">
                Burmese python Sep. 2013 (Python_molurus_bivittatus-5.0.2/pytBiv1) (pytBiv1)
            </option><option value="hapBur1">
                Burton's mouthbreeder Oct 2011 (AstBur1.0/hapBur1) (hapBur1)
            </option><option value="otoGar1">
                Bushbaby Dec. 2006 (Broad/otoGar1) (otoGar1)
            </option><option value="otoGar3">
                Bushbaby Mar. 2011 (Broad/otoGar3) (otoGar3)
            </option><option value="caeAng1">
                C. angaria Oct. 2010 (WS225/caeAng1) (caeAng1)
            </option><option value="caePb2">
                C. brenneri Feb. 2008 (WUGSC 6.0.1/caePb2) (caePb2)
            </option><option value="caePb1">
                C. brenneri Jan. 2007 (WUGSC 4.0/caePb1) (caePb1)
            </option><option value="caePb3">
                C. brenneri Nov. 2010 (C. brenneri 6.0.1b/caePb3) (caePb3)
            </option><option value="cb4">
                C. briggsae Apr. 2011 (WS225/cb4) (cb4)
            </option><option value="cb2">
                C. briggsae Aug 2005 (WUGSC prelim/cb2) (cb2)
            </option><option value="cb3">
                C. briggsae Jan. 2007 (WUGSC 1.0/cb3) (cb3)
            </option><option value="cb1">
                C. briggsae July 2002 (WormBase cb25.agp8/cb1) (cb1)
            </option><option value="ce5">
                C. elegans Aug. 2007 (WS180/ce5) (ce5)
            </option><option value="ce7">
                C. elegans Feb 2009 (WS200/ce7) (ce7)
            </option><option value="ce11">
                C. elegans Feb. 2013 (WBcel235/ce11) (ce11)
            </option><option value="ce9">
                C. elegans Jan 2010 (WS210/ce9) (ce9)
            </option><option value="ce4">
                C. elegans Jan. 2007 (WS170/ce4) (ce4)
            </option><option value="ce8">
                C. elegans Jun 2009 (WS204/ce8) (ce8)
            </option><option value="ce2">
                C. elegans Mar. 2004 (WS120/ce2) (ce2)
            </option><option value="ce3">
                C. elegans March 2005 (WS140/ce3) (ce3)
            </option><option value="ce6">
                C. elegans May 2008 (WS190/ce6) (ce6)
            </option><option value="ce10">
                C. elegans Oct. 2010 (WS220/ce10) (ce10)
            </option><option value="ci3">
                C. intestinalis Apr. 2011 (Kyoto KH/ci3) (ci3)
            </option><option value="ci1">
                C. intestinalis Dec. 2002 (JGI 1.0/ci1) (ci1)
            </option><option value="ci2">
                C. intestinalis Mar. 2005 (JGI 2.1/ci2) (ci2)
            </option><option value="caeJap3">
                C. japonica  Sep. 2010 (WUGSC 7.0.1/caeJap3) (caeJap3)
            </option><option value="caeJap4">
                C. japonica Aug. 2010 (WUSTL 7.0.1/caeJap4) (caeJap4)
            </option><option value="caeJap2">
                C. japonica Jan. 2009 (WUGSC 4.0.1/caeJap2) (caeJap2)
            </option><option value="caeJap1">
                C. japonica Mar. 2008 (WUGSC 3.0.2/caeJap1) (caeJap1)
            </option><option value="caeRem4">
                C. remanei Jul. 2007 (WS220/caeRem4) (caeRem4)
            </option><option value="caeRem2">
                C. remanei Mar. 2006 (WUGSC 1.0/caeRem2) (caeRem2)
            </option><option value="caeRem1">
                C. remanei March 2005 (WUGSC prelim/caeRem1) (caeRem1)
            </option><option value="caeRem3">
                C. remanei May 2007 (WUGSC 15.0.1/caeRem3) (caeRem3)
            </option><option value="cioSav1">
                C. savignyi Apr. 2003 (Broad/cioSav1) (cioSav1)
            </option><option value="cioSav2">
                C. savignyi July 2005 (Sidow Lab 2.0/cioSav2) (cioSav2)
            </option><option value="caeSp111">
                C. sp. 11 ju1373 Nov. 2010 (WUSTL 3.0.1/caeSp111) (caeSp111)
            </option><option value="caeSp71">
                C. sp. 7 ju1286 Nov. 2010 (WUSTL 4.0.1/caeSp71) (caeSp71)
            </option><option value="caeSp91">
                C. sp. 9 ju1422 Nov. 2010 (WUSTL 3.0.1/caeSp91) (caeSp91)
            </option><option value="cacTus3">
                Cactus Nov. 2010 (CactusV3/cacTus3) (cacTus3)
            </option><option value="cacTus2">
                Cactus Oct. 2010 (Ngan10.1/cacTus2) (cacTus2)
            </option><option value="cacTus1">
                Cactus Sep. 2010 (CactusV1/cacTus1) (cacTus1)
            </option><option value="Cajanus_cajan_Penguin_Genomics_v5.0">
                Cajanus cajan Penguin Genomics v5.0
            </option><option value="caldSacc_DSM8903">
                Caldicellulosiruptor saccharolyticus DSM 8903 (caldSacc_DSM8903)
            </option><option value="caldMaqu1">
                Caldivirga maquilingensis IC-167 (caldMaqu1)
            </option><option value="17159">
                Campylobacter concisus 13826 (17159)
            </option><option value="17161">
                Campylobacter curvus 525.92 (17161)
            </option><option value="campFetu_82_40">
                Campylobacter fetus subsp. fetus 82-40 (campFetu_82_40)
            </option><option value="20083">
                Campylobacter hominis ATCC BAA-381 (20083)
            </option><option value="campJeju_RM1221">
                Campylobacter jejuni RM1221 (campJeju_RM1221)
            </option><option value="17163">
                Campylobacter jejuni subsp. doylei 269.97 (17163)
            </option><option value="campJeju_81_176">
                Campylobacter jejuni subsp. jejuni 81-176 (campJeju_81_176)
            </option><option value="17953">
                Campylobacter jejuni subsp. jejuni 81116 (17953)
            </option><option value="campJeju">
                Campylobacter jejuni subsp. jejuni NCTC 11168 (campJeju)
            </option><option value="serCan1">
                Canary 15 Jan-2014 (SCA1/serCan1) (serCan1)
            </option><option value="blocFlor">
                Candidatus Blochmannia floridanus (blocFlor)
            </option><option value="13875">
                Candidatus Blochmannia pennsylvanicus str. BPEN (13875)
            </option><option value="candCars_RUDDII">
                Candidatus Carsonella ruddii PV (candCars_RUDDII)
            </option><option value="21047">
                Candidatus Desulforudis audaxviator MP104C (21047)
            </option><option value="16525">
                Candidatus Korarchaeum cryptofilum OPF8 (16525)
            </option><option value="methBoon1">
                Candidatus Methanoregula boonei 6A8 (methBoon1)
            </option><option value="candPela_UBIQUE_HTCC1">
                Candidatus Pelagibacter ubique HTCC1062 (candPela_UBIQUE_HTCC1)
            </option><option value="paraSp_UWE25">
                Candidatus Protochlamydia amoebophila UWE25 (paraSp_UWE25)
            </option><option value="16841">
                Candidatus Ruthia magnifica str. Cm (Calyptogena magnifica) (16841)
            </option><option value="19617">
                Candidatus Sulcia muelleri GWSS (19617)
            </option><option value="18267">
                Candidatus Vesicomyosocius okutanii HA (18267)
            </option><option value="eleEdw1">
                Cape elephant shrew Aug 2012 (EleEdw1.0/eleEdw1) (eleEdw1)
            </option><option value="chrAsi1">
                Cape golden mole Aug 2012 (ChrAsi1.0/chrAsi1) (chrAsi1)
            </option><option value="Capsella_rubella_JGI_annotation_v1.0_on_assembly_v1">
                Capsella rubella JGI annotation v1.0 on assembly v1
            </option><option value="Capsella_rubella_JGI_v1.0">
                Capsella rubella JGI v1.0
            </option><option value="Capsicum_annuum_Pepper_Genome_v2.0">
                Capsicum annuum Pepper Genome v2.0
            </option><option value="carbHydr_Z_2901">
                Carboxydothermus hydrogenoformans Z-2901 (carbHydr_Z_2901)
            </option><option value="Carica_papaya_Hawaii_Agriculture_Research_Center">
                Carica papaya Hawaii Agriculture Research Center
            </option><option value="Carica_papaya_JGI_ASGPB0.4">
                Carica papaya JGI ASGPB0.4
            </option><option value="felCat4">
                Cat Dec. 2008 (NHGRI/GTB V17e/felCat4) (felCat4)
            </option><option value="felCat3">
                Cat Mar. 2006 (Broad/felCat3) (felCat3)
            </option><option value="felCat8">
                Cat Nov. 2014 (ICGSC Felis_catus_8.0/felCat8) (felCat8)
            </option><option value="felCat9">
                Cat Nov. 2017 (Felis_catus_9.0/felCat9) (felCat9)
            </option><option value="felCat5">
                Cat Sep. 2011 (ICGSC Felis_catus 6.2/felCat5) (felCat5)
            </option><option value="catArr1">
                Catarrhini June 13. 2006 (UCSC Recon/catArr1) (catArr1)
            </option><option value="caulCres">
                Caulobacter crescentus CB15 (caulCres)
            </option><option value="16306">
                Caulobacter sp. K31 (16306)
            </option><option value="Cenchrus_americanus_v1.0">
                Cenchrus americanus v1.0
            </option><option value="centromeres1">
                CentromereY 2013 GrCh38 (centromeres1)
            </option><option value="Chenopodium_quinoa_JGI_v1.0">
                Chenopodium quinoa JGI v1.0
            </option><option value="galGal5">
                Chicken Dec 2015 (Gallus_gallus-5.0/galGal5) (galGal5)
            </option><option value="galGal2">
                Chicken Feb. 2004 (WUGSC 1.0/galGal2) (galGal2)
            </option><option value="galGal6">
                Chicken Mar. 2018 (GRCg6a/galGal6) (galGal6)
            </option><option value="galGal3">
                Chicken May 2006 (WUGSC 2.1/galGal3) (galGal3)
            </option><option value="galGal4">
                Chicken Nov. 2011 (ICGSC Gallus_gallus-4.0/galGal4) (galGal4)
            </option><option value="GCF_016700215.2">
                Chicken Oct. 2021 (bGalGal1.pat.whiteleghornlayer.GRCg7w_WZ) (GCF_016700215.2)
            </option><option value="panTro4">
                Chimp Feb. 2011 (CSAC 2.1.4/panTro4) (panTro4)
            </option><option value="panTro6">
                Chimp Jan. 2018 (Clint_PTRv2/panTro6) (panTro6)
            </option><option value="panTro2">
                Chimp Mar. 2006 (CGSC 2.1/panTro2) (panTro2)
            </option><option value="panTro5">
                Chimp May 2016 (Pan_tro 3.0/panTro5) (panTro5)
            </option><option value="panTro1">
                Chimp Nov. 2003 (CGSC 1.1/panTro1) (panTro1)
            </option><option value="panTro3">
                Chimp Oct. 2010 (CGSC 2.1.3/panTro3) (panTro3)
            </option><option value="chiLan1">
                Chinchilla May 2012  (ChiLan1.0/chiLan1) (chiLan1)
            </option><option value="allSin1">
                Chinese alligator 29 Aug 2013 (ASM45574v1/allSin1) (allSin1)
            </option><option value="criGriChoV1">
                Chinese hamster Aug. 2011 (CHO K1 cell line/criGriChoV1) (criGriChoV1)
            </option><option value="criGri1">
                Chinese hamster Jul. 2013 (C_griseus_v1.0/criGri1) (criGri1)
            </option><option value="criGriChoV2">
                Chinese hamster Jun. 2017 (CHOK1S_HZDv1/criGriChoV2) (criGriChoV2)
            </option><option value="manPen1">
                Chinese pangolin Aug 2014 (M_pentadactyla-1.1.1/manPen1) (manPen1)
            </option><option value="pelSin1">
                Chinese softshell turtle Oct 2011 (PelSin_1.0/pelSin1) (pelSin1)
            </option><option value="tupChi1">
                Chinese tree shrew Jan 2013 (TupChi_1.0/tupChi1) (tupChi1)
            </option><option value="229">
                Chlamydia muridarum Nigg (229)
            </option><option value="28583">
                Chlamydia trachomatis 434/Bu (28583)
            </option><option value="13885">
                Chlamydia trachomatis A/HAR-13 (13885)
            </option><option value="chlaTrac">
                Chlamydia trachomatis D/UW-3/CX (chlaTrac)
            </option><option value="28585">
                Chlamydia trachomatis L2b/UCH-1/proctitis (28585)
            </option><option value="Chlamydomonas_reinhardtii_JGI_v5.0_assembly_annot_v5.3.1_based_on_Augustus_u11.6">
                Chlamydomonas reinhardtii JGI v5.0 assembly, annot v5.3.1 based on Augustus u11.6
            </option><option value="Chlamydomonas_reinhardtii_JGI_v5.5">
                Chlamydomonas reinhardtii JGI v5.5
            </option><option value="355">
                Chlamydophila abortus S26/3 (355)
            </option><option value="228">
                Chlamydophila caviae GPIC (228)
            </option><option value="370">
                Chlamydophila felis Fe/C-56 (370)
            </option><option value="247">
                Chlamydophila pneumoniae AR39 (247)
            </option><option value="chlaPneu_CWL029">
                Chlamydophila pneumoniae CWL029 (chlaPneu_CWL029)
            </option><option value="257">
                Chlamydophila pneumoniae J138 (257)
            </option><option value="420">
                Chlamydophila pneumoniae TW-183 (420)
            </option><option value="chloChlo_CAD3">
                Chlorobium chlorochromatii CaD3 (chloChlo_CAD3)
            </option><option value="12609">
                Chlorobium phaeobacteroides DSM 266 (12609)
            </option><option value="chloTepi_TLS">
                Chlorobium tepidum TLS (chloTepi_TLS)
            </option><option value="59">
                Chloroflexus aurantiacus J-10-fl (59)
            </option><option value="chroViol">
                Chromobacterium violaceum ATCC 12472 (chroViol)
            </option><option value="chroSale_DSM3043">
                Chromohalobacter salexigens DSM 3043 (chroSale_DSM3043)
            </option><option value="capCar1">
                Chuck-will's-widow Jun 2014 (ASM70074v1/capCar1) (capCar1)
            </option><option value="Cicer_arietinum_LIS_v1.0">
                Cicer arietinum LIS v1.0
            </option><option value="12716">
                Citrobacter koseri ATCC BAA-895 (12716)
            </option><option value="Citrullus_lanatus_Cucurbit_Genomics_Database">
                Citrullus lanatus Cucurbit Genomics Database
            </option><option value="Citrullus_lanatus_Cucurbit_Genomics_Database_v1.0">
                Citrullus lanatus Cucurbit Genomics Database v1.0
            </option><option value="Citrus_clementina_JGI_v1.0">
                Citrus clementina JGI v1.0
            </option><option value="Citrus_sinensis_JGI_v1_assembly_and_v1.0_annotation">
                Citrus sinensis JGI v1 assembly and v1.0 annotation
            </option><option value="clavMich_NCPPB_382">
                Clavibacter michiganensis subsp. michiganensis NCPPB 382 (clavMich_NCPPB_382)
            </option><option value="184">
                Clavibacter michiganensis subsp. sepedonicus (184)
            </option><option value="77">
                Clostridium acetobutylicum ATCC 824 (77)
            </option><option value="12637">
                Clostridium beijerinckii NCIMB 8052 (12637)
            </option><option value="19517">
                Clostridium botulinum A str. ATCC 19397 (19517)
            </option><option value="193">
                Clostridium botulinum A str. ATCC 3502 (193)
            </option><option value="19521">
                Clostridium botulinum A str. Hall (19521)
            </option><option value="28507">
                Clostridium botulinum A3 str. Loch Maree (28507)
            </option><option value="28505">
                Clostridium botulinum B1 str. Okra (28505)
            </option><option value="19519">
                Clostridium botulinum F str. Langeland (19519)
            </option><option value="78">
                Clostridium difficile 630 (78)
            </option><option value="19065">
                Clostridium kluyveri DSM 555 (19065)
            </option><option value="16820">
                Clostridium novyi NT (16820)
            </option><option value="304">
                Clostridium perfringens ATCC 13124 (304)
            </option><option value="12521">
                Clostridium perfringens SM101 (12521)
            </option><option value="79">
                Clostridium perfringens str. 13 (79)
            </option><option value="16184">
                Clostridium phytofermentans ISDg (16184)
            </option><option value="81">
                Clostridium tetani E88 (81)
            </option><option value="314">
                Clostridium thermocellum ATCC 27405 (314)
            </option><option value="latCha1">
                Coelacanth Aug. 2011 (Broad/latCha1) (latCha1)
            </option><option value="Coffea_canephora_Coffee_Genome_Hub_v1">
                Coffea canephora Coffee Genome Hub v1
            </option><option value="ficAlb2">
                Collared flycatcher Jun 2013 (FicAlb1.5/ficAlb2) (ficAlb2)
            </option><option value="colwPsyc_34H">
                Colwellia psychrerythraea 34H (colwPsyc_34H)
            </option><option value="cucCan1">
                Common cuckoo Jun 2014 (ASM70932v1/cucCan1) (cucCan1)
            </option><option value="Corchorus_olitorius_GenBank_v1.0">
                Corchorus olitorius GenBank v1.0
            </option><option value="87">
                Corynebacterium diphtheriae NCTC 13129 (87)
            </option><option value="coryEffi_YS_314">
                Corynebacterium efficiens YS-314 (coryEffi_YS_314)
            </option><option value="13760">
                Corynebacterium glutamicum ATCC 13032 (13760)
            </option><option value="307">
                Corynebacterium glutamicum ATCC 13032 (307)
            </option><option value="19193">
                Corynebacterium glutamicum R (19193)
            </option><option value="13967">
                Corynebacterium jeikeium K411 (13967)
            </option><option value="bosTau5">
                Cow Apr. 2009 (Baylor Btau_4.2/bosTau5) (bosTau5)
            </option><option value="bosTau9">
                Cow Apr. 2018 (ARS-UCD1.2/bosTau9) (bosTau9)
            </option><option value="bosTau3">
                Cow Aug. 2006 (Baylor 3.1/bosTau3) (bosTau3)
            </option><option value="bosTauMd3">
                Cow Aug. 2009 (UMD3.1) (bosTauMd3)
            </option><option value="bosTau8">
                Cow Jun. 2014 (Bos_taurus_UMD_3.1.1/bosTau8) (bosTau8)
            </option><option value="bosTau2">
                Cow Mar. 2005 (Baylor 2.0/bosTau2) (bosTau2)
            </option><option value="bosTau6">
                Cow Nov. 2009 (Bos_taurus_UMD_3.1/bosTau6) (bosTau6)
            </option><option value="bosTau4">
                Cow Oct. 2007 (Baylor 4.0/bosTau4) (bosTau4)
            </option><option value="bosTau7">
                Cow Oct. 2011 (Baylor Btau_4.6.1/bosTau7) (bosTau7)
            </option><option value="16721">
                Coxiella burnetii Dugway 5J108-111 (16721)
            </option><option value="16791">
                Coxiella burnetii RSA 331 (16791)
            </option><option value="coxiBurn">
                Coxiella burnetii RSA 493 (coxiBurn)
            </option><option value="macFas5">
                Crab-eating macaque Jun. 2013 (Macaca_fascicularis_5.0/macFas5) (macFas5)
            </option><option value="nipNip1">
                Crested ibis Jun 2014 (ASM70822v1/nipNip1) (nipNip1)
            </option><option value="balPav1">
                Crowned crain Jun 2014 (ASM70989v1/balPav1) (balPav1)
            </option><option value="lepDis1">
                Cuckoo roller May 2014 (ASM69178v1/lepDis1) (lepDis1)
            </option><option value="Cucumis_melo_Melonomics_v3.5">
                Cucumis melo Melonomics v3.5
            </option><option value="Cucumis_sativus_L._JGI_v1.0">
                Cucumis sativus L. JGI v1.0
            </option><option value="cytoHutc_ATCC33406">
                Cytophaga hutchinsonii ATCC 33406 (cytoHutc_ATCC33406)
            </option><option value="droAlb1">
                D. albomicans 21 May 2012 (DroAlb_1.0/droAlb1) (droAlb1)
            </option><option value="droAna2">
                D. ananassae Aug. 2005 (Agencourt prelim/droAna2) (droAna2)
            </option><option value="droAna3">
                D. ananassae Feb. 2006 (droAna3)
            </option><option value="droAna1">
                D. ananassae July 2004 (TIGR/droAna1) (droAna1)
            </option><option value="droBia2">
                D. biarmipes 04 Mar 2013 (Dbia_2.0/droBia2) (droBia2)
            </option><option value="droBip2">
                D. bipectinata 04 Mar 2013 (Dbip_2.0/droBip2) (droBip2)
            </option><option value="droEle2">
                D. elegans 04 Mar 2013 (Dele_2.0/droEle2) (droEle2)
            </option><option value="droEre1">
                D. erecta Aug. 2005 (Agencourt prelim/droEre1) (droEre1)
            </option><option value="droEre2">
                D. erecta Feb. 2006 (droEre2)
            </option><option value="droEug2">
                D. eugracilis 04 Mar 2013 (Deug_2.0/droEug2) (droEug2)
            </option><option value="droFic2">
                D. ficusphila 04 Mar 2013 (Dfic_2.0/droFic2) (droFic2)
            </option><option value="droGri1">
                D. grimshawi Aug. 2005 (Agencourt prelim/droGri1) (droGri1)
            </option><option value="droGri2">
                D. grimshawi Feb. 2006 (droGri2)
            </option><option value="droKik2">
                D. kikkawai 04 Mar 2013 (Dkik_2.0/droKik2) (droKik2)
            </option><option value="dm2">
                D. melanogaster Apr. 2004 (BDGP R4/dm2) (dm2)
            </option><option value="dm3">
                D. melanogaster Apr. 2006 (BDGP R5/dm3) (dm3)
            </option><option value="dm6">
                D. melanogaster Aug. 2014 (BDGP Release 6 + ISO1 MT/dm6) (dm6)
            </option><option value="dm1">
                D. melanogaster Jan. 2003 (BDGP R3/dm1) (dm1)
            </option><option value="droMir2">
                D. miranda 19 Apr 2013 (DroMir_2.2/droMir2) (droMir2)
            </option><option value="droMoj1">
                D. mojavensis Aug. 2004 (Agencourt prelim/droMoj1) (droMoj1)
            </option><option value="droMoj2">
                D. mojavensis Aug. 2005 (Agencourt prelim/droMoj2) (droMoj2)
            </option><option value="droMoj3">
                D. mojavensis Feb. 2006 (droMoj3)
            </option><option value="droPer1">
                D. persimilis Oct. 2005 (Broad/droPer1) (droPer1)
            </option><option value="dp4">
                D. pseudoobscura (dp4) (dp4)
            </option><option value="droPse3">
                D. pseudoobscura 11 Apr 2013 (Dpse_3.0/droPse3) (droPse3)
            </option><option value="dp2">
                D. pseudoobscura Aug. 2003 (Baylor freeze1/dp2) (dp2)
            </option><option value="dp3">
                D. pseudoobscura Nov. 2004 (FlyBase 1.03/dp3) (dp3)
            </option><option value="droRho2">
                D. rhopaloa 22 Feb 2013 (Drho_2.0/droRho2) (droRho2)
            </option><option value="droSec1">
                D. sechellia Oct. 2005 (Broad/droSec1) (droSec1)
            </option><option value="droSim1">
                D. simulans Apr. 2005 (WUGSC mosaic 1.0/droSim1) (droSim1)
            </option><option value="droSuz1">
                D. suzukii 30 Sep 2013 (Dsuzukii.v01/droSuz1) (droSuz1)
            </option><option value="droTak2">
                D. takahashii 04 Mar 2013 (Dtak_2.0/droTak2) (droTak2)
            </option><option value="droVir2">
                D. virilis Aug. 2005 (Agencourt prelim/droVir2) (droVir2)
            </option><option value="droVir3">
                D. virilis Feb. 2006 (droVir3)
            </option><option value="droVir1">
                D. virilis July 2004 (Agencourt prelim/droVir1) (droVir1)
            </option><option value="droWil2">
                D. willistoni 03 Aug 2006 (dwil_caf1/droWil2) (droWil2)
            </option><option value="droWil1">
                D. willistoni Feb. 2006 (droWil1)
            </option><option value="droYak3">
                D. yakuba 27 Jun 2006 (dyak_caf1/droYak3) (droYak3)
            </option><option value="droYak1">
                D. yakuba Apr. 2004 (WUGSC 1.0/droYak1) (droYak1)
            </option><option value="droYak2">
                D. yakuba Nov. 2005 (WUGSC 7.1/droYak2) (droYak2)
            </option><option value="pelCri1">
                Dalmatian pelican May 2014 (ASM68737v1/pelCri1) (pelCri1)
            </option><option value="Daucus_carota_JGI_v2.0">
                Daucus carota JGI v2.0
            </option><option value="myoDav1">
                David's myotis (bat) Aug 2012 (ASM32734v1/myoDav1) (myoDav1)
            </option><option value="dechArom_RCB">
                Dechloromonas aromatica RCB (dechArom_RCB)
            </option><option value="IscaW1">
                Deer Tick (Ixodes scapularis) (IscaW1)
            </option><option value="dehaEthe_195">
                Dehalococcoides ethenogenes 195 (dehaEthe_195)
            </option><option value="15770">
                Dehalococcoides sp. BAV1 (15770)
            </option><option value="15604">
                Dehalococcoides sp. CBDB1 (15604)
            </option><option value="deinGeot_DSM11300">
                Deinococcus geothermalis DSM 11300 (deinGeot_DSM11300)
            </option><option value="deinRadi">
                Deinococcus radiodurans R1 (deinRadi)
            </option><option value="17413">
                Delftia acidovorans SPH-1 (17413)
            </option><option value="desuHafn_Y51">
                Desulfitobacterium hafniense Y51 (desuHafn_Y51)
            </option><option value="18007">
                Desulfococcus oleovorans Hxd3 (18007)
            </option><option value="desuPsyc_LSV54">
                Desulfotalea psychrophila LSv54 (desuPsyc_LSV54)
            </option><option value="desuRedu_MI_1">
                Desulfotomaculum reducens MI-1 (desuRedu_MI_1)
            </option><option value="329">
                Desulfovibrio desulfuricans G20 (329)
            </option><option value="17227">
                Desulfovibrio vulgaris subsp. vulgaris DP4 (17227)
            </option><option value="desuVulg_HILDENBOROUG">
                Desulfovibrio vulgaris subsp. vulgaris str. Hildenborough (desuVulg_HILDENBOROUG)
            </option><option value="dichNodo_VCS1703A">
                Dichelobacter nodosus VCS1703A (dichNodo_VCS1703A)
            </option><option value="17417">
                Dinoroseobacter shibae DFL 12 (17417)
            </option><option value="canFam1">
                Dog July 2004 (Broad/canFam1) (canFam1)
            </option><option value="canFam4">
                Dog Mar. 2020 (UU_Cfam_GSD_1.0/canFam4) (canFam4)
            </option><option value="canFamPoodle1">
                Dog May 2003 (TIGR Poodle/canFamPoodle1) (canFamPoodle1)
            </option><option value="canFam2">
                Dog May 2005 (Broad/canFam2) (canFam2)
            </option><option value="canFam5">
                Dog May 2019 (UMICH_Zoey_3.1/canFam5) (canFam5)
            </option><option value="canFam6">
                Dog Oct. 2020 (Dog10K_Boxer_Tasha/canFam6) (canFam6)
            </option><option value="turTru1">
                Dolphin Feb. 2008 (Broad/turTru1) (turTru1)
            </option><option value="turTru2">
                Dolphin Oct. 2011 (Baylor Ttru_1.4/turTru2) (turTru2)
            </option><option value="capHir1">
                Domestic goat May 2012 (CHIR_1.0/capHir1) (capHir1)
            </option><option value="picPub1">
                Downy woodpecker Jun 2014 (ASM69900v1/picPub1) (picPub1)
            </option><option value="eboVir2">
                Ebola virus 49 sequences: 5 Ebola, 2 Marburg strains (eboVir2)
            </option><option value="eboVir1">
                Ebola virus Aug. 2014 (46 strains eboVir1/eboVir1) (eboVir1)
            </option><option value="eboVir3">
                Ebola virus Sierra Leone 2014 (G3683/KM034562.1/eboVir3) (eboVir3)
            </option><option value="ecoRef3">
                Ecoref Feb. 2013 (ecoRef3/ecoRef3) (ecoRef3)
            </option><option value="ecoRef4">
                Ecoref Feb. 2013 (ecoRef4/ecoRef4) (ecoRef4)
            </option><option value="ecoRef1">
                Ecoref Jan. 2013 (ecoRef1/ecoRef1) (ecoRef1)
            </option><option value="ecoRef2">
                Ecoref Jan. 2013 (ecoRef2/ecoRef2) (ecoRef2)
            </option><option value="10694">
                Ehrlichia canis str. Jake (10694)
            </option><option value="325">
                Ehrlichia chaffeensis str. Arkansas (325)
            </option><option value="13356">
                Ehrlichia ruminantium str. Gardel (13356)
            </option><option value="13355">
                Ehrlichia ruminantium str. Welgevonden (13355)
            </option><option value="ehrlRumi_WELGEVONDEN">
                Ehrlichia ruminantium str. Welgevonden (ehrlRumi_WELGEVONDEN)
            </option><option value="Elaeis_guineensis_EG5.1">
                Elaeis guineensis EG5.1
            </option><option value="Elaeis_guineensis_NCBI_Genome_v1.0">
                Elaeis guineensis NCBI Genome v1.0
            </option><option value="loxAfr2">
                Elephant Jul. 2008 (Broad/loxAfr2) (loxAfr2)
            </option><option value="loxAfr3">
                Elephant Jul. 2009 (Broad/loxAfr3) (loxAfr3)
            </option><option value="loxAfr1">
                Elephant May 2005 (Broad/loxAfr1) (loxAfr1)
            </option><option value="calMil1">
                Elephant shark Dec. 2013 (Callorhinchus_milii-6.1.3/calMil1) (calMil1)
            </option><option value="aptFor1">
                Emperor penguin 06 Jun 2014 (ASM69914v1/aptFor1) (aptFor1)
            </option><option value="12720">
                Enterobacter sakazakii ATCC BAA-894 (12720)
            </option><option value="ente638">
                Enterobacter sp. 638 (ente638)
            </option><option value="enteFaec_V583">
                Enterococcus faecalis V583 (enteFaec_V583)
            </option><option value="Erythranthe_guttata_JGI_v2.0">
                Erythranthe guttata JGI v2.0
            </option><option value="erytLito_HTCC2594">
                Erythrobacter litoralis HTCC2594 (erytLito_HTCC2594)
            </option><option value="16235">
                Escherichia coli 536 (16235)
            </option><option value="eschColi_APEC_O1">
                Escherichia coli APEC O1 (eschColi_APEC_O1)
            </option><option value="18083">
                Escherichia coli C str. ATCC 8739 (18083)
            </option><option value="eschColi_CFT073">
                Escherichia coli CFT073 (eschColi_CFT073)
            </option><option value="20079">
                Escherichia coli DH10B (20079)
            </option><option value="13960">
                Escherichia coli E24377A (13960)
            </option><option value="eschColi_EC4115">
                Escherichia coli EC4115 (eschColi_EC4115)
            </option><option value="13959">
                Escherichia coli HS (13959)
            </option><option value="eschColi_K12">
                Escherichia coli K12 (eschColi_K12)
            </option><option value="259">
                Escherichia coli O157:H7 EDL933 (259)
            </option><option value="eschColi_O157H7">
                Escherichia coli O157:H7 str. Sakai (eschColi_O157H7)
            </option><option value="19469">
                Escherichia coli SECEC SMS-3-5 (19469)
            </option><option value="eschColi_TW14359">
                Escherichia coli TW14359 (eschColi_TW14359)
            </option><option value="16259">
                Escherichia coli UTI89 (16259)
            </option><option value="16351">
                Escherichia coli W3110 (16351)
            </option><option value="euaGli13">
                Euarchontoglires Apr. 24. 2006 (UCSC Recon/euaGli13) (euaGli13)
            </option><option value="Eucalyptus_grandis_JGI_assembly_v1.0_annotation_v1.1">
                Eucalyptus grandis JGI assembly v1.0, annotation v1.1
            </option><option value="Eucalyptus_grandis_JGI_v2.0">
                Eucalyptus grandis JGI v2.0
            </option><option value="dicLab1">
                European sea bass Apr 2014 (seabass_V1.0/dicLab1) (dicLab1)
            </option><option value="eutHer13">
                Eutheria Apr. 24. 2006 (UCSC Recon/eutHer13) (eutHer13)
            </option><option value="evoSim1">
                Evosim Feb. 2013 (evoSim1/evoSim1) (evoSim1)
            </option><option value="musFur1">
                Ferret  Apr. 2011 (MusPutFur1.0/musFur1) (musFur1)
            </option><option value="16719">
                Fervidobacterium nodosum Rt17-B1 (16719)
            </option><option value="18981">
                Finegoldia magna ATCC 29328 (18981)
            </option><option value="Schizosaccharomyces_pombe_1.1">
                Fission Yeast (S. pombe) (Schizosaccharomyces_pombe_1.1)
            </option><option value="sebRub1">
                Flag rockfish 22 Oct-2013 (SRub1.0/sebRub1) (sebRub1)
            </option><option value="flavJohn_UW101">
                Flavobacterium johnsoniae UW101 (flavJohn_UW101)
            </option><option value="19979">
                Flavobacterium psychrophilum JIP02/86 (19979)
            </option><option value="AaloF1">
                Foshan strain genomic contigs sequences, AaloF1 assembly.
            </option><option value="Fragaria_vesca_JGI_v1.1">
                Fragaria vesca JGI v1.1
            </option><option value="Fragaria_vesca_Strawberry_Genome_1.0">
                Fragaria vesca Strawberry Genome 1.0
            </option><option value="27853">
                Francisella philomiragia subsp. philomiragia ATCC 25017 (27853)
            </option><option value="16421">
                Francisella tularensis subsp. holarctica (16421)
            </option><option value="20197">
                Francisella tularensis subsp. holarctica FTA (20197)
            </option><option value="17265">
                Francisella tularensis subsp. holarctica OSU18 (17265)
            </option><option value="16088">
                Francisella tularensis subsp. novicida U112 (16088)
            </option><option value="17375">
                Francisella tularensis subsp. tularensis FSC198 (17375)
            </option><option value="franTula_TULARENSIS">
                Francisella tularensis subsp. tularensis SCHU S4 (franTula_TULARENSIS)
            </option><option value="18459">
                Francisella tularensis subsp. tularensis WY96-3418 (18459)
            </option><option value="17403">
                Frankia alni ACN14a (17403)
            </option><option value="franCcI3">
                Frankia sp. CcI3 (franCcI3)
            </option><option value="13915">
                Frankia sp. EAN1pec (13915)
            </option><option value="fr1">
                Fugu Aug. 2002 (JGI 3.0/fr1) (fr1)
            </option><option value="fr2">
                Fugu Oct. 2004 (JGI 4.0/fr2) (fr2)
            </option><option value="fr3">
                Fugu Oct. 2011 (FUGU5/fr3) (fr3)
            </option><option value="fusoNucl">
                Fusobacterium nucleatum subsp. nucleatum ATCC 25586 (fusoNucl)
            </option><option value="hg19Patch2">
                GRCh37.p2 Aug. 2009 (GRCh37.p2/hg19Patch2) (hg19Patch2)
            </option><option value="hg38Patch2">
                GRCh38.p2 Dec 2014 (hg38Patch2)
            </option><option value="thaSir1">
                Garter snake Jun. 2015 (Thamnophis_sirtalis-6.0/thaSir1) (thaSir1)
            </option><option value="geobKaus_HTA426">
                Geobacillus kaustophilus HTA426 (geobKaus_HTA426)
            </option><option value="geobTher_NG80_2">
                Geobacillus thermodenitrificans NG80-2 (geobTher_NG80_2)
            </option><option value="geobMeta_GS15">
                Geobacter metallireducens GS-15 (geobMeta_GS15)
            </option><option value="geobSulf">
                Geobacter sulfurreducens PCA (geobSulf)
            </option><option value="geobUran_RF4">
                Geobacter uraniireducens Rf4 (geobUran_RF4)
            </option><option value="gavGan0">
                Gharial APR 2009 (preliminary/gavGan0) (gavGan0)
            </option><option value="nomLeu1">
                Gibbon Jan. 2010 (GGSC Nleu1.0/nomLeu1) (nomLeu1)
            </option><option value="nomLeu2">
                Gibbon Jun. 2011 (GGSC Nleu1.1/nomLeu2) (nomLeu2)
            </option><option value="nomLeu3">
                Gibbon Oct. 2012 (GGSC Nleu3.0/nomLeu3) (nomLeu3)
            </option><option value="gliRes13">
                Glires Apr. 24. 2006 (UCSC Recon/gliRes13) (gliRes13)
            </option><option value="gloeViol">
                Gloeobacter violaceus PCC 7421 (gloeViol)
            </option><option value="377">
                Gluconacetobacter diazotrophicus PAl 5 (377)
            </option><option value="glucOxyd_621H">
                Gluconobacter oxydans 621H (glucOxyd_621H)
            </option><option value="Glycine_max_JGI_Glyma1.1_annotation_of_the_chromosome-based_Glyma1_assembly">
                Glycine max JGI Glyma1.1 annotation of the chromosome-based Glyma1 assembly
            </option><option value="Glycine_max_JGI_Wm82.a2.v1">
                Glycine max JGI Wm82.a2.v1
            </option><option value="aquChr1">
                Golden eagle May 2014 (AquilaChrysaetos1/aquChr1) (aquChr1)
            </option><option value="aquChr2">
                Golden eagle Oct. 2014 (aquChr-1.0.2/aquChr2) (aquChr2)
            </option><option value="mesAur1">
                Golden hamster Mar 2013 (MesAur1.0/mesAur1) (mesAur1)
            </option><option value="rhiRox1">
                Golden snub-nosed monkey Oct. 2014 (Rrox_v1/rhiRox1) (rhiRox1)
            </option><option value="manVit1">
                Golden-collared manakin May 2014 (ASM69201v1/manVit1) (manVit1)
            </option><option value="gorGor2">
                Gorilla Aug. 2009 (Sanger 4/gorGor2) (gorGor2)
            </option><option value="gorGor6">
                Gorilla Aug. 2019 (Kamilah_GGO_v0/gorGor6) (gorGor6)
            </option><option value="gorGor4">
                Gorilla Dec 2014 (gorGor4.1/gorGor4) (gorGor4)
            </option><option value="gorGor5">
                Gorilla Mar. 2016 (GSMRT3/gorGor5) (gorGor5)
            </option><option value="gorGor3">
                Gorilla May 2011 (gorGor3.1/gorGor3) (gorGor3)
            </option><option value="gorGor1">
                Gorilla Oct. 2008 (Sanger 0.1/gorGor1) (gorGor1)
            </option><option value="Gossypium_raimondii_JGI_annotation_v2.1_on_assembly_v2.0">
                Gossypium raimondii JGI annotation v2.1 on assembly v2.0
            </option><option value="Gossypium_raimondii_JGI_v2.1">
                Gossypium raimondii JGI v2.1
            </option><option value="gramFors_KT0803">
                Gramella forsetii KT0803 (gramFors_KT0803)
            </option><option value="granBeth_CGDNIH1">
                Granulibacter bethesdensis CGDNIH1 (granBeth_CGDNIH1)
            </option><option value="hg19Patch10">
                Grch37.p10 Sep. 2012 (GRCh37.p10/hg19Patch10) (hg19Patch10)
            </option><option value="hg19Patch5">
                Grch37.p5 Jun. 2011 (GRCh37.p5/hg19Patch5) (hg19Patch5)
            </option><option value="hg19Patch9">
                Grch37.p9 Jul. 2012 (GRCh37.p9/hg19Patch9) (hg19Patch9)
            </option><option value="mm10Patch1">
                Grcm38.p1 Aug. 2012 (GRCm38.p1/mm10Patch1) (mm10Patch1)
            </option><option value="phaCar1">
                Great cormorant Jun 2014 (ASM70892v1/phaCar1) (phaCar1)
            </option><option value="rhiFer1">
                Greater horseshoe bat Sep. 2013 (ASM46549v1/rhiFer1) (rhiFer1)
            </option><option value="chlSab2">
                Green monkey 25 Mar-2014 (Chlorocebus_sabeus 1.1/chlSab2) (chlSab2)
            </option><option value="cheMyd1">
                Green seaturtle Mar 2013 (CheMyd_1.0/cheMyd1) (cheMyd1)
            </option><option value="cavPor3">
                Guinea pig Feb. 2008 (Broad/cavPor3) (cavPor3)
            </option><option value="poeRet1">
                Guppy Apr. 2014 (Guppy_female_1.0/poeRet1) (poeRet1)
            </option><option value="haeCon1">
                H. contortus Dec. 2009 (WormBase WS210/haeCon1) (haeCon1)
            </option><option value="38">
                Haemophilus ducreyi 35000HP (38)
            </option><option value="11752">
                Haemophilus influenzae 86-028NP (11752)
            </option><option value="16400">
                Haemophilus influenzae PittEE (16400)
            </option><option value="16401">
                Haemophilus influenzae PittGG (16401)
            </option><option value="haemInfl_KW20">
                Haemophilus influenzae Rd KW20 (haemInfl_KW20)
            </option><option value="haemSomn_129PT">
                Haemophilus somnus 129PT (haemSomn_129PT)
            </option><option value="388">
                Haemophilus somnus 2336 (388)
            </option><option value="haheChej_KCTC_2396">
                Hahella chejuensis KCTC 2396 (haheChej_KCTC_2396)
            </option><option value="halMar1">
                Haloarcula marismortui ATCC 43049 (halMar1)
            </option><option value="106">
                Halobacterium salinarum R1 (106)
            </option><option value="haloHalo1">
                Halobacterium sp. NRC-1 (haloHalo1)
            </option><option value="haloWals1">
                Haloquadratum walsbyi DSM 16790 (haloWals1)
            </option><option value="haloHalo_SL1">
                Halorhodospira halophila SL1 (haloHalo_SL1)
            </option><option value="neoSch1">
                Hawaiian monk seal Jun. 2017 (ASM220157v1/neoSch1) (neoSch1)
            </option><option value="PhumU1">
                Head Louse (Pediculus humanus) (PhumU1)
            </option><option value="eriEur1">
                Hedgehog June 2006 (Broad/eriEur1) (eriEur1)
            </option><option value="eriEur2">
                Hedgehog May 2012 (EriEur2.0/eriEur2) (eriEur2)
            </option><option value="heliAcin_SHEEBA">
                Helicobacter acinonychis str. Sheeba (heliAcin_SHEEBA)
            </option><option value="heliHepa">
                Helicobacter hepaticus ATCC 51449 (heliHepa)
            </option><option value="heliPylo_26695">
                Helicobacter pylori 26695 (heliPylo_26695)
            </option><option value="heliPylo_HPAG1">
                Helicobacter pylori HPAG1 (heliPylo_HPAG1)
            </option><option value="heliPylo_J99">
                Helicobacter pylori J99 (heliPylo_J99)
            </option><option value="13427">
                Heliobacterium modesticaldum Ice1 (13427)
            </option><option value="hermArse">
                Herminiimonas arsenicoxydans (hermArse)
            </option><option value="16523">
                Herpetosiphon aurantiacus ATCC 23779 (16523)
            </option><option value="Hevea_brasiliensis_NCBI_Genomes_v1">
                Hevea brasiliensis NCBI Genomes v1
            </option><option value="hg19LggInv">
                Hg19 lgg inversion Mar 2013 (1/hg19LggInv) (hg19LggInv)
            </option><option value="hlaRef1">
                Hlaref Apr. 2012 (hlaRefV1/hlaRef1) (hlaRef1)
            </option><option value="hlaRef2">
                Hlaref Apr. 2012 (hlaRefV2/hlaRef2) (hlaRef2)
            </option><option value="hlaRef3">
                Hlaref Apr. 2012 (hlaRefV3/hlaRef3) (hlaRef3)
            </option><option value="opiHoa1">
                Hoatzin May 2014 (ASM69207v1/opiHoa1) (opiHoa1)
            </option><option value="homIni13">
                Hominidae Apr. 24. 2006 (UCSC Recon/homIni13) (homIni13)
            </option><option value="homIni14">
                Hominidae Oct. 1. 2006 (UCSC Recon/homIni14) (homIni14)
            </option><option value="GRCh38">
                Homo Sapiens (GRCh38) Transcriptome + ERCC
            </option><option value="Homo_sapiens_nuHg19_mtrCRS">
                Homo sapiens (hg19 with mtDNA replaced with rCRS) (Homo_sapiens_nuHg19_mtrCRS)
            </option><option value="hg_g1k_v37">
                Homo sapiens b37 (hg_g1k_v37)
            </option><option value="CHM1">
                Homo sapiens hydatidiform mole assembly chm1 June 2013  (1/CHM1) (CHM1)
            </option><option value="apiMel3">
                Honeybee (Apis mellifera): apiMel3 (apiMel3)
            </option><option value="corCor1">
                Hooded crow Aug 2014 (Hooded_Crow_genome/corCor1) (corCor1)
            </option><option value="Hordeum_vulgare_Ensembl_Genomes_ASM32608v1">
                Hordeum vulgare Ensembl Genomes ASM32608v1
            </option><option value="Hordeum_vulgare_IBC_PGSB_v2">
                Hordeum vulgare IBC PGSB v2
            </option><option value="Hordeum_vulgare_MIPS">
                Hordeum vulgare MIPS
            </option><option value="equCab1">
                Horse Jan. 2007 (Broad/equCab1) (equCab1)
            </option><option value="equCab3">
                Horse Jan. 2018 (EquCab3.0/equCab3) (equCab3)
            </option><option value="equCab2">
                Horse Sep. 2007 (Broad/equCab2) (equCab2)
            </option><option value="chlUnd1">
                Houbara bustard Jun 2014 (ASM69519v1/chlUnd1) (chlUnd1)
            </option><option value="hg15">
                Human Apr. 2003 (NCBI33/hg15) (hg15)
            </option><option value="CHM13_T2T_v2.0">
                Human CHM13 2.0 (T2T Consortium) Jan. 2022
            </option><option value="hg38">
                Human Dec. 2013 (GRCh38/hg38) (hg38)
            </option><option value="hg19">
                Human Feb. 2009 (GRCh37/hg19) (hg19)
            </option><option value="CHM13_T2T_v1.0">
                Human Jan. 2021 isolate CHM13 (T2T Consortium) female
            </option><option value="hs1">
                Human Jan. 2022 (T2T CHM13v2.0/hs1) (hs1)
            </option><option value="hg16">
                Human July 2003 (NCBI34/hg16) (hg16)
            </option><option value="hg18">
                Human Mar. 2006 (NCBI36/hg18) (hg18)
            </option><option value="hg17">
                Human May 2004 (NCBI35/hg17) (hg17)
            </option><option value="15217">
                Human herpesvirus 1 (15217)
            </option><option value="repeats2">
                Human repeats Nov. 2012 (1/repeats2) (repeats2)
            </option><option value="Hydra_JCVI">
                Hydra magnipapillata str. 105 (Hydra_JCVI)
            </option><option value="hypeButy1">
                Hyperthermus butylicus DSM 5456 (hypeButy1)
            </option><option value="hyphNept_ATCC15444">
                Hyphomonas neptunium ATCC 15444 (hyphNept_ATCC15444)
            </option><option value="idioLoih_L2TR">
                Idiomarina loihiensis L2TR (idioLoih_L2TR)
            </option><option value="13914">
                Ignicoccus hospitalis KIN4/I (13914)
            </option><option value="megLyr1">
                Indian false vampire Sep. 2013 (ASM46534v1/megLyr1) (megLyr1)
            </option><option value="h7n7_360722">
                Influenza A virus (A/seal/Mass/1-SC35M/1980(H7N7))
            </option><option value="venter1">
                J. Craig Venter Sep. 2007 (HuRef/venter1) (venter1)
            </option><option value="jannCCS1">
                Jannaschia sp. CCS1 (jannCCS1)
            </option><option value="16549">
                Janthinobacterium sp. Marseille (16549)
            </option><option value="angJap1">
                Japanese eel 18 Mar-2014 (japanese_eel_genome_v1_25_oct_2011_japonica_c401b400k25m200_sspacepremiumk3a02n24_extra.final.scaffolds/angJap1) (angJap1)
            </option><option value="cotJap1">
                Japanese quail 26 Jul-2013 (Coja_1.0/cotJap1) (cotJap1)
            </option><option value="dipOrd1">
                Kangaroo rat Jul. 2008 (Broad/dipOrd1) (dipOrd1)
            </option><option value="chaVoc2">
                Killdeer Aug 2014 (ASM70802v2/chaVoc2) (chaVoc2)
            </option><option value="chaVoc1">
                Killdeer Jun 2014 (ASM70802v1/chaVoc1) (chaVoc1)
            </option><option value="orcOrc1">
                Killer whale Jan 2013 (Oorc_1.1/orcOrc1) (orcOrc1)
            </option><option value="10689">
                Kineococcus radiotolerans SRS30216 (10689)
            </option><option value="kirRef">
                Kirref Apr. 2011 (kirRef/kirRef) (kirRef)
            </option><option value="kirRefV2">
                Kirrefv2 Apr. 2011 (kirRefV2/kirRefV2) (kirRefV2)
            </option><option value="kirRefV3">
                Kirrefv3 Apr. 2011 (kirRefV3/kirRefV3) (kirRefV3)
            </option><option value="kirRefV4">
                Kirrefv4 Apr. 2011 (kirRefV4/kirRefV4) (kirRefV4)
            </option><option value="kirRefV5">
                Kirrefv5 Apr. 2011 (kirRefV5/kirRefV5) (kirRefV5)
            </option><option value="31">
                Klebsiella pneumoniae subsp. pneumoniae MGH 78578 (31)
            </option><option value="Homo_sapiens_AK1">
                Korean Man (Homo_sapiens_AK1)
            </option><option value="82">
                Lactobacillus acidophilus NCFM (82)
            </option><option value="404">
                Lactobacillus brevis ATCC 367 (404)
            </option><option value="402">
                Lactobacillus casei ATCC 334 (402)
            </option><option value="16871">
                Lactobacillus delbrueckii subsp. bulgaricus ATCC 11842 (16871)
            </option><option value="403">
                Lactobacillus delbrueckii subsp. bulgaricus ATCC BAA-365 (403)
            </option><option value="84">
                Lactobacillus gasseri ATCC 33323 (84)
            </option><option value="17811">
                Lactobacillus helveticus DPC 4571 (17811)
            </option><option value="9638">
                Lactobacillus johnsonii NCC 533 (9638)
            </option><option value="lactPlan">
                Lactobacillus plantarum WCFS1 (lactPlan)
            </option><option value="15766">
                Lactobacillus reuteri F275 (15766)
            </option><option value="13435">
                Lactobacillus sakei subsp. sakei 23K (13435)
            </option><option value="lactSali_UCC118">
                Lactobacillus salivarius UCC118 (lactSali_UCC118)
            </option><option value="18797">
                Lactococcus lactis subsp. cremoris MG1363 (18797)
            </option><option value="401">
                Lactococcus lactis subsp. cremoris SK11 (401)
            </option><option value="lactLact">
                Lactococcus lactis subsp. lactis Il1403 (lactLact)
            </option><option value="petMar3">
                Lamprey Dec. 2017 (Pmar_germline 1.0/petMar3) (petMar3)
            </option><option value="petMar1">
                Lamprey Mar. 2007 (WUGSC 3.0/petMar1) (petMar1)
            </option><option value="petMar2">
                Lamprey Sep. 2010 (WUGSC 7.0/petMar2) (petMar2)
            </option><option value="braFlo2">
                Lancelet Apr. 2008 (JGI v2.0/braFlo2) (braFlo2)
            </option><option value="braFlo1">
                Lancelet Mar. 2006 (JGI 1.0/braFlo1) (braFlo1)
            </option><option value="lauRas13">
                Laurasiatheria Apr. 24. 2006 (UCSC Recon/lauRas13) (lauRas13)
            </option><option value="lawsIntr_PHE_MN1_00">
                Lawsonia intracellularis PHE/MN1-00 (lawsIntr_PHE_MN1_00)
            </option><option value="17491">
                Legionella pneumophila str. Corby (17491)
            </option><option value="13126">
                Legionella pneumophila str. Lens (13126)
            </option><option value="13127">
                Legionella pneumophila str. Paris (13127)
            </option><option value="legiPneu_PHILADELPHIA">
                Legionella pneumophila subsp. pneumophila str. Philadelphia 1 (legiPneu_PHILADELPHIA)
            </option><option value="leifXyli_XYLI_CTCB0">
                Leifsonia xyli subsp. xyli str. CTCB07 (leifXyli_XYLI_CTCB0)
            </option><option value="lMaj5">
                Leishmania major 2005 (lMaj5)
            </option><option value="16148">
                Leptospira borgpetersenii serovar Hardjo-bovis JB197 (16148)
            </option><option value="16146">
                Leptospira borgpetersenii serovar Hardjo-bovis L550 (16146)
            </option><option value="10687">
                Leptospira interrogans serovar Copenhageni str. Fiocruz L1-130 (10687)
            </option><option value="leptInte">
                Leptospira interrogans serovar Lai str. 56601 (leptInte)
            </option><option value="20039">
                Leptothrix cholodnii SP-6 (20039)
            </option><option value="jacJac1">
                Lesser Egyptian jerboa May 2012 (JacJac1.0/jacJac1) (jacJac1)
            </option><option value="16062">
                Leuconostoc citreum KM20 (16062)
            </option><option value="leucMese_ATCC8293">
                Leuconostoc mesenteroides subsp. mesenteroides ATCC 8293 (leucMese_ATCC8293)
            </option><option value="lotGig0">
                Limpet Jul. 2013 (lotGig0/lotGig0) (lotGig0)
            </option><option value="listInno">
                Listeria innocua Clip11262 (listInno)
            </option><option value="276">
                Listeria monocytogenes EGD-e (276)
            </option><option value="85">
                Listeria monocytogenes str. 4b F2365 (85)
            </option><option value="13443">
                Listeria welshimeri serovar 6b str. SLCC5334 (13443)
            </option><option value="egrGar1">
                Little egret May 2014 (ASM68718v1/egrGar1) (egrGar1)
            </option><option value="anoCar1">
                Lizard Feb. 2007 (Broad/anoCar1) (anoCar1)
            </option><option value="anoCar2">
                Lizard May 2010 (Broad AnoCar2.0/anoCar2) (anoCar2)
            </option><option value="Lotus_japonicus_Kazusa_2.5">
                Lotus japonicus Kazusa 2.5
            </option><option value="19619">
                Lysinibacillus sphaericus C3-41 (19619)
            </option><option value="musDom2">
                M. domestica 22 Apr 2013 (Musca_domestica-2.0.2/musDom2) (musDom2)
            </option><option value="melHap1">
                M. hapla Sep. 2008 (M. hapla VW9 WS210/melHap1) (melHap1)
            </option><option value="melInc1">
                M. incognita Feb. 2008 (M. incognita WS210/melInc1) (melInc1)
            </option><option value="mmtv">
                MMTV Nov. 2009 (MMTV/mmtv) (mmtv)
            </option><option value="magnMC1">
                Magnetococcus sp. MC-1 (magnMC1)
            </option><option value="magnMagn_AMB_1">
                Magnetospirillum magneticum AMB-1 (magnMagn_AMB_1)
            </option><option value="Zea_mays_B73_RefGen_v2">
                Maize (Zea mays) (Zea_mays_B73_RefGen_v2)
            </option><option value="galVar1">
                Malayan flying lemur Jun. 2014 (G_variegatus-3.0.2/galVar1) (galVar1)
            </option><option value="anaPla1">
                Mallard duck Apr 2013 (BGI_duck_1.0/anaPla1) (anaPla1)
            </option><option value="Malus_domestica_IASMA">
                Malus domestica IASMA
            </option><option value="Malus_domestica_Penguin_Genomics_v2">
                Malus domestica Penguin Genomics v2
            </option><option value="triMan1">
                Manatee Oct. 2011 (Broad v1.0/triMan1) (triMan1)
            </option><option value="Manihot_esculenta_Cassava4">
                Manihot esculenta Cassava4
            </option><option value="Manihot_esculenta_JGI_v6.1">
                Manihot esculenta JGI v6.1
            </option><option value="mannSucc_MBEL55E">
                Mannheimia succiniciproducens MBEL55E (mannSucc_MBEL55E)
            </option><option value="marVir1">
                Marburg virus Sep. 2014 (NC_024781.1/marVir1) (marVir1)
            </option><option value="Marchantia_polymorpha_JGI_v3.1">
                Marchantia polymorpha JGI v3.1
            </option><option value="mariMari_MCS10">
                Maricaulis maris MCS10 (mariMari_MCS10)
            </option><option value="mariAqua_VT8">
                Marinobacter aquaeolei VT8 (mariAqua_VT8)
            </option><option value="17445">
                Marinomonas sp. MWYL1 (17445)
            </option><option value="calJac1">
                Marmoset June 2007 (WUGSC 2.0.2/calJac1) (calJac1)
            </option><option value="calJac3">
                Marmoset March 2009 (WUGSC 3.2/calJac3) (calJac3)
            </option><option value="calJac4">
                Marmoset May 2020 (Callithrix_jacchus_cj1700_1.1/calJac4) (calJac4)
            </option><option value="oryLat1">
                Medaka Apr. 2006 (NIG/UT MEDAKA1/oryLat1) (oryLat1)
            </option><option value="oryLat2">
                Medaka Oct. 2005 (NIG/UT MEDAKA1/oryLat2) (oryLat2)
            </option><option value="Medicago_truncatula_JCVI_4.0">
                Medicago truncatula JCVI 4.0
            </option><option value="Medicago_truncatula_JGI_Mt4.0v1">
                Medicago truncatula JGI Mt4.0v1
            </option><option value="geoFor1">
                Medium ground finch Apr. 2012 (GeoFor_1.0/geoFor1) (geoFor1)
            </option><option value="pteVam1">
                Megabat Jul. 2008 (Broad/pteVam1) (pteVam1)
            </option><option value="mesoFlor_L1">
                Mesoplasma florum L1 (mesoFlor_L1)
            </option><option value="mesoLoti">
                Mesorhizobium loti MAFF303099 (mesoLoti)
            </option><option value="10690">
                Mesorhizobium sp. BNC1 (10690)
            </option><option value="metaSedu">
                Metallosphaera sedula DSM 5348 (metaSedu)
            </option><option value="methSmit1">
                Methanobrevibacter smithii ATCC 35061 (methSmit1)
            </option><option value="methJann1">
                Methanocaldococcus jannaschii DSM 2661 (methJann1)
            </option><option value="methBurt2">
                Methanococcoides burtonii DSM 6242 (methBurt2)
            </option><option value="methAeol1">
                Methanococcus aeolicus Nankai-3 (methAeol1)
            </option><option value="methMari_C5_1">
                Methanococcus maripaludis C5 (methMari_C5_1)
            </option><option value="19639">
                Methanococcus maripaludis C6 (19639)
            </option><option value="methMari_C7">
                Methanococcus maripaludis C7 (methMari_C7)
            </option><option value="metMar1">
                Methanococcus maripaludis S2 (metMar1)
            </option><option value="methVann1">
                Methanococcus vannielii SB (methVann1)
            </option><option value="methLabrZ_1">
                Methanocorpusculum labreanum Z (methLabrZ_1)
            </option><option value="mculMari1">
                Methanoculleus marisnigri JR1 (mculMari1)
            </option><option value="methKand1">
                Methanopyrus kandleri AV19 (methKand1)
            </option><option value="methTherPT1">
                Methanosaeta thermophila PT (methTherPT1)
            </option><option value="metAce1">
                Methanosarcina acetivorans C2A (metAce1)
            </option><option value="methBark1">
                Methanosarcina barkeri str. Fusaro (methBark1)
            </option><option value="methMaze1">
                Methanosarcina mazei Go1 (methMaze1)
            </option><option value="methStad1">
                Methanosphaera stadtmanae DSM 3091 (methStad1)
            </option><option value="methHung1">
                Methanospirillum hungatei JF-1 (methHung1)
            </option><option value="methTher1">
                Methanothermobacter thermautotrophicus str. Delta H (methTher1)
            </option><option value="methPetr_PM1">
                Methylibium petroleiphilum PM1 (methPetr_PM1)
            </option><option value="methFlag_KT">
                Methylobacillus flagellatus KT (methFlag_KT)
            </option><option value="18637">
                Methylobacterium extorquens PA1 (18637)
            </option><option value="18817">
                Methylobacterium radiotolerans JCM 2831 (18817)
            </option><option value="18809">
                Methylobacterium sp. 4-46 (18809)
            </option><option value="methCaps_BATH">
                Methylococcus capsulatus str. Bath (methCaps_BATH)
            </option><option value="astMex1">
                Mexican tetra (cavefish) Apr 2013 (Astyanax_mexicanus-1.0.2/astMex1) (astMex1)
            </option><option value="mhcRefnoNs2">
                Mhcref Apr. 2011 (mhcRefnoNsV2/mhcRefnoNs2) (mhcRefnoNs2)
            </option><option value="mhcRef">
                Mhcref Jan. 2011 (mhcRefV1/mhcRef) (mhcRef)
            </option><option value="mhcRefnoNs10">
                Mhcref July 2011 (mhcRefnoNsV10/mhcRefnoNs10) (mhcRefnoNs10)
            </option><option value="mhcRefnoNs11">
                Mhcref July 2011 (mhcRefnoNsV11/mhcRefnoNs11) (mhcRefnoNs11)
            </option><option value="mhcRefnoNs12">
                Mhcref July 2011 (mhcRefnoNsV12/mhcRefnoNs12) (mhcRefnoNs12)
            </option><option value="mhcRefnoNs13">
                Mhcref July 2011 (mhcRefnoNsV13/mhcRefnoNs13) (mhcRefnoNs13)
            </option><option value="mhcRefnoNs14">
                Mhcref July 2011 (mhcRefnoNsV14/mhcRefnoNs14) (mhcRefnoNs14)
            </option><option value="mhcRefnoNs15">
                Mhcref July 2011 (mhcRefnoNsV15/mhcRefnoNs15) (mhcRefnoNs15)
            </option><option value="mhcRefnoNs16">
                Mhcref July 2011 (mhcRefnoNsV16/mhcRefnoNs16) (mhcRefnoNs16)
            </option><option value="mhcRefnoNs17">
                Mhcref July 2011 (mhcRefnoNsV17/mhcRefnoNs17) (mhcRefnoNs17)
            </option><option value="mhcRefnoNs5">
                Mhcref July 2011 (mhcRefnoNsV5/mhcRefnoNs5) (mhcRefnoNs5)
            </option><option value="mhcRefnoNs6">
                Mhcref July 2011 (mhcRefnoNsV6/mhcRefnoNs6) (mhcRefnoNs6)
            </option><option value="mhcRefnoNs7">
                Mhcref July 2011 (mhcRefnoNsV7/mhcRefnoNs7) (mhcRefnoNs7)
            </option><option value="mhcRefnoNs8">
                Mhcref July 2011 (mhcRefnoNsV8/mhcRefnoNs8) (mhcRefnoNs8)
            </option><option value="mhcRefnoNs9">
                Mhcref July 2011 (mhcRefnoNsV8/mhcRefnoNs9) (mhcRefnoNs9)
            </option><option value="mhcRefnoNs">
                Mhcref June 2011 (mhcRefnoNsV1/mhcRefnoNs) (mhcRefnoNs)
            </option><option value="mhcRefnoNs3">
                Mhcref June 2011 (mhcRefnoNsV3/mhcRefnoNs3) (mhcRefnoNs3)
            </option><option value="mhcRefnoNs4">
                Mhcref June 2011 (mhcRefnoNsV4/mhcRefnoNs4) (mhcRefnoNs4)
            </option><option value="mhcRefnoNs22">
                Mhcref Mar. 2012 (mhcRefnoNsV22/mhcRefnoNs22) (mhcRefnoNs22)
            </option><option value="mhcRefnoNs23">
                Mhcref Mar. 2012 (mhcRefnoNsV23/mhcRefnoNs23) (mhcRefnoNs23)
            </option><option value="mhcRefnoNs24">
                Mhcref Mar. 2012 (mhcRefnoNsV24/mhcRefnoNs24) (mhcRefnoNs24)
            </option><option value="mhcRefnoNs18">
                Mhcref Oct. 2011 (mhcRefnoNsV18/mhcRefnoNs18) (mhcRefnoNs18)
            </option><option value="mhcRefnoNs19">
                Mhcref Oct. 2011 (mhcRefnoNsV19/mhcRefnoNs19) (mhcRefnoNs19)
            </option><option value="mhcRefnoNs20">
                Mhcref Oct. 2011 (mhcRefnoNsV20/mhcRefnoNs20) (mhcRefnoNs20)
            </option><option value="mhcRefnoNs21">
                Mhcref Oct. 2011 (mhcRefnoNsV21/mhcRefnoNs21) (mhcRefnoNs21)
            </option><option value="myoLuc2">
                Microbat Jul. 2010 (Broad Institute Myoluc2.0/myoLuc2) (myoLuc2)
            </option><option value="myoLuc1">
                Microbat Mar. 2006 (Broad/myoLuc1) (myoLuc1)
            </option><option value="27835">
                Microcystis aeruginosa NIES-843 (27835)
            </option><option value="Micromonas_commoda_JGI_v3.0">
                Micromonas commoda JGI v3.0
            </option><option value="balAcu1">
                Minke whale Oct. 2013 (BalAcu1.0/balAcu1) (balAcu1)
            </option><option value="mm10Strains1">
                Mm10haps Dec. 2011 (Mouse strains/mm10Strains1) (mm10Strains1)
            </option><option value="mpxvRivers">
                Monkeypox virus MPXV-M5312_HM12_Rivers (MT903340.1/GCF_014621545.1) (mpxvRivers)
            </option><option value="moorTher_ATCC39073">
                Moorella thermoacetica ATCC 39073 (moorTher_ATCC39073)
            </option><option value="AaegL1">
                Mosquito (Aedes aegypti) (AaegL1)
            </option><option value="AgamP3">
                Mosquito (Anopheles gambiae) (AgamP3)
            </option><option value="CpipJ1">
                Mosquito (Culex quinquefasciatus) (CpipJ1)
            </option><option value="mm10">
                Mouse (Mus Musculus): mm10 Full
            </option><option value="mm7">
                Mouse Aug. 2005 (NCBI35/mm7) (mm7)
            </option><option value="mm8">
                Mouse Feb. 2006 (NCBI36/mm8) (mm8)
            </option><option value="mm9">
                Mouse July 2007 (NCBI37/mm9) (mm9)
            </option><option value="mm39">
                Mouse Jun. 2020 (GRCm39/mm39) (mm39)
            </option><option value="mm5">
                Mouse May 2004 (mm5)
            </option><option value="micMur1">
                Mouse lemur Jul. 2007 (Broad/micMur1) (micMur1)
            </option><option value="micMur2">
                Mouse lemur May 2015 (Mouse lemur/micMur2) (micMur2)
            </option><option value="Musa_acuminata_Banana_Genome_v1.0">
                Musa acuminata Banana Genome v1.0
            </option><option value="Musa_acuminata_Banana_Genome_v2.0">
                Musa acuminata Banana Genome v2.0
            </option><option value="Musa_acuminata_Genescope-Cirad">
                Musa acuminata Genescope-Cirad
            </option><option value="15691">
                Mycobacterium abscessus (15691)
            </option><option value="88">
                Mycobacterium avium 104 (88)
            </option><option value="91">
                Mycobacterium avium subsp. paratuberculosis K-10 (91)
            </option><option value="89">
                Mycobacterium bovis AF2122/97 (89)
            </option><option value="18059">
                Mycobacterium bovis BCG str. Pasteur 1173P2 (18059)
            </option><option value="15760">
                Mycobacterium gilvum PYR-GCK (15760)
            </option><option value="90">
                Mycobacterium leprae TN (90)
            </option><option value="92">
                Mycobacterium smegmatis str. MC2 155 (92)
            </option><option value="16079">
                Mycobacterium sp. JLS (16079)
            </option><option value="16081">
                Mycobacterium sp. KMS (16081)
            </option><option value="15762">
                Mycobacterium sp. MCS (15762)
            </option><option value="223">
                Mycobacterium tuberculosis CDC1551 (223)
            </option><option value="15642">
                Mycobacterium tuberculosis F11 (15642)
            </option><option value="18883">
                Mycobacterium tuberculosis H37Ra (18883)
            </option><option value="mycoTube_H37RV">
                Mycobacterium tuberculosis H37Rv (mycoTube_H37RV)
            </option><option value="16230">
                Mycobacterium ulcerans Agy99 (16230)
            </option><option value="15761">
                Mycobacterium vanbaalenii PYR-1 (15761)
            </option><option value="16095">
                Mycoplasma agalactiae PG2 (16095)
            </option><option value="16208">
                Mycoplasma capricolum subsp. capricolum ATCC 27343 (16208)
            </option><option value="409">
                Mycoplasma gallisepticum R (409)
            </option><option value="mycoGeni">
                Mycoplasma genitalium G37 (mycoGeni)
            </option><option value="13120">
                Mycoplasma hyopneumoniae 232 (13120)
            </option><option value="10639">
                Mycoplasma hyopneumoniae 7448 (10639)
            </option><option value="10675">
                Mycoplasma hyopneumoniae J (10675)
            </option><option value="10697">
                Mycoplasma mobile 163K (10697)
            </option><option value="10616">
                Mycoplasma mycoides subsp. mycoides SC str. PG1 (10616)
            </option><option value="176">
                Mycoplasma penetrans HF-2 (176)
            </option><option value="99">
                Mycoplasma pneumoniae M129 (99)
            </option><option value="100">
                Mycoplasma pulmonis UAB CTIP (100)
            </option><option value="10676">
                Mycoplasma synoviae 53 (10676)
            </option><option value="myxoXant_DK_1622">
                Myxococcus xanthus DK 1622 (myxoXant_DK_1622)
            </option><option value="hetGla2">
                Naked mole-rat Jan. 2012 (Broad HetGla_female_1.0/hetGla2) (hetGla2)
            </option><option value="hetGla1">
                Naked mole-rat Jul. 2011 (BGI HetGla_1.0/hetGla1) (hetGla1)
            </option><option value="nanEqu1">
                Nanoarchaeum equitans Kin4-M (nanEqu1)
            </option><option value="natrPhar1">
                Natronomonas pharaonis DSM 2160 (natrPhar1)
            </option><option value="neisGono_FA1090_1">
                Neisseria gonorrhoeae FA 1090 (neisGono_FA1090_1)
            </option><option value="16393">
                Neisseria meningitidis 053442 (16393)
            </option><option value="neisMeni_FAM18_1">
                Neisseria meningitidis FAM18 (neisMeni_FAM18_1)
            </option><option value="neisMeni_MC58_1">
                Neisseria meningitidis MC58 (neisMeni_MC58_1)
            </option><option value="neisMeni_Z2491_1">
                Neisseria meningitidis Z2491 (neisMeni_Z2491_1)
            </option><option value="Nelumbo_nucifera_LOTUS-DB_v1.1">
                Nelumbo nucifera LOTUS-DB v1.1
            </option><option value="neorSenn_MIYAYAMA">
                Neorickettsia sennetsu str. Miyayama (neorSenn_MIYAYAMA)
            </option><option value="oreNil1">
                Nile tilapia Jan. 2011 (Broad oreNil1.0/oreNil1) (oreNil1)
            </option><option value="oreNil2">
                Nile tilapia Jan. 2011 (Broad oreNil1.1/oreNil2) (oreNil2)
            </option><option value="18963">
                Nitratiruptor sp. SB155-2 (18963)
            </option><option value="13473">
                Nitrobacter hamburgensis X14 (13473)
            </option><option value="nitrWino_NB_255">
                Nitrobacter winogradskyi Nb-255 (nitrWino_NB_255)
            </option><option value="nitrOcea_ATCC19707">
                Nitrosococcus oceani ATCC 19707 (nitrOcea_ATCC19707)
            </option><option value="nitrEuro">
                Nitrosomonas europaea ATCC 19718 (nitrEuro)
            </option><option value="13913">
                Nitrosomonas eutropha C91 (13913)
            </option><option value="19265">
                Nitrosopumilus maritimus SCM1 (19265)
            </option><option value="nitrMult_ATCC25196">
                Nitrosospira multiformis ATCC 25196 (nitrMult_ATCC25196)
            </option><option value="nocaFarc_IFM10152">
                Nocardia farcinica IFM 10152 (nocaFarc_IFM10152)
            </option><option value="nocaJS61">
                Nocardioides sp. JS614 (nocaJS61)
            </option><option value="nonAfr13">
                Non-Afrotheria Apr. 24. 2006 (UCSC Recon/nonAfr13) (nonAfr13)
            </option><option value="merNub1">
                Northern carmine bee-eater May 2014 (ASM69184v1/merNub1) (merNub1)
            </option><option value="fulGla1">
                Northern fulmar May 2014 (ASM69083v1/fulGla1) (fulGla1)
            </option><option value="esoLuc1">
                Northern pike Jul 2014 (EsoLuc1.0/esoLuc1) (esoLuc1)
            </option><option value="nostSp">
                Nostoc sp. PCC 7120 (nostSp)
            </option><option value="calMil0">
                Notyet Oct. 2011 (notYet1/calMil0) (calMil0)
            </option><option value="novoArom_DSM12444">
                Novosphingobium aromaticivorans DSM 12444 (novoArom_DSM12444)
            </option><option value="oceaIhey">
                Oceanobacillus iheyensis HTE831 (oceaIhey)
            </option><option value="19485">
                Ochrobactrum anthropi ATCC 49188 (19485)
            </option><option value="oenoOeni_PSU_1">
                Oenococcus oeni PSU-1 (oenoOeni_PSU_1)
            </option><option value="od01">
                Oikopleura dioica (od01)
            </option><option value="onioYell_PHYTOPLASMA">
                Onion yellows phytoplasma OY-M (onioYell_PHYTOPLASMA)
            </option><option value="monDom4">
                Opossum Jan. 2006 (Broad/monDom4) (monDom4)
            </option><option value="monDom1">
                Opossum Oct. 2004 (Broad prelim/monDom1) (monDom1)
            </option><option value="monDom5">
                Opossum Oct. 2006 (Broad/monDom5) (monDom5)
            </option><option value="ponAbe3">
                Orangutan Jan. 2018 (Susie_PABv2/ponAbe3) (ponAbe3)
            </option><option value="ponAbe2">
                Orangutan July 2007 (WUGSC 2.0.2/ponAbe2) (ponAbe2)
            </option><option value="orieTsut_BORYONG">
                Orientia tsutsugamushi str. Boryong (orieTsut_BORYONG)
            </option><option value="Oropetium_thomaeum_JGI_v1.0">
                Oropetium thomaeum JGI v1.0
            </option><option value="Oryza_brachyantha_Ensembl_Plants_v1.4b">
                Oryza brachyantha Ensembl Plants v1.4b
            </option><option value="Oryza_sativa_ssp._indica_9311_BGF_2005">
                Oryza sativa ssp. indica 9311_BGF_2005
            </option><option value="Oryza_sativa_ssp._indica_MBKBASE_v1.0">
                Oryza sativa ssp. indica MBKBASE v1.0
            </option><option value="Oryza_sativa_ssp._japonica_JGI_v7.0">
                Oryza sativa ssp. japonica JGI v7.0
            </option><option value="Oryza_sativa_ssp._japonica_MSU_RGAP_7">
                Oryza sativa ssp. japonica MSU RGAP 7
            </option><option value="Ostreococcus_lucimarinus_JGI_2.0">
                Ostreococcus lucimarinus JGI 2.0
            </option><option value="Ostreococcus_lucimarinus_JGI_v2.0_assembly_and_annotation">
                Ostreococcus lucimarinus JGI v2.0 assembly and annotation
            </option><option value="strCam1">
                Ostrich 06 Jun-2014 (ASM69896v1/strCam1) (strCam1)
            </option><option value="falciparum">
                P. falciparum Plasmodium falciparum (?/falciparum) (falciparum)
            </option><option value="priPac2">
                P. pacificus Dec. 2008 (WUGSC 5.0.1/priPac2) (priPac2)
            </option><option value="priPac1">
                P. pacificus Feb. 2007 (WUGSC 5.0/priPac1) (priPac1)
            </option><option value="odoRosDiv1">
                Pacific walrus Jan 2013 (Oros_1.0/odoRosDiv1) (odoRosDiv1)
            </option><option value="chrPic1">
                Painted turtle Dec. 2011 (v3.0.1/chrPic1) (chrPic1)
            </option><option value="chrPic2">
                Painted turtle Mar. 2014 (v3.0.3/chrPic2) (chrPic2)
            </option><option value="ailMel1">
                Panda Dec. 2009 (BGI-Shenzhen 1.0/ailMel1) (ailMel1)
            </option><option value="13485">
                Parabacteroides distasonis ATCC 8503 (13485)
            </option><option value="paraDeni_PD1222">
                Paracoccus denitrificans PD1222 (paraDeni_PD1222)
            </option><option value="ptePar1">
                Parnell's mustached bat Sep. 2013 (ASM46540v1/ptePar1) (ptePar1)
            </option><option value="amaVit1">
                Parrot Jan. 2013 (AV1/amaVit1) (amaVit1)
            </option><option value="17639">
                Parvibaculum lavamentivorans DS-1 (17639)
            </option><option value="pastMult">
                Pasteurella multocida subsp. multocida str. Pm70 (pastMult)
            </option><option value="erwiCaro_ATROSEPTICA">
                Pectobacterium atrosepticum SCRI1043 (erwiCaro_ATROSEPTICA)
            </option><option value="pediPent_ATCC25745">
                Pediococcus pentosaceus ATCC 25745 (pediPent_ATCC25745)
            </option><option value="peloCarb">
                Pelobacter carbinolicus DSM 2380 (peloCarb)
            </option><option value="13384">
                Pelobacter propionicus DSM 2379 (13384)
            </option><option value="peloLute_DSM273">
                Pelodictyon luteolum DSM 273 (peloLute_DSM273)
            </option><option value="peloTher_SI">
                Pelotomaculum thermopropionicum SI (peloTher_SI)
            </option><option value="falPer1">
                Peregrine falcon Feb 2013 (F_peregrinus_v1.0/falPer1) (falPer1)
            </option><option value="17679">
                Petrotoga mobilis SJ95 (17679)
            </option><option value="Petunia_axillaris_Sol_Genomics_v1.6.2">
                Petunia axillaris Sol Genomics v1.6.2
            </option><option value="Phalaenopsis_equestris_Genomics_v1.0">
                Phalaenopsis equestris Genomics v1.0
            </option><option value="photProf_SS9">
                Photobacterium profundum SS9 (photProf_SS9)
            </option><option value="photLumi">
                Photorhabdus luminescens subsp. laumondii TTO1 (photLumi)
            </option><option value="Phyllostachys_edulis_NCGR_v1.0">
                Phyllostachys edulis NCGR v1.0
            </option><option value="Physcomitrella_patens_JGI_assembly_release_v1.1_and_COSMOSS_annotation_v1.6">
                Physcomitrella patens JGI assembly release v1.1 and COSMOSS annotation v1.6
            </option><option value="Physcomitrella_patens_JGI_v3.3">
                Physcomitrella patens JGI v3.3
            </option><option value="picrTorr1">
                Picrophilus torridus DSM 9790 (picrTorr1)
            </option><option value="susScr3">
                Pig Aug. 2011 (SGSC Sscrofa10.2/susScr3) (susScr3)
            </option><option value="susScr11">
                Pig Feb. 2017 (Sscrofa11.1/susScr11) (susScr11)
            </option><option value="Sscrofa9.58">
                Pig May 2010 (Sscrofa9.58)
            </option><option value="susScr2">
                Pig Nov. 2009 (SGSC Sscrofa9.2/susScr2) (susScr2)
            </option><option value="ochPri2">
                Pika Jul. 2008 (Broad/ochPri2) (ochPri2)
            </option><option value="ochPri3">
                Pika May 2012 (OchPri3.0/ochPri3) (ochPri3)
            </option><option value="plasFalc3D7V12">
                Plasmodium falciparum 3d7 10 Sept 2014 (V12.0/plasFalc3D7V12) (plasFalc3D7V12)
            </option><option value="ornAna2">
                Platypus Feb. 2007 (ASM227v2/ornAna2) (ornAna2)
            </option><option value="ornAna1">
                Platypus Mar. 2007 (WUGSC 5.0.1/ornAna1) (ornAna1)
            </option><option value="ursMar1">
                Polar bear 09 May-2014 (UrsMar_1.0/ursMar1) (ursMar1)
            </option><option value="13418">
                Polaromonas naphthalenivorans CJ2 (13418)
            </option><option value="polaJS66">
                Polaromonas sp. JS666 (polaJS66)
            </option><option value="polyQLWP">
                Polynucleobacter sp. QLW-P1DMWA-1 (polyQLWP)
            </option><option value="Ptrichocarpa_156">
                Poplar (Populus trichocarpa) (Ptrichocarpa_156)
            </option><option value="Populus_trichocarpa_JGI_assembly_release_v3.0_annotation_v3.0">
                Populus trichocarpa JGI assembly release v3.0, annotation v3.0
            </option><option value="Populus_trichocarpa_JGI_v3.1">
                Populus trichocarpa JGI v3.1
            </option><option value="porpGing_W83">
                Porphyromonas gingivalis W83 (porpGing_W83)
            </option><option value="perManBai1">
                Prairie deer mouse 03 Dec-2013 (Pman_1.0/perManBai1) (perManBai1)
            </option><option value="micOch1">
                Prairie vole Oct 2012 (MicOch1.0/micOch1) (micOch1)
            </option><option value="priMat13">
                Primate Apr. 24. 2006 (UCSC Recon/priMat13) (priMat13)
            </option><option value="neoBri1">
                Princess of Burundi May 2011 (NeoBri1.0/neoBri1) (neoBri1)
            </option><option value="nasLar1">
                Proboscis monkey Nov. 2014 (Charlie1.0/nasLar1) (nasLar1)
            </option><option value="13548">
                Prochlorococcus marinus str. AS9601 (13548)
            </option><option value="13551">
                Prochlorococcus marinus str. MIT 9211 (13551)
            </option><option value="18633">
                Prochlorococcus marinus str. MIT 9215 (18633)
            </option><option value="15746">
                Prochlorococcus marinus str. MIT 9301 (15746)
            </option><option value="13496">
                Prochlorococcus marinus str. MIT 9303 (13496)
            </option><option value="13910">
                Prochlorococcus marinus str. MIT 9312 (13910)
            </option><option value="220">
                Prochlorococcus marinus str. MIT 9313 (220)
            </option><option value="13617">
                Prochlorococcus marinus str. MIT 9515 (13617)
            </option><option value="15660">
                Prochlorococcus marinus str. NATL1A (15660)
            </option><option value="13911">
                Prochlorococcus marinus str. NATL2A (13911)
            </option><option value="procMari_CCMP1375">
                Prochlorococcus marinus subsp. marinus str. CCMP1375 (procMari_CCMP1375)
            </option><option value="213">
                Prochlorococcus marinus subsp. pastoris str. CCMP1986 (213)
            </option><option value="propAcne_KPA171202">
                Propionibacterium acnes KPA171202 (propAcne_KPA171202)
            </option><option value="12607">
                Prosthecochloris vibrioformis DSM 265 (12607)
            </option><option value="Prunus_persica_JGI_release_v1.0">
                Prunus persica JGI release v1.0
            </option><option value="Prunus_persica_JGI_v2.1">
                Prunus persica JGI v2.1
            </option><option value="equPrz1">
                Przewalski horse Jun 2014 (Burgud/equPrz1) (equPrz1)
            </option><option value="13454">
                Pseudoalteromonas atlantica T6c (13454)
            </option><option value="pseuHalo_TAC125">
                Pseudoalteromonas haloplanktis TAC125 (pseuHalo_TAC125)
            </option><option value="16720">
                Pseudomonas aeruginosa PA7 (16720)
            </option><option value="pseuAeru">
                Pseudomonas aeruginosa PAO1 (pseuAeru)
            </option><option value="386">
                Pseudomonas aeruginosa UCBPP-PA14 (386)
            </option><option value="16800">
                Pseudomonas entomophila L48 (16800)
            </option><option value="327">
                Pseudomonas fluorescens Pf-5 (327)
            </option><option value="12">
                Pseudomonas fluorescens PfO-1 (12)
            </option><option value="17457">
                Pseudomonas mendocina ymp (17457)
            </option><option value="13909">
                Pseudomonas putida F1 (13909)
            </option><option value="17629">
                Pseudomonas putida GB-1 (17629)
            </option><option value="267">
                Pseudomonas putida KT2440 (267)
            </option><option value="17053">
                Pseudomonas putida W619 (17053)
            </option><option value="16817">
                Pseudomonas stutzeri A1501 (16817)
            </option><option value="12416">
                Pseudomonas syringae pv. phaseolicola 1448A (12416)
            </option><option value="323">
                Pseudomonas syringae pv. syringae B728a (323)
            </option><option value="359">
                Pseudomonas syringae pv. tomato str. DC3000 (359)
            </option><option value="psycArct_273_4">
                Psychrobacter arcticus 273-4 (psycArct_273_4)
            </option><option value="13920">
                Psychrobacter cryohalolentis K5 (13920)
            </option><option value="15759">
                Psychrobacter sp. PRwf-1 (15759)
            </option><option value="psycIngr_37">
                Psychromonas ingrahamii 37 (psycIngr_37)
            </option><option value="punNye1">
                Pundamilia nyererei Oct 2011 (PunNye1.0/punNye1) (punNye1)
            </option><option value="Spur_v2.6">
                Purple Sea Urchin (Strongylocentrotus purpuratus) v2.6 (Spur_v2.6)
            </option><option value="pyrAer1">
                Pyrobaculum aerophilum str. IM2 (pyrAer1)
            </option><option value="pyroArse1">
                Pyrobaculum arsenaticum DSM 13514 (pyroArse1)
            </option><option value="pyroCali1">
                Pyrobaculum calidifontis JCM 11548 (pyroCali1)
            </option><option value="pyroIsla1">
                Pyrobaculum islandicum DSM 4184 (pyroIsla1)
            </option><option value="pyrAby1">
                Pyrococcus abyssi GE5 (pyrAby1)
            </option><option value="pyrFur2">
                Pyrococcus furiosus DSM 3638 (pyrFur2)
            </option><option value="pyrHor1">
                Pyrococcus horikoshii OT3 (pyrHor1)
            </option><option value="Pyrus_bretschneideri_Pear_Genome_Project_v1.0">
                Pyrus bretschneideri Pear Genome Project v1.0
            </option><option value="oryCun2">
                Rabbit Apr. 2009 (Broad/oryCun2) (oryCun2)
            </option><option value="oryCun1">
                Rabbit May 2005 (Broad/oryCun1) (oryCun1)
            </option><option value="13603">
                Ralstonia eutropha H16 (13603)
            </option><option value="ralsEutr_JMP134">
                Ralstonia eutropha JMP134 (ralsEutr_JMP134)
            </option><option value="250">
                Ralstonia metallidurans CH34 (250)
            </option><option value="ralsSola">
                Ralstonia solanacearum GMI1000 (ralsSola)
            </option><option value="rn6">
                Rat Jul. 2014 (RGSC 6.0/rn6) (rn6)
            </option><option value="rn3">
                Rat June 2003 (Baylor 3.1/rn3) (rn3)
            </option><option value="rn5">
                Rat Mar. 2012 (RGSC 5.0/rn5) (rn5)
            </option><option value="rn4">
                Rat Nov. 2004 (Baylor 3.4/rn4) (rn4)
            </option><option value="rn7">
                Rat Nov. 2020 (mRatBN7.2/rn7) (rn7)
            </option><option value="Tcas_3.0">
                Red Flour Beetle (Tribolium castaneum) (Tcas_3.0)
            </option><option value="tauEry1">
                Red crested turaco Jun 2014 (ASM70936v1/tauEry1) (tauEry1)
            </option><option value="carCri1">
                Red-legged seriema May 2014 (ASM69053v1/carCri1) (carCri1)
            </option><option value="gavSte1">
                Red-throated loon May 2014 (ASM69087v1/gavSte1) (gavSte1)
            </option><option value="19227">
                Renibacterium salmoninarum ATCC 33209 (19227)
            </option><option value="repBase0">
                RepBase Jun. 2012 (hg19/repBase0) (repBase0)
            </option><option value="repBase1">
                RepBase Sep. 2012 (17.08/repBase1) (repBase1)
            </option><option value="resEbo1">
                Reston ebolavirus Aug. 2014 (NC_004161.1/resEbo1) (resEbo1)
            </option><option value="rheMac10">
                Rhesus Feb. 2019 (Mmul_10/rheMac10) (rheMac10)
            </option><option value="rheMac2">
                Rhesus Jan. 2006 (MGSC Merged 1.0/rheMac2) (rheMac2)
            </option><option value="rheMac7">
                Rhesus Nov. 2014 (UMD MacaM_Assembly_v7/rheMac7) (rheMac7)
            </option><option value="rheMac8">
                Rhesus Nov. 2015 (BCM Mmul_8.0.1/rheMac8) (rheMac8)
            </option><option value="rheMac3">
                Rhesus Oct. 2010 (BGI CR_1.0/rheMac3) (rheMac3)
            </option><option value="bucRhi1">
                Rhinoceros hornbill Jun 2014 (ASM71030v1/bucRhi1) (bucRhi1)
            </option><option value="rhizEtli_CFN_42">
                Rhizobium etli CFN 42 (rhizEtli_CFN_42)
            </option><option value="344">
                Rhizobium leguminosarum bv. viciae 3841 (344)
            </option><option value="rhodSpha_2_4_1">
                Rhodobacter sphaeroides 2.4.1 (rhodSpha_2_4_1)
            </option><option value="15755">
                Rhodobacter sphaeroides ATCC 17025 (15755)
            </option><option value="15754">
                Rhodobacter sphaeroides ATCC 17029 (15754)
            </option><option value="rhodRHA1">
                Rhodococcus sp. RHA1 (rhodRHA1)
            </option><option value="13908">
                Rhodoferax ferrireducens T118 (13908)
            </option><option value="pireSp">
                Rhodopirellula baltica SH 1 (pireSp)
            </option><option value="15751">
                Rhodopseudomonas palustris BisA53 (15751)
            </option><option value="15750">
                Rhodopseudomonas palustris BisB18 (15750)
            </option><option value="15749">
                Rhodopseudomonas palustris BisB5 (15749)
            </option><option value="rhodPalu_CGA009">
                Rhodopseudomonas palustris CGA009 (rhodPalu_CGA009)
            </option><option value="15747">
                Rhodopseudomonas palustris HaA2 (15747)
            </option><option value="rhodRubr_ATCC11170">
                Rhodospirillum rubrum ATCC 11170 (rhodRubr_ATCC11170)
            </option><option value="oryza_sativa_japonica_nipponbare_IRGSP4.0">
                Rice (Oryza sativa ssp. japonica var. Nipponbare) (oryza_sativa_japonica_nipponbare_IRGSP4.0)
            </option><option value="Ricinus_communis_JCVI_1.0">
                Ricinus communis JCVI 1.0
            </option><option value="Ricinus_communis_JGI_v0.1">
                Ricinus communis JGI v0.1
            </option><option value="12953">
                Rickettsia akari str. Hartford (12953)
            </option><option value="17237">
                Rickettsia bellii OSU 85-389 (17237)
            </option><option value="rickBell_RML369_C">
                Rickettsia bellii RML369-C (rickBell_RML369_C)
            </option><option value="12952">
                Rickettsia canadensis str. McKiel (12952)
            </option><option value="42">
                Rickettsia conorii str. Malish 7 (42)
            </option><option value="13884">
                Rickettsia felis URRWXCal2 (13884)
            </option><option value="18271">
                Rickettsia massiliae MTU5 (18271)
            </option><option value="43">
                Rickettsia prowazekii str. Madrid E (43)
            </option><option value="9636">
                Rickettsia rickettsii str. 'Sheila Smith' (9636)
            </option><option value="19943">
                Rickettsia rickettsii str. Iowa (19943)
            </option><option value="10679">
                Rickettsia typhi str. Wilmington (10679)
            </option><option value="acaChl1">
                Rifleman 27 May 2014 (ASM69581v1/acaChl1) (acaChl1)
            </option><option value="proCap1">
                Rock hyrax Jul. 2008 (Broad/proCap1) (proCap1)
            </option><option value="colLiv2.1">
                Rock pigeon Dec 2017 (Cliv_2.1/colLiv2)
            </option><option value="colLiv1">
                Rock pigeon Feb 2013 (Cliv_1.0/colLiv1) (colLiv1)
            </option><option value="rodEnt13">
                Rodent Apr. 24. 2006 (UCSC Recon/rodEnt13) (rodEnt13)
            </option><option value="13462">
                Roseiflexus castenholzii DSM 13941 (13462)
            </option><option value="16190">
                Roseiflexus sp. RS-1 (16190)
            </option><option value="roseDeni_OCH_114">
                Roseobacter denitrificans OCh 114 (roseDeni_OCH_114)
            </option><option value="rubrXyla_DSM9941">
                Rubrobacter xylanophilus DSM 9941 (rubrXyla_DSM9941)
            </option><option value="Saccharomyces_cerevisiae_S288C_SGD2010">
                S. cerevisae str. S288C (Saccharomyces_cerevisiae_S288C_SGD2010)
            </option><option value="sacCer3">
                S. cerevisiae Apr. 2011 (SacCer_Apr2011/sacCer3) (sacCer3)
            </option><option value="sacCer2">
                S. cerevisiae June 2008 (SGD/sacCer2) (sacCer2)
            </option><option value="sacCer1">
                S. cerevisiae Oct. 2003 (SGD/sacCer1) (sacCer1)
            </option><option value="SL4.0">
                S. lycopersicum (Tomato) Feb 2019 (SL4.0)
            </option><option value="strPur1">
                S. purpuratus Apr. 2005 (Baylor 1.1/strPur1) (strPur1)
            </option><option value="strPur3">
                S. purpuratus Feb. 2009 (BCM-HGSC Spur_v2.5/strPur3) (strPur3)
            </option><option value="strPur4">
                S. purpuratus Jun. 2011 (BCM-HGSC Spur_v3.1/strPur4) (strPur4)
            </option><option value="strPur2">
                S. purpuratus Sep. 2006 (Baylor 2.1/strPur2) (strPur2)
            </option><option value="s.enterica_NC_016810.1">
                S.enterica_NC_016810.1
            </option><option value="sc1">
                SARS coronavirus Apr. 2003 (GenBank Apr. 14 '03/sc1) (sc1)
            </option><option value="wuhCor1">
                SARS-CoV-2 Jan. 2020 (NC_045512.2) (wuhCor1)
            </option><option value="NC_045512.2">
                SARS-CoV-2 isolate Wuhan-Hu-1, complete genome (NC_045512)
            </option><option value="saccDegr_2_40">
                Saccharophagus degradans 2-40 (saccDegr_2_40)
            </option><option value="saccEryt_NRRL_2338">
                Saccharopolyspora erythraea NRRL 2338 (saccEryt_NRRL_2338)
            </option><option value="Saccharum_spontaneum_v20190103">
                Saccharum spontaneum v20190103
            </option><option value="falChe1">
                Saker falcon Feb 2013 (F_cherrug_v1.0/falChe1) (falChe1)
            </option><option value="saliRube_DSM13855">
                Salinibacter ruber DSM 13855 (saliRube_DSM13855)
            </option><option value="17109">
                Salinispora arenicola CNS-205 (17109)
            </option><option value="saliTrop_CNB_440">
                Salinispora tropica CNB-440 (saliTrop_CNB_440)
            </option><option value="S. enterica (NC_016810.1)">
                Salmonella enterica (NC_016810.1)
            </option><option value="13030">
                Salmonella enterica subsp. arizonae serovar 62:z4,z23:-- (13030)
            </option><option value="9618">
                Salmonella enterica subsp. enterica serovar Choleraesuis str. SC-B67 (9618)
            </option><option value="salmEnte_PARATYPI_ATC">
                Salmonella enterica subsp. enterica serovar Paratyphi A str. ATCC 9150 (salmEnte_PARATYPI_ATC)
            </option><option value="27803">
                Salmonella enterica subsp. enterica serovar Paratyphi B str. SPB7 (27803)
            </option><option value="salmTyph">
                Salmonella enterica subsp. enterica serovar Typhi str. CT18 (salmTyph)
            </option><option value="salmTyph_TY2">
                Salmonella enterica subsp. enterica serovar Typhi str. Ty2 (salmTyph_TY2)
            </option><option value="241">
                Salmonella typhimurium LT2 (241)
            </option><option value="croPor0">
                Saltwater crocodile June 2011 (UCSC cPoro_v0.1c45/croPor0) (croPor0)
            </option><option value="araMac1">
                Scarlet Macaw Jun 2013 (SMACv1.1/araMac1) (araMac1)
            </option><option value="Schrenkiella_parvula_Thellungiella_v2.0_TpV84">
                Schrenkiella parvula Thellungiella v2.0 (TpV84)
            </option><option value="aplCal0">
                Sea hare Jul. 2013 (aplCal0/aplCal0) (aplCal0)
            </option><option value="aplCal1">
                Sea hare Sept. 2008 (Broad 2.0/aplCal1) (aplCal1)
            </option><option value="Selaginella_moellendorffii_JGI_v1.0">
                Selaginella moellendorffii JGI v1.0
            </option><option value="Selaginella_moellendorffii_NCBIr100">
                Selaginella moellendorffii NCBIr100
            </option><option value="17459">
                Serratia proteamaculans 568 (17459)
            </option><option value="Setaria_italica_JGI_8.3X_chromosome-scale_assembly_release_2.0_annotation_version_2.1">
                Setaria italica JGI 8.3X chromosome-scale assembly release 2.0, annotation version 2.1
            </option><option value="Setaria_italica_JGI_v2.2">
                Setaria italica JGI v2.2
            </option><option value="oviAri3">
                Sheep Aug. 2012 (ISGC Oar_v3.1/oviAri3) (oviAri3)
            </option><option value="oviAri1">
                Sheep Feb. 2010 (ISGC Ovis_aries_1.0/oviAri1) (oviAri1)
            </option><option value="oviAri4">
                Sheep Nov. 2015 (Oar_v4.0/oviAri4) (oviAri4)
            </option><option value="cypVar1">
                Sheepshead minnow Jul 2014 (C_variegatus-1.0/cypVar1) (cypVar1)
            </option><option value="shewAmaz">
                Shewanella amazonensis SB2B (shewAmaz)
            </option><option value="shewBalt">
                Shewanella baltica OS155 (shewBalt)
            </option><option value="17643">
                Shewanella baltica OS185 (17643)
            </option><option value="13389">
                Shewanella baltica OS195 (13389)
            </option><option value="shewDeni">
                Shewanella denitrificans OS217 (shewDeni)
            </option><option value="shewFrig">
                Shewanella frigidimarina NCIMB 400 (shewFrig)
            </option><option value="20241">
                Shewanella halifaxensis HAW-EB4 (20241)
            </option><option value="shewLoihPV4">
                Shewanella loihica PV-4 (shewLoihPV4)
            </option><option value="shewOnei">
                Shewanella oneidensis MR-1 (shewOnei)
            </option><option value="17415">
                Shewanella pealeana ATCC 700345 (17415)
            </option><option value="shewPutrCN32">
                Shewanella putrefaciens CN-32 (shewPutrCN32)
            </option><option value="18789">
                Shewanella sediminis HAW-EB3 (18789)
            </option><option value="shewANA3">
                Shewanella sp. ANA-3 (shewANA3)
            </option><option value="shewMR4">
                Shewanella sp. MR-4 (shewMR4)
            </option><option value="shewMR7">
                Shewanella sp. MR-7 (shewMR7)
            </option><option value="shewW318">
                Shewanella sp. W3-18-1 (shewW318)
            </option><option value="17455">
                Shewanella woodyi ATCC 51908 (17455)
            </option><option value="13146">
                Shigella boydii Sb227 (13146)
            </option><option value="13145">
                Shigella dysenteriae Sd197 (13145)
            </option><option value="408">
                Shigella flexneri 2a str. 2457T (408)
            </option><option value="shigFlex_2A">
                Shigella flexneri 2a str. 301 (shigFlex_2A)
            </option><option value="16375">
                Shigella flexneri 5 str. 8401 (16375)
            </option><option value="13151">
                Shigella sonnei Ss046 (13151)
            </option><option value="sorAra2">
                Shrew Aug. 2008 (Broad/sorAra2) (sorAra2)
            </option><option value="sorAra1">
                Shrew June 2006 (Broad/sorAra1) (sorAra1)
            </option><option value="siliPome_DSS_3">
                Silicibacter pomeroyi DSS-3 (siliPome_DSS_3)
            </option><option value="13040">
                Silicibacter sp. TM1040 (13040)
            </option><option value="Bombyx_mori_p50T_2.0">
                Silkworm (Bombyx_mori_p50T_2.0)
            </option><option value="simHuman">
                Simhuman Oct. 2011 (simHuman/simHuman) (simHuman)
            </option><option value="simHumanMammal">
                Simhumanmammal Oct. 2011 (simHumanMammal/simHumanMammal) (simHumanMammal)
            </option><option value="simHumanMam">
                Simmam Jan. 2011 (SimMamV1/simHumanMam) (simHumanMam)
            </option><option value="simHumanPrim">
                Simprim Jan. 2011 (SimPrimV1/simHumanPrim) (simHumanPrim)
            </option><option value="simRefMam">
                Simrefmam Jan. 2011 (SimRefMamV1/simRefMam) (simRefMam)
            </option><option value="16304">
                Sinorhizobium medicae WSM419 (16304)
            </option><option value="sinoMeli">
                Sinorhizobium meliloti 1021 (sinoMeli)
            </option><option value="choHof1">
                Sloth Jul. 2008 (Broad/choHof1) (choHof1)
            </option><option value="bioGla0">
                Snail Jun. 2013 (bioGla0/bioGla0) (bioGla0)
            </option><option value="sodaGlos_MORSITANS">
                Sodalis glossinidius str. 'morsitans' (sodaGlos_MORSITANS)
            </option><option value="Solanum_lycopersicum_ITAG_2.3">
                Solanum lycopersicum ITAG 2.3
            </option><option value="Solanum_lycopersicum_Sol_Genomics_itag2.4">
                Solanum lycopersicum Sol Genomics itag2.4
            </option><option value="Solanum_tuberosum_ITAG_001">
                Solanum tuberosum ITAG 001
            </option><option value="Solanum_tuberosum_JGI_v4.03">
                Solanum tuberosum JGI v4.03
            </option><option value="soliUsit_ELLIN6076">
                Solibacter usitatus Ellin6076 (soliUsit_ELLIN6076)
            </option><option value="28111">
                Sorangium cellulosum 'So ce 56' (28111)
            </option><option value="Sorghum_bicolor_JGI_1.4">
                Sorghum bicolor JGI 1.4
            </option><option value="Sorghum_bicolor_JGI_v3.1">
                Sorghum bicolor JGI v3.1
            </option><option value="xipMac1">
                Southern platyfish Jan 2012 (Xiphophorus_maculatus-4.4.2/xipMac1) (xipMac1)
            </option><option value="enhLutNer1">
                Southern sea otter Jun. 2019 (ASM641071v1/enhLutNer1) (enhLutNer1)
            </option><option value="colStr1">
                Speckled mousebird May 2014 (ASM69071v1/colStr1) (colStr1)
            </option><option value="phyCat1">
                Sperm whale Sep. 2013 (Physeter_macrocephalus-2.0.2/phyCat1) (phyCat1)
            </option><option value="17343">
                Sphingomonas wittichii RW1 (17343)
            </option><option value="sphiAlas_RB2256">
                Sphingopyxis alaskensis RB2256 (sphiAlas_RB2256)
            </option><option value="apaSpi1">
                Spiny softshell turtle May 2013 (ASM38561v1/apaSpi1) (apaSpi1)
            </option><option value="Spirodela_polyrhiza_JGI_v2.0">
                Spirodela polyrhiza JGI v2.0
            </option><option value="lepOcu1">
                Spotted gar Dec 2011 (LepOcu1/lepOcu1) (lepOcu1)
            </option><option value="speTri1">
                Squirrel Feb. 2008 (Broad/speTri1) (speTri1)
            </option><option value="speTri2">
                Squirrel Nov. 2011 (Broad/speTri2) (speTri2)
            </option><option value="saiBol1">
                Squirrel monkey Oct. 2011 (Broad/saiBol1) (saiBol1)
            </option><option value="63">
                Staphylococcus aureus RF122 (63)
            </option><option value="238">
                Staphylococcus aureus subsp. aureus COL (238)
            </option><option value="15758">
                Staphylococcus aureus subsp. aureus JH1 (15758)
            </option><option value="15757">
                Staphylococcus aureus subsp. aureus JH9 (15757)
            </option><option value="265">
                Staphylococcus aureus subsp. aureus MRSA252 (265)
            </option><option value="266">
                Staphylococcus aureus subsp. aureus MSSA476 (266)
            </option><option value="306">
                Staphylococcus aureus subsp. aureus MW2 (306)
            </option><option value="18509">
                Staphylococcus aureus subsp. aureus Mu3 (18509)
            </option><option value="stapAure_MU50">
                Staphylococcus aureus subsp. aureus Mu50 (stapAure_MU50)
            </option><option value="264">
                Staphylococcus aureus subsp. aureus N315 (264)
            </option><option value="237">
                Staphylococcus aureus subsp. aureus NCTC 8325 (237)
            </option><option value="16313">
                Staphylococcus aureus subsp. aureus USA300 (16313)
            </option><option value="19489">
                Staphylococcus aureus subsp. aureus USA300_TCH1516 (19489)
            </option><option value="18801">
                Staphylococcus aureus subsp. aureus str. Newman (18801)
            </option><option value="279">
                Staphylococcus epidermidis ATCC 12228 (279)
            </option><option value="64">
                Staphylococcus epidermidis RP62A (64)
            </option><option value="12508">
                Staphylococcus haemolyticus JCSC1435 (12508)
            </option><option value="15596">
                Staphylococcus saprophyticus subsp. saprophyticus ATCC 15305 (15596)
            </option><option value="stapMari1">
                Staphylothermus marinus F1 (stapMari1)
            </option><option value="conCri1">
                Star-nosed mole Mar 2012 (ConCri1.0/conCri1) (conCri1)
            </option><option value="nemVec1">
                Starlet sea anemone Jun. 2007 (JGI-PDF/nemVec1) (nemVec1)
            </option><option value="gasAcu1">
                Stickleback Feb. 2006 (Broad/gasAcu1) (gasAcu1)
            </option><option value="eidHel1">
                Straw colored fruit bat 13 Sep 2013 (ASM46528v1/eidHel1) (eidHel1)
            </option><option value="330">
                Streptococcus agalactiae 2603V/R (330)
            </option><option value="326">
                Streptococcus agalactiae A909 (326)
            </option><option value="334">
                Streptococcus agalactiae NEM316 (334)
            </option><option value="66">
                Streptococcus gordonii str. Challis substr. CH1 (66)
            </option><option value="333">
                Streptococcus mutans UA159 (333)
            </option><option value="16374">
                Streptococcus pneumoniae D39 (16374)
            </option><option value="28035">
                Streptococcus pneumoniae Hungary19A-6 (28035)
            </option><option value="278">
                Streptococcus pneumoniae R6 (278)
            </option><option value="277">
                Streptococcus pneumoniae TIGR4 (277)
            </option><option value="strePyog_M1_GAS">
                Streptococcus pyogenes M1 GAS (strePyog_M1_GAS)
            </option><option value="16364">
                Streptococcus pyogenes MGAS10270 (16364)
            </option><option value="12469">
                Streptococcus pyogenes MGAS10394 (12469)
            </option><option value="16366">
                Streptococcus pyogenes MGAS10750 (16366)
            </option><option value="16365">
                Streptococcus pyogenes MGAS2096 (16365)
            </option><option value="311">
                Streptococcus pyogenes MGAS315 (311)
            </option><option value="13888">
                Streptococcus pyogenes MGAS5005 (13888)
            </option><option value="13887">
                Streptococcus pyogenes MGAS6180 (13887)
            </option><option value="286">
                Streptococcus pyogenes MGAS8232 (286)
            </option><option value="16363">
                Streptococcus pyogenes MGAS9429 (16363)
            </option><option value="301">
                Streptococcus pyogenes SSI-1 (301)
            </option><option value="270">
                Streptococcus pyogenes str. Manfredo (270)
            </option><option value="13942">
                Streptococcus sanguinis SK36 (13942)
            </option><option value="17153">
                Streptococcus suis 05ZYH33 (17153)
            </option><option value="17155">
                Streptococcus suis 98HAH33 (17155)
            </option><option value="13163">
                Streptococcus thermophilus CNRZ1066 (13163)
            </option><option value="13773">
                Streptococcus thermophilus LMD-9 (13773)
            </option><option value="13162">
                Streptococcus thermophilus LMG 18311 (13162)
            </option><option value="189">
                Streptomyces avermitilis MA-4680 (189)
            </option><option value="streCoel">
                Streptomyces coelicolor A3(2) (streCoel)
            </option><option value="sudEbo1">
                Sudan ebolavirus Aug. 2014 (NC_006432.1/sudEbo1) (sudEbo1)
            </option><option value="sulfAcid1">
                Sulfolobus acidocaldarius DSM 639 (sulfAcid1)
            </option><option value="sulSol1">
                Sulfolobus solfataricus P2 (sulSol1)
            </option><option value="sulfToko1">
                Sulfolobus tokodaii str. 7 (sulfToko1)
            </option><option value="thioDeni_ATCC33889">
                Sulfurimonas denitrificans DSM 1251 (thioDeni_ATCC33889)
            </option><option value="18965">
                Sulfurovum sp. NBC37-1 (18965)
            </option><option value="eurHel1">
                Sunbittern May 2014 (ASM69077v1/eurHel1) (eurHel1)
            </option><option value="symbTher_IAM14863">
                Symbiobacterium thermophilum IAM 14863 (symbTher_IAM14863)
            </option><option value="13282">
                Synechococcus elongatus PCC 6301 (13282)
            </option><option value="10645">
                Synechococcus elongatus PCC 7942 (10645)
            </option><option value="12530">
                Synechococcus sp. CC9311 (12530)
            </option><option value="13643">
                Synechococcus sp. CC9605 (13643)
            </option><option value="13655">
                Synechococcus sp. CC9902 (13655)
            </option><option value="16252">
                Synechococcus sp. JA-2-3B'a(2-13) (16252)
            </option><option value="16251">
                Synechococcus sp. JA-3-3Ab (16251)
            </option><option value="28247">
                Synechococcus sp. PCC 7002 (28247)
            </option><option value="13654">
                Synechococcus sp. RCC307 (13654)
            </option><option value="13642">
                Synechococcus sp. WH 7803 (13642)
            </option><option value="syneSp_WH8102">
                Synechococcus sp. WH 8102 (syneSp_WH8102)
            </option><option value="synePCC6">
                Synechocystis sp. PCC 6803 (synePCC6)
            </option><option value="syntFuma_MPOB">
                Syntrophobacter fumaroxidans MPOB (syntFuma_MPOB)
            </option><option value="syntWolf_GOETTINGEN">
                Syntrophomonas wolfei subsp. wolfei str. Goettingen (syntWolf_GOETTINGEN)
            </option><option value="syntAcid_SB">
                Syntrophus aciditrophicus SB (syntAcid_SB)
            </option><option value="triCas2">
                T. castaneum Sep. 2005 (Baylor 2.0/triCas2) (triCas2)
            </option><option value="taiEbo1">
                Tai forest ebolavirus Aug. 2014 (NC_014372.1/taiEbo1) (taiEbo1)
            </option><option value="Tarenaya_hassleriana_RefSeq_v1">
                Tarenaya hassleriana RefSeq v1
            </option><option value="tarSyr1">
                Tarsier Aug. 2008 (Broad/tarSyr1) (tarSyr1)
            </option><option value="tarSyr2">
                Tarsier Sep. 2013 (Tarsius_syrichta-2.0.1/tarSyr2) (tarSyr2)
            </option><option value="sarHar1">
                Tasmanian devil Feb. 2011 (WTSI Devil_ref v7.0/sarHar1) (sarHar1)
            </option><option value="echTel1">
                Tenrec July 2005 (Broad/echTel1) (echTel1)
            </option><option value="echTel2">
                Tenrec Nov. 2012 (Broad/echTel2) (echTel2)
            </option><option value="tetNig1">
                Tetraodon Feb. 2004 (Genoscope 7/tetNig1) (tetNig1)
            </option><option value="tetNig2">
                Tetraodon Mar. 2007 (Genoscope 8.0/tetNig2) (tetNig2)
            </option><option value="Thellungiella_parvula_TpV84">
                Thellungiella parvula TpV84
            </option><option value="Theobroma_cacao_D._Gilbert_public_gene_set_8_Mar_2012_on_assembly_v1.1">
                Theobroma cacao D. Gilbert public gene set 8 Mar 2012 on assembly v1.1
            </option><option value="Tcacao_1.0">
                Theobroma cacao Dec 2010 (CIRAD/Theobroma cacao Belizian Criollo) (Tcacao_1.0)
            </option><option value="Theobroma_cacao_RefSeq_v1.1">
                Theobroma cacao RefSeq v1.1
            </option><option value="13901">
                Thermoanaerobacter pseudethanolicus ATCC 33223 (13901)
            </option><option value="16394">
                Thermoanaerobacter sp. X514 (16394)
            </option><option value="therTeng">
                Thermoanaerobacter tengcongensis MB4 (therTeng)
            </option><option value="therFusc_YX">
                Thermobifida fusca YX (therFusc_YX)
            </option><option value="therKoda1">
                Thermococcus kodakarensis KOD1 (therKoda1)
            </option><option value="therPend1">
                Thermofilum pendens Hrk 5 (therPend1)
            </option><option value="therAcid1">
                Thermoplasma acidophilum DSM 1728 (therAcid1)
            </option><option value="therVolc1">
                Thermoplasma volcanium GSS1 (therVolc1)
            </option><option value="17249">
                Thermosipho melanesiensis BI429 (17249)
            </option><option value="therElon">
                Thermosynechococcus elongatus BP-1 (therElon)
            </option><option value="15644">
                Thermotoga lettingae TMO (15644)
            </option><option value="therMari">
                Thermotoga maritima MSB8 (therMari)
            </option><option value="therPetr_RKU_1">
                Thermotoga petrophila RKU-1 (therPetr_RKU_1)
            </option><option value="19543">
                Thermotoga sp. RQ2 (19543)
            </option><option value="therTher_HB27">
                Thermus thermophilus HB27 (therTher_HB27)
            </option><option value="therTher_HB8">
                Thermus thermophilus HB8 (therTher_HB8)
            </option><option value="thioDeni_ATCC25259">
                Thiobacillus denitrificans ATCC 25259 (thioDeni_ATCC25259)
            </option><option value="thioCrun_XCL_2">
                Thiomicrospira crunogena XCL-2 (thioCrun_XCL_2)
            </option><option value="panHod1">
                Tibetan antelope May 2013 (PHO1.0/panHod1) (panHod1)
            </option><option value="nanPar1">
                Tibetan frog Mar. 2015 (BGI_ZX_2015/nanPar1) (nanPar1)
            </option><option value="pseHum1">
                Tibetan ground jay Jan 2013 (PseHum1.0/pseHum1) (pseHum1)
            </option><option value="sebNig1">
                Tiger rockfish 22 Oct-2013 (Snig1.0/sebNig1) (sebNig1)
            </option><option value="cynSem1">
                Tongue sole 28 Jan-2014 (Cse_v1.0/cynSem1) (cynSem1)
            </option><option value="tupBel1">
                Tree shrew Dec. 2006 (Broad/tupBel1) (tupBel1)
            </option><option value="4">
                Treponema denticola ATCC 35405 (4)
            </option><option value="trepPall">
                Treponema pallidum subsp. pallidum str. Nichols (trepPall)
            </option><option value="tricEryt_IMS101">
                Trichodesmium erythraeum IMS101 (tricEryt_IMS101)
            </option><option value="Trifolium_pratense_JGI_v2">
                Trifolium pratense JGI v2
            </option><option value="Triticum_aestivum_Ensembl_Plants_TGACv1">
                Triticum aestivum Ensembl Plants TGACv1
            </option><option value="Triticum_aestivum_IWGSC1.1">
                Triticum aestivum IWGSC1.1
            </option><option value="Triticum_turgidum_Svevo_v1.0">
                Triticum turgidum Svevo v1.0
            </option><option value="tropWhip_TW08_27">
                Tropheryma whipplei TW08/27 (tropWhip_TW08_27)
            </option><option value="95">
                Tropheryma whipplei str. Twist (95)
            </option><option value="TriTrypDB-37_TbruceiLister427">
                Trypanosoma brucei 427 Lister strain genome build 37
            </option><option value="melGal1">
                Turkey Dec. 2009 (TGC Turkey_2.01/melGal1) (melGal1)
            </option><option value="melGal5">
                Turkey Nov. 2014 (Turkey_5.0/melGal5) (melGal5)
            </option><option value="nanGal1">
                Upper Galilee mountains blind mole rat Jun 2014 (S.galili_v1.0/nanGal1) (nanGal1)
            </option><option value="19087">
                Ureaplasma parvum serovar 3 str. ATCC 27815 (19087)
            </option><option value="ureaUrea">
                Ureaplasma parvum serovar 3 str. ATCC 700970 (ureaUrea)
            </option><option value="Utricularia_gibba_CoGe_v4">
                Utricularia gibba CoGe v4
            </option><option value="vermEise_EF01_2">
                Verminephrobacter eiseniae EF01-2 (vermEise_EF01_2)
            </option><option value="vibrChol1">
                Vibrio cholerae O1 biovar eltor str. N16961 (vibrChol1)
            </option><option value="vibrChol_O395_1">
                Vibrio cholerae O395 (vibrChol_O395_1)
            </option><option value="vibrFisc_ES114_1">
                Vibrio fischeri ES114 (vibrFisc_ES114_1)
            </option><option value="19857">
                Vibrio harveyi ATCC BAA-1116 (19857)
            </option><option value="vibrPara1">
                Vibrio parahaemolyticus RIMD 2210633 (vibrPara1)
            </option><option value="vibrVuln_CMCP6_1">
                Vibrio vulnificus CMCP6 (vibrVuln_CMCP6_1)
            </option><option value="vibrVuln_YJ016_1">
                Vibrio vulnificus YJ016 (vibrVuln_YJ016_1)
            </option><option value="Vigna_radiata_var._radiata_LIS_v6">
                Vigna radiata var. radiata LIS v6
            </option><option value="Vitis_vinifera_JGI_12xMarch2010">
                Vitis vinifera JGI 12xMarch2010
            </option><option value="Vitis_vinifera_genescope_v1">
                Vitis vinifera genescope v1
            </option><option value="macEug1">
                Wallaby Nov. 2007 (Baylor 1.0/macEug1) (macEug1)
            </option><option value="macEug2">
                Wallaby Sep. 2009 (TWGS Meug_1.1/macEug2) (macEug2)
            </option><option value="bubBub1">
                Water buffalo Sep. 2013 (UMD_CASPUR_WB_2.0/bubBub1) (bubBub1)
            </option><option value="lepWed1">
                Weddell seal Mar 2013 (LepWed1.0/lepWed1) (lepWed1)
            </option><option value="GCF_019279795.1">
                West African lungfish Jul 2021 (Protopterus annectens, PAN1.0) (GCF_019279795.1)
            </option><option value="cerSim1">
                White rhinoceros May 2012 (CerSimSim1.0/cerSim1) (cerSim1)
            </option><option value="tinGut2">
                White throated tinamou Aug 2014 (ASM70537v2/tinGut2) (tinGut2)
            </option><option value="tinGut1">
                White throated tinamou Jun 2014 (ASM70537v1/tinGut1) (tinGut1)
            </option><option value="GCF_002102435.1">
                White-tailed deer Apr 2017 (Odocoileus virginianus) (GCF_002102435.1, Ovir.te_1.0)
            </option><option value="halAlb1">
                White-tailed eagle May 2014 (ASM69140v1/halAlb1) (halAlb1)
            </option><option value="phaLep1">
                White-tailed tropicbird May 2014 (ASM68728v1/phaLep1) (phaLep1)
            </option><option value="zonAlb1">
                White-throated sparrow Apr 2013 (ASM38545v1/zonAlb1) (zonAlb1)
            </option><option value="wiggBrev">
                Wigglesworthia glossinidia endosymbiont of Glossina brevipalpis (wiggBrev)
            </option><option value="bosMut1">
                Wild yak Oct 2011 (BosGru_v2.0/bosMut1) (bosMut1)
            </option><option value="wolbEndo_OF_DROSOPHIL">
                Wolbachia endosymbiont of Drosophila melanogaster (wolbEndo_OF_DROSOPHIL)
            </option><option value="12475">
                Wolbachia endosymbiont strain TRS of Brugia malayi (12475)
            </option><option value="woliSucc">
                Wolinella succinogenes DSM 1740 (woliSucc)
            </option><option value="xenTro2">
                X. tropicalis Aug. 2005 (JGI 4.1/xenTro2) (xenTro2)
            </option><option value="xenTro9">
                X. tropicalis Jul. 2016 (Xenopus_tropicalis_v9.1/xenTro9) (xenTro9)
            </option><option value="xenTro3">
                X. tropicalis Nov. 2009 (JGI 4.2/xenTro3) (xenTro3)
            </option><option value="xenTro10">
                X. tropicalis Nov. 2019 (UCB_Xtro_10.0/xenTro10) (xenTro10)
            </option><option value="xenTro1">
                X. tropicalis Oct. 2004 (JGI 3.0/xenTro1) (xenTro1)
            </option><option value="xenTro7">
                X. tropicalis Sep. 2012 (JGI 7.0/xenTro7) (xenTro7)
            </option><option value="15756">
                Xanthobacter autotrophicus Py2 (15756)
            </option><option value="297">
                Xanthomonas axonopodis pv. citri str. 306 (297)
            </option><option value="15">
                Xanthomonas campestris pv. campestris str. 8004 (15)
            </option><option value="xantCamp">
                Xanthomonas campestris pv. campestris str. ATCC 33913 (xantCamp)
            </option><option value="13649">
                Xanthomonas campestris pv. vesicatoria str. 85-10 (13649)
            </option><option value="12931">
                Xanthomonas oryzae pv. oryzae KACC10331 (12931)
            </option><option value="16297">
                Xanthomonas oryzae pv. oryzae MAFF 311018 (16297)
            </option><option value="Xanthomonas_oryzae_PXO99A">
                Xanthomonas oryzae pv. oryzae PXO99A (Xanthomonas_oryzae_PXO99A)
            </option><option value="xenTro9.1">
                Xenopus tropicalis (xenTro9.1)
            </option><option value="xyleFast">
                Xylella fastidiosa 9a5c (xyleFast)
            </option><option value="17823">
                Xylella fastidiosa M12 (17823)
            </option><option value="285">
                Xylella fastidiosa Temecula1 (285)
            </option><option value="lipVex1">
                Yangtze river dolphin 31 Jul 2013 (Lipotes_vexillifer_v1/lipVex1) (lipVex1)
            </option><option value="pteGut1">
                Yellow-throated sandgrouse Jun 2014 (ASM69924v1/pteGut1) (pteGut1)
            </option><option value="takFla1">
                Yellowbelly pufferfish May 2013 (version 1 of Takifugu flavidus genome/takFla1) (takFla1)
            </option><option value="190">
                Yersinia enterocolitica subsp. enterocolitica 8081 (190)
            </option><option value="16067">
                Yersinia pestis Angola (16067)
            </option><option value="16645">
                Yersinia pestis Antiqua (16645)
            </option><option value="yersPest_CO92">
                Yersinia pestis CO92 (yersPest_CO92)
            </option><option value="288">
                Yersinia pestis KIM (288)
            </option><option value="16646">
                Yersinia pestis Nepal516 (16646)
            </option><option value="16700">
                Yersinia pestis Pestoides F (16700)
            </option><option value="10638">
                Yersinia pestis biovar Microtus str. 91001 (10638)
            </option><option value="16070">
                Yersinia pseudotuberculosis IP 31758 (16070)
            </option><option value="12950">
                Yersinia pseudotuberculosis IP 32953 (12950)
            </option><option value="28743">
                Yersinia pseudotuberculosis YPIII (28743)
            </option><option value="zaiEbo1">
                Zaire ebolavirus Aug. 2014 (NC_002549.1/zaiEbo1) (zaiEbo1)
            </option><option value="Zea_mays_B73_AGP_v4.0">
                Zea mays B73 AGP v4.0
            </option><option value="Zea_mays_PH207_v1.0">
                Zea mays PH207 v1.0
            </option><option value="mayZeb1">
                Zebra Mbuna Mar 2012 (MetZeb1.1/mayZeb1) (mayZeb1)
            </option><option value="taeGut2">
                Zebra finch Feb. 2013 (WashU taeGut324/taeGut2) (taeGut2)
            </option><option value="taeGut1">
                Zebra finch Jul. 2008 (WUGSC 3.2.4/taeGut1) (taeGut1)
            </option><option value="danRer11_wo_alt">
                Zebrafish (GRCz11/danRer11) without alt contigs
            </option><option value="danRer6">
                Zebrafish Dec. 2008 (Zv8/danRer6) (danRer6)
            </option><option value="danRer7">
                Zebrafish Jul. 2010 (Zv9/danRer7) (danRer7)
            </option><option value="danRer5">
                Zebrafish July 2007 (Zv7/danRer5) (danRer5)
            </option><option value="danRer4">
                Zebrafish Mar. 2006 (Zv6/danRer4) (danRer4)
            </option><option value="danRer3">
                Zebrafish May 2005 (Zv5/danRer3) (danRer3)
            </option><option value="danRer11">
                Zebrafish May 2017 (GRCz11/danRer11) (danRer11)
            </option><option value="danRer10">
                Zebrafish Sep. 2014 (GRCz10/danRer10) (danRer10)
            </option><option value="Zostera_marina_JGI_v2.2">
                Zostera marina JGI v2.2
            </option><option value="Zoysia_japonica_ssp._nagirizaki_Zoysia_Genome_Database_v1.1">
                Zoysia japonica ssp. nagirizaki Zoysia Genome Database v1.1
            </option><option value="zymoMobi_ZM4">
                Zymomonas mobilis subsp. mobilis ZM4 (zymoMobi_ZM4)
            </option><option value="canFam3">
                canFam3
            </option><option value="homPan20">
                chimp/human Jun. 2008 (UCSC Recon/homPan20) (homPan20)
            </option><option value="hg19Haps">
                hg19Haplotypes Feb. 2009 (GRCh37/hg19Haps) (hg19Haps)
            </option><option value="mm9_ERCC">
                mm9 with ERCC spike-in
            </option><option value="homIni20">
                orang/human Jun. 2008 (UCSC Recon/homIni20) (homIni20)
            </option><option value="pUC18">
                pUC18 Plasmid (pUC18)
            </option><option value="phiX">
                phiX174 (phiX)
            </option><option value="tarIhg38">
                tarInv Dec. 2013 (GRCh38Tar/tarIhg38) (tarIhg38)
            </option><option value="uncuMeth_RCI">
                uncultured methanogenic archaeon RC-I (uncuMeth_RCI)
            </option></select></div> <div class="upload-buttons"><button id="btn-close" title="Close" type="button" class="btn ui-button-default upload-close btn-secondary">
            Close
        </button> <button id="btn-reset" title="Reset" type="button" disabled="disabled" class="btn ui-button-default btn-secondary disabled">
            Reset
        </button> <button id="btn-stop" title="Pause" type="button" disabled="disabled" class="btn ui-button-default btn-secondary disabled">
            Pause
        </button> <button id="btn-start" title="Start" type="button" disabled="disabled" class="btn ui-button-default upload-start btn-secondary disabled">
            Start
        </button> <button id="btn-new" title="Paste/Fetch data" type="button" class="btn ui-button-default upload-paste btn-secondary"><span class="fa fa-edit"></span>Paste/Fetch data
        </button> <!----> <button id="btn-local" title="Choose local files" type="button" class="btn ui-button-default btn-secondary"><span class="fa fa-laptop"></span>Choose local files
        </button></div></div></div> <div role="tabpanel" id="composite" aria-hidden="true" aria-labelledby="tab-title-link-composite" class="tab-pane" style="display: none;"><div class="upload-view-composite"><div class="upload-top"><div class="upload-top-info">&nbsp;</div></div> <div class="upload-box" style="height: 300px;"><div class="upload-helper">Select a composite type</div> <table class="upload-table ui-table-striped" style="display: none;"><thead><tr><th></th> <th></th> <th>Description</th> <th>Name</th> <th>Size</th> <th>Settings</th> <th>Status</th></tr></thead> <tbody></tbody></table></div> <div class="upload-footer"><span class="upload-footer-title">Composite Type:</span> <div class="select2-container upload-footer-extension" id="s2id_autogen21"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-22">Select</span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen22" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-22" id="s2id_autogen22"><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen22_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-22" id="s2id_autogen22_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-22">   </ul></div></div><select tabindex="-1" title="" class="select2-offscreen"><option value="_select_">Select</option><option value="affybatch">affybatch</option><option value="analyze75">analyze75</option><option value="eset">eset</option><option value="fped">fped</option><option value="fphe">fphe</option><option value="ideaspre">ideaspre</option><option value="imzml">imzml</option><option value="isa-json">isa-json</option><option value="isa-tab">isa-tab</option><option value="ldindep">ldindep</option><option value="lped">lped</option><option value="ludwig_model">ludwig_model</option><option value="malist">malist</option><option value="neostore.zip">neostore.zip</option><option value="obfs">obfs</option><option value="pbed">pbed</option><option value="pphe">pphe</option><option value="rdata.se">rdata.se</option><option value="shp">shp</option><option value="snpsiftdbnsfp">snpsiftdbnsfp</option><option value="spalndba">spalndba</option><option value="spalndbnp">spalndbnp</option><option value="splib">splib</option><option value="velvet">velvet</option><option value="wiff">wiff</option><option value="wiff2">wiff2</option></select> <span class="upload-footer-extension-info upload-icon-button fa fa-search"></span> <span class="upload-footer-title">Genome/Build:</span> <div class="select2-container upload-footer-genome" id="s2id_autogen23"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-24">
                unspecified (?)
            </span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen24" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-24" id="s2id_autogen24"><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen24_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-24" id="s2id_autogen24_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-24">   </ul></div></div><select tabindex="-1" title="" class="select2-offscreen"><option value="?">
                unspecified (?)
            </option><option value="anoGam1">
                A. gambiae Feb. 2003 (IAGEC MOZ2/anoGam1) (anoGam1)
            </option><option value="anoGam3">
                A. gambiae Oct. 2006 (AgamP3/anoGam3) (anoGam3)
            </option><option value="apiMel4">
                A. mellifera 04 Nov 2010 (Amel_4.5/apiMel4) (apiMel4)
            </option><option value="apiMel2">
                A. mellifera Jan. 2005 (Baylor 2.0/apiMel2) (apiMel2)
            </option><option value="apiMel1">
                A. mellifera July 2004 (Baylor 1.2/apiMel1) (apiMel1)
            </option><option value="GCF_001704415.2">
                ARS1 Aug. 2016 Capra hircus (goat) (San Clemente 2016 USDA) (GCF_001704415.2)
            </option><option value="oryAfe1">
                Aardvark May 2012 (OryAfe1.0/oryAfe1) (oryAfe1)
            </option><option value="12997">
                Acaryochloris marina MBIC11017 (12997)
            </option><option value="19259">
                Acholeplasma laidlawii PG-8A (19259)
            </option><option value="acidCryp_JF_5">
                Acidiphilium cryptum JF-5 (acidCryp_JF_5)
            </option><option value="acidBact_ELLIN345">
                Acidobacteria bacterium Ellin345 (acidBact_ELLIN345)
            </option><option value="acidCell_11B">
                Acidothermus cellulolyticus 11B (acidCell_11B)
            </option><option value="15708">
                Acidovorax avenae subsp. citrulli AAC00-1 (15708)
            </option><option value="acidJS42">
                Acidovorax sp. JS42 (acidJS42)
            </option><option value="13001">
                Acinetobacter baumannii (13001)
            </option><option value="28921">
                Acinetobacter baumannii (28921)
            </option><option value="17477">
                Acinetobacter baumannii ATCC 17978 (17477)
            </option><option value="aciBauTYTH_1">
                Acinetobacter baumannii Oct. 2012 (aciBauTYTH_1/aciBauTYTH_1) (aciBauTYTH_1)
            </option><option value="acinSp_ADP1">
                Acinetobacter sp. ADP1 (acinSp_ADP1)
            </option><option value="Actinidia_chinensis_FEI_Lab_v1.0">
                Actinidia chinensis FEI Lab v1.0
            </option><option value="actiPleu_L20">
                Actinobacillus pleuropneumoniae L20 (actiPleu_L20)
            </option><option value="19135">
                Actinobacillus pleuropneumoniae serovar 3 str. JL03 (19135)
            </option><option value="13370">
                Actinobacillus succinogenes 130Z (13370)
            </option><option value="pygAde1">
                Adelie penguin Jun 2014 (ASM69910v1/pygAde1) (pygAde1)
            </option><option value="aeroHydr_ATCC7966">
                Aeromonas hydrophila subsp. hydrophila ATCC 7966 (aeroHydr_ATCC7966)
            </option><option value="16723">
                Aeromonas salmonicida subsp. salmonicida A449 (16723)
            </option><option value="aerPer1">
                Aeropyrum pernix K1 (aerPer1)
            </option><option value="xenLae9.2">
                African Clawed Frog (xenLae9.2)
            </option><option value="xenLae2">
                African clawed frog Aug. 2016 (Xenopus_laevis_v2/xenLae2) (xenLae2)
            </option><option value="afrOth13">
                Afrotheria Apr. 24. 2006 (UCSC Recon/afrOth13) (afrOth13)
            </option><option value="283">
                Agrobacterium tumefaciens str. C58 (283)
            </option><option value="alcaBork_SK2">
                Alcanivorax borkumensis SK2 (alcaBork_SK2)
            </option><option value="alkaEhrl_MLHE_1">
                Alkalilimnicola ehrlichei MLHE-1 (alkaEhrl_MLHE_1)
            </option><option value="13006">
                Alkaliphilus metalliredigens QYMF (13006)
            </option><option value="16083">
                Alkaliphilus oremlandii OhILAs (16083)
            </option><option value="vicPac1">
                Alpaca Jul. 2008 (Broad/vicPac1) (vicPac1)
            </option><option value="vicPac2">
                Alpaca Mar. 2013 (Vicugna_pacos-2.0.1/vicPac2) (vicPac2)
            </option><option value="poeFor1">
                Amazon molly 28 Oct-2013 (Poecilia_formosa-5.1.2/poeFor1) (poeFor1)
            </option><option value="Amborella_trichopoda_Amborella_V1.0">
                Amborella trichopoda Amborella V1.0
            </option><option value="Amborella_trichopoda_JGI_v1.0">
                Amborella trichopoda JGI v1.0
            </option><option value="allMis1">
                American alligator Aug. 2012 (allMis0.2/allMis1) (allMis1)
            </option><option value="allMis0">
                American alligator November 2011 (UCSC aMiss_v0.1c27/allMis0) (allMis0)
            </option><option value="corBra1">
                American crow Jun 2014 (ASM69197v1/corBra1) (corBra1)
            </option><option value="phoRub1">
                American flamingo May 2014 (ASM68726v1/phoRub1) (phoRub1)
            </option><option value="panTig1">
                Amur tiger 06 Sep 2013 (PanTig1.0/panTig1) (panTig1)
            </option><option value="anabVari_ATCC29413">
                Anabaena variabilis ATCC 29413 (anabVari_ATCC29413)
            </option><option value="anaeDeha_2CP_C">
                Anaeromyxobacter dehalogenans 2CP-C (anaeDeha_2CP_C)
            </option><option value="17729">
                Anaeromyxobacter sp. Fw109-5 (17729)
            </option><option value="Ananas_comosus_JGI_v3.0">
                Ananas comosus JGI v3.0
            </option><option value="anapMarg_ST_MARIES">
                Anaplasma marginale str. St. Maries (anapMarg_ST_MARIES)
            </option><option value="336">
                Anaplasma phagocytophilum HZ (336)
            </option><option value="calAnn1">
                Anna's hummingbird Jun 2014 (ASM69908v1/calAnn1) (calAnn1)
            </option><option value="Apostasia_shenzhenica_v1.0">
                Apostasia shenzhenica v1.0
            </option><option value="aquiAeol">
                Aquifex aeolicus VF5 (aquiAeol)
            </option><option value="Araly1">
                Arabidopsis lyrata (Araly1)
            </option><option value="Arabidopsis_lyrata_JGI_v1.0">
                Arabidopsis lyrata JGI v1.0
            </option><option value="Arabidopsis_lyrata_JGI_v2.1">
                Arabidopsis lyrata JGI v2.1
            </option><option value="TAIR10">
                Arabidopsis thaliana
            </option><option value="Arabidopsis_thaliana_COL-0_Araport11">
                Arabidopsis thaliana COL-0 Araport11
            </option><option value="Arabidopsis_thaliana_COL-0_TAIR10">
                Arabidopsis thaliana COL-0 TAIR10
            </option><option value="Arabidopsis_thaliana_TAIR10">
                Arabidopsis thaliana TAIR10 (Arabidopsis_thaliana_TAIR10) (Arabidopsis_thaliana_TAIR10)
            </option><option value="araTha1">
                Arabidopsis thaliana TAIR7 (araTha1)
            </option><option value="arabidopsis_tair8">
                Arabidopsis thaliana TAIR8 (arabidopsis_tair8)
            </option><option value="arabidopsis">
                Arabidopsis thaliana TAIR9 (arabidopsis)
            </option><option value="Arachis_ipaensis_Peanut_Base_v1.0">
                Arachis ipaensis Peanut Base v1.0
            </option><option value="archFulg1">
                Archaeoglobus fulgidus DSM 4304 (archFulg1)
            </option><option value="16319">
                Arcobacter butzleri RM4018 (16319)
            </option><option value="letCam1">
                Arctic lamprey Sep 2013 (LetJap1.0/letCam1) (letCam1)
            </option><option value="dasNov3">
                Armadillo Dec. 2011 (Baylor/dasNov3) (dasNov3)
            </option><option value="dasNov2">
                Armadillo Jul. 2008 (Broad/dasNov2) (dasNov2)
            </option><option value="dasNov1">
                Armadillo May 2005 (Broad/dasNov1) (dasNov1)
            </option><option value="12512">
                Arthrobacter aurescens TC1 (12512)
            </option><option value="arthFB24">
                Arthrobacter sp. FB24 (arthFB24)
            </option><option value="Asparagus_officinalis_v1.1">
                Asparagus officinalis v1.1
            </option><option value="13478">
                Aster yellows witches'-broom phytoplasma AYWB (13478)
            </option><option value="gadMor1">
                Atlantic cod May 2010 (Genofisk GadMor_May2010/gadMor1) (gadMor1)
            </option><option value="13217">
                Azoarcus sp. BH72 (13217)
            </option><option value="azoaSp_EBN1">
                Azoarcus sp. EbN1 (azoaSp_EBN1)
            </option><option value="azorCaul2">
                Azorhizobium caulinodans ORS 571 (azorCaul2)
            </option><option value="bruMal1">
                B. malayi Sep. 2007 (TIGR 1.0/bruMal1) (bruMal1)
            </option><option value="papAnu4">
                Baboon Apr. 2017 (Panu_3.0/papAnu4) (papAnu4)
            </option><option value="papAnu2">
                Baboon Mar. 2012 (Baylor Panu_2.0/papAnu2) (papAnu2)
            </option><option value="papHam1">
                Baboon Nov. 2008 (Baylor Pham_1.0/papHam1) (papHam1)
            </option><option value="13403">
                Bacillus amyloliquefaciens FZB42 (13403)
            </option><option value="10784">
                Bacillus anthracis str. 'Ames Ancestor' (10784)
            </option><option value="baciAnth_AMES">
                Bacillus anthracis str. Ames (baciAnth_AMES)
            </option><option value="10878">
                Bacillus anthracis str. Sterne (10878)
            </option><option value="74">
                Bacillus cereus ATCC 10987 (74)
            </option><option value="384">
                Bacillus cereus ATCC 14579 (384)
            </option><option value="12468">
                Bacillus cereus E33L (12468)
            </option><option value="13624">
                Bacillus cereus subsp. cytotoxis NVH 391-98 (13624)
            </option><option value="13291">
                Bacillus clausii KSM-K16 (13291)
            </option><option value="baciHalo">
                Bacillus halodurans C-125 (baciHalo)
            </option><option value="12388">
                Bacillus licheniformis ATCC 14580 (12388)
            </option><option value="13082">
                Bacillus licheniformis ATCC 14580 (13082)
            </option><option value="20391">
                Bacillus pumilus SAFR-032 (20391)
            </option><option value="baciSubt">
                Bacillus subtilis subsp. subtilis str. 168 (baciSubt)
            </option><option value="10877">
                Bacillus thuringiensis serovar konkukian str. 97-27 (10877)
            </option><option value="18255">
                Bacillus thuringiensis str. Al Hakam (18255)
            </option><option value="13623">
                Bacillus weihenstephanensis KBAB4 (13623)
            </option><option value="46">
                Bacteroides fragilis NCTC 9343 (46)
            </option><option value="13067">
                Bacteroides fragilis YCH46 (13067)
            </option><option value="bactThet_VPI_5482">
                Bacteroides thetaiotaomicron VPI-5482 (bactThet_VPI_5482)
            </option><option value="13378">
                Bacteroides vulgatus ATCC 8482 (13378)
            </option><option value="camFer1">
                Bactrian camel Dec 2011 (CB1/camFer1) (camFer1)
            </option><option value="halLeu1">
                Bald eagle Aug 2014 (Haliaeetus_leucocephalus-4.0/halLeu1) (halLeu1)
            </option><option value="apaVit1">
                Bar tailed trogon 16 Jun 2014 (ASM70340v1/apaVit1) (apaVit1)
            </option><option value="tytAlb1">
                Barn owl May 2014 (ASM68720v1/tytAlb1) (tytAlb1)
            </option><option value="16249">
                Bartonella bacilliformis KC583 (16249)
            </option><option value="bartHens_HOUSTON_1">
                Bartonella henselae str. Houston-1 (bartHens_HOUSTON_1)
            </option><option value="44">
                Bartonella quintana str. Toulouse (44)
            </option><option value="28109">
                Bartonella tribocorum CIP 105476 (28109)
            </option><option value="baumCica_HOMALODISCA">
                Baumannia cicadellinicola str. Hc (Homalodisca coagulata) (baumCica_HOMALODISCA)
            </option><option value="bdelBact">
                Bdellovibrio bacteriovorus HD100 (bdelBact)
            </option><option value="Beta_vulgaris_RefBeet_1.1">
                Beta vulgaris RefBeet 1.1
            </option><option value="Beta_vulgaris_RefBeet_v1.2">
                Beta vulgaris RefBeet v1.2
            </option><option value="stePar1">
                Bicolor damselfish May 2014 (Stegastes_partitus-1.0.2/stePar1) (stePar1)
            </option><option value="16321">
                Bifidobacterium adolescentis ATCC 15703 (16321)
            </option><option value="bifiLong">
                Bifidobacterium longum NCC2705 (bifiLong)
            </option><option value="eptFus1">
                Big brown bat Jul 2012 (EptFus1.0/eptFus1) (eptFus1)
            </option><option value="bisBis1">
                Bison Oct. 2014 (Bison_UMD1.0/bisBis1) (bisBis1)
            </option><option value="pteAle1">
                Black flying-fox Aug 2012 (ASM32557v1/pteAle1) (pteAle1)
            </option><option value="notCor1">
                Black rockcod Jul 2014 (NC01/notCor1) (notCor1)
            </option><option value="panPan2">
                Bonobo Aug. 2015 (MPI-EVA panpan1.1/panPan2) (panPan2)
            </option><option value="panPan1">
                Bonobo May 2012 (Max-Planck/panPan1) (panPan1)
            </option><option value="panPan3">
                Bonobo May 2020 (Mhudiblu_PPA_v0/panPan3) (panPan3)
            </option><option value="bordBron">
                Bordetella bronchiseptica RB50 (bordBron)
            </option><option value="25">
                Bordetella parapertussis 12822 (25)
            </option><option value="26">
                Bordetella pertussis Tohama I (26)
            </option><option value="28135">
                Bordetella petrii DSM 12804 (28135)
            </option><option value="borEut13">
                Boreoeutherian Apr. 24. 2006 (UCSC Recon/borEut13) (borEut13)
            </option><option value="canHg12">
                Boreoeutherian Nov. 19. 2005 (UCSC Recon/canHg12) (canHg12)
            </option><option value="17057">
                Borrelia afzelii PKo (17057)
            </option><option value="borrBurg">
                Borrelia burgdorferi B31 (borrBurg)
            </option><option value="12554">
                Borrelia garinii PBi (12554)
            </option><option value="Brachypodium_distachyon_JGI_v3.1">
                Brachypodium distachyon JGI v3.1
            </option><option value="Brachypodium_distachyon_MIPS_1.2">
                Brachypodium distachyon MIPS 1.2
            </option><option value="bradJapo">
                Bradyrhizobium japonicum USDA 110 (bradJapo)
            </option><option value="16137">
                Bradyrhizobium sp. BTAi1 (16137)
            </option><option value="19575">
                Bradyrhizobium sp. ORS278 (19575)
            </option><option value="myoBra1">
                Brandt's myotis (bat) 28 Jun 2013 (ASM41265v1/myoBra1) (myoBra1)
            </option><option value="braNey1">
                Braney test Jul. 2013 (braNey1/braNey1) (braNey1)
            </option><option value="Brassica_oleracea_Ensembl_Genomes_v2.1">
                Brassica oleracea Ensembl Genomes v2.1
            </option><option value="Brassica_rapa_Annotation_v1.2_on_assembly_v1.1_Phytozome_9.1">
                Brassica rapa Annotation v1.2 on assembly v1.1 (Phytozome 9.1)
            </option><option value="Brassica_rapa_JGI_FPsc1.3">
                Brassica rapa JGI FPsc1.3
            </option><option value="aptMan1">
                Brown kiwi Jun. 2015 (MPI-EVA AptMant0/aptMan1) (aptMan1)
            </option><option value="mesUni1">
                Brown roatelo Jun 2014 (ASM69576v1/mesUni1) (mesUni1)
            </option><option value="9619">
                Brucella abortus biovar 1 str. 9-941 (9619)
            </option><option value="20243">
                Brucella canis ATCC 23365 (20243)
            </option><option value="brucMeli">
                Brucella melitensis 16M (brucMeli)
            </option><option value="16203">
                Brucella melitensis biovar Abortus 2308 (16203)
            </option><option value="12514">
                Brucella ovis ATCC 25840 (12514)
            </option><option value="320">
                Brucella suis 1330 (320)
            </option><option value="20371">
                Brucella suis ATCC 23445 (20371)
            </option><option value="octDeg1">
                Brush-tailed rat Apr 2012 (OctDeg1.0/octDeg1) (octDeg1)
            </option><option value="buchSp">
                Buchnera aphidicola str. APS (Acyrthosiphon pisum) (buchSp)
            </option><option value="256">
                Buchnera aphidicola str. Bp (Baizongia pistaciae) (256)
            </option><option value="16372">
                Buchnera aphidicola str. Cc (Cinara cedri) (16372)
            </option><option value="312">
                Buchnera aphidicola str. Sg (Schizaphis graminum) (312)
            </option><option value="melUnd1">
                Budgerigar Sep. 2011 (WUSTL v6.3/melUnd1) (melUnd1)
            </option><option value="bunEbo1">
                Bundibugyo ebolavirus Aug. 2014 (NC_014373.1/bunEbo1) (bunEbo1)
            </option><option value="burkCepa_AMMD">
                Burkholderia ambifaria AMMD (burkCepa_AMMD)
            </option><option value="burkCeno_AU_1054">
                Burkholderia cenocepacia AU 1054 (burkCeno_AU_1054)
            </option><option value="burkCeno_HI2424">
                Burkholderia cenocepacia HI2424 (burkCeno_HI2424)
            </option><option value="17929">
                Burkholderia cenocepacia MC0-3 (17929)
            </option><option value="burkMall_ATCC23344">
                Burkholderia mallei ATCC 23344 (burkMall_ATCC23344)
            </option><option value="13943">
                Burkholderia mallei NCTC 10229 (13943)
            </option><option value="13946">
                Burkholderia mallei NCTC 10247 (13946)
            </option><option value="13947">
                Burkholderia mallei SAVP1 (13947)
            </option><option value="17407">
                Burkholderia multivorans ATCC 17616 (17407)
            </option><option value="burkPseu_1106A">
                Burkholderia pseudomallei 1106a (burkPseu_1106A)
            </option><option value="13954">
                Burkholderia pseudomallei 1710b (13954)
            </option><option value="13953">
                Burkholderia pseudomallei 668 (13953)
            </option><option value="178">
                Burkholderia pseudomallei K96243 (178)
            </option><option value="burk383">
                Burkholderia sp. 383 (burk383)
            </option><option value="burkThai_E264">
                Burkholderia thailandensis E264 (burkThai_E264)
            </option><option value="burkViet_G4">
                Burkholderia vietnamiensis G4 (burkViet_G4)
            </option><option value="burkXeno_LB400">
                Burkholderia xenovorans LB400 (burkXeno_LB400)
            </option><option value="pytBiv1">
                Burmese python Sep. 2013 (Python_molurus_bivittatus-5.0.2/pytBiv1) (pytBiv1)
            </option><option value="hapBur1">
                Burton's mouthbreeder Oct 2011 (AstBur1.0/hapBur1) (hapBur1)
            </option><option value="otoGar1">
                Bushbaby Dec. 2006 (Broad/otoGar1) (otoGar1)
            </option><option value="otoGar3">
                Bushbaby Mar. 2011 (Broad/otoGar3) (otoGar3)
            </option><option value="caeAng1">
                C. angaria Oct. 2010 (WS225/caeAng1) (caeAng1)
            </option><option value="caePb2">
                C. brenneri Feb. 2008 (WUGSC 6.0.1/caePb2) (caePb2)
            </option><option value="caePb1">
                C. brenneri Jan. 2007 (WUGSC 4.0/caePb1) (caePb1)
            </option><option value="caePb3">
                C. brenneri Nov. 2010 (C. brenneri 6.0.1b/caePb3) (caePb3)
            </option><option value="cb4">
                C. briggsae Apr. 2011 (WS225/cb4) (cb4)
            </option><option value="cb2">
                C. briggsae Aug 2005 (WUGSC prelim/cb2) (cb2)
            </option><option value="cb3">
                C. briggsae Jan. 2007 (WUGSC 1.0/cb3) (cb3)
            </option><option value="cb1">
                C. briggsae July 2002 (WormBase cb25.agp8/cb1) (cb1)
            </option><option value="ce5">
                C. elegans Aug. 2007 (WS180/ce5) (ce5)
            </option><option value="ce7">
                C. elegans Feb 2009 (WS200/ce7) (ce7)
            </option><option value="ce11">
                C. elegans Feb. 2013 (WBcel235/ce11) (ce11)
            </option><option value="ce9">
                C. elegans Jan 2010 (WS210/ce9) (ce9)
            </option><option value="ce4">
                C. elegans Jan. 2007 (WS170/ce4) (ce4)
            </option><option value="ce8">
                C. elegans Jun 2009 (WS204/ce8) (ce8)
            </option><option value="ce2">
                C. elegans Mar. 2004 (WS120/ce2) (ce2)
            </option><option value="ce3">
                C. elegans March 2005 (WS140/ce3) (ce3)
            </option><option value="ce6">
                C. elegans May 2008 (WS190/ce6) (ce6)
            </option><option value="ce10">
                C. elegans Oct. 2010 (WS220/ce10) (ce10)
            </option><option value="ci3">
                C. intestinalis Apr. 2011 (Kyoto KH/ci3) (ci3)
            </option><option value="ci1">
                C. intestinalis Dec. 2002 (JGI 1.0/ci1) (ci1)
            </option><option value="ci2">
                C. intestinalis Mar. 2005 (JGI 2.1/ci2) (ci2)
            </option><option value="caeJap3">
                C. japonica  Sep. 2010 (WUGSC 7.0.1/caeJap3) (caeJap3)
            </option><option value="caeJap4">
                C. japonica Aug. 2010 (WUSTL 7.0.1/caeJap4) (caeJap4)
            </option><option value="caeJap2">
                C. japonica Jan. 2009 (WUGSC 4.0.1/caeJap2) (caeJap2)
            </option><option value="caeJap1">
                C. japonica Mar. 2008 (WUGSC 3.0.2/caeJap1) (caeJap1)
            </option><option value="caeRem4">
                C. remanei Jul. 2007 (WS220/caeRem4) (caeRem4)
            </option><option value="caeRem2">
                C. remanei Mar. 2006 (WUGSC 1.0/caeRem2) (caeRem2)
            </option><option value="caeRem1">
                C. remanei March 2005 (WUGSC prelim/caeRem1) (caeRem1)
            </option><option value="caeRem3">
                C. remanei May 2007 (WUGSC 15.0.1/caeRem3) (caeRem3)
            </option><option value="cioSav1">
                C. savignyi Apr. 2003 (Broad/cioSav1) (cioSav1)
            </option><option value="cioSav2">
                C. savignyi July 2005 (Sidow Lab 2.0/cioSav2) (cioSav2)
            </option><option value="caeSp111">
                C. sp. 11 ju1373 Nov. 2010 (WUSTL 3.0.1/caeSp111) (caeSp111)
            </option><option value="caeSp71">
                C. sp. 7 ju1286 Nov. 2010 (WUSTL 4.0.1/caeSp71) (caeSp71)
            </option><option value="caeSp91">
                C. sp. 9 ju1422 Nov. 2010 (WUSTL 3.0.1/caeSp91) (caeSp91)
            </option><option value="cacTus3">
                Cactus Nov. 2010 (CactusV3/cacTus3) (cacTus3)
            </option><option value="cacTus2">
                Cactus Oct. 2010 (Ngan10.1/cacTus2) (cacTus2)
            </option><option value="cacTus1">
                Cactus Sep. 2010 (CactusV1/cacTus1) (cacTus1)
            </option><option value="Cajanus_cajan_Penguin_Genomics_v5.0">
                Cajanus cajan Penguin Genomics v5.0
            </option><option value="caldSacc_DSM8903">
                Caldicellulosiruptor saccharolyticus DSM 8903 (caldSacc_DSM8903)
            </option><option value="caldMaqu1">
                Caldivirga maquilingensis IC-167 (caldMaqu1)
            </option><option value="17159">
                Campylobacter concisus 13826 (17159)
            </option><option value="17161">
                Campylobacter curvus 525.92 (17161)
            </option><option value="campFetu_82_40">
                Campylobacter fetus subsp. fetus 82-40 (campFetu_82_40)
            </option><option value="20083">
                Campylobacter hominis ATCC BAA-381 (20083)
            </option><option value="campJeju_RM1221">
                Campylobacter jejuni RM1221 (campJeju_RM1221)
            </option><option value="17163">
                Campylobacter jejuni subsp. doylei 269.97 (17163)
            </option><option value="campJeju_81_176">
                Campylobacter jejuni subsp. jejuni 81-176 (campJeju_81_176)
            </option><option value="17953">
                Campylobacter jejuni subsp. jejuni 81116 (17953)
            </option><option value="campJeju">
                Campylobacter jejuni subsp. jejuni NCTC 11168 (campJeju)
            </option><option value="serCan1">
                Canary 15 Jan-2014 (SCA1/serCan1) (serCan1)
            </option><option value="blocFlor">
                Candidatus Blochmannia floridanus (blocFlor)
            </option><option value="13875">
                Candidatus Blochmannia pennsylvanicus str. BPEN (13875)
            </option><option value="candCars_RUDDII">
                Candidatus Carsonella ruddii PV (candCars_RUDDII)
            </option><option value="21047">
                Candidatus Desulforudis audaxviator MP104C (21047)
            </option><option value="16525">
                Candidatus Korarchaeum cryptofilum OPF8 (16525)
            </option><option value="methBoon1">
                Candidatus Methanoregula boonei 6A8 (methBoon1)
            </option><option value="candPela_UBIQUE_HTCC1">
                Candidatus Pelagibacter ubique HTCC1062 (candPela_UBIQUE_HTCC1)
            </option><option value="paraSp_UWE25">
                Candidatus Protochlamydia amoebophila UWE25 (paraSp_UWE25)
            </option><option value="16841">
                Candidatus Ruthia magnifica str. Cm (Calyptogena magnifica) (16841)
            </option><option value="19617">
                Candidatus Sulcia muelleri GWSS (19617)
            </option><option value="18267">
                Candidatus Vesicomyosocius okutanii HA (18267)
            </option><option value="eleEdw1">
                Cape elephant shrew Aug 2012 (EleEdw1.0/eleEdw1) (eleEdw1)
            </option><option value="chrAsi1">
                Cape golden mole Aug 2012 (ChrAsi1.0/chrAsi1) (chrAsi1)
            </option><option value="Capsella_rubella_JGI_annotation_v1.0_on_assembly_v1">
                Capsella rubella JGI annotation v1.0 on assembly v1
            </option><option value="Capsella_rubella_JGI_v1.0">
                Capsella rubella JGI v1.0
            </option><option value="Capsicum_annuum_Pepper_Genome_v2.0">
                Capsicum annuum Pepper Genome v2.0
            </option><option value="carbHydr_Z_2901">
                Carboxydothermus hydrogenoformans Z-2901 (carbHydr_Z_2901)
            </option><option value="Carica_papaya_Hawaii_Agriculture_Research_Center">
                Carica papaya Hawaii Agriculture Research Center
            </option><option value="Carica_papaya_JGI_ASGPB0.4">
                Carica papaya JGI ASGPB0.4
            </option><option value="felCat4">
                Cat Dec. 2008 (NHGRI/GTB V17e/felCat4) (felCat4)
            </option><option value="felCat3">
                Cat Mar. 2006 (Broad/felCat3) (felCat3)
            </option><option value="felCat8">
                Cat Nov. 2014 (ICGSC Felis_catus_8.0/felCat8) (felCat8)
            </option><option value="felCat9">
                Cat Nov. 2017 (Felis_catus_9.0/felCat9) (felCat9)
            </option><option value="felCat5">
                Cat Sep. 2011 (ICGSC Felis_catus 6.2/felCat5) (felCat5)
            </option><option value="catArr1">
                Catarrhini June 13. 2006 (UCSC Recon/catArr1) (catArr1)
            </option><option value="caulCres">
                Caulobacter crescentus CB15 (caulCres)
            </option><option value="16306">
                Caulobacter sp. K31 (16306)
            </option><option value="Cenchrus_americanus_v1.0">
                Cenchrus americanus v1.0
            </option><option value="centromeres1">
                CentromereY 2013 GrCh38 (centromeres1)
            </option><option value="Chenopodium_quinoa_JGI_v1.0">
                Chenopodium quinoa JGI v1.0
            </option><option value="galGal5">
                Chicken Dec 2015 (Gallus_gallus-5.0/galGal5) (galGal5)
            </option><option value="galGal2">
                Chicken Feb. 2004 (WUGSC 1.0/galGal2) (galGal2)
            </option><option value="galGal6">
                Chicken Mar. 2018 (GRCg6a/galGal6) (galGal6)
            </option><option value="galGal3">
                Chicken May 2006 (WUGSC 2.1/galGal3) (galGal3)
            </option><option value="galGal4">
                Chicken Nov. 2011 (ICGSC Gallus_gallus-4.0/galGal4) (galGal4)
            </option><option value="GCF_016700215.2">
                Chicken Oct. 2021 (bGalGal1.pat.whiteleghornlayer.GRCg7w_WZ) (GCF_016700215.2)
            </option><option value="panTro4">
                Chimp Feb. 2011 (CSAC 2.1.4/panTro4) (panTro4)
            </option><option value="panTro6">
                Chimp Jan. 2018 (Clint_PTRv2/panTro6) (panTro6)
            </option><option value="panTro2">
                Chimp Mar. 2006 (CGSC 2.1/panTro2) (panTro2)
            </option><option value="panTro5">
                Chimp May 2016 (Pan_tro 3.0/panTro5) (panTro5)
            </option><option value="panTro1">
                Chimp Nov. 2003 (CGSC 1.1/panTro1) (panTro1)
            </option><option value="panTro3">
                Chimp Oct. 2010 (CGSC 2.1.3/panTro3) (panTro3)
            </option><option value="chiLan1">
                Chinchilla May 2012  (ChiLan1.0/chiLan1) (chiLan1)
            </option><option value="allSin1">
                Chinese alligator 29 Aug 2013 (ASM45574v1/allSin1) (allSin1)
            </option><option value="criGriChoV1">
                Chinese hamster Aug. 2011 (CHO K1 cell line/criGriChoV1) (criGriChoV1)
            </option><option value="criGri1">
                Chinese hamster Jul. 2013 (C_griseus_v1.0/criGri1) (criGri1)
            </option><option value="criGriChoV2">
                Chinese hamster Jun. 2017 (CHOK1S_HZDv1/criGriChoV2) (criGriChoV2)
            </option><option value="manPen1">
                Chinese pangolin Aug 2014 (M_pentadactyla-1.1.1/manPen1) (manPen1)
            </option><option value="pelSin1">
                Chinese softshell turtle Oct 2011 (PelSin_1.0/pelSin1) (pelSin1)
            </option><option value="tupChi1">
                Chinese tree shrew Jan 2013 (TupChi_1.0/tupChi1) (tupChi1)
            </option><option value="229">
                Chlamydia muridarum Nigg (229)
            </option><option value="28583">
                Chlamydia trachomatis 434/Bu (28583)
            </option><option value="13885">
                Chlamydia trachomatis A/HAR-13 (13885)
            </option><option value="chlaTrac">
                Chlamydia trachomatis D/UW-3/CX (chlaTrac)
            </option><option value="28585">
                Chlamydia trachomatis L2b/UCH-1/proctitis (28585)
            </option><option value="Chlamydomonas_reinhardtii_JGI_v5.0_assembly_annot_v5.3.1_based_on_Augustus_u11.6">
                Chlamydomonas reinhardtii JGI v5.0 assembly, annot v5.3.1 based on Augustus u11.6
            </option><option value="Chlamydomonas_reinhardtii_JGI_v5.5">
                Chlamydomonas reinhardtii JGI v5.5
            </option><option value="355">
                Chlamydophila abortus S26/3 (355)
            </option><option value="228">
                Chlamydophila caviae GPIC (228)
            </option><option value="370">
                Chlamydophila felis Fe/C-56 (370)
            </option><option value="247">
                Chlamydophila pneumoniae AR39 (247)
            </option><option value="chlaPneu_CWL029">
                Chlamydophila pneumoniae CWL029 (chlaPneu_CWL029)
            </option><option value="257">
                Chlamydophila pneumoniae J138 (257)
            </option><option value="420">
                Chlamydophila pneumoniae TW-183 (420)
            </option><option value="chloChlo_CAD3">
                Chlorobium chlorochromatii CaD3 (chloChlo_CAD3)
            </option><option value="12609">
                Chlorobium phaeobacteroides DSM 266 (12609)
            </option><option value="chloTepi_TLS">
                Chlorobium tepidum TLS (chloTepi_TLS)
            </option><option value="59">
                Chloroflexus aurantiacus J-10-fl (59)
            </option><option value="chroViol">
                Chromobacterium violaceum ATCC 12472 (chroViol)
            </option><option value="chroSale_DSM3043">
                Chromohalobacter salexigens DSM 3043 (chroSale_DSM3043)
            </option><option value="capCar1">
                Chuck-will's-widow Jun 2014 (ASM70074v1/capCar1) (capCar1)
            </option><option value="Cicer_arietinum_LIS_v1.0">
                Cicer arietinum LIS v1.0
            </option><option value="12716">
                Citrobacter koseri ATCC BAA-895 (12716)
            </option><option value="Citrullus_lanatus_Cucurbit_Genomics_Database">
                Citrullus lanatus Cucurbit Genomics Database
            </option><option value="Citrullus_lanatus_Cucurbit_Genomics_Database_v1.0">
                Citrullus lanatus Cucurbit Genomics Database v1.0
            </option><option value="Citrus_clementina_JGI_v1.0">
                Citrus clementina JGI v1.0
            </option><option value="Citrus_sinensis_JGI_v1_assembly_and_v1.0_annotation">
                Citrus sinensis JGI v1 assembly and v1.0 annotation
            </option><option value="clavMich_NCPPB_382">
                Clavibacter michiganensis subsp. michiganensis NCPPB 382 (clavMich_NCPPB_382)
            </option><option value="184">
                Clavibacter michiganensis subsp. sepedonicus (184)
            </option><option value="77">
                Clostridium acetobutylicum ATCC 824 (77)
            </option><option value="12637">
                Clostridium beijerinckii NCIMB 8052 (12637)
            </option><option value="19517">
                Clostridium botulinum A str. ATCC 19397 (19517)
            </option><option value="193">
                Clostridium botulinum A str. ATCC 3502 (193)
            </option><option value="19521">
                Clostridium botulinum A str. Hall (19521)
            </option><option value="28507">
                Clostridium botulinum A3 str. Loch Maree (28507)
            </option><option value="28505">
                Clostridium botulinum B1 str. Okra (28505)
            </option><option value="19519">
                Clostridium botulinum F str. Langeland (19519)
            </option><option value="78">
                Clostridium difficile 630 (78)
            </option><option value="19065">
                Clostridium kluyveri DSM 555 (19065)
            </option><option value="16820">
                Clostridium novyi NT (16820)
            </option><option value="304">
                Clostridium perfringens ATCC 13124 (304)
            </option><option value="12521">
                Clostridium perfringens SM101 (12521)
            </option><option value="79">
                Clostridium perfringens str. 13 (79)
            </option><option value="16184">
                Clostridium phytofermentans ISDg (16184)
            </option><option value="81">
                Clostridium tetani E88 (81)
            </option><option value="314">
                Clostridium thermocellum ATCC 27405 (314)
            </option><option value="latCha1">
                Coelacanth Aug. 2011 (Broad/latCha1) (latCha1)
            </option><option value="Coffea_canephora_Coffee_Genome_Hub_v1">
                Coffea canephora Coffee Genome Hub v1
            </option><option value="ficAlb2">
                Collared flycatcher Jun 2013 (FicAlb1.5/ficAlb2) (ficAlb2)
            </option><option value="colwPsyc_34H">
                Colwellia psychrerythraea 34H (colwPsyc_34H)
            </option><option value="cucCan1">
                Common cuckoo Jun 2014 (ASM70932v1/cucCan1) (cucCan1)
            </option><option value="Corchorus_olitorius_GenBank_v1.0">
                Corchorus olitorius GenBank v1.0
            </option><option value="87">
                Corynebacterium diphtheriae NCTC 13129 (87)
            </option><option value="coryEffi_YS_314">
                Corynebacterium efficiens YS-314 (coryEffi_YS_314)
            </option><option value="13760">
                Corynebacterium glutamicum ATCC 13032 (13760)
            </option><option value="307">
                Corynebacterium glutamicum ATCC 13032 (307)
            </option><option value="19193">
                Corynebacterium glutamicum R (19193)
            </option><option value="13967">
                Corynebacterium jeikeium K411 (13967)
            </option><option value="bosTau5">
                Cow Apr. 2009 (Baylor Btau_4.2/bosTau5) (bosTau5)
            </option><option value="bosTau9">
                Cow Apr. 2018 (ARS-UCD1.2/bosTau9) (bosTau9)
            </option><option value="bosTau3">
                Cow Aug. 2006 (Baylor 3.1/bosTau3) (bosTau3)
            </option><option value="bosTauMd3">
                Cow Aug. 2009 (UMD3.1) (bosTauMd3)
            </option><option value="bosTau8">
                Cow Jun. 2014 (Bos_taurus_UMD_3.1.1/bosTau8) (bosTau8)
            </option><option value="bosTau2">
                Cow Mar. 2005 (Baylor 2.0/bosTau2) (bosTau2)
            </option><option value="bosTau6">
                Cow Nov. 2009 (Bos_taurus_UMD_3.1/bosTau6) (bosTau6)
            </option><option value="bosTau4">
                Cow Oct. 2007 (Baylor 4.0/bosTau4) (bosTau4)
            </option><option value="bosTau7">
                Cow Oct. 2011 (Baylor Btau_4.6.1/bosTau7) (bosTau7)
            </option><option value="16721">
                Coxiella burnetii Dugway 5J108-111 (16721)
            </option><option value="16791">
                Coxiella burnetii RSA 331 (16791)
            </option><option value="coxiBurn">
                Coxiella burnetii RSA 493 (coxiBurn)
            </option><option value="macFas5">
                Crab-eating macaque Jun. 2013 (Macaca_fascicularis_5.0/macFas5) (macFas5)
            </option><option value="nipNip1">
                Crested ibis Jun 2014 (ASM70822v1/nipNip1) (nipNip1)
            </option><option value="balPav1">
                Crowned crain Jun 2014 (ASM70989v1/balPav1) (balPav1)
            </option><option value="lepDis1">
                Cuckoo roller May 2014 (ASM69178v1/lepDis1) (lepDis1)
            </option><option value="Cucumis_melo_Melonomics_v3.5">
                Cucumis melo Melonomics v3.5
            </option><option value="Cucumis_sativus_L._JGI_v1.0">
                Cucumis sativus L. JGI v1.0
            </option><option value="cytoHutc_ATCC33406">
                Cytophaga hutchinsonii ATCC 33406 (cytoHutc_ATCC33406)
            </option><option value="droAlb1">
                D. albomicans 21 May 2012 (DroAlb_1.0/droAlb1) (droAlb1)
            </option><option value="droAna2">
                D. ananassae Aug. 2005 (Agencourt prelim/droAna2) (droAna2)
            </option><option value="droAna3">
                D. ananassae Feb. 2006 (droAna3)
            </option><option value="droAna1">
                D. ananassae July 2004 (TIGR/droAna1) (droAna1)
            </option><option value="droBia2">
                D. biarmipes 04 Mar 2013 (Dbia_2.0/droBia2) (droBia2)
            </option><option value="droBip2">
                D. bipectinata 04 Mar 2013 (Dbip_2.0/droBip2) (droBip2)
            </option><option value="droEle2">
                D. elegans 04 Mar 2013 (Dele_2.0/droEle2) (droEle2)
            </option><option value="droEre1">
                D. erecta Aug. 2005 (Agencourt prelim/droEre1) (droEre1)
            </option><option value="droEre2">
                D. erecta Feb. 2006 (droEre2)
            </option><option value="droEug2">
                D. eugracilis 04 Mar 2013 (Deug_2.0/droEug2) (droEug2)
            </option><option value="droFic2">
                D. ficusphila 04 Mar 2013 (Dfic_2.0/droFic2) (droFic2)
            </option><option value="droGri1">
                D. grimshawi Aug. 2005 (Agencourt prelim/droGri1) (droGri1)
            </option><option value="droGri2">
                D. grimshawi Feb. 2006 (droGri2)
            </option><option value="droKik2">
                D. kikkawai 04 Mar 2013 (Dkik_2.0/droKik2) (droKik2)
            </option><option value="dm2">
                D. melanogaster Apr. 2004 (BDGP R4/dm2) (dm2)
            </option><option value="dm3">
                D. melanogaster Apr. 2006 (BDGP R5/dm3) (dm3)
            </option><option value="dm6">
                D. melanogaster Aug. 2014 (BDGP Release 6 + ISO1 MT/dm6) (dm6)
            </option><option value="dm1">
                D. melanogaster Jan. 2003 (BDGP R3/dm1) (dm1)
            </option><option value="droMir2">
                D. miranda 19 Apr 2013 (DroMir_2.2/droMir2) (droMir2)
            </option><option value="droMoj1">
                D. mojavensis Aug. 2004 (Agencourt prelim/droMoj1) (droMoj1)
            </option><option value="droMoj2">
                D. mojavensis Aug. 2005 (Agencourt prelim/droMoj2) (droMoj2)
            </option><option value="droMoj3">
                D. mojavensis Feb. 2006 (droMoj3)
            </option><option value="droPer1">
                D. persimilis Oct. 2005 (Broad/droPer1) (droPer1)
            </option><option value="dp4">
                D. pseudoobscura (dp4) (dp4)
            </option><option value="droPse3">
                D. pseudoobscura 11 Apr 2013 (Dpse_3.0/droPse3) (droPse3)
            </option><option value="dp2">
                D. pseudoobscura Aug. 2003 (Baylor freeze1/dp2) (dp2)
            </option><option value="dp3">
                D. pseudoobscura Nov. 2004 (FlyBase 1.03/dp3) (dp3)
            </option><option value="droRho2">
                D. rhopaloa 22 Feb 2013 (Drho_2.0/droRho2) (droRho2)
            </option><option value="droSec1">
                D. sechellia Oct. 2005 (Broad/droSec1) (droSec1)
            </option><option value="droSim1">
                D. simulans Apr. 2005 (WUGSC mosaic 1.0/droSim1) (droSim1)
            </option><option value="droSuz1">
                D. suzukii 30 Sep 2013 (Dsuzukii.v01/droSuz1) (droSuz1)
            </option><option value="droTak2">
                D. takahashii 04 Mar 2013 (Dtak_2.0/droTak2) (droTak2)
            </option><option value="droVir2">
                D. virilis Aug. 2005 (Agencourt prelim/droVir2) (droVir2)
            </option><option value="droVir3">
                D. virilis Feb. 2006 (droVir3)
            </option><option value="droVir1">
                D. virilis July 2004 (Agencourt prelim/droVir1) (droVir1)
            </option><option value="droWil2">
                D. willistoni 03 Aug 2006 (dwil_caf1/droWil2) (droWil2)
            </option><option value="droWil1">
                D. willistoni Feb. 2006 (droWil1)
            </option><option value="droYak3">
                D. yakuba 27 Jun 2006 (dyak_caf1/droYak3) (droYak3)
            </option><option value="droYak1">
                D. yakuba Apr. 2004 (WUGSC 1.0/droYak1) (droYak1)
            </option><option value="droYak2">
                D. yakuba Nov. 2005 (WUGSC 7.1/droYak2) (droYak2)
            </option><option value="pelCri1">
                Dalmatian pelican May 2014 (ASM68737v1/pelCri1) (pelCri1)
            </option><option value="Daucus_carota_JGI_v2.0">
                Daucus carota JGI v2.0
            </option><option value="myoDav1">
                David's myotis (bat) Aug 2012 (ASM32734v1/myoDav1) (myoDav1)
            </option><option value="dechArom_RCB">
                Dechloromonas aromatica RCB (dechArom_RCB)
            </option><option value="IscaW1">
                Deer Tick (Ixodes scapularis) (IscaW1)
            </option><option value="dehaEthe_195">
                Dehalococcoides ethenogenes 195 (dehaEthe_195)
            </option><option value="15770">
                Dehalococcoides sp. BAV1 (15770)
            </option><option value="15604">
                Dehalococcoides sp. CBDB1 (15604)
            </option><option value="deinGeot_DSM11300">
                Deinococcus geothermalis DSM 11300 (deinGeot_DSM11300)
            </option><option value="deinRadi">
                Deinococcus radiodurans R1 (deinRadi)
            </option><option value="17413">
                Delftia acidovorans SPH-1 (17413)
            </option><option value="desuHafn_Y51">
                Desulfitobacterium hafniense Y51 (desuHafn_Y51)
            </option><option value="18007">
                Desulfococcus oleovorans Hxd3 (18007)
            </option><option value="desuPsyc_LSV54">
                Desulfotalea psychrophila LSv54 (desuPsyc_LSV54)
            </option><option value="desuRedu_MI_1">
                Desulfotomaculum reducens MI-1 (desuRedu_MI_1)
            </option><option value="329">
                Desulfovibrio desulfuricans G20 (329)
            </option><option value="17227">
                Desulfovibrio vulgaris subsp. vulgaris DP4 (17227)
            </option><option value="desuVulg_HILDENBOROUG">
                Desulfovibrio vulgaris subsp. vulgaris str. Hildenborough (desuVulg_HILDENBOROUG)
            </option><option value="dichNodo_VCS1703A">
                Dichelobacter nodosus VCS1703A (dichNodo_VCS1703A)
            </option><option value="17417">
                Dinoroseobacter shibae DFL 12 (17417)
            </option><option value="canFam1">
                Dog July 2004 (Broad/canFam1) (canFam1)
            </option><option value="canFam4">
                Dog Mar. 2020 (UU_Cfam_GSD_1.0/canFam4) (canFam4)
            </option><option value="canFamPoodle1">
                Dog May 2003 (TIGR Poodle/canFamPoodle1) (canFamPoodle1)
            </option><option value="canFam2">
                Dog May 2005 (Broad/canFam2) (canFam2)
            </option><option value="canFam5">
                Dog May 2019 (UMICH_Zoey_3.1/canFam5) (canFam5)
            </option><option value="canFam6">
                Dog Oct. 2020 (Dog10K_Boxer_Tasha/canFam6) (canFam6)
            </option><option value="turTru1">
                Dolphin Feb. 2008 (Broad/turTru1) (turTru1)
            </option><option value="turTru2">
                Dolphin Oct. 2011 (Baylor Ttru_1.4/turTru2) (turTru2)
            </option><option value="capHir1">
                Domestic goat May 2012 (CHIR_1.0/capHir1) (capHir1)
            </option><option value="picPub1">
                Downy woodpecker Jun 2014 (ASM69900v1/picPub1) (picPub1)
            </option><option value="eboVir2">
                Ebola virus 49 sequences: 5 Ebola, 2 Marburg strains (eboVir2)
            </option><option value="eboVir1">
                Ebola virus Aug. 2014 (46 strains eboVir1/eboVir1) (eboVir1)
            </option><option value="eboVir3">
                Ebola virus Sierra Leone 2014 (G3683/KM034562.1/eboVir3) (eboVir3)
            </option><option value="ecoRef3">
                Ecoref Feb. 2013 (ecoRef3/ecoRef3) (ecoRef3)
            </option><option value="ecoRef4">
                Ecoref Feb. 2013 (ecoRef4/ecoRef4) (ecoRef4)
            </option><option value="ecoRef1">
                Ecoref Jan. 2013 (ecoRef1/ecoRef1) (ecoRef1)
            </option><option value="ecoRef2">
                Ecoref Jan. 2013 (ecoRef2/ecoRef2) (ecoRef2)
            </option><option value="10694">
                Ehrlichia canis str. Jake (10694)
            </option><option value="325">
                Ehrlichia chaffeensis str. Arkansas (325)
            </option><option value="13356">
                Ehrlichia ruminantium str. Gardel (13356)
            </option><option value="13355">
                Ehrlichia ruminantium str. Welgevonden (13355)
            </option><option value="ehrlRumi_WELGEVONDEN">
                Ehrlichia ruminantium str. Welgevonden (ehrlRumi_WELGEVONDEN)
            </option><option value="Elaeis_guineensis_EG5.1">
                Elaeis guineensis EG5.1
            </option><option value="Elaeis_guineensis_NCBI_Genome_v1.0">
                Elaeis guineensis NCBI Genome v1.0
            </option><option value="loxAfr2">
                Elephant Jul. 2008 (Broad/loxAfr2) (loxAfr2)
            </option><option value="loxAfr3">
                Elephant Jul. 2009 (Broad/loxAfr3) (loxAfr3)
            </option><option value="loxAfr1">
                Elephant May 2005 (Broad/loxAfr1) (loxAfr1)
            </option><option value="calMil1">
                Elephant shark Dec. 2013 (Callorhinchus_milii-6.1.3/calMil1) (calMil1)
            </option><option value="aptFor1">
                Emperor penguin 06 Jun 2014 (ASM69914v1/aptFor1) (aptFor1)
            </option><option value="12720">
                Enterobacter sakazakii ATCC BAA-894 (12720)
            </option><option value="ente638">
                Enterobacter sp. 638 (ente638)
            </option><option value="enteFaec_V583">
                Enterococcus faecalis V583 (enteFaec_V583)
            </option><option value="Erythranthe_guttata_JGI_v2.0">
                Erythranthe guttata JGI v2.0
            </option><option value="erytLito_HTCC2594">
                Erythrobacter litoralis HTCC2594 (erytLito_HTCC2594)
            </option><option value="16235">
                Escherichia coli 536 (16235)
            </option><option value="eschColi_APEC_O1">
                Escherichia coli APEC O1 (eschColi_APEC_O1)
            </option><option value="18083">
                Escherichia coli C str. ATCC 8739 (18083)
            </option><option value="eschColi_CFT073">
                Escherichia coli CFT073 (eschColi_CFT073)
            </option><option value="20079">
                Escherichia coli DH10B (20079)
            </option><option value="13960">
                Escherichia coli E24377A (13960)
            </option><option value="eschColi_EC4115">
                Escherichia coli EC4115 (eschColi_EC4115)
            </option><option value="13959">
                Escherichia coli HS (13959)
            </option><option value="eschColi_K12">
                Escherichia coli K12 (eschColi_K12)
            </option><option value="259">
                Escherichia coli O157:H7 EDL933 (259)
            </option><option value="eschColi_O157H7">
                Escherichia coli O157:H7 str. Sakai (eschColi_O157H7)
            </option><option value="19469">
                Escherichia coli SECEC SMS-3-5 (19469)
            </option><option value="eschColi_TW14359">
                Escherichia coli TW14359 (eschColi_TW14359)
            </option><option value="16259">
                Escherichia coli UTI89 (16259)
            </option><option value="16351">
                Escherichia coli W3110 (16351)
            </option><option value="euaGli13">
                Euarchontoglires Apr. 24. 2006 (UCSC Recon/euaGli13) (euaGli13)
            </option><option value="Eucalyptus_grandis_JGI_assembly_v1.0_annotation_v1.1">
                Eucalyptus grandis JGI assembly v1.0, annotation v1.1
            </option><option value="Eucalyptus_grandis_JGI_v2.0">
                Eucalyptus grandis JGI v2.0
            </option><option value="dicLab1">
                European sea bass Apr 2014 (seabass_V1.0/dicLab1) (dicLab1)
            </option><option value="eutHer13">
                Eutheria Apr. 24. 2006 (UCSC Recon/eutHer13) (eutHer13)
            </option><option value="evoSim1">
                Evosim Feb. 2013 (evoSim1/evoSim1) (evoSim1)
            </option><option value="musFur1">
                Ferret  Apr. 2011 (MusPutFur1.0/musFur1) (musFur1)
            </option><option value="16719">
                Fervidobacterium nodosum Rt17-B1 (16719)
            </option><option value="18981">
                Finegoldia magna ATCC 29328 (18981)
            </option><option value="Schizosaccharomyces_pombe_1.1">
                Fission Yeast (S. pombe) (Schizosaccharomyces_pombe_1.1)
            </option><option value="sebRub1">
                Flag rockfish 22 Oct-2013 (SRub1.0/sebRub1) (sebRub1)
            </option><option value="flavJohn_UW101">
                Flavobacterium johnsoniae UW101 (flavJohn_UW101)
            </option><option value="19979">
                Flavobacterium psychrophilum JIP02/86 (19979)
            </option><option value="AaloF1">
                Foshan strain genomic contigs sequences, AaloF1 assembly.
            </option><option value="Fragaria_vesca_JGI_v1.1">
                Fragaria vesca JGI v1.1
            </option><option value="Fragaria_vesca_Strawberry_Genome_1.0">
                Fragaria vesca Strawberry Genome 1.0
            </option><option value="27853">
                Francisella philomiragia subsp. philomiragia ATCC 25017 (27853)
            </option><option value="16421">
                Francisella tularensis subsp. holarctica (16421)
            </option><option value="20197">
                Francisella tularensis subsp. holarctica FTA (20197)
            </option><option value="17265">
                Francisella tularensis subsp. holarctica OSU18 (17265)
            </option><option value="16088">
                Francisella tularensis subsp. novicida U112 (16088)
            </option><option value="17375">
                Francisella tularensis subsp. tularensis FSC198 (17375)
            </option><option value="franTula_TULARENSIS">
                Francisella tularensis subsp. tularensis SCHU S4 (franTula_TULARENSIS)
            </option><option value="18459">
                Francisella tularensis subsp. tularensis WY96-3418 (18459)
            </option><option value="17403">
                Frankia alni ACN14a (17403)
            </option><option value="franCcI3">
                Frankia sp. CcI3 (franCcI3)
            </option><option value="13915">
                Frankia sp. EAN1pec (13915)
            </option><option value="fr1">
                Fugu Aug. 2002 (JGI 3.0/fr1) (fr1)
            </option><option value="fr2">
                Fugu Oct. 2004 (JGI 4.0/fr2) (fr2)
            </option><option value="fr3">
                Fugu Oct. 2011 (FUGU5/fr3) (fr3)
            </option><option value="fusoNucl">
                Fusobacterium nucleatum subsp. nucleatum ATCC 25586 (fusoNucl)
            </option><option value="hg19Patch2">
                GRCh37.p2 Aug. 2009 (GRCh37.p2/hg19Patch2) (hg19Patch2)
            </option><option value="hg38Patch2">
                GRCh38.p2 Dec 2014 (hg38Patch2)
            </option><option value="thaSir1">
                Garter snake Jun. 2015 (Thamnophis_sirtalis-6.0/thaSir1) (thaSir1)
            </option><option value="geobKaus_HTA426">
                Geobacillus kaustophilus HTA426 (geobKaus_HTA426)
            </option><option value="geobTher_NG80_2">
                Geobacillus thermodenitrificans NG80-2 (geobTher_NG80_2)
            </option><option value="geobMeta_GS15">
                Geobacter metallireducens GS-15 (geobMeta_GS15)
            </option><option value="geobSulf">
                Geobacter sulfurreducens PCA (geobSulf)
            </option><option value="geobUran_RF4">
                Geobacter uraniireducens Rf4 (geobUran_RF4)
            </option><option value="gavGan0">
                Gharial APR 2009 (preliminary/gavGan0) (gavGan0)
            </option><option value="nomLeu1">
                Gibbon Jan. 2010 (GGSC Nleu1.0/nomLeu1) (nomLeu1)
            </option><option value="nomLeu2">
                Gibbon Jun. 2011 (GGSC Nleu1.1/nomLeu2) (nomLeu2)
            </option><option value="nomLeu3">
                Gibbon Oct. 2012 (GGSC Nleu3.0/nomLeu3) (nomLeu3)
            </option><option value="gliRes13">
                Glires Apr. 24. 2006 (UCSC Recon/gliRes13) (gliRes13)
            </option><option value="gloeViol">
                Gloeobacter violaceus PCC 7421 (gloeViol)
            </option><option value="377">
                Gluconacetobacter diazotrophicus PAl 5 (377)
            </option><option value="glucOxyd_621H">
                Gluconobacter oxydans 621H (glucOxyd_621H)
            </option><option value="Glycine_max_JGI_Glyma1.1_annotation_of_the_chromosome-based_Glyma1_assembly">
                Glycine max JGI Glyma1.1 annotation of the chromosome-based Glyma1 assembly
            </option><option value="Glycine_max_JGI_Wm82.a2.v1">
                Glycine max JGI Wm82.a2.v1
            </option><option value="aquChr1">
                Golden eagle May 2014 (AquilaChrysaetos1/aquChr1) (aquChr1)
            </option><option value="aquChr2">
                Golden eagle Oct. 2014 (aquChr-1.0.2/aquChr2) (aquChr2)
            </option><option value="mesAur1">
                Golden hamster Mar 2013 (MesAur1.0/mesAur1) (mesAur1)
            </option><option value="rhiRox1">
                Golden snub-nosed monkey Oct. 2014 (Rrox_v1/rhiRox1) (rhiRox1)
            </option><option value="manVit1">
                Golden-collared manakin May 2014 (ASM69201v1/manVit1) (manVit1)
            </option><option value="gorGor2">
                Gorilla Aug. 2009 (Sanger 4/gorGor2) (gorGor2)
            </option><option value="gorGor6">
                Gorilla Aug. 2019 (Kamilah_GGO_v0/gorGor6) (gorGor6)
            </option><option value="gorGor4">
                Gorilla Dec 2014 (gorGor4.1/gorGor4) (gorGor4)
            </option><option value="gorGor5">
                Gorilla Mar. 2016 (GSMRT3/gorGor5) (gorGor5)
            </option><option value="gorGor3">
                Gorilla May 2011 (gorGor3.1/gorGor3) (gorGor3)
            </option><option value="gorGor1">
                Gorilla Oct. 2008 (Sanger 0.1/gorGor1) (gorGor1)
            </option><option value="Gossypium_raimondii_JGI_annotation_v2.1_on_assembly_v2.0">
                Gossypium raimondii JGI annotation v2.1 on assembly v2.0
            </option><option value="Gossypium_raimondii_JGI_v2.1">
                Gossypium raimondii JGI v2.1
            </option><option value="gramFors_KT0803">
                Gramella forsetii KT0803 (gramFors_KT0803)
            </option><option value="granBeth_CGDNIH1">
                Granulibacter bethesdensis CGDNIH1 (granBeth_CGDNIH1)
            </option><option value="hg19Patch10">
                Grch37.p10 Sep. 2012 (GRCh37.p10/hg19Patch10) (hg19Patch10)
            </option><option value="hg19Patch5">
                Grch37.p5 Jun. 2011 (GRCh37.p5/hg19Patch5) (hg19Patch5)
            </option><option value="hg19Patch9">
                Grch37.p9 Jul. 2012 (GRCh37.p9/hg19Patch9) (hg19Patch9)
            </option><option value="mm10Patch1">
                Grcm38.p1 Aug. 2012 (GRCm38.p1/mm10Patch1) (mm10Patch1)
            </option><option value="phaCar1">
                Great cormorant Jun 2014 (ASM70892v1/phaCar1) (phaCar1)
            </option><option value="rhiFer1">
                Greater horseshoe bat Sep. 2013 (ASM46549v1/rhiFer1) (rhiFer1)
            </option><option value="chlSab2">
                Green monkey 25 Mar-2014 (Chlorocebus_sabeus 1.1/chlSab2) (chlSab2)
            </option><option value="cheMyd1">
                Green seaturtle Mar 2013 (CheMyd_1.0/cheMyd1) (cheMyd1)
            </option><option value="cavPor3">
                Guinea pig Feb. 2008 (Broad/cavPor3) (cavPor3)
            </option><option value="poeRet1">
                Guppy Apr. 2014 (Guppy_female_1.0/poeRet1) (poeRet1)
            </option><option value="haeCon1">
                H. contortus Dec. 2009 (WormBase WS210/haeCon1) (haeCon1)
            </option><option value="38">
                Haemophilus ducreyi 35000HP (38)
            </option><option value="11752">
                Haemophilus influenzae 86-028NP (11752)
            </option><option value="16400">
                Haemophilus influenzae PittEE (16400)
            </option><option value="16401">
                Haemophilus influenzae PittGG (16401)
            </option><option value="haemInfl_KW20">
                Haemophilus influenzae Rd KW20 (haemInfl_KW20)
            </option><option value="haemSomn_129PT">
                Haemophilus somnus 129PT (haemSomn_129PT)
            </option><option value="388">
                Haemophilus somnus 2336 (388)
            </option><option value="haheChej_KCTC_2396">
                Hahella chejuensis KCTC 2396 (haheChej_KCTC_2396)
            </option><option value="halMar1">
                Haloarcula marismortui ATCC 43049 (halMar1)
            </option><option value="106">
                Halobacterium salinarum R1 (106)
            </option><option value="haloHalo1">
                Halobacterium sp. NRC-1 (haloHalo1)
            </option><option value="haloWals1">
                Haloquadratum walsbyi DSM 16790 (haloWals1)
            </option><option value="haloHalo_SL1">
                Halorhodospira halophila SL1 (haloHalo_SL1)
            </option><option value="neoSch1">
                Hawaiian monk seal Jun. 2017 (ASM220157v1/neoSch1) (neoSch1)
            </option><option value="PhumU1">
                Head Louse (Pediculus humanus) (PhumU1)
            </option><option value="eriEur1">
                Hedgehog June 2006 (Broad/eriEur1) (eriEur1)
            </option><option value="eriEur2">
                Hedgehog May 2012 (EriEur2.0/eriEur2) (eriEur2)
            </option><option value="heliAcin_SHEEBA">
                Helicobacter acinonychis str. Sheeba (heliAcin_SHEEBA)
            </option><option value="heliHepa">
                Helicobacter hepaticus ATCC 51449 (heliHepa)
            </option><option value="heliPylo_26695">
                Helicobacter pylori 26695 (heliPylo_26695)
            </option><option value="heliPylo_HPAG1">
                Helicobacter pylori HPAG1 (heliPylo_HPAG1)
            </option><option value="heliPylo_J99">
                Helicobacter pylori J99 (heliPylo_J99)
            </option><option value="13427">
                Heliobacterium modesticaldum Ice1 (13427)
            </option><option value="hermArse">
                Herminiimonas arsenicoxydans (hermArse)
            </option><option value="16523">
                Herpetosiphon aurantiacus ATCC 23779 (16523)
            </option><option value="Hevea_brasiliensis_NCBI_Genomes_v1">
                Hevea brasiliensis NCBI Genomes v1
            </option><option value="hg19LggInv">
                Hg19 lgg inversion Mar 2013 (1/hg19LggInv) (hg19LggInv)
            </option><option value="hlaRef1">
                Hlaref Apr. 2012 (hlaRefV1/hlaRef1) (hlaRef1)
            </option><option value="hlaRef2">
                Hlaref Apr. 2012 (hlaRefV2/hlaRef2) (hlaRef2)
            </option><option value="hlaRef3">
                Hlaref Apr. 2012 (hlaRefV3/hlaRef3) (hlaRef3)
            </option><option value="opiHoa1">
                Hoatzin May 2014 (ASM69207v1/opiHoa1) (opiHoa1)
            </option><option value="homIni13">
                Hominidae Apr. 24. 2006 (UCSC Recon/homIni13) (homIni13)
            </option><option value="homIni14">
                Hominidae Oct. 1. 2006 (UCSC Recon/homIni14) (homIni14)
            </option><option value="GRCh38">
                Homo Sapiens (GRCh38) Transcriptome + ERCC
            </option><option value="Homo_sapiens_nuHg19_mtrCRS">
                Homo sapiens (hg19 with mtDNA replaced with rCRS) (Homo_sapiens_nuHg19_mtrCRS)
            </option><option value="hg_g1k_v37">
                Homo sapiens b37 (hg_g1k_v37)
            </option><option value="CHM1">
                Homo sapiens hydatidiform mole assembly chm1 June 2013  (1/CHM1) (CHM1)
            </option><option value="apiMel3">
                Honeybee (Apis mellifera): apiMel3 (apiMel3)
            </option><option value="corCor1">
                Hooded crow Aug 2014 (Hooded_Crow_genome/corCor1) (corCor1)
            </option><option value="Hordeum_vulgare_Ensembl_Genomes_ASM32608v1">
                Hordeum vulgare Ensembl Genomes ASM32608v1
            </option><option value="Hordeum_vulgare_IBC_PGSB_v2">
                Hordeum vulgare IBC PGSB v2
            </option><option value="Hordeum_vulgare_MIPS">
                Hordeum vulgare MIPS
            </option><option value="equCab1">
                Horse Jan. 2007 (Broad/equCab1) (equCab1)
            </option><option value="equCab3">
                Horse Jan. 2018 (EquCab3.0/equCab3) (equCab3)
            </option><option value="equCab2">
                Horse Sep. 2007 (Broad/equCab2) (equCab2)
            </option><option value="chlUnd1">
                Houbara bustard Jun 2014 (ASM69519v1/chlUnd1) (chlUnd1)
            </option><option value="hg15">
                Human Apr. 2003 (NCBI33/hg15) (hg15)
            </option><option value="CHM13_T2T_v2.0">
                Human CHM13 2.0 (T2T Consortium) Jan. 2022
            </option><option value="hg38">
                Human Dec. 2013 (GRCh38/hg38) (hg38)
            </option><option value="hg19">
                Human Feb. 2009 (GRCh37/hg19) (hg19)
            </option><option value="CHM13_T2T_v1.0">
                Human Jan. 2021 isolate CHM13 (T2T Consortium) female
            </option><option value="hs1">
                Human Jan. 2022 (T2T CHM13v2.0/hs1) (hs1)
            </option><option value="hg16">
                Human July 2003 (NCBI34/hg16) (hg16)
            </option><option value="hg18">
                Human Mar. 2006 (NCBI36/hg18) (hg18)
            </option><option value="hg17">
                Human May 2004 (NCBI35/hg17) (hg17)
            </option><option value="15217">
                Human herpesvirus 1 (15217)
            </option><option value="repeats2">
                Human repeats Nov. 2012 (1/repeats2) (repeats2)
            </option><option value="Hydra_JCVI">
                Hydra magnipapillata str. 105 (Hydra_JCVI)
            </option><option value="hypeButy1">
                Hyperthermus butylicus DSM 5456 (hypeButy1)
            </option><option value="hyphNept_ATCC15444">
                Hyphomonas neptunium ATCC 15444 (hyphNept_ATCC15444)
            </option><option value="idioLoih_L2TR">
                Idiomarina loihiensis L2TR (idioLoih_L2TR)
            </option><option value="13914">
                Ignicoccus hospitalis KIN4/I (13914)
            </option><option value="megLyr1">
                Indian false vampire Sep. 2013 (ASM46534v1/megLyr1) (megLyr1)
            </option><option value="h7n7_360722">
                Influenza A virus (A/seal/Mass/1-SC35M/1980(H7N7))
            </option><option value="venter1">
                J. Craig Venter Sep. 2007 (HuRef/venter1) (venter1)
            </option><option value="jannCCS1">
                Jannaschia sp. CCS1 (jannCCS1)
            </option><option value="16549">
                Janthinobacterium sp. Marseille (16549)
            </option><option value="angJap1">
                Japanese eel 18 Mar-2014 (japanese_eel_genome_v1_25_oct_2011_japonica_c401b400k25m200_sspacepremiumk3a02n24_extra.final.scaffolds/angJap1) (angJap1)
            </option><option value="cotJap1">
                Japanese quail 26 Jul-2013 (Coja_1.0/cotJap1) (cotJap1)
            </option><option value="dipOrd1">
                Kangaroo rat Jul. 2008 (Broad/dipOrd1) (dipOrd1)
            </option><option value="chaVoc2">
                Killdeer Aug 2014 (ASM70802v2/chaVoc2) (chaVoc2)
            </option><option value="chaVoc1">
                Killdeer Jun 2014 (ASM70802v1/chaVoc1) (chaVoc1)
            </option><option value="orcOrc1">
                Killer whale Jan 2013 (Oorc_1.1/orcOrc1) (orcOrc1)
            </option><option value="10689">
                Kineococcus radiotolerans SRS30216 (10689)
            </option><option value="kirRef">
                Kirref Apr. 2011 (kirRef/kirRef) (kirRef)
            </option><option value="kirRefV2">
                Kirrefv2 Apr. 2011 (kirRefV2/kirRefV2) (kirRefV2)
            </option><option value="kirRefV3">
                Kirrefv3 Apr. 2011 (kirRefV3/kirRefV3) (kirRefV3)
            </option><option value="kirRefV4">
                Kirrefv4 Apr. 2011 (kirRefV4/kirRefV4) (kirRefV4)
            </option><option value="kirRefV5">
                Kirrefv5 Apr. 2011 (kirRefV5/kirRefV5) (kirRefV5)
            </option><option value="31">
                Klebsiella pneumoniae subsp. pneumoniae MGH 78578 (31)
            </option><option value="Homo_sapiens_AK1">
                Korean Man (Homo_sapiens_AK1)
            </option><option value="82">
                Lactobacillus acidophilus NCFM (82)
            </option><option value="404">
                Lactobacillus brevis ATCC 367 (404)
            </option><option value="402">
                Lactobacillus casei ATCC 334 (402)
            </option><option value="16871">
                Lactobacillus delbrueckii subsp. bulgaricus ATCC 11842 (16871)
            </option><option value="403">
                Lactobacillus delbrueckii subsp. bulgaricus ATCC BAA-365 (403)
            </option><option value="84">
                Lactobacillus gasseri ATCC 33323 (84)
            </option><option value="17811">
                Lactobacillus helveticus DPC 4571 (17811)
            </option><option value="9638">
                Lactobacillus johnsonii NCC 533 (9638)
            </option><option value="lactPlan">
                Lactobacillus plantarum WCFS1 (lactPlan)
            </option><option value="15766">
                Lactobacillus reuteri F275 (15766)
            </option><option value="13435">
                Lactobacillus sakei subsp. sakei 23K (13435)
            </option><option value="lactSali_UCC118">
                Lactobacillus salivarius UCC118 (lactSali_UCC118)
            </option><option value="18797">
                Lactococcus lactis subsp. cremoris MG1363 (18797)
            </option><option value="401">
                Lactococcus lactis subsp. cremoris SK11 (401)
            </option><option value="lactLact">
                Lactococcus lactis subsp. lactis Il1403 (lactLact)
            </option><option value="petMar3">
                Lamprey Dec. 2017 (Pmar_germline 1.0/petMar3) (petMar3)
            </option><option value="petMar1">
                Lamprey Mar. 2007 (WUGSC 3.0/petMar1) (petMar1)
            </option><option value="petMar2">
                Lamprey Sep. 2010 (WUGSC 7.0/petMar2) (petMar2)
            </option><option value="braFlo2">
                Lancelet Apr. 2008 (JGI v2.0/braFlo2) (braFlo2)
            </option><option value="braFlo1">
                Lancelet Mar. 2006 (JGI 1.0/braFlo1) (braFlo1)
            </option><option value="lauRas13">
                Laurasiatheria Apr. 24. 2006 (UCSC Recon/lauRas13) (lauRas13)
            </option><option value="lawsIntr_PHE_MN1_00">
                Lawsonia intracellularis PHE/MN1-00 (lawsIntr_PHE_MN1_00)
            </option><option value="17491">
                Legionella pneumophila str. Corby (17491)
            </option><option value="13126">
                Legionella pneumophila str. Lens (13126)
            </option><option value="13127">
                Legionella pneumophila str. Paris (13127)
            </option><option value="legiPneu_PHILADELPHIA">
                Legionella pneumophila subsp. pneumophila str. Philadelphia 1 (legiPneu_PHILADELPHIA)
            </option><option value="leifXyli_XYLI_CTCB0">
                Leifsonia xyli subsp. xyli str. CTCB07 (leifXyli_XYLI_CTCB0)
            </option><option value="lMaj5">
                Leishmania major 2005 (lMaj5)
            </option><option value="16148">
                Leptospira borgpetersenii serovar Hardjo-bovis JB197 (16148)
            </option><option value="16146">
                Leptospira borgpetersenii serovar Hardjo-bovis L550 (16146)
            </option><option value="10687">
                Leptospira interrogans serovar Copenhageni str. Fiocruz L1-130 (10687)
            </option><option value="leptInte">
                Leptospira interrogans serovar Lai str. 56601 (leptInte)
            </option><option value="20039">
                Leptothrix cholodnii SP-6 (20039)
            </option><option value="jacJac1">
                Lesser Egyptian jerboa May 2012 (JacJac1.0/jacJac1) (jacJac1)
            </option><option value="16062">
                Leuconostoc citreum KM20 (16062)
            </option><option value="leucMese_ATCC8293">
                Leuconostoc mesenteroides subsp. mesenteroides ATCC 8293 (leucMese_ATCC8293)
            </option><option value="lotGig0">
                Limpet Jul. 2013 (lotGig0/lotGig0) (lotGig0)
            </option><option value="listInno">
                Listeria innocua Clip11262 (listInno)
            </option><option value="276">
                Listeria monocytogenes EGD-e (276)
            </option><option value="85">
                Listeria monocytogenes str. 4b F2365 (85)
            </option><option value="13443">
                Listeria welshimeri serovar 6b str. SLCC5334 (13443)
            </option><option value="egrGar1">
                Little egret May 2014 (ASM68718v1/egrGar1) (egrGar1)
            </option><option value="anoCar1">
                Lizard Feb. 2007 (Broad/anoCar1) (anoCar1)
            </option><option value="anoCar2">
                Lizard May 2010 (Broad AnoCar2.0/anoCar2) (anoCar2)
            </option><option value="Lotus_japonicus_Kazusa_2.5">
                Lotus japonicus Kazusa 2.5
            </option><option value="19619">
                Lysinibacillus sphaericus C3-41 (19619)
            </option><option value="musDom2">
                M. domestica 22 Apr 2013 (Musca_domestica-2.0.2/musDom2) (musDom2)
            </option><option value="melHap1">
                M. hapla Sep. 2008 (M. hapla VW9 WS210/melHap1) (melHap1)
            </option><option value="melInc1">
                M. incognita Feb. 2008 (M. incognita WS210/melInc1) (melInc1)
            </option><option value="mmtv">
                MMTV Nov. 2009 (MMTV/mmtv) (mmtv)
            </option><option value="magnMC1">
                Magnetococcus sp. MC-1 (magnMC1)
            </option><option value="magnMagn_AMB_1">
                Magnetospirillum magneticum AMB-1 (magnMagn_AMB_1)
            </option><option value="Zea_mays_B73_RefGen_v2">
                Maize (Zea mays) (Zea_mays_B73_RefGen_v2)
            </option><option value="galVar1">
                Malayan flying lemur Jun. 2014 (G_variegatus-3.0.2/galVar1) (galVar1)
            </option><option value="anaPla1">
                Mallard duck Apr 2013 (BGI_duck_1.0/anaPla1) (anaPla1)
            </option><option value="Malus_domestica_IASMA">
                Malus domestica IASMA
            </option><option value="Malus_domestica_Penguin_Genomics_v2">
                Malus domestica Penguin Genomics v2
            </option><option value="triMan1">
                Manatee Oct. 2011 (Broad v1.0/triMan1) (triMan1)
            </option><option value="Manihot_esculenta_Cassava4">
                Manihot esculenta Cassava4
            </option><option value="Manihot_esculenta_JGI_v6.1">
                Manihot esculenta JGI v6.1
            </option><option value="mannSucc_MBEL55E">
                Mannheimia succiniciproducens MBEL55E (mannSucc_MBEL55E)
            </option><option value="marVir1">
                Marburg virus Sep. 2014 (NC_024781.1/marVir1) (marVir1)
            </option><option value="Marchantia_polymorpha_JGI_v3.1">
                Marchantia polymorpha JGI v3.1
            </option><option value="mariMari_MCS10">
                Maricaulis maris MCS10 (mariMari_MCS10)
            </option><option value="mariAqua_VT8">
                Marinobacter aquaeolei VT8 (mariAqua_VT8)
            </option><option value="17445">
                Marinomonas sp. MWYL1 (17445)
            </option><option value="calJac1">
                Marmoset June 2007 (WUGSC 2.0.2/calJac1) (calJac1)
            </option><option value="calJac3">
                Marmoset March 2009 (WUGSC 3.2/calJac3) (calJac3)
            </option><option value="calJac4">
                Marmoset May 2020 (Callithrix_jacchus_cj1700_1.1/calJac4) (calJac4)
            </option><option value="oryLat1">
                Medaka Apr. 2006 (NIG/UT MEDAKA1/oryLat1) (oryLat1)
            </option><option value="oryLat2">
                Medaka Oct. 2005 (NIG/UT MEDAKA1/oryLat2) (oryLat2)
            </option><option value="Medicago_truncatula_JCVI_4.0">
                Medicago truncatula JCVI 4.0
            </option><option value="Medicago_truncatula_JGI_Mt4.0v1">
                Medicago truncatula JGI Mt4.0v1
            </option><option value="geoFor1">
                Medium ground finch Apr. 2012 (GeoFor_1.0/geoFor1) (geoFor1)
            </option><option value="pteVam1">
                Megabat Jul. 2008 (Broad/pteVam1) (pteVam1)
            </option><option value="mesoFlor_L1">
                Mesoplasma florum L1 (mesoFlor_L1)
            </option><option value="mesoLoti">
                Mesorhizobium loti MAFF303099 (mesoLoti)
            </option><option value="10690">
                Mesorhizobium sp. BNC1 (10690)
            </option><option value="metaSedu">
                Metallosphaera sedula DSM 5348 (metaSedu)
            </option><option value="methSmit1">
                Methanobrevibacter smithii ATCC 35061 (methSmit1)
            </option><option value="methJann1">
                Methanocaldococcus jannaschii DSM 2661 (methJann1)
            </option><option value="methBurt2">
                Methanococcoides burtonii DSM 6242 (methBurt2)
            </option><option value="methAeol1">
                Methanococcus aeolicus Nankai-3 (methAeol1)
            </option><option value="methMari_C5_1">
                Methanococcus maripaludis C5 (methMari_C5_1)
            </option><option value="19639">
                Methanococcus maripaludis C6 (19639)
            </option><option value="methMari_C7">
                Methanococcus maripaludis C7 (methMari_C7)
            </option><option value="metMar1">
                Methanococcus maripaludis S2 (metMar1)
            </option><option value="methVann1">
                Methanococcus vannielii SB (methVann1)
            </option><option value="methLabrZ_1">
                Methanocorpusculum labreanum Z (methLabrZ_1)
            </option><option value="mculMari1">
                Methanoculleus marisnigri JR1 (mculMari1)
            </option><option value="methKand1">
                Methanopyrus kandleri AV19 (methKand1)
            </option><option value="methTherPT1">
                Methanosaeta thermophila PT (methTherPT1)
            </option><option value="metAce1">
                Methanosarcina acetivorans C2A (metAce1)
            </option><option value="methBark1">
                Methanosarcina barkeri str. Fusaro (methBark1)
            </option><option value="methMaze1">
                Methanosarcina mazei Go1 (methMaze1)
            </option><option value="methStad1">
                Methanosphaera stadtmanae DSM 3091 (methStad1)
            </option><option value="methHung1">
                Methanospirillum hungatei JF-1 (methHung1)
            </option><option value="methTher1">
                Methanothermobacter thermautotrophicus str. Delta H (methTher1)
            </option><option value="methPetr_PM1">
                Methylibium petroleiphilum PM1 (methPetr_PM1)
            </option><option value="methFlag_KT">
                Methylobacillus flagellatus KT (methFlag_KT)
            </option><option value="18637">
                Methylobacterium extorquens PA1 (18637)
            </option><option value="18817">
                Methylobacterium radiotolerans JCM 2831 (18817)
            </option><option value="18809">
                Methylobacterium sp. 4-46 (18809)
            </option><option value="methCaps_BATH">
                Methylococcus capsulatus str. Bath (methCaps_BATH)
            </option><option value="astMex1">
                Mexican tetra (cavefish) Apr 2013 (Astyanax_mexicanus-1.0.2/astMex1) (astMex1)
            </option><option value="mhcRefnoNs2">
                Mhcref Apr. 2011 (mhcRefnoNsV2/mhcRefnoNs2) (mhcRefnoNs2)
            </option><option value="mhcRef">
                Mhcref Jan. 2011 (mhcRefV1/mhcRef) (mhcRef)
            </option><option value="mhcRefnoNs10">
                Mhcref July 2011 (mhcRefnoNsV10/mhcRefnoNs10) (mhcRefnoNs10)
            </option><option value="mhcRefnoNs11">
                Mhcref July 2011 (mhcRefnoNsV11/mhcRefnoNs11) (mhcRefnoNs11)
            </option><option value="mhcRefnoNs12">
                Mhcref July 2011 (mhcRefnoNsV12/mhcRefnoNs12) (mhcRefnoNs12)
            </option><option value="mhcRefnoNs13">
                Mhcref July 2011 (mhcRefnoNsV13/mhcRefnoNs13) (mhcRefnoNs13)
            </option><option value="mhcRefnoNs14">
                Mhcref July 2011 (mhcRefnoNsV14/mhcRefnoNs14) (mhcRefnoNs14)
            </option><option value="mhcRefnoNs15">
                Mhcref July 2011 (mhcRefnoNsV15/mhcRefnoNs15) (mhcRefnoNs15)
            </option><option value="mhcRefnoNs16">
                Mhcref July 2011 (mhcRefnoNsV16/mhcRefnoNs16) (mhcRefnoNs16)
            </option><option value="mhcRefnoNs17">
                Mhcref July 2011 (mhcRefnoNsV17/mhcRefnoNs17) (mhcRefnoNs17)
            </option><option value="mhcRefnoNs5">
                Mhcref July 2011 (mhcRefnoNsV5/mhcRefnoNs5) (mhcRefnoNs5)
            </option><option value="mhcRefnoNs6">
                Mhcref July 2011 (mhcRefnoNsV6/mhcRefnoNs6) (mhcRefnoNs6)
            </option><option value="mhcRefnoNs7">
                Mhcref July 2011 (mhcRefnoNsV7/mhcRefnoNs7) (mhcRefnoNs7)
            </option><option value="mhcRefnoNs8">
                Mhcref July 2011 (mhcRefnoNsV8/mhcRefnoNs8) (mhcRefnoNs8)
            </option><option value="mhcRefnoNs9">
                Mhcref July 2011 (mhcRefnoNsV8/mhcRefnoNs9) (mhcRefnoNs9)
            </option><option value="mhcRefnoNs">
                Mhcref June 2011 (mhcRefnoNsV1/mhcRefnoNs) (mhcRefnoNs)
            </option><option value="mhcRefnoNs3">
                Mhcref June 2011 (mhcRefnoNsV3/mhcRefnoNs3) (mhcRefnoNs3)
            </option><option value="mhcRefnoNs4">
                Mhcref June 2011 (mhcRefnoNsV4/mhcRefnoNs4) (mhcRefnoNs4)
            </option><option value="mhcRefnoNs22">
                Mhcref Mar. 2012 (mhcRefnoNsV22/mhcRefnoNs22) (mhcRefnoNs22)
            </option><option value="mhcRefnoNs23">
                Mhcref Mar. 2012 (mhcRefnoNsV23/mhcRefnoNs23) (mhcRefnoNs23)
            </option><option value="mhcRefnoNs24">
                Mhcref Mar. 2012 (mhcRefnoNsV24/mhcRefnoNs24) (mhcRefnoNs24)
            </option><option value="mhcRefnoNs18">
                Mhcref Oct. 2011 (mhcRefnoNsV18/mhcRefnoNs18) (mhcRefnoNs18)
            </option><option value="mhcRefnoNs19">
                Mhcref Oct. 2011 (mhcRefnoNsV19/mhcRefnoNs19) (mhcRefnoNs19)
            </option><option value="mhcRefnoNs20">
                Mhcref Oct. 2011 (mhcRefnoNsV20/mhcRefnoNs20) (mhcRefnoNs20)
            </option><option value="mhcRefnoNs21">
                Mhcref Oct. 2011 (mhcRefnoNsV21/mhcRefnoNs21) (mhcRefnoNs21)
            </option><option value="myoLuc2">
                Microbat Jul. 2010 (Broad Institute Myoluc2.0/myoLuc2) (myoLuc2)
            </option><option value="myoLuc1">
                Microbat Mar. 2006 (Broad/myoLuc1) (myoLuc1)
            </option><option value="27835">
                Microcystis aeruginosa NIES-843 (27835)
            </option><option value="Micromonas_commoda_JGI_v3.0">
                Micromonas commoda JGI v3.0
            </option><option value="balAcu1">
                Minke whale Oct. 2013 (BalAcu1.0/balAcu1) (balAcu1)
            </option><option value="mm10Strains1">
                Mm10haps Dec. 2011 (Mouse strains/mm10Strains1) (mm10Strains1)
            </option><option value="mpxvRivers">
                Monkeypox virus MPXV-M5312_HM12_Rivers (MT903340.1/GCF_014621545.1) (mpxvRivers)
            </option><option value="moorTher_ATCC39073">
                Moorella thermoacetica ATCC 39073 (moorTher_ATCC39073)
            </option><option value="AaegL1">
                Mosquito (Aedes aegypti) (AaegL1)
            </option><option value="AgamP3">
                Mosquito (Anopheles gambiae) (AgamP3)
            </option><option value="CpipJ1">
                Mosquito (Culex quinquefasciatus) (CpipJ1)
            </option><option value="mm10">
                Mouse (Mus Musculus): mm10 Full
            </option><option value="mm7">
                Mouse Aug. 2005 (NCBI35/mm7) (mm7)
            </option><option value="mm8">
                Mouse Feb. 2006 (NCBI36/mm8) (mm8)
            </option><option value="mm9">
                Mouse July 2007 (NCBI37/mm9) (mm9)
            </option><option value="mm39">
                Mouse Jun. 2020 (GRCm39/mm39) (mm39)
            </option><option value="mm5">
                Mouse May 2004 (mm5)
            </option><option value="micMur1">
                Mouse lemur Jul. 2007 (Broad/micMur1) (micMur1)
            </option><option value="micMur2">
                Mouse lemur May 2015 (Mouse lemur/micMur2) (micMur2)
            </option><option value="Musa_acuminata_Banana_Genome_v1.0">
                Musa acuminata Banana Genome v1.0
            </option><option value="Musa_acuminata_Banana_Genome_v2.0">
                Musa acuminata Banana Genome v2.0
            </option><option value="Musa_acuminata_Genescope-Cirad">
                Musa acuminata Genescope-Cirad
            </option><option value="15691">
                Mycobacterium abscessus (15691)
            </option><option value="88">
                Mycobacterium avium 104 (88)
            </option><option value="91">
                Mycobacterium avium subsp. paratuberculosis K-10 (91)
            </option><option value="89">
                Mycobacterium bovis AF2122/97 (89)
            </option><option value="18059">
                Mycobacterium bovis BCG str. Pasteur 1173P2 (18059)
            </option><option value="15760">
                Mycobacterium gilvum PYR-GCK (15760)
            </option><option value="90">
                Mycobacterium leprae TN (90)
            </option><option value="92">
                Mycobacterium smegmatis str. MC2 155 (92)
            </option><option value="16079">
                Mycobacterium sp. JLS (16079)
            </option><option value="16081">
                Mycobacterium sp. KMS (16081)
            </option><option value="15762">
                Mycobacterium sp. MCS (15762)
            </option><option value="223">
                Mycobacterium tuberculosis CDC1551 (223)
            </option><option value="15642">
                Mycobacterium tuberculosis F11 (15642)
            </option><option value="18883">
                Mycobacterium tuberculosis H37Ra (18883)
            </option><option value="mycoTube_H37RV">
                Mycobacterium tuberculosis H37Rv (mycoTube_H37RV)
            </option><option value="16230">
                Mycobacterium ulcerans Agy99 (16230)
            </option><option value="15761">
                Mycobacterium vanbaalenii PYR-1 (15761)
            </option><option value="16095">
                Mycoplasma agalactiae PG2 (16095)
            </option><option value="16208">
                Mycoplasma capricolum subsp. capricolum ATCC 27343 (16208)
            </option><option value="409">
                Mycoplasma gallisepticum R (409)
            </option><option value="mycoGeni">
                Mycoplasma genitalium G37 (mycoGeni)
            </option><option value="13120">
                Mycoplasma hyopneumoniae 232 (13120)
            </option><option value="10639">
                Mycoplasma hyopneumoniae 7448 (10639)
            </option><option value="10675">
                Mycoplasma hyopneumoniae J (10675)
            </option><option value="10697">
                Mycoplasma mobile 163K (10697)
            </option><option value="10616">
                Mycoplasma mycoides subsp. mycoides SC str. PG1 (10616)
            </option><option value="176">
                Mycoplasma penetrans HF-2 (176)
            </option><option value="99">
                Mycoplasma pneumoniae M129 (99)
            </option><option value="100">
                Mycoplasma pulmonis UAB CTIP (100)
            </option><option value="10676">
                Mycoplasma synoviae 53 (10676)
            </option><option value="myxoXant_DK_1622">
                Myxococcus xanthus DK 1622 (myxoXant_DK_1622)
            </option><option value="hetGla2">
                Naked mole-rat Jan. 2012 (Broad HetGla_female_1.0/hetGla2) (hetGla2)
            </option><option value="hetGla1">
                Naked mole-rat Jul. 2011 (BGI HetGla_1.0/hetGla1) (hetGla1)
            </option><option value="nanEqu1">
                Nanoarchaeum equitans Kin4-M (nanEqu1)
            </option><option value="natrPhar1">
                Natronomonas pharaonis DSM 2160 (natrPhar1)
            </option><option value="neisGono_FA1090_1">
                Neisseria gonorrhoeae FA 1090 (neisGono_FA1090_1)
            </option><option value="16393">
                Neisseria meningitidis 053442 (16393)
            </option><option value="neisMeni_FAM18_1">
                Neisseria meningitidis FAM18 (neisMeni_FAM18_1)
            </option><option value="neisMeni_MC58_1">
                Neisseria meningitidis MC58 (neisMeni_MC58_1)
            </option><option value="neisMeni_Z2491_1">
                Neisseria meningitidis Z2491 (neisMeni_Z2491_1)
            </option><option value="Nelumbo_nucifera_LOTUS-DB_v1.1">
                Nelumbo nucifera LOTUS-DB v1.1
            </option><option value="neorSenn_MIYAYAMA">
                Neorickettsia sennetsu str. Miyayama (neorSenn_MIYAYAMA)
            </option><option value="oreNil1">
                Nile tilapia Jan. 2011 (Broad oreNil1.0/oreNil1) (oreNil1)
            </option><option value="oreNil2">
                Nile tilapia Jan. 2011 (Broad oreNil1.1/oreNil2) (oreNil2)
            </option><option value="18963">
                Nitratiruptor sp. SB155-2 (18963)
            </option><option value="13473">
                Nitrobacter hamburgensis X14 (13473)
            </option><option value="nitrWino_NB_255">
                Nitrobacter winogradskyi Nb-255 (nitrWino_NB_255)
            </option><option value="nitrOcea_ATCC19707">
                Nitrosococcus oceani ATCC 19707 (nitrOcea_ATCC19707)
            </option><option value="nitrEuro">
                Nitrosomonas europaea ATCC 19718 (nitrEuro)
            </option><option value="13913">
                Nitrosomonas eutropha C91 (13913)
            </option><option value="19265">
                Nitrosopumilus maritimus SCM1 (19265)
            </option><option value="nitrMult_ATCC25196">
                Nitrosospira multiformis ATCC 25196 (nitrMult_ATCC25196)
            </option><option value="nocaFarc_IFM10152">
                Nocardia farcinica IFM 10152 (nocaFarc_IFM10152)
            </option><option value="nocaJS61">
                Nocardioides sp. JS614 (nocaJS61)
            </option><option value="nonAfr13">
                Non-Afrotheria Apr. 24. 2006 (UCSC Recon/nonAfr13) (nonAfr13)
            </option><option value="merNub1">
                Northern carmine bee-eater May 2014 (ASM69184v1/merNub1) (merNub1)
            </option><option value="fulGla1">
                Northern fulmar May 2014 (ASM69083v1/fulGla1) (fulGla1)
            </option><option value="esoLuc1">
                Northern pike Jul 2014 (EsoLuc1.0/esoLuc1) (esoLuc1)
            </option><option value="nostSp">
                Nostoc sp. PCC 7120 (nostSp)
            </option><option value="calMil0">
                Notyet Oct. 2011 (notYet1/calMil0) (calMil0)
            </option><option value="novoArom_DSM12444">
                Novosphingobium aromaticivorans DSM 12444 (novoArom_DSM12444)
            </option><option value="oceaIhey">
                Oceanobacillus iheyensis HTE831 (oceaIhey)
            </option><option value="19485">
                Ochrobactrum anthropi ATCC 49188 (19485)
            </option><option value="oenoOeni_PSU_1">
                Oenococcus oeni PSU-1 (oenoOeni_PSU_1)
            </option><option value="od01">
                Oikopleura dioica (od01)
            </option><option value="onioYell_PHYTOPLASMA">
                Onion yellows phytoplasma OY-M (onioYell_PHYTOPLASMA)
            </option><option value="monDom4">
                Opossum Jan. 2006 (Broad/monDom4) (monDom4)
            </option><option value="monDom1">
                Opossum Oct. 2004 (Broad prelim/monDom1) (monDom1)
            </option><option value="monDom5">
                Opossum Oct. 2006 (Broad/monDom5) (monDom5)
            </option><option value="ponAbe3">
                Orangutan Jan. 2018 (Susie_PABv2/ponAbe3) (ponAbe3)
            </option><option value="ponAbe2">
                Orangutan July 2007 (WUGSC 2.0.2/ponAbe2) (ponAbe2)
            </option><option value="orieTsut_BORYONG">
                Orientia tsutsugamushi str. Boryong (orieTsut_BORYONG)
            </option><option value="Oropetium_thomaeum_JGI_v1.0">
                Oropetium thomaeum JGI v1.0
            </option><option value="Oryza_brachyantha_Ensembl_Plants_v1.4b">
                Oryza brachyantha Ensembl Plants v1.4b
            </option><option value="Oryza_sativa_ssp._indica_9311_BGF_2005">
                Oryza sativa ssp. indica 9311_BGF_2005
            </option><option value="Oryza_sativa_ssp._indica_MBKBASE_v1.0">
                Oryza sativa ssp. indica MBKBASE v1.0
            </option><option value="Oryza_sativa_ssp._japonica_JGI_v7.0">
                Oryza sativa ssp. japonica JGI v7.0
            </option><option value="Oryza_sativa_ssp._japonica_MSU_RGAP_7">
                Oryza sativa ssp. japonica MSU RGAP 7
            </option><option value="Ostreococcus_lucimarinus_JGI_2.0">
                Ostreococcus lucimarinus JGI 2.0
            </option><option value="Ostreococcus_lucimarinus_JGI_v2.0_assembly_and_annotation">
                Ostreococcus lucimarinus JGI v2.0 assembly and annotation
            </option><option value="strCam1">
                Ostrich 06 Jun-2014 (ASM69896v1/strCam1) (strCam1)
            </option><option value="falciparum">
                P. falciparum Plasmodium falciparum (?/falciparum) (falciparum)
            </option><option value="priPac2">
                P. pacificus Dec. 2008 (WUGSC 5.0.1/priPac2) (priPac2)
            </option><option value="priPac1">
                P. pacificus Feb. 2007 (WUGSC 5.0/priPac1) (priPac1)
            </option><option value="odoRosDiv1">
                Pacific walrus Jan 2013 (Oros_1.0/odoRosDiv1) (odoRosDiv1)
            </option><option value="chrPic1">
                Painted turtle Dec. 2011 (v3.0.1/chrPic1) (chrPic1)
            </option><option value="chrPic2">
                Painted turtle Mar. 2014 (v3.0.3/chrPic2) (chrPic2)
            </option><option value="ailMel1">
                Panda Dec. 2009 (BGI-Shenzhen 1.0/ailMel1) (ailMel1)
            </option><option value="13485">
                Parabacteroides distasonis ATCC 8503 (13485)
            </option><option value="paraDeni_PD1222">
                Paracoccus denitrificans PD1222 (paraDeni_PD1222)
            </option><option value="ptePar1">
                Parnell's mustached bat Sep. 2013 (ASM46540v1/ptePar1) (ptePar1)
            </option><option value="amaVit1">
                Parrot Jan. 2013 (AV1/amaVit1) (amaVit1)
            </option><option value="17639">
                Parvibaculum lavamentivorans DS-1 (17639)
            </option><option value="pastMult">
                Pasteurella multocida subsp. multocida str. Pm70 (pastMult)
            </option><option value="erwiCaro_ATROSEPTICA">
                Pectobacterium atrosepticum SCRI1043 (erwiCaro_ATROSEPTICA)
            </option><option value="pediPent_ATCC25745">
                Pediococcus pentosaceus ATCC 25745 (pediPent_ATCC25745)
            </option><option value="peloCarb">
                Pelobacter carbinolicus DSM 2380 (peloCarb)
            </option><option value="13384">
                Pelobacter propionicus DSM 2379 (13384)
            </option><option value="peloLute_DSM273">
                Pelodictyon luteolum DSM 273 (peloLute_DSM273)
            </option><option value="peloTher_SI">
                Pelotomaculum thermopropionicum SI (peloTher_SI)
            </option><option value="falPer1">
                Peregrine falcon Feb 2013 (F_peregrinus_v1.0/falPer1) (falPer1)
            </option><option value="17679">
                Petrotoga mobilis SJ95 (17679)
            </option><option value="Petunia_axillaris_Sol_Genomics_v1.6.2">
                Petunia axillaris Sol Genomics v1.6.2
            </option><option value="Phalaenopsis_equestris_Genomics_v1.0">
                Phalaenopsis equestris Genomics v1.0
            </option><option value="photProf_SS9">
                Photobacterium profundum SS9 (photProf_SS9)
            </option><option value="photLumi">
                Photorhabdus luminescens subsp. laumondii TTO1 (photLumi)
            </option><option value="Phyllostachys_edulis_NCGR_v1.0">
                Phyllostachys edulis NCGR v1.0
            </option><option value="Physcomitrella_patens_JGI_assembly_release_v1.1_and_COSMOSS_annotation_v1.6">
                Physcomitrella patens JGI assembly release v1.1 and COSMOSS annotation v1.6
            </option><option value="Physcomitrella_patens_JGI_v3.3">
                Physcomitrella patens JGI v3.3
            </option><option value="picrTorr1">
                Picrophilus torridus DSM 9790 (picrTorr1)
            </option><option value="susScr3">
                Pig Aug. 2011 (SGSC Sscrofa10.2/susScr3) (susScr3)
            </option><option value="susScr11">
                Pig Feb. 2017 (Sscrofa11.1/susScr11) (susScr11)
            </option><option value="Sscrofa9.58">
                Pig May 2010 (Sscrofa9.58)
            </option><option value="susScr2">
                Pig Nov. 2009 (SGSC Sscrofa9.2/susScr2) (susScr2)
            </option><option value="ochPri2">
                Pika Jul. 2008 (Broad/ochPri2) (ochPri2)
            </option><option value="ochPri3">
                Pika May 2012 (OchPri3.0/ochPri3) (ochPri3)
            </option><option value="plasFalc3D7V12">
                Plasmodium falciparum 3d7 10 Sept 2014 (V12.0/plasFalc3D7V12) (plasFalc3D7V12)
            </option><option value="ornAna2">
                Platypus Feb. 2007 (ASM227v2/ornAna2) (ornAna2)
            </option><option value="ornAna1">
                Platypus Mar. 2007 (WUGSC 5.0.1/ornAna1) (ornAna1)
            </option><option value="ursMar1">
                Polar bear 09 May-2014 (UrsMar_1.0/ursMar1) (ursMar1)
            </option><option value="13418">
                Polaromonas naphthalenivorans CJ2 (13418)
            </option><option value="polaJS66">
                Polaromonas sp. JS666 (polaJS66)
            </option><option value="polyQLWP">
                Polynucleobacter sp. QLW-P1DMWA-1 (polyQLWP)
            </option><option value="Ptrichocarpa_156">
                Poplar (Populus trichocarpa) (Ptrichocarpa_156)
            </option><option value="Populus_trichocarpa_JGI_assembly_release_v3.0_annotation_v3.0">
                Populus trichocarpa JGI assembly release v3.0, annotation v3.0
            </option><option value="Populus_trichocarpa_JGI_v3.1">
                Populus trichocarpa JGI v3.1
            </option><option value="porpGing_W83">
                Porphyromonas gingivalis W83 (porpGing_W83)
            </option><option value="perManBai1">
                Prairie deer mouse 03 Dec-2013 (Pman_1.0/perManBai1) (perManBai1)
            </option><option value="micOch1">
                Prairie vole Oct 2012 (MicOch1.0/micOch1) (micOch1)
            </option><option value="priMat13">
                Primate Apr. 24. 2006 (UCSC Recon/priMat13) (priMat13)
            </option><option value="neoBri1">
                Princess of Burundi May 2011 (NeoBri1.0/neoBri1) (neoBri1)
            </option><option value="nasLar1">
                Proboscis monkey Nov. 2014 (Charlie1.0/nasLar1) (nasLar1)
            </option><option value="13548">
                Prochlorococcus marinus str. AS9601 (13548)
            </option><option value="13551">
                Prochlorococcus marinus str. MIT 9211 (13551)
            </option><option value="18633">
                Prochlorococcus marinus str. MIT 9215 (18633)
            </option><option value="15746">
                Prochlorococcus marinus str. MIT 9301 (15746)
            </option><option value="13496">
                Prochlorococcus marinus str. MIT 9303 (13496)
            </option><option value="13910">
                Prochlorococcus marinus str. MIT 9312 (13910)
            </option><option value="220">
                Prochlorococcus marinus str. MIT 9313 (220)
            </option><option value="13617">
                Prochlorococcus marinus str. MIT 9515 (13617)
            </option><option value="15660">
                Prochlorococcus marinus str. NATL1A (15660)
            </option><option value="13911">
                Prochlorococcus marinus str. NATL2A (13911)
            </option><option value="procMari_CCMP1375">
                Prochlorococcus marinus subsp. marinus str. CCMP1375 (procMari_CCMP1375)
            </option><option value="213">
                Prochlorococcus marinus subsp. pastoris str. CCMP1986 (213)
            </option><option value="propAcne_KPA171202">
                Propionibacterium acnes KPA171202 (propAcne_KPA171202)
            </option><option value="12607">
                Prosthecochloris vibrioformis DSM 265 (12607)
            </option><option value="Prunus_persica_JGI_release_v1.0">
                Prunus persica JGI release v1.0
            </option><option value="Prunus_persica_JGI_v2.1">
                Prunus persica JGI v2.1
            </option><option value="equPrz1">
                Przewalski horse Jun 2014 (Burgud/equPrz1) (equPrz1)
            </option><option value="13454">
                Pseudoalteromonas atlantica T6c (13454)
            </option><option value="pseuHalo_TAC125">
                Pseudoalteromonas haloplanktis TAC125 (pseuHalo_TAC125)
            </option><option value="16720">
                Pseudomonas aeruginosa PA7 (16720)
            </option><option value="pseuAeru">
                Pseudomonas aeruginosa PAO1 (pseuAeru)
            </option><option value="386">
                Pseudomonas aeruginosa UCBPP-PA14 (386)
            </option><option value="16800">
                Pseudomonas entomophila L48 (16800)
            </option><option value="327">
                Pseudomonas fluorescens Pf-5 (327)
            </option><option value="12">
                Pseudomonas fluorescens PfO-1 (12)
            </option><option value="17457">
                Pseudomonas mendocina ymp (17457)
            </option><option value="13909">
                Pseudomonas putida F1 (13909)
            </option><option value="17629">
                Pseudomonas putida GB-1 (17629)
            </option><option value="267">
                Pseudomonas putida KT2440 (267)
            </option><option value="17053">
                Pseudomonas putida W619 (17053)
            </option><option value="16817">
                Pseudomonas stutzeri A1501 (16817)
            </option><option value="12416">
                Pseudomonas syringae pv. phaseolicola 1448A (12416)
            </option><option value="323">
                Pseudomonas syringae pv. syringae B728a (323)
            </option><option value="359">
                Pseudomonas syringae pv. tomato str. DC3000 (359)
            </option><option value="psycArct_273_4">
                Psychrobacter arcticus 273-4 (psycArct_273_4)
            </option><option value="13920">
                Psychrobacter cryohalolentis K5 (13920)
            </option><option value="15759">
                Psychrobacter sp. PRwf-1 (15759)
            </option><option value="psycIngr_37">
                Psychromonas ingrahamii 37 (psycIngr_37)
            </option><option value="punNye1">
                Pundamilia nyererei Oct 2011 (PunNye1.0/punNye1) (punNye1)
            </option><option value="Spur_v2.6">
                Purple Sea Urchin (Strongylocentrotus purpuratus) v2.6 (Spur_v2.6)
            </option><option value="pyrAer1">
                Pyrobaculum aerophilum str. IM2 (pyrAer1)
            </option><option value="pyroArse1">
                Pyrobaculum arsenaticum DSM 13514 (pyroArse1)
            </option><option value="pyroCali1">
                Pyrobaculum calidifontis JCM 11548 (pyroCali1)
            </option><option value="pyroIsla1">
                Pyrobaculum islandicum DSM 4184 (pyroIsla1)
            </option><option value="pyrAby1">
                Pyrococcus abyssi GE5 (pyrAby1)
            </option><option value="pyrFur2">
                Pyrococcus furiosus DSM 3638 (pyrFur2)
            </option><option value="pyrHor1">
                Pyrococcus horikoshii OT3 (pyrHor1)
            </option><option value="Pyrus_bretschneideri_Pear_Genome_Project_v1.0">
                Pyrus bretschneideri Pear Genome Project v1.0
            </option><option value="oryCun2">
                Rabbit Apr. 2009 (Broad/oryCun2) (oryCun2)
            </option><option value="oryCun1">
                Rabbit May 2005 (Broad/oryCun1) (oryCun1)
            </option><option value="13603">
                Ralstonia eutropha H16 (13603)
            </option><option value="ralsEutr_JMP134">
                Ralstonia eutropha JMP134 (ralsEutr_JMP134)
            </option><option value="250">
                Ralstonia metallidurans CH34 (250)
            </option><option value="ralsSola">
                Ralstonia solanacearum GMI1000 (ralsSola)
            </option><option value="rn6">
                Rat Jul. 2014 (RGSC 6.0/rn6) (rn6)
            </option><option value="rn3">
                Rat June 2003 (Baylor 3.1/rn3) (rn3)
            </option><option value="rn5">
                Rat Mar. 2012 (RGSC 5.0/rn5) (rn5)
            </option><option value="rn4">
                Rat Nov. 2004 (Baylor 3.4/rn4) (rn4)
            </option><option value="rn7">
                Rat Nov. 2020 (mRatBN7.2/rn7) (rn7)
            </option><option value="Tcas_3.0">
                Red Flour Beetle (Tribolium castaneum) (Tcas_3.0)
            </option><option value="tauEry1">
                Red crested turaco Jun 2014 (ASM70936v1/tauEry1) (tauEry1)
            </option><option value="carCri1">
                Red-legged seriema May 2014 (ASM69053v1/carCri1) (carCri1)
            </option><option value="gavSte1">
                Red-throated loon May 2014 (ASM69087v1/gavSte1) (gavSte1)
            </option><option value="19227">
                Renibacterium salmoninarum ATCC 33209 (19227)
            </option><option value="repBase0">
                RepBase Jun. 2012 (hg19/repBase0) (repBase0)
            </option><option value="repBase1">
                RepBase Sep. 2012 (17.08/repBase1) (repBase1)
            </option><option value="resEbo1">
                Reston ebolavirus Aug. 2014 (NC_004161.1/resEbo1) (resEbo1)
            </option><option value="rheMac10">
                Rhesus Feb. 2019 (Mmul_10/rheMac10) (rheMac10)
            </option><option value="rheMac2">
                Rhesus Jan. 2006 (MGSC Merged 1.0/rheMac2) (rheMac2)
            </option><option value="rheMac7">
                Rhesus Nov. 2014 (UMD MacaM_Assembly_v7/rheMac7) (rheMac7)
            </option><option value="rheMac8">
                Rhesus Nov. 2015 (BCM Mmul_8.0.1/rheMac8) (rheMac8)
            </option><option value="rheMac3">
                Rhesus Oct. 2010 (BGI CR_1.0/rheMac3) (rheMac3)
            </option><option value="bucRhi1">
                Rhinoceros hornbill Jun 2014 (ASM71030v1/bucRhi1) (bucRhi1)
            </option><option value="rhizEtli_CFN_42">
                Rhizobium etli CFN 42 (rhizEtli_CFN_42)
            </option><option value="344">
                Rhizobium leguminosarum bv. viciae 3841 (344)
            </option><option value="rhodSpha_2_4_1">
                Rhodobacter sphaeroides 2.4.1 (rhodSpha_2_4_1)
            </option><option value="15755">
                Rhodobacter sphaeroides ATCC 17025 (15755)
            </option><option value="15754">
                Rhodobacter sphaeroides ATCC 17029 (15754)
            </option><option value="rhodRHA1">
                Rhodococcus sp. RHA1 (rhodRHA1)
            </option><option value="13908">
                Rhodoferax ferrireducens T118 (13908)
            </option><option value="pireSp">
                Rhodopirellula baltica SH 1 (pireSp)
            </option><option value="15751">
                Rhodopseudomonas palustris BisA53 (15751)
            </option><option value="15750">
                Rhodopseudomonas palustris BisB18 (15750)
            </option><option value="15749">
                Rhodopseudomonas palustris BisB5 (15749)
            </option><option value="rhodPalu_CGA009">
                Rhodopseudomonas palustris CGA009 (rhodPalu_CGA009)
            </option><option value="15747">
                Rhodopseudomonas palustris HaA2 (15747)
            </option><option value="rhodRubr_ATCC11170">
                Rhodospirillum rubrum ATCC 11170 (rhodRubr_ATCC11170)
            </option><option value="oryza_sativa_japonica_nipponbare_IRGSP4.0">
                Rice (Oryza sativa ssp. japonica var. Nipponbare) (oryza_sativa_japonica_nipponbare_IRGSP4.0)
            </option><option value="Ricinus_communis_JCVI_1.0">
                Ricinus communis JCVI 1.0
            </option><option value="Ricinus_communis_JGI_v0.1">
                Ricinus communis JGI v0.1
            </option><option value="12953">
                Rickettsia akari str. Hartford (12953)
            </option><option value="17237">
                Rickettsia bellii OSU 85-389 (17237)
            </option><option value="rickBell_RML369_C">
                Rickettsia bellii RML369-C (rickBell_RML369_C)
            </option><option value="12952">
                Rickettsia canadensis str. McKiel (12952)
            </option><option value="42">
                Rickettsia conorii str. Malish 7 (42)
            </option><option value="13884">
                Rickettsia felis URRWXCal2 (13884)
            </option><option value="18271">
                Rickettsia massiliae MTU5 (18271)
            </option><option value="43">
                Rickettsia prowazekii str. Madrid E (43)
            </option><option value="9636">
                Rickettsia rickettsii str. 'Sheila Smith' (9636)
            </option><option value="19943">
                Rickettsia rickettsii str. Iowa (19943)
            </option><option value="10679">
                Rickettsia typhi str. Wilmington (10679)
            </option><option value="acaChl1">
                Rifleman 27 May 2014 (ASM69581v1/acaChl1) (acaChl1)
            </option><option value="proCap1">
                Rock hyrax Jul. 2008 (Broad/proCap1) (proCap1)
            </option><option value="colLiv2.1">
                Rock pigeon Dec 2017 (Cliv_2.1/colLiv2)
            </option><option value="colLiv1">
                Rock pigeon Feb 2013 (Cliv_1.0/colLiv1) (colLiv1)
            </option><option value="rodEnt13">
                Rodent Apr. 24. 2006 (UCSC Recon/rodEnt13) (rodEnt13)
            </option><option value="13462">
                Roseiflexus castenholzii DSM 13941 (13462)
            </option><option value="16190">
                Roseiflexus sp. RS-1 (16190)
            </option><option value="roseDeni_OCH_114">
                Roseobacter denitrificans OCh 114 (roseDeni_OCH_114)
            </option><option value="rubrXyla_DSM9941">
                Rubrobacter xylanophilus DSM 9941 (rubrXyla_DSM9941)
            </option><option value="Saccharomyces_cerevisiae_S288C_SGD2010">
                S. cerevisae str. S288C (Saccharomyces_cerevisiae_S288C_SGD2010)
            </option><option value="sacCer3">
                S. cerevisiae Apr. 2011 (SacCer_Apr2011/sacCer3) (sacCer3)
            </option><option value="sacCer2">
                S. cerevisiae June 2008 (SGD/sacCer2) (sacCer2)
            </option><option value="sacCer1">
                S. cerevisiae Oct. 2003 (SGD/sacCer1) (sacCer1)
            </option><option value="SL4.0">
                S. lycopersicum (Tomato) Feb 2019 (SL4.0)
            </option><option value="strPur1">
                S. purpuratus Apr. 2005 (Baylor 1.1/strPur1) (strPur1)
            </option><option value="strPur3">
                S. purpuratus Feb. 2009 (BCM-HGSC Spur_v2.5/strPur3) (strPur3)
            </option><option value="strPur4">
                S. purpuratus Jun. 2011 (BCM-HGSC Spur_v3.1/strPur4) (strPur4)
            </option><option value="strPur2">
                S. purpuratus Sep. 2006 (Baylor 2.1/strPur2) (strPur2)
            </option><option value="s.enterica_NC_016810.1">
                S.enterica_NC_016810.1
            </option><option value="sc1">
                SARS coronavirus Apr. 2003 (GenBank Apr. 14 '03/sc1) (sc1)
            </option><option value="wuhCor1">
                SARS-CoV-2 Jan. 2020 (NC_045512.2) (wuhCor1)
            </option><option value="NC_045512.2">
                SARS-CoV-2 isolate Wuhan-Hu-1, complete genome (NC_045512)
            </option><option value="saccDegr_2_40">
                Saccharophagus degradans 2-40 (saccDegr_2_40)
            </option><option value="saccEryt_NRRL_2338">
                Saccharopolyspora erythraea NRRL 2338 (saccEryt_NRRL_2338)
            </option><option value="Saccharum_spontaneum_v20190103">
                Saccharum spontaneum v20190103
            </option><option value="falChe1">
                Saker falcon Feb 2013 (F_cherrug_v1.0/falChe1) (falChe1)
            </option><option value="saliRube_DSM13855">
                Salinibacter ruber DSM 13855 (saliRube_DSM13855)
            </option><option value="17109">
                Salinispora arenicola CNS-205 (17109)
            </option><option value="saliTrop_CNB_440">
                Salinispora tropica CNB-440 (saliTrop_CNB_440)
            </option><option value="S. enterica (NC_016810.1)">
                Salmonella enterica (NC_016810.1)
            </option><option value="13030">
                Salmonella enterica subsp. arizonae serovar 62:z4,z23:-- (13030)
            </option><option value="9618">
                Salmonella enterica subsp. enterica serovar Choleraesuis str. SC-B67 (9618)
            </option><option value="salmEnte_PARATYPI_ATC">
                Salmonella enterica subsp. enterica serovar Paratyphi A str. ATCC 9150 (salmEnte_PARATYPI_ATC)
            </option><option value="27803">
                Salmonella enterica subsp. enterica serovar Paratyphi B str. SPB7 (27803)
            </option><option value="salmTyph">
                Salmonella enterica subsp. enterica serovar Typhi str. CT18 (salmTyph)
            </option><option value="salmTyph_TY2">
                Salmonella enterica subsp. enterica serovar Typhi str. Ty2 (salmTyph_TY2)
            </option><option value="241">
                Salmonella typhimurium LT2 (241)
            </option><option value="croPor0">
                Saltwater crocodile June 2011 (UCSC cPoro_v0.1c45/croPor0) (croPor0)
            </option><option value="araMac1">
                Scarlet Macaw Jun 2013 (SMACv1.1/araMac1) (araMac1)
            </option><option value="Schrenkiella_parvula_Thellungiella_v2.0_TpV84">
                Schrenkiella parvula Thellungiella v2.0 (TpV84)
            </option><option value="aplCal0">
                Sea hare Jul. 2013 (aplCal0/aplCal0) (aplCal0)
            </option><option value="aplCal1">
                Sea hare Sept. 2008 (Broad 2.0/aplCal1) (aplCal1)
            </option><option value="Selaginella_moellendorffii_JGI_v1.0">
                Selaginella moellendorffii JGI v1.0
            </option><option value="Selaginella_moellendorffii_NCBIr100">
                Selaginella moellendorffii NCBIr100
            </option><option value="17459">
                Serratia proteamaculans 568 (17459)
            </option><option value="Setaria_italica_JGI_8.3X_chromosome-scale_assembly_release_2.0_annotation_version_2.1">
                Setaria italica JGI 8.3X chromosome-scale assembly release 2.0, annotation version 2.1
            </option><option value="Setaria_italica_JGI_v2.2">
                Setaria italica JGI v2.2
            </option><option value="oviAri3">
                Sheep Aug. 2012 (ISGC Oar_v3.1/oviAri3) (oviAri3)
            </option><option value="oviAri1">
                Sheep Feb. 2010 (ISGC Ovis_aries_1.0/oviAri1) (oviAri1)
            </option><option value="oviAri4">
                Sheep Nov. 2015 (Oar_v4.0/oviAri4) (oviAri4)
            </option><option value="cypVar1">
                Sheepshead minnow Jul 2014 (C_variegatus-1.0/cypVar1) (cypVar1)
            </option><option value="shewAmaz">
                Shewanella amazonensis SB2B (shewAmaz)
            </option><option value="shewBalt">
                Shewanella baltica OS155 (shewBalt)
            </option><option value="17643">
                Shewanella baltica OS185 (17643)
            </option><option value="13389">
                Shewanella baltica OS195 (13389)
            </option><option value="shewDeni">
                Shewanella denitrificans OS217 (shewDeni)
            </option><option value="shewFrig">
                Shewanella frigidimarina NCIMB 400 (shewFrig)
            </option><option value="20241">
                Shewanella halifaxensis HAW-EB4 (20241)
            </option><option value="shewLoihPV4">
                Shewanella loihica PV-4 (shewLoihPV4)
            </option><option value="shewOnei">
                Shewanella oneidensis MR-1 (shewOnei)
            </option><option value="17415">
                Shewanella pealeana ATCC 700345 (17415)
            </option><option value="shewPutrCN32">
                Shewanella putrefaciens CN-32 (shewPutrCN32)
            </option><option value="18789">
                Shewanella sediminis HAW-EB3 (18789)
            </option><option value="shewANA3">
                Shewanella sp. ANA-3 (shewANA3)
            </option><option value="shewMR4">
                Shewanella sp. MR-4 (shewMR4)
            </option><option value="shewMR7">
                Shewanella sp. MR-7 (shewMR7)
            </option><option value="shewW318">
                Shewanella sp. W3-18-1 (shewW318)
            </option><option value="17455">
                Shewanella woodyi ATCC 51908 (17455)
            </option><option value="13146">
                Shigella boydii Sb227 (13146)
            </option><option value="13145">
                Shigella dysenteriae Sd197 (13145)
            </option><option value="408">
                Shigella flexneri 2a str. 2457T (408)
            </option><option value="shigFlex_2A">
                Shigella flexneri 2a str. 301 (shigFlex_2A)
            </option><option value="16375">
                Shigella flexneri 5 str. 8401 (16375)
            </option><option value="13151">
                Shigella sonnei Ss046 (13151)
            </option><option value="sorAra2">
                Shrew Aug. 2008 (Broad/sorAra2) (sorAra2)
            </option><option value="sorAra1">
                Shrew June 2006 (Broad/sorAra1) (sorAra1)
            </option><option value="siliPome_DSS_3">
                Silicibacter pomeroyi DSS-3 (siliPome_DSS_3)
            </option><option value="13040">
                Silicibacter sp. TM1040 (13040)
            </option><option value="Bombyx_mori_p50T_2.0">
                Silkworm (Bombyx_mori_p50T_2.0)
            </option><option value="simHuman">
                Simhuman Oct. 2011 (simHuman/simHuman) (simHuman)
            </option><option value="simHumanMammal">
                Simhumanmammal Oct. 2011 (simHumanMammal/simHumanMammal) (simHumanMammal)
            </option><option value="simHumanMam">
                Simmam Jan. 2011 (SimMamV1/simHumanMam) (simHumanMam)
            </option><option value="simHumanPrim">
                Simprim Jan. 2011 (SimPrimV1/simHumanPrim) (simHumanPrim)
            </option><option value="simRefMam">
                Simrefmam Jan. 2011 (SimRefMamV1/simRefMam) (simRefMam)
            </option><option value="16304">
                Sinorhizobium medicae WSM419 (16304)
            </option><option value="sinoMeli">
                Sinorhizobium meliloti 1021 (sinoMeli)
            </option><option value="choHof1">
                Sloth Jul. 2008 (Broad/choHof1) (choHof1)
            </option><option value="bioGla0">
                Snail Jun. 2013 (bioGla0/bioGla0) (bioGla0)
            </option><option value="sodaGlos_MORSITANS">
                Sodalis glossinidius str. 'morsitans' (sodaGlos_MORSITANS)
            </option><option value="Solanum_lycopersicum_ITAG_2.3">
                Solanum lycopersicum ITAG 2.3
            </option><option value="Solanum_lycopersicum_Sol_Genomics_itag2.4">
                Solanum lycopersicum Sol Genomics itag2.4
            </option><option value="Solanum_tuberosum_ITAG_001">
                Solanum tuberosum ITAG 001
            </option><option value="Solanum_tuberosum_JGI_v4.03">
                Solanum tuberosum JGI v4.03
            </option><option value="soliUsit_ELLIN6076">
                Solibacter usitatus Ellin6076 (soliUsit_ELLIN6076)
            </option><option value="28111">
                Sorangium cellulosum 'So ce 56' (28111)
            </option><option value="Sorghum_bicolor_JGI_1.4">
                Sorghum bicolor JGI 1.4
            </option><option value="Sorghum_bicolor_JGI_v3.1">
                Sorghum bicolor JGI v3.1
            </option><option value="xipMac1">
                Southern platyfish Jan 2012 (Xiphophorus_maculatus-4.4.2/xipMac1) (xipMac1)
            </option><option value="enhLutNer1">
                Southern sea otter Jun. 2019 (ASM641071v1/enhLutNer1) (enhLutNer1)
            </option><option value="colStr1">
                Speckled mousebird May 2014 (ASM69071v1/colStr1) (colStr1)
            </option><option value="phyCat1">
                Sperm whale Sep. 2013 (Physeter_macrocephalus-2.0.2/phyCat1) (phyCat1)
            </option><option value="17343">
                Sphingomonas wittichii RW1 (17343)
            </option><option value="sphiAlas_RB2256">
                Sphingopyxis alaskensis RB2256 (sphiAlas_RB2256)
            </option><option value="apaSpi1">
                Spiny softshell turtle May 2013 (ASM38561v1/apaSpi1) (apaSpi1)
            </option><option value="Spirodela_polyrhiza_JGI_v2.0">
                Spirodela polyrhiza JGI v2.0
            </option><option value="lepOcu1">
                Spotted gar Dec 2011 (LepOcu1/lepOcu1) (lepOcu1)
            </option><option value="speTri1">
                Squirrel Feb. 2008 (Broad/speTri1) (speTri1)
            </option><option value="speTri2">
                Squirrel Nov. 2011 (Broad/speTri2) (speTri2)
            </option><option value="saiBol1">
                Squirrel monkey Oct. 2011 (Broad/saiBol1) (saiBol1)
            </option><option value="63">
                Staphylococcus aureus RF122 (63)
            </option><option value="238">
                Staphylococcus aureus subsp. aureus COL (238)
            </option><option value="15758">
                Staphylococcus aureus subsp. aureus JH1 (15758)
            </option><option value="15757">
                Staphylococcus aureus subsp. aureus JH9 (15757)
            </option><option value="265">
                Staphylococcus aureus subsp. aureus MRSA252 (265)
            </option><option value="266">
                Staphylococcus aureus subsp. aureus MSSA476 (266)
            </option><option value="306">
                Staphylococcus aureus subsp. aureus MW2 (306)
            </option><option value="18509">
                Staphylococcus aureus subsp. aureus Mu3 (18509)
            </option><option value="stapAure_MU50">
                Staphylococcus aureus subsp. aureus Mu50 (stapAure_MU50)
            </option><option value="264">
                Staphylococcus aureus subsp. aureus N315 (264)
            </option><option value="237">
                Staphylococcus aureus subsp. aureus NCTC 8325 (237)
            </option><option value="16313">
                Staphylococcus aureus subsp. aureus USA300 (16313)
            </option><option value="19489">
                Staphylococcus aureus subsp. aureus USA300_TCH1516 (19489)
            </option><option value="18801">
                Staphylococcus aureus subsp. aureus str. Newman (18801)
            </option><option value="279">
                Staphylococcus epidermidis ATCC 12228 (279)
            </option><option value="64">
                Staphylococcus epidermidis RP62A (64)
            </option><option value="12508">
                Staphylococcus haemolyticus JCSC1435 (12508)
            </option><option value="15596">
                Staphylococcus saprophyticus subsp. saprophyticus ATCC 15305 (15596)
            </option><option value="stapMari1">
                Staphylothermus marinus F1 (stapMari1)
            </option><option value="conCri1">
                Star-nosed mole Mar 2012 (ConCri1.0/conCri1) (conCri1)
            </option><option value="nemVec1">
                Starlet sea anemone Jun. 2007 (JGI-PDF/nemVec1) (nemVec1)
            </option><option value="gasAcu1">
                Stickleback Feb. 2006 (Broad/gasAcu1) (gasAcu1)
            </option><option value="eidHel1">
                Straw colored fruit bat 13 Sep 2013 (ASM46528v1/eidHel1) (eidHel1)
            </option><option value="330">
                Streptococcus agalactiae 2603V/R (330)
            </option><option value="326">
                Streptococcus agalactiae A909 (326)
            </option><option value="334">
                Streptococcus agalactiae NEM316 (334)
            </option><option value="66">
                Streptococcus gordonii str. Challis substr. CH1 (66)
            </option><option value="333">
                Streptococcus mutans UA159 (333)
            </option><option value="16374">
                Streptococcus pneumoniae D39 (16374)
            </option><option value="28035">
                Streptococcus pneumoniae Hungary19A-6 (28035)
            </option><option value="278">
                Streptococcus pneumoniae R6 (278)
            </option><option value="277">
                Streptococcus pneumoniae TIGR4 (277)
            </option><option value="strePyog_M1_GAS">
                Streptococcus pyogenes M1 GAS (strePyog_M1_GAS)
            </option><option value="16364">
                Streptococcus pyogenes MGAS10270 (16364)
            </option><option value="12469">
                Streptococcus pyogenes MGAS10394 (12469)
            </option><option value="16366">
                Streptococcus pyogenes MGAS10750 (16366)
            </option><option value="16365">
                Streptococcus pyogenes MGAS2096 (16365)
            </option><option value="311">
                Streptococcus pyogenes MGAS315 (311)
            </option><option value="13888">
                Streptococcus pyogenes MGAS5005 (13888)
            </option><option value="13887">
                Streptococcus pyogenes MGAS6180 (13887)
            </option><option value="286">
                Streptococcus pyogenes MGAS8232 (286)
            </option><option value="16363">
                Streptococcus pyogenes MGAS9429 (16363)
            </option><option value="301">
                Streptococcus pyogenes SSI-1 (301)
            </option><option value="270">
                Streptococcus pyogenes str. Manfredo (270)
            </option><option value="13942">
                Streptococcus sanguinis SK36 (13942)
            </option><option value="17153">
                Streptococcus suis 05ZYH33 (17153)
            </option><option value="17155">
                Streptococcus suis 98HAH33 (17155)
            </option><option value="13163">
                Streptococcus thermophilus CNRZ1066 (13163)
            </option><option value="13773">
                Streptococcus thermophilus LMD-9 (13773)
            </option><option value="13162">
                Streptococcus thermophilus LMG 18311 (13162)
            </option><option value="189">
                Streptomyces avermitilis MA-4680 (189)
            </option><option value="streCoel">
                Streptomyces coelicolor A3(2) (streCoel)
            </option><option value="sudEbo1">
                Sudan ebolavirus Aug. 2014 (NC_006432.1/sudEbo1) (sudEbo1)
            </option><option value="sulfAcid1">
                Sulfolobus acidocaldarius DSM 639 (sulfAcid1)
            </option><option value="sulSol1">
                Sulfolobus solfataricus P2 (sulSol1)
            </option><option value="sulfToko1">
                Sulfolobus tokodaii str. 7 (sulfToko1)
            </option><option value="thioDeni_ATCC33889">
                Sulfurimonas denitrificans DSM 1251 (thioDeni_ATCC33889)
            </option><option value="18965">
                Sulfurovum sp. NBC37-1 (18965)
            </option><option value="eurHel1">
                Sunbittern May 2014 (ASM69077v1/eurHel1) (eurHel1)
            </option><option value="symbTher_IAM14863">
                Symbiobacterium thermophilum IAM 14863 (symbTher_IAM14863)
            </option><option value="13282">
                Synechococcus elongatus PCC 6301 (13282)
            </option><option value="10645">
                Synechococcus elongatus PCC 7942 (10645)
            </option><option value="12530">
                Synechococcus sp. CC9311 (12530)
            </option><option value="13643">
                Synechococcus sp. CC9605 (13643)
            </option><option value="13655">
                Synechococcus sp. CC9902 (13655)
            </option><option value="16252">
                Synechococcus sp. JA-2-3B'a(2-13) (16252)
            </option><option value="16251">
                Synechococcus sp. JA-3-3Ab (16251)
            </option><option value="28247">
                Synechococcus sp. PCC 7002 (28247)
            </option><option value="13654">
                Synechococcus sp. RCC307 (13654)
            </option><option value="13642">
                Synechococcus sp. WH 7803 (13642)
            </option><option value="syneSp_WH8102">
                Synechococcus sp. WH 8102 (syneSp_WH8102)
            </option><option value="synePCC6">
                Synechocystis sp. PCC 6803 (synePCC6)
            </option><option value="syntFuma_MPOB">
                Syntrophobacter fumaroxidans MPOB (syntFuma_MPOB)
            </option><option value="syntWolf_GOETTINGEN">
                Syntrophomonas wolfei subsp. wolfei str. Goettingen (syntWolf_GOETTINGEN)
            </option><option value="syntAcid_SB">
                Syntrophus aciditrophicus SB (syntAcid_SB)
            </option><option value="triCas2">
                T. castaneum Sep. 2005 (Baylor 2.0/triCas2) (triCas2)
            </option><option value="taiEbo1">
                Tai forest ebolavirus Aug. 2014 (NC_014372.1/taiEbo1) (taiEbo1)
            </option><option value="Tarenaya_hassleriana_RefSeq_v1">
                Tarenaya hassleriana RefSeq v1
            </option><option value="tarSyr1">
                Tarsier Aug. 2008 (Broad/tarSyr1) (tarSyr1)
            </option><option value="tarSyr2">
                Tarsier Sep. 2013 (Tarsius_syrichta-2.0.1/tarSyr2) (tarSyr2)
            </option><option value="sarHar1">
                Tasmanian devil Feb. 2011 (WTSI Devil_ref v7.0/sarHar1) (sarHar1)
            </option><option value="echTel1">
                Tenrec July 2005 (Broad/echTel1) (echTel1)
            </option><option value="echTel2">
                Tenrec Nov. 2012 (Broad/echTel2) (echTel2)
            </option><option value="tetNig1">
                Tetraodon Feb. 2004 (Genoscope 7/tetNig1) (tetNig1)
            </option><option value="tetNig2">
                Tetraodon Mar. 2007 (Genoscope 8.0/tetNig2) (tetNig2)
            </option><option value="Thellungiella_parvula_TpV84">
                Thellungiella parvula TpV84
            </option><option value="Theobroma_cacao_D._Gilbert_public_gene_set_8_Mar_2012_on_assembly_v1.1">
                Theobroma cacao D. Gilbert public gene set 8 Mar 2012 on assembly v1.1
            </option><option value="Tcacao_1.0">
                Theobroma cacao Dec 2010 (CIRAD/Theobroma cacao Belizian Criollo) (Tcacao_1.0)
            </option><option value="Theobroma_cacao_RefSeq_v1.1">
                Theobroma cacao RefSeq v1.1
            </option><option value="13901">
                Thermoanaerobacter pseudethanolicus ATCC 33223 (13901)
            </option><option value="16394">
                Thermoanaerobacter sp. X514 (16394)
            </option><option value="therTeng">
                Thermoanaerobacter tengcongensis MB4 (therTeng)
            </option><option value="therFusc_YX">
                Thermobifida fusca YX (therFusc_YX)
            </option><option value="therKoda1">
                Thermococcus kodakarensis KOD1 (therKoda1)
            </option><option value="therPend1">
                Thermofilum pendens Hrk 5 (therPend1)
            </option><option value="therAcid1">
                Thermoplasma acidophilum DSM 1728 (therAcid1)
            </option><option value="therVolc1">
                Thermoplasma volcanium GSS1 (therVolc1)
            </option><option value="17249">
                Thermosipho melanesiensis BI429 (17249)
            </option><option value="therElon">
                Thermosynechococcus elongatus BP-1 (therElon)
            </option><option value="15644">
                Thermotoga lettingae TMO (15644)
            </option><option value="therMari">
                Thermotoga maritima MSB8 (therMari)
            </option><option value="therPetr_RKU_1">
                Thermotoga petrophila RKU-1 (therPetr_RKU_1)
            </option><option value="19543">
                Thermotoga sp. RQ2 (19543)
            </option><option value="therTher_HB27">
                Thermus thermophilus HB27 (therTher_HB27)
            </option><option value="therTher_HB8">
                Thermus thermophilus HB8 (therTher_HB8)
            </option><option value="thioDeni_ATCC25259">
                Thiobacillus denitrificans ATCC 25259 (thioDeni_ATCC25259)
            </option><option value="thioCrun_XCL_2">
                Thiomicrospira crunogena XCL-2 (thioCrun_XCL_2)
            </option><option value="panHod1">
                Tibetan antelope May 2013 (PHO1.0/panHod1) (panHod1)
            </option><option value="nanPar1">
                Tibetan frog Mar. 2015 (BGI_ZX_2015/nanPar1) (nanPar1)
            </option><option value="pseHum1">
                Tibetan ground jay Jan 2013 (PseHum1.0/pseHum1) (pseHum1)
            </option><option value="sebNig1">
                Tiger rockfish 22 Oct-2013 (Snig1.0/sebNig1) (sebNig1)
            </option><option value="cynSem1">
                Tongue sole 28 Jan-2014 (Cse_v1.0/cynSem1) (cynSem1)
            </option><option value="tupBel1">
                Tree shrew Dec. 2006 (Broad/tupBel1) (tupBel1)
            </option><option value="4">
                Treponema denticola ATCC 35405 (4)
            </option><option value="trepPall">
                Treponema pallidum subsp. pallidum str. Nichols (trepPall)
            </option><option value="tricEryt_IMS101">
                Trichodesmium erythraeum IMS101 (tricEryt_IMS101)
            </option><option value="Trifolium_pratense_JGI_v2">
                Trifolium pratense JGI v2
            </option><option value="Triticum_aestivum_Ensembl_Plants_TGACv1">
                Triticum aestivum Ensembl Plants TGACv1
            </option><option value="Triticum_aestivum_IWGSC1.1">
                Triticum aestivum IWGSC1.1
            </option><option value="Triticum_turgidum_Svevo_v1.0">
                Triticum turgidum Svevo v1.0
            </option><option value="tropWhip_TW08_27">
                Tropheryma whipplei TW08/27 (tropWhip_TW08_27)
            </option><option value="95">
                Tropheryma whipplei str. Twist (95)
            </option><option value="TriTrypDB-37_TbruceiLister427">
                Trypanosoma brucei 427 Lister strain genome build 37
            </option><option value="melGal1">
                Turkey Dec. 2009 (TGC Turkey_2.01/melGal1) (melGal1)
            </option><option value="melGal5">
                Turkey Nov. 2014 (Turkey_5.0/melGal5) (melGal5)
            </option><option value="nanGal1">
                Upper Galilee mountains blind mole rat Jun 2014 (S.galili_v1.0/nanGal1) (nanGal1)
            </option><option value="19087">
                Ureaplasma parvum serovar 3 str. ATCC 27815 (19087)
            </option><option value="ureaUrea">
                Ureaplasma parvum serovar 3 str. ATCC 700970 (ureaUrea)
            </option><option value="Utricularia_gibba_CoGe_v4">
                Utricularia gibba CoGe v4
            </option><option value="vermEise_EF01_2">
                Verminephrobacter eiseniae EF01-2 (vermEise_EF01_2)
            </option><option value="vibrChol1">
                Vibrio cholerae O1 biovar eltor str. N16961 (vibrChol1)
            </option><option value="vibrChol_O395_1">
                Vibrio cholerae O395 (vibrChol_O395_1)
            </option><option value="vibrFisc_ES114_1">
                Vibrio fischeri ES114 (vibrFisc_ES114_1)
            </option><option value="19857">
                Vibrio harveyi ATCC BAA-1116 (19857)
            </option><option value="vibrPara1">
                Vibrio parahaemolyticus RIMD 2210633 (vibrPara1)
            </option><option value="vibrVuln_CMCP6_1">
                Vibrio vulnificus CMCP6 (vibrVuln_CMCP6_1)
            </option><option value="vibrVuln_YJ016_1">
                Vibrio vulnificus YJ016 (vibrVuln_YJ016_1)
            </option><option value="Vigna_radiata_var._radiata_LIS_v6">
                Vigna radiata var. radiata LIS v6
            </option><option value="Vitis_vinifera_JGI_12xMarch2010">
                Vitis vinifera JGI 12xMarch2010
            </option><option value="Vitis_vinifera_genescope_v1">
                Vitis vinifera genescope v1
            </option><option value="macEug1">
                Wallaby Nov. 2007 (Baylor 1.0/macEug1) (macEug1)
            </option><option value="macEug2">
                Wallaby Sep. 2009 (TWGS Meug_1.1/macEug2) (macEug2)
            </option><option value="bubBub1">
                Water buffalo Sep. 2013 (UMD_CASPUR_WB_2.0/bubBub1) (bubBub1)
            </option><option value="lepWed1">
                Weddell seal Mar 2013 (LepWed1.0/lepWed1) (lepWed1)
            </option><option value="GCF_019279795.1">
                West African lungfish Jul 2021 (Protopterus annectens, PAN1.0) (GCF_019279795.1)
            </option><option value="cerSim1">
                White rhinoceros May 2012 (CerSimSim1.0/cerSim1) (cerSim1)
            </option><option value="tinGut2">
                White throated tinamou Aug 2014 (ASM70537v2/tinGut2) (tinGut2)
            </option><option value="tinGut1">
                White throated tinamou Jun 2014 (ASM70537v1/tinGut1) (tinGut1)
            </option><option value="GCF_002102435.1">
                White-tailed deer Apr 2017 (Odocoileus virginianus) (GCF_002102435.1, Ovir.te_1.0)
            </option><option value="halAlb1">
                White-tailed eagle May 2014 (ASM69140v1/halAlb1) (halAlb1)
            </option><option value="phaLep1">
                White-tailed tropicbird May 2014 (ASM68728v1/phaLep1) (phaLep1)
            </option><option value="zonAlb1">
                White-throated sparrow Apr 2013 (ASM38545v1/zonAlb1) (zonAlb1)
            </option><option value="wiggBrev">
                Wigglesworthia glossinidia endosymbiont of Glossina brevipalpis (wiggBrev)
            </option><option value="bosMut1">
                Wild yak Oct 2011 (BosGru_v2.0/bosMut1) (bosMut1)
            </option><option value="wolbEndo_OF_DROSOPHIL">
                Wolbachia endosymbiont of Drosophila melanogaster (wolbEndo_OF_DROSOPHIL)
            </option><option value="12475">
                Wolbachia endosymbiont strain TRS of Brugia malayi (12475)
            </option><option value="woliSucc">
                Wolinella succinogenes DSM 1740 (woliSucc)
            </option><option value="xenTro2">
                X. tropicalis Aug. 2005 (JGI 4.1/xenTro2) (xenTro2)
            </option><option value="xenTro9">
                X. tropicalis Jul. 2016 (Xenopus_tropicalis_v9.1/xenTro9) (xenTro9)
            </option><option value="xenTro3">
                X. tropicalis Nov. 2009 (JGI 4.2/xenTro3) (xenTro3)
            </option><option value="xenTro10">
                X. tropicalis Nov. 2019 (UCB_Xtro_10.0/xenTro10) (xenTro10)
            </option><option value="xenTro1">
                X. tropicalis Oct. 2004 (JGI 3.0/xenTro1) (xenTro1)
            </option><option value="xenTro7">
                X. tropicalis Sep. 2012 (JGI 7.0/xenTro7) (xenTro7)
            </option><option value="15756">
                Xanthobacter autotrophicus Py2 (15756)
            </option><option value="297">
                Xanthomonas axonopodis pv. citri str. 306 (297)
            </option><option value="15">
                Xanthomonas campestris pv. campestris str. 8004 (15)
            </option><option value="xantCamp">
                Xanthomonas campestris pv. campestris str. ATCC 33913 (xantCamp)
            </option><option value="13649">
                Xanthomonas campestris pv. vesicatoria str. 85-10 (13649)
            </option><option value="12931">
                Xanthomonas oryzae pv. oryzae KACC10331 (12931)
            </option><option value="16297">
                Xanthomonas oryzae pv. oryzae MAFF 311018 (16297)
            </option><option value="Xanthomonas_oryzae_PXO99A">
                Xanthomonas oryzae pv. oryzae PXO99A (Xanthomonas_oryzae_PXO99A)
            </option><option value="xenTro9.1">
                Xenopus tropicalis (xenTro9.1)
            </option><option value="xyleFast">
                Xylella fastidiosa 9a5c (xyleFast)
            </option><option value="17823">
                Xylella fastidiosa M12 (17823)
            </option><option value="285">
                Xylella fastidiosa Temecula1 (285)
            </option><option value="lipVex1">
                Yangtze river dolphin 31 Jul 2013 (Lipotes_vexillifer_v1/lipVex1) (lipVex1)
            </option><option value="pteGut1">
                Yellow-throated sandgrouse Jun 2014 (ASM69924v1/pteGut1) (pteGut1)
            </option><option value="takFla1">
                Yellowbelly pufferfish May 2013 (version 1 of Takifugu flavidus genome/takFla1) (takFla1)
            </option><option value="190">
                Yersinia enterocolitica subsp. enterocolitica 8081 (190)
            </option><option value="16067">
                Yersinia pestis Angola (16067)
            </option><option value="16645">
                Yersinia pestis Antiqua (16645)
            </option><option value="yersPest_CO92">
                Yersinia pestis CO92 (yersPest_CO92)
            </option><option value="288">
                Yersinia pestis KIM (288)
            </option><option value="16646">
                Yersinia pestis Nepal516 (16646)
            </option><option value="16700">
                Yersinia pestis Pestoides F (16700)
            </option><option value="10638">
                Yersinia pestis biovar Microtus str. 91001 (10638)
            </option><option value="16070">
                Yersinia pseudotuberculosis IP 31758 (16070)
            </option><option value="12950">
                Yersinia pseudotuberculosis IP 32953 (12950)
            </option><option value="28743">
                Yersinia pseudotuberculosis YPIII (28743)
            </option><option value="zaiEbo1">
                Zaire ebolavirus Aug. 2014 (NC_002549.1/zaiEbo1) (zaiEbo1)
            </option><option value="Zea_mays_B73_AGP_v4.0">
                Zea mays B73 AGP v4.0
            </option><option value="Zea_mays_PH207_v1.0">
                Zea mays PH207 v1.0
            </option><option value="mayZeb1">
                Zebra Mbuna Mar 2012 (MetZeb1.1/mayZeb1) (mayZeb1)
            </option><option value="taeGut2">
                Zebra finch Feb. 2013 (WashU taeGut324/taeGut2) (taeGut2)
            </option><option value="taeGut1">
                Zebra finch Jul. 2008 (WUGSC 3.2.4/taeGut1) (taeGut1)
            </option><option value="danRer11_wo_alt">
                Zebrafish (GRCz11/danRer11) without alt contigs
            </option><option value="danRer6">
                Zebrafish Dec. 2008 (Zv8/danRer6) (danRer6)
            </option><option value="danRer7">
                Zebrafish Jul. 2010 (Zv9/danRer7) (danRer7)
            </option><option value="danRer5">
                Zebrafish July 2007 (Zv7/danRer5) (danRer5)
            </option><option value="danRer4">
                Zebrafish Mar. 2006 (Zv6/danRer4) (danRer4)
            </option><option value="danRer3">
                Zebrafish May 2005 (Zv5/danRer3) (danRer3)
            </option><option value="danRer11">
                Zebrafish May 2017 (GRCz11/danRer11) (danRer11)
            </option><option value="danRer10">
                Zebrafish Sep. 2014 (GRCz10/danRer10) (danRer10)
            </option><option value="Zostera_marina_JGI_v2.2">
                Zostera marina JGI v2.2
            </option><option value="Zoysia_japonica_ssp._nagirizaki_Zoysia_Genome_Database_v1.1">
                Zoysia japonica ssp. nagirizaki Zoysia Genome Database v1.1
            </option><option value="zymoMobi_ZM4">
                Zymomonas mobilis subsp. mobilis ZM4 (zymoMobi_ZM4)
            </option><option value="canFam3">
                canFam3
            </option><option value="homPan20">
                chimp/human Jun. 2008 (UCSC Recon/homPan20) (homPan20)
            </option><option value="hg19Haps">
                hg19Haplotypes Feb. 2009 (GRCh37/hg19Haps) (hg19Haps)
            </option><option value="mm9_ERCC">
                mm9 with ERCC spike-in
            </option><option value="homIni20">
                orang/human Jun. 2008 (UCSC Recon/homIni20) (homIni20)
            </option><option value="pUC18">
                pUC18 Plasmid (pUC18)
            </option><option value="phiX">
                phiX174 (phiX)
            </option><option value="tarIhg38">
                tarInv Dec. 2013 (GRCh38Tar/tarIhg38) (tarIhg38)
            </option><option value="uncuMeth_RCI">
                uncultured methanogenic archaeon RC-I (uncuMeth_RCI)
            </option></select></div> <div class="upload-buttons"><button title="Close" type="button" class="btn ui-button-default btn-secondary">
            Close
        </button> <button id="btn-start" title="Start" type="button" disabled="disabled" class="btn ui-button-default btn-secondary disabled">
            Start
        </button> <button id="btn-reset" title="Reset" type="button" class="btn ui-button-default btn-secondary">
            Reset
        </button></div></div></div> <div role="tabpanel" id="collection" aria-hidden="true" aria-labelledby="tab-title-link-collection" class="tab-pane" style="display: none;"><div class="upload-view-default"><div class="upload-top"><div class="upload-top-info">&nbsp;</div></div> <div class="upload-box" style="height: 300px;"><div class="upload-helper"><i class="fa fa-files-o"></i>Drop files here</div> <table class="upload-table ui-table-striped" style="display: none;"><thead><tr><th>Size</th> <th>Status</th> <th></th></tr></thead> <tbody></tbody></table><input type="file" style="display: none" multiple=""></div> <div class="upload-footer"><span class="upload-footer-title">Collection Type:</span> <div class="select2-container upload-footer-collection-type" id="s2id_autogen25"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-26">List</span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen26" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-26" id="s2id_autogen26"><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen26_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-26" id="s2id_autogen26_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-26">   </ul></div></div><select tabindex="-1" title="" class="select2-offscreen"><option value="list">List</option> <option value="paired">Pair</option> <option value="list:paired">List of Pairs</option></select> <span class="upload-footer-title">File Type:</span> <div class="select2-container upload-footer-extension" id="s2id_autogen27"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-28">Auto-detect</span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen28" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-28" id="s2id_autogen28"><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen28_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-28" id="s2id_autogen28_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-28">   </ul></div></div><select tabindex="-1" title="" class="select2-offscreen"><option value="auto">Auto-detect</option><option value="ab1">ab1</option><option value="agilentbrukeryep.d.tar">agilentbrukeryep.d.tar</option><option value="agilentmasshunter.d.tar">agilentmasshunter.d.tar</option><option value="agp">agp</option><option value="ampvis2">ampvis2</option><option value="arb">arb</option><option value="arb.gz">arb.gz</option><option value="arff">arff</option><option value="asn1">asn1</option><option value="asn1-binary">asn1-binary</option><option value="augustus">augustus</option><option value="avi">avi</option><option value="axt">axt</option><option value="bam">bam</option><option value="bcf">bcf</option><option value="bcf_uncompressed">bcf_uncompressed</option><option value="bcsl.model">bcsl.model</option><option value="bcsl.ts">bcsl.ts</option><option value="bed">bed</option><option value="bed12">bed12</option><option value="bed6">bed6</option><option value="bed_tabix.gz">bed_tabix.gz</option><option value="bedgraph">bedgraph</option><option value="bedstrict">bedstrict</option><option value="bgzip">bgzip</option><option value="bigbed">bigbed</option><option value="bigwig">bigwig</option><option value="bil">bil</option><option value="biom1">biom1</option><option value="biom2">biom2</option><option value="blastxml">blastxml</option><option value="blib">blib</option><option value="bref3">bref3</option><option value="brukerbaf.d.tar">brukerbaf.d.tar</option><option value="brukertdf.d.tar">brukertdf.d.tar</option><option value="bus">bus</option><option value="castep">castep</option><option value="cel">cel</option><option value="cell">cell</option><option value="chira.sqlite">chira.sqlite</option><option value="chrint">chrint</option><option value="cif">cif</option><option value="cisml">cisml</option><option value="ckpt">ckpt</option><option value="cmap">cmap</option><option value="cml">cml</option><option value="consensusxml">consensusxml</option><option value="cool">cool</option><option value="cps">cps</option><option value="cpt">cpt</option><option value="cram">cram</option><option value="csfasta">csfasta</option><option value="csv">csv</option><option value="ct">ct</option><option value="ctl.result">ctl.result</option><option value="cuffdiff.sqlite">cuffdiff.sqlite</option><option value="cxb">cxb</option><option value="d3_hierarchy">d3_hierarchy</option><option value="daa">daa</option><option value="dada2_dada">dada2_dada</option><option value="dada2_errorrates">dada2_errorrates</option><option value="dada2_mergepairs">dada2_mergepairs</option><option value="dada2_sequencetable">dada2_sequencetable</option><option value="dada2_uniques">dada2_uniques</option><option value="dbn">dbn</option><option value="dbnsfp.tabular">dbnsfp.tabular</option><option value="dcd">dcd</option><option value="deeptools_compute_matrix_archive">deeptools_compute_matrix_archive</option><option value="deeptools_coverage_matrix">deeptools_coverage_matrix</option><option value="den_fmt">den_fmt</option><option value="dlib">dlib</option><option value="drf">drf</option><option value="dta">dta</option><option value="dta2d">dta2d</option><option value="dzi">dzi</option><option value="edr">edr</option><option value="edta">edta</option><option value="eland">eland</option><option value="elandmulti">elandmulti</option><option value="elib">elib</option><option value="embl">embl</option><option value="encodepeak">encodepeak</option><option value="excel.xls">excel.xls</option><option value="expression.json">expression.json</option><option value="extxyz">extxyz</option><option value="fai">fai</option><option value="fast5">fast5</option><option value="fast5.tar">fast5.tar</option><option value="fast5.tar.bz2">fast5.tar.bz2</option><option value="fast5.tar.gz">fast5.tar.gz</option><option value="fasta">fasta</option><option value="fasta.gz">fasta.gz</option><option value="fastg">fastg</option><option value="fastq">fastq</option><option value="fastq.bz2">fastq.bz2</option><option value="fastq.gz">fastq.gz</option><option value="fastqcssanger">fastqcssanger</option><option value="fastqcssanger.bz2">fastqcssanger.bz2</option><option value="fastqcssanger.gz">fastqcssanger.gz</option><option value="fastqillumina">fastqillumina</option><option value="fastqillumina.bz2">fastqillumina.bz2</option><option value="fastqillumina.gz">fastqillumina.gz</option><option value="fastqsanger">fastqsanger</option><option value="fastqsanger.bz2">fastqsanger.bz2</option><option value="fastqsanger.gz">fastqsanger.gz</option><option value="fastqsolexa">fastqsolexa</option><option value="fastqsolexa.bz2">fastqsolexa.bz2</option><option value="fastqsolexa.gz">fastqsolexa.gz</option><option value="fcs">fcs</option><option value="featurexml">featurexml</option><option value="ffdata">ffdata</option><option value="ffindex">ffindex</option><option value="fits">fits</option><option value="flowclr">flowclr</option><option value="flowframe">flowframe</option><option value="flowmfi">flowmfi</option><option value="flowscore">flowscore</option><option value="flowset">flowset</option><option value="flowstat1">flowstat1</option><option value="flowstat2">flowstat2</option><option value="flowstat3">flowstat3</option><option value="flowtext">flowtext</option><option value="flv">flv</option><option value="fps">fps</option><option value="fqtoc">fqtoc</option><option value="fsom">fsom</option><option value="gafa.sqlite">gafa.sqlite</option><option value="gal">gal</option><option value="gatk_dbsnp">gatk_dbsnp</option><option value="gatk_interval">gatk_interval</option><option value="gatk_recal">gatk_recal</option><option value="gatk_report">gatk_report</option><option value="gatk_tranche">gatk_tranche</option><option value="gemini.sqlite">gemini.sqlite</option><option value="genbank">genbank</option><option value="genbank.gz">genbank.gz</option><option value="genenotebook">genenotebook</option><option value="geojson">geojson</option><option value="gfa1">gfa1</option><option value="gfa1.gz">gfa1.gz</option><option value="gfa2">gfa2</option><option value="gfa2.gz">gfa2.gz</option><option value="gff">gff</option><option value="gff3">gff3</option><option value="gff3.bz2">gff3.bz2</option><option value="gff3.gz">gff3.gz</option><option value="gff_tabix.gz">gff_tabix.gz</option><option value="gii">gii</option><option value="gii.gz">gii.gz</option><option value="gmsh.geo">gmsh.geo</option><option value="gmsh.msh">gmsh.msh</option><option value="gpr">gpr</option><option value="grd">grd</option><option value="grd.tgz">grd.tgz</option><option value="grib">grib</option><option value="gro">gro</option><option value="gtf">gtf</option><option value="h5">h5</option><option value="h5ad">h5ad</option><option value="h5mlm">h5mlm</option><option value="hdf4">hdf4</option><option value="hdr">hdr</option><option value="hdt">hdt</option><option value="hep.root">hep.root</option><option value="hexrd.accepted_orientations">hexrd.accepted_orientations</option><option value="hexrd.eta_ome.npz">hexrd.eta_ome.npz</option><option value="hexrd.images.npz">hexrd.images.npz</option><option value="hexrd.materials.h5">hexrd.materials.h5</option><option value="hexrd.scored_orientations.npz">hexrd.scored_orientations.npz</option><option value="hexrd.yml">hexrd.yml</option><option value="hhr">hhr</option><option value="hic_matrix">hic_matrix</option><option value="hlf">hlf</option><option value="hmm2">hmm2</option><option value="hmm3">hmm3</option><option value="ibd">ibd</option><option value="icm">icm</option><option value="idat">idat</option><option value="idat">idat</option><option value="idpdb">idpdb</option><option value="idxml">idxml</option><option value="imgt.json">imgt.json</option><option value="inchi">inchi</option><option value="inpcrd">inpcrd</option><option value="intermine_tabular">intermine_tabular</option><option value="interval">interval</option><option value="ipynb">ipynb</option><option value="itp">itp</option><option value="jellyfish">jellyfish</option><option value="jp2">jp2</option><option value="jpg">jpg</option><option value="json">json</option><option value="jsonld">jsonld</option><option value="juicer_medium_tabix.gz">juicer_medium_tabix.gz</option><option value="kallisto.idx">kallisto.idx</option><option value="kroenik">kroenik</option><option value="lav">lav</option><option value="len">len</option><option value="loom">loom</option><option value="maf">maf</option><option value="mascotxml">mascotxml</option><option value="maskinfo-asn1">maskinfo-asn1</option><option value="maskinfo-asn1-binary">maskinfo-asn1-binary</option><option value="mat">mat</option><option value="mcool">mcool</option><option value="mdp">mdp</option><option value="memepsp">memepsp</option><option value="memexml">memexml</option><option value="meryldb">meryldb</option><option value="metacyto_clr.txt">metacyto_clr.txt</option><option value="metacyto_stats.txt">metacyto_stats.txt</option><option value="metacyto_summary.txt">metacyto_summary.txt</option><option value="mgf">mgf</option><option value="mkv">mkv</option><option value="mol">mol</option><option value="mol2">mol2</option><option value="mongodb">mongodb</option><option value="mothur.accnos">mothur.accnos</option><option value="mothur.align">mothur.align</option><option value="mothur.align.check">mothur.align.check</option><option value="mothur.align.report">mothur.align.report</option><option value="mothur.axes">mothur.axes</option><option value="mothur.cons.taxonomy">mothur.cons.taxonomy</option><option value="mothur.count_table">mothur.count_table</option><option value="mothur.design">mothur.design</option><option value="mothur.dist">mothur.dist</option><option value="mothur.filter">mothur.filter</option><option value="mothur.filtered.masked.quan">mothur.filtered.masked.quan</option><option value="mothur.filtered.quan">mothur.filtered.quan</option><option value="mothur.freq">mothur.freq</option><option value="mothur.groups">mothur.groups</option><option value="mothur.list">mothur.list</option><option value="mothur.lower.dist">mothur.lower.dist</option><option value="mothur.map">mothur.map</option><option value="mothur.masked.quan">mothur.masked.quan</option><option value="mothur.names">mothur.names</option><option value="mothur.oligos">mothur.oligos</option><option value="mothur.otu">mothur.otu</option><option value="mothur.otu.corr">mothur.otu.corr</option><option value="mothur.otulabels">mothur.otulabels</option><option value="mothur.pair.dist">mothur.pair.dist</option><option value="mothur.quan">mothur.quan</option><option value="mothur.rabund">mothur.rabund</option><option value="mothur.rdp.taxonomy">mothur.rdp.taxonomy</option><option value="mothur.ref.taxonomy">mothur.ref.taxonomy</option><option value="mothur.relabund">mothur.relabund</option><option value="mothur.sabund">mothur.sabund</option><option value="mothur.seq.taxonomy">mothur.seq.taxonomy</option><option value="mothur.sff.flow">mothur.sff.flow</option><option value="mothur.shared">mothur.shared</option><option value="mothur.square.dist">mothur.square.dist</option><option value="mothur.summary">mothur.summary</option><option value="mothur.tax.summary">mothur.tax.summary</option><option value="mothur.tre">mothur.tre</option><option value="mov">mov</option><option value="mp3">mp3</option><option value="mp4">mp4</option><option value="mpg">mpg</option><option value="mrc">mrc</option><option value="mrm">mrm</option><option value="ms2">ms2</option><option value="msh">msh</option><option value="msp">msp</option><option value="mtx">mtx</option><option value="mz.sqlite">mz.sqlite</option><option value="mz5">mz5</option><option value="mzdata">mzdata</option><option value="mzid">mzid</option><option value="mzml">mzml</option><option value="mzq">mzq</option><option value="mzqc">mzqc</option><option value="mztab">mztab</option><option value="mztab2">mztab2</option><option value="mzxml">mzxml</option><option value="n3">n3</option><option value="ncbi_genome_dataset.zip">ncbi_genome_dataset.zip</option><option value="ncbitaxonomy.sqlite">ncbitaxonomy.sqlite</option><option value="ndx">ndx</option><option value="neper.mscell">neper.mscell</option><option value="neper.points">neper.points</option><option value="neper.points.tsv">neper.points.tsv</option><option value="neper.tesr">neper.tesr</option><option value="neper.tess">neper.tess</option><option value="netcdf">netcdf</option><option value="newick">newick</option><option value="nex">nex</option><option value="nhx">nhx</option><option value="nii1">nii1</option><option value="nii1.gz">nii1.gz</option><option value="nii2">nii2</option><option value="nii2.gz">nii2.gz</option><option value="nmrml">nmrml</option><option value="no_unzip.zip">no_unzip.zip</option><option value="npz">npz</option><option value="nt">nt</option><option value="obo">obo</option><option value="odgi">odgi</option><option value="ome.tiff">ome.tiff</option><option value="onnx">onnx</option><option value="osw">osw</option><option value="owl">owl</option><option value="oxlicg">oxlicg</option><option value="oxligl">oxligl</option><option value="oxling">oxling</option><option value="oxliss">oxliss</option><option value="oxlist">oxlist</option><option value="oxlits">oxlits</option><option value="paf">paf</option><option value="paf.gz">paf.gz</option><option value="par">par</option><option value="param">param</option><option value="paramxml">paramxml</option><option value="parquet">parquet</option><option value="pdb">pdb</option><option value="pdbqt">pdbqt</option><option value="pdf">pdf</option><option value="peff">peff</option><option value="peplist">peplist</option><option value="peptideshaker_archive">peptideshaker_archive</option><option value="pepxml">pepxml</option><option value="pepxml.tsv">pepxml.tsv</option><option value="phar">phar</option><option value="phylip">phylip</option><option value="phyloseq">phyloseq</option><option value="phyloxml">phyloxml</option><option value="picard_interval_list">picard_interval_list</option><option value="pileup">pileup</option><option value="pithya.model">pithya.model</option><option value="pithya.property">pithya.property</option><option value="pithya.result">pithya.result</option><option value="plyascii">plyascii</option><option value="plybinary">plybinary</option><option value="png">png</option><option value="postgresql">postgresql</option><option value="pov">pov</option><option value="pqp">pqp</option><option value="pqr">pqr</option><option value="pretext">pretext</option><option value="prmtop">prmtop</option><option value="probam">probam</option><option value="probed">probed</option><option value="protobuf2">protobuf2</option><option value="protobuf3">protobuf3</option><option value="protxml">protxml</option><option value="protxml.tsv">protxml.tsv</option><option value="psl">psl</option><option value="psms">psms</option><option value="pssm-asn1">pssm-asn1</option><option value="ptkscmp">ptkscmp</option><option value="qcml">qcml</option><option value="qcml">qcml</option><option value="qiime2.tabular">qiime2.tabular</option><option value="qname_sorted.bam">qname_sorted.bam</option><option value="qual454">qual454</option><option value="qualillumina">qualillumina</option><option value="qualsolexa">qualsolexa</option><option value="qualsolid">qualsolid</option><option value="qza">qza</option><option value="qzv">qzv</option><option value="rdata">rdata</option><option value="rdata.camera.negative">rdata.camera.negative</option><option value="rdata.camera.positive">rdata.camera.positive</option><option value="rdata.camera.quick">rdata.camera.quick</option><option value="rdata.msnbase.raw">rdata.msnbase.raw</option><option value="rdata.sce">rdata.sce</option><option value="rdata.xcms.fillpeaks">rdata.xcms.fillpeaks</option><option value="rdata.xcms.findchrompeaks">rdata.xcms.findchrompeaks</option><option value="rdata.xcms.group">rdata.xcms.group</option><option value="rdata.xcms.raw">rdata.xcms.raw</option><option value="rdata.xcms.retcor">rdata.xcms.retcor</option><option value="rdf">rdf</option><option value="rdock_as">rdock_as</option><option value="rds">rds</option><option value="rma6">rma6</option><option value="rna_eps">rna_eps</option><option value="sam">sam</option><option value="sbml">sbml</option><option value="scf">scf</option><option value="scidx">scidx</option><option value="scool">scool</option><option value="scool">scool</option><option value="sdf">sdf</option><option value="searchgui_archive">searchgui_archive</option><option value="sf3">sf3</option><option value="sff">sff</option><option value="sif">sif</option><option value="smat">smat</option><option value="smi">smi</option><option value="snaphmm">snaphmm</option><option value="snpeffdb">snpeffdb</option><option value="snpmatrix">snpmatrix</option><option value="snptest">snptest</option><option value="source.c">source.c</option><option value="source.cpp">source.cpp</option><option value="source.cs">source.cs</option><option value="source.go">source.go</option><option value="source.h">source.h</option><option value="source.py">source.py</option><option value="source.rs">source.rs</option><option value="spec.xml">spec.xml</option><option value="splib_noindex">splib_noindex</option><option value="sqlite">sqlite</option><option value="sqmass">sqmass</option><option value="sra">sra</option><option value="sra_manifest.tabular">sra_manifest.tabular</option><option value="star">star</option><option value="stl">stl</option><option value="stockholm">stockholm</option><option value="storm.check">storm.check</option><option value="storm.sample">storm.sample</option><option value="svg">svg</option><option value="tabular">tabular</option><option value="tabular.gz">tabular.gz</option><option value="tandem">tandem</option><option value="tar">tar</option><option value="taxonomy">taxonomy</option><option value="tck">tck</option><option value="textgrid">textgrid</option><option value="tgz">tgz</option><option value="thermo.raw">thermo.raw</option><option value="tiff">tiff</option><option value="toml">toml</option><option value="top">top</option><option value="tpr">tpr</option><option value="trackhub">trackhub</option><option value="trafoxml">trafoxml</option><option value="trafoxml">trafoxml</option><option value="traml">traml</option><option value="trk">trk</option><option value="trr">trr</option><option value="tsv">tsv</option><option value="ttl">ttl</option><option value="twobit">twobit</option><option value="txt">txt</option><option value="uniprotxml">uniprotxml</option><option value="unsorted.bam">unsorted.bam</option><option value="vcf">vcf</option><option value="vcf_bgzip">vcf_bgzip</option><option value="vel">vel</option><option value="vg">vg</option><option value="visium.tar.gz">visium.tar.gz</option><option value="vtkascii">vtkascii</option><option value="vtkbinary">vtkbinary</option><option value="watersmasslynx.raw.tar">watersmasslynx.raw.tar</option><option value="wav">wav</option><option value="wiff.tar">wiff.tar</option><option value="wiff2.tar">wiff2.tar</option><option value="wig">wig</option><option value="xg">xg</option><option value="xgmml">xgmml</option><option value="xlsx">xlsx</option><option value="xmfa">xmfa</option><option value="xml">xml</option><option value="xquest.xml">xquest.xml</option><option value="xsd">xsd</option><option value="xtc">xtc</option><option value="xvg">xvg</option><option value="xyz">xyz</option><option value="yaml">yaml</option><option value="zip">zip</option><option value="zset.geof">zset.geof</option></select> <span class="upload-footer-extension-info upload-icon-button fa fa-search"></span> <span class="upload-footer-title">Genome (set all):</span> <div class="select2-container upload-footer-genome" id="s2id_autogen29"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-30">
                unspecified (?)
            </span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen30" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-30" id="s2id_autogen30"><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen30_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-30" id="s2id_autogen30_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-30">   </ul></div></div><select tabindex="-1" title="" class="select2-offscreen"><option value="?">
                unspecified (?)
            </option><option value="anoGam1">
                A. gambiae Feb. 2003 (IAGEC MOZ2/anoGam1) (anoGam1)
            </option><option value="anoGam3">
                A. gambiae Oct. 2006 (AgamP3/anoGam3) (anoGam3)
            </option><option value="apiMel4">
                A. mellifera 04 Nov 2010 (Amel_4.5/apiMel4) (apiMel4)
            </option><option value="apiMel2">
                A. mellifera Jan. 2005 (Baylor 2.0/apiMel2) (apiMel2)
            </option><option value="apiMel1">
                A. mellifera July 2004 (Baylor 1.2/apiMel1) (apiMel1)
            </option><option value="GCF_001704415.2">
                ARS1 Aug. 2016 Capra hircus (goat) (San Clemente 2016 USDA) (GCF_001704415.2)
            </option><option value="oryAfe1">
                Aardvark May 2012 (OryAfe1.0/oryAfe1) (oryAfe1)
            </option><option value="12997">
                Acaryochloris marina MBIC11017 (12997)
            </option><option value="19259">
                Acholeplasma laidlawii PG-8A (19259)
            </option><option value="acidCryp_JF_5">
                Acidiphilium cryptum JF-5 (acidCryp_JF_5)
            </option><option value="acidBact_ELLIN345">
                Acidobacteria bacterium Ellin345 (acidBact_ELLIN345)
            </option><option value="acidCell_11B">
                Acidothermus cellulolyticus 11B (acidCell_11B)
            </option><option value="15708">
                Acidovorax avenae subsp. citrulli AAC00-1 (15708)
            </option><option value="acidJS42">
                Acidovorax sp. JS42 (acidJS42)
            </option><option value="13001">
                Acinetobacter baumannii (13001)
            </option><option value="28921">
                Acinetobacter baumannii (28921)
            </option><option value="17477">
                Acinetobacter baumannii ATCC 17978 (17477)
            </option><option value="aciBauTYTH_1">
                Acinetobacter baumannii Oct. 2012 (aciBauTYTH_1/aciBauTYTH_1) (aciBauTYTH_1)
            </option><option value="acinSp_ADP1">
                Acinetobacter sp. ADP1 (acinSp_ADP1)
            </option><option value="Actinidia_chinensis_FEI_Lab_v1.0">
                Actinidia chinensis FEI Lab v1.0
            </option><option value="actiPleu_L20">
                Actinobacillus pleuropneumoniae L20 (actiPleu_L20)
            </option><option value="19135">
                Actinobacillus pleuropneumoniae serovar 3 str. JL03 (19135)
            </option><option value="13370">
                Actinobacillus succinogenes 130Z (13370)
            </option><option value="pygAde1">
                Adelie penguin Jun 2014 (ASM69910v1/pygAde1) (pygAde1)
            </option><option value="aeroHydr_ATCC7966">
                Aeromonas hydrophila subsp. hydrophila ATCC 7966 (aeroHydr_ATCC7966)
            </option><option value="16723">
                Aeromonas salmonicida subsp. salmonicida A449 (16723)
            </option><option value="aerPer1">
                Aeropyrum pernix K1 (aerPer1)
            </option><option value="xenLae9.2">
                African Clawed Frog (xenLae9.2)
            </option><option value="xenLae2">
                African clawed frog Aug. 2016 (Xenopus_laevis_v2/xenLae2) (xenLae2)
            </option><option value="afrOth13">
                Afrotheria Apr. 24. 2006 (UCSC Recon/afrOth13) (afrOth13)
            </option><option value="283">
                Agrobacterium tumefaciens str. C58 (283)
            </option><option value="alcaBork_SK2">
                Alcanivorax borkumensis SK2 (alcaBork_SK2)
            </option><option value="alkaEhrl_MLHE_1">
                Alkalilimnicola ehrlichei MLHE-1 (alkaEhrl_MLHE_1)
            </option><option value="13006">
                Alkaliphilus metalliredigens QYMF (13006)
            </option><option value="16083">
                Alkaliphilus oremlandii OhILAs (16083)
            </option><option value="vicPac1">
                Alpaca Jul. 2008 (Broad/vicPac1) (vicPac1)
            </option><option value="vicPac2">
                Alpaca Mar. 2013 (Vicugna_pacos-2.0.1/vicPac2) (vicPac2)
            </option><option value="poeFor1">
                Amazon molly 28 Oct-2013 (Poecilia_formosa-5.1.2/poeFor1) (poeFor1)
            </option><option value="Amborella_trichopoda_Amborella_V1.0">
                Amborella trichopoda Amborella V1.0
            </option><option value="Amborella_trichopoda_JGI_v1.0">
                Amborella trichopoda JGI v1.0
            </option><option value="allMis1">
                American alligator Aug. 2012 (allMis0.2/allMis1) (allMis1)
            </option><option value="allMis0">
                American alligator November 2011 (UCSC aMiss_v0.1c27/allMis0) (allMis0)
            </option><option value="corBra1">
                American crow Jun 2014 (ASM69197v1/corBra1) (corBra1)
            </option><option value="phoRub1">
                American flamingo May 2014 (ASM68726v1/phoRub1) (phoRub1)
            </option><option value="panTig1">
                Amur tiger 06 Sep 2013 (PanTig1.0/panTig1) (panTig1)
            </option><option value="anabVari_ATCC29413">
                Anabaena variabilis ATCC 29413 (anabVari_ATCC29413)
            </option><option value="anaeDeha_2CP_C">
                Anaeromyxobacter dehalogenans 2CP-C (anaeDeha_2CP_C)
            </option><option value="17729">
                Anaeromyxobacter sp. Fw109-5 (17729)
            </option><option value="Ananas_comosus_JGI_v3.0">
                Ananas comosus JGI v3.0
            </option><option value="anapMarg_ST_MARIES">
                Anaplasma marginale str. St. Maries (anapMarg_ST_MARIES)
            </option><option value="336">
                Anaplasma phagocytophilum HZ (336)
            </option><option value="calAnn1">
                Anna's hummingbird Jun 2014 (ASM69908v1/calAnn1) (calAnn1)
            </option><option value="Apostasia_shenzhenica_v1.0">
                Apostasia shenzhenica v1.0
            </option><option value="aquiAeol">
                Aquifex aeolicus VF5 (aquiAeol)
            </option><option value="Araly1">
                Arabidopsis lyrata (Araly1)
            </option><option value="Arabidopsis_lyrata_JGI_v1.0">
                Arabidopsis lyrata JGI v1.0
            </option><option value="Arabidopsis_lyrata_JGI_v2.1">
                Arabidopsis lyrata JGI v2.1
            </option><option value="TAIR10">
                Arabidopsis thaliana
            </option><option value="Arabidopsis_thaliana_COL-0_Araport11">
                Arabidopsis thaliana COL-0 Araport11
            </option><option value="Arabidopsis_thaliana_COL-0_TAIR10">
                Arabidopsis thaliana COL-0 TAIR10
            </option><option value="Arabidopsis_thaliana_TAIR10">
                Arabidopsis thaliana TAIR10 (Arabidopsis_thaliana_TAIR10) (Arabidopsis_thaliana_TAIR10)
            </option><option value="araTha1">
                Arabidopsis thaliana TAIR7 (araTha1)
            </option><option value="arabidopsis_tair8">
                Arabidopsis thaliana TAIR8 (arabidopsis_tair8)
            </option><option value="arabidopsis">
                Arabidopsis thaliana TAIR9 (arabidopsis)
            </option><option value="Arachis_ipaensis_Peanut_Base_v1.0">
                Arachis ipaensis Peanut Base v1.0
            </option><option value="archFulg1">
                Archaeoglobus fulgidus DSM 4304 (archFulg1)
            </option><option value="16319">
                Arcobacter butzleri RM4018 (16319)
            </option><option value="letCam1">
                Arctic lamprey Sep 2013 (LetJap1.0/letCam1) (letCam1)
            </option><option value="dasNov3">
                Armadillo Dec. 2011 (Baylor/dasNov3) (dasNov3)
            </option><option value="dasNov2">
                Armadillo Jul. 2008 (Broad/dasNov2) (dasNov2)
            </option><option value="dasNov1">
                Armadillo May 2005 (Broad/dasNov1) (dasNov1)
            </option><option value="12512">
                Arthrobacter aurescens TC1 (12512)
            </option><option value="arthFB24">
                Arthrobacter sp. FB24 (arthFB24)
            </option><option value="Asparagus_officinalis_v1.1">
                Asparagus officinalis v1.1
            </option><option value="13478">
                Aster yellows witches'-broom phytoplasma AYWB (13478)
            </option><option value="gadMor1">
                Atlantic cod May 2010 (Genofisk GadMor_May2010/gadMor1) (gadMor1)
            </option><option value="13217">
                Azoarcus sp. BH72 (13217)
            </option><option value="azoaSp_EBN1">
                Azoarcus sp. EbN1 (azoaSp_EBN1)
            </option><option value="azorCaul2">
                Azorhizobium caulinodans ORS 571 (azorCaul2)
            </option><option value="bruMal1">
                B. malayi Sep. 2007 (TIGR 1.0/bruMal1) (bruMal1)
            </option><option value="papAnu4">
                Baboon Apr. 2017 (Panu_3.0/papAnu4) (papAnu4)
            </option><option value="papAnu2">
                Baboon Mar. 2012 (Baylor Panu_2.0/papAnu2) (papAnu2)
            </option><option value="papHam1">
                Baboon Nov. 2008 (Baylor Pham_1.0/papHam1) (papHam1)
            </option><option value="13403">
                Bacillus amyloliquefaciens FZB42 (13403)
            </option><option value="10784">
                Bacillus anthracis str. 'Ames Ancestor' (10784)
            </option><option value="baciAnth_AMES">
                Bacillus anthracis str. Ames (baciAnth_AMES)
            </option><option value="10878">
                Bacillus anthracis str. Sterne (10878)
            </option><option value="74">
                Bacillus cereus ATCC 10987 (74)
            </option><option value="384">
                Bacillus cereus ATCC 14579 (384)
            </option><option value="12468">
                Bacillus cereus E33L (12468)
            </option><option value="13624">
                Bacillus cereus subsp. cytotoxis NVH 391-98 (13624)
            </option><option value="13291">
                Bacillus clausii KSM-K16 (13291)
            </option><option value="baciHalo">
                Bacillus halodurans C-125 (baciHalo)
            </option><option value="12388">
                Bacillus licheniformis ATCC 14580 (12388)
            </option><option value="13082">
                Bacillus licheniformis ATCC 14580 (13082)
            </option><option value="20391">
                Bacillus pumilus SAFR-032 (20391)
            </option><option value="baciSubt">
                Bacillus subtilis subsp. subtilis str. 168 (baciSubt)
            </option><option value="10877">
                Bacillus thuringiensis serovar konkukian str. 97-27 (10877)
            </option><option value="18255">
                Bacillus thuringiensis str. Al Hakam (18255)
            </option><option value="13623">
                Bacillus weihenstephanensis KBAB4 (13623)
            </option><option value="46">
                Bacteroides fragilis NCTC 9343 (46)
            </option><option value="13067">
                Bacteroides fragilis YCH46 (13067)
            </option><option value="bactThet_VPI_5482">
                Bacteroides thetaiotaomicron VPI-5482 (bactThet_VPI_5482)
            </option><option value="13378">
                Bacteroides vulgatus ATCC 8482 (13378)
            </option><option value="camFer1">
                Bactrian camel Dec 2011 (CB1/camFer1) (camFer1)
            </option><option value="halLeu1">
                Bald eagle Aug 2014 (Haliaeetus_leucocephalus-4.0/halLeu1) (halLeu1)
            </option><option value="apaVit1">
                Bar tailed trogon 16 Jun 2014 (ASM70340v1/apaVit1) (apaVit1)
            </option><option value="tytAlb1">
                Barn owl May 2014 (ASM68720v1/tytAlb1) (tytAlb1)
            </option><option value="16249">
                Bartonella bacilliformis KC583 (16249)
            </option><option value="bartHens_HOUSTON_1">
                Bartonella henselae str. Houston-1 (bartHens_HOUSTON_1)
            </option><option value="44">
                Bartonella quintana str. Toulouse (44)
            </option><option value="28109">
                Bartonella tribocorum CIP 105476 (28109)
            </option><option value="baumCica_HOMALODISCA">
                Baumannia cicadellinicola str. Hc (Homalodisca coagulata) (baumCica_HOMALODISCA)
            </option><option value="bdelBact">
                Bdellovibrio bacteriovorus HD100 (bdelBact)
            </option><option value="Beta_vulgaris_RefBeet_1.1">
                Beta vulgaris RefBeet 1.1
            </option><option value="Beta_vulgaris_RefBeet_v1.2">
                Beta vulgaris RefBeet v1.2
            </option><option value="stePar1">
                Bicolor damselfish May 2014 (Stegastes_partitus-1.0.2/stePar1) (stePar1)
            </option><option value="16321">
                Bifidobacterium adolescentis ATCC 15703 (16321)
            </option><option value="bifiLong">
                Bifidobacterium longum NCC2705 (bifiLong)
            </option><option value="eptFus1">
                Big brown bat Jul 2012 (EptFus1.0/eptFus1) (eptFus1)
            </option><option value="bisBis1">
                Bison Oct. 2014 (Bison_UMD1.0/bisBis1) (bisBis1)
            </option><option value="pteAle1">
                Black flying-fox Aug 2012 (ASM32557v1/pteAle1) (pteAle1)
            </option><option value="notCor1">
                Black rockcod Jul 2014 (NC01/notCor1) (notCor1)
            </option><option value="panPan2">
                Bonobo Aug. 2015 (MPI-EVA panpan1.1/panPan2) (panPan2)
            </option><option value="panPan1">
                Bonobo May 2012 (Max-Planck/panPan1) (panPan1)
            </option><option value="panPan3">
                Bonobo May 2020 (Mhudiblu_PPA_v0/panPan3) (panPan3)
            </option><option value="bordBron">
                Bordetella bronchiseptica RB50 (bordBron)
            </option><option value="25">
                Bordetella parapertussis 12822 (25)
            </option><option value="26">
                Bordetella pertussis Tohama I (26)
            </option><option value="28135">
                Bordetella petrii DSM 12804 (28135)
            </option><option value="borEut13">
                Boreoeutherian Apr. 24. 2006 (UCSC Recon/borEut13) (borEut13)
            </option><option value="canHg12">
                Boreoeutherian Nov. 19. 2005 (UCSC Recon/canHg12) (canHg12)
            </option><option value="17057">
                Borrelia afzelii PKo (17057)
            </option><option value="borrBurg">
                Borrelia burgdorferi B31 (borrBurg)
            </option><option value="12554">
                Borrelia garinii PBi (12554)
            </option><option value="Brachypodium_distachyon_JGI_v3.1">
                Brachypodium distachyon JGI v3.1
            </option><option value="Brachypodium_distachyon_MIPS_1.2">
                Brachypodium distachyon MIPS 1.2
            </option><option value="bradJapo">
                Bradyrhizobium japonicum USDA 110 (bradJapo)
            </option><option value="16137">
                Bradyrhizobium sp. BTAi1 (16137)
            </option><option value="19575">
                Bradyrhizobium sp. ORS278 (19575)
            </option><option value="myoBra1">
                Brandt's myotis (bat) 28 Jun 2013 (ASM41265v1/myoBra1) (myoBra1)
            </option><option value="braNey1">
                Braney test Jul. 2013 (braNey1/braNey1) (braNey1)
            </option><option value="Brassica_oleracea_Ensembl_Genomes_v2.1">
                Brassica oleracea Ensembl Genomes v2.1
            </option><option value="Brassica_rapa_Annotation_v1.2_on_assembly_v1.1_Phytozome_9.1">
                Brassica rapa Annotation v1.2 on assembly v1.1 (Phytozome 9.1)
            </option><option value="Brassica_rapa_JGI_FPsc1.3">
                Brassica rapa JGI FPsc1.3
            </option><option value="aptMan1">
                Brown kiwi Jun. 2015 (MPI-EVA AptMant0/aptMan1) (aptMan1)
            </option><option value="mesUni1">
                Brown roatelo Jun 2014 (ASM69576v1/mesUni1) (mesUni1)
            </option><option value="9619">
                Brucella abortus biovar 1 str. 9-941 (9619)
            </option><option value="20243">
                Brucella canis ATCC 23365 (20243)
            </option><option value="brucMeli">
                Brucella melitensis 16M (brucMeli)
            </option><option value="16203">
                Brucella melitensis biovar Abortus 2308 (16203)
            </option><option value="12514">
                Brucella ovis ATCC 25840 (12514)
            </option><option value="320">
                Brucella suis 1330 (320)
            </option><option value="20371">
                Brucella suis ATCC 23445 (20371)
            </option><option value="octDeg1">
                Brush-tailed rat Apr 2012 (OctDeg1.0/octDeg1) (octDeg1)
            </option><option value="buchSp">
                Buchnera aphidicola str. APS (Acyrthosiphon pisum) (buchSp)
            </option><option value="256">
                Buchnera aphidicola str. Bp (Baizongia pistaciae) (256)
            </option><option value="16372">
                Buchnera aphidicola str. Cc (Cinara cedri) (16372)
            </option><option value="312">
                Buchnera aphidicola str. Sg (Schizaphis graminum) (312)
            </option><option value="melUnd1">
                Budgerigar Sep. 2011 (WUSTL v6.3/melUnd1) (melUnd1)
            </option><option value="bunEbo1">
                Bundibugyo ebolavirus Aug. 2014 (NC_014373.1/bunEbo1) (bunEbo1)
            </option><option value="burkCepa_AMMD">
                Burkholderia ambifaria AMMD (burkCepa_AMMD)
            </option><option value="burkCeno_AU_1054">
                Burkholderia cenocepacia AU 1054 (burkCeno_AU_1054)
            </option><option value="burkCeno_HI2424">
                Burkholderia cenocepacia HI2424 (burkCeno_HI2424)
            </option><option value="17929">
                Burkholderia cenocepacia MC0-3 (17929)
            </option><option value="burkMall_ATCC23344">
                Burkholderia mallei ATCC 23344 (burkMall_ATCC23344)
            </option><option value="13943">
                Burkholderia mallei NCTC 10229 (13943)
            </option><option value="13946">
                Burkholderia mallei NCTC 10247 (13946)
            </option><option value="13947">
                Burkholderia mallei SAVP1 (13947)
            </option><option value="17407">
                Burkholderia multivorans ATCC 17616 (17407)
            </option><option value="burkPseu_1106A">
                Burkholderia pseudomallei 1106a (burkPseu_1106A)
            </option><option value="13954">
                Burkholderia pseudomallei 1710b (13954)
            </option><option value="13953">
                Burkholderia pseudomallei 668 (13953)
            </option><option value="178">
                Burkholderia pseudomallei K96243 (178)
            </option><option value="burk383">
                Burkholderia sp. 383 (burk383)
            </option><option value="burkThai_E264">
                Burkholderia thailandensis E264 (burkThai_E264)
            </option><option value="burkViet_G4">
                Burkholderia vietnamiensis G4 (burkViet_G4)
            </option><option value="burkXeno_LB400">
                Burkholderia xenovorans LB400 (burkXeno_LB400)
            </option><option value="pytBiv1">
                Burmese python Sep. 2013 (Python_molurus_bivittatus-5.0.2/pytBiv1) (pytBiv1)
            </option><option value="hapBur1">
                Burton's mouthbreeder Oct 2011 (AstBur1.0/hapBur1) (hapBur1)
            </option><option value="otoGar1">
                Bushbaby Dec. 2006 (Broad/otoGar1) (otoGar1)
            </option><option value="otoGar3">
                Bushbaby Mar. 2011 (Broad/otoGar3) (otoGar3)
            </option><option value="caeAng1">
                C. angaria Oct. 2010 (WS225/caeAng1) (caeAng1)
            </option><option value="caePb2">
                C. brenneri Feb. 2008 (WUGSC 6.0.1/caePb2) (caePb2)
            </option><option value="caePb1">
                C. brenneri Jan. 2007 (WUGSC 4.0/caePb1) (caePb1)
            </option><option value="caePb3">
                C. brenneri Nov. 2010 (C. brenneri 6.0.1b/caePb3) (caePb3)
            </option><option value="cb4">
                C. briggsae Apr. 2011 (WS225/cb4) (cb4)
            </option><option value="cb2">
                C. briggsae Aug 2005 (WUGSC prelim/cb2) (cb2)
            </option><option value="cb3">
                C. briggsae Jan. 2007 (WUGSC 1.0/cb3) (cb3)
            </option><option value="cb1">
                C. briggsae July 2002 (WormBase cb25.agp8/cb1) (cb1)
            </option><option value="ce5">
                C. elegans Aug. 2007 (WS180/ce5) (ce5)
            </option><option value="ce7">
                C. elegans Feb 2009 (WS200/ce7) (ce7)
            </option><option value="ce11">
                C. elegans Feb. 2013 (WBcel235/ce11) (ce11)
            </option><option value="ce9">
                C. elegans Jan 2010 (WS210/ce9) (ce9)
            </option><option value="ce4">
                C. elegans Jan. 2007 (WS170/ce4) (ce4)
            </option><option value="ce8">
                C. elegans Jun 2009 (WS204/ce8) (ce8)
            </option><option value="ce2">
                C. elegans Mar. 2004 (WS120/ce2) (ce2)
            </option><option value="ce3">
                C. elegans March 2005 (WS140/ce3) (ce3)
            </option><option value="ce6">
                C. elegans May 2008 (WS190/ce6) (ce6)
            </option><option value="ce10">
                C. elegans Oct. 2010 (WS220/ce10) (ce10)
            </option><option value="ci3">
                C. intestinalis Apr. 2011 (Kyoto KH/ci3) (ci3)
            </option><option value="ci1">
                C. intestinalis Dec. 2002 (JGI 1.0/ci1) (ci1)
            </option><option value="ci2">
                C. intestinalis Mar. 2005 (JGI 2.1/ci2) (ci2)
            </option><option value="caeJap3">
                C. japonica  Sep. 2010 (WUGSC 7.0.1/caeJap3) (caeJap3)
            </option><option value="caeJap4">
                C. japonica Aug. 2010 (WUSTL 7.0.1/caeJap4) (caeJap4)
            </option><option value="caeJap2">
                C. japonica Jan. 2009 (WUGSC 4.0.1/caeJap2) (caeJap2)
            </option><option value="caeJap1">
                C. japonica Mar. 2008 (WUGSC 3.0.2/caeJap1) (caeJap1)
            </option><option value="caeRem4">
                C. remanei Jul. 2007 (WS220/caeRem4) (caeRem4)
            </option><option value="caeRem2">
                C. remanei Mar. 2006 (WUGSC 1.0/caeRem2) (caeRem2)
            </option><option value="caeRem1">
                C. remanei March 2005 (WUGSC prelim/caeRem1) (caeRem1)
            </option><option value="caeRem3">
                C. remanei May 2007 (WUGSC 15.0.1/caeRem3) (caeRem3)
            </option><option value="cioSav1">
                C. savignyi Apr. 2003 (Broad/cioSav1) (cioSav1)
            </option><option value="cioSav2">
                C. savignyi July 2005 (Sidow Lab 2.0/cioSav2) (cioSav2)
            </option><option value="caeSp111">
                C. sp. 11 ju1373 Nov. 2010 (WUSTL 3.0.1/caeSp111) (caeSp111)
            </option><option value="caeSp71">
                C. sp. 7 ju1286 Nov. 2010 (WUSTL 4.0.1/caeSp71) (caeSp71)
            </option><option value="caeSp91">
                C. sp. 9 ju1422 Nov. 2010 (WUSTL 3.0.1/caeSp91) (caeSp91)
            </option><option value="cacTus3">
                Cactus Nov. 2010 (CactusV3/cacTus3) (cacTus3)
            </option><option value="cacTus2">
                Cactus Oct. 2010 (Ngan10.1/cacTus2) (cacTus2)
            </option><option value="cacTus1">
                Cactus Sep. 2010 (CactusV1/cacTus1) (cacTus1)
            </option><option value="Cajanus_cajan_Penguin_Genomics_v5.0">
                Cajanus cajan Penguin Genomics v5.0
            </option><option value="caldSacc_DSM8903">
                Caldicellulosiruptor saccharolyticus DSM 8903 (caldSacc_DSM8903)
            </option><option value="caldMaqu1">
                Caldivirga maquilingensis IC-167 (caldMaqu1)
            </option><option value="17159">
                Campylobacter concisus 13826 (17159)
            </option><option value="17161">
                Campylobacter curvus 525.92 (17161)
            </option><option value="campFetu_82_40">
                Campylobacter fetus subsp. fetus 82-40 (campFetu_82_40)
            </option><option value="20083">
                Campylobacter hominis ATCC BAA-381 (20083)
            </option><option value="campJeju_RM1221">
                Campylobacter jejuni RM1221 (campJeju_RM1221)
            </option><option value="17163">
                Campylobacter jejuni subsp. doylei 269.97 (17163)
            </option><option value="campJeju_81_176">
                Campylobacter jejuni subsp. jejuni 81-176 (campJeju_81_176)
            </option><option value="17953">
                Campylobacter jejuni subsp. jejuni 81116 (17953)
            </option><option value="campJeju">
                Campylobacter jejuni subsp. jejuni NCTC 11168 (campJeju)
            </option><option value="serCan1">
                Canary 15 Jan-2014 (SCA1/serCan1) (serCan1)
            </option><option value="blocFlor">
                Candidatus Blochmannia floridanus (blocFlor)
            </option><option value="13875">
                Candidatus Blochmannia pennsylvanicus str. BPEN (13875)
            </option><option value="candCars_RUDDII">
                Candidatus Carsonella ruddii PV (candCars_RUDDII)
            </option><option value="21047">
                Candidatus Desulforudis audaxviator MP104C (21047)
            </option><option value="16525">
                Candidatus Korarchaeum cryptofilum OPF8 (16525)
            </option><option value="methBoon1">
                Candidatus Methanoregula boonei 6A8 (methBoon1)
            </option><option value="candPela_UBIQUE_HTCC1">
                Candidatus Pelagibacter ubique HTCC1062 (candPela_UBIQUE_HTCC1)
            </option><option value="paraSp_UWE25">
                Candidatus Protochlamydia amoebophila UWE25 (paraSp_UWE25)
            </option><option value="16841">
                Candidatus Ruthia magnifica str. Cm (Calyptogena magnifica) (16841)
            </option><option value="19617">
                Candidatus Sulcia muelleri GWSS (19617)
            </option><option value="18267">
                Candidatus Vesicomyosocius okutanii HA (18267)
            </option><option value="eleEdw1">
                Cape elephant shrew Aug 2012 (EleEdw1.0/eleEdw1) (eleEdw1)
            </option><option value="chrAsi1">
                Cape golden mole Aug 2012 (ChrAsi1.0/chrAsi1) (chrAsi1)
            </option><option value="Capsella_rubella_JGI_annotation_v1.0_on_assembly_v1">
                Capsella rubella JGI annotation v1.0 on assembly v1
            </option><option value="Capsella_rubella_JGI_v1.0">
                Capsella rubella JGI v1.0
            </option><option value="Capsicum_annuum_Pepper_Genome_v2.0">
                Capsicum annuum Pepper Genome v2.0
            </option><option value="carbHydr_Z_2901">
                Carboxydothermus hydrogenoformans Z-2901 (carbHydr_Z_2901)
            </option><option value="Carica_papaya_Hawaii_Agriculture_Research_Center">
                Carica papaya Hawaii Agriculture Research Center
            </option><option value="Carica_papaya_JGI_ASGPB0.4">
                Carica papaya JGI ASGPB0.4
            </option><option value="felCat4">
                Cat Dec. 2008 (NHGRI/GTB V17e/felCat4) (felCat4)
            </option><option value="felCat3">
                Cat Mar. 2006 (Broad/felCat3) (felCat3)
            </option><option value="felCat8">
                Cat Nov. 2014 (ICGSC Felis_catus_8.0/felCat8) (felCat8)
            </option><option value="felCat9">
                Cat Nov. 2017 (Felis_catus_9.0/felCat9) (felCat9)
            </option><option value="felCat5">
                Cat Sep. 2011 (ICGSC Felis_catus 6.2/felCat5) (felCat5)
            </option><option value="catArr1">
                Catarrhini June 13. 2006 (UCSC Recon/catArr1) (catArr1)
            </option><option value="caulCres">
                Caulobacter crescentus CB15 (caulCres)
            </option><option value="16306">
                Caulobacter sp. K31 (16306)
            </option><option value="Cenchrus_americanus_v1.0">
                Cenchrus americanus v1.0
            </option><option value="centromeres1">
                CentromereY 2013 GrCh38 (centromeres1)
            </option><option value="Chenopodium_quinoa_JGI_v1.0">
                Chenopodium quinoa JGI v1.0
            </option><option value="galGal5">
                Chicken Dec 2015 (Gallus_gallus-5.0/galGal5) (galGal5)
            </option><option value="galGal2">
                Chicken Feb. 2004 (WUGSC 1.0/galGal2) (galGal2)
            </option><option value="galGal6">
                Chicken Mar. 2018 (GRCg6a/galGal6) (galGal6)
            </option><option value="galGal3">
                Chicken May 2006 (WUGSC 2.1/galGal3) (galGal3)
            </option><option value="galGal4">
                Chicken Nov. 2011 (ICGSC Gallus_gallus-4.0/galGal4) (galGal4)
            </option><option value="GCF_016700215.2">
                Chicken Oct. 2021 (bGalGal1.pat.whiteleghornlayer.GRCg7w_WZ) (GCF_016700215.2)
            </option><option value="panTro4">
                Chimp Feb. 2011 (CSAC 2.1.4/panTro4) (panTro4)
            </option><option value="panTro6">
                Chimp Jan. 2018 (Clint_PTRv2/panTro6) (panTro6)
            </option><option value="panTro2">
                Chimp Mar. 2006 (CGSC 2.1/panTro2) (panTro2)
            </option><option value="panTro5">
                Chimp May 2016 (Pan_tro 3.0/panTro5) (panTro5)
            </option><option value="panTro1">
                Chimp Nov. 2003 (CGSC 1.1/panTro1) (panTro1)
            </option><option value="panTro3">
                Chimp Oct. 2010 (CGSC 2.1.3/panTro3) (panTro3)
            </option><option value="chiLan1">
                Chinchilla May 2012  (ChiLan1.0/chiLan1) (chiLan1)
            </option><option value="allSin1">
                Chinese alligator 29 Aug 2013 (ASM45574v1/allSin1) (allSin1)
            </option><option value="criGriChoV1">
                Chinese hamster Aug. 2011 (CHO K1 cell line/criGriChoV1) (criGriChoV1)
            </option><option value="criGri1">
                Chinese hamster Jul. 2013 (C_griseus_v1.0/criGri1) (criGri1)
            </option><option value="criGriChoV2">
                Chinese hamster Jun. 2017 (CHOK1S_HZDv1/criGriChoV2) (criGriChoV2)
            </option><option value="manPen1">
                Chinese pangolin Aug 2014 (M_pentadactyla-1.1.1/manPen1) (manPen1)
            </option><option value="pelSin1">
                Chinese softshell turtle Oct 2011 (PelSin_1.0/pelSin1) (pelSin1)
            </option><option value="tupChi1">
                Chinese tree shrew Jan 2013 (TupChi_1.0/tupChi1) (tupChi1)
            </option><option value="229">
                Chlamydia muridarum Nigg (229)
            </option><option value="28583">
                Chlamydia trachomatis 434/Bu (28583)
            </option><option value="13885">
                Chlamydia trachomatis A/HAR-13 (13885)
            </option><option value="chlaTrac">
                Chlamydia trachomatis D/UW-3/CX (chlaTrac)
            </option><option value="28585">
                Chlamydia trachomatis L2b/UCH-1/proctitis (28585)
            </option><option value="Chlamydomonas_reinhardtii_JGI_v5.0_assembly_annot_v5.3.1_based_on_Augustus_u11.6">
                Chlamydomonas reinhardtii JGI v5.0 assembly, annot v5.3.1 based on Augustus u11.6
            </option><option value="Chlamydomonas_reinhardtii_JGI_v5.5">
                Chlamydomonas reinhardtii JGI v5.5
            </option><option value="355">
                Chlamydophila abortus S26/3 (355)
            </option><option value="228">
                Chlamydophila caviae GPIC (228)
            </option><option value="370">
                Chlamydophila felis Fe/C-56 (370)
            </option><option value="247">
                Chlamydophila pneumoniae AR39 (247)
            </option><option value="chlaPneu_CWL029">
                Chlamydophila pneumoniae CWL029 (chlaPneu_CWL029)
            </option><option value="257">
                Chlamydophila pneumoniae J138 (257)
            </option><option value="420">
                Chlamydophila pneumoniae TW-183 (420)
            </option><option value="chloChlo_CAD3">
                Chlorobium chlorochromatii CaD3 (chloChlo_CAD3)
            </option><option value="12609">
                Chlorobium phaeobacteroides DSM 266 (12609)
            </option><option value="chloTepi_TLS">
                Chlorobium tepidum TLS (chloTepi_TLS)
            </option><option value="59">
                Chloroflexus aurantiacus J-10-fl (59)
            </option><option value="chroViol">
                Chromobacterium violaceum ATCC 12472 (chroViol)
            </option><option value="chroSale_DSM3043">
                Chromohalobacter salexigens DSM 3043 (chroSale_DSM3043)
            </option><option value="capCar1">
                Chuck-will's-widow Jun 2014 (ASM70074v1/capCar1) (capCar1)
            </option><option value="Cicer_arietinum_LIS_v1.0">
                Cicer arietinum LIS v1.0
            </option><option value="12716">
                Citrobacter koseri ATCC BAA-895 (12716)
            </option><option value="Citrullus_lanatus_Cucurbit_Genomics_Database">
                Citrullus lanatus Cucurbit Genomics Database
            </option><option value="Citrullus_lanatus_Cucurbit_Genomics_Database_v1.0">
                Citrullus lanatus Cucurbit Genomics Database v1.0
            </option><option value="Citrus_clementina_JGI_v1.0">
                Citrus clementina JGI v1.0
            </option><option value="Citrus_sinensis_JGI_v1_assembly_and_v1.0_annotation">
                Citrus sinensis JGI v1 assembly and v1.0 annotation
            </option><option value="clavMich_NCPPB_382">
                Clavibacter michiganensis subsp. michiganensis NCPPB 382 (clavMich_NCPPB_382)
            </option><option value="184">
                Clavibacter michiganensis subsp. sepedonicus (184)
            </option><option value="77">
                Clostridium acetobutylicum ATCC 824 (77)
            </option><option value="12637">
                Clostridium beijerinckii NCIMB 8052 (12637)
            </option><option value="19517">
                Clostridium botulinum A str. ATCC 19397 (19517)
            </option><option value="193">
                Clostridium botulinum A str. ATCC 3502 (193)
            </option><option value="19521">
                Clostridium botulinum A str. Hall (19521)
            </option><option value="28507">
                Clostridium botulinum A3 str. Loch Maree (28507)
            </option><option value="28505">
                Clostridium botulinum B1 str. Okra (28505)
            </option><option value="19519">
                Clostridium botulinum F str. Langeland (19519)
            </option><option value="78">
                Clostridium difficile 630 (78)
            </option><option value="19065">
                Clostridium kluyveri DSM 555 (19065)
            </option><option value="16820">
                Clostridium novyi NT (16820)
            </option><option value="304">
                Clostridium perfringens ATCC 13124 (304)
            </option><option value="12521">
                Clostridium perfringens SM101 (12521)
            </option><option value="79">
                Clostridium perfringens str. 13 (79)
            </option><option value="16184">
                Clostridium phytofermentans ISDg (16184)
            </option><option value="81">
                Clostridium tetani E88 (81)
            </option><option value="314">
                Clostridium thermocellum ATCC 27405 (314)
            </option><option value="latCha1">
                Coelacanth Aug. 2011 (Broad/latCha1) (latCha1)
            </option><option value="Coffea_canephora_Coffee_Genome_Hub_v1">
                Coffea canephora Coffee Genome Hub v1
            </option><option value="ficAlb2">
                Collared flycatcher Jun 2013 (FicAlb1.5/ficAlb2) (ficAlb2)
            </option><option value="colwPsyc_34H">
                Colwellia psychrerythraea 34H (colwPsyc_34H)
            </option><option value="cucCan1">
                Common cuckoo Jun 2014 (ASM70932v1/cucCan1) (cucCan1)
            </option><option value="Corchorus_olitorius_GenBank_v1.0">
                Corchorus olitorius GenBank v1.0
            </option><option value="87">
                Corynebacterium diphtheriae NCTC 13129 (87)
            </option><option value="coryEffi_YS_314">
                Corynebacterium efficiens YS-314 (coryEffi_YS_314)
            </option><option value="13760">
                Corynebacterium glutamicum ATCC 13032 (13760)
            </option><option value="307">
                Corynebacterium glutamicum ATCC 13032 (307)
            </option><option value="19193">
                Corynebacterium glutamicum R (19193)
            </option><option value="13967">
                Corynebacterium jeikeium K411 (13967)
            </option><option value="bosTau5">
                Cow Apr. 2009 (Baylor Btau_4.2/bosTau5) (bosTau5)
            </option><option value="bosTau9">
                Cow Apr. 2018 (ARS-UCD1.2/bosTau9) (bosTau9)
            </option><option value="bosTau3">
                Cow Aug. 2006 (Baylor 3.1/bosTau3) (bosTau3)
            </option><option value="bosTauMd3">
                Cow Aug. 2009 (UMD3.1) (bosTauMd3)
            </option><option value="bosTau8">
                Cow Jun. 2014 (Bos_taurus_UMD_3.1.1/bosTau8) (bosTau8)
            </option><option value="bosTau2">
                Cow Mar. 2005 (Baylor 2.0/bosTau2) (bosTau2)
            </option><option value="bosTau6">
                Cow Nov. 2009 (Bos_taurus_UMD_3.1/bosTau6) (bosTau6)
            </option><option value="bosTau4">
                Cow Oct. 2007 (Baylor 4.0/bosTau4) (bosTau4)
            </option><option value="bosTau7">
                Cow Oct. 2011 (Baylor Btau_4.6.1/bosTau7) (bosTau7)
            </option><option value="16721">
                Coxiella burnetii Dugway 5J108-111 (16721)
            </option><option value="16791">
                Coxiella burnetii RSA 331 (16791)
            </option><option value="coxiBurn">
                Coxiella burnetii RSA 493 (coxiBurn)
            </option><option value="macFas5">
                Crab-eating macaque Jun. 2013 (Macaca_fascicularis_5.0/macFas5) (macFas5)
            </option><option value="nipNip1">
                Crested ibis Jun 2014 (ASM70822v1/nipNip1) (nipNip1)
            </option><option value="balPav1">
                Crowned crain Jun 2014 (ASM70989v1/balPav1) (balPav1)
            </option><option value="lepDis1">
                Cuckoo roller May 2014 (ASM69178v1/lepDis1) (lepDis1)
            </option><option value="Cucumis_melo_Melonomics_v3.5">
                Cucumis melo Melonomics v3.5
            </option><option value="Cucumis_sativus_L._JGI_v1.0">
                Cucumis sativus L. JGI v1.0
            </option><option value="cytoHutc_ATCC33406">
                Cytophaga hutchinsonii ATCC 33406 (cytoHutc_ATCC33406)
            </option><option value="droAlb1">
                D. albomicans 21 May 2012 (DroAlb_1.0/droAlb1) (droAlb1)
            </option><option value="droAna2">
                D. ananassae Aug. 2005 (Agencourt prelim/droAna2) (droAna2)
            </option><option value="droAna3">
                D. ananassae Feb. 2006 (droAna3)
            </option><option value="droAna1">
                D. ananassae July 2004 (TIGR/droAna1) (droAna1)
            </option><option value="droBia2">
                D. biarmipes 04 Mar 2013 (Dbia_2.0/droBia2) (droBia2)
            </option><option value="droBip2">
                D. bipectinata 04 Mar 2013 (Dbip_2.0/droBip2) (droBip2)
            </option><option value="droEle2">
                D. elegans 04 Mar 2013 (Dele_2.0/droEle2) (droEle2)
            </option><option value="droEre1">
                D. erecta Aug. 2005 (Agencourt prelim/droEre1) (droEre1)
            </option><option value="droEre2">
                D. erecta Feb. 2006 (droEre2)
            </option><option value="droEug2">
                D. eugracilis 04 Mar 2013 (Deug_2.0/droEug2) (droEug2)
            </option><option value="droFic2">
                D. ficusphila 04 Mar 2013 (Dfic_2.0/droFic2) (droFic2)
            </option><option value="droGri1">
                D. grimshawi Aug. 2005 (Agencourt prelim/droGri1) (droGri1)
            </option><option value="droGri2">
                D. grimshawi Feb. 2006 (droGri2)
            </option><option value="droKik2">
                D. kikkawai 04 Mar 2013 (Dkik_2.0/droKik2) (droKik2)
            </option><option value="dm2">
                D. melanogaster Apr. 2004 (BDGP R4/dm2) (dm2)
            </option><option value="dm3">
                D. melanogaster Apr. 2006 (BDGP R5/dm3) (dm3)
            </option><option value="dm6">
                D. melanogaster Aug. 2014 (BDGP Release 6 + ISO1 MT/dm6) (dm6)
            </option><option value="dm1">
                D. melanogaster Jan. 2003 (BDGP R3/dm1) (dm1)
            </option><option value="droMir2">
                D. miranda 19 Apr 2013 (DroMir_2.2/droMir2) (droMir2)
            </option><option value="droMoj1">
                D. mojavensis Aug. 2004 (Agencourt prelim/droMoj1) (droMoj1)
            </option><option value="droMoj2">
                D. mojavensis Aug. 2005 (Agencourt prelim/droMoj2) (droMoj2)
            </option><option value="droMoj3">
                D. mojavensis Feb. 2006 (droMoj3)
            </option><option value="droPer1">
                D. persimilis Oct. 2005 (Broad/droPer1) (droPer1)
            </option><option value="dp4">
                D. pseudoobscura (dp4) (dp4)
            </option><option value="droPse3">
                D. pseudoobscura 11 Apr 2013 (Dpse_3.0/droPse3) (droPse3)
            </option><option value="dp2">
                D. pseudoobscura Aug. 2003 (Baylor freeze1/dp2) (dp2)
            </option><option value="dp3">
                D. pseudoobscura Nov. 2004 (FlyBase 1.03/dp3) (dp3)
            </option><option value="droRho2">
                D. rhopaloa 22 Feb 2013 (Drho_2.0/droRho2) (droRho2)
            </option><option value="droSec1">
                D. sechellia Oct. 2005 (Broad/droSec1) (droSec1)
            </option><option value="droSim1">
                D. simulans Apr. 2005 (WUGSC mosaic 1.0/droSim1) (droSim1)
            </option><option value="droSuz1">
                D. suzukii 30 Sep 2013 (Dsuzukii.v01/droSuz1) (droSuz1)
            </option><option value="droTak2">
                D. takahashii 04 Mar 2013 (Dtak_2.0/droTak2) (droTak2)
            </option><option value="droVir2">
                D. virilis Aug. 2005 (Agencourt prelim/droVir2) (droVir2)
            </option><option value="droVir3">
                D. virilis Feb. 2006 (droVir3)
            </option><option value="droVir1">
                D. virilis July 2004 (Agencourt prelim/droVir1) (droVir1)
            </option><option value="droWil2">
                D. willistoni 03 Aug 2006 (dwil_caf1/droWil2) (droWil2)
            </option><option value="droWil1">
                D. willistoni Feb. 2006 (droWil1)
            </option><option value="droYak3">
                D. yakuba 27 Jun 2006 (dyak_caf1/droYak3) (droYak3)
            </option><option value="droYak1">
                D. yakuba Apr. 2004 (WUGSC 1.0/droYak1) (droYak1)
            </option><option value="droYak2">
                D. yakuba Nov. 2005 (WUGSC 7.1/droYak2) (droYak2)
            </option><option value="pelCri1">
                Dalmatian pelican May 2014 (ASM68737v1/pelCri1) (pelCri1)
            </option><option value="Daucus_carota_JGI_v2.0">
                Daucus carota JGI v2.0
            </option><option value="myoDav1">
                David's myotis (bat) Aug 2012 (ASM32734v1/myoDav1) (myoDav1)
            </option><option value="dechArom_RCB">
                Dechloromonas aromatica RCB (dechArom_RCB)
            </option><option value="IscaW1">
                Deer Tick (Ixodes scapularis) (IscaW1)
            </option><option value="dehaEthe_195">
                Dehalococcoides ethenogenes 195 (dehaEthe_195)
            </option><option value="15770">
                Dehalococcoides sp. BAV1 (15770)
            </option><option value="15604">
                Dehalococcoides sp. CBDB1 (15604)
            </option><option value="deinGeot_DSM11300">
                Deinococcus geothermalis DSM 11300 (deinGeot_DSM11300)
            </option><option value="deinRadi">
                Deinococcus radiodurans R1 (deinRadi)
            </option><option value="17413">
                Delftia acidovorans SPH-1 (17413)
            </option><option value="desuHafn_Y51">
                Desulfitobacterium hafniense Y51 (desuHafn_Y51)
            </option><option value="18007">
                Desulfococcus oleovorans Hxd3 (18007)
            </option><option value="desuPsyc_LSV54">
                Desulfotalea psychrophila LSv54 (desuPsyc_LSV54)
            </option><option value="desuRedu_MI_1">
                Desulfotomaculum reducens MI-1 (desuRedu_MI_1)
            </option><option value="329">
                Desulfovibrio desulfuricans G20 (329)
            </option><option value="17227">
                Desulfovibrio vulgaris subsp. vulgaris DP4 (17227)
            </option><option value="desuVulg_HILDENBOROUG">
                Desulfovibrio vulgaris subsp. vulgaris str. Hildenborough (desuVulg_HILDENBOROUG)
            </option><option value="dichNodo_VCS1703A">
                Dichelobacter nodosus VCS1703A (dichNodo_VCS1703A)
            </option><option value="17417">
                Dinoroseobacter shibae DFL 12 (17417)
            </option><option value="canFam1">
                Dog July 2004 (Broad/canFam1) (canFam1)
            </option><option value="canFam4">
                Dog Mar. 2020 (UU_Cfam_GSD_1.0/canFam4) (canFam4)
            </option><option value="canFamPoodle1">
                Dog May 2003 (TIGR Poodle/canFamPoodle1) (canFamPoodle1)
            </option><option value="canFam2">
                Dog May 2005 (Broad/canFam2) (canFam2)
            </option><option value="canFam5">
                Dog May 2019 (UMICH_Zoey_3.1/canFam5) (canFam5)
            </option><option value="canFam6">
                Dog Oct. 2020 (Dog10K_Boxer_Tasha/canFam6) (canFam6)
            </option><option value="turTru1">
                Dolphin Feb. 2008 (Broad/turTru1) (turTru1)
            </option><option value="turTru2">
                Dolphin Oct. 2011 (Baylor Ttru_1.4/turTru2) (turTru2)
            </option><option value="capHir1">
                Domestic goat May 2012 (CHIR_1.0/capHir1) (capHir1)
            </option><option value="picPub1">
                Downy woodpecker Jun 2014 (ASM69900v1/picPub1) (picPub1)
            </option><option value="eboVir2">
                Ebola virus 49 sequences: 5 Ebola, 2 Marburg strains (eboVir2)
            </option><option value="eboVir1">
                Ebola virus Aug. 2014 (46 strains eboVir1/eboVir1) (eboVir1)
            </option><option value="eboVir3">
                Ebola virus Sierra Leone 2014 (G3683/KM034562.1/eboVir3) (eboVir3)
            </option><option value="ecoRef3">
                Ecoref Feb. 2013 (ecoRef3/ecoRef3) (ecoRef3)
            </option><option value="ecoRef4">
                Ecoref Feb. 2013 (ecoRef4/ecoRef4) (ecoRef4)
            </option><option value="ecoRef1">
                Ecoref Jan. 2013 (ecoRef1/ecoRef1) (ecoRef1)
            </option><option value="ecoRef2">
                Ecoref Jan. 2013 (ecoRef2/ecoRef2) (ecoRef2)
            </option><option value="10694">
                Ehrlichia canis str. Jake (10694)
            </option><option value="325">
                Ehrlichia chaffeensis str. Arkansas (325)
            </option><option value="13356">
                Ehrlichia ruminantium str. Gardel (13356)
            </option><option value="13355">
                Ehrlichia ruminantium str. Welgevonden (13355)
            </option><option value="ehrlRumi_WELGEVONDEN">
                Ehrlichia ruminantium str. Welgevonden (ehrlRumi_WELGEVONDEN)
            </option><option value="Elaeis_guineensis_EG5.1">
                Elaeis guineensis EG5.1
            </option><option value="Elaeis_guineensis_NCBI_Genome_v1.0">
                Elaeis guineensis NCBI Genome v1.0
            </option><option value="loxAfr2">
                Elephant Jul. 2008 (Broad/loxAfr2) (loxAfr2)
            </option><option value="loxAfr3">
                Elephant Jul. 2009 (Broad/loxAfr3) (loxAfr3)
            </option><option value="loxAfr1">
                Elephant May 2005 (Broad/loxAfr1) (loxAfr1)
            </option><option value="calMil1">
                Elephant shark Dec. 2013 (Callorhinchus_milii-6.1.3/calMil1) (calMil1)
            </option><option value="aptFor1">
                Emperor penguin 06 Jun 2014 (ASM69914v1/aptFor1) (aptFor1)
            </option><option value="12720">
                Enterobacter sakazakii ATCC BAA-894 (12720)
            </option><option value="ente638">
                Enterobacter sp. 638 (ente638)
            </option><option value="enteFaec_V583">
                Enterococcus faecalis V583 (enteFaec_V583)
            </option><option value="Erythranthe_guttata_JGI_v2.0">
                Erythranthe guttata JGI v2.0
            </option><option value="erytLito_HTCC2594">
                Erythrobacter litoralis HTCC2594 (erytLito_HTCC2594)
            </option><option value="16235">
                Escherichia coli 536 (16235)
            </option><option value="eschColi_APEC_O1">
                Escherichia coli APEC O1 (eschColi_APEC_O1)
            </option><option value="18083">
                Escherichia coli C str. ATCC 8739 (18083)
            </option><option value="eschColi_CFT073">
                Escherichia coli CFT073 (eschColi_CFT073)
            </option><option value="20079">
                Escherichia coli DH10B (20079)
            </option><option value="13960">
                Escherichia coli E24377A (13960)
            </option><option value="eschColi_EC4115">
                Escherichia coli EC4115 (eschColi_EC4115)
            </option><option value="13959">
                Escherichia coli HS (13959)
            </option><option value="eschColi_K12">
                Escherichia coli K12 (eschColi_K12)
            </option><option value="259">
                Escherichia coli O157:H7 EDL933 (259)
            </option><option value="eschColi_O157H7">
                Escherichia coli O157:H7 str. Sakai (eschColi_O157H7)
            </option><option value="19469">
                Escherichia coli SECEC SMS-3-5 (19469)
            </option><option value="eschColi_TW14359">
                Escherichia coli TW14359 (eschColi_TW14359)
            </option><option value="16259">
                Escherichia coli UTI89 (16259)
            </option><option value="16351">
                Escherichia coli W3110 (16351)
            </option><option value="euaGli13">
                Euarchontoglires Apr. 24. 2006 (UCSC Recon/euaGli13) (euaGli13)
            </option><option value="Eucalyptus_grandis_JGI_assembly_v1.0_annotation_v1.1">
                Eucalyptus grandis JGI assembly v1.0, annotation v1.1
            </option><option value="Eucalyptus_grandis_JGI_v2.0">
                Eucalyptus grandis JGI v2.0
            </option><option value="dicLab1">
                European sea bass Apr 2014 (seabass_V1.0/dicLab1) (dicLab1)
            </option><option value="eutHer13">
                Eutheria Apr. 24. 2006 (UCSC Recon/eutHer13) (eutHer13)
            </option><option value="evoSim1">
                Evosim Feb. 2013 (evoSim1/evoSim1) (evoSim1)
            </option><option value="musFur1">
                Ferret  Apr. 2011 (MusPutFur1.0/musFur1) (musFur1)
            </option><option value="16719">
                Fervidobacterium nodosum Rt17-B1 (16719)
            </option><option value="18981">
                Finegoldia magna ATCC 29328 (18981)
            </option><option value="Schizosaccharomyces_pombe_1.1">
                Fission Yeast (S. pombe) (Schizosaccharomyces_pombe_1.1)
            </option><option value="sebRub1">
                Flag rockfish 22 Oct-2013 (SRub1.0/sebRub1) (sebRub1)
            </option><option value="flavJohn_UW101">
                Flavobacterium johnsoniae UW101 (flavJohn_UW101)
            </option><option value="19979">
                Flavobacterium psychrophilum JIP02/86 (19979)
            </option><option value="AaloF1">
                Foshan strain genomic contigs sequences, AaloF1 assembly.
            </option><option value="Fragaria_vesca_JGI_v1.1">
                Fragaria vesca JGI v1.1
            </option><option value="Fragaria_vesca_Strawberry_Genome_1.0">
                Fragaria vesca Strawberry Genome 1.0
            </option><option value="27853">
                Francisella philomiragia subsp. philomiragia ATCC 25017 (27853)
            </option><option value="16421">
                Francisella tularensis subsp. holarctica (16421)
            </option><option value="20197">
                Francisella tularensis subsp. holarctica FTA (20197)
            </option><option value="17265">
                Francisella tularensis subsp. holarctica OSU18 (17265)
            </option><option value="16088">
                Francisella tularensis subsp. novicida U112 (16088)
            </option><option value="17375">
                Francisella tularensis subsp. tularensis FSC198 (17375)
            </option><option value="franTula_TULARENSIS">
                Francisella tularensis subsp. tularensis SCHU S4 (franTula_TULARENSIS)
            </option><option value="18459">
                Francisella tularensis subsp. tularensis WY96-3418 (18459)
            </option><option value="17403">
                Frankia alni ACN14a (17403)
            </option><option value="franCcI3">
                Frankia sp. CcI3 (franCcI3)
            </option><option value="13915">
                Frankia sp. EAN1pec (13915)
            </option><option value="fr1">
                Fugu Aug. 2002 (JGI 3.0/fr1) (fr1)
            </option><option value="fr2">
                Fugu Oct. 2004 (JGI 4.0/fr2) (fr2)
            </option><option value="fr3">
                Fugu Oct. 2011 (FUGU5/fr3) (fr3)
            </option><option value="fusoNucl">
                Fusobacterium nucleatum subsp. nucleatum ATCC 25586 (fusoNucl)
            </option><option value="hg19Patch2">
                GRCh37.p2 Aug. 2009 (GRCh37.p2/hg19Patch2) (hg19Patch2)
            </option><option value="hg38Patch2">
                GRCh38.p2 Dec 2014 (hg38Patch2)
            </option><option value="thaSir1">
                Garter snake Jun. 2015 (Thamnophis_sirtalis-6.0/thaSir1) (thaSir1)
            </option><option value="geobKaus_HTA426">
                Geobacillus kaustophilus HTA426 (geobKaus_HTA426)
            </option><option value="geobTher_NG80_2">
                Geobacillus thermodenitrificans NG80-2 (geobTher_NG80_2)
            </option><option value="geobMeta_GS15">
                Geobacter metallireducens GS-15 (geobMeta_GS15)
            </option><option value="geobSulf">
                Geobacter sulfurreducens PCA (geobSulf)
            </option><option value="geobUran_RF4">
                Geobacter uraniireducens Rf4 (geobUran_RF4)
            </option><option value="gavGan0">
                Gharial APR 2009 (preliminary/gavGan0) (gavGan0)
            </option><option value="nomLeu1">
                Gibbon Jan. 2010 (GGSC Nleu1.0/nomLeu1) (nomLeu1)
            </option><option value="nomLeu2">
                Gibbon Jun. 2011 (GGSC Nleu1.1/nomLeu2) (nomLeu2)
            </option><option value="nomLeu3">
                Gibbon Oct. 2012 (GGSC Nleu3.0/nomLeu3) (nomLeu3)
            </option><option value="gliRes13">
                Glires Apr. 24. 2006 (UCSC Recon/gliRes13) (gliRes13)
            </option><option value="gloeViol">
                Gloeobacter violaceus PCC 7421 (gloeViol)
            </option><option value="377">
                Gluconacetobacter diazotrophicus PAl 5 (377)
            </option><option value="glucOxyd_621H">
                Gluconobacter oxydans 621H (glucOxyd_621H)
            </option><option value="Glycine_max_JGI_Glyma1.1_annotation_of_the_chromosome-based_Glyma1_assembly">
                Glycine max JGI Glyma1.1 annotation of the chromosome-based Glyma1 assembly
            </option><option value="Glycine_max_JGI_Wm82.a2.v1">
                Glycine max JGI Wm82.a2.v1
            </option><option value="aquChr1">
                Golden eagle May 2014 (AquilaChrysaetos1/aquChr1) (aquChr1)
            </option><option value="aquChr2">
                Golden eagle Oct. 2014 (aquChr-1.0.2/aquChr2) (aquChr2)
            </option><option value="mesAur1">
                Golden hamster Mar 2013 (MesAur1.0/mesAur1) (mesAur1)
            </option><option value="rhiRox1">
                Golden snub-nosed monkey Oct. 2014 (Rrox_v1/rhiRox1) (rhiRox1)
            </option><option value="manVit1">
                Golden-collared manakin May 2014 (ASM69201v1/manVit1) (manVit1)
            </option><option value="gorGor2">
                Gorilla Aug. 2009 (Sanger 4/gorGor2) (gorGor2)
            </option><option value="gorGor6">
                Gorilla Aug. 2019 (Kamilah_GGO_v0/gorGor6) (gorGor6)
            </option><option value="gorGor4">
                Gorilla Dec 2014 (gorGor4.1/gorGor4) (gorGor4)
            </option><option value="gorGor5">
                Gorilla Mar. 2016 (GSMRT3/gorGor5) (gorGor5)
            </option><option value="gorGor3">
                Gorilla May 2011 (gorGor3.1/gorGor3) (gorGor3)
            </option><option value="gorGor1">
                Gorilla Oct. 2008 (Sanger 0.1/gorGor1) (gorGor1)
            </option><option value="Gossypium_raimondii_JGI_annotation_v2.1_on_assembly_v2.0">
                Gossypium raimondii JGI annotation v2.1 on assembly v2.0
            </option><option value="Gossypium_raimondii_JGI_v2.1">
                Gossypium raimondii JGI v2.1
            </option><option value="gramFors_KT0803">
                Gramella forsetii KT0803 (gramFors_KT0803)
            </option><option value="granBeth_CGDNIH1">
                Granulibacter bethesdensis CGDNIH1 (granBeth_CGDNIH1)
            </option><option value="hg19Patch10">
                Grch37.p10 Sep. 2012 (GRCh37.p10/hg19Patch10) (hg19Patch10)
            </option><option value="hg19Patch5">
                Grch37.p5 Jun. 2011 (GRCh37.p5/hg19Patch5) (hg19Patch5)
            </option><option value="hg19Patch9">
                Grch37.p9 Jul. 2012 (GRCh37.p9/hg19Patch9) (hg19Patch9)
            </option><option value="mm10Patch1">
                Grcm38.p1 Aug. 2012 (GRCm38.p1/mm10Patch1) (mm10Patch1)
            </option><option value="phaCar1">
                Great cormorant Jun 2014 (ASM70892v1/phaCar1) (phaCar1)
            </option><option value="rhiFer1">
                Greater horseshoe bat Sep. 2013 (ASM46549v1/rhiFer1) (rhiFer1)
            </option><option value="chlSab2">
                Green monkey 25 Mar-2014 (Chlorocebus_sabeus 1.1/chlSab2) (chlSab2)
            </option><option value="cheMyd1">
                Green seaturtle Mar 2013 (CheMyd_1.0/cheMyd1) (cheMyd1)
            </option><option value="cavPor3">
                Guinea pig Feb. 2008 (Broad/cavPor3) (cavPor3)
            </option><option value="poeRet1">
                Guppy Apr. 2014 (Guppy_female_1.0/poeRet1) (poeRet1)
            </option><option value="haeCon1">
                H. contortus Dec. 2009 (WormBase WS210/haeCon1) (haeCon1)
            </option><option value="38">
                Haemophilus ducreyi 35000HP (38)
            </option><option value="11752">
                Haemophilus influenzae 86-028NP (11752)
            </option><option value="16400">
                Haemophilus influenzae PittEE (16400)
            </option><option value="16401">
                Haemophilus influenzae PittGG (16401)
            </option><option value="haemInfl_KW20">
                Haemophilus influenzae Rd KW20 (haemInfl_KW20)
            </option><option value="haemSomn_129PT">
                Haemophilus somnus 129PT (haemSomn_129PT)
            </option><option value="388">
                Haemophilus somnus 2336 (388)
            </option><option value="haheChej_KCTC_2396">
                Hahella chejuensis KCTC 2396 (haheChej_KCTC_2396)
            </option><option value="halMar1">
                Haloarcula marismortui ATCC 43049 (halMar1)
            </option><option value="106">
                Halobacterium salinarum R1 (106)
            </option><option value="haloHalo1">
                Halobacterium sp. NRC-1 (haloHalo1)
            </option><option value="haloWals1">
                Haloquadratum walsbyi DSM 16790 (haloWals1)
            </option><option value="haloHalo_SL1">
                Halorhodospira halophila SL1 (haloHalo_SL1)
            </option><option value="neoSch1">
                Hawaiian monk seal Jun. 2017 (ASM220157v1/neoSch1) (neoSch1)
            </option><option value="PhumU1">
                Head Louse (Pediculus humanus) (PhumU1)
            </option><option value="eriEur1">
                Hedgehog June 2006 (Broad/eriEur1) (eriEur1)
            </option><option value="eriEur2">
                Hedgehog May 2012 (EriEur2.0/eriEur2) (eriEur2)
            </option><option value="heliAcin_SHEEBA">
                Helicobacter acinonychis str. Sheeba (heliAcin_SHEEBA)
            </option><option value="heliHepa">
                Helicobacter hepaticus ATCC 51449 (heliHepa)
            </option><option value="heliPylo_26695">
                Helicobacter pylori 26695 (heliPylo_26695)
            </option><option value="heliPylo_HPAG1">
                Helicobacter pylori HPAG1 (heliPylo_HPAG1)
            </option><option value="heliPylo_J99">
                Helicobacter pylori J99 (heliPylo_J99)
            </option><option value="13427">
                Heliobacterium modesticaldum Ice1 (13427)
            </option><option value="hermArse">
                Herminiimonas arsenicoxydans (hermArse)
            </option><option value="16523">
                Herpetosiphon aurantiacus ATCC 23779 (16523)
            </option><option value="Hevea_brasiliensis_NCBI_Genomes_v1">
                Hevea brasiliensis NCBI Genomes v1
            </option><option value="hg19LggInv">
                Hg19 lgg inversion Mar 2013 (1/hg19LggInv) (hg19LggInv)
            </option><option value="hlaRef1">
                Hlaref Apr. 2012 (hlaRefV1/hlaRef1) (hlaRef1)
            </option><option value="hlaRef2">
                Hlaref Apr. 2012 (hlaRefV2/hlaRef2) (hlaRef2)
            </option><option value="hlaRef3">
                Hlaref Apr. 2012 (hlaRefV3/hlaRef3) (hlaRef3)
            </option><option value="opiHoa1">
                Hoatzin May 2014 (ASM69207v1/opiHoa1) (opiHoa1)
            </option><option value="homIni13">
                Hominidae Apr. 24. 2006 (UCSC Recon/homIni13) (homIni13)
            </option><option value="homIni14">
                Hominidae Oct. 1. 2006 (UCSC Recon/homIni14) (homIni14)
            </option><option value="GRCh38">
                Homo Sapiens (GRCh38) Transcriptome + ERCC
            </option><option value="Homo_sapiens_nuHg19_mtrCRS">
                Homo sapiens (hg19 with mtDNA replaced with rCRS) (Homo_sapiens_nuHg19_mtrCRS)
            </option><option value="hg_g1k_v37">
                Homo sapiens b37 (hg_g1k_v37)
            </option><option value="CHM1">
                Homo sapiens hydatidiform mole assembly chm1 June 2013  (1/CHM1) (CHM1)
            </option><option value="apiMel3">
                Honeybee (Apis mellifera): apiMel3 (apiMel3)
            </option><option value="corCor1">
                Hooded crow Aug 2014 (Hooded_Crow_genome/corCor1) (corCor1)
            </option><option value="Hordeum_vulgare_Ensembl_Genomes_ASM32608v1">
                Hordeum vulgare Ensembl Genomes ASM32608v1
            </option><option value="Hordeum_vulgare_IBC_PGSB_v2">
                Hordeum vulgare IBC PGSB v2
            </option><option value="Hordeum_vulgare_MIPS">
                Hordeum vulgare MIPS
            </option><option value="equCab1">
                Horse Jan. 2007 (Broad/equCab1) (equCab1)
            </option><option value="equCab3">
                Horse Jan. 2018 (EquCab3.0/equCab3) (equCab3)
            </option><option value="equCab2">
                Horse Sep. 2007 (Broad/equCab2) (equCab2)
            </option><option value="chlUnd1">
                Houbara bustard Jun 2014 (ASM69519v1/chlUnd1) (chlUnd1)
            </option><option value="hg15">
                Human Apr. 2003 (NCBI33/hg15) (hg15)
            </option><option value="CHM13_T2T_v2.0">
                Human CHM13 2.0 (T2T Consortium) Jan. 2022
            </option><option value="hg38">
                Human Dec. 2013 (GRCh38/hg38) (hg38)
            </option><option value="hg19">
                Human Feb. 2009 (GRCh37/hg19) (hg19)
            </option><option value="CHM13_T2T_v1.0">
                Human Jan. 2021 isolate CHM13 (T2T Consortium) female
            </option><option value="hs1">
                Human Jan. 2022 (T2T CHM13v2.0/hs1) (hs1)
            </option><option value="hg16">
                Human July 2003 (NCBI34/hg16) (hg16)
            </option><option value="hg18">
                Human Mar. 2006 (NCBI36/hg18) (hg18)
            </option><option value="hg17">
                Human May 2004 (NCBI35/hg17) (hg17)
            </option><option value="15217">
                Human herpesvirus 1 (15217)
            </option><option value="repeats2">
                Human repeats Nov. 2012 (1/repeats2) (repeats2)
            </option><option value="Hydra_JCVI">
                Hydra magnipapillata str. 105 (Hydra_JCVI)
            </option><option value="hypeButy1">
                Hyperthermus butylicus DSM 5456 (hypeButy1)
            </option><option value="hyphNept_ATCC15444">
                Hyphomonas neptunium ATCC 15444 (hyphNept_ATCC15444)
            </option><option value="idioLoih_L2TR">
                Idiomarina loihiensis L2TR (idioLoih_L2TR)
            </option><option value="13914">
                Ignicoccus hospitalis KIN4/I (13914)
            </option><option value="megLyr1">
                Indian false vampire Sep. 2013 (ASM46534v1/megLyr1) (megLyr1)
            </option><option value="h7n7_360722">
                Influenza A virus (A/seal/Mass/1-SC35M/1980(H7N7))
            </option><option value="venter1">
                J. Craig Venter Sep. 2007 (HuRef/venter1) (venter1)
            </option><option value="jannCCS1">
                Jannaschia sp. CCS1 (jannCCS1)
            </option><option value="16549">
                Janthinobacterium sp. Marseille (16549)
            </option><option value="angJap1">
                Japanese eel 18 Mar-2014 (japanese_eel_genome_v1_25_oct_2011_japonica_c401b400k25m200_sspacepremiumk3a02n24_extra.final.scaffolds/angJap1) (angJap1)
            </option><option value="cotJap1">
                Japanese quail 26 Jul-2013 (Coja_1.0/cotJap1) (cotJap1)
            </option><option value="dipOrd1">
                Kangaroo rat Jul. 2008 (Broad/dipOrd1) (dipOrd1)
            </option><option value="chaVoc2">
                Killdeer Aug 2014 (ASM70802v2/chaVoc2) (chaVoc2)
            </option><option value="chaVoc1">
                Killdeer Jun 2014 (ASM70802v1/chaVoc1) (chaVoc1)
            </option><option value="orcOrc1">
                Killer whale Jan 2013 (Oorc_1.1/orcOrc1) (orcOrc1)
            </option><option value="10689">
                Kineococcus radiotolerans SRS30216 (10689)
            </option><option value="kirRef">
                Kirref Apr. 2011 (kirRef/kirRef) (kirRef)
            </option><option value="kirRefV2">
                Kirrefv2 Apr. 2011 (kirRefV2/kirRefV2) (kirRefV2)
            </option><option value="kirRefV3">
                Kirrefv3 Apr. 2011 (kirRefV3/kirRefV3) (kirRefV3)
            </option><option value="kirRefV4">
                Kirrefv4 Apr. 2011 (kirRefV4/kirRefV4) (kirRefV4)
            </option><option value="kirRefV5">
                Kirrefv5 Apr. 2011 (kirRefV5/kirRefV5) (kirRefV5)
            </option><option value="31">
                Klebsiella pneumoniae subsp. pneumoniae MGH 78578 (31)
            </option><option value="Homo_sapiens_AK1">
                Korean Man (Homo_sapiens_AK1)
            </option><option value="82">
                Lactobacillus acidophilus NCFM (82)
            </option><option value="404">
                Lactobacillus brevis ATCC 367 (404)
            </option><option value="402">
                Lactobacillus casei ATCC 334 (402)
            </option><option value="16871">
                Lactobacillus delbrueckii subsp. bulgaricus ATCC 11842 (16871)
            </option><option value="403">
                Lactobacillus delbrueckii subsp. bulgaricus ATCC BAA-365 (403)
            </option><option value="84">
                Lactobacillus gasseri ATCC 33323 (84)
            </option><option value="17811">
                Lactobacillus helveticus DPC 4571 (17811)
            </option><option value="9638">
                Lactobacillus johnsonii NCC 533 (9638)
            </option><option value="lactPlan">
                Lactobacillus plantarum WCFS1 (lactPlan)
            </option><option value="15766">
                Lactobacillus reuteri F275 (15766)
            </option><option value="13435">
                Lactobacillus sakei subsp. sakei 23K (13435)
            </option><option value="lactSali_UCC118">
                Lactobacillus salivarius UCC118 (lactSali_UCC118)
            </option><option value="18797">
                Lactococcus lactis subsp. cremoris MG1363 (18797)
            </option><option value="401">
                Lactococcus lactis subsp. cremoris SK11 (401)
            </option><option value="lactLact">
                Lactococcus lactis subsp. lactis Il1403 (lactLact)
            </option><option value="petMar3">
                Lamprey Dec. 2017 (Pmar_germline 1.0/petMar3) (petMar3)
            </option><option value="petMar1">
                Lamprey Mar. 2007 (WUGSC 3.0/petMar1) (petMar1)
            </option><option value="petMar2">
                Lamprey Sep. 2010 (WUGSC 7.0/petMar2) (petMar2)
            </option><option value="braFlo2">
                Lancelet Apr. 2008 (JGI v2.0/braFlo2) (braFlo2)
            </option><option value="braFlo1">
                Lancelet Mar. 2006 (JGI 1.0/braFlo1) (braFlo1)
            </option><option value="lauRas13">
                Laurasiatheria Apr. 24. 2006 (UCSC Recon/lauRas13) (lauRas13)
            </option><option value="lawsIntr_PHE_MN1_00">
                Lawsonia intracellularis PHE/MN1-00 (lawsIntr_PHE_MN1_00)
            </option><option value="17491">
                Legionella pneumophila str. Corby (17491)
            </option><option value="13126">
                Legionella pneumophila str. Lens (13126)
            </option><option value="13127">
                Legionella pneumophila str. Paris (13127)
            </option><option value="legiPneu_PHILADELPHIA">
                Legionella pneumophila subsp. pneumophila str. Philadelphia 1 (legiPneu_PHILADELPHIA)
            </option><option value="leifXyli_XYLI_CTCB0">
                Leifsonia xyli subsp. xyli str. CTCB07 (leifXyli_XYLI_CTCB0)
            </option><option value="lMaj5">
                Leishmania major 2005 (lMaj5)
            </option><option value="16148">
                Leptospira borgpetersenii serovar Hardjo-bovis JB197 (16148)
            </option><option value="16146">
                Leptospira borgpetersenii serovar Hardjo-bovis L550 (16146)
            </option><option value="10687">
                Leptospira interrogans serovar Copenhageni str. Fiocruz L1-130 (10687)
            </option><option value="leptInte">
                Leptospira interrogans serovar Lai str. 56601 (leptInte)
            </option><option value="20039">
                Leptothrix cholodnii SP-6 (20039)
            </option><option value="jacJac1">
                Lesser Egyptian jerboa May 2012 (JacJac1.0/jacJac1) (jacJac1)
            </option><option value="16062">
                Leuconostoc citreum KM20 (16062)
            </option><option value="leucMese_ATCC8293">
                Leuconostoc mesenteroides subsp. mesenteroides ATCC 8293 (leucMese_ATCC8293)
            </option><option value="lotGig0">
                Limpet Jul. 2013 (lotGig0/lotGig0) (lotGig0)
            </option><option value="listInno">
                Listeria innocua Clip11262 (listInno)
            </option><option value="276">
                Listeria monocytogenes EGD-e (276)
            </option><option value="85">
                Listeria monocytogenes str. 4b F2365 (85)
            </option><option value="13443">
                Listeria welshimeri serovar 6b str. SLCC5334 (13443)
            </option><option value="egrGar1">
                Little egret May 2014 (ASM68718v1/egrGar1) (egrGar1)
            </option><option value="anoCar1">
                Lizard Feb. 2007 (Broad/anoCar1) (anoCar1)
            </option><option value="anoCar2">
                Lizard May 2010 (Broad AnoCar2.0/anoCar2) (anoCar2)
            </option><option value="Lotus_japonicus_Kazusa_2.5">
                Lotus japonicus Kazusa 2.5
            </option><option value="19619">
                Lysinibacillus sphaericus C3-41 (19619)
            </option><option value="musDom2">
                M. domestica 22 Apr 2013 (Musca_domestica-2.0.2/musDom2) (musDom2)
            </option><option value="melHap1">
                M. hapla Sep. 2008 (M. hapla VW9 WS210/melHap1) (melHap1)
            </option><option value="melInc1">
                M. incognita Feb. 2008 (M. incognita WS210/melInc1) (melInc1)
            </option><option value="mmtv">
                MMTV Nov. 2009 (MMTV/mmtv) (mmtv)
            </option><option value="magnMC1">
                Magnetococcus sp. MC-1 (magnMC1)
            </option><option value="magnMagn_AMB_1">
                Magnetospirillum magneticum AMB-1 (magnMagn_AMB_1)
            </option><option value="Zea_mays_B73_RefGen_v2">
                Maize (Zea mays) (Zea_mays_B73_RefGen_v2)
            </option><option value="galVar1">
                Malayan flying lemur Jun. 2014 (G_variegatus-3.0.2/galVar1) (galVar1)
            </option><option value="anaPla1">
                Mallard duck Apr 2013 (BGI_duck_1.0/anaPla1) (anaPla1)
            </option><option value="Malus_domestica_IASMA">
                Malus domestica IASMA
            </option><option value="Malus_domestica_Penguin_Genomics_v2">
                Malus domestica Penguin Genomics v2
            </option><option value="triMan1">
                Manatee Oct. 2011 (Broad v1.0/triMan1) (triMan1)
            </option><option value="Manihot_esculenta_Cassava4">
                Manihot esculenta Cassava4
            </option><option value="Manihot_esculenta_JGI_v6.1">
                Manihot esculenta JGI v6.1
            </option><option value="mannSucc_MBEL55E">
                Mannheimia succiniciproducens MBEL55E (mannSucc_MBEL55E)
            </option><option value="marVir1">
                Marburg virus Sep. 2014 (NC_024781.1/marVir1) (marVir1)
            </option><option value="Marchantia_polymorpha_JGI_v3.1">
                Marchantia polymorpha JGI v3.1
            </option><option value="mariMari_MCS10">
                Maricaulis maris MCS10 (mariMari_MCS10)
            </option><option value="mariAqua_VT8">
                Marinobacter aquaeolei VT8 (mariAqua_VT8)
            </option><option value="17445">
                Marinomonas sp. MWYL1 (17445)
            </option><option value="calJac1">
                Marmoset June 2007 (WUGSC 2.0.2/calJac1) (calJac1)
            </option><option value="calJac3">
                Marmoset March 2009 (WUGSC 3.2/calJac3) (calJac3)
            </option><option value="calJac4">
                Marmoset May 2020 (Callithrix_jacchus_cj1700_1.1/calJac4) (calJac4)
            </option><option value="oryLat1">
                Medaka Apr. 2006 (NIG/UT MEDAKA1/oryLat1) (oryLat1)
            </option><option value="oryLat2">
                Medaka Oct. 2005 (NIG/UT MEDAKA1/oryLat2) (oryLat2)
            </option><option value="Medicago_truncatula_JCVI_4.0">
                Medicago truncatula JCVI 4.0
            </option><option value="Medicago_truncatula_JGI_Mt4.0v1">
                Medicago truncatula JGI Mt4.0v1
            </option><option value="geoFor1">
                Medium ground finch Apr. 2012 (GeoFor_1.0/geoFor1) (geoFor1)
            </option><option value="pteVam1">
                Megabat Jul. 2008 (Broad/pteVam1) (pteVam1)
            </option><option value="mesoFlor_L1">
                Mesoplasma florum L1 (mesoFlor_L1)
            </option><option value="mesoLoti">
                Mesorhizobium loti MAFF303099 (mesoLoti)
            </option><option value="10690">
                Mesorhizobium sp. BNC1 (10690)
            </option><option value="metaSedu">
                Metallosphaera sedula DSM 5348 (metaSedu)
            </option><option value="methSmit1">
                Methanobrevibacter smithii ATCC 35061 (methSmit1)
            </option><option value="methJann1">
                Methanocaldococcus jannaschii DSM 2661 (methJann1)
            </option><option value="methBurt2">
                Methanococcoides burtonii DSM 6242 (methBurt2)
            </option><option value="methAeol1">
                Methanococcus aeolicus Nankai-3 (methAeol1)
            </option><option value="methMari_C5_1">
                Methanococcus maripaludis C5 (methMari_C5_1)
            </option><option value="19639">
                Methanococcus maripaludis C6 (19639)
            </option><option value="methMari_C7">
                Methanococcus maripaludis C7 (methMari_C7)
            </option><option value="metMar1">
                Methanococcus maripaludis S2 (metMar1)
            </option><option value="methVann1">
                Methanococcus vannielii SB (methVann1)
            </option><option value="methLabrZ_1">
                Methanocorpusculum labreanum Z (methLabrZ_1)
            </option><option value="mculMari1">
                Methanoculleus marisnigri JR1 (mculMari1)
            </option><option value="methKand1">
                Methanopyrus kandleri AV19 (methKand1)
            </option><option value="methTherPT1">
                Methanosaeta thermophila PT (methTherPT1)
            </option><option value="metAce1">
                Methanosarcina acetivorans C2A (metAce1)
            </option><option value="methBark1">
                Methanosarcina barkeri str. Fusaro (methBark1)
            </option><option value="methMaze1">
                Methanosarcina mazei Go1 (methMaze1)
            </option><option value="methStad1">
                Methanosphaera stadtmanae DSM 3091 (methStad1)
            </option><option value="methHung1">
                Methanospirillum hungatei JF-1 (methHung1)
            </option><option value="methTher1">
                Methanothermobacter thermautotrophicus str. Delta H (methTher1)
            </option><option value="methPetr_PM1">
                Methylibium petroleiphilum PM1 (methPetr_PM1)
            </option><option value="methFlag_KT">
                Methylobacillus flagellatus KT (methFlag_KT)
            </option><option value="18637">
                Methylobacterium extorquens PA1 (18637)
            </option><option value="18817">
                Methylobacterium radiotolerans JCM 2831 (18817)
            </option><option value="18809">
                Methylobacterium sp. 4-46 (18809)
            </option><option value="methCaps_BATH">
                Methylococcus capsulatus str. Bath (methCaps_BATH)
            </option><option value="astMex1">
                Mexican tetra (cavefish) Apr 2013 (Astyanax_mexicanus-1.0.2/astMex1) (astMex1)
            </option><option value="mhcRefnoNs2">
                Mhcref Apr. 2011 (mhcRefnoNsV2/mhcRefnoNs2) (mhcRefnoNs2)
            </option><option value="mhcRef">
                Mhcref Jan. 2011 (mhcRefV1/mhcRef) (mhcRef)
            </option><option value="mhcRefnoNs10">
                Mhcref July 2011 (mhcRefnoNsV10/mhcRefnoNs10) (mhcRefnoNs10)
            </option><option value="mhcRefnoNs11">
                Mhcref July 2011 (mhcRefnoNsV11/mhcRefnoNs11) (mhcRefnoNs11)
            </option><option value="mhcRefnoNs12">
                Mhcref July 2011 (mhcRefnoNsV12/mhcRefnoNs12) (mhcRefnoNs12)
            </option><option value="mhcRefnoNs13">
                Mhcref July 2011 (mhcRefnoNsV13/mhcRefnoNs13) (mhcRefnoNs13)
            </option><option value="mhcRefnoNs14">
                Mhcref July 2011 (mhcRefnoNsV14/mhcRefnoNs14) (mhcRefnoNs14)
            </option><option value="mhcRefnoNs15">
                Mhcref July 2011 (mhcRefnoNsV15/mhcRefnoNs15) (mhcRefnoNs15)
            </option><option value="mhcRefnoNs16">
                Mhcref July 2011 (mhcRefnoNsV16/mhcRefnoNs16) (mhcRefnoNs16)
            </option><option value="mhcRefnoNs17">
                Mhcref July 2011 (mhcRefnoNsV17/mhcRefnoNs17) (mhcRefnoNs17)
            </option><option value="mhcRefnoNs5">
                Mhcref July 2011 (mhcRefnoNsV5/mhcRefnoNs5) (mhcRefnoNs5)
            </option><option value="mhcRefnoNs6">
                Mhcref July 2011 (mhcRefnoNsV6/mhcRefnoNs6) (mhcRefnoNs6)
            </option><option value="mhcRefnoNs7">
                Mhcref July 2011 (mhcRefnoNsV7/mhcRefnoNs7) (mhcRefnoNs7)
            </option><option value="mhcRefnoNs8">
                Mhcref July 2011 (mhcRefnoNsV8/mhcRefnoNs8) (mhcRefnoNs8)
            </option><option value="mhcRefnoNs9">
                Mhcref July 2011 (mhcRefnoNsV8/mhcRefnoNs9) (mhcRefnoNs9)
            </option><option value="mhcRefnoNs">
                Mhcref June 2011 (mhcRefnoNsV1/mhcRefnoNs) (mhcRefnoNs)
            </option><option value="mhcRefnoNs3">
                Mhcref June 2011 (mhcRefnoNsV3/mhcRefnoNs3) (mhcRefnoNs3)
            </option><option value="mhcRefnoNs4">
                Mhcref June 2011 (mhcRefnoNsV4/mhcRefnoNs4) (mhcRefnoNs4)
            </option><option value="mhcRefnoNs22">
                Mhcref Mar. 2012 (mhcRefnoNsV22/mhcRefnoNs22) (mhcRefnoNs22)
            </option><option value="mhcRefnoNs23">
                Mhcref Mar. 2012 (mhcRefnoNsV23/mhcRefnoNs23) (mhcRefnoNs23)
            </option><option value="mhcRefnoNs24">
                Mhcref Mar. 2012 (mhcRefnoNsV24/mhcRefnoNs24) (mhcRefnoNs24)
            </option><option value="mhcRefnoNs18">
                Mhcref Oct. 2011 (mhcRefnoNsV18/mhcRefnoNs18) (mhcRefnoNs18)
            </option><option value="mhcRefnoNs19">
                Mhcref Oct. 2011 (mhcRefnoNsV19/mhcRefnoNs19) (mhcRefnoNs19)
            </option><option value="mhcRefnoNs20">
                Mhcref Oct. 2011 (mhcRefnoNsV20/mhcRefnoNs20) (mhcRefnoNs20)
            </option><option value="mhcRefnoNs21">
                Mhcref Oct. 2011 (mhcRefnoNsV21/mhcRefnoNs21) (mhcRefnoNs21)
            </option><option value="myoLuc2">
                Microbat Jul. 2010 (Broad Institute Myoluc2.0/myoLuc2) (myoLuc2)
            </option><option value="myoLuc1">
                Microbat Mar. 2006 (Broad/myoLuc1) (myoLuc1)
            </option><option value="27835">
                Microcystis aeruginosa NIES-843 (27835)
            </option><option value="Micromonas_commoda_JGI_v3.0">
                Micromonas commoda JGI v3.0
            </option><option value="balAcu1">
                Minke whale Oct. 2013 (BalAcu1.0/balAcu1) (balAcu1)
            </option><option value="mm10Strains1">
                Mm10haps Dec. 2011 (Mouse strains/mm10Strains1) (mm10Strains1)
            </option><option value="mpxvRivers">
                Monkeypox virus MPXV-M5312_HM12_Rivers (MT903340.1/GCF_014621545.1) (mpxvRivers)
            </option><option value="moorTher_ATCC39073">
                Moorella thermoacetica ATCC 39073 (moorTher_ATCC39073)
            </option><option value="AaegL1">
                Mosquito (Aedes aegypti) (AaegL1)
            </option><option value="AgamP3">
                Mosquito (Anopheles gambiae) (AgamP3)
            </option><option value="CpipJ1">
                Mosquito (Culex quinquefasciatus) (CpipJ1)
            </option><option value="mm10">
                Mouse (Mus Musculus): mm10 Full
            </option><option value="mm7">
                Mouse Aug. 2005 (NCBI35/mm7) (mm7)
            </option><option value="mm8">
                Mouse Feb. 2006 (NCBI36/mm8) (mm8)
            </option><option value="mm9">
                Mouse July 2007 (NCBI37/mm9) (mm9)
            </option><option value="mm39">
                Mouse Jun. 2020 (GRCm39/mm39) (mm39)
            </option><option value="mm5">
                Mouse May 2004 (mm5)
            </option><option value="micMur1">
                Mouse lemur Jul. 2007 (Broad/micMur1) (micMur1)
            </option><option value="micMur2">
                Mouse lemur May 2015 (Mouse lemur/micMur2) (micMur2)
            </option><option value="Musa_acuminata_Banana_Genome_v1.0">
                Musa acuminata Banana Genome v1.0
            </option><option value="Musa_acuminata_Banana_Genome_v2.0">
                Musa acuminata Banana Genome v2.0
            </option><option value="Musa_acuminata_Genescope-Cirad">
                Musa acuminata Genescope-Cirad
            </option><option value="15691">
                Mycobacterium abscessus (15691)
            </option><option value="88">
                Mycobacterium avium 104 (88)
            </option><option value="91">
                Mycobacterium avium subsp. paratuberculosis K-10 (91)
            </option><option value="89">
                Mycobacterium bovis AF2122/97 (89)
            </option><option value="18059">
                Mycobacterium bovis BCG str. Pasteur 1173P2 (18059)
            </option><option value="15760">
                Mycobacterium gilvum PYR-GCK (15760)
            </option><option value="90">
                Mycobacterium leprae TN (90)
            </option><option value="92">
                Mycobacterium smegmatis str. MC2 155 (92)
            </option><option value="16079">
                Mycobacterium sp. JLS (16079)
            </option><option value="16081">
                Mycobacterium sp. KMS (16081)
            </option><option value="15762">
                Mycobacterium sp. MCS (15762)
            </option><option value="223">
                Mycobacterium tuberculosis CDC1551 (223)
            </option><option value="15642">
                Mycobacterium tuberculosis F11 (15642)
            </option><option value="18883">
                Mycobacterium tuberculosis H37Ra (18883)
            </option><option value="mycoTube_H37RV">
                Mycobacterium tuberculosis H37Rv (mycoTube_H37RV)
            </option><option value="16230">
                Mycobacterium ulcerans Agy99 (16230)
            </option><option value="15761">
                Mycobacterium vanbaalenii PYR-1 (15761)
            </option><option value="16095">
                Mycoplasma agalactiae PG2 (16095)
            </option><option value="16208">
                Mycoplasma capricolum subsp. capricolum ATCC 27343 (16208)
            </option><option value="409">
                Mycoplasma gallisepticum R (409)
            </option><option value="mycoGeni">
                Mycoplasma genitalium G37 (mycoGeni)
            </option><option value="13120">
                Mycoplasma hyopneumoniae 232 (13120)
            </option><option value="10639">
                Mycoplasma hyopneumoniae 7448 (10639)
            </option><option value="10675">
                Mycoplasma hyopneumoniae J (10675)
            </option><option value="10697">
                Mycoplasma mobile 163K (10697)
            </option><option value="10616">
                Mycoplasma mycoides subsp. mycoides SC str. PG1 (10616)
            </option><option value="176">
                Mycoplasma penetrans HF-2 (176)
            </option><option value="99">
                Mycoplasma pneumoniae M129 (99)
            </option><option value="100">
                Mycoplasma pulmonis UAB CTIP (100)
            </option><option value="10676">
                Mycoplasma synoviae 53 (10676)
            </option><option value="myxoXant_DK_1622">
                Myxococcus xanthus DK 1622 (myxoXant_DK_1622)
            </option><option value="hetGla2">
                Naked mole-rat Jan. 2012 (Broad HetGla_female_1.0/hetGla2) (hetGla2)
            </option><option value="hetGla1">
                Naked mole-rat Jul. 2011 (BGI HetGla_1.0/hetGla1) (hetGla1)
            </option><option value="nanEqu1">
                Nanoarchaeum equitans Kin4-M (nanEqu1)
            </option><option value="natrPhar1">
                Natronomonas pharaonis DSM 2160 (natrPhar1)
            </option><option value="neisGono_FA1090_1">
                Neisseria gonorrhoeae FA 1090 (neisGono_FA1090_1)
            </option><option value="16393">
                Neisseria meningitidis 053442 (16393)
            </option><option value="neisMeni_FAM18_1">
                Neisseria meningitidis FAM18 (neisMeni_FAM18_1)
            </option><option value="neisMeni_MC58_1">
                Neisseria meningitidis MC58 (neisMeni_MC58_1)
            </option><option value="neisMeni_Z2491_1">
                Neisseria meningitidis Z2491 (neisMeni_Z2491_1)
            </option><option value="Nelumbo_nucifera_LOTUS-DB_v1.1">
                Nelumbo nucifera LOTUS-DB v1.1
            </option><option value="neorSenn_MIYAYAMA">
                Neorickettsia sennetsu str. Miyayama (neorSenn_MIYAYAMA)
            </option><option value="oreNil1">
                Nile tilapia Jan. 2011 (Broad oreNil1.0/oreNil1) (oreNil1)
            </option><option value="oreNil2">
                Nile tilapia Jan. 2011 (Broad oreNil1.1/oreNil2) (oreNil2)
            </option><option value="18963">
                Nitratiruptor sp. SB155-2 (18963)
            </option><option value="13473">
                Nitrobacter hamburgensis X14 (13473)
            </option><option value="nitrWino_NB_255">
                Nitrobacter winogradskyi Nb-255 (nitrWino_NB_255)
            </option><option value="nitrOcea_ATCC19707">
                Nitrosococcus oceani ATCC 19707 (nitrOcea_ATCC19707)
            </option><option value="nitrEuro">
                Nitrosomonas europaea ATCC 19718 (nitrEuro)
            </option><option value="13913">
                Nitrosomonas eutropha C91 (13913)
            </option><option value="19265">
                Nitrosopumilus maritimus SCM1 (19265)
            </option><option value="nitrMult_ATCC25196">
                Nitrosospira multiformis ATCC 25196 (nitrMult_ATCC25196)
            </option><option value="nocaFarc_IFM10152">
                Nocardia farcinica IFM 10152 (nocaFarc_IFM10152)
            </option><option value="nocaJS61">
                Nocardioides sp. JS614 (nocaJS61)
            </option><option value="nonAfr13">
                Non-Afrotheria Apr. 24. 2006 (UCSC Recon/nonAfr13) (nonAfr13)
            </option><option value="merNub1">
                Northern carmine bee-eater May 2014 (ASM69184v1/merNub1) (merNub1)
            </option><option value="fulGla1">
                Northern fulmar May 2014 (ASM69083v1/fulGla1) (fulGla1)
            </option><option value="esoLuc1">
                Northern pike Jul 2014 (EsoLuc1.0/esoLuc1) (esoLuc1)
            </option><option value="nostSp">
                Nostoc sp. PCC 7120 (nostSp)
            </option><option value="calMil0">
                Notyet Oct. 2011 (notYet1/calMil0) (calMil0)
            </option><option value="novoArom_DSM12444">
                Novosphingobium aromaticivorans DSM 12444 (novoArom_DSM12444)
            </option><option value="oceaIhey">
                Oceanobacillus iheyensis HTE831 (oceaIhey)
            </option><option value="19485">
                Ochrobactrum anthropi ATCC 49188 (19485)
            </option><option value="oenoOeni_PSU_1">
                Oenococcus oeni PSU-1 (oenoOeni_PSU_1)
            </option><option value="od01">
                Oikopleura dioica (od01)
            </option><option value="onioYell_PHYTOPLASMA">
                Onion yellows phytoplasma OY-M (onioYell_PHYTOPLASMA)
            </option><option value="monDom4">
                Opossum Jan. 2006 (Broad/monDom4) (monDom4)
            </option><option value="monDom1">
                Opossum Oct. 2004 (Broad prelim/monDom1) (monDom1)
            </option><option value="monDom5">
                Opossum Oct. 2006 (Broad/monDom5) (monDom5)
            </option><option value="ponAbe3">
                Orangutan Jan. 2018 (Susie_PABv2/ponAbe3) (ponAbe3)
            </option><option value="ponAbe2">
                Orangutan July 2007 (WUGSC 2.0.2/ponAbe2) (ponAbe2)
            </option><option value="orieTsut_BORYONG">
                Orientia tsutsugamushi str. Boryong (orieTsut_BORYONG)
            </option><option value="Oropetium_thomaeum_JGI_v1.0">
                Oropetium thomaeum JGI v1.0
            </option><option value="Oryza_brachyantha_Ensembl_Plants_v1.4b">
                Oryza brachyantha Ensembl Plants v1.4b
            </option><option value="Oryza_sativa_ssp._indica_9311_BGF_2005">
                Oryza sativa ssp. indica 9311_BGF_2005
            </option><option value="Oryza_sativa_ssp._indica_MBKBASE_v1.0">
                Oryza sativa ssp. indica MBKBASE v1.0
            </option><option value="Oryza_sativa_ssp._japonica_JGI_v7.0">
                Oryza sativa ssp. japonica JGI v7.0
            </option><option value="Oryza_sativa_ssp._japonica_MSU_RGAP_7">
                Oryza sativa ssp. japonica MSU RGAP 7
            </option><option value="Ostreococcus_lucimarinus_JGI_2.0">
                Ostreococcus lucimarinus JGI 2.0
            </option><option value="Ostreococcus_lucimarinus_JGI_v2.0_assembly_and_annotation">
                Ostreococcus lucimarinus JGI v2.0 assembly and annotation
            </option><option value="strCam1">
                Ostrich 06 Jun-2014 (ASM69896v1/strCam1) (strCam1)
            </option><option value="falciparum">
                P. falciparum Plasmodium falciparum (?/falciparum) (falciparum)
            </option><option value="priPac2">
                P. pacificus Dec. 2008 (WUGSC 5.0.1/priPac2) (priPac2)
            </option><option value="priPac1">
                P. pacificus Feb. 2007 (WUGSC 5.0/priPac1) (priPac1)
            </option><option value="odoRosDiv1">
                Pacific walrus Jan 2013 (Oros_1.0/odoRosDiv1) (odoRosDiv1)
            </option><option value="chrPic1">
                Painted turtle Dec. 2011 (v3.0.1/chrPic1) (chrPic1)
            </option><option value="chrPic2">
                Painted turtle Mar. 2014 (v3.0.3/chrPic2) (chrPic2)
            </option><option value="ailMel1">
                Panda Dec. 2009 (BGI-Shenzhen 1.0/ailMel1) (ailMel1)
            </option><option value="13485">
                Parabacteroides distasonis ATCC 8503 (13485)
            </option><option value="paraDeni_PD1222">
                Paracoccus denitrificans PD1222 (paraDeni_PD1222)
            </option><option value="ptePar1">
                Parnell's mustached bat Sep. 2013 (ASM46540v1/ptePar1) (ptePar1)
            </option><option value="amaVit1">
                Parrot Jan. 2013 (AV1/amaVit1) (amaVit1)
            </option><option value="17639">
                Parvibaculum lavamentivorans DS-1 (17639)
            </option><option value="pastMult">
                Pasteurella multocida subsp. multocida str. Pm70 (pastMult)
            </option><option value="erwiCaro_ATROSEPTICA">
                Pectobacterium atrosepticum SCRI1043 (erwiCaro_ATROSEPTICA)
            </option><option value="pediPent_ATCC25745">
                Pediococcus pentosaceus ATCC 25745 (pediPent_ATCC25745)
            </option><option value="peloCarb">
                Pelobacter carbinolicus DSM 2380 (peloCarb)
            </option><option value="13384">
                Pelobacter propionicus DSM 2379 (13384)
            </option><option value="peloLute_DSM273">
                Pelodictyon luteolum DSM 273 (peloLute_DSM273)
            </option><option value="peloTher_SI">
                Pelotomaculum thermopropionicum SI (peloTher_SI)
            </option><option value="falPer1">
                Peregrine falcon Feb 2013 (F_peregrinus_v1.0/falPer1) (falPer1)
            </option><option value="17679">
                Petrotoga mobilis SJ95 (17679)
            </option><option value="Petunia_axillaris_Sol_Genomics_v1.6.2">
                Petunia axillaris Sol Genomics v1.6.2
            </option><option value="Phalaenopsis_equestris_Genomics_v1.0">
                Phalaenopsis equestris Genomics v1.0
            </option><option value="photProf_SS9">
                Photobacterium profundum SS9 (photProf_SS9)
            </option><option value="photLumi">
                Photorhabdus luminescens subsp. laumondii TTO1 (photLumi)
            </option><option value="Phyllostachys_edulis_NCGR_v1.0">
                Phyllostachys edulis NCGR v1.0
            </option><option value="Physcomitrella_patens_JGI_assembly_release_v1.1_and_COSMOSS_annotation_v1.6">
                Physcomitrella patens JGI assembly release v1.1 and COSMOSS annotation v1.6
            </option><option value="Physcomitrella_patens_JGI_v3.3">
                Physcomitrella patens JGI v3.3
            </option><option value="picrTorr1">
                Picrophilus torridus DSM 9790 (picrTorr1)
            </option><option value="susScr3">
                Pig Aug. 2011 (SGSC Sscrofa10.2/susScr3) (susScr3)
            </option><option value="susScr11">
                Pig Feb. 2017 (Sscrofa11.1/susScr11) (susScr11)
            </option><option value="Sscrofa9.58">
                Pig May 2010 (Sscrofa9.58)
            </option><option value="susScr2">
                Pig Nov. 2009 (SGSC Sscrofa9.2/susScr2) (susScr2)
            </option><option value="ochPri2">
                Pika Jul. 2008 (Broad/ochPri2) (ochPri2)
            </option><option value="ochPri3">
                Pika May 2012 (OchPri3.0/ochPri3) (ochPri3)
            </option><option value="plasFalc3D7V12">
                Plasmodium falciparum 3d7 10 Sept 2014 (V12.0/plasFalc3D7V12) (plasFalc3D7V12)
            </option><option value="ornAna2">
                Platypus Feb. 2007 (ASM227v2/ornAna2) (ornAna2)
            </option><option value="ornAna1">
                Platypus Mar. 2007 (WUGSC 5.0.1/ornAna1) (ornAna1)
            </option><option value="ursMar1">
                Polar bear 09 May-2014 (UrsMar_1.0/ursMar1) (ursMar1)
            </option><option value="13418">
                Polaromonas naphthalenivorans CJ2 (13418)
            </option><option value="polaJS66">
                Polaromonas sp. JS666 (polaJS66)
            </option><option value="polyQLWP">
                Polynucleobacter sp. QLW-P1DMWA-1 (polyQLWP)
            </option><option value="Ptrichocarpa_156">
                Poplar (Populus trichocarpa) (Ptrichocarpa_156)
            </option><option value="Populus_trichocarpa_JGI_assembly_release_v3.0_annotation_v3.0">
                Populus trichocarpa JGI assembly release v3.0, annotation v3.0
            </option><option value="Populus_trichocarpa_JGI_v3.1">
                Populus trichocarpa JGI v3.1
            </option><option value="porpGing_W83">
                Porphyromonas gingivalis W83 (porpGing_W83)
            </option><option value="perManBai1">
                Prairie deer mouse 03 Dec-2013 (Pman_1.0/perManBai1) (perManBai1)
            </option><option value="micOch1">
                Prairie vole Oct 2012 (MicOch1.0/micOch1) (micOch1)
            </option><option value="priMat13">
                Primate Apr. 24. 2006 (UCSC Recon/priMat13) (priMat13)
            </option><option value="neoBri1">
                Princess of Burundi May 2011 (NeoBri1.0/neoBri1) (neoBri1)
            </option><option value="nasLar1">
                Proboscis monkey Nov. 2014 (Charlie1.0/nasLar1) (nasLar1)
            </option><option value="13548">
                Prochlorococcus marinus str. AS9601 (13548)
            </option><option value="13551">
                Prochlorococcus marinus str. MIT 9211 (13551)
            </option><option value="18633">
                Prochlorococcus marinus str. MIT 9215 (18633)
            </option><option value="15746">
                Prochlorococcus marinus str. MIT 9301 (15746)
            </option><option value="13496">
                Prochlorococcus marinus str. MIT 9303 (13496)
            </option><option value="13910">
                Prochlorococcus marinus str. MIT 9312 (13910)
            </option><option value="220">
                Prochlorococcus marinus str. MIT 9313 (220)
            </option><option value="13617">
                Prochlorococcus marinus str. MIT 9515 (13617)
            </option><option value="15660">
                Prochlorococcus marinus str. NATL1A (15660)
            </option><option value="13911">
                Prochlorococcus marinus str. NATL2A (13911)
            </option><option value="procMari_CCMP1375">
                Prochlorococcus marinus subsp. marinus str. CCMP1375 (procMari_CCMP1375)
            </option><option value="213">
                Prochlorococcus marinus subsp. pastoris str. CCMP1986 (213)
            </option><option value="propAcne_KPA171202">
                Propionibacterium acnes KPA171202 (propAcne_KPA171202)
            </option><option value="12607">
                Prosthecochloris vibrioformis DSM 265 (12607)
            </option><option value="Prunus_persica_JGI_release_v1.0">
                Prunus persica JGI release v1.0
            </option><option value="Prunus_persica_JGI_v2.1">
                Prunus persica JGI v2.1
            </option><option value="equPrz1">
                Przewalski horse Jun 2014 (Burgud/equPrz1) (equPrz1)
            </option><option value="13454">
                Pseudoalteromonas atlantica T6c (13454)
            </option><option value="pseuHalo_TAC125">
                Pseudoalteromonas haloplanktis TAC125 (pseuHalo_TAC125)
            </option><option value="16720">
                Pseudomonas aeruginosa PA7 (16720)
            </option><option value="pseuAeru">
                Pseudomonas aeruginosa PAO1 (pseuAeru)
            </option><option value="386">
                Pseudomonas aeruginosa UCBPP-PA14 (386)
            </option><option value="16800">
                Pseudomonas entomophila L48 (16800)
            </option><option value="327">
                Pseudomonas fluorescens Pf-5 (327)
            </option><option value="12">
                Pseudomonas fluorescens PfO-1 (12)
            </option><option value="17457">
                Pseudomonas mendocina ymp (17457)
            </option><option value="13909">
                Pseudomonas putida F1 (13909)
            </option><option value="17629">
                Pseudomonas putida GB-1 (17629)
            </option><option value="267">
                Pseudomonas putida KT2440 (267)
            </option><option value="17053">
                Pseudomonas putida W619 (17053)
            </option><option value="16817">
                Pseudomonas stutzeri A1501 (16817)
            </option><option value="12416">
                Pseudomonas syringae pv. phaseolicola 1448A (12416)
            </option><option value="323">
                Pseudomonas syringae pv. syringae B728a (323)
            </option><option value="359">
                Pseudomonas syringae pv. tomato str. DC3000 (359)
            </option><option value="psycArct_273_4">
                Psychrobacter arcticus 273-4 (psycArct_273_4)
            </option><option value="13920">
                Psychrobacter cryohalolentis K5 (13920)
            </option><option value="15759">
                Psychrobacter sp. PRwf-1 (15759)
            </option><option value="psycIngr_37">
                Psychromonas ingrahamii 37 (psycIngr_37)
            </option><option value="punNye1">
                Pundamilia nyererei Oct 2011 (PunNye1.0/punNye1) (punNye1)
            </option><option value="Spur_v2.6">
                Purple Sea Urchin (Strongylocentrotus purpuratus) v2.6 (Spur_v2.6)
            </option><option value="pyrAer1">
                Pyrobaculum aerophilum str. IM2 (pyrAer1)
            </option><option value="pyroArse1">
                Pyrobaculum arsenaticum DSM 13514 (pyroArse1)
            </option><option value="pyroCali1">
                Pyrobaculum calidifontis JCM 11548 (pyroCali1)
            </option><option value="pyroIsla1">
                Pyrobaculum islandicum DSM 4184 (pyroIsla1)
            </option><option value="pyrAby1">
                Pyrococcus abyssi GE5 (pyrAby1)
            </option><option value="pyrFur2">
                Pyrococcus furiosus DSM 3638 (pyrFur2)
            </option><option value="pyrHor1">
                Pyrococcus horikoshii OT3 (pyrHor1)
            </option><option value="Pyrus_bretschneideri_Pear_Genome_Project_v1.0">
                Pyrus bretschneideri Pear Genome Project v1.0
            </option><option value="oryCun2">
                Rabbit Apr. 2009 (Broad/oryCun2) (oryCun2)
            </option><option value="oryCun1">
                Rabbit May 2005 (Broad/oryCun1) (oryCun1)
            </option><option value="13603">
                Ralstonia eutropha H16 (13603)
            </option><option value="ralsEutr_JMP134">
                Ralstonia eutropha JMP134 (ralsEutr_JMP134)
            </option><option value="250">
                Ralstonia metallidurans CH34 (250)
            </option><option value="ralsSola">
                Ralstonia solanacearum GMI1000 (ralsSola)
            </option><option value="rn6">
                Rat Jul. 2014 (RGSC 6.0/rn6) (rn6)
            </option><option value="rn3">
                Rat June 2003 (Baylor 3.1/rn3) (rn3)
            </option><option value="rn5">
                Rat Mar. 2012 (RGSC 5.0/rn5) (rn5)
            </option><option value="rn4">
                Rat Nov. 2004 (Baylor 3.4/rn4) (rn4)
            </option><option value="rn7">
                Rat Nov. 2020 (mRatBN7.2/rn7) (rn7)
            </option><option value="Tcas_3.0">
                Red Flour Beetle (Tribolium castaneum) (Tcas_3.0)
            </option><option value="tauEry1">
                Red crested turaco Jun 2014 (ASM70936v1/tauEry1) (tauEry1)
            </option><option value="carCri1">
                Red-legged seriema May 2014 (ASM69053v1/carCri1) (carCri1)
            </option><option value="gavSte1">
                Red-throated loon May 2014 (ASM69087v1/gavSte1) (gavSte1)
            </option><option value="19227">
                Renibacterium salmoninarum ATCC 33209 (19227)
            </option><option value="repBase0">
                RepBase Jun. 2012 (hg19/repBase0) (repBase0)
            </option><option value="repBase1">
                RepBase Sep. 2012 (17.08/repBase1) (repBase1)
            </option><option value="resEbo1">
                Reston ebolavirus Aug. 2014 (NC_004161.1/resEbo1) (resEbo1)
            </option><option value="rheMac10">
                Rhesus Feb. 2019 (Mmul_10/rheMac10) (rheMac10)
            </option><option value="rheMac2">
                Rhesus Jan. 2006 (MGSC Merged 1.0/rheMac2) (rheMac2)
            </option><option value="rheMac7">
                Rhesus Nov. 2014 (UMD MacaM_Assembly_v7/rheMac7) (rheMac7)
            </option><option value="rheMac8">
                Rhesus Nov. 2015 (BCM Mmul_8.0.1/rheMac8) (rheMac8)
            </option><option value="rheMac3">
                Rhesus Oct. 2010 (BGI CR_1.0/rheMac3) (rheMac3)
            </option><option value="bucRhi1">
                Rhinoceros hornbill Jun 2014 (ASM71030v1/bucRhi1) (bucRhi1)
            </option><option value="rhizEtli_CFN_42">
                Rhizobium etli CFN 42 (rhizEtli_CFN_42)
            </option><option value="344">
                Rhizobium leguminosarum bv. viciae 3841 (344)
            </option><option value="rhodSpha_2_4_1">
                Rhodobacter sphaeroides 2.4.1 (rhodSpha_2_4_1)
            </option><option value="15755">
                Rhodobacter sphaeroides ATCC 17025 (15755)
            </option><option value="15754">
                Rhodobacter sphaeroides ATCC 17029 (15754)
            </option><option value="rhodRHA1">
                Rhodococcus sp. RHA1 (rhodRHA1)
            </option><option value="13908">
                Rhodoferax ferrireducens T118 (13908)
            </option><option value="pireSp">
                Rhodopirellula baltica SH 1 (pireSp)
            </option><option value="15751">
                Rhodopseudomonas palustris BisA53 (15751)
            </option><option value="15750">
                Rhodopseudomonas palustris BisB18 (15750)
            </option><option value="15749">
                Rhodopseudomonas palustris BisB5 (15749)
            </option><option value="rhodPalu_CGA009">
                Rhodopseudomonas palustris CGA009 (rhodPalu_CGA009)
            </option><option value="15747">
                Rhodopseudomonas palustris HaA2 (15747)
            </option><option value="rhodRubr_ATCC11170">
                Rhodospirillum rubrum ATCC 11170 (rhodRubr_ATCC11170)
            </option><option value="oryza_sativa_japonica_nipponbare_IRGSP4.0">
                Rice (Oryza sativa ssp. japonica var. Nipponbare) (oryza_sativa_japonica_nipponbare_IRGSP4.0)
            </option><option value="Ricinus_communis_JCVI_1.0">
                Ricinus communis JCVI 1.0
            </option><option value="Ricinus_communis_JGI_v0.1">
                Ricinus communis JGI v0.1
            </option><option value="12953">
                Rickettsia akari str. Hartford (12953)
            </option><option value="17237">
                Rickettsia bellii OSU 85-389 (17237)
            </option><option value="rickBell_RML369_C">
                Rickettsia bellii RML369-C (rickBell_RML369_C)
            </option><option value="12952">
                Rickettsia canadensis str. McKiel (12952)
            </option><option value="42">
                Rickettsia conorii str. Malish 7 (42)
            </option><option value="13884">
                Rickettsia felis URRWXCal2 (13884)
            </option><option value="18271">
                Rickettsia massiliae MTU5 (18271)
            </option><option value="43">
                Rickettsia prowazekii str. Madrid E (43)
            </option><option value="9636">
                Rickettsia rickettsii str. 'Sheila Smith' (9636)
            </option><option value="19943">
                Rickettsia rickettsii str. Iowa (19943)
            </option><option value="10679">
                Rickettsia typhi str. Wilmington (10679)
            </option><option value="acaChl1">
                Rifleman 27 May 2014 (ASM69581v1/acaChl1) (acaChl1)
            </option><option value="proCap1">
                Rock hyrax Jul. 2008 (Broad/proCap1) (proCap1)
            </option><option value="colLiv2.1">
                Rock pigeon Dec 2017 (Cliv_2.1/colLiv2)
            </option><option value="colLiv1">
                Rock pigeon Feb 2013 (Cliv_1.0/colLiv1) (colLiv1)
            </option><option value="rodEnt13">
                Rodent Apr. 24. 2006 (UCSC Recon/rodEnt13) (rodEnt13)
            </option><option value="13462">
                Roseiflexus castenholzii DSM 13941 (13462)
            </option><option value="16190">
                Roseiflexus sp. RS-1 (16190)
            </option><option value="roseDeni_OCH_114">
                Roseobacter denitrificans OCh 114 (roseDeni_OCH_114)
            </option><option value="rubrXyla_DSM9941">
                Rubrobacter xylanophilus DSM 9941 (rubrXyla_DSM9941)
            </option><option value="Saccharomyces_cerevisiae_S288C_SGD2010">
                S. cerevisae str. S288C (Saccharomyces_cerevisiae_S288C_SGD2010)
            </option><option value="sacCer3">
                S. cerevisiae Apr. 2011 (SacCer_Apr2011/sacCer3) (sacCer3)
            </option><option value="sacCer2">
                S. cerevisiae June 2008 (SGD/sacCer2) (sacCer2)
            </option><option value="sacCer1">
                S. cerevisiae Oct. 2003 (SGD/sacCer1) (sacCer1)
            </option><option value="SL4.0">
                S. lycopersicum (Tomato) Feb 2019 (SL4.0)
            </option><option value="strPur1">
                S. purpuratus Apr. 2005 (Baylor 1.1/strPur1) (strPur1)
            </option><option value="strPur3">
                S. purpuratus Feb. 2009 (BCM-HGSC Spur_v2.5/strPur3) (strPur3)
            </option><option value="strPur4">
                S. purpuratus Jun. 2011 (BCM-HGSC Spur_v3.1/strPur4) (strPur4)
            </option><option value="strPur2">
                S. purpuratus Sep. 2006 (Baylor 2.1/strPur2) (strPur2)
            </option><option value="s.enterica_NC_016810.1">
                S.enterica_NC_016810.1
            </option><option value="sc1">
                SARS coronavirus Apr. 2003 (GenBank Apr. 14 '03/sc1) (sc1)
            </option><option value="wuhCor1">
                SARS-CoV-2 Jan. 2020 (NC_045512.2) (wuhCor1)
            </option><option value="NC_045512.2">
                SARS-CoV-2 isolate Wuhan-Hu-1, complete genome (NC_045512)
            </option><option value="saccDegr_2_40">
                Saccharophagus degradans 2-40 (saccDegr_2_40)
            </option><option value="saccEryt_NRRL_2338">
                Saccharopolyspora erythraea NRRL 2338 (saccEryt_NRRL_2338)
            </option><option value="Saccharum_spontaneum_v20190103">
                Saccharum spontaneum v20190103
            </option><option value="falChe1">
                Saker falcon Feb 2013 (F_cherrug_v1.0/falChe1) (falChe1)
            </option><option value="saliRube_DSM13855">
                Salinibacter ruber DSM 13855 (saliRube_DSM13855)
            </option><option value="17109">
                Salinispora arenicola CNS-205 (17109)
            </option><option value="saliTrop_CNB_440">
                Salinispora tropica CNB-440 (saliTrop_CNB_440)
            </option><option value="S. enterica (NC_016810.1)">
                Salmonella enterica (NC_016810.1)
            </option><option value="13030">
                Salmonella enterica subsp. arizonae serovar 62:z4,z23:-- (13030)
            </option><option value="9618">
                Salmonella enterica subsp. enterica serovar Choleraesuis str. SC-B67 (9618)
            </option><option value="salmEnte_PARATYPI_ATC">
                Salmonella enterica subsp. enterica serovar Paratyphi A str. ATCC 9150 (salmEnte_PARATYPI_ATC)
            </option><option value="27803">
                Salmonella enterica subsp. enterica serovar Paratyphi B str. SPB7 (27803)
            </option><option value="salmTyph">
                Salmonella enterica subsp. enterica serovar Typhi str. CT18 (salmTyph)
            </option><option value="salmTyph_TY2">
                Salmonella enterica subsp. enterica serovar Typhi str. Ty2 (salmTyph_TY2)
            </option><option value="241">
                Salmonella typhimurium LT2 (241)
            </option><option value="croPor0">
                Saltwater crocodile June 2011 (UCSC cPoro_v0.1c45/croPor0) (croPor0)
            </option><option value="araMac1">
                Scarlet Macaw Jun 2013 (SMACv1.1/araMac1) (araMac1)
            </option><option value="Schrenkiella_parvula_Thellungiella_v2.0_TpV84">
                Schrenkiella parvula Thellungiella v2.0 (TpV84)
            </option><option value="aplCal0">
                Sea hare Jul. 2013 (aplCal0/aplCal0) (aplCal0)
            </option><option value="aplCal1">
                Sea hare Sept. 2008 (Broad 2.0/aplCal1) (aplCal1)
            </option><option value="Selaginella_moellendorffii_JGI_v1.0">
                Selaginella moellendorffii JGI v1.0
            </option><option value="Selaginella_moellendorffii_NCBIr100">
                Selaginella moellendorffii NCBIr100
            </option><option value="17459">
                Serratia proteamaculans 568 (17459)
            </option><option value="Setaria_italica_JGI_8.3X_chromosome-scale_assembly_release_2.0_annotation_version_2.1">
                Setaria italica JGI 8.3X chromosome-scale assembly release 2.0, annotation version 2.1
            </option><option value="Setaria_italica_JGI_v2.2">
                Setaria italica JGI v2.2
            </option><option value="oviAri3">
                Sheep Aug. 2012 (ISGC Oar_v3.1/oviAri3) (oviAri3)
            </option><option value="oviAri1">
                Sheep Feb. 2010 (ISGC Ovis_aries_1.0/oviAri1) (oviAri1)
            </option><option value="oviAri4">
                Sheep Nov. 2015 (Oar_v4.0/oviAri4) (oviAri4)
            </option><option value="cypVar1">
                Sheepshead minnow Jul 2014 (C_variegatus-1.0/cypVar1) (cypVar1)
            </option><option value="shewAmaz">
                Shewanella amazonensis SB2B (shewAmaz)
            </option><option value="shewBalt">
                Shewanella baltica OS155 (shewBalt)
            </option><option value="17643">
                Shewanella baltica OS185 (17643)
            </option><option value="13389">
                Shewanella baltica OS195 (13389)
            </option><option value="shewDeni">
                Shewanella denitrificans OS217 (shewDeni)
            </option><option value="shewFrig">
                Shewanella frigidimarina NCIMB 400 (shewFrig)
            </option><option value="20241">
                Shewanella halifaxensis HAW-EB4 (20241)
            </option><option value="shewLoihPV4">
                Shewanella loihica PV-4 (shewLoihPV4)
            </option><option value="shewOnei">
                Shewanella oneidensis MR-1 (shewOnei)
            </option><option value="17415">
                Shewanella pealeana ATCC 700345 (17415)
            </option><option value="shewPutrCN32">
                Shewanella putrefaciens CN-32 (shewPutrCN32)
            </option><option value="18789">
                Shewanella sediminis HAW-EB3 (18789)
            </option><option value="shewANA3">
                Shewanella sp. ANA-3 (shewANA3)
            </option><option value="shewMR4">
                Shewanella sp. MR-4 (shewMR4)
            </option><option value="shewMR7">
                Shewanella sp. MR-7 (shewMR7)
            </option><option value="shewW318">
                Shewanella sp. W3-18-1 (shewW318)
            </option><option value="17455">
                Shewanella woodyi ATCC 51908 (17455)
            </option><option value="13146">
                Shigella boydii Sb227 (13146)
            </option><option value="13145">
                Shigella dysenteriae Sd197 (13145)
            </option><option value="408">
                Shigella flexneri 2a str. 2457T (408)
            </option><option value="shigFlex_2A">
                Shigella flexneri 2a str. 301 (shigFlex_2A)
            </option><option value="16375">
                Shigella flexneri 5 str. 8401 (16375)
            </option><option value="13151">
                Shigella sonnei Ss046 (13151)
            </option><option value="sorAra2">
                Shrew Aug. 2008 (Broad/sorAra2) (sorAra2)
            </option><option value="sorAra1">
                Shrew June 2006 (Broad/sorAra1) (sorAra1)
            </option><option value="siliPome_DSS_3">
                Silicibacter pomeroyi DSS-3 (siliPome_DSS_3)
            </option><option value="13040">
                Silicibacter sp. TM1040 (13040)
            </option><option value="Bombyx_mori_p50T_2.0">
                Silkworm (Bombyx_mori_p50T_2.0)
            </option><option value="simHuman">
                Simhuman Oct. 2011 (simHuman/simHuman) (simHuman)
            </option><option value="simHumanMammal">
                Simhumanmammal Oct. 2011 (simHumanMammal/simHumanMammal) (simHumanMammal)
            </option><option value="simHumanMam">
                Simmam Jan. 2011 (SimMamV1/simHumanMam) (simHumanMam)
            </option><option value="simHumanPrim">
                Simprim Jan. 2011 (SimPrimV1/simHumanPrim) (simHumanPrim)
            </option><option value="simRefMam">
                Simrefmam Jan. 2011 (SimRefMamV1/simRefMam) (simRefMam)
            </option><option value="16304">
                Sinorhizobium medicae WSM419 (16304)
            </option><option value="sinoMeli">
                Sinorhizobium meliloti 1021 (sinoMeli)
            </option><option value="choHof1">
                Sloth Jul. 2008 (Broad/choHof1) (choHof1)
            </option><option value="bioGla0">
                Snail Jun. 2013 (bioGla0/bioGla0) (bioGla0)
            </option><option value="sodaGlos_MORSITANS">
                Sodalis glossinidius str. 'morsitans' (sodaGlos_MORSITANS)
            </option><option value="Solanum_lycopersicum_ITAG_2.3">
                Solanum lycopersicum ITAG 2.3
            </option><option value="Solanum_lycopersicum_Sol_Genomics_itag2.4">
                Solanum lycopersicum Sol Genomics itag2.4
            </option><option value="Solanum_tuberosum_ITAG_001">
                Solanum tuberosum ITAG 001
            </option><option value="Solanum_tuberosum_JGI_v4.03">
                Solanum tuberosum JGI v4.03
            </option><option value="soliUsit_ELLIN6076">
                Solibacter usitatus Ellin6076 (soliUsit_ELLIN6076)
            </option><option value="28111">
                Sorangium cellulosum 'So ce 56' (28111)
            </option><option value="Sorghum_bicolor_JGI_1.4">
                Sorghum bicolor JGI 1.4
            </option><option value="Sorghum_bicolor_JGI_v3.1">
                Sorghum bicolor JGI v3.1
            </option><option value="xipMac1">
                Southern platyfish Jan 2012 (Xiphophorus_maculatus-4.4.2/xipMac1) (xipMac1)
            </option><option value="enhLutNer1">
                Southern sea otter Jun. 2019 (ASM641071v1/enhLutNer1) (enhLutNer1)
            </option><option value="colStr1">
                Speckled mousebird May 2014 (ASM69071v1/colStr1) (colStr1)
            </option><option value="phyCat1">
                Sperm whale Sep. 2013 (Physeter_macrocephalus-2.0.2/phyCat1) (phyCat1)
            </option><option value="17343">
                Sphingomonas wittichii RW1 (17343)
            </option><option value="sphiAlas_RB2256">
                Sphingopyxis alaskensis RB2256 (sphiAlas_RB2256)
            </option><option value="apaSpi1">
                Spiny softshell turtle May 2013 (ASM38561v1/apaSpi1) (apaSpi1)
            </option><option value="Spirodela_polyrhiza_JGI_v2.0">
                Spirodela polyrhiza JGI v2.0
            </option><option value="lepOcu1">
                Spotted gar Dec 2011 (LepOcu1/lepOcu1) (lepOcu1)
            </option><option value="speTri1">
                Squirrel Feb. 2008 (Broad/speTri1) (speTri1)
            </option><option value="speTri2">
                Squirrel Nov. 2011 (Broad/speTri2) (speTri2)
            </option><option value="saiBol1">
                Squirrel monkey Oct. 2011 (Broad/saiBol1) (saiBol1)
            </option><option value="63">
                Staphylococcus aureus RF122 (63)
            </option><option value="238">
                Staphylococcus aureus subsp. aureus COL (238)
            </option><option value="15758">
                Staphylococcus aureus subsp. aureus JH1 (15758)
            </option><option value="15757">
                Staphylococcus aureus subsp. aureus JH9 (15757)
            </option><option value="265">
                Staphylococcus aureus subsp. aureus MRSA252 (265)
            </option><option value="266">
                Staphylococcus aureus subsp. aureus MSSA476 (266)
            </option><option value="306">
                Staphylococcus aureus subsp. aureus MW2 (306)
            </option><option value="18509">
                Staphylococcus aureus subsp. aureus Mu3 (18509)
            </option><option value="stapAure_MU50">
                Staphylococcus aureus subsp. aureus Mu50 (stapAure_MU50)
            </option><option value="264">
                Staphylococcus aureus subsp. aureus N315 (264)
            </option><option value="237">
                Staphylococcus aureus subsp. aureus NCTC 8325 (237)
            </option><option value="16313">
                Staphylococcus aureus subsp. aureus USA300 (16313)
            </option><option value="19489">
                Staphylococcus aureus subsp. aureus USA300_TCH1516 (19489)
            </option><option value="18801">
                Staphylococcus aureus subsp. aureus str. Newman (18801)
            </option><option value="279">
                Staphylococcus epidermidis ATCC 12228 (279)
            </option><option value="64">
                Staphylococcus epidermidis RP62A (64)
            </option><option value="12508">
                Staphylococcus haemolyticus JCSC1435 (12508)
            </option><option value="15596">
                Staphylococcus saprophyticus subsp. saprophyticus ATCC 15305 (15596)
            </option><option value="stapMari1">
                Staphylothermus marinus F1 (stapMari1)
            </option><option value="conCri1">
                Star-nosed mole Mar 2012 (ConCri1.0/conCri1) (conCri1)
            </option><option value="nemVec1">
                Starlet sea anemone Jun. 2007 (JGI-PDF/nemVec1) (nemVec1)
            </option><option value="gasAcu1">
                Stickleback Feb. 2006 (Broad/gasAcu1) (gasAcu1)
            </option><option value="eidHel1">
                Straw colored fruit bat 13 Sep 2013 (ASM46528v1/eidHel1) (eidHel1)
            </option><option value="330">
                Streptococcus agalactiae 2603V/R (330)
            </option><option value="326">
                Streptococcus agalactiae A909 (326)
            </option><option value="334">
                Streptococcus agalactiae NEM316 (334)
            </option><option value="66">
                Streptococcus gordonii str. Challis substr. CH1 (66)
            </option><option value="333">
                Streptococcus mutans UA159 (333)
            </option><option value="16374">
                Streptococcus pneumoniae D39 (16374)
            </option><option value="28035">
                Streptococcus pneumoniae Hungary19A-6 (28035)
            </option><option value="278">
                Streptococcus pneumoniae R6 (278)
            </option><option value="277">
                Streptococcus pneumoniae TIGR4 (277)
            </option><option value="strePyog_M1_GAS">
                Streptococcus pyogenes M1 GAS (strePyog_M1_GAS)
            </option><option value="16364">
                Streptococcus pyogenes MGAS10270 (16364)
            </option><option value="12469">
                Streptococcus pyogenes MGAS10394 (12469)
            </option><option value="16366">
                Streptococcus pyogenes MGAS10750 (16366)
            </option><option value="16365">
                Streptococcus pyogenes MGAS2096 (16365)
            </option><option value="311">
                Streptococcus pyogenes MGAS315 (311)
            </option><option value="13888">
                Streptococcus pyogenes MGAS5005 (13888)
            </option><option value="13887">
                Streptococcus pyogenes MGAS6180 (13887)
            </option><option value="286">
                Streptococcus pyogenes MGAS8232 (286)
            </option><option value="16363">
                Streptococcus pyogenes MGAS9429 (16363)
            </option><option value="301">
                Streptococcus pyogenes SSI-1 (301)
            </option><option value="270">
                Streptococcus pyogenes str. Manfredo (270)
            </option><option value="13942">
                Streptococcus sanguinis SK36 (13942)
            </option><option value="17153">
                Streptococcus suis 05ZYH33 (17153)
            </option><option value="17155">
                Streptococcus suis 98HAH33 (17155)
            </option><option value="13163">
                Streptococcus thermophilus CNRZ1066 (13163)
            </option><option value="13773">
                Streptococcus thermophilus LMD-9 (13773)
            </option><option value="13162">
                Streptococcus thermophilus LMG 18311 (13162)
            </option><option value="189">
                Streptomyces avermitilis MA-4680 (189)
            </option><option value="streCoel">
                Streptomyces coelicolor A3(2) (streCoel)
            </option><option value="sudEbo1">
                Sudan ebolavirus Aug. 2014 (NC_006432.1/sudEbo1) (sudEbo1)
            </option><option value="sulfAcid1">
                Sulfolobus acidocaldarius DSM 639 (sulfAcid1)
            </option><option value="sulSol1">
                Sulfolobus solfataricus P2 (sulSol1)
            </option><option value="sulfToko1">
                Sulfolobus tokodaii str. 7 (sulfToko1)
            </option><option value="thioDeni_ATCC33889">
                Sulfurimonas denitrificans DSM 1251 (thioDeni_ATCC33889)
            </option><option value="18965">
                Sulfurovum sp. NBC37-1 (18965)
            </option><option value="eurHel1">
                Sunbittern May 2014 (ASM69077v1/eurHel1) (eurHel1)
            </option><option value="symbTher_IAM14863">
                Symbiobacterium thermophilum IAM 14863 (symbTher_IAM14863)
            </option><option value="13282">
                Synechococcus elongatus PCC 6301 (13282)
            </option><option value="10645">
                Synechococcus elongatus PCC 7942 (10645)
            </option><option value="12530">
                Synechococcus sp. CC9311 (12530)
            </option><option value="13643">
                Synechococcus sp. CC9605 (13643)
            </option><option value="13655">
                Synechococcus sp. CC9902 (13655)
            </option><option value="16252">
                Synechococcus sp. JA-2-3B'a(2-13) (16252)
            </option><option value="16251">
                Synechococcus sp. JA-3-3Ab (16251)
            </option><option value="28247">
                Synechococcus sp. PCC 7002 (28247)
            </option><option value="13654">
                Synechococcus sp. RCC307 (13654)
            </option><option value="13642">
                Synechococcus sp. WH 7803 (13642)
            </option><option value="syneSp_WH8102">
                Synechococcus sp. WH 8102 (syneSp_WH8102)
            </option><option value="synePCC6">
                Synechocystis sp. PCC 6803 (synePCC6)
            </option><option value="syntFuma_MPOB">
                Syntrophobacter fumaroxidans MPOB (syntFuma_MPOB)
            </option><option value="syntWolf_GOETTINGEN">
                Syntrophomonas wolfei subsp. wolfei str. Goettingen (syntWolf_GOETTINGEN)
            </option><option value="syntAcid_SB">
                Syntrophus aciditrophicus SB (syntAcid_SB)
            </option><option value="triCas2">
                T. castaneum Sep. 2005 (Baylor 2.0/triCas2) (triCas2)
            </option><option value="taiEbo1">
                Tai forest ebolavirus Aug. 2014 (NC_014372.1/taiEbo1) (taiEbo1)
            </option><option value="Tarenaya_hassleriana_RefSeq_v1">
                Tarenaya hassleriana RefSeq v1
            </option><option value="tarSyr1">
                Tarsier Aug. 2008 (Broad/tarSyr1) (tarSyr1)
            </option><option value="tarSyr2">
                Tarsier Sep. 2013 (Tarsius_syrichta-2.0.1/tarSyr2) (tarSyr2)
            </option><option value="sarHar1">
                Tasmanian devil Feb. 2011 (WTSI Devil_ref v7.0/sarHar1) (sarHar1)
            </option><option value="echTel1">
                Tenrec July 2005 (Broad/echTel1) (echTel1)
            </option><option value="echTel2">
                Tenrec Nov. 2012 (Broad/echTel2) (echTel2)
            </option><option value="tetNig1">
                Tetraodon Feb. 2004 (Genoscope 7/tetNig1) (tetNig1)
            </option><option value="tetNig2">
                Tetraodon Mar. 2007 (Genoscope 8.0/tetNig2) (tetNig2)
            </option><option value="Thellungiella_parvula_TpV84">
                Thellungiella parvula TpV84
            </option><option value="Theobroma_cacao_D._Gilbert_public_gene_set_8_Mar_2012_on_assembly_v1.1">
                Theobroma cacao D. Gilbert public gene set 8 Mar 2012 on assembly v1.1
            </option><option value="Tcacao_1.0">
                Theobroma cacao Dec 2010 (CIRAD/Theobroma cacao Belizian Criollo) (Tcacao_1.0)
            </option><option value="Theobroma_cacao_RefSeq_v1.1">
                Theobroma cacao RefSeq v1.1
            </option><option value="13901">
                Thermoanaerobacter pseudethanolicus ATCC 33223 (13901)
            </option><option value="16394">
                Thermoanaerobacter sp. X514 (16394)
            </option><option value="therTeng">
                Thermoanaerobacter tengcongensis MB4 (therTeng)
            </option><option value="therFusc_YX">
                Thermobifida fusca YX (therFusc_YX)
            </option><option value="therKoda1">
                Thermococcus kodakarensis KOD1 (therKoda1)
            </option><option value="therPend1">
                Thermofilum pendens Hrk 5 (therPend1)
            </option><option value="therAcid1">
                Thermoplasma acidophilum DSM 1728 (therAcid1)
            </option><option value="therVolc1">
                Thermoplasma volcanium GSS1 (therVolc1)
            </option><option value="17249">
                Thermosipho melanesiensis BI429 (17249)
            </option><option value="therElon">
                Thermosynechococcus elongatus BP-1 (therElon)
            </option><option value="15644">
                Thermotoga lettingae TMO (15644)
            </option><option value="therMari">
                Thermotoga maritima MSB8 (therMari)
            </option><option value="therPetr_RKU_1">
                Thermotoga petrophila RKU-1 (therPetr_RKU_1)
            </option><option value="19543">
                Thermotoga sp. RQ2 (19543)
            </option><option value="therTher_HB27">
                Thermus thermophilus HB27 (therTher_HB27)
            </option><option value="therTher_HB8">
                Thermus thermophilus HB8 (therTher_HB8)
            </option><option value="thioDeni_ATCC25259">
                Thiobacillus denitrificans ATCC 25259 (thioDeni_ATCC25259)
            </option><option value="thioCrun_XCL_2">
                Thiomicrospira crunogena XCL-2 (thioCrun_XCL_2)
            </option><option value="panHod1">
                Tibetan antelope May 2013 (PHO1.0/panHod1) (panHod1)
            </option><option value="nanPar1">
                Tibetan frog Mar. 2015 (BGI_ZX_2015/nanPar1) (nanPar1)
            </option><option value="pseHum1">
                Tibetan ground jay Jan 2013 (PseHum1.0/pseHum1) (pseHum1)
            </option><option value="sebNig1">
                Tiger rockfish 22 Oct-2013 (Snig1.0/sebNig1) (sebNig1)
            </option><option value="cynSem1">
                Tongue sole 28 Jan-2014 (Cse_v1.0/cynSem1) (cynSem1)
            </option><option value="tupBel1">
                Tree shrew Dec. 2006 (Broad/tupBel1) (tupBel1)
            </option><option value="4">
                Treponema denticola ATCC 35405 (4)
            </option><option value="trepPall">
                Treponema pallidum subsp. pallidum str. Nichols (trepPall)
            </option><option value="tricEryt_IMS101">
                Trichodesmium erythraeum IMS101 (tricEryt_IMS101)
            </option><option value="Trifolium_pratense_JGI_v2">
                Trifolium pratense JGI v2
            </option><option value="Triticum_aestivum_Ensembl_Plants_TGACv1">
                Triticum aestivum Ensembl Plants TGACv1
            </option><option value="Triticum_aestivum_IWGSC1.1">
                Triticum aestivum IWGSC1.1
            </option><option value="Triticum_turgidum_Svevo_v1.0">
                Triticum turgidum Svevo v1.0
            </option><option value="tropWhip_TW08_27">
                Tropheryma whipplei TW08/27 (tropWhip_TW08_27)
            </option><option value="95">
                Tropheryma whipplei str. Twist (95)
            </option><option value="TriTrypDB-37_TbruceiLister427">
                Trypanosoma brucei 427 Lister strain genome build 37
            </option><option value="melGal1">
                Turkey Dec. 2009 (TGC Turkey_2.01/melGal1) (melGal1)
            </option><option value="melGal5">
                Turkey Nov. 2014 (Turkey_5.0/melGal5) (melGal5)
            </option><option value="nanGal1">
                Upper Galilee mountains blind mole rat Jun 2014 (S.galili_v1.0/nanGal1) (nanGal1)
            </option><option value="19087">
                Ureaplasma parvum serovar 3 str. ATCC 27815 (19087)
            </option><option value="ureaUrea">
                Ureaplasma parvum serovar 3 str. ATCC 700970 (ureaUrea)
            </option><option value="Utricularia_gibba_CoGe_v4">
                Utricularia gibba CoGe v4
            </option><option value="vermEise_EF01_2">
                Verminephrobacter eiseniae EF01-2 (vermEise_EF01_2)
            </option><option value="vibrChol1">
                Vibrio cholerae O1 biovar eltor str. N16961 (vibrChol1)
            </option><option value="vibrChol_O395_1">
                Vibrio cholerae O395 (vibrChol_O395_1)
            </option><option value="vibrFisc_ES114_1">
                Vibrio fischeri ES114 (vibrFisc_ES114_1)
            </option><option value="19857">
                Vibrio harveyi ATCC BAA-1116 (19857)
            </option><option value="vibrPara1">
                Vibrio parahaemolyticus RIMD 2210633 (vibrPara1)
            </option><option value="vibrVuln_CMCP6_1">
                Vibrio vulnificus CMCP6 (vibrVuln_CMCP6_1)
            </option><option value="vibrVuln_YJ016_1">
                Vibrio vulnificus YJ016 (vibrVuln_YJ016_1)
            </option><option value="Vigna_radiata_var._radiata_LIS_v6">
                Vigna radiata var. radiata LIS v6
            </option><option value="Vitis_vinifera_JGI_12xMarch2010">
                Vitis vinifera JGI 12xMarch2010
            </option><option value="Vitis_vinifera_genescope_v1">
                Vitis vinifera genescope v1
            </option><option value="macEug1">
                Wallaby Nov. 2007 (Baylor 1.0/macEug1) (macEug1)
            </option><option value="macEug2">
                Wallaby Sep. 2009 (TWGS Meug_1.1/macEug2) (macEug2)
            </option><option value="bubBub1">
                Water buffalo Sep. 2013 (UMD_CASPUR_WB_2.0/bubBub1) (bubBub1)
            </option><option value="lepWed1">
                Weddell seal Mar 2013 (LepWed1.0/lepWed1) (lepWed1)
            </option><option value="GCF_019279795.1">
                West African lungfish Jul 2021 (Protopterus annectens, PAN1.0) (GCF_019279795.1)
            </option><option value="cerSim1">
                White rhinoceros May 2012 (CerSimSim1.0/cerSim1) (cerSim1)
            </option><option value="tinGut2">
                White throated tinamou Aug 2014 (ASM70537v2/tinGut2) (tinGut2)
            </option><option value="tinGut1">
                White throated tinamou Jun 2014 (ASM70537v1/tinGut1) (tinGut1)
            </option><option value="GCF_002102435.1">
                White-tailed deer Apr 2017 (Odocoileus virginianus) (GCF_002102435.1, Ovir.te_1.0)
            </option><option value="halAlb1">
                White-tailed eagle May 2014 (ASM69140v1/halAlb1) (halAlb1)
            </option><option value="phaLep1">
                White-tailed tropicbird May 2014 (ASM68728v1/phaLep1) (phaLep1)
            </option><option value="zonAlb1">
                White-throated sparrow Apr 2013 (ASM38545v1/zonAlb1) (zonAlb1)
            </option><option value="wiggBrev">
                Wigglesworthia glossinidia endosymbiont of Glossina brevipalpis (wiggBrev)
            </option><option value="bosMut1">
                Wild yak Oct 2011 (BosGru_v2.0/bosMut1) (bosMut1)
            </option><option value="wolbEndo_OF_DROSOPHIL">
                Wolbachia endosymbiont of Drosophila melanogaster (wolbEndo_OF_DROSOPHIL)
            </option><option value="12475">
                Wolbachia endosymbiont strain TRS of Brugia malayi (12475)
            </option><option value="woliSucc">
                Wolinella succinogenes DSM 1740 (woliSucc)
            </option><option value="xenTro2">
                X. tropicalis Aug. 2005 (JGI 4.1/xenTro2) (xenTro2)
            </option><option value="xenTro9">
                X. tropicalis Jul. 2016 (Xenopus_tropicalis_v9.1/xenTro9) (xenTro9)
            </option><option value="xenTro3">
                X. tropicalis Nov. 2009 (JGI 4.2/xenTro3) (xenTro3)
            </option><option value="xenTro10">
                X. tropicalis Nov. 2019 (UCB_Xtro_10.0/xenTro10) (xenTro10)
            </option><option value="xenTro1">
                X. tropicalis Oct. 2004 (JGI 3.0/xenTro1) (xenTro1)
            </option><option value="xenTro7">
                X. tropicalis Sep. 2012 (JGI 7.0/xenTro7) (xenTro7)
            </option><option value="15756">
                Xanthobacter autotrophicus Py2 (15756)
            </option><option value="297">
                Xanthomonas axonopodis pv. citri str. 306 (297)
            </option><option value="15">
                Xanthomonas campestris pv. campestris str. 8004 (15)
            </option><option value="xantCamp">
                Xanthomonas campestris pv. campestris str. ATCC 33913 (xantCamp)
            </option><option value="13649">
                Xanthomonas campestris pv. vesicatoria str. 85-10 (13649)
            </option><option value="12931">
                Xanthomonas oryzae pv. oryzae KACC10331 (12931)
            </option><option value="16297">
                Xanthomonas oryzae pv. oryzae MAFF 311018 (16297)
            </option><option value="Xanthomonas_oryzae_PXO99A">
                Xanthomonas oryzae pv. oryzae PXO99A (Xanthomonas_oryzae_PXO99A)
            </option><option value="xenTro9.1">
                Xenopus tropicalis (xenTro9.1)
            </option><option value="xyleFast">
                Xylella fastidiosa 9a5c (xyleFast)
            </option><option value="17823">
                Xylella fastidiosa M12 (17823)
            </option><option value="285">
                Xylella fastidiosa Temecula1 (285)
            </option><option value="lipVex1">
                Yangtze river dolphin 31 Jul 2013 (Lipotes_vexillifer_v1/lipVex1) (lipVex1)
            </option><option value="pteGut1">
                Yellow-throated sandgrouse Jun 2014 (ASM69924v1/pteGut1) (pteGut1)
            </option><option value="takFla1">
                Yellowbelly pufferfish May 2013 (version 1 of Takifugu flavidus genome/takFla1) (takFla1)
            </option><option value="190">
                Yersinia enterocolitica subsp. enterocolitica 8081 (190)
            </option><option value="16067">
                Yersinia pestis Angola (16067)
            </option><option value="16645">
                Yersinia pestis Antiqua (16645)
            </option><option value="yersPest_CO92">
                Yersinia pestis CO92 (yersPest_CO92)
            </option><option value="288">
                Yersinia pestis KIM (288)
            </option><option value="16646">
                Yersinia pestis Nepal516 (16646)
            </option><option value="16700">
                Yersinia pestis Pestoides F (16700)
            </option><option value="10638">
                Yersinia pestis biovar Microtus str. 91001 (10638)
            </option><option value="16070">
                Yersinia pseudotuberculosis IP 31758 (16070)
            </option><option value="12950">
                Yersinia pseudotuberculosis IP 32953 (12950)
            </option><option value="28743">
                Yersinia pseudotuberculosis YPIII (28743)
            </option><option value="zaiEbo1">
                Zaire ebolavirus Aug. 2014 (NC_002549.1/zaiEbo1) (zaiEbo1)
            </option><option value="Zea_mays_B73_AGP_v4.0">
                Zea mays B73 AGP v4.0
            </option><option value="Zea_mays_PH207_v1.0">
                Zea mays PH207 v1.0
            </option><option value="mayZeb1">
                Zebra Mbuna Mar 2012 (MetZeb1.1/mayZeb1) (mayZeb1)
            </option><option value="taeGut2">
                Zebra finch Feb. 2013 (WashU taeGut324/taeGut2) (taeGut2)
            </option><option value="taeGut1">
                Zebra finch Jul. 2008 (WUGSC 3.2.4/taeGut1) (taeGut1)
            </option><option value="danRer11_wo_alt">
                Zebrafish (GRCz11/danRer11) without alt contigs
            </option><option value="danRer6">
                Zebrafish Dec. 2008 (Zv8/danRer6) (danRer6)
            </option><option value="danRer7">
                Zebrafish Jul. 2010 (Zv9/danRer7) (danRer7)
            </option><option value="danRer5">
                Zebrafish July 2007 (Zv7/danRer5) (danRer5)
            </option><option value="danRer4">
                Zebrafish Mar. 2006 (Zv6/danRer4) (danRer4)
            </option><option value="danRer3">
                Zebrafish May 2005 (Zv5/danRer3) (danRer3)
            </option><option value="danRer11">
                Zebrafish May 2017 (GRCz11/danRer11) (danRer11)
            </option><option value="danRer10">
                Zebrafish Sep. 2014 (GRCz10/danRer10) (danRer10)
            </option><option value="Zostera_marina_JGI_v2.2">
                Zostera marina JGI v2.2
            </option><option value="Zoysia_japonica_ssp._nagirizaki_Zoysia_Genome_Database_v1.1">
                Zoysia japonica ssp. nagirizaki Zoysia Genome Database v1.1
            </option><option value="zymoMobi_ZM4">
                Zymomonas mobilis subsp. mobilis ZM4 (zymoMobi_ZM4)
            </option><option value="canFam3">
                canFam3
            </option><option value="homPan20">
                chimp/human Jun. 2008 (UCSC Recon/homPan20) (homPan20)
            </option><option value="hg19Haps">
                hg19Haplotypes Feb. 2009 (GRCh37/hg19Haps) (hg19Haps)
            </option><option value="mm9_ERCC">
                mm9 with ERCC spike-in
            </option><option value="homIni20">
                orang/human Jun. 2008 (UCSC Recon/homIni20) (homIni20)
            </option><option value="pUC18">
                pUC18 Plasmid (pUC18)
            </option><option value="phiX">
                phiX174 (phiX)
            </option><option value="tarIhg38">
                tarInv Dec. 2013 (GRCh38Tar/tarIhg38) (tarIhg38)
            </option><option value="uncuMeth_RCI">
                uncultured methanogenic archaeon RC-I (uncuMeth_RCI)
            </option></select></div> <div class="upload-buttons"><button id="btn-close" title="Close" type="button" class="btn ui-button-default btn-secondary">
            Close
        </button> <button id="btn-reset" title="Reset" type="button" disabled="disabled" class="btn ui-button-default btn-secondary disabled">
            Reset
        </button> <button id="btn-stop" title="Pause" type="button" disabled="disabled" class="btn ui-button-default btn-secondary disabled">
            Pause
        </button> <button id="btn-build" title="Build" type="button" disabled="disabled" class="btn ui-button-default btn-secondary disabled">
            Build
        </button> <button id="btn-start" title="Start" type="button" disabled="disabled" class="btn ui-button-default btn-secondary disabled">
            Start
        </button> <button id="btn-new" title="Paste/Fetch data" type="button" class="btn ui-button-default btn-secondary"><span class="fa fa-edit"></span>Paste/Fetch data
        </button> <!----> <button id="btn-local" title="Choose local files" type="button" class="btn ui-button-default btn-secondary"><span class="fa fa-laptop"></span>Choose local files
        </button></div></div></div> <div role="tabpanel" id="rule-based" aria-hidden="true" aria-labelledby="tab-title-link-rule-based" class="tab-pane" style="display: none;"><div class="upload-view-default"><div class="upload-top"><div class="upload-top-info">Tabular source data to extract collection files and metadata from</div></div> <div class="upload-box" style="height: 335px;"><span class="float-left" style="width: 25%; display: inline; height: 100%;"><div class="upload-rule-option"><div class="upload-rule-option-title">Upload data as</div> <div class="rule-data-type"><div class="select2-container upload-footer-selection" id="s2id_autogen31"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-32">Datasets</span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen32" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-32" id="s2id_autogen32"><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen32_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-32" id="s2id_autogen32_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-32">   </ul></div></div><select tabindex="-1" title="" class="select2-offscreen"><option value="datasets">Datasets</option> <option value="collections">Collection(s)</option></select></div></div> <div class="upload-rule-option"><div class="upload-rule-option-title">Load tabular data from</div> <div class="rule-select-type"><div class="select2-container upload-footer-selection" id="s2id_autogen33"><a href="javascript:void(0)" class="select2-choice" tabindex="-1">   <span class="select2-chosen" id="select2-chosen-34">Pasted Table</span><abbr class="select2-search-choice-close"></abbr>   <span class="select2-arrow" role="presentation"><b role="presentation"></b></span></a><label for="s2id_autogen34" class="select2-offscreen"></label><input class="select2-focusser select2-offscreen" type="text" aria-haspopup="true" role="button" aria-labelledby="select2-chosen-34" id="s2id_autogen34"><div class="select2-drop select2-display-none select2-with-searchbox">   <div class="select2-search">       <label for="s2id_autogen34_search" class="select2-offscreen"></label>       <input type="text" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" class="select2-input" role="combobox" aria-expanded="true" aria-autocomplete="list" aria-owns="select2-results-34" id="s2id_autogen34_search" placeholder="">   </div>   <ul class="select2-results" role="listbox" id="select2-results-34">   </ul></div></div><select tabindex="-1" title="" class="select2-offscreen"><option value="paste">Pasted Table</option> <option value="dataset">History Dataset</option> <!----> <option value="remote_files">Remote Files Directory</option></select></div></div> <!----></span> <span style="display: inline; float: right; width: 75%; height: 300px;"><textarea class="upload-rule-source-content form-control" style="height: 100%;"></textarea></span></div> <div class="clear"></div> <div class="upload-buttons"><button id="btn-close" title="Close" type="button" class="btn ui-button-default btn-secondary">
            Close
        </button> <button id="btn-build" title="Build" type="button" disabled="disabled" class="btn ui-button-default btn-secondary disabled">
            Build
        </button> <button id="btn-reset" title="Reset" type="button" disabled="disabled" class="btn ui-button-default btn-secondary disabled">
            Reset
        </button></div></div></div><!----></div></div></div><!----></div><!----></div></div><!----></div> <!----> <span data-v-14171f27="" id="drag-ghost" class="py-2 px-3 rounded"><svg data-v-14171f27="" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="paper-plane" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="mr-1 svg-inline--fa fa-paper-plane"><path data-v-14171f27="" fill="currentColor" d="M476 3.2L12.5 270.6c-18.1 10.4-15.8 35.6 2.2 43.2L121 358.4l287.3-253.2c5.5-4.9 13.3 2.6 8.6 8.3L176 407v80.5c0 23.6 28.5 32.9 42.5 15.8L282 426l124.6 52.2c14.2 6 30.4-2.9 33-18.2l72-432C515 7.8 493.3-6.8 476 3.2z" class=""></path></svg> <span data-v-14171f27="" class="text-break text-center font-weight-bold">
    Draggable
</span></span></div>
        </main>

        
    <noscript>
        <div class="overlay overlay-background noscript-overlay">
            <div>
                <h3 class="title">Javascript Required for Galaxy</h3>
                <div>
                    The Galaxy analysis interface requires a browser with Javascript enabled.<br>
                    Please enable Javascript and refresh this page.
                </div>
            </div>
        </div>
    </noscript>

        
    <script src="/static/dist/libs.bundled.js?v=1703115796000" type="text/javascript"></script>
<script src="/static/dist/analysis.bundled.js?v=1703115796000" type="text/javascript"></script>

        

    <script type="text/javascript">
        console.debug("Initializing javascript application:", "app");

        // js-app.mako
        var options = {"root": "/", "user": {"id": null, "total_disk_usage": 0.0, "nice_total_disk_usage": "0 bytes", "quota_percent": null}, "config": {"post_user_logout_href": "/root/login?is_logout_redirect=true", "enable_unique_workflow_defaults": false, "chunk_upload_size": 0, "require_login": false, "allow_user_creation": true, "select_type_workflow_threshold": -1, "datatypes_disable_auto": false, "object_store_ids_allowing_selection": [], "version_major": "23.1", "message_box_visible": false, "message_box_class": "warning", "user_library_import_dir_available": false, "fixed_delegated_auth": false, "plausible_domain": null, "enable_notification_system": true, "simplified_workflow_run_ui": "prefer", "enable_quotas": true, "citations_export_message_html": "When writing up your analysis, remember to include all references that should be cited in order to completely describe your work. Also, please remember to <a href=\"https://galaxyproject.org/citing-galaxy\">cite Galaxy<\/a>.", "tool_training_recommendations_api_url": "https://training.galaxyproject.org/training-material/api/top-tools.json", "visualizations_visible": true, "message_box_content": null, "simplified_workflow_run_ui_job_cache": "off", "matomo_site_id": null, "ga_code": null, "has_user_tool_filters": true, "oidc": {"elixir": {"icon": "https://lifescience-ri.eu/fileadmin/lifescience-ri/media/Images/button-login-small.png"}, "keycloak": {"icon": "https://galaxyproject.eu/assets/media/DataPLANT_logo.png"}}, "registration_warning_message": "Please register only one account - we provide this service free of charge and have limited computational resources. Multi-accounts are tracked and will be subjected to account termination and data deletion.", "version_extra": null, "quota_url": "https://galaxyproject.org/support/account-quotas/", "logo_url": "/", "matomo_server": null, "simplified_workflow_run_ui_target_history": "prefer_current", "welcome_url": "/static/welcome.html", "markdown_to_pdf_available": true, "allow_user_dataset_purge": true, "server_mail_configured": true, "object_store_allows_id_selection": false, "admin_tool_recommendations_path": "/opt/galaxy/config/tool_recommendations_overwrite.yml", "allow_user_impersonation": true, "enable_oidc": true, "brand": "Europe", "toolbox_auto_sort": true, "screencasts_url": "https://www.youtube.com/c/galaxyproject", "lims_doc_url": "https://usegalaxy.org/u/rkchak/p/sts", "python": [3, 11], "quota_source_labels": [], "upload_from_form_button": "always-on", "enable_account_interface": true, "ftp_upload_site": "ftp://ftp.usegalaxy.eu", "wiki_url": "https://galaxyproject.org/", "enable_tool_source_display": true, "enable_beta_markdown_export": true, "use_remote_user": null, "server_startttime": 1703115796000, "cookie_domain": null, "logo_src": "/static/favicon.svg", "support_url": "https://galaxyproject.org/support/", "helpsite_url": "https://help.galaxyproject.org/c/usegalaxy-eu-support", "tool_training_recommendations_link": "https://training.galaxyproject.org/training-material/by-tool/{training_tool_identifier}.html", "panel_views": {"default": {"id": "default", "model_class": "type", "name": "Full Tool Panel", "description": "Galaxy's fully configured toolbox panel with all sections, tools, and configured workflows loaded.", "view_type": "default", "searchable": true}, "ontology:edam_operations": {"id": "ontology:edam_operations", "model_class": "EdamToolPanelView", "name": "EDAM Operations", "description": "Tools are grouped using annotated EDAM operation information (if available).", "view_type": "ontology", "searchable": true}, "ontology:edam_topics": {"id": "ontology:edam_topics", "model_class": "EdamToolPanelView", "name": "EDAM Topics", "description": "Tools are grouped using annotated EDAM topic information (if available).", "view_type": "ontology", "searchable": true}}, "plausible_server": null, "default_locale": "auto", "prefer_custos_login": false, "logo_src_secondary": null, "enable_beacon_integration": true, "release_doc_base_url": "https://docs.galaxyproject.org/en/release_", "expose_user_email": false, "themes": {"europe": {"--masthead-color": "#003399"}, "lightblue": {"--masthead-color": "#384E77", "--masthead-text-color": "white", "--masthead-text-hover": "#E6F9AF", "--masthead-text-active": "white", "--masthead-link-color": "transparent", "--masthead-link-hover": "transparent", "--masthead-link-active": "#18314F"}, "pride": {"--masthead-color": "linear-gradient(120deg,\n  #3c476d 0px 200px,\n  #fdda0f 202px 210px,\n  #fff 212px 220px,\n  #f4b0c9 222px 230px,\n  #7ccee6 232px 240px,\n  #93540c 242px 250px,\n  #000 252px 260px,\n  transparent 262px),\nlinear-gradient(270deg,\n  #3c476d 0px 120px,\n  #3c476d00 120px),\nlinear-gradient(180deg,\n  #FE0000 16.66%,\n  #FD8C00 16.66% 33.32%,\n  #FFE500 33.32% 49.98%,\n  #119F0B 49.98% 66.64%,\n  #0644B3 66.64% 83.3%,\n  #C22EDC 83.3%)\n", "--masthead-text-color": "white", "--masthead-text-hover": "gold", "--masthead-text-active": "white", "--masthead-link-color": "#3c476d", "--masthead-link-hover": "#323a53", "--masthead-link-active": "#6170a6"}, "smoky": {"--masthead-color": "#0C0F0A", "--masthead-text-color": "white", "--masthead-text-hover": "#FBFF12", "--masthead-text-active": "white", "--masthead-link-color": "transparent", "--masthead-link-hover": "transparent", "--masthead-link-active": "#FF206E"}}, "version_minor": "4.dev0", "show_welcome_with_login": true, "aws_estimate": true, "citation_url": "https://galaxyproject.org/citing-galaxy", "mailing_join_addr": null, "file_sources_configured": true, "terms_url": "https://usegalaxy.eu/terms", "single_user": false, "enable_celery_tasks": true, "inactivity_box_content": "Your account has not been activated yet.  Feel free to browse around and see what's available, but you won't be able to upload data or run jobs until you have verified your email address.", "remote_user_logout_href": null, "welcome_directory": "plugins/welcome_page/new_user/static/topics/", "nginx_upload_path": "/_upload", "is_admin_user": false, "tool_training_recommendations": true, "topk_recommendations": 20, "interactivetools_enable": true, "enable_tool_recommendations": true, "overwrite_model_recommendations": false, "default_panel_view": "default", "tool_recommendation_model_path": "https://github.com/galaxyproject/galaxy-test-data/raw/master/tool_recommendation_model_v_0.2.hdf5", "active_view": "analysis", "enable_webhooks": true, "show_inactivity_warning": null, "tool_shed_urls": ["https://toolshed.g2.bx.psu.edu/"], "tool_dynamic_configs": ["/opt/galaxy/mutable-config/shed_tool_conf.xml"], "stored_workflow_menu_entries": []}, "params": {"tool_id": "toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.74+galaxy0", "version": "latest"}, "session_csrf_token": "c3ae71f65be7de5514087340a696ba67"};
        var bootstrapped = {};

        config.set({
            options: options,
            bootstrapped: bootstrapped,
            form_input_auto_focus: false
        });
    </script>


    
        <script type="text/javascript">

            var sentry = {};
                sentry.sentry_dsn_public = "https://b9bc8b8fadab4eefa8622428d6920fef@sentry.galaxyproject.org/7"

            config.set({
                sentry: sentry
            });

        </script>


    







<ul class="vakata-context"></ul><div id="jstree-marker" style="display: none;">&nbsp;</div><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><span role="status" aria-live="polite" class="select2-hidden-accessible"></span><div id="tool-source-viewer___BV_modal_outer_" style="position: absolute; z-index: 1040;"><div id="tool-source-viewer" role="dialog" aria-labelledby="tool-source-viewer___BV_modal_title_" aria-describedby="tool-source-viewer___BV_modal_body_" class="modal fade show" aria-modal="true" style="display: block; padding-left: 0px;"><div class="modal-dialog modal-lg"><span tabindex="0"></span><div id="tool-source-viewer___BV_modal_content_" tabindex="-1" class="modal-content"><header id="tool-source-viewer___BV_modal_header_" class="modal-header"><h5 id="tool-source-viewer___BV_modal_title_" class="modal-title">Tool Source for toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.74+galaxy0</h5><button type="button" aria-label="Close" class="close">×</button></header><div id="tool-source-viewer___BV_modal_body_" class="modal-body"><pre class="language-xml" id="toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.74+galaxy0" tabindex="0"><code class="language-xml"><span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>tool</span> <span class="token attr-name">id</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc<span class="token punctuation">"</span></span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>FastQC<span class="token punctuation">"</span></span> <span class="token attr-name">version</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>0.74+galaxy0<span class="token punctuation">"</span></span><span class="token punctuation">&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>description</span><span class="token punctuation">&gt;</span></span>
        Read Quality reports
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>description</span><span class="token punctuation">&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>xrefs</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>xref</span> <span class="token attr-name">type</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>bio.tools<span class="token punctuation">"</span></span><span class="token punctuation">&gt;</span></span>
            fastqc
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>xref</span><span class="token punctuation">&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>xrefs</span><span class="token punctuation">&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>requirements</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>requirement</span> <span class="token attr-name">type</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>package<span class="token punctuation">"</span></span> <span class="token attr-name">version</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>0.12.1<span class="token punctuation">"</span></span><span class="token punctuation">&gt;</span></span>
            fastqc
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>requirement</span><span class="token punctuation">&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>requirements</span><span class="token punctuation">&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>stdio</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>exit_code</span> <span class="token attr-name">range</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>1:<span class="token punctuation">"</span></span> <span class="token attr-name">level</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fatal<span class="token punctuation">"</span></span> <span class="token attr-name">description</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>FastQC returned non zero exit code<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>exit_code</span> <span class="token attr-name">range</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>:-1<span class="token punctuation">"</span></span> <span class="token attr-name">level</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fatal<span class="token punctuation">"</span></span> <span class="token attr-name">description</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>FastQC returned non zero exit code<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>regex</span> <span class="token attr-name">match</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>There is insufficient memory for the Java Runtime Environment<span class="token punctuation">"</span></span> <span class="token attr-name">source</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>stdout<span class="token punctuation">"</span></span> <span class="token attr-name">level</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fatal_oom<span class="token punctuation">"</span></span> <span class="token attr-name">description</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>Out of memory error occurred<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>stdio</span><span class="token punctuation">&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>command</span><span class="token punctuation">&gt;</span></span>
        #import re
        #set input_name = re.sub('[^\w\-\s]', '_', str($input_file.element_identifier))

        #if $input_file.ext.endswith('.gz'):
            #set input_file_sl = $input_name + '.gz'
        #elif $input_file.ext.endswith('.bz2'):
            #set input_file_sl = $input_name + '.bz2'
        #else
            #set input_file_sl = $input_name
        #end if

        #if 'bam' in $input_file.ext:
            #set format = 'bam'
        #elif 'sam' in $input_file.ext:
            #set format = 'sam'
        #else
            #set format = 'fastq'
        #end if

        ln -s '${input_file}' '${input_file_sl}' <span class="token entity named-entity" title="&amp;">&amp;amp;</span><span class="token entity named-entity" title="&amp;">&amp;amp;</span>
        mkdir -p '${html_file.files_path}' <span class="token entity named-entity" title="&amp;">&amp;amp;</span><span class="token entity named-entity" title="&amp;">&amp;amp;</span>
        fastqc
            --outdir '${html_file.files_path}'
            #if $contaminants.dataset and str($contaminants) <span class="token entity named-entity" title=">">&amp;gt;</span> ''
                --contaminants '${contaminants}'
            #end if

            #if $adapters.dataset and str($adapters) <span class="token entity named-entity" title=">">&amp;gt;</span> ''
                --adapters '${adapters}'
            #end if

            #if $limits.dataset and str($limits) <span class="token entity named-entity" title=">">&amp;gt;</span> ''
                --limits '${limits}'
            #end if
            --threads \${GALAXY_SLOTS:-2}
            --quiet
            --extract
            #if $min_length:
                --min_length $min_length
            #end if
            $nogroup
            --kmers $kmers
            -f '${format}'
            '${input_file_sl}'

        <span class="token entity named-entity" title="&amp;">&amp;amp;</span><span class="token entity named-entity" title="&amp;">&amp;amp;</span> cp '${html_file.files_path}'/*/fastqc_data.txt output.txt
        <span class="token entity named-entity" title="&amp;">&amp;amp;</span><span class="token entity named-entity" title="&amp;">&amp;amp;</span> cp '${html_file.files_path}'/*\.html output.html
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>command</span><span class="token punctuation">&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>inputs</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">format</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastq,fastq.gz,fastq.bz2,bam,sam<span class="token punctuation">"</span></span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>input_file<span class="token punctuation">"</span></span> <span class="token attr-name">type</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>data<span class="token punctuation">"</span></span> <span class="token attr-name">label</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>Raw read data from your current history<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>contaminants<span class="token punctuation">"</span></span> <span class="token attr-name">type</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>data<span class="token punctuation">"</span></span> <span class="token attr-name">format</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>tabular<span class="token punctuation">"</span></span> <span class="token attr-name">optional</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>true<span class="token punctuation">"</span></span> <span class="token attr-name">label</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>Contaminant list<span class="token punctuation">"</span></span> <span class="token attr-name">help</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>tab delimited file with 2 columns: name and sequence.  For example: Illumina Small RNA RT Primer CAAGCAGAAGACGGCATACGA<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">argument</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>--adapters<span class="token punctuation">"</span></span> <span class="token attr-name">type</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>data<span class="token punctuation">"</span></span> <span class="token attr-name">format</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>tabular<span class="token punctuation">"</span></span> <span class="token attr-name">optional</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>true<span class="token punctuation">"</span></span> <span class="token attr-name">label</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>Adapter list<span class="token punctuation">"</span></span> <span class="token attr-name">help</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>List of adapters adapter sequences which will be explicity searched against the library. It should be a tab-delimited file with 2 columns: name and sequence.<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>limits<span class="token punctuation">"</span></span> <span class="token attr-name">type</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>data<span class="token punctuation">"</span></span> <span class="token attr-name">format</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>txt<span class="token punctuation">"</span></span> <span class="token attr-name">optional</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>true<span class="token punctuation">"</span></span> <span class="token attr-name">label</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>Submodule and Limit specifing file<span class="token punctuation">"</span></span> <span class="token attr-name">help</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>a file that specifies which submodules are to be executed (default=all) and also specifies the thresholds for the each submodules warning parameter<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">argument</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>--nogroup<span class="token punctuation">"</span></span> <span class="token attr-name">type</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>boolean<span class="token punctuation">"</span></span> <span class="token attr-name">truevalue</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>--nogroup<span class="token punctuation">"</span></span> <span class="token attr-name">falsevalue</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span><span class="token punctuation">"</span></span> <span class="token attr-name">checked</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>False<span class="token punctuation">"</span></span> <span class="token attr-name">label</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>Disable grouping of bases for reads <span class="token entity named-entity" title=">">&amp;gt;</span>50bp<span class="token punctuation">"</span></span> <span class="token attr-name">help</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>Using this option will cause fastqc to crash and burn if you use it on really long reads, and your plots may end up a ridiculous size. You have been warned!<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">argument</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>--min_length<span class="token punctuation">"</span></span> <span class="token attr-name">type</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>integer<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span><span class="token punctuation">"</span></span> <span class="token attr-name">optional</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>true<span class="token punctuation">"</span></span> <span class="token attr-name">label</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>Lower limit on the length of the sequence to be shown in the report<span class="token punctuation">"</span></span> <span class="token attr-name">help</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span> As long as you set this to a value greater or equal to your longest read length then this will be the sequence length used to create your read groups.  This can be useful for making directly comaparable statistics from datasets with somewhat variable read lengths.<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">argument</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>--kmers<span class="token punctuation">"</span></span> <span class="token attr-name">type</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>integer<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>7<span class="token punctuation">"</span></span> <span class="token attr-name">min</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>2<span class="token punctuation">"</span></span> <span class="token attr-name">max</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>10<span class="token punctuation">"</span></span> <span class="token attr-name">label</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>Length of Kmer to look for<span class="token punctuation">"</span></span> <span class="token attr-name">help</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>Note: the Kmer test is disabled and needs to be enabled using a custom Submodule and limits file<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>inputs</span><span class="token punctuation">&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>outputs</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>data</span> <span class="token attr-name">format</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html<span class="token punctuation">"</span></span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html_file<span class="token punctuation">"</span></span> <span class="token attr-name">from_work_dir</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>output.html<span class="token punctuation">"</span></span> <span class="token attr-name">label</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>${tool.name} on ${on_string}: Webpage<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>data</span> <span class="token attr-name">format</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>txt<span class="token punctuation">"</span></span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>text_file<span class="token punctuation">"</span></span> <span class="token attr-name">from_work_dir</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>output.txt<span class="token punctuation">"</span></span> <span class="token attr-name">label</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>${tool.name} on ${on_string}: RawData<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>outputs</span><span class="token punctuation">&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>tests</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>test</span><span class="token punctuation">&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>input_file<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>1000trimmed.fastq<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_report.html<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html<span class="token punctuation">"</span></span> <span class="token attr-name">lines_diff</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>2<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>text_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_data.txt<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>txt<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>test</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>test</span><span class="token punctuation">&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>input_file<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>1000trimmed.fastq<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>contaminants<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_contaminants.txt<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>tabular<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_report_contaminants.html<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html<span class="token punctuation">"</span></span> <span class="token attr-name">lines_diff</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>2<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>text_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_data_contaminants.txt<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>txt<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>test</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>test</span><span class="token punctuation">&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>input_file<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>1000trimmed.fastq<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>adapters<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_adapters.txt<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>tabular<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_report_adapters.html<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html<span class="token punctuation">"</span></span> <span class="token attr-name">lines_diff</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>2<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>text_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_data_adapters.txt<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>txt<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>test</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>test</span><span class="token punctuation">&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>input_file<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>1000trimmed.fastq<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>limits<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_customlimits.txt<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>txt<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_report_customlimits.html<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html<span class="token punctuation">"</span></span> <span class="token attr-name">lines_diff</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>2<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>text_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_data_customlimits.txt<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>txt<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>test</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>test</span><span class="token punctuation">&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>input_file<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>1000trimmed.fastq<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastq<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>kmers<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>3<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>limits<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_customlimits.txt<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>txt<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_report_kmer.html<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html<span class="token punctuation">"</span></span> <span class="token attr-name">lines_diff</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>2<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>text_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_data_kmer.txt<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>txt<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>assert_command</span><span class="token punctuation">&gt;</span></span>
                <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>has_text</span> <span class="token attr-name">text</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>--kmers 3<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>assert_command</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>test</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>test</span><span class="token punctuation">&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>input_file<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>1000trimmed.fastq<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>min_length<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>108<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_report_min_length.html<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html<span class="token punctuation">"</span></span> <span class="token attr-name">lines_diff</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>2<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>text_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_data_min_length.txt<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>txt<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>test</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>test</span><span class="token punctuation">&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>input_file<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>1000trimmed.fastq<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastq<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>nogroup<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>--nogroup<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_report_nogroup.html<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html<span class="token punctuation">"</span></span> <span class="token attr-name">lines_diff</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>2<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>text_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_data_nogroup.txt<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>txt<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>assert_command</span><span class="token punctuation">&gt;</span></span>
                <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>has_text</span> <span class="token attr-name">text</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>--nogroup<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>assert_command</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>test</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>test</span><span class="token punctuation">&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>param</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>input_file<span class="token punctuation">"</span></span> <span class="token attr-name">value</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>hisat_output_1.bam<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>bam<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_report_hisat.html<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>html<span class="token punctuation">"</span></span> <span class="token attr-name">lines_diff</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>2<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>output</span> <span class="token attr-name">name</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>text_file<span class="token punctuation">"</span></span> <span class="token attr-name">file</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>fastqc_data_hisat.txt<span class="token punctuation">"</span></span> <span class="token attr-name">ftype</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>txt<span class="token punctuation">"</span></span><span class="token punctuation">/&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>test</span><span class="token punctuation">&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>tests</span><span class="token punctuation">&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>help</span><span class="token punctuation">&gt;</span></span>
        .. class:: infomark

**Purpose**

FastQC aims to provide a simple way to do some quality control checks on raw
sequence data coming from high throughput sequencing pipelines.
It provides a set of analyses which you can use to get a quick
impression of whether your data has any problems of
which you should be aware before doing any further analysis.

The main functions of FastQC are:

- Import of data from BAM, SAM or FastQ/FastQ.gz files (any variant),
- Providing a quick overview to tell you in which areas there may be problems
- Summary graphs and tables to quickly assess your data
- Export of results to an HTML based permanent report
- Offline operation to allow automated generation of reports without running the interactive application

-----

.. class:: infomark

**FastQC**

This is a Galaxy wrapper. It merely exposes the external package FastQC_ which is documented at FastQC_
Kindly acknowledge it as well as this tool if you use it.
FastQC incorporates the Picard-tools_ libraries for SAM/BAM processing.

The contaminants file parameter was borrowed from the independently developed
fastqcwrapper contributed to the Galaxy Community Tool Shed by J. Johnson.
Adaption to version 0.11.2 by T. McGowan.

-----

.. class:: infomark

**Inputs and outputs**

FastQC_ is the best place to look for documentation - it's very good.
A summary follows below for those in a tearing hurry.

This wrapper will accept a Galaxy fastq, fastq.gz, sam or bam as the input read file to check.
It will also take an optional file containing a list of contaminants information, in the form of
a tab-delimited file with 2 columns, name and sequence. As another option the tool takes a custom
limits.txt file that allows setting the warning thresholds for the different modules and also specifies
which modules to include in the output.

The tool produces a basic text and a HTML output file that contain all of the results, including the following:

- Basic Statistics
- Per base sequence quality
- Per sequence quality scores
- Per base sequence content
- Per base GC content
- Per sequence GC content
- Per base N content
- Sequence Length Distribution
- Sequence Duplication Levels
- Overrepresented sequences
- Kmer Content

All except Basic Statistics and Overrepresented sequences are plots.
 .. _FastQC: http://www.bioinformatics.babraham.ac.uk/projects/fastqc/
 .. _Picard-tools: https://broadinstitute.github.io/picard/
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>help</span><span class="token punctuation">&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>citations</span><span class="token punctuation">&gt;</span></span>
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>citation</span> <span class="token attr-name">type</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>bibtex<span class="token punctuation">"</span></span><span class="token punctuation">&gt;</span></span>
            @unpublished{andrews_s,
            author = {Andrews, S.},
            keywords = {bioinformatics, ngs, qc},
            priority = {2},
            title = {{FastQC A Quality Control tool for High Throughput Sequence Data}},
            url = {http://www.bioinformatics.babraham.ac.uk/projects/fastqc/}
        }
        <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>citation</span><span class="token punctuation">&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>citations</span><span class="token punctuation">&gt;</span></span>
<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>tool</span><span class="token punctuation">&gt;</span></span></code></pre></div><footer id="tool-source-viewer___BV_modal_footer_" class="modal-footer"><!----><button type="button" class="btn btn-primary">Close</button></footer></div><span tabindex="0"></span></div></div><div id="tool-source-viewer___BV_modal_backdrop_" class="modal-backdrop"></div></div></body></html>